{
  "nbformat": 4,
  "nbformat_minor": 0,
  "metadata": {
    "colab": {
      "provenance": [],
      "machine_shape": "hm",
      "gpuType": "A100",
      "authorship_tag": "ABX9TyN9iQcskJrRIIgpJEvD5j6r",
      "include_colab_link": true
    },
    "kernelspec": {
      "name": "python3",
      "display_name": "Python 3"
    },
    "language_info": {
      "name": "python"
    },
    "accelerator": "GPU",
    "widgets": {
      "application/vnd.jupyter.widget-state+json": {
        "cdae6812242242ddb245ddf6f12e9a8a": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "HBoxModel",
          "model_module_version": "1.5.0",
          "state": {
            "_dom_classes": [],
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "HBoxModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/controls",
            "_view_module_version": "1.5.0",
            "_view_name": "HBoxView",
            "box_style": "",
            "children": [
              "IPY_MODEL_5a9781b987fe4b47b6f3587e75446e24",
              "IPY_MODEL_f315e6ffd6f7479f9b413414b3aa66a6",
              "IPY_MODEL_b0bc553bdbf44bed8ee7a4b487451b4b"
            ],
            "layout": "IPY_MODEL_56fdedbfd10a4b59a095308407366643"
          }
        },
        "5a9781b987fe4b47b6f3587e75446e24": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "HTMLModel",
          "model_module_version": "1.5.0",
          "state": {
            "_dom_classes": [],
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "HTMLModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/controls",
            "_view_module_version": "1.5.0",
            "_view_name": "HTMLView",
            "description": "",
            "description_tooltip": null,
            "layout": "IPY_MODEL_554a89277d3043d68d9b06fa38245c24",
            "placeholder": "​",
            "style": "IPY_MODEL_35a600674e0f446eb9b353c2c6481581",
            "value": "Loading checkpoint shards: 100%"
          }
        },
        "f315e6ffd6f7479f9b413414b3aa66a6": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "FloatProgressModel",
          "model_module_version": "1.5.0",
          "state": {
            "_dom_classes": [],
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "FloatProgressModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/controls",
            "_view_module_version": "1.5.0",
            "_view_name": "ProgressView",
            "bar_style": "success",
            "description": "",
            "description_tooltip": null,
            "layout": "IPY_MODEL_e8080aeb4ec9462e94dc9b7035a9bf04",
            "max": 2,
            "min": 0,
            "orientation": "horizontal",
            "style": "IPY_MODEL_0165c1007b224108ad38f6d7408a4036",
            "value": 2
          }
        },
        "b0bc553bdbf44bed8ee7a4b487451b4b": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "HTMLModel",
          "model_module_version": "1.5.0",
          "state": {
            "_dom_classes": [],
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "HTMLModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/controls",
            "_view_module_version": "1.5.0",
            "_view_name": "HTMLView",
            "description": "",
            "description_tooltip": null,
            "layout": "IPY_MODEL_0bab08a1f34a477a9022f4da601bd177",
            "placeholder": "​",
            "style": "IPY_MODEL_a2952d2dc2c34d3a8810053c96466b1f",
            "value": " 2/2 [00:02&lt;00:00,  1.22s/it]"
          }
        },
        "56fdedbfd10a4b59a095308407366643": {
          "model_module": "@jupyter-widgets/base",
          "model_name": "LayoutModel",
          "model_module_version": "1.2.0",
          "state": {
            "_model_module": "@jupyter-widgets/base",
            "_model_module_version": "1.2.0",
            "_model_name": "LayoutModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "LayoutView",
            "align_content": null,
            "align_items": null,
            "align_self": null,
            "border": null,
            "bottom": null,
            "display": null,
            "flex": null,
            "flex_flow": null,
            "grid_area": null,
            "grid_auto_columns": null,
            "grid_auto_flow": null,
            "grid_auto_rows": null,
            "grid_column": null,
            "grid_gap": null,
            "grid_row": null,
            "grid_template_areas": null,
            "grid_template_columns": null,
            "grid_template_rows": null,
            "height": null,
            "justify_content": null,
            "justify_items": null,
            "left": null,
            "margin": null,
            "max_height": null,
            "max_width": null,
            "min_height": null,
            "min_width": null,
            "object_fit": null,
            "object_position": null,
            "order": null,
            "overflow": null,
            "overflow_x": null,
            "overflow_y": null,
            "padding": null,
            "right": null,
            "top": null,
            "visibility": null,
            "width": null
          }
        },
        "554a89277d3043d68d9b06fa38245c24": {
          "model_module": "@jupyter-widgets/base",
          "model_name": "LayoutModel",
          "model_module_version": "1.2.0",
          "state": {
            "_model_module": "@jupyter-widgets/base",
            "_model_module_version": "1.2.0",
            "_model_name": "LayoutModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "LayoutView",
            "align_content": null,
            "align_items": null,
            "align_self": null,
            "border": null,
            "bottom": null,
            "display": null,
            "flex": null,
            "flex_flow": null,
            "grid_area": null,
            "grid_auto_columns": null,
            "grid_auto_flow": null,
            "grid_auto_rows": null,
            "grid_column": null,
            "grid_gap": null,
            "grid_row": null,
            "grid_template_areas": null,
            "grid_template_columns": null,
            "grid_template_rows": null,
            "height": null,
            "justify_content": null,
            "justify_items": null,
            "left": null,
            "margin": null,
            "max_height": null,
            "max_width": null,
            "min_height": null,
            "min_width": null,
            "object_fit": null,
            "object_position": null,
            "order": null,
            "overflow": null,
            "overflow_x": null,
            "overflow_y": null,
            "padding": null,
            "right": null,
            "top": null,
            "visibility": null,
            "width": null
          }
        },
        "35a600674e0f446eb9b353c2c6481581": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "DescriptionStyleModel",
          "model_module_version": "1.5.0",
          "state": {
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "DescriptionStyleModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "StyleView",
            "description_width": ""
          }
        },
        "e8080aeb4ec9462e94dc9b7035a9bf04": {
          "model_module": "@jupyter-widgets/base",
          "model_name": "LayoutModel",
          "model_module_version": "1.2.0",
          "state": {
            "_model_module": "@jupyter-widgets/base",
            "_model_module_version": "1.2.0",
            "_model_name": "LayoutModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "LayoutView",
            "align_content": null,
            "align_items": null,
            "align_self": null,
            "border": null,
            "bottom": null,
            "display": null,
            "flex": null,
            "flex_flow": null,
            "grid_area": null,
            "grid_auto_columns": null,
            "grid_auto_flow": null,
            "grid_auto_rows": null,
            "grid_column": null,
            "grid_gap": null,
            "grid_row": null,
            "grid_template_areas": null,
            "grid_template_columns": null,
            "grid_template_rows": null,
            "height": null,
            "justify_content": null,
            "justify_items": null,
            "left": null,
            "margin": null,
            "max_height": null,
            "max_width": null,
            "min_height": null,
            "min_width": null,
            "object_fit": null,
            "object_position": null,
            "order": null,
            "overflow": null,
            "overflow_x": null,
            "overflow_y": null,
            "padding": null,
            "right": null,
            "top": null,
            "visibility": null,
            "width": null
          }
        },
        "0165c1007b224108ad38f6d7408a4036": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "ProgressStyleModel",
          "model_module_version": "1.5.0",
          "state": {
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "ProgressStyleModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "StyleView",
            "bar_color": null,
            "description_width": ""
          }
        },
        "0bab08a1f34a477a9022f4da601bd177": {
          "model_module": "@jupyter-widgets/base",
          "model_name": "LayoutModel",
          "model_module_version": "1.2.0",
          "state": {
            "_model_module": "@jupyter-widgets/base",
            "_model_module_version": "1.2.0",
            "_model_name": "LayoutModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "LayoutView",
            "align_content": null,
            "align_items": null,
            "align_self": null,
            "border": null,
            "bottom": null,
            "display": null,
            "flex": null,
            "flex_flow": null,
            "grid_area": null,
            "grid_auto_columns": null,
            "grid_auto_flow": null,
            "grid_auto_rows": null,
            "grid_column": null,
            "grid_gap": null,
            "grid_row": null,
            "grid_template_areas": null,
            "grid_template_columns": null,
            "grid_template_rows": null,
            "height": null,
            "justify_content": null,
            "justify_items": null,
            "left": null,
            "margin": null,
            "max_height": null,
            "max_width": null,
            "min_height": null,
            "min_width": null,
            "object_fit": null,
            "object_position": null,
            "order": null,
            "overflow": null,
            "overflow_x": null,
            "overflow_y": null,
            "padding": null,
            "right": null,
            "top": null,
            "visibility": null,
            "width": null
          }
        },
        "a2952d2dc2c34d3a8810053c96466b1f": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "DescriptionStyleModel",
          "model_module_version": "1.5.0",
          "state": {
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "DescriptionStyleModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "StyleView",
            "description_width": ""
          }
        }
      }
    }
  },
  "cells": [
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "view-in-github",
        "colab_type": "text"
      },
      "source": [
        "<a href=\"https://colab.research.google.com/github/summerdevlin46/recruitment_test/blob/main/bsc_recruitment_test.ipynb\" target=\"_parent\"><img src=\"https://colab.research.google.com/assets/colab-badge.svg\" alt=\"Open In Colab\"/></a>"
      ]
    },
    {
      "cell_type": "markdown",
      "source": [
        "#BSC Recruitment Test"
      ],
      "metadata": {
        "id": "8sNEi4utMGhe"
      }
    },
    {
      "cell_type": "markdown",
      "source": [
        "## Setting up environment"
      ],
      "metadata": {
        "id": "zlSrG5wZMWWn"
      }
    },
    {
      "cell_type": "code",
      "execution_count": 20,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "kNWWWiRBVmUj",
        "outputId": "afc09e56-0bba-4cc7-b67a-d3cca0adcd6f"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "fatal: destination path 'recruitment_test' already exists and is not an empty directory.\n"
          ]
        }
      ],
      "source": [
        "  !git clone https://github.com/summerdevlin46/recruitment_test"
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "from huggingface_hub import login\n",
        "from google.colab import userdata\n",
        "\n",
        "access_token_read = userdata.get('HF_TOKEN')\n",
        "\n",
        "login(token = access_token_read)"
      ],
      "metadata": {
        "id": "8c4XQGhMgj9m"
      },
      "execution_count": 22,
      "outputs": []
    },
    {
      "cell_type": "code",
      "source": [
        "import json\n",
        "\n",
        "# Load the JSON data\n",
        "with open(\"recruitment_test/data/articles.json\", \"r\") as file:\n",
        "    data = json.load(file)"
      ],
      "metadata": {
        "id": "1ZvXUlzQXQIf"
      },
      "execution_count": 23,
      "outputs": []
    },
    {
      "cell_type": "markdown",
      "source": [
        "## Preprocessing JSON Files"
      ],
      "metadata": {
        "id": "DpKhhWPoMgTf"
      }
    },
    {
      "cell_type": "markdown",
      "source": [
        "I removed any entries that contained null for each value, which left us with 36 entries containing usable data."
      ],
      "metadata": {
        "id": "bHwv2jmDMgRc"
      }
    },
    {
      "cell_type": "code",
      "source": [
        "# Filter out the entries with all null values except for pmcid\n",
        "filtered_articles = [entry for entry in data if any(value is not None for key, value in entry.items() if key != 'pmcid')]\n",
        "\n",
        "# Print filtered data\n",
        "print(json.dumps(filtered_articles, indent=4))\n",
        "\n",
        "# Number of entries after filtering\n",
        "#print(\"Number of entries after filtering:\", len(filtered_articles))\n"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "collapsed": true,
        "id": "dYIyr73Hji_-",
        "outputId": "402ddb49-fee6-47c7-e21d-b066ae5aad02"
      },
      "execution_count": 24,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "[\n",
            "    {\n",
            "        \"pmcid\": \"PMC10167034\",\n",
            "        \"abstract\": null,\n",
            "        \"intro\": \"1 Introduction Sepsis remains a leading cause of death in intensive care units (ICU), with a mortality rate of approximately 15-25% ( 1 \\u2013 3 ). Vascular dysfunction, a feared complication of sepsis, can result in acute respiratory distress syndrome (ARDS), acute renal failure (ARF), and even mortality ( 4 , 5 ). However, existing prognostic tools are hindered by complexity and biases leading to imperfect predictive capabilities ( 6 \\u2013 9 ). Further, the heterogeneity of sepsis at the individual patient level has also hindered therapeutic advances in the field ( 10 ). Decades of investigative efforts in the absence of sepsis endotyping have yielded no conclusively effective therapies to date. Understanding the heterogeneity of sepsis and subgrouping septic patients for personalized treatment may hold the promise of targeted sepsis therapy. Extracellular vesicles (EVs) are membranous nanovesicles formed by outward budding of the plasma membrane, apoptosis, or exocytosis of multivesicular bodies, and are released from almost every cell type ( 11 \\u2013 13 ). EVs express molecules on their membranes that are specific for their parental cells and contain functional molecules including protein and miRNA ( 11 ). Circulating EVs encompass EVs from platelets, endothelial cells (ECs), neutrophils, monocytes, lymphocytes, erythrocytes, and their precursors, and play a critical role in maintaining vascular homeostasis in physiological conditions ( 13 , 14 ). The presence of circulating EVs and their role as biomarkers and mediators in sepsis pathophysiology have been explored ( 11 ). Recent studies suggest that EV caspase-1 activity can induce EC injury and that sepsis-induced pyroptosis of monocytes and ECs can release caspase-1 containing EVs ( 15 \\u2013 18 ). Conversely, EV miR-126 has been shown to stabilize the endothelium in sepsis while modulating sepsis-related organ failure ( 19 \\u2013 22 ). However, the potential role of circulating total or cell-specific EVs in septic patients and their association with sepsis outcomes such as organ failure and mortality are still largely unknown. We isolated circulating total and cell-specific EVs from septic patients and analyzed the association of EV function/contents with sepsis outcomes. Our data demonstrated that EV caspase-1 activity and miR-126 levels are associated with the development of organ failures and mortality and may serve as novel biomarkers and/or therapeutic targets for sepsis.\",\n",
            "        \"method\": \"2 Materials and methods 2.1 Human donors Septic (n=96) and non-septic critically ill (n=12) hospitalized patients were enrolled during 2015-2018 at the Medical University of South Carolina. We screened all admissions for the presence of sepsis based on the American College of Chest Physicians/Society of Critical Care Medicine consensus definition in use at the time of enrollment initiation ( 23 ). Additional inclusion criteria included age \\u226518 years and admission into the hospital within the previous 24 hours. We excluded immunocompromised patients as defined by immunosuppressive medication use, leukopenia, current hematologic malignancy, and history of stem cell transplant. Additional exclusion criteria included: transfer in from other hospitals if subjects had spent > 24 hours in an ICU at the time of screening and care goals consistent with comfort measures only at the time of screening. Healthy control subjects (n=45) were recruited through local advertising. Informed consent for study participation and publication of results was obtained from all research subjects or their legally authorized representatives. Organ failures, including the occurrence of ARDS or ARF, were adjudicated by study investigators using the Berlin definition and kidney disease: Improving Global Outcomes (KDIGO) criteria, respectively ( 24 , 25 ). ICU and hospital length of stay, discharge destination and vital status were captured. Basic demographics were recorded from healthy controls. These studies were in accordance with the 1964 Helsinki declaration and its later amendments or comparable ethical standards and approved by the Institutional Review Board at the Medical University of South Carolina. The entire human cohort demographics are listed in Table 1 . Table 1 Demographic characteristics of cohort. N=153 Healthy Control Septic patients without ARDS Septic patients with ARDS non-septic critical illnesses Subject size 45 75 21 12 Sex, Females (%), 34 (75.6%) 35 (46.7%) 8 (38.1%) 7 (58.3%) Age \\u00b1 SEM (years) 33.6 \\u00b1 2.0 58.5 \\u00b1 1.9 58.7 \\u00b1 3.8 52.3 \\u00b1 6.3 Race (%) White 31 (68.9%) 44 (58.7%) 14 (66.7%), 9 (75%) Black 10 (22.2%) 26 (34.7%) 7 (33.3%) 3 (25%) Other 2 (8.8%) 5 (6.6%) 0 0 Mortality rate (%) N/A 9.3% 57.1% 25% SEM, standard error of mean; N/A, not applicable. 2.2 Human plasma collection Plasma samples were collected from septic subjects and non-septic critically ill subjects on day 1, and 3 after hospital admission, and from healthy controls. Peripheral blood was collected in tubes containing disodium EDTA, and then centrifuged at 2,000 x g for 15 minutes at 4\\u00b0C to obtain the plasma. The collected plasma samples were aliquoted and stored at -80\\u00b0C for further analysis. 2.3 Extracellular vesicle isolation and characterization Circulating total EVs were isolated from frozen human plasma as described ( 26 , 27 ). In brief, 1 ml of the collected plasma samples were centrifuged at 2,000 x g for 30 minutes at 4\\u00b0C, the supernatant was further centrifuged at 12,000 x g for 30 minutes at 4\\u00b0C. Then the supernatant was ultracentrifuged at 100,000 x g for 2 hours at 4\\u00b0C to pellet the EVs. EVs pellet was washed once in PBS and ultracentrifuged again at 100,000 x g for 2 hours at 4\\u00b0C. The pelleted EVs were then suspended in PBS and kept at -80\\u00b0C for further analysis. Circulating EVs derived from monocytes or endothelial cells were isolated as described by Goetzl et al. ( 28 ). Briefly, circulating total EVs were precipitated with ExoQuick (SBI). Monocyte cell-specific EVs were enriched by sequential immunoprecipitation with biotinylated anti-CD14 antibody (Thermo Fisher Scientific), and CD14 + EVs were considered monocyte-derived EVs. Endothelial cell-specific EVs were enriched by sequential immunoprecipitation with the biotinylated anti-CD31 (Thermo Fisher Scientific) and anti-CD146 antibodies (Novus Biologicals). CD31 + /CD146 + EVs were considered endothelial cell-derived EVs. EVs were isolated from 1 ml of human plasma, fixed with 4% paraformaldehyde and their morphology was detected using transmission electron microscopy in the Electron Microscopy Facility at the University of Connecticut Health Center. The size distribution and the total number of EVs were analyzed by nanoparticle tracking analysis (NTA) with ZetaView PMX 120 according to the manufacturer\\u2019s instructions (Particle Metrix, Meerbusch, Germany). The presence of specific EV markers (CD9, CD63 and CD81) and absence of common contaminants (Calnexin and GM130) were determined by western blot. 2.4 Cell culture Human Lung Microvascular Endothelial Cells (HMVEC) were purchased from Lonza and cultured in EBM-2 supplemented with EGM-2 MV SingleQuot (Lonza, Allendale, NJ, USA) containing 5% FBS and 1% penicillin/streptomycin in a CO 2 incubator with 5% CO 2 . 2.5 Transendothelial electrical resistance assay 1.5 x 10 3 cells (HMVEC) were seeded into each well of 8 well arrays (8W10E+, Applied Biophysics) and cultured for 24 hours. Total circulating EVs at 1.5 x 10 10 /ml were added into each well and transendothelial electrical resistance (TEER) assays were performed using an ECIS instrument (Applied Biophysics). At 2 hours after EV addition, the resistance of HMVECs was compared to that of its starting point (%), recorded and analyzed. 2.6 Caspase-1 activity Caspase-1 activity in circulating total EVs (1.5 x 10 10 particles), monocyte-derived EVs (5.5 x 10 9 particles) and EC-derived EVs (3 x 10 7 ) was measured by a Caspase-Glo\\u00ae1 Inflammasome Assay kit according to the instructions (Promega). The catalytically active caspase-1 was detected by caspase cleavage of selective caspase-1 substrate (Z-WEHD). In brief, we transferred the same amount of EVs in 100 \\u00b5l PBS into the corresponding wells of a new white 96-well plate. We then added 100 \\u00b5l of Caspase-Glo\\u00ae 1 reagent and incubated the plate at room temperature for 1 hour before measuring luminescence using a plate-reader. 2.7 Real-Time RT-PCR Total RNA was extracted from EC-specific EVs using the miRNeasy Serum/Plasma kit (Qiagen, Germantown, MD) according to the manufacturer\\u2019s instructions. For miRNA expression, the RNA was reverse transcribed using a QIAGEN miRNA Reverse Transcription Kit (Qiagen). An identical amount of Caenorhabditis elegans miR-39 was spiked into each sample to allow for the normalization of Cq values. 2.8 Western blot EVs were lysed with ice-cold RIPA lysis buffer (Abcam) containing protease and phosphatase inhibitors (Cell Signaling). All lysed samples were kept on ice for 30 min and centrifuged for 10 min at 4 \\u00b0C at 10,000\\u00d7 g . The cell lysates were collected, and protein concentrations were measured using a DC protein assay (Bio-Rad). A total of 20 \\u03bcg of EV protein was loaded into each lane for western blot. All exosome-specific primary antibodies including the anti-CD9, anti-CD63 and the anti-CD81 antibodies (System Biosciences) were used at 1:1000 dilution, and exosome-validated peroxidase-labeled secondary antibody was used at 1: 10000 dilution. Anti-calnexin, anti-GM130, anti-beta-actin and anti-GAPDH antibodies were from Cell Signaling. The immunoreactive protein bands were visualized by ECL detection kit (GE Healthcare). The images of the Western blots are shown in Supplementary Figures 1 and 2 . 2.9 Next-generation sequencing EVs (approximately 10 12 particles) were isolated from 500 \\u03bcl of plasma taken on days 1 and 3 from subjects with sepsis or non-septic critical illnesses using qEV columns (Izon, Medford, MA). RNAs were isolated from plasma EVs using miRNeasy Serum/Plasma kit (Qiagen, Germantown, MD) according to the manufacturer\\u2019s instructions. Next-generation sequencing was performed at MUSC next-generation sequencing facility using standard protocols. 2.10 Data analysis Statistical significance for caspase-1 activity or miR-126 levels in healthy controls, sepsis with ARDS, and non-ARDS septic patients were evaluated using analysis of variance (ANOVA). Associations between caspase-1 activity with the occurrence of mortality, ARDS and ARF among septic patients were evaluated using Wilcoxon rank sum tests. Associations between expression levels of EV miR-126-3p and miR-126-5p levels with mortality, ARDS, and ARF among septic patients were evaluated using a linear mixed model approach. Models included fixed effects for patient status (e.g. died vs. survived), day, and the interaction between patient status and day and a random subject effect to account for measures collected on the same patients. Differences between patient status over time were evaluated using linear contrasts from the model. P-values were Bonferroni adjusted for all comparisons considered. Analyses were conducted using GraphPad Prism and SAS software. A value of p <0.05 was considered statistically significant.\",\n",
            "        \"subjects\": null,\n",
            "        \"results\": \"3 Results 3.1 Characterization of EVs We first characterized total, EC-derived or monocyte-derived EVs collected from plasma samples. Transmission electron microscopy was used to analyze EVs in all groups, and they exhibited typical morphology ( Figure 1A ). As shown in Figure 1B , the concentration of isolated total, EC-derived or monocyte-derived EVs was approximately 5.2 \\u00d7 10 12 particles/ml, 4.5 \\u00d7 10 9 particles/ml and 3.5 \\u00d7 10 11 particles/ml respectively. All EVs showed a similar size distribution profile with isolated particles within the 30\\u2013120 nm range ( Figure 1B ). Moreover, we observed that the total number of isolated EVs, including those derived from monocytes or endothelial cells, was similar between healthy controls and patients (data not shown). Western blot analysis validated the presence of EV markers CD9, CD63 and CD81 in total, monocyte- and EC-derived EVs, while common contaminants calnexin and GM130 were absent ( Figure 1C ). Additionally, both \\u03b2-actin and GAPDH were not detected in total, monocyte-, or EC-derived EVs ( Figure 1C ). Figure 1 EV characterization. Analysis of EV morphology using transmission electron microscopy (A) . Scale bar: 100 nm. EV size distribution and concentration were measured by nanoparticle tracking analysis (NTA) with ZetaView (B) . The concentration of isolated total, monocyte-derived or EC-derived EVs was approximately 5.2 \\u00d7 10 12 particles/ml, 3.5 \\u00d7 10 11 particles/ml and 4.5 \\u00d7 10 9 particles/ml, respectively. All EVs showed a similar size distribution profile with isolated particles within the 30\\u2013120 nm range. EV markers (CD9, CD63 and CD81) and common contaminants (calnexin and GM130) in total, monocyte- and EC-derived EVs were detected by western blot (C) . EV, Extracellular vesicles. Cell Lysate: human brain pericytes. 3.2 The impact of circulating total EVs on vascular permeability is associated with the development of ARDS Since vascular injury plays a pivotal role in sepsis-related organ failures ( 4 , 5 ), we first assessed the impact of circulating total EVs on vascular permeability in vitro using HMVECs and analyzed this impact\\u2019s association with sepsis outcomes including mortality, ARDS and ARF. We found that circulating EVs from healthy controls did not induce an increase in vascular permeability (n=7, Figure 2A ). EVs from some septic patients did not induce vascular permeability (n=4, Figure 2B ) while EVs of the same concentration from other septic patients induced increased vascular permeability (n=3, Figure 2C ). Interestingly, total circulating EVs from sepsis survivors (n=73) versus non-survivors (n=19) showed no significantly different impact on transendothelial resistance ( Figure 2D ). In contrast, a significantly lower transendothelial resistance was induced by EVs from septic patients with ARDS (n=21) compared to those without ARDS (n=71) (p=0.02, Figure 2E ). Circulating total EVs from septic patients with ARF (n=40) compared to those without ARF (n=52) showed no difference in their impact on transendothelial resistance ( Figure 2F ). These data demonstrate that ARDS is associated with the presence of circulating EVs which induce EC permeability. Figure 2 Circulating EV-induced vascular permeability is associated with ARDS patients. Circulating total EVs were isolated from healthy controls (N=7) and septic patients (N=96). Among septic patients, 21 patients developed ARDS and 75 did not. These EVs at 1.5 x 10 10 EVs/ml were used in TEER assays which was measured in HMVEC for 6-24 hours by an ECIS instrument. Representative results from 7 healthy controls (A) and 7 septic patients (B, C) are shown. The association of TEER level with mortality (D) , ARDS development (E) and acute renal failure (F) were analyzed using Wilcoxon rank sum tests. Resistance levels are expressed as the level of impedance at 2 hours divided by the level of impedance prior to addition of EVs. Boxes present the 25 th , 50 th , and 75 th percentiles of resistance in each group. Whiskers extend 1.5 +/- the inner quartile range (i.e. difference between the 75 th and 25 th percentile). Grey points are the observed resistance values (% of original) for subjects within each group. EV, Extracellular vesicles; ARDS, acute respiratory distress syndrome; TEER, transendothelial electrical resistance; HMVEC, Human Lung Microvascular Endothelial Cell. 3.3 Circulating total EV caspase-1 activity is associated with the development of ARDS To explore the relationship between EV content and sepsis outcomes, we determined caspase-1 activity in total circulating EVs from healthy controls and septic patients and analyzed its association with sepsis outcomes including mortality, ARDS and ARF. We found that caspase-1 activity was significantly increased in EVs from septic patients compared to those from healthy controls (p<0.05) and that among septic patients, EVs from patients who experienced ARDS (n=21) contained significantly increased caspase-1 activity than EVs from patients without ARDS (n=63) (p<0.05, Figure 3A ). Circulating total EV caspase-1 activity from sepsis survivors (n=64) and non-survivors (n=17) showed no significant difference ( Figure 3B ). In contrast, a significantly higher total EV caspase-1 activity from septic patients with ARDS (n=21) compared to non-ARDS septic patients (n=60) was observed (p<0.05, Figure 3C ). Circulating total EV caspase-1 activity from septic patients with ARF (n=34) and without ARF (n=47) was not significantly different ( Figure 3D ). These data demonstrate that circulating total EV caspase-1 activity is associated with ARDS development during sepsis, but not with mortality or ARF of sepsis. Figure 3 Circulating total EV caspase-1 activity is associated with ARDS development. Total plasma EVs were isolated from 45 healthy controls and 84 septic patients (21 ARDS and 63 non-ARDS), and caspase-1 activity (A) was determined. * p < 0.05 compared to healthy controls, # p<0.05 compared to non-ARDS septic patients. N=21-63/group. Statistical significance was determined by analysis of variance (ANOVA). The association of caspase-1 activity with mortality (B) , ARDS development (C) and acute renal failure (D) were analyzed using Wilcoxon rank sum tests. Boxes present the 25 th , 50 th , and 75 th percentiles of caspase-1 activity in each group. Whiskers extend 1.5 +/- the inner quartile range (i.e. difference between the 75 th and 25 th percentile). Grey points are the caspase-1 activity values for subjects within each group. EV, Extracellular vesicles; ARDS, acute respiratory distress syndrome. 3.4 Circulating monocyte-derived EV caspase-1 activity is associated with the development of ARDS To explore the relationship between cell-specific EV content and sepsis outcomes, we examined caspase-1 activity in circulating monocyte-derived EVs. We found that caspase-1 activity was significantly increased in monocyte-derived EVs from septic patients with ARDS compared to those from healthy controls and septic patients without ARDS (N=10; p<0.05, Figure 4A ). Monocyte-derived EV caspase-1 activity from sepsis survivors (n=14) and non-survivors (n=6) showed no significant difference ( Figure 4B ). In contrast, a significantly higher monocyte-derived EV caspase-1 activity from septic patients with ARDS (n=10) compared to non-ARDS septic patients (n=10) was observed (p<0.05, Figure 4C ). Additionally, monocyte-derived EV caspase-1 activity from septic patients with (n=9) and without (n=11) ARF was comparable ( Figure 4D ). These data demonstrate that monocyte-derived EV caspase-1 activity is associated with ARDS development during sepsis, but not with mortality or ARF of sepsis Figure 4 Circulating monocyte-derived EV caspase-1 activity is associated with ARDS development. Circulating monocyte-derived EVs were isolated from 10 healthy controls and 20 septic patients (10 ARDS and 10 non-ARDS), and caspase-1 activity (A) was determined. * p < 0.05 compared to healthy controls, # p<0.05 compared to non-ARDS septic patients. N=10/group. Statistical significance was determined by analysis of variance (ANOVA). The association of caspase-1 activity with mortality (B) , ARDS development (C) and acute renal failure (D) were analyzed using Wilcoxon rank sum tests for the analysis. Boxes present the 25 th , 50 th , and 75 th percentiles of caspase-1 activity in each group. Whiskers extend 1.5 +/- the inner quartile range (i.e. difference between the 75 th and 25 th percentile). Grey points are the caspase-1 activity values for subjects within each group. EV, Extracellular vesicles; ARDS, acute respiratory distress syndrome. 3.5 Circulating EC-derived EV caspase-1 activity is associated with the development of ARDS We further examined caspase-1 activity in circulating EC-derived EVs and analyzed its association with sepsis outcomes including mortality, ARDS and ARF. We found that caspase-1 activity was significantly increased in EC-derived EVs from ARDS patients (n=14) compared to those from healthy controls (n=11) and non-ARDS septic patients (n=14; p<0.05, Figure 5A ). Interestingly, EC-derived EV caspase-1 activity from sepsis survivors (n=19) and non-survivors (n=9) showed no significant difference ( Figure 5B ). In contrast, a significantly higher EC-derived EV caspase-1 activity from septic patients with ARDS (n=14) compared to non-ARDS septic patients (n=14) was observed (p<0.01, Figure 5C ). Additionally, EC-derived EV caspase-1 activity from septic patients with ARF (n =11) and without (n=17) ARF was comparable ( Figure 5D ). These data demonstrate that EC-derived EV caspase-1 activity is associated with ARDS development during sepsis, but not with mortality or ARF of sepsis. Figure 5 Circulating endothelial cell-derived EV caspase-1 activity is associated with ARDS development. Circulating endothelial cell-derived EVs were isolated from 11 healthy controls and 28 septic patients (14 ARDS and 14 non-ARDS), and caspase-1 activity (A) was determined. * p < 0.05 compared to healthy controls, # p<0.05 compared to non-ARDS septic patients. N=11-14/group. Statistical significance was determined by analysis of variance (ANOVA). The association of caspase-1 activity with mortality (B) , ARDS development (C) and acute renal failure (D) were analyzed using Wilcoxon rank sum tests. Boxes present the 25 th , 50 th , and 75 th percentiles of caspase-1 activity in each group. Whiskers extend 1.5 +/- the inner quartile range (i.e. difference between the 75 th and 25 th percentile). Grey points are the caspase-1 activity values for subjects within each group. EV, Extracellular vesicles; ARDS, acute respiratory distress syndrome. 3.6 Circulating EC-derived EV miR-126-3p levels were decreased in ARDS patients In addition to caspase-1, miR-126 has been implicated in vascular injury during sepsis and is one of the most abundant miRNAs in the EC-derived EVs ( 18 , 20 , 21 ). Thus, we next determined miR-126 levels in circulating EC-derived EVs from day 1 plasma samples. MiR-126-3p levels were significantly decreased in ARDS patients (n=5) compared to healthy control group (n=8; p<0.05, Figure 6A ), while miR-126-5p levels were not different among the investigated groups (n=8, Figure 6B ). Figure 6 Circulating EV miR-126 levels and its association with clinical outcomes of sepsis. Circulating endothelial-cell-derived EVs were isolated, miRNA-126 levels (A, B) were determined in healthy controls, non-ARDS septic patients, and ARDS patients. *p < 0.05 compared to healthy controls. N=5-8/group. Statistical significance was determined by analysis of variance (ANOVA). Circulating total EV miRNA levels in sepsis and ICU control patients were determined by next generation sequencing. Association between EV miR-126-3p (C) or miR-126-5p (D) levels with sepsis outcomes (mortality; ARDS; acute renal failure) were analyzed using a linear mixed model approach. Differences between patient statuses over time were evaluated using linear contrasts from the model. P-values were Bonferroni adjusted for all comparisons considered. N=12/group. EV, Extracellular vesicles. 3.7 Circulating total EV miR-126 levels are associated with sepsis outcomes To further analyze the relationship between EV miR-126 levels and sepsis outcomes, RNAs were isolated from circulating total EVs taken on days 1 and 3 from subjects with sepsis and non-septic critical illnesses controls. Next-generation sequencing identified that 51 miRNAs were differentially expressed between the two groups with miR-126-3p and -5p being among the most differentially expressed (adjusted p values of 0.03 and 0.003, respectively; data not shown). The temporary decline of EV miR-126-3p levels from day 1 to day 3 was significantly associated with ARDS development but not mortality or ARF (n=12, Figure 6C ). An overall decline in EV miR-126-5p levels from day 1 to day 3 is significantly associated with increased mortality, ARDS and ARF (n=12, Figure 6D ).\",\n",
            "        \"discussion\": \"4 Discussion Currently, there are no reliable biomarkers to predict sepsis outcomes. Caspase-1 and miR-126 in EVs have been implicated in lung injury and sepsis ( 18 , 20 , 21 ). In the present study, our findings demonstrated previously unappreciated associations of circulating EV caspase-1/miR-126 levels with sepsis outcomes. First, the impact of circulating total EVs on endothelial injury is associated with ARDS development during sepsis; second, caspase-1 activity in circulating EVs an associated with ARDS development during sepsis; third, a decline in total EV miR-126-5p levels from day 1 to day 3 is associated with mortality, ARDS and ARF during sepsis. Therefore, circulating EV function/contents may predict patient outcomes at an early time point, and lead to early clinical interventions to improve outcomes of septic patients. During sepsis, leukocytes, endothelial cells, and platelets produce large amounts of EVs, which carry a variety of RNAs and proteins to modulate recipient cell function. Circulating EVs can be released and taken up by recipient cells quickly, thus, the pool of circulating EVs is dynamic and reflective of the temporal changes in cellular activation ( 11 ). Accordingly, the EV profile of each individual may change throughout the course of sepsis offering a window into the longitudinal states of cellular activation. A deeper understanding of circulating total/cell-specific EVs and their function/contents over the course of sepsis development may pave the way to identify novel biomarkers and subgroup septic patients for future personalized treatment. EC dysfunction, manifested by increases in vascular permeability, is an important feature of sepsis, and plays a critical role in the pathogenesis of multi-organ failure ( 5 , 29 , 30 ). Platelet-derived EVs from septic patients have been reported to induce endothelial cell apoptosis ( 31 ). However, the role of circulating EVs from patients with sepsis on endothelial dysfunction, and their connections with sepsis outcomes remain largely unknown. Our data demonstrated that circulating EVs from a subset of septic patients induce vascular permeability while EVs from others did not. Interestingly, septic patients with circulating EVs that induced vascular permeability were more likely to develop ARDS compared to those patients who did not, suggesting that the impact of circulating EVs on vascular injury may contribute to ARDS development during sepsis. Thus, a deeper understanding of circulating EV properties and biology may allow for sepsis endotyping and lead to novel therapeutic strategies. Caspase-1, a member of the cysteinyl proteases family, plays a critical role in regulating inflammatory responses, apoptosis, and pyroptosis, three known features of sepsis ( 32 \\u2013 35 ). A growing body of evidence has confirmed that inhibiting caspase-1 is beneficial in sepsis, as it can reduce inflammation, apoptosis and pyroptosis, and attenuate organ injury ( 34 , 36 \\u2013 39 ). In addition, circulating EVs isolated from septic patients induce lymphocyte apoptosis via caspase-1 ( 33 ). However, circulating EV caspase-1 and its association with sepsis outcomes remains largely unknown. In agreement with previous findings that circulating EV caspase-1 activity was significantly enhanced in septic patients compared with controls ( 33 ), we further demonstrated that caspase-1 activity in circulating EVs was significantly higher in septic patients who experienced ARDS as compared to those who did not. This observation was further seen in EVs derived from both monocytes and endothelial cells suggesting a potentially important role of these cells in cellular cross-talk in the setting of sepsis-related organ failure. This is consistent with previous findings that EVs from cultured monocytes can cause lung vascular endothelial cell injury via its caspase-1 content ( 17 , 18 ). Thus, circulating monocyte EV caspase-1 may induce vascular injury and contribute to ARDS development during sepsis. These initial studies demonstrate the feasibility of using cell-specific EV caspase-1 to develop novel biomarkers and subgroup septic patients. Our previous studies suggested that EC EV miR-126 improves the outcomes of lipopolysaccharide-induced acute lung injury and cecal ligation and puncture-induced sepsis via reducing inflammation and vascular permeability in an animal model ( 20 , 21 ). Recent studies have further confirmed that delivering miR-126 to septic animals can alleviate multiple organ dysfunction including lung, liver, kidney, brain, and heart by reducing inflammation and vascular injury ( 19 , 22 , 40 ). However, circulating EV miR-126 content and its association with human sepsis outcomes remains largely unknown. Here we extended previous observations that serum miR-126-3p levels were decreased among septic patients compared to control subjects ( 41 ) by demonstrating that circulating EV miR-126-3p levels are significantly decreased in septic patients who experience ARDS as compared to healthy controls. In addition, we demonstrated for the first time that a decline in circulating total EV miR-126-5p levels is associated with mortality, ARDS and ARF while decreased miR-126-3p levels is associated with ARDS development during sepsis. These data indicate that further investigations on circulating EV miR-126 levels may provide novel biomarkers or therapeutic targets for sepsis. There are several limitations of this study. First, we detected the effect of circulating total EVs on EC permeability and its association with sepsis outcomes; however, the effect of cell-specific EVs from platelets, ECs, neutrophils, monocytes, and other cell types on EC permeability remains to be further investigated. Second, we examined caspase-1 levels in monocyte- or EC-derived EVs and their association with clinical features in a small cohort of septic patients. However, further validation studies in larger cohorts of septic patients are necessary. Last, the association of circulating total EV miR-126 levels with sepsis outcomes has been determined in this study; however, cell-specific EV miR-126 levels in a large cohort of septic patients and their connections to sepsis outcomes need further investigation. In summary, we discovered novel associations between circulating EV function/contents and sepsis outcomes, which may lead to the discovery of novel biomarkers for sepsis and possible new therapeutic approaches.\",\n",
            "        \"conclusion\": null\n",
            "    },\n",
            "    {\n",
            "        \"pmcid\": \"PMC10191296\",\n",
            "        \"abstract\": null,\n",
            "        \"intro\": \"Introduction The global prevalence of neurodegenerative diseases, such as Parkinson disease (PD), Alzheimer disease (AD), and amyotrophic lateral sclerosis (ALS), is increasing due to an aging population and increasing life expectancy in most parts of the world [ 1 \\u2013 3 ]. Currently, treatments for neurodegenerative diseases are largely focused on symptom management making it urgent to identify disease-modifying therapies. Pharmaceutical treatments can emerge through the synthesis of new drugs or the repurposing of existing drugs. The latter approach is cost-effective as FDA-approved medications typically have well-established pharmacokinetic and safety profiles. In this study, we used a large, population-based administrative claims dataset, specifically Part D pharmacy claims data from United States (U.S.) Medicare beneficiaries, to identify prescription drugs associated with a lower risk of PD, AD, and/or ALS. To maximize our ability to identify potentially protective drugs, we screened medications according to their biological targets and pharmacological actions on those targets, rather than as individual drugs. We hypothesized that the three neurodegenerative diseases share some common neurodegenerative mechanisms and that epidemiologic evidence of medications associated with a lower risk of multiple neurodegenerative diseases would represent high-priority candidates for disease-modifying clinical trials. This work suggested one biologic target as a priority for further research.\",\n",
            "        \"method\": \"Materials and methods Study overview Based on U.S. Medicare beneficiaries \\u226566 years of age with Part D pharmacy coverage (claims data), we constructed a large, population-based case-control study with incident AD, PD, and ALS cases in 2009, along with comparable beneficiaries without these conditions in the same year. Medicare is the national health insurance used by >98% of Americans age \\u226565. Following identification of candidate neuroprotective medications from this case-control study, we attempted to replicate our strongest findings using the cohort design. Specifically, we constructed a large, Medicare-based follow-up study from 2010\\u20132014 while applying both traditional and active comparator approaches. The study was approved by the Washington University Human Research Protection Office and the Centers for Medicare and Medicaid Services (CMS), which only released data after de-identification. Informed consent was not required in this records-based study, which was classified as not involving human subjects research; we obtained a waiver. Study eligibility criteria All cases and controls met study criteria designed to ensure complete case ascertainment and a population-based sample of both cases and controls [ 4 ]. Briefly, both met all of the following criteria in 2009: 1) age \\u226566 years, 11 months (age-eligible for Medicare \\u22652 years) but \\u226490, 2) Medicare Part A/B coverage and no Part C coverage, and 3) residence in the U.S. We determined the above from the Medicare summary file for 2009, which enumerates basic demographic and medical information for all beneficiaries. During this time period, approximately a quarter of beneficiaries otherwise eligible for our study had Part C coverage [ 4 ]. For the present study, we also required Medicare Part D (pharmacy) coverage and \\u22651 prescription claim in 2006\\u20132007. Part D is an optional Medicare program that covers prescription medications, which began in 2006. We required and restricted to Part D claims in these years because this approach ensured that all medications were first prescribed >1 year before PD/AD/ALS diagnosis (or control reference date) in 2009, in order to minimize detection of associations due to prodromal disease symptomatology. For the follow-up component, all of the same criteria applied. In addition, we required beneficiaries to have survived, without diagnosis of neurodegenerative disease, to January 1, 2010, the start of follow-up. Identification of PD, AD, and ALS cases We identified PD, AD, and/or ALS on the basis of International Classification of Diseases, Ninth Revision, Clinical Modification (ICD-9-CM) diagnosis codes contained in comprehensive claims data in 2004\\u20132009, specifically for PD (ICD-9-CM 332 or 332.0), AD (ICD-9-CM 331 or 331.0), and ALS (ICD-9-CM 335.2 or 335.20). In the case-control study, we restricted to incident cases in 2009, i.e., excluded prevalent cases in all years (2004\\u20132009), by including only those PD/AD/ALS cases with a diagnosis code for the respective condition in 2009, but no prior years (2004\\u20132008). We excluded from the PD case group 4.6% of potential cases with a diagnosis code for atypical parkinsonism (ICD-9-CM 333.0) or dementia with Lewy bodies (ICD-9-CM 331.82) [ 4 ]. We included all remaining incident PD cases who met study eligibility criteria. AD is the most common neurodegenerative disease, and we also included a 5% random sample of all incident AD cases who met study eligibility criteria. ALS is rare and we included all incident ALS cases who met study eligibility criteria. We applied the same case identification approach during follow-up in 2010\\u20132014. We identified 42,885 incident cases of PD/AD/ALS in 2009. In order to leverage independent findings across the three neurodegenerative conditions and identify the most promising neuroprotective drug candidates, we classified cases into one of four groups: PD only (N = 28,679), AD only (N = 8,332), ALS only (N = 1,341), or a \\u201cmixed\\u201d PD/AD/ALS case category (N = 4,533), containing a code for >1 of these conditions. We retained this mixed case group because beneficiaries who obtain a code for only one condition might differ from those in the mixed group in ways that are not restricted to certainty of (the single) diagnosis. In addition, this mixed group provided for a case group in which we hypothesized we could replicate associations that we observed across multiple individual case groups, particularly for PD and AD. During the five years of follow-up in 2010\\u20132014 we identified 18,293 additional incident cases of neurodegenerative disease. Selection of controls In the case-control study, controls were a random sample (N = 334,387) of all beneficiaries enumerated in the 2009 summary file who met the same criteria, but without any of the above ICD-9-CM diagnosis codes for PD, AD, ALS, other motor neuron disease (ICD-9-CM 335.21, 335.22, 335.23, 335.24, or 335.29), atypical parkinsonism, or dementia with Lewy bodies. Because our large sample size provided good statistical power, we opted to adjust for covariates rather than match controls to cases [ 5 ]. We used all controls for comparisons to each case group. In addition, a random sample of all controls who survived to January 1, 2010 were available for the follow-up study (N = 207,764). By definition these controls were free of neurodegenerative disease on that date. Assessment of medication use and their biological targets We used Part D pharmacy claims data from 2006\\u20132007 to identify all medications used by cases and controls in the study period, effectively applying >1 year of exposure lagging. We classified a beneficiary as having used a medication if there was \\u22651 prescription event (fill) of the respective medication. We then identified the active ingredient(s) in each medication. In total, we identified use of 768 active ingredients. For each active ingredient, we next identified the biological targets, such as specified metabolic enzymes or receptors, and the pharmacological actions on each target, i.e., activator, blocker, or other. We identified these targets and actions through DrugBank [ 6 ]. Presented results are based on targets and actions in this database by February 2023, and we manually reviewed active ingredients in our dataset to ensure linkage when active ingredient names differed. The 768 active ingredients in our dataset corresponded to 723 targets from DrugBank, which we collapsed by similarity to 561 unique targets. We then simultaneously considered the pharmacological action on each target, yielding 723 biological target-action pairs. Of these, 141 target-action pairs had >10 beneficiaries in each case group and in the control group (S1 Table in S1 File ), and we focused on these most commonly observed target-action pairs to meet CMS reporting requirements. Assessment of covariates We obtained each beneficiary\\u2019s birth date (age), sex, and race/ethnicity from the 2009 Medicare summary file. We determined indicators of overall use of medical care from 2008, so that post-diagnosis care would not be included. Use of care is a strong potential confounder of the association between various medical conditions and PD [ 7 ], and therefore also is a potential confounder in a pharmacoepidemiologic study. As direct indicators of use of care we obtained the number of outpatient/physician (carrier) visits and number of days hospitalized as an inpatient including in a skilled nursing facility from the summary file from 2008. From these same summary files in 2007\\u20132008 we used a count of all medical conditions tracked by the Chronic Conditions Data Warehouse (CCW) other than AD and sex-specific conditions. These included acute myocardial infarction, rheumatoid/osteoarthritis, atrial fibrillation, cataract, cancer of the colon, cancer of the lung, chronic kidney disease, chronic obstructive pulmonary disease (COPD), congestive heart failure, depression, diabetes, fracture of the hip/pelvis, glaucoma, ischemic heart disease, osteoporosis, and stroke/transient ischemic attack. These 16 conditions largely overlap with those used to calculate the Charlson Comorbidity Index, and as such, the number of these conditions can serve as an indirect indicator of use of medical care. In addition to this sum of medical conditions as an a priori covariate, we retained a single dichotomous variable for each of these 16 conditions as additional covariates available for post-hoc sensitivity analyses (selected according to potential indications for the respective prescribed medications). We also combined the COPD and lung cancer variables as an indicator of smoking, given that most individuals with COPD or lung cancer are current or previous tobacco smokers. We had no missing covariate data, with the exception of race/ethnicity for a small percentage of beneficiaries who we combined with those of an \\u201cother\\u201d race/ethnicity. Statistical analysis We used R version 4.2.1 to prepare figures and otherwise conducted all analyses using Stata/MP version 14.2 or 17.0 [ 8 ]. In our case-control study, we used multinomial (polytomous) logistic regression to estimate adjusted odds ratios (ORs) and 95% confidence intervals (CIs) between each target-action pair and each neurodegenerative disease group relative to all controls. We generated one model per target-action pair, with the outcome in five categories (PD only, AD only, ALS only, mixed PD/AD/ALS cases, or none of these outcomes [controls]). In the follow-study, we used Cox proportional hazards regression in our primary analysis and competing-risks survival analysis in sensitivity analysis, as detailed below. In all analyses, we adjusted a priori for age, as a continuous variable, using Harrell\\u2019s method [ 9 ], to allow for flexible adjustment, recognizing that the shape of the strong association between age and each of the three neurodegenerative conditions differs. We also adjusted for sex and race/ethnicity (White, Black, Hispanic, and Asian/Pacific Islander [API]/other/unknown), use of medical care, and number of CCW conditions (as detailed above, each as a continuous variable), and a dichotomous indicator of smoking (COPD or lung cancer). In addition, we confirmed all reported associations in sensitivity analyses by simultaneously adjusting for dichotomous variables for selected CCW medical conditions and medications that could potentially confound the association of interest. Because we aimed to identify neuroprotective medications, we primarily report inverse associations from our discovery step, i.e., case-control study. To avoid missing associations due to insufficient statistical power, we used two-sided \\u03b1 = 0.05 to determine significance, rather than correcting for multiple comparisons. Instead, we considered both the magnitude and consistency of the associations across the four case groups. As a measure of both magnitude and consistency, we calculated the mean OR across these four case groups for each biological target-action pair. To explore the consistency of the potential inverse associations for a target-action pair further, we examined associations for the individual medications in the respective target-action pair, as well as by sex in sex-specific models (two models per target-action pair). In the cohort analyses, we focused on selected candidate medications and pre-hypothesized the direction of effect, but retained our more conservative two-sided \\u03b1 = 0.05. We attempted to replicate the strongest inverse associations in a cohort study that included an active comparator group [ 10 ], i.e., individuals who use different medications with the same indication as the medication of interest. This approach is used to address confounding by indication and ensure greater comparability of the groups. This approach can also be combined with the new user design [ 10 ]. However, we did not simultaneously apply the new user design because of the known diagnostic delay in ALS [ 11 ] and the long prodromal period for AD and PD [ 12 ], which together could imply that the observed exposures could not have occurred before disease onset. Rather, similar to the initial study, we applied a two-year exposure lag to exclude medications first used between disease onset and diagnosis. In this cohort we followed the sample of controls who survived to January 1, 2010 forward for incident neurodegenerative disease until death or end of 2014, i.e., up to five years. We also censored individuals if and when they developed atypical parkinsonism, developed dementia with Lewy bodies, or enrolled in Part C coverage. Total time at risk and under observation was 781,486 years in this study. We used Cox proportional hazards regression, while accounting for the same covariates as of baseline, both with and without relevant CCW medical conditions and other medications. We used age as the time scale [ 13 ] to account for age and then included all other covariates as independent variables in the model. We estimated hazards ratios (HRs) and 95% CIs from the Cox model. We checked the proportional hazards assumption using Schoenfeld residuals [ 14 ]. In sensitivity analyses we excluded PD/AD/ALS in progressively more years of follow-up to further minimize potential effects of reverse causation, given that only two years of exposure lagging was possible because Part D coverage first became available to Medicare beneficiaries in 2006. In additional sensitivity analyses we conducted competing-risks survival regression based on Fine and Gray\\u2019s proportional subhazards model [ 15 ]. In these analyses we treated the start of Part C coverage and death as competing risks. Because PD, AD, and ALS are all rare, and because we did not use incidence density sampling of controls, we interpreted ORs from our case-control study as relative risks, rather than as ORs or incidence rate ratios [ 16 ]. The latter are most comparable to HRs, but for simplicity we interpreted HRs from our cohort study as approximations of relative risk over the five years of follow-up when the proportional hazards assumption was met.\",\n",
            "        \"subjects\": null,\n",
            "        \"results\": \"Results Participant characteristics In each case group and in controls, a majority (>80%) of beneficiaries were non-Hispanic white, and >50% were female ( Table 1 ). PD and ALS were each more common in men than women, and in non-Hispanic Whites than each of the other racial/ethnic groups. In contrast, AD was more common in women than in men, and more common in Black and Hispanic beneficiaries than in White or especially Asian/Pacific Islander beneficiaries, who had the lowest AD risk. Risk of PD and AD increased with age, whereas risk of ALS peaked in the 70s. When accounting for these demographic differences, history of smoking (COPD or lung cancer) was less common in PD cases as compared to controls. In the 1\\u20132 years prior to diagnosis with a neurodegenerative condition, use of medical care in cases was generally greater than among controls. 10.1371/journal.pone.0285011.t001 Table 1 Characteristics of beneficiaries with and without incident PD, AD, and/or ALS, U.S. Medicare 2009. PD only N = 28,679 % OR (95% CI) a AD only N = 8,332 % OR (95% CI) a ALS only N = 1,341 % OR (95% CI) a Mixed PD/AD/ ALS N = 4,533 % OR (95% CI) a Controls N = 334,387 % Age, years 66\\u201369 8.1 1.0 (Reference) 3.7 1.0 (Reference) 12.5 1.0 (Reference) 4.8 1.0 (Reference) 13.6 70\\u201374 22.6 1.20 (1.14, 1.26) 12.5 1.47 (1.29, 1.67) 31.8 1.13 (0.95, 1.36) 14.2 1.29 (1.11, 1.51) 29.9 75\\u201379 25.0 1.58 (1.51, 1.66) 21.0 2.97 (2.63, 3.35) 23.2 1.02 (0.84, 1.23) 23.2 2.51 (2.17, 2.91) 23.7 80\\u201390 44.2 1.90 (1.81, 1.99) 62.8 5.91 (5.26, 6.63) 32.5 1.02 (0.85, 1.23) 57.8 4.18 (3.63, 4.81) 32.9 Mean (SD) 78.4 (6.1) \\u2014 81.0 (5.9) \\u2014 76.6 (6.1) \\u2014 80.4 (5.9) \\u2014 76.6 (6.2) Sex Male 43.1 1.0 (Reference) 28.8 1.0 (Reference) 45.4 1.0 (Reference) 39.1 1.0 (Reference) 35.6 Female 56.9 0.65 (0.64, 0.67) 71.2 1.14 (1.09, 1.20) 54.6 0.64 (0.58, 0.72) 60.9 0.72 (0.68, 0.76) 64.4 Race/ethnicity White 86.7 1.0 (Reference) 80.6 1.0 (Reference) 88.1 1.0 (Reference) 81.3 1.0 (Reference) 84.1 Black 5.8 0.74 (0.71, 0.78) 10.5 1.43 (1.33, 1.54) 6.6 0.84 (0.68, 1.05) 8.7 1.18 (1.06, 1.31) 7.7 Hispanic 2.8 0.91 (0.84, 0.98) 4.3 1.47 (1.31, 1.64) 1.3 0.47 (0.30, 0.75) 4.8 1.61 (1.40, 1.85) 2.7 API/other/unknown 4.7 0.85 (0.80, 0.90) 4.6 0.91 (0.82, 1.01) 3.9 0.69 (0.52, 0.91) 5.2 1.00 (0.88, 1.15) 5.5 Use of medical care b Annual number of physician/outpatient visits, mean (SD) 16.2 (12.9) 1.006 (1.006, 1.007) 13.4 (11.7) 0.995 (0.994, 0.996) 17.1 (13.8) 1.012 (1.010, 1.014) 14.5 (12.3) 0.998 (0.997, 1.000) 12.5 (10.9) Annual number of days in a hospital or SNF, mean (SD) 4.9 (17.6) 1.007 (1.007, 1.008) 6.3 (20.1) 1.009 (1.008, 1.010) 3.4 (12.8) 1.005 (1.002, 1.08) 6.9 (21.3) 1.009 (1.008, 1.011) 1.8 (9.9) Number of selected medical conditions b , c 3.5 (2.1) 1.16 (1.15, 1.17) 3.5 (2.1) 1.14 (1.13, 1.16) 3.0 (2.0) 1.02 (0.99, 1.06) 3.6 (2.2) 1.20 (1.18, 1.22) 2.7 (1.9) COPD or lung cancer b 20.5 0.86 (0.83, 0.89) 20.7 0.99 (0.93, 1.06) 20.4 1.10 (0.95, 1.29) 20.7 0.85 (0.78, 0.92) 15.3 a Adjusted for all variables shown. ORs for use of care/conditions are per visit, day, or condition. ORs for COPD/lung cancer are relative to beneficiaries without COPD and lung cancer. b Prior to diagnosis, specifically in 2008 (visits or days) or 2007\\u20132008 (medical conditions, including COPD and lung cancer). c Of all (16) conditions available from the 2007\\u20132008 Medicare beneficiary annual summary files other than AD/dementia and sex-specific cancers: Acute myocardial infarction, rheumatoid/osteoarthritis, atrial fibrillation, cataract, cancer of the colon, cancer of the lung, chronic kidney disease, chronic obstructive pulmonary disease (COPD), congestive heart failure, depression, diabetes, fracture of the hip/pelvis, glaucoma, ischemic heart disease, osteoporosis, and stroke/transient ischemic attack. c Of all (16) conditions available from the 2007\\u20132008 Medicare beneficiary annual summary files other than AD/dementia and sex-specific cancers: Acute myocardial infarction, rheumatoid/osteoarthritis, atrial fibrillation, cataract, cancer of the colon, cancer of the lung, chronic kidney disease, chronic obstructive pulmonary disease (COPD), congestive heart failure, depression, diabetes, fracture of the hip/pelvis, glaucoma, ischemic heart disease, osteoporosis, and stroke/transient ischemic attack. Neurodegenerative disease associations in the case-control (discovery) study Thirty target-action pairs were significantly associated with lower risk of \\u22651 neurodegenerative disease ( Table 2 and Fig 1 and S2 Table in S1 File ). Four of these target-action pairs had a mean OR \\u22640.80 (\\u226520% average reduction in risk) across all four case groups ( Table 2 ). All four neurodegenerative case groups were significantly inversely associated with one target-action pair, tubulin alpha/beta chain blockers (represented by six medications, including the anti-gout medication colchicine, target-action pair mean OR = 0.76). However, this target-action pair association appeared stronger in men (mean OR = 0.67) than women (mean OR = 0.90), and was inconsistent across the individual medications within this target-action pair. Specifically, there was no evidence of lower risk of any of the neurodegenerative conditions in relation to any of the tubulin alpha/beta chain blockers other than colchicine. Use of three of these medications was too infrequent to consider individually, while the OR point estimates for the remaining tubulin alpha/beta chain blockers were universally at or above null for all four neurodegenerative conditions (ORs ranged from 0.99 to 1.40 with a mean of 1.20 for griseofulvin, and ORs ranged from 0.97 to 2.06 with a mean of 1.55 for mebendazole). 10.1371/journal.pone.0285011.g001 Fig 1 Medication target-action pairs significantly inversely associated with at least one neurodegenerative condition, U.S. Medicare 2009. Legend: Thirty target-action pairs were significantly associated with lower risk of \\u22651 neurodegenerative disease. Four or these target-action pairs had a mean odds ratio \\u22640.80 (\\u226520% average reduction in risk) across all four case groups ( Table 2 ) and the remainder had a mean odds ratio <1.00 (S2 Table in S1 File ). 10.1371/journal.pone.0285011.t002 Table 2 Medication target-action pairs significantly inversely associated with neurodegenerative condition(s) and \\u226520% average reduction in relative risk, U.S. Medicare 2009. Adjusted OR (95% CI) a Target (action) Medication(s) PD only N = 28,679 AD only N = 8,332 ALS only N = 1,341 Mixed PD/AD/ALS N = 4,533 Mean OR Tubulin alpha/beta chain (blocker) Albendazole, colchicine, griseofulvin, mebendazole, paclitaxel, podofilox 0.85 (0.78, 0.92) 0.75 (0.64, 0.88) 0.63 (0.42, 0.95) 0.79 (0.65, 0.96) 0.76 Xanthine dehydrogenase/oxidase (blocker) Allopurinol b 0.84 (0.79, 0.89) 0.69 (0.61, 0.78) 0.87 (0.66, 1.13) 0.66 (0.56, 0.78) 0.77 Bifunctional purine biosynthesis protein PURH (blocker) Methotrexate 0.90 (0.80, 1.01) 0.74 (0.58, 0.94) 0.74 (0.43, 1.28) 0.78 (0.58, 1.06) 0.79 Vascular endothelial growth factor A (other) b Carvedilol b 0.81 (0.77, 0.86) 0.90 (0.81, 0.99) 0.78 (0.59, 1.01) 0.72 (0.62, 0.82) 0.80 a OR comparing beneficiaries who did vs. did not use any of the listed medications prior to diagnosis/reference, adjusted for age (continuous, using Harrell\\u2019s method), sex, race/ethnicity (4 categories: White, Black, Hispanic, and API/other/unknown), use of medical care (total number of physician/outpatient visits, total number of days in a hospital/skilled nursing facility, and total number of selected medical conditions, each prior to case diagnosis or control reference as continuous variables), and indicator of smoking (lung cancer or COPD). b Carvedilol also blocks effects of xanthine dehydrogenase/oxidase, but this is an off-target effect, so is not classified as such here. Abbreviations: AD = Alzheimer disease; ALS = amyotrophic lateral sclerosis; API = Asian/Pacific Islander; COPD = chronic obstructive pulmonary disease; CI = confidence interval; OR = odds ratio; PD = Parkinson disease. The three other target-action pairs with inverse associations suggested across all four case groups and with a mean OR \\u22640.80 were represented by one medication each: xanthine dehydrogenase/oxidase blockers (the anti-gout medication allopurinol, mean OR = 0.77), bifunctional purine biosynthesis protein PURH blockers (methotrexate, typically used to treat psoriasis, rheumatoid arthritis, and cancer, mean OR = 0.79), and targets of vascular endothelial growth factor A (the beta blocker carvedilol, used for hypertension and heart failure, mean OR = 0.80) ( Table 2 ). Results were relatively similar in men and women for both allopurinol (mean OR = 0.75 and 0.79, respectively) and carvedilol (mean OR = 0.79 and 0.81, respectively). OR point estimates for both men and women for all four case groups were <1.0 for both allopurinol and carvedilol, although some CIs were quite wide. Methotrexate use was uncommon in men, but for all case groups with >10 male cases, ORs in men and women were similar (PD ORs 0.83\\u20130.88; AD ORs 0.75\\u20130.76). The remaining 26 target-action pairs with at least one significant inverse association had more modest mean ORs across the four case groups ( Fig 1 and S2 Table in S1 File ). Among these target-action pairs were hypoxia-inducible factor 1-alpha activators due to the inverse associations for carvedilol, as well as for hydralazine (captured alone in another target-action pair here). In addition, one target-action pair demonstrated significant inverse associations for all four neurodegenerative disease groups, medications with \\u201cother\\u201d effects on nuclear receptor (represented by ten medications including atorvastatin, flutamide, ketoconazole, spironolactone, warfarin, and selected chemotherapeutic agents, mean OR = 0.86). We confirmed this association in both men (mean OR = 0.83) and women (mean OR = 0.87). We observed less consistent results across disease groups for most of the remaining target-action pairs, including the lactoylglutathione lyase blocking medication, indomethacin, and targets of angiotensin-converting enzyme (chloroquine and hydroxychloroquine), which were not associated with ALS ( Fig 1 and S2 Table in S1 File ). Frequently, results were consistently inverse for both PD and AD but not ALS, or were only inverse for ALS. The latter associations were most commonly for target-action pairs that included medications used to treat type 2 diabetes or hypercholesterolemia. Among the target-action pairs with a mean OR\\u22651.0 (S3 Table in S1 File ), none were in agreement or contrast with the associations noted above for xanthine oxidase/dehydrogenase (use of activators was too rare to examine) or bifunctional purine biosynthesis protein PURH (no other approved medications are known to otherwise target this pathway). In contrast to the targets mentioned above and an off-target blockade of effects from xanthine oxidase/dehydrogenase, target-action pairs for carvedilol (adrenergic receptor and potassium channel blocking) were not inversely associated with PD, AD, or ALS. Insofar as we could apply further adjustments, the associations of greatest interest were not altered markedly by adjustment for the appropriate available CCW conditions or other medications with a strong confounding potential. When we adjusted allopurinol associations for colchicine, indomethacin, and rheumatoid/osteoarthritis (CCW variable closest to gout), ORs for each of the four neurodegenerative groups and allopurinol were attenuated by 5\\u20139%. When we adjusted carvedilol associations for other beta blockers and cardiovascular disease, or adjusted methotrexate associations for rheumatoid/osteoarthritis, chloroquine, and hydroxychloroquine, ORs were altered up or down by \\u22644%. Neurodegenerative disease associations in the cohort (replication) study Given the robustness of the associations for allopurinol, carvedilol, and methotrexate, we selected all three for potential replication in our follow-up study. For allopurinol, we selected colchicine and indomethacin as active comparator medications. These medications were the ones most commonly prescribed gout in the Medicare formulary at the time we conducted our study. For carvedilol, we selected all other systemic (non-ophthalmic) beta blockers observed in our data: acebutolol, atenolol, betaxolol, bisoprolol, labetalol, metoprolol, nadolol, penbutolol, pindolol, propranolol, and sotalol. We focused on beta blockers rather than all possible medications that can be used for hypertension/cardiovascular disease in order to ensure that the active comparator group was as similar as possible to the carvedilol group. Finally, for methotrexate our active comparator medications were the following medications observed in our data that are used to treat rheumatoid arthritis: hydroxychloroquine, chloroquine, sulfasalazine, leflunomide, etanercept, and infliximab. When we examined the associations between the three candidate medications and any PD/AD/ALS in the replication cohort study, our primary findings, i.e., for allopurinol and carvedilol, were largely confirmed. We again observed clear inverse associations for allopurinol ( Table 3 ). HR point estimates for allopurinol were consistently inverse, even when restricted to users of allopurinol who did not use other gout medications. In addition, HR point estimates grew further below null when using an active comparator group or with greater exclusion of PD/AD/ALS diagnosed in the first years of follow-up. After excluding diagnoses in the first four years of follow-up, the HR most comparable to our initial analysis (HR = 0.77, 95% CI 0.62, 0.95) was essentially same as the mean OR across all PD/AD/ALS case groups from the initial analysis (OR = 0.77) (Tables 2 and 3 ). The same was true for carvedilol (HR = 0.81, 95% CI 0.67, 0.99 vs. mean OR = 0.80) ( Table 2 ; S4 Table in S1 File ). In contrast, we were unable to confirm an association for methotrexate (all HRs \\u22650.87 and with wide CIs). Competing-risks survival regression did not materially alter conclusions for any of the three medications. Relative to the non-exposed comparator group, use of colchicine/indomethacin without allopurinol was not associated with neurodegenerative disease (HR = 1.02, 95% CI 0.92, 1.12) and risk of neurodegenerative disease remained strongly inversely associated with allopurinol use both with (HR = 0.80, 95% 0.69, 0.91) and without (HR = 0.83, 95% CI 0.75, 0.92) the use of other gout medications. Beta blockers other than carvedilol were not associated with neurodegenerative disease either alone (HR = 0.99, 95% CI 95, 1.02) or in combination with carvedilol (HR = 0.96, 95% CI 0.85, 1.08), whereas carvedilol alone was associated with markedly lower risk (HR = 0.82, 95% CI 0.75, 0.90) as compared to the non-exposed comparator group. 10.1371/journal.pone.0285011.t003 Table 3 Allopurinol, other gout medications, and risk of neurodegenerative disease, U.S. Medicare 2010\\u20132014. PD/AD/ALS in specified years of follow-up Total Incident PD/AD/ALS Allopurinol category Any allopurinol Non-exposed comparator HR (95% CI) a Active comparator HR (95% CI) a Basic model HR (95% CI) a Full model HR (95% CI) a , b Years 1\\u20135 (all years) N = 207,764 n (%) N = 18,293 n (%) No gout medications 195,264 (94.0) 17,208 (94.1) 1.0 (Reference) \\u2014 1.0 (Reference) 1.0 (Reference) Colchicine/indomethacin only 4,586 (2.2) 437 (2.4) 1.05 (0.96, 1.16) 1.0 (Reference) Allopurinol and colchicine/indomethacin 2,610 (1.3) 213 (1.2) 0.88 (0.77, 1.004) 0.83 (0.71, 0.98) 0.86 (0.80, 0.93) 0.86 (0.79, 0.93) Allopurinol only 5,304 (2.6) 435 (2.4) 0.86 (0.78, 0.95) 0.82 (0.71, 0.93) Years 2\\u20135 N = 203,470 c n (%) N = 13,999 n (%) No gout medications 191,229 (93.4) 13,173 (94.1) 1.0 (Reference) \\u2014 1.0 (Reference) 1.0 (Reference) Colchicine/indomethacin only 4,479 (2.2) 330 (2.4) 1.05 (0.94, 1.17) 1.0 (Reference) Allopurinol and colchicine/indomethacin 2,550 (1.3) 153 (1.1) 0.85 (0.72, 0.99) 0.80 (0.66, 0.98) 0.88 (0.80, 0.96) 0.88 (0.80, 0.96) Allopurinol only 5,212 (2.6) 343 (2.5) 0.89 (0.80, 0.99) 0.85 (0.73, 0.99) Years 3\\u20135 N = 199,496 c n (%) N = 10,025 n (%) No gout medications 187,495 (94.0) 9,439 (94.2) 1.0 (Reference) \\u2014 1.0 (Reference) 1.0 (Reference) Colchicine/indomethacin only 4,391 (2.2) 242 (2.4) 1.09 (0.96, 1.23) 1.0 (Reference) Allopurinol and colchicine/indomethacin 2,504 (1.3) 107 (1.1) 0.84 (0.69, 1.02) 0.77 (0.62, 0.97) 0.86 (0.77, 0.95) 0.85 (0.76, 0.95) Allopurinol only 5,106 (2.6) 237 (2.4) 0.87 (0.76, 0.99) 0.80 (0.67, 0.96) Years 4\\u20135 N = 195,717 c n (%) N = 6,246 n (%) No gout medications 183,941 (94.0) 5,885 (94.2) 1.0 (Reference) \\u2014 1.0 (Reference) 1.0 (Reference) Colchicine/indomethacin only 4,298 (2.2) 149 (2.4) 1.09 (0.92, 1.28) 1.0 (Reference) Allopurinol and colchicine/indomethacin 2,470 (1.3) 73 (1.2) 0.94 (0.74, 1.18) 0.86 (0.65, 1.14) 0.86 (0.75, 0.98) 0.85 (0.73, 0.98) Allopurinol only 5,008 (2.6) 139 (2.2) 0.82 (0.70, 0.98) 0.76 (0.60, 0.96) Year 5 N = 192,514 c n (%) N = 3,043 n (%) No gout medications 180,930 (94.0) 2,874 (94.5) 1.0 (Reference) \\u2014 1.0 (Reference) 1.0 (Reference) Colchicine/indomethacin only 4,226 (2.2) 77 (2.5) 1.16 (0.92, 1.45) 1.0 (Reference) Allopurinol and colchicine/indomethacin 2,423 (1.3) 26 (0.9) 0.70 (0.47, 1.03) 0.60 (0.39, 0.94) 0.77 (0.62, 0.95) 0.77 (0.62, 0.96) Allopurinol only 4,935 (2.6) 66 (2.2) 0.81 (0.63, 1.03) 0.70 (0.50, 0.97) a With age as the time scale to account for age, and adjusted sex, race/ethnicity (4 categories), use of medical care (number of physician/outpatient visits, number of days in a hospital/skilled nursing facility, and number of selected medical conditions, each prior to case diagnosis or control reference as continuous variables), and indicator of smoking (lung cancer or chronic obstructive pulmonary disease). b Also adjusted for rheumatoid arthritis/osteoarthritis, colchicine, and indomethacin. c Excludes incident PD/AD/ALS in the prior year(s) of follow-up. Abbreviations: AD = Alzheimer disease; ALS = amyotrophic lateral sclerosis; CI = confidence interval; HR = hazard ratio; PD = Parkinson disease.\",\n",
            "        \"discussion\": null,\n",
            "        \"conclusion\": \"Discussion In this large, population-based study, we took a unique approach to investigate the association between various medication categories, as defined by biological target and pharmacological action on that target, in relation to the three most common neurodegenerative diseases: AD, PD, and ALS. These findings suggest a possible new direction for repurposing or developing medications for neuroprotection. Medications that block xanthine dehydrogenase/oxidase (allopurinol) or inhibit the enzyme\\u2019s effects (carvedilol) [ 17 ] were inversely associated with all three diseases, even though the two medications in this pathway have different indications. Carvedilol is used for hypertension and heart disease, while allopurinol is used to lower uric acid levels, mainly for gout. Residual confounding by smoking could create an inverse association between carvedilol and PD, but prior analyses demonstrate that the inverse carvedilol-PD association is unlikely due to confounding by either smoking or indication [ 18 , 19 ]. In our present work we also demonstrated that other medications commonly used for gout, colchicine and indomethacin, are unlikely to cause the inverse associations for allopurinol. First, we demonstrated that the inverse association between allopurinol and neurodegenerative disease was not confounded by either of these medications. Second, of several medications that block tubulin alpha/beta chain, colchicine was the only one inversely associated with neurodegenerative disease, further confirming that the association for colchicine is not causal, but confounded by indication or allopurinol. Confounding by indication as a potential cause of the allopurinol association is more difficult to rule out, as gout is inversely associated with all three neurodegenerative diseases [ 20 \\u2013 28 ], and allopurinol is primarily used for gout. Thus, although three prior studies reported inverse associations between allopurinol and PD, similar in magnitude to ours [ 29 \\u2013 31 ], confounding by indication remains universally possible. However, a Mendelian randomization study found that serum uric acid and PD are not associated [ 32 ]. In addition, randomized trials of inosine to increase serum uric acid failed to slow progression of PD [ 33 , 34 ], which could suggest that previously observed inverse gout-PD associations were due to allopurinol rather than uric acid. Both arguments against confounding by indication also apply to ALS [ 32 , 35 ]. With regard to AD, Mendelian randomization studies conflict as to whether uric acid and AD are associated [ 32 , 36 ], but we note that the inverse allopurinol association was particularly strong for AD in our study. Finally, in our cohort analyses we confirmed the association between allopurinol and neurodegenerative disease in an active comparator design. Taken together, these findings provide potential new insights to long-established observations and suggest possible new directions for repurposing or developing medications for neuroprotection. The other most consistent finding across the three neurodegenerative diseases in our case-control analysis was a potentially protective effect of bifunctional purine biosynthesis protein PURH blockers. This target-action pair is represented by a single medication (methotrexate), which is indicated for different conditions (rheumatoid arthritis, psoriasis, and cancer) typically than either allopurinol or carvedilol. We previously found no evidence of confounding by indication for an inverse methotrexate-PD association in our population [ 37 ]. Another study found a stronger inverse methotrexate-PD association [ 30 ], but did not observe the association when restricting to rheumatoid arthritis patients [ 38 ]. Still, another study restricted to rheumatoid arthritis patients did find an inverse methotrexate-dementia association very similar to our association for AD [ 39 ]. Thus, findings for methotrexate are mixed both in the literature and in our own work, given the lack of association with neurodegenerative disease in our cohort analysis. Nonetheless, methotrexate, as a PURH blocker, remains of possible interest in light of our findings for the xanthine dehydrogenase/oxidase pathway, since PURH catalyzes the last two steps in the biosynthetic pathway that serves as the primary source of purine nucleotides [ 40 ]. As part of this process, PURH generates inosine monophosphate, a component of hypoxanthine. Xanthine dehydrogenase, and a closely related derivative enzyme, xanthine oxidase, are involved in purine metabolism. Xanthine dehydrogenase catalyzes the transformation of xanthine to uric acid. Xanthine oxidase catalyzes the same reaction, as well as the transformation of hypoxanthine to the purine xanthine, creating reactive oxygen species in both processes [ 41 ]. Accordingly, allopurinol, carvedilol, and methotrexate potentially reduce reactive oxygen species (by either limiting reactions catalyzed by xanthine oxidase or the hypoxanthine precursor to these reactions) or their effects [ 17 ]. Ultimately, further experimental and epidemiologic studies will be required to understand whether our findings are causal and related. Carvedilol protects not only against xanthine oxidase mediated oxidative stress [ 17 ], but also crosses the blood-brain barrier and protects against lipid peroxidation in brain homogenates due to its carbazole moiety [ 17 , 42 ]. Carvedilol possibly also protects against oxidative stress via an effect on hypoxia inducible factor 1-alpha [ 43 ]. Similarly, higher serum uric acid levels might lower risk and/or progression of neurodegenerative diseases by protecting neurons through anti-oxidant mechanisms [ 26 \\u2013 28 , 44 ]. Thus, it is possible that our findings are indicative of a critical role for oxidative stress in neurodegenerative disease rather than the importance of allopurinol and the xanthine oxidase/dehydrogenase pathway. Key strengths of our analyses were the restriction to incident cases and application of exposure lagging through our exclusion of medications used in the years immediately prior to diagnosis with a neurodegenerative disease. Despite these strengths and our intriguing findings, we must note some potential limitations to this administrative data study. Our findings are based on Medicare beneficiaries >65 years old and may not be generalizable to those with younger onset of neurodegenerative disease. In addition, because the definition of neurodegenerative diseases was based on administrative diagnosis codes, our case ascertainment method may have resulted in misclassification of some beneficiaries due to misdiagnosis or lack of diagnosis. Nonetheless, the demographic characteristics of our cases relative to controls were similar to that seen in other published studies [ 45 \\u2013 48 ], providing some reassurance that our case groups were sufficiently accurate to provide meaningful results. We also could not fully investigate the time course of disease and long-term use of medications as they relate to disease onset because we were limited to prescription claims data from the most recent years prior to neurodegenerative disease diagnosis. As such, we may have misclassified some beneficiaries as not taking a medication if they stopped using it before our study period. The above measurement error in both our outcome and exposure variables would likely tend to bias ORs toward null. Thus, we might have missed some true associations, and the ones we report are likely stronger than observed. With that caveat, we recognize that the potential neuroprotective effects we observed were modest. We also might have missed some true associations due to combining medications based on biological target-action pairs, rather than chemical structure, which also could have biological effects and be relevant to neurodegenerative disease risk. Finally, the medication associations we studied relate to disease risk, rather than progression, yet medications that impact disease risk may not necessarily impact disease progression. In conclusion, by classifying medications according to their specific action on their biological targets, we identified a target-action pair associated with a lower risk of developing PD, AD, and ALS. Blockade of xanthine dehydrogenase/oxidase or its effects, regardless of indication, was associated with a modestly reduced risk of all three neurodegenerative diseases. This mechanism, and possibly also related mechanisms as suggested by our secondary findings for bifunctional purine biosynthesis protein PURH, may warrant further study as a potential disease-modifying target for all three diseases.\"\n",
            "    },\n",
            "    {\n",
            "        \"pmcid\": \"PMC10390885\",\n",
            "        \"abstract\": null,\n",
            "        \"intro\": \"Introduction Patients with breast cancer predominantly present with tumors that depend on the estrogen receptor (ER) for tumor growth and progression. Such tumors are positive for estrogen receptor alpha (ER + ) encoded by the ESR1 gene, and negative for HER2 \\u2212 encoded by the ERBB2 gene ( 1 ). Endocrine therapy, the preferred first-line treatment for advanced or unresectable ER + breast cancer ( 2 ), yields superior efficacy and toxicity profile in comparison with chemotherapy ( 3, 4 ), and exploits biochemical mechanisms that include selective ER modulation ( 5 ), aromatase inhibition (AI; ref. 6 ), and selective estrogen receptor degradation (SERD; ref. 7 ). Patients initially experience therapeutic benefit, but eventually develop resistance to endocrine treatments due to any of several mechanisms ( 8 ), including acquisition of ESR1 mutations ( 9 ). Studies have demonstrated that patients with tumors expressing ESR1 mutations have improved progression-free survival (PFS) after treatment with fulvestrant, a SERD therapeutic, versus treatment with exemestane, an AI therapeutic, whereas patients with tumors expressing wild-type ESR1 have similar PFS after treatment with either therapeutic ( 10, 11 ). The success with fulvestrant in patients who acquire ESR1 mutations ( 10, 11 ) as well as the limitations of this therapeutic with respect to route of delivery and suboptimal exposure ( 12, 13 ) highlight an unmet need for novel SERD therapeutics with improved oral bioavailability for treatment of tumors harboring ESR1 mutations ( 14 ). GDC-0927 is an orally bioavailable small-molecule SERD that immobilizes ER, fully antagonizes its response to estrogen, and induces its proteasomal degradation ( 15, 16 ). GDC-0927 demonstrated dose-dependent antitumor activity in ESR1 mutant and wild-type patient-derived xenograft models of ER + breast cancer; the in vivo efficacious dose range was 10\\u2013100 mg/kg/day, which was well tolerated. This first-in-human study investigated GDC-0927 in postmenopausal women with ER + /HER2 \\u2212 locally advanced or metastatic breast cancer (MBC).\",\n",
            "        \"method\": \"Patients and Methods Study design This was an open-label, multicenter phase I clinical trial. The primary objective was to investigate the MTD and/or recommended phase II dose (RP2D) in patients with advanced or metastatic breast cancer. Secondary objectives included evaluation of the safety, antitumor activity at the RP2D, and pharmacokinetics of single and multiple doses of GDC-0927. The protocol was approved by Institutional Review Boards at participating institutions and written informed consent was obtained from patients prior to performing any procedures. The study was conducted in conformance with the ICH E6 guideline for Good Clinical Practice and the principles of the Declaration of Helsinki or the laws and regulations of the country in which the research was conducted, whichever afforded greater protection to the individual. Patients Postmenopausal women with histologic or cytologic diagnosis of locally assessed ER + /HER2 \\u2212 adenocarcinoma of the breast, with evidence of metastatic disease or locally recurrent disease not amenable to resection or radiotherapy, with progressive disease after greater than or equal to 6 months of treatment with hormonal therapy, were eligible to enroll. An ER + tumor was defined as staining in \\u22651% cells by IHC. HER2 \\u2212 breast cancer was defined as an IHC result of 0 or 1+ for cellular membrane protein expression, or a FISH result showing HER2/CEP17 ratio of <1.8, or an average of <4 copies of HER2 gene per nucleus for systems without an internal central probe. Inclusion criteria were \\u226518 years of age with an Eastern Cooperative Oncology Group (ECOG) performance status \\u22642 and adequate organ function [absolute neutrophil count \\u22651,500/\\u03bcL; platelets \\u2265100,000/\\u03bcL; aspartate transaminase (AST) and alanine aminotransferase (ALT) \\u22643 \\u00d7 upper limit of normal (ULN) or \\u22645 \\u00d7 ULN if liver function abnormalities were due to underlying malignancy; total serum bilirubin \\u22641.5 \\u00d7 ULN regardless of liver involvement secondary to tumor; serum creatinine \\u22641.5 \\u00d7 ULN]. Other inclusion criteria were \\u22642 prior chemotherapies in the advanced or metastatic setting, \\u22652 months since last use of tamoxifen, \\u22656 months since last use of fulvestrant, \\u22652 weeks since last use of any other endocrine therapy, and \\u22653 weeks since last use of any chemotherapy. The inclusion of patients with increased serum indirect bilirubin (\\u22643 \\u00d7 ULN) due to Gilbert syndrome was permitted. In the dose-expansion cohorts, eligible patients had measurable disease or evaluable disease. Prior treatment with another oral SERD was not allowed. Other exclusion criteria were: untreated brain metastasis; current treatment with any systemic anticancer therapies or experimental treatments; current use of warfarin, phenytoin, or proton pump inhibitors; diagnosis of any secondary malignancy within prior 2 years except for adequately treated basal cell or squamous cell skin cancer, or carcinoma in situ ; active inflammatory bowel disease or chronic diarrhea, short bowel syndrome, or upper gastrointestinal (GI) surgery; human immunodeficiency virus infection; clinically significant history of liver disease; major surgery within 4 weeks prior to enrollment; radiotherapy within 2 weeks prior to enrollment; any of the following occurrences within 12 months of enrollment\\u2014myocardial infarction, severe/unstable angina, ongoing cardiac dysrhythmias of grade \\u22652, atrial fibrillation of any grade, coronary/peripheral artery bypass graft, symptomatic congestive heart failure, or cerebrovascular accident including transient ischemic attack; and any other condition deemed inappropriate for study. Study treatments During dose escalation, GDC-0927 was administered orally as a single dose on day \\u22127 for pharmacokinetic evaluation during the lead-in period. Starting on day 1, patients received their treatment once daily. Dose-limiting toxicities (DLT) were evaluated from day \\u22127 through the first cycle (28 days) of treatment, for a total of 35 days. Patients were assigned to escalating doses of GDC-0927 using a standard 3+3 design. The starting dose was 600 mg/day followed by dose escalation in 400 mg increments. An expansion cohort was enrolled at the maximum administered dose (MAD) to further characterize the safety, pharmacokinetics, and antitumor activity of GDC-0927. Dose reductions were not allowed during the DLT assessment window, but were allowed once the dose cohort was closed to further enrollment. Dose delays were allowed during the DLT assessment window and for <28 days in consecutive cycles. Study assessments Toxicity was graded according to the NCI Common Toxicity Criteria for Adverse Events (Version 4.0). A DLT was defined as any of the following adverse events (AE) deemed by the investigator to be related to the study drug: Any grade \\u22653 non-hematologic toxicity (excluding alopecia of any grade and grade 3 or 4 nausea, vomiting, and diarrhea that reversed to grade \\u22642 within 24 hours with maximal medical therapy); any grade \\u22653 hematologic toxicity lasting >7 days; any grade toxicity leading to study drug interruption lasting >7 days; any case of potential drug-induced liver injury (elevated ALT or AST in combination with either elevated bilirubin or clinical jaundice, as defined by Hy's law); treatment-emergent ALT or AST (>3 \\u00d7 baseline value) in combination with total bilirubin (>2 \\u00d7 ULN of which \\u226535% is direct bilirubin); treatment-emergent ALT or AST >3 \\u00d7 baseline value in combination with clinical jaundice. Safety was assessed through summaries of AEs, changes in laboratory test results, changes in vital signs, and GDC-0927 exposure; all patients who received study treatment on day 1 of cycle 1 were included in the analyses. A full plasma pharmacokinetic profile was obtained for GDC-0927 and analyzed using noncompartmental methods for all patients in the study. Determination of steady-state trough concentrations (before dosing) of GDC-0927 was performed for all patients in dose-escalation and dose-expansion cohorts up to cycle 3 day 1. Objective response and clinical benefit rates were derived according to RECIST v1.1. [ 18 F]-fluoroestradiol (FES)\\u2013PET imaging was performed at baseline and at cycle 2 day 3 (\\u226518 hours after dosing) in dose escalation to quantify ER expression in tumors and to assess for pharmacodynamic response. The mean percent change from baseline in SUV Max (corrected for background activity) for up to five index lesions was calculated for each patient as described elsewhere ( 17 ). Disease in the liver and some areas of the GI tract was considered unevaluable in the FES analysis because of confounding normal physiologic uptake of the tracer. Protein levels for ER, progesterone receptor (PR), and Ki67 were measured by IHC in baseline and available on-treatment (cycle 2 or 3, day 8) biopsies. All IHC cases were stained by a single laboratory and scored by a single pathologist to avoid interlaboratory and interobserver variability. Ki67 was scored as percentage of positively stained tumor cells. ER and PR protein levels are represented by Histoscores (H-scores) calculated with a range of 0\\u2013300 using the formula: (3 \\u00d7 percentage of positively stained tumor cell nuclei at strong intensity) + (2 \\u00d7 percentage of positively stained tumor cell nuclei at intermediate intensity) + (1 \\u00d7 percentage of positively stained tumor cell nuclei at weak intensity). Gene expression analysis in available matched baseline and on-treatment biopsies was performed using the RNA Access Library Preparation Kit (Illumina) followed by paired-end (2\\u00d750b, 50M reads) sequencing on the HiSeq (Illumina). Circulating tumor DNA (ctDNA) was analyzed to gain insights into potential causal relationships between the clinical activity of GDC-0927 and resistance mechanisms to prior endocrine therapy. Hotspot mutations in ESR1 , PIK3CA , and AKT1 were analyzed in baseline, on-treatment, and end-of-treatment plasma-derived ctDNA samples using the BEAMing digital PCR assay (Sysmex Inostics, Inc.). ESR1 mutations were defined as nucleotide substitutions that resulted in the following amino acid changes: E380Q, S463P, V534E, P535H, L536H/P/Q/R, Y537C/N/S, and D538G. Similarly defined, AKT1 mutations were limited to E17K, and PIK3CA mutations included C420R, E542K, E545G/K, Q546K, M1043I, and H1047L/R/Y. A subset of 1,400 mg dose plasma samples were analyzed with Foundation Assay for Circulating Tumor DNA (FACT), a next-generation sequencing ctDNA assay (Foundation Medicine) which covers genomic alterations in full exonic coding regions of 62 commonly altered genes. Samples for pharmacokinetic analyses following a single dose of GDC-0927 were collected on lead-in days \\u22127, \\u22126, \\u22125, \\u22124, \\u22123, and on day 1 of cycle 1. Steady-state samples were collected on day 1 of cycle 2. Single-dose and steady-state pharmacokinetic parameters for GDC-0927 were estimated on the basis of concentration data and nominal time. Plasma concentrations were used to perform a noncompartmental pharmacokinetics analysis with the use of Phoenix WinNonlin (version 8.2, Certera). Antitumor activity was defined as objective response and clinical benefit rates derived according to RECIST v1.1 and summarized by dose level and cohort. Separately, the maximal response in patients with measurable disease at any time on study was reported using waterfall plots. Clinical benefit was defined as either (i) patients with confirmed responses [complete response (CR) or partial response (PR)], or (ii) patients with 24-week or longer outcomes of unconfirmed responses (uCR, uPR), stable disease (SD), or non-CR/non-progressive disease (non-CR/PD). Statistical analysis Because of the exploratory nature of this study, no confirmatory inferential analyses were planned, and no imputation for missing data was done. Descriptive statistics (such as means, medians, SDs, and ranges for continuous data and percentages for categorical data) were used to summarize patient characteristics, treatment administration/compliance, safety parameters, pharmacokinetic parameters, antitumor activity endpoints, and exploratory biomarkers. A total of 60 patients were planned for enrollment at the MTD and/or RP2D to provide a more robust safety profile and a sample size for a preliminary assessment of antitumor activity. Data availability To request access to data that support the findings of this study, see here ( https://vivli.org/members/enquiries-about-studies-not-listed-on-the-vivli-platform/ ), and for up-to-date details on Roche's Global Policy on the Sharing of Clinical Information, see here ( https://www.roche.com/innovation/process/clinical-trials/data-sharing/ ).\",\n",
            "        \"subjects\": null,\n",
            "        \"results\": \"Results Study population Between March 2015 and January 2020, a total of 42 patients received GDC-0927 in dose-escalation (600 mg, n = 3; 1,000 mg, n = 3; 1,400 mg, n = 6) and dose-expansion cohorts (1,400 mg, n = 30) at six sites in the United States and eight sites in Spain (Supplementary Fig. S1). Patients in dose expansion had advanced or MBC that had progressed following previous treatments with no more than two endocrine therapies. AIs were part of prior therapeutics for all patients ( n = 42, 100%). Patients had a median of 4 (range, 1\\u20138) prior cancer therapies across all settings (neoadjuvant, adjuvant, and metastatic), including chemotherapy ( n = 35, 83%), tamoxifen ( n = 28, 67%), CDK4/6 inhibitors ( n = 19, 45%), and fulvestrant ( n = 18, 43%). Twenty-nine patients (69%) had measurable disease. Bone-only disease was present in 6 patients (14%), and 32 patients (76%) had visceral disease. The majority of patients were identified as White ( n = 33, 79%) with a median age of 56 years (range, 38\\u201378). Other demographics and baseline characteristics, including ESR1, PIK3CA, and AKT1 mutation status, are provided in Table 1 . Representativeness of study patients is shown in Supplementary Table S1. Table 1. Baseline demographics and patient characteristics. All patients ( N = 42) Age, years Median 56 Range 38\\u201378 Race Asian 1 (2%) Black or African American 4 (10%) White 33 (79%) Unknown 4 (10%) Baseline ECOG performance status 0 35 (83%) 1 7 (17%) Number of ESR1 mutations at baseline 0 20 (48%) 1 13 (31%) 2 1 (2%) 3 3 (7%) 4 2 (5%) 5 2 (5%) 6 1 (2%) Number of PIK3CA mutations at baseline 0 28 (67%) 1 12 (29%) 2 2 (5%) AKT1 mutation status at baseline Mutation detected 4 (10%) Mutation not detected 38 (90%) MTD and study drug exposure All dose-expansion patients were treated with GDC-0927 at the MAD of 1,400 mg/day. The MTD was not reached at doses up to the MAD. The median duration of treatment was 98.5 days (range, 32\\u20131,127 days; Fig. 1 ). Four patients discontinued from the study following study termination by the sponsor, and transitioned to another Genentech-sponsored oral SERD study. Figure 1. Swimmer plot of time on study. Figure 1. Swimmer plot of time on study. Safety Forty-two patients who received at least one dose of GDC-0927 were included in the safety analysis. There were no reports of DLTs, treatment-related serious adverse events (SAE), deaths, or AEs leading to withdrawal. Overall, 37 patients (88%) experienced at least one AE regardless of causality, the most frequent of which were nausea ( n = 14, 33%), constipation, diarrhea ( n = 9, 21% each), arthralgia, fatigue, hot flush ( n = 8, 19% each), back pain, vomiting ( n = 7, 17% each), anemia, and headache ( n = 6, 14% each; Table 2 ). A total of 30 patients (71%) experienced at least one AE considered related to GDC-0927 by the investigators, the most frequent of which were nausea, hot flushes ( n = 8, 19% each), constipation ( n = 6, 14%), diarrhea, and fatigue ( n = 5, 12% each; Table 2 ). Nine patients (21%) experienced at least one grade 3 AE considered unrelated to GDC-0927, including 7 patients each with distinct AEs and 2 patients with the same AE of grade 3 back pain; there were no grade 4 or grade 5 AEs reported in this study. Seven patients (17%) experienced at least one SAE considered to be unrelated to GDC-0927. Table 2. AEs in \\u226510% of patients overall ( N = 42). Any grade Grade 3 a Adverse events regardless of causality Patients with any adverse events, n (%) 37 (88) 9 (21) Nausea 14 (33) 0 Constipation 9 (21) 0 Diarrhea 9 (21) 0 Arthralgia 8 (19) 0 Fatigue 8 (19) 0 Hot flush 8 (19) 0 Back pain 7 (17) 2 (5) Vomiting 7 (17) 0 Anemia 6 (14) 0 Headache 6 (14) 0 Aspartate aminotransferase (AST) increased 5 (12) 0 Asthenia 5 (12) 1 (2) Cough 5 (12) 0 Pain in extremity 5 (12) 0 Adverse events considered related to study drug Patients with at least one adverse event, n (%) 30 (71) 0 Total number of events 109 0 Hot flush 8 (19) 0 Nausea 8 (19) 0 Constipation 6 (14) 0 Diarrhea 5 (12) 0 Fatigue 5 (12) 0 a There were no grade 4 or grade 5 events in this study. Per protocol, AEs of special interest (AESI) were events reported to the sponsor within 24 hours after learning of the event, regardless of relationship to study drug. Causality was determined by the investigator, and causal attribution guidance was provided in the protocol. Two patients (5%) experienced AESI of grade 2 deep vein thrombosis and grade 2 jugular vein thrombosis, both considered unrelated to GDC-0927 by the investigators. There were laboratory abnormalities noted in both hematologic and clinical chemistry parameters, none of which were considered clinically significant. None of the patients experienced any clinically significant vital sign changes. None of the AEs led to withdrawal from the study or treatment. There were no dose modifications due to AEs. Seven patients (17%) experienced AEs that led to dose interruptions. Pharmacokinetics Overall, a total of 39 patients were included in the pharmacokinetic analysis and pooled according to treatment dose level; 3 patients with missing pharmacokinetic data were not included in the analysis. The mean GDC-0927 plasma concentration\\u2013nominal time profiles following single- and multiple-dose oral administration of GDC-0927 as tablets (600, 1,000, 1,400 mg) in patients with breast cancer are presented in Fig. 2A and B . The average C max and AUC 0\\u201324 of GDC-0927 increased with increasing dose in the dose range investigated in this study. Single-dose and steady-state mean pharmacokinetic parameters are presented in Supplementary Table S2. Following once-daily dosing, mean steady-state exposure (AUC 0\\u201324 ) across all dose levels was 1.6-fold higher compared with single-dose exposure, and was consistent with the observed half-life and frequency of dosing. Figure 2. GDC-0927 plasma concentration (mean \\u00b1 SD) following single dose ( A ) and at steady state ( B ). Figure 2. GDC-0927 plasma concentration (mean \\u00b1 SD) following single dose (A) and at steady state (B). Pharmacodynamics Twelve patients underwent FES-PET imaging; 9 patients with FES-avid disease at baseline had >90% reduction in FES uptake at cycle 2, day 3 while receiving GDC-0927 doses of 600 mg ( n = 1), 1,000 mg ( n = 3), and 1,400 mg ( n = 5), including 6 patients with baseline ESR1 mutations ( Fig. 3A \\u2013 C ). Three patients had no FES-avid disease at baseline despite having ER H-scores of 160, 180, and 300; this discrepancy could be due to all 3 patients having liver ( n = 2) or ovarian ( n = 1) metastatic lesions that may have been unevaluable due to high tracer uptake in adjacent normal tissue. Two of the 3 patients on study for 182 days (liver) and 204 days (ovarian) had SD and achieved clinical benefit; 1 patient had PD and was on study for 62 days (liver). Figure 3. FES-PET scans demonstrate tumor avidity at baseline and target engagement during GDC-0927 exposure. A, Target engagement across all doses; reduction of FES uptake is independent of ESR1 mutation status. B and C, Representative FES-PET scan images at baseline and on treatment (cycle 2, day 3, 18\\u201324 hours after cycle 2, day 2 dose) for 2 patients with tumors harboring ESR1 mutations dosed at 1,000 mg. Figure 3. FES-PET scans demonstrate tumor avidity at baseline and target engagement during GDC-0927 exposure. A, Target engagement across all doses; reduction of FES uptake is independent of ESR1 mutation status. B and C, Representative FES-PET scan images at baseline and on treatment (cycle 2, day 3, 18\\u201324 hours after cycle 2, day 2 dose) for 2 patients with tumors harboring ESR1 mutations dosed at 1,000 mg. Nine patients provided paired baseline and on-treatment (cycle 2 or 3, day 8) tissue samples and ER, PR, and Ki67 protein levels were evaluated by IHC (Supplementary Table S3). Matched gene expression by RNA-sequencing and IHC results were evaluated in baseline and on-treatment tumor biopsy pairs from 6 patients, 4 of whom also had baseline FES-PET. A predefined, experimentally derived 38-gene ER signature z-score was utilized to determine ER signaling pathway activity collated from a core set of 21 estrogen-induced and 17 estrogen-repressed genes ( 16 ). In general, patients showed evidence of on-target pathway modulation and reduced proliferative activity by GDC-0927 as demonstrated by decreases in these biomarkers (ER signature z-score, ER H-score, PR H-score, and Ki67 IHC) upon treatment ( Fig. 4A \\u2013 D ). PGR , the gene-encoding PR, is a downstream transcriptional target of ER and included in the ER signature z-score; analysis of this gene by itself (Supplementary Fig. S2) demonstrated a trend similar to PR IHC in the 6 patients ( Fig. 4C ). Furthermore, in an analysis of patients whose on-treatment clinical benefit status (yes/no) was available, baseline protein levels of ER and PR ( n = 21 and n = 18, respectively) were not significantly different between patients with or without clinical benefit postdose (Supplementary Fig. S3A). Figure 4. Biomarker analyses of tumor tissue and ctDNA patient samples. Changes in ER pathway activity ( A ) by gene expression and in IHC protein levels for ER ( B ), PR ( C ), and Ki67 ( D ), in matched baseline and on-treatment patient biopsies ( n = 6). A\\u2013D, Each individual patient is represented by the same line color throughout; 5 patients were dosed at 1,400 mg and 1 patient (black line) at 1,000 mg; 2 patients with baseline PR H-scores of 0 were excluded from the PR panel. MAF in paired baseline and postbaseline patient ctDNA samples assayed for ESR1 MAF ( n = 19; E ) and PIK3CA / AKT1 MAF ( n = 12; F ). E and F, Postbaseline samples were the first ctDNA sample available after treatment was initiated (either cycle 3 day 1 or end-of-treatment), available for 19 of 20 patients with baseline ctDNA samples with ESR1 mutations. MAF is the sum MAF of all eligible mutations if >1 mutation detected in a sample. A\\u2013F, Dashed lines indicate patients with clinical benefit defined as \\u226524-week outcomes of unconfirmed responses (uCR, uPR), SD, or non-CR/PD. Figure 4. Biomarker analyses of tumor tissue and ctDNA patient samples. Changes in ER pathway activity (A) by gene expression and in IHC protein levels for ER (B), PR (C), and Ki67 (D), in matched baseline and on-treatment patient biopsies (n = 6). A\\u2013D, Each individual patient is represented by the same line color throughout; 5 patients were dosed at 1,400 mg and 1 patient (black line) at 1,000 mg; 2 patients with baseline PR H-scores of 0 were excluded from the PR panel. MAF in paired baseline and postbaseline patient ctDNA samples assayed for ESR1 MAF (n = 19; E) and PIK3CA/AKT1 MAF (n = 12; F). E and F, Postbaseline samples were the first ctDNA sample available after treatment was initiated (either cycle 3 day 1 or end-of-treatment), available for 19 of 20 patients with baseline ctDNA samples with ESR1 mutations. MAF is the sum MAF of all eligible mutations if >1 mutation detected in a sample. A\\u2013F, Dashed lines indicate patients with clinical benefit defined as \\u226524-week outcomes of unconfirmed responses (uCR, uPR), SD, or non-CR/PD. Clinical activity Twelve patients (29%) achieved clinical benefit, defined as either (i) patients with confirmed responses (CR or PR), or (ii) patients with 24-week or longer outcomes of unconfirmed responses (uCR, uPR), SD, or non-CR/PD ( Fig. 1 ). Four patients (9.5%) with clinical benefit received treatment on study beyond 2 years (34.2, 35.8, 36.2, and 37 months). There were no confirmed CRs or PRs on this study. Seventeen patients showed a confirmed best overall response of SD (41%) that included patients dosed at 600 mg ( n = 1), 1,000 mg ( n = 2), and 1,400 mg ( n = 14). Five patients showed a confirmed best overall response of non-CR/PD (14%); all 5 patients were from the 1,400 mg dose level. All 42 patients were included in the PFS analysis. The earliest contributing event was disease progression in 34 patients (81%); the remaining 8 patients (19%) were without any event. The median PFS was 3.3 months [95% confidence interval (CI): 1.9\\u20135.3 months]. ctDNA and clinical activity Baseline patient ctDNA samples were available for all patients in this study and samples from 22 (52%) patients had at least one detectable ESR1 mutation ( Table 1 ). The predominant ESR1 mutations were D538G, Y537S, and Y537N, and eight (19%) of the samples classified as ESR1 mutant had more than one ESR1 mutation detected demonstrating tumor heterogeneity (Supplementary Fig. S4A and S4B). PIK3CA and AKT1 mutations were also observed in 14 (33%) and four (10%) baseline patient ctDNA samples, respectively. Six of the 22 (27%) patients with baseline ctDNA ESR1 mutation(s) achieved clinical benefit, as well as 6 of the 20 (30%) patients without detectable baseline ctDNA ESR1 mutations. Similar to the lack of a correlation between baseline ER and PR protein levels and clinical benefit, baseline ESR1 mutant allele frequencies (MAF) were not significantly different between patients with or without clinical benefit ( n = 23; Supplementary Fig. S3A). At the 1,400 mg dose, the median PFS was 3.52 months (95% CI: 1.8\\u20136.4) in patients with tumors without detectable ESR1 mutations ( n = 17), and 3.55 months (95% CI: 1.8\\u20136.0) in patients with tumors expressing baseline ESR1 mutations ( n = 19). ESR1 data were available for 36 of 36 patients dosed at 1,400 mg (Supplementary Fig. S5). MAF generally declined in the first postbaseline (either cycle 3 day 1 or end-of-treatment) ctDNA samples collected for both ESR1 (16 of 21 patients; postbaseline sample unavailable for 1 patient with baseline ESR1 mutation detected) and PIK3CA/AKT1 (8 of 13; postbaseline sample unavailable for 2 patients with baseline PIK3CA/AKT1 mutations detected; Fig. 4E and F ). The majority (15/16) of the ESR1 MAF reductions were greater than 95% relative to baseline and while 5 of 16 (31%) of the patients with a decline in ESR1 MAF also achieved clinical benefit, in general, a decline in ESR1 MAF did not correlate with clinical benefit (Supplementary Fig. S3B). Six patients displayed increased ESR1 MAF in the first postbaseline sample, among whom 1 (17%) achieved clinical benefit and 4 had an increase in more than one variant. Clinical response was not contingent on ESR1 MAF dynamics ( \\u03c7 2 P = 0.35, OR = 3.9; Supplementary Fig. S3B). Of the 23 patients with baseline samples available (all at the 1,400 mg dose) for analysis using the FACT ctDNA assay, 19 harbored between one and four total short variant gene alterations at baseline, the majority of which were substitutions (Supplementary Fig. S6A and S6B). The top mutated genes following ESR1 / PIK3CA / AKT1 were TP53 (26.1%), CDH1 (8.7%), NF1 (4.3%), and PTEN (4.3%; Supplementary Fig. S6C). Each patient's mutation landscape and the associated clinical response for the patient are indicated in Supplementary Fig. S6C. ESR1 and PIK3CA mutations were mostly mutually exclusive (log 10 OR \\u22120.363). Consistent with BEAMing results, ESR1 mutations were less prevalent in on-treatment samples (35% on-treatment vs. 52% pretreatment). In a comparison of 53 matched ctDNA samples with both BEAMing and FACT results collected at baseline, cycle 3 day 1 or end-of-treatment, an overall concordance of 89% for ESR1 mutation status (mutant vs. no mutation detected) was observed between the two assays. Six samples with ESR1 mutation(s) detected via BEAMing had no alterations detected via FACT, which may be a reflection of the differing limits of detection between digital PCR and next-generation sequencing assays (Supplementary Fig. S4A and S4B).\",\n",
            "        \"discussion\": \"Discussion Resistance to endocrine therapeutics ( 18 ) is a major clinical challenge in the MBC setting. While there is heterogeneity in the mechanisms of resistance ( 19 ), those associated with retention or reactivation of ER signaling, including acquisition of ESR1 mutations, are potentially sensitive to SERD therapeutics. Several SERDs have recently entered clinical development for ER + advanced and MBC, highlighting an immense interest in next-generation SERDs resulting from the fulvestrant experience ( 20 ). The limitations of fulvestrant include poor pharmaceutical properties dictating intramuscular injections, thereby limiting the administered dose and potentially achieving an incomplete receptor blockade. Therapeutics that are mechanistically similar to fulvestrant but have an improved bioavailability may have best-in-disease potential ( 21 ). In this study, the pharmacokinetic parameters were predictable, with increases in GDC-0927 exposure with dose. This study enrolled a heavily pretreated patient population with ER + /HER \\u2212 advanced or MBC; prior therapeutics included AIs for all patients while many patients received radiotherapy, tamoxifen, fulvestrant, CDK4/6 inhibitors, PI3K pathway inhibitors, and a few patients received investigational therapeutics. Encouraging antitumor activity was seen in this study, which was comparable in subgroups with or without ESR1 mutations. GDC-0927 had a safe AE profile in patients who were treated orally, once daily, at doses up to 1,400 mg/day, which was the MAD. The MTD was not reached. AEs were predominately grade 1 or 2 in severity. There were no grade 4 or 5 AEs. No DLTs, treatment related SAEs, AESIs, death due to AEs, or events leading to withdrawal from the study treatment were reported. Most of the AEs were managed by dose interruption, medication, and/or supportive care. This study demonstrated that GDC-0927 greatly reduced ER availability, as measured by FES-PET, which indicated a reduction of >90% in uptake by the tumors postdose on cycle 2, day 3, in comparison to baseline. This reduction in ER availability was consistent, regardless of the ESR1 mutation status. FES-PET response was unaffected by the presence of PIK3CA mutations. Other recent studies have also used FES-PET to assess the pharmacodynamic effects of novel SERDs ( 22, 23 ), including elacestrant ( 24 ). The presence of mutant ER was supported by ctDNA analyses that demonstrated the predominant ESR1 mutations in this study to be D538G, Y537S, and Y537N at baseline. Seven of 16 patients (44%) who exhibited decreased ESR1 MAF at their first on-treatment visit showed SD. However, similar to another study ( 25 ), ESR1 MAF dynamics in this small study were not predictive of clinical response. Reduced ER and PR levels, Ki67 positivity, and expression of ER target genes were observed in on-treatment biopsies, further demonstrating the on-target activity of GDC-0927. While ER tumor activity and ctDNA dynamics were effective in assessing GDC-0927 dosing, baseline parameters including ER IHC, PR IHC, ESR1 MAF levels, and dynamics were not predictive of clinical benefit, strengthening the need to develop better prognostic biomarkers in late line HR + /HER2 \\u2212 patients. Furthermore, the conclusions from the biomarker analyses in this study are limited because of small sample sizes. Larger studies, such as those ongoing with the next generation of oral SERDs, will be better powered to interrogate both baseline levels and dynamics of these biomarkers and draw conclusions with clinical outcomes. The current development of ctDNA assays that infer tumor ER target gene expression (e.g., cell-free fragmentomics and epigenomics) may serve as more informative biomarkers in the near future. One key parameter in drug discovery surrounding SERD compounds has been the optimization of ER degradation, along with attention to pharmacokinetic properties. Fulvestrant is a full ER antagonist; it antagonizes estrogen while lacking any ER agonist potential under estrogen-deprived conditions. Full ER suppression by fulvestrant is achieved by immobilizing ER ( 16 ) and promoting ER protein degradation ( 26 ). Tamoxifen is a partial ER antagonist that does suppress estrogen, but promotes weak ER agonism ( 27 ). In preclinical studies, GDC-0927 demonstrated ER degradation in all ER + cell lines tested and retained a full antagonist profile, even in lines where GDC-0810 displayed partial agonism; notably, GDC-0927 lacked ER agonism in the uterus. While FES-PET imaging showed >90% reduction in FES uptake, ER protein expression only declined by an average of approximately 40%. Importantly, ER IHC represents just a single tumor snapshot, whereas FES-PET accounts for global tumor heterogeneity. Regardless, these results may suggest that some ER protein can remain in the tumor following SERD treatment and exhibit minimal binding of estradiol, and that FES-PET studies may represent a combination of effects on ligand binding and ER protein levels. This is consistent with evidence from previous studies showing that ER degradation by fulvestrant is not the mechanistic driver of ER antagonism ( 16, 28 ). Several oral SERDS are in development, and elacestrant (GS2-02) was recently approved by the U.S. FDA for patients with ER + , HER2 \\u2212 , ESR1 -mutated advanced or MBC as evaluated in the EMERALD trial ( 29 ), with elacestrant becoming the first oral SERD with improved efficacy over fulvestrant in the second- and third-line MBC setting. The AMEERA-5 trial ( 30 ) evaluating amcenestrant (SAR439859) in a combination study was discontinued in mid-2022 when results did not meet the continuation criteria and development of the amcenestrant program was subsequently stopped. The SERENA-6 ( 31 ) and SERENA-4 ( 32 ) trials are evaluating camizestrant (AZD9833) in combination treatments, with results expected in the near future; results for the SERENA-2 study showed camizestrant 75 and 150 mg had a median PFS of 7.2 and 7.7 months, respectively, compared with 3.7 months for patients treated with fulvestrant ( 33 ). The EMBER-3 trial is evaluating imlunestrant (Ly3484356). In a phase Ia/Ib/IIa study, GDC-0810 was reported to have a favorable safety and tolerability profile with preliminary antitumor activity in heavily pretreated patients with ER + advanced/MBC, regardless of ESR1 mutation status, although the program has since been discontinued ( 34 ). Primary results from the acelERA trial investigating giredestrant (GDC-9545) did not meet its primary endpoint, but giredestrant showed a numerical improvement over the comparator arm, which was more pronounced in the population with ESR1 -mutated tumors ( 35 ). The sponsor discontinued the development of GDC-0927 in October 2017. The decision to halt enrollment was not due to any safety concerns, but was based on the totality of clinical data, including data from another SERD under development ( 36\\u201338 ). Four patients who were still receiving GDC-0927 and discontinued because of study termination were transitioned to a different Genentech sponsored oral SERD study with giredestrant.\",\n",
            "        \"conclusion\": null\n",
            "    },\n",
            "    {\n",
            "        \"pmcid\": \"PMC2383879\",\n",
            "        \"abstract\": null,\n",
            "        \"intro\": null,\n",
            "        \"method\": \"Methods Early-onset CAD family-based sample (GENECARD) GENECARD is a collaborative study involving six international investigative sites that make up the GENECARD Study Network. The study is coordinated at Duke University. All study participants signed a consent form approved by the responsible institutional review board or local ethics committee. The study design has been previously reported [ 27 ]. Briefly, the initial genome-wide linkage screen within the GENECARD study was composed of 493 affected sibling pairs in 420 families, where at least two siblings met the criteria for early-onset CAD (eoCAD) [ 15 ]. In order to obtain a more genetically homogenous sample, using a priori defined analyses, our eoCAD sample was stratified into acute coronary syndrome (ACS) families. The presence of ACS was defined as diagnosis of myocardial infarction or unstable angina in at least two affected siblings (228 families) [ 15 ]. ACS is a serious manifestation of CAD which includes the subgroup of MI, and is diagnosed by the presence of at least two of the following three signs/symptoms: chest pain typical of CAD, changes on electrocardiogram, and/or positive serum biomarkers for myocardial infarction (MI). Of the 985 affected individuals in the study, 697 met ACS criteria, with 71 individuals with unstable angina only (89% with MI). Given these data, and since MI and ACS represent similar clinical phenotypes, for simplicity we consider the ACS phenotype as primarily a myocardial infarction phenotype. In order to increase the power to detect association in this sample, we included additional individuals from the selected ACS families that were sampled after the initial linkage screen was completed. A total of eight additional parents and 28 additional unaffected siblings were added to the 228 ACS families. Furthermore, two additional families qualified for the ACS phenotype since the original screen and were also included in the analysis. The characteristics of the eoCAD (n = 420) and ACS (n = 230) GENECARD families are summarized in Table 1 . Table 1 Clinical characteristics of samples used in this study Variable eoCAD Probands ACS Probands MI Cases Controls Aorta Samples (n = 422) (n = 228) (n = 368) (n = 289) (n = 88) Age of onset (SD) 44.2 (5.8) 43.9 (6.1)\\u2020 51.5 (10.2)\\u2020 69.0 (7.0) exam 37.8 (13.2) Race: Caucasian 92.9% 93%\\u2020 100%\\u2020 100.0% 87.9% African American 2.1% 2.2% 0.0% 0.0% 8.6% American Indian 4.0% 3.0% 0.0% 0.0% 0.0% Asian 0.7% 0.9% 0.0% 0.0% 2.0% Hispanic 0.0% 0.0% 0.0% 0.0% 3.5% Other (unknown) 0.2% 0.4% 0.0% 0.0% 0.0% Sex: Male 70.6% 72%\\u2020 83%\\u2020 43.0% 55.2% Family history of CAD 100.0% 100%\\u2020 58%\\u2020 28.0% Body Mass Index (SD) 29.9 (5.8) 29.9 (6.3) 29.9 (6.8) 28.2 (6.5) Smoking 78.0% 84.0%\\u2020 72%\\u2020 39.0% Diabetes 22.3% 22.0% 29.0% 15.0% Hypertension 56.9% 57%\\u2020 66%\\u2020 66.0% MI 65.9%* 89.0%*\\u2020 100%\\u2020 0.0% Data not available Sys/Dias BP (SD) 140.1/84.0* (21.5/11.4) 140.6\\u2020/87.0* (22.0/12.0) 138.0\\u2020/74.0 (24.0/12.0) 149.0/76.0 (23.0/13.0) Total Cholesterol (SD) 239.1 (64.8) 235.0\\u2020 (67.0) 189.0\\u2020 (56.0) 189.0 (43.0) LDL (SD) 146.1 (54.3) 161.0\\u2020 (125.0) 108\\u2020 (42.0) 105.0 (35.0) HDL (SD) 38.67 (16.3) 40.0 (14.0) 39.0 (12.0) 51.0 (19.0) Triglycerides (SD) 230.1 (138.8) 221.0 (157.0) 226.0 (239.0) 169.0 (133.0) *significant difference between eoCAD and ACS probands (p \\u2264 0.05) \\u2020significant difference between ACS probands and MI cases (p \\u2264 0.05) MI case-control sample (CATHGEN) CATHGEN participants were recruited sequentially through the cardiac catheterization laboratories at Duke University Hospital (Durham, North Carolina, United States) with approval from the Duke Institutional Review Board. All participants undergoing catheterization were offered participation in the study and signed informed consent. Medical history and clinical data were collected and stored in the Duke Information System for Cardiovascular Care database maintained at the Duke Clinical Research Institute [ 28 ]. Controls and cases were chosen on the basis of extent of CAD as measured by the CAD index (CADi). CADi is a numerical summary of coronary angiographic data that incorporates the extent and anatomical distribution of coronary disease [ 29 ]. CADi has been shown to be a better predictor of clinical outcome than extent of CAD [ 30 ]. Affected status was determined by the presence of significant CAD defined as a CADi \\u2265 32 [ 31 ], as we have used for previous studies [ 22 , 23 ]. For patients older than 55 years of age, a higher CADi threshold (CADi \\u2265 74) was used to adjust for the higher baseline extent of CAD in this group. Given the linkage to MI in the GENECARD study, we also selected a subgroup of Caucasian individuals who experienced MI from the overall CATHGEN group, similar to the GENECARD ACS families. MI was defined as documentation of MI in the medical history. Controls were defined as \\u226560 years of age, with no CAD as demonstrated by coronary angiography and no documented history of cerebrovascular or peripheral vascular disease, myocardial infarction, transplant, or interventional or surgical coronary revascularization procedures [ 32 ]. A comparison of clinical characteristics between GENECARD probands and CATHGEN MI cases and unaffected controls is presented in Table 1 . As a replication data set, the CATHGEN sample provides 80% power to detect effect sizes between 1.25 and 1.45, for allele frequencies ranging from 0.1 to 0.5. The CATHGEN MI cases were also stratified into a young affected group (CAD AOO \\u2264 55), which provides a consistent comparison for the GENECARD family study. Human donor aorta samples and expression Human aorta samples were collected from heart transplant donors as previously described [ 33 ]. DNA and RNA were isolated from each sample. Eighty-eight samples in total were analyzed representing fifty-eight unique samples, as 30 individuals had more than one sample. As harvested tissue was obtained from deceased heart donors, the clinical data associated with these aortas is very limited and consists of age, sex, and race (Table 1 ). Genotyping was performed using Applied Biosystems Taqman allelic discrimination assays and expression profiling was performed using Affymetrix GeneChip U95Av2 (Affymetrix, Santa Clara, CA). Expression signal intensity values were log 2 transformed and normalized using quantile normalization. We analyzed cis effects of FAM5C variants on FAM5C expression using the Affymetrix tag 34442_at, representing the 3' end of FAM5C . Because some subjects had multiple samples while others did not, we treated each individual and sample separately. The expression level of the tag was modeled using multiple linear regression including age, sex, race and additive genotype. To account for repeated measures (i.e., multiple sections per subjects) and as a validation, a mixed model as implemented in the SAS PROC MIXED procedure was utilized (SAS Institute Inc., Cary, NC). Peak-wide SNP selection The approximate 30 Mb region of linkage on chromosome 1q contains 138 genes (NCBI Build 36 in Ensembl). We used an iterative fine mapping approach that involved selecting SNPs with a minor allele frequency of at least 5% in the Caucasian population spaced at 100 kilobases (kb) [ 34 ]. These SNPs were intercalated with at least one previously identified and confirmed (validated) SNP to represent positional candidate genes from NCBI Build 35 (120 genes). When possible, coding SNPs, SNPs that altered a splice site or that were contained within putative regulatory regions of multi-species sequence conservation (chimp, mouse, rat, dog, chicken, Fugu, zebrafish) were selected for genotyping. If the gene was greater than 100 kb in size, multiple SNPs were selected using the same criteria. Additionally, SNPs were selected from regions that do not contain annotated genes at an average SNP to SNP distance of 100 kb, while taking into account regions of multi-species sequence conservation. Using these criteria, 231 intragenic and 226 intergenic SNPs were selected for genotype analysis within the chromosome 1q linkage region with an average spacing of 66 kb. Twenty-seven genes contained validated coding SNPs with allele frequencies >5%; twenty-two genes contained synonymous SNPs with allele frequencies >5%; eleven SNPs are found within untranslated regions (5' or 3' UTR); and one hundred and seventy-one intronic SNPs were selected, the reference allele is conserved within fifty-five of these (Additional files, Table 1 ). SNP genotyping A total of 457 SNPs were genotyped at two genotyping facilities. 384 SNPs were genotyped at the University of Washington through a grant from SeattleSNPs (see Availability and requirements section for URL). Genotyping in the GENECARD sample was performed by SeattleSNPs using the Illumina BeadStation 500 G SNP genotyping system (Illumina, San Diego, California, United States). Each Sentrix Array generates 384 genotypes for 96 individuals; within each individual array experiment, four samples purchased from Coriell were included as quality control samples. The Coriell samples were of Centre d'Etude du Polymorphisme Humain (CEPH) (Caucasian) origin, Asian and Hispanic ethnicity and were used for gender checking and to assess sample quality. In addition, one sample per plate was duplicated for internal quality control. These samples were used to identify possible sample plating errors and genotype calling inconsistencies. Twenty-two SNPs did not cluster well and were not called. The call rate for the remaining 362 SNPs was 99.8%. One replicate error was identified in SNP rs2370025, all other SNPs yielded greater than 99.999% concordance among the replicates. Ninety-five additional inter- and intragenic SNPs were assayed at the Center for Human Genetics using Taqman allelic discrimination assays. A total of 15 quality control samples \\u2013 composed of six reference genotype controls in duplicate, two CEPH pedigree individuals, and one no-template sample \\u2013 were included in each quadrant of the 384-well plate. SNPs that showed mismatches on quality-control samples were reviewed by an independent genotyping supervisor for potential genotyping errors. All SNPs examined were successfully genotyped for 95% or more of the individuals in the study. Error rate estimates for SNPs meeting the quality control benchmarks were determined to be less than 0.2%. Statistical analysis All SNPs were tested for deviations (p \\u2264 0.0001) from Hardy-Weinberg equilibrium (HWE) in the affected and unaffected race-stratified groups. Two SNPs deviated from HWE in the peak-wide association study, rs1503122 (cases) and rs438781 (cases and controls). Neither of these SNPs was statistically significant in our association analysis. Interestingly, rs438781 lies within a reported copy number polymorphism, which is most likely the cause of the disequilibrium. In CATHGEN, one SNP, rs9427746, significantly deviated from HWE (p < 0.0001) in the Caucasian control group. This deviation was neither due to mixed racial backgrounds as all of the samples were from Caucasian origin, nor was it due to assay dropout as all of our controls are randomly interspersed with our cases in the 384 well plates. This deviation is either a significant departure representing an association or due to random chance. Linkage disequilibrium between pairs of SNPs was assessed using the Graphical Overview of Linkage Disequilibrium package [ 35 ] and displayed using Haploview [ 36 ]. In the initial screen, two-point linkage using a dominant and recessive model was performed using Fastlink [ 37 , 38 ] and Homog [ 39 ] and family-based association was tested using the Association in the Presence of Linkage (APL) test [ 40 ]. We analyzed the GENECARD families together regardless of race because these families formed the basis of the original linkage evidence. We chose to use the APL test for family based association because this test incorporates data from affected sibling pairs with available parental data and unaffected siblings in the analyses, effectively using all available information in the GENECARD families. The APL software infers missing parental genotypes, appropriately accounts for the non-independence of affected siblings and calculates a robust estimate of the variance. APL results from markers with variance estimates of less than five are viewed as less reliable [ 41 ]. A single SNP in TROVE2 was significant in our original screen with a variance less than five and did not replicate. In addition to APL, family-based association of FAM5C SNPs was performed using the Pedigree Disequilibrium Test (PDT) [ 42 , 43 ]. We employed this additional test at the gene level in order to obtain the most information about each polymorphism. The PDT test uses related trios and discordant sibships from extended pedigrees and like other family-based association tests is robust to population stratification. In order to determine if replicated SNPs could account for the MI linkage signal we employed Linkage and Association Modeling in Pedigrees (LAMP). LAMP estimates the degree of linkage disequilibrium (LD) between a candidate SNP and the putative disease locus through joint modeling of linkage and association [ 44 , 45 ]. Allelic association in the CATHGEN MI cases and controls was evaluated using multivariable logistic regression modeling adjusted for sex, and known CAD risk factors (history of hypertension, history of diabetes mellitus, body mass index, history of dyslipidemia, and smoking history) as covariates. These adjustments could hypothetically allow us to control for competing genetic pathways that are independent risk factors for CAD, thereby allowing us to detect a separate CAD genetic effect. SAS 9.1 (SAS Institute, Cary, North Carolina, United States) was used for statistical analysis. FAM5C sequencing To identify novel SNPs within seven regions of multi-species sequence conservation localized to the last intron of FAM5C (see Additional files, Figure 1 , arrows), PCR amplicons were generated using standard conditions and sequenced using ABI Big Dye v3.1 and an ABI 3730 automated sequencer (primers available upon request). Sequencing data were analyzed using Sequencher software (GeneCodes, Ann Arbor, Michigan, United States). All amplicons were generated within 16 affected and 16 unaffected GENECARD individuals (16 Caucasian males and 16 Caucasian females). Primer sets for each region are listed in Additional files, Table 2 . Table 2 LAMP Analysis in GENECARD ACS families and replication in CATHGEN MI cases and controls SNP LOCATION (bp) Alleles LINKAGE TEST (df = 3) ASSOCIATION TEST (df = 1) OTHER LINKED VARIANTS (df = 2) CATHGEN MI Logistic regression\\u2020 CATHGEN Minor Allele Frequency Major Minor LOD pvalue LOD pvalue LOD pvalue Allele pvalue Allele OR (95% CI)* RS7514392 188,323,178 C g 2.95 0.004 0.33 0.220 1.69 0.020 - - - RS10920501 188,328,568 A t 2.98 0.003 0.96 0.040 2.27 0.005 0.018 0.612 (0.408\\u20130.919) 0.18 RS12142564 188,338,626 C t 2.80 0.005 1.43 0.010 2.53 0.003 0.322 0.799 (0.508\\u20131.257) 0.11 RS2185836 188,349,217 A g 2.89 0.004 0.76 0.060 2.82 0.002 0.273 0.791 (0.521\\u20131.202) 0.45 RS2419370 188,375,504 T c 2.50 0.009 0.56 0.110 2.49 0.003 0.003 1.925 (1.251\\u20132.961) 0.44 RS2990996 188,411,850 C a 3.19 0.002 0.16 0.400 3.18 0.001 0.115 1.441 (0.915\\u20132.271) 0.50 RS815343 188,422,256 C t 2.97 0.003 0.34 0.210 2.31 0.005 0.005 0.574 (0.392\\u20130.842) 0.39 RS1891586 188,430,617 T g 2.95 0.004 0.04 0.700 2.93 0.001 0.003 0.563 (0.385\\u20130.823) 0.23 RS9427746 188,438,263 G c 2.86 0.004 0.01 0.810 2.50 0.003 0.007 0.587 (0.399\\u20130.863) 0.40 RS10920678 188,506,530 G a 2.99 0.003 0.01 0.850 2.77 0.002 0.159 0.753 (0.508\\u20131.117) 0.43 RS480692 188,526,284 T c 3.29 0.002 0.41 0.200 3.42 0.000 0.028 1.591 (1.052\\u20132.405) 0.43 RS12724000 188,618,300 C t 2.78 0.005 0.04 0.660 2.60 0.003 0.031 0.645 (0.434\\u20130.961) 0.17 RS872177 188,659,068 T a 2.93 0.004 0.05 0.600 2.90 0.001 0.188 0.776 (0.533\\u20131.132) 0.31 RS2134098 188,660,894 G c 2.82 0.005 0.89 0.040 3.00 0.001 0.131 0.679 (0.411\\u20131.122) 0.08 RS10920722 188,694,717 A t 2.65 0.007 0.07 0.600 2.37 0.004 0.009 0.589 (0.397\\u20130.874) 0.19 RS10920725 188,707,160 T c 2.65 0.007 0.13 0.400 2.45 0.004 0.007 0.584 (0.394\\u20130.865) 0.20 RS11581737 188,726,896 G a 2.87 0.004 0.18 0.400 2.76 0.002 0.031 0.642 (0.429\\u20130.960) 0.16 *Odds Ratio (OR) calculated for minor allele, - not tested \\u2020adjusted for sex, history of hypertension, history of diabetes mellitus, body mass index, history of dyslipidemia, and smoking history Figure 1 Chromosome 1 peak-wide mapping of genes involved in myocardial infarction . 457 SNPs were genotyped across the 1 LOD down region of linkage on chromosome 1 in the GENECARD ACS sample. Association was assessed using APL. The -log 10 of the p-value is plotted according to chromosome position (closed circles). Markers with a p-value less than 0.05 fall above the dashed line; labels identify these genes and intergenic SNPs. Markers in bold exhibited modest evidence for linkage (LOD \\u2265 1) in two point analysis. Sixteen markers that were associated (p \\u2264 0.05) in the GENECARD ACS sample were genotyped in a MI case-control population (open circles). Markers which show replication are underlined. Aortic endothelial and smooth muscle cell culture Cryopreserved human aortic smooth muscle (AoSMC) and human aortic endothelial cells (HAEC) were obtained from Lonza (Walkersville, MD) at passage 3 and cultured using SGM-2 or EGM-2 media (Lonza) until the indicated passage. Cells were grown in T-75 SoLo Flasks Nunclon\\u2122\\u0394 (NUNC, Rochester, NY) according to manufacturer's instructions (see Availability and requirements section for URL). Reverse transcriptase PCR (RT-PCR) RNA from 20 human tissues (Human Total RNA Master Panel II) was obtained from Clontech (Mountain View, CA). Lot number 6020107 contained RNA from the following tissue types: Fetal Brain, Spinal Cord, Skeletal Muscle, Lung, Trachea, Fetal Liver, Liver, Bone Marrow, Kidney, Thymus, Thyroid, Adrenal Gland, Salivary Gland, Uterus, Testes, Prostate, and Placenta. RNA from cultured aortic smooth muscle and endothelial cells was isolated using a Ribopure Kit from Applied Biosystems (Foster City, CA). cDNA was synthesized using the protocol for cDNA synthesis in the Illumina Total Prep RNA Amplification kit from Applied Biosystems (Foster City, CA). Primers corresponding to exon 6 and exon 7 of FAM5C were used for PCR from cDNA (5'-ACAACAGTGACTTTGAGGAATCAGA-3' and 5'-GCGCTGAAAATTAGAATCCATTG-3'). Primers corresponding to exons 8 and 9 of GAPDH were used as a template control (5'-CTCCTCCACCTTTGACGCTG-3', and 5'-AGGGGAGATTCAGTGTGGTG-3'). 20 ng of cDNA template was used for each reaction and Platinum Taq polymerase (Invitrogen, Carlsbad, CA) was used for all PCR reactions. PCR reaction conditions were as follows: Step 1: 95\\u00b0C for 5 minutes, Step 2: 30 cycles ( FAM5C ) or 20 cycles (GAPDH) of 95\\u00b0C for 1 minute followed by 62\\u00b0C for 1 minute, and 72\\u00b0C for 1 min, and Step 3, 72\\u00b0C 10 min. FAM5C real time RT-PCR Applied Biosystems Taqman Gene expression assays were used to perform quantitative (real time) RT-PCR (FAM5C, Hs00982332_m1 and GAPDH, Hs02758991_g1). The following reaction components were used for each probe: 2 uL cDNA, 5 ul Custom TaqMan SNP Master Mix (Applied Biosystems, Foster City, CA), 0.5 ul of assay, and 2.5 ul water. Reactions were performed in a single 384 well plate in triplicate using an ABI PRISM \\u00ae 7900 HT Sequence Detection System. PCR reaction conditions were as follows: Step 1: 50\\u00b0C for 2 minutes, Step 2: 95\\u00b0C for 10 minutes, Step 3: 40 cycles of 95\\u00b0C for 15 seconds followed by 60\\u00b0C for 1 minute. Expression relative to GAPDH was calculated using 2 \\u0394Ct [ 46 ] and levels were normalized to heart expression.\",\n",
            "        \"subjects\": null,\n",
            "        \"results\": null,\n",
            "        \"discussion\": null,\n",
            "        \"conclusion\": null\n",
            "    },\n",
            "    {\n",
            "        \"pmcid\": \"PMC2693442\",\n",
            "        \"abstract\": null,\n",
            "        \"intro\": null,\n",
            "        \"method\": \"Method Tissue samples Twenty paraffin blocks of cases previously diagnosed as Warthin's tumor were collected from the archives of the Oral Pathology Department, Faculty of Dentistry, Ain Shams University. Haematoxylin and Eosin stained sections were prepared for each specimen to confirm the diagnosis. From each of the selected paraffin blocks three sections of 4 \\u03bcm thick were cut and mounted on positively charged slides (Biogenex Corp, California, USA) for immunohistochemical staining. Reagents Monoclonal antibodies against mdm2, p27 Kip1 and bcl-2 were used. These antibodies recognized their corresponding proteins in paraffin-embedded sections. Monoclonal IgG-Kappa antibody against mdm2 was purchased from (Zymed Laboratories Inc., USA). It recognizes epitope located within the amino acid 26\\u2013169 of the human mdm2 protein. While monoclonal antibody against p27 Kip1 was purchased from (DAKO Corporation, USA) and monoclonal antibody against bcl-2 was purchased from (Zymed Laboratories Inc., USA). A streptavidin-biotin immunoperoxidase staining system was used for immuno-detection. It included the following reagents: \\u2022 Monoclonal Linking Reagent: biotinylated anti-mouse immunoglobulin \\u2022 Streptavidin Enzyme Label: horseradish peroxidase-conjugated streptavidin \\u2022 Buffered Substrate: Hydrochloric acid (HCl) buffer, pH 7.5 \\u2022 Enzyme Substrate: 0.6% hydrogen peroxide \\u2022 DAB Chromogen Concentrate: 3.3' diaminobenzidine in chromogen solution \\u2022 Buffered Wash Solution: phosphate buffered saline (PBS), 0.1 mol/l, pH 7.4 \\u00b1 2 The detection systems were purchased from DAKO Corporation, USA and Zymed Laboratories Inc., USA. The retrieval solution was purchased from BioGenex, USA. Immunohistochemistry Sections were deparaffinized in hot xylene. Rehydration of slides was performed by immersing slides in descending alcohol concentrations and then washed in distilled water. The sections were rinsed in two washes of phosphate-buffered saline for 5 minutes each (PBS; pH 7.4). For staining with mdm2 antibody, the sections were microwave antigen retrieved with citrate buffer solution (pH 4.8 for 15 minutes). P27 Kip-1 and bcl-2 antibodies required antigen retrieval by microwave processing in 0.2 M glycine buffer (Glyca, pH 6.2 \\u00b1 0.2, for 15 min.). Sections were again rinsed in two washes of PBS. Endogenous peroxidase activity was blocked with 3% hydrogen peroxide in water. The primary antibodies anti-mdm2, anti-p27 Kip-1 , and anti-bcl-2 were applied and the slides were incubated in a humid chamber overnight at room temperature. The sections were then washed in PBS. The secondary (Link) antibody (Biotinylated anti-mouse IgG, dilution 1:25) was then applied and the slides were incubated for 20 minutes at room temperature. After washing with PBS, the slides were incubated with peroxidase-labelled streptavidin complex for 20 minutes at room temperature, and then washed with PBS. The slides were then incubated with a solution of 3% diamino-benzidine (DAB) as the chromogen for 20 minutes. Finally, the tissues were lightly counter stained with Mayer's Haematoxylin and mounted with DPX permanent mounting medium. Assessment In this study, the immunostaining of each antibody was assessed in 5 serial sections from each specimen. For mdm2, tumor cells were considered immunopositive when they displayed a brownish nuclear and/or cytoplasmic immunoreactivity, p27 Kip-1 immunostaining was considered positive when tumor cells showed a nuclear immunoreactivity while bcl-2 was considered positive if the tumor cells showed cytoplasmic or perinuclear localization of immunoreactivity.\",\n",
            "        \"subjects\": null,\n",
            "        \"results\": null,\n",
            "        \"discussion\": null,\n",
            "        \"conclusion\": null\n",
            "    },\n",
            "    {\n",
            "        \"pmcid\": \"PMC3145824\",\n",
            "        \"abstract\": \"Notch signaling is often and aberrantly activated by hypoxia during tumor progression; however, the exact pathological role of hypoxia-induced Notch signaling in tumor metastasis is as yet poorly understood. In this study, we aimed to define the mechanism of Notch ligand activation by hypoxia in both primary tumor and bone stromal cells in the metastatic niche and to clarify their roles in tumor progression. We have analyzed the expression profiles of various Notch liagnds in 779 breast cancer patients in GEO database and found that the expression of Jagged2 among all five ligands is most significantly correlated with the overall- and metastasis-free survival of breast cancer patients. The results of our immunohistochemical (IHC) analysis for Jagged2 in 61 clinical samples also revealed that both Jagged2 and Notch signaling were strongly up-regulated at the hypoxic invasive front. Activation of Jagged2 by hypoxia in tumor cells induced EMT and also promoted cell survival\",\n",
            "        \"intro\": \"Introduction Oxygen homeostasis plays a critical role in carcinogenesis and tumor progression ( Schindl et al., 2002 ; Zhong et al., 1999 ; Bertout et al., 2009 ). Hypoxia is known to promote aggressive phenotypes of tumor and induces their invasiveness and metastasis; however, the exact molecular mechanism of hypoxia-mediated metastasis is not well defined yet. Under hypoxic condition, tumor cells up-regulate the expression of hypoxia-inducible factors (HIFs) which function as master regulators to control a series of down-stream events such as activating angiogenic and cell survival pathways as well as promoting epithelial to mesenchymal transition (EMT) ( Pouyssegur et al., 2006 ; Haase et al., 2009 ). Hypoxic areas are often found in the central necrotic regions of a tumor mass ( Lambin et al., 1998 ; Tomes et al., 2003 ). Interestingly, however, the expression levels of several hypoxia-related proteins, including HIF-1\\u03b1, Glut-1(Glucose transporter 1) and CA9 (Carbonic anhydrase 9) were also found to be frequently elevated at the margins of solid tumors which is known as invasive front ( Horree et al., 2007 ). In this context, it is noteworthy that cancer stem-like cells are often seen at the invasive front in several types of cancers ( Das et al., 2008 ). Bone is one of the most frequently metastasized organs in breast cancer due to its unique microenvironment ( Roodman, 2004 ). Many regions of the bone are known to be hypoxic and this hypoxic niche in bone microenvironment is believed to promote self renewal of hematopoietic stem cells ( Parmar et al., 2007 ). Therefore, cancer stem-like cells may also take advantage of such an environment and establish metastatic colony in the bone. In fact, Das et al used \\u201cinjured conditioned medium\\u201d which is derived from bone marrow stromal cells exposed to hypoxia and oxidative stress to isolate post-hypoxic side population of cancer cells with stem cell property, indicating a potential role of hypoxic bone microenvironment in promoting cancer stem cell growth ( Das et al., 2008 ). Notch signaling is an evolutionarily conserved pathway that plays a critical role in self-renewal of stem cells, and aberrant expression of this pathway is often observed in various types of cancers ( Leong & Karsan, 2006 ; Koch & Radtke, 2007 ; Kiaris et al., 2004 ; Klinakis et al., 2006 ; Lee et al., 2007 ; Stylianou et al., 2006 ). Notch signaling is also known to be activated under hypoxia in aggressive tumors ( Sahlgren et al., 2008 ). Recently, it has been shown that hypoxia-induced HIF1\\u03b1 forms a complex with Notch1 to enhance the stability of the protein and its activity ( Sahlgren et al., 2008 ; Gustafsson et al., 2005 ; Bertout et al., 2009 ; Main et al., 2010 ).However, since the activation of Notch signaling relies on the receptor\\u2013ligand interaction, the exact role of hypoxic microenvironment in Notch activation is not well clarified. Previous studies showed that not only Notch signaling but also Notch ligands were up-regulated in high grade cancers, indicating the prognostic value of Notch ligands in clinical settings ( Mullendore et al., 2009 ; Nam et al., 2008 ). There are five Notch ligands (Jagged1-2 and Delta like 1, 3 and 4) that have been identified in mammals ( Leong & Karsan, 2006 ; D'Souza et al. ). They are functionally similar and appear to be redundant; however, expressions of these genes are differentially regulated during embryogenesis as well as in tumor progression. Choi et al showed that despite their high sequence similarity, Jagged1 and Jagged2 are differentially regulated according to their distinct biological roles in non-small lung cancer cells ( Choi et al., 2009 ). Notably, over-expression of Jagged2 has been found in over 90% of pancreatic cancer cell lines ( Mullendore et al., 2009 ). In breast cancer, increased Jagged2 mRNA was found in the MDA-MB-435 derived Br4 variant cell line which specifically metastasizes to the brain, and knocking down of Jagged2 significantly decreased migration and invasion abilities of the Br4 cells ( Nam et al., 2008 ), suggesting that Jagged2 plays an important role in tumor metastasis in breast cancer although the underlying mechanisms of the aberrant Jagged2 expression is yet to be defined. By analyzing microarray data from multiple patient cohorts, we recently found that the expression of Jagged2 among all five ligands is most significantly correlated with the overall- and metastasis-free survival of breast cancer patients, indicating the critical role of Jagged2 in breast cancer metastasis. In this report, we provide evidence to show that Jagged2 is significantly elevated by hypoxia in both cancer and bone stromal cells, which concomitantly activate Notch signal in cancer cells. We also show that this hypoxia-induced Notch activation is essential to promote EMT and cell survival at the hypoxic invasive front and that the hypoxia-induced Jagged2 expression in bone marrow stromal cells promotes self-renewal of cancer stem-like cells by activating Notch signaling.\",\n",
            "        \"method\": \"Materials and Methods Cell culture and reagents Human breast carcinoma cell lines, MCF7 and MDA231were purchased from American Type Culture Collection. MDA231LM and MDA231BoM were kind gifts from Dr. Massague (Memorial Sloan-Kettering Cancer Center). Cells were maintained in RPMI 1640 supplemented with 10% FBS and grown at 37 \\u00b0C in a 5% CO 2 atmosphere. Cells grown in hypoxia were maintained in hypoxia chamber filled with mixture of gases (1%O 2 , 5%CO 2 , 94%N 2 ) at 37\\u00b0C. N-[N-(3,5-Difluorophenacetyl)-L-alanyl]-S-phenylglycine t-butyl ester (DAPT) was purchased from Sigma Chemical Co. Plasmids construction PGL2pro-4XCBF Notch reporter was a kind gift from Dr. Hayward (Johns Hopkins University). NICD cDNA with a Myc-tag was provided by Dr. Bresnick (University of Wisconsin). The tetracycline-inducible system T-Rex (Invitrogen) was used to create a cell line with inducible NICD expression. First, the Myc-NICD cDNA was amplified by PCR and cloned into pcDNA5/TO (Invitrogen). The human breast cancer cell line MDA231BoM was transfected with pcDNA6/TR encoding the Tet repressor, and a stable cell line (MDA231BoM/Tet) was generated. Then, the pcDNA5/TO/Myc-NICD expression plasmid was stably transfected into the MDA231BoM/Tet cell, and the resultant clones were designated as MDA231BoM Tet-Myc-NICD. To generate the Jagged2 reporter plasmid, the promoter of Jagged2 (from +15 to \\u22121,352 bp) was amplified by PCR and the product was cloned into the PGL3-basic plasmid. The resultant clones were designated as PGL3-Jagged2. pCDNA3-Jagged2-myc was a kind gift from Dr Sisodia (University of Chicago) and Jagged2 sequence was sub cloned into pSin-EF2-Pur vector between EcoRI and SpeI sites. Western blot Western blot analysis was performed as described previously using antibodies against cleaved Notch1 (1:200; Val1744; Cell Signaling Technology), Jagged2 (1:200; R&D Systems Inc.), \\u03b1 -tubulin (1:1,000 Cell Signaling Technology), HIF1\\u03b1 (1:200; BD Bioscience), phospho-Akt1 (1:500; Ser473; Cell Signaling Technology), and Akt1 (1:500; Cell Signaling Technology) ( Furuta et al., 2008 ). Quantitative real-time PCR Total RNA was isolated from the cells and reverse transcribed. The cDNA was then amplified with a pair of forward and reverse primers for the following genes: Jagged1(5\\u2019-TCGCTGTATCTGTCCACCTG - 3\\u2019 and 5\\u2019-AGTCACTGGCACGGTTGTAG - 3\\u2019), Jagged2 (5\\u2019- CTCCTCATTCGGGGTGGTAT - 3\\u2019 and 5\\u2019- GTCGTCATCCCCTTCCAGT -3\\u2019), DLL1 (5\\u2019- CAGGGTTGCACATTTCTCC -3\\u2019 and 5\\u2019- GCACGGACCTCAAGTACTCC-3\\u2019), DLL3 (5\\u2019- CCTGCGCGCTGAATGTC-3\\u2019 and 5\\u2019- CATCGAAACCTGGAGAGAGG-3\\u2019), DLL4(5\\u2019-CACACACTGGACTATAATCTGG -3\\u2019 and 5\\u2019- ACACATTCGTTCCTCTCTTCTG-3\\u2019)and \\u03b2-actin (5\\u2019-TGAGACCTTCAACACCCCAGCCATG-3\\u2019 and 5\\u2019-CGTAGATGGGCACAGTGTGGGTG-3\\u2019). PCR reactions were performed using DNA engine opticon 2 system (MJ Research) and the Maxima\\u00ae SYBR Green qPCR Master Mix (Fermentas Life Science). The thermal cycling conditions composed of an initial denaturation step at 95\\u00b0C for 5 min followed by 40 cycles of PCR using the following profile: 94 \\u00b0C, 30 s; 58 \\u00b0C, 30 s; and 72 \\u00b0C, 30 s. Reporter assay PGL3-Jagged2 plasmid was transfected to breast cancer cell line MCF7 by using Lipofectamine 2000 (Invitrogen). After 48 h, the cells were collected and the luciferase activities were then measured by using Dual-Luciferase Reporter Assay System (Promega). The Renilla expression plasmid phRG-TK (Promega) was used as an internal control. MTT cell viability assay Cells were seeded in 96-well plates and cultured under hypoxia (1%O 2 ) for the indicated time, and the viability was assessed by MTT assay (Promega). Immunohistochemistry Human breast cancer specimens were obtained from surgical pathology archives of the Akita Red Cross Hospital and Iwate Medical School. Human breast cancer tissue array was purchased from Biomax (BR1503). The sections from paraffin-embedded tissue specimens were deparaffinized and antigens were retrieved by heating the slides in 10 mM sodium citrate (pH 6.0) at 85 \\u00b0C for 30min. The slides were treated with 3% H 2 O 2 and then incubated overnight at 4 \\u00b0C with anti-NICD rabbit polyclonal antibody (1:500; Abcam 8387 lot#329981 .), anti-Jagged1 rabbit polyclonal antibody (1:200; Cell Signaling Technology), anti-Jagged2 goat polyclonal antibody (1:200; R&D Systems, Inc.), anti-DLL1 mouse monoclonal antibody (1:100; R&D Systems, Inc.), anti-DLL3 goat polyclonal antibody (1:200; Santa Cruz Biotechnology), anti-DLL4 rabbit polyclonal antibody(1:200; Santa Cruz Biotechnology), anti-glut1 antibody (1:100; Santa Cruz Biotechnology) and anti-CA9 mouse monoclonal antibody (1:100; R&D Systems, Inc.). The sections were then incubated with secondary antibodies and visualized using Envision-plus kit (Dako Corp.) or ABC staining system (Santa Cruz Biotechnology). Results of the immunohistochemistry were judged based on the intensity of staining, comparing the tumor cells and the normal glands on the same slide. Grading of the NICD expression levels was done by two independent persons without prior knowledge of the patient data. Cancer stem-like cells isolation Magnetic activated cell sorting (MACS) method was used to isolate cancer stem-like cells. Briefly, 5\\u00d710 7 cells were suspended in 1ml MACS buffer and treated with DNase for 10mins at 37 \\u00b0C. Cells were then suspended in 100\\u00b5l of MACS buffer and further incubated with biotin-conjugated anti-CD24, APC-conjugated anti-CD44 and biotin-conjugated anti-ESA antibodies followed by incubation with anti-biotin antibody and anti-APC antibodies at room temperature for 10 min. CD24 \\u2212 , CD44 + and ESA + fractions were collected by serial passages through magnetic columns according to the manufacturer\\u2019s protocol. Immunocytochemistry Cells fixed with 70% ethanol were washed with PBS and blocked by 2%BSA for 1hr. Cells were then washed again with PBS and incubated with primary anti-NICD (Val1744; Cell Signaling Technology) and anti-Jagged2 (1:200; R&D Systems, Inc) antibodies overnight at 4 \\u00b0C. Cells were further incubated with anti-rabbit IgG Alexa Fluor (R) 555molecular probe (Cell Signaling Technology) and anti-goat FITC (Invitrogen) for 1hr at room temperature. Fluorescence images were taken by a fluorescent microscope. In vitro invasion assay Cell invasion assay was performed as described previously using Matrigel invasion chamber (Beckton Dickinson, Bedford, MA) ( Bandyopadhyay et al., 2006 ). The cells that invaded through the membrane were stained with tetrazolium dye and their numbers were counted under microscope. FACS (Fluorescence-activated cell sorting) 10 4 Cells were suspended in FACS buffer (PBS with 0.1% BSA and 0.1% Triton X100) followed by incubation with FITC conjugated anti-CD24, APC conjugated anti-CD44 and PE conjugated anti-ESA for 15mins at room temperature. Cells were then washed with PBS and re-suspended in PBS for FACS analysis. For NICD expression analysis, cells were incubated with anti-NICD antibody (1:200, Val1744; Cell Signaling Technology) for 15 min at room temperature followed by incubation with anti-rabbit IgG Alexa Fluor (R) 555molecular probe for 10 mins at room temperature. Cells were then suspended in PBS and subjected to FACS analysis. ALDH (Aldehyde dehydrogenase) staining and FACS analysis 10 4 MDA231BoM stem-like cells were first labeled with Cell Tracker Dye (CellVue Burgundy cell labeling kit, eBioscience) and seeded on the monolayer of bone marrow stromal cells and they were co-cultured for 48 hrs. Cells were then collected and ALDH staining was performed according to the manufacturer\\u2019s instructions (ALDEFLUOR\\u00ae stem cell technologies), followed by analyzing the cells by FACS to measure the number of cancer stem-like cells.\",\n",
            "        \"subjects\": null,\n",
            "        \"results\": \"Results Jagged2 expression is correlated with poor overall- and metastasis-free survival of breast cancer patients To understand the role of Notch signaling in the progression of breast cancer, we first examined the expression profile of all known Notch ligands (Jagged1, Jagged2, DLL1, DLL3, DLL4) and their prognostic values in breast cancer patients, using three existing microarray cohort data bases that contain the information of overall- and metastasis-free survival for a total of 779 breast cancer patients. As shown in Fig. 1A\\u2013B , we found that only Jagged2 among all five Notch ligands is significantly correlated with overall- and metastasis-free survival. On the other hand, the expression of other ligands failed to predict overall or metastasis-free survival in patients. Importantly, this trend was observed in all three independent cohorts, suggesting that Jagged2 plays a critical role in breast tumor progression ( supplemental Fig. S1A ). We also examined the relationship between expression of Notch receptors (Notch 1,2,3,and 4) and overall- or metastasis- free survival of patients in the GSE7390 cohort, and found that none of the notch receptors had significant co-relation to patient survival ( Supplemental Fig. S1B ). Furthermore, we conducted a multivariate Cox regression analysis for Jagged2, tumor size and ER status using another independent cohort database. The results of this analysis revealed that Jagged2 is a statistically significant independent prognostic factor, which is consistent with our result of survival analysis ( Fig. 1C ). The odds ratio for Jagged2 is 2.874 (95% CI 1.446\\u20135.712, P<0.003), indicating that the risk of patients with increased Jagged2 expression within a specific time is 2.9 times higher than the risk of patients with Jagged2 negativity. Thus, over-expression of Jagged2 can be a strong predicator of overall- and metastasis-free survival. Because Jagged2 has potential value as a prognostic marker, it is interesting to know whether the expression of this gene varies in different subsets of breast cancers. Therefore we performed immunohistochemical analysis for Jagged2 expression using a tissue microarray which included 71 patient samples. However, we found that the Jagged2 expression had no correlation to the status of ER, PR or Her2 ( Supplemental Fig. S1C ). This result was also confirmed by analyzing GSE7390 cohort data for which ER status of the patients was available ( Supplemental Fig. S1D ) Notch signaling and Notch ligand are activated at hypoxic invasive front To further evaluate the role of Notch signaling in breast tumor progression, we examined the expression of cleaved Notch1 in 61 human breast tumor samples by immunohistochemistry (IHC) using NICD (Notch intracellular domain)-specific antibody. As shown in Fig. 2A and B , we observed that NICD is strongly expressed in both the nucleus and cytosol in normal gland of breast tissue; however, it was gradually decreased during tumor progression. In ductal carcinoma in situ (DCIS), an early stage of breast cancer, the expression of NICD was slightly decreased although strong nuclear expression was still observed. On the other hand, overall NICD expression was significantly reduced inside poorly differentiated tumor. Our tissue array results indicate that the expression of NICD had no significant correlation with the status of ER, PR or Her2, although it was significantly correlated with the expression of Jagged 2 and metastasis status of the patients ( Supplemental Fig. S2A\\u2013B ). Interestingly, the expression of NICD was significantly up-regulated at the invasive front of the same tumors ( Fig. 2C ). This striking difference suggests the critical role of Notch signaling in invasion and metastatic tumor progression of breast cancer. Because activation of Notch signaling requires ligand interaction, we sought a possibility that Jagged2 expression, which is significantly correlated to patient survival as shown in Fig. 1 , is also expressed at the invasive front. Indeed, we found that both NICD and Jagged2 are strongly expressed at the invasive front while they are drastically reduced inside the tumors ( Fig. 2C ), suggesting that Jagged2 is the major ligand to activate Notch signaling at the invasive front. To further verify this notion, we examined the expression of other Notch ligands (Jagged1, DLL1, DLL 3 and DLL4) using the consecutively sectioned slide for the same area of the tumor. We found that Jagged2 among all other Notch ligands was indeed most strongly expressed in the invasive front ( Supplemental Fig. S2C ). Because Notch signaling is often activated by hypoxia ( Sahlgren et al., 2008 ) and the invasive front of tumor mass is known to be hypoxic ( Horree et al., 2007 ), we tested our hypothesis that Jagged2 is up-regulated at the invasive front through hypoxic signaling by performing immunohistochemistry using antibodies for well established hypoxia markers, CA9 and Glut1, as well as NICD. As shown in Fig. 2D , we found that CA9 and Glu1 were strongly expressed at the invasive front and these markers coincided with the elevated expression of NICD. These results strongly suggest that the activated Notch signaling at the invasive front is caused by increased expression of Jagged2 which is mediated by local hypoxia. Hypoxia up-regulates Jagged2 in breast cancer cells To provide further evidence that Jagged2 is activated by hypoxia in vitro , we examined the effect of hypoxia on Jagged2 expression in two different breast cancer cell lines, MCF7 and MDA-MB 231(MDA231), under normoxia (21%O 2 ) and hypoxia (1%O 2 ). As shown in Fig. 3A , we found that mRNA and protein levels of Jagged2 were significantly elevated in both MCF7 and MDA231 cells under hypoxia. However, we found no significant induction of other Notch ligands (Jagged1, DLL1, 2, 3 and 4) in either MDA231 or MCF7 cells under hypoxic condition ( Supplemental Fig. S3A\\u2013B ). In addition, the results of Jagged2 reporter assay indicate that hypoxia can significantly up-regulate the promoter activity of Jagged2 ( Fig. 3A ). However, we found no significant HIF1\\u03b1 binding sequence on Jagged2 promoter region. Therefore, to further verify that Notch signaling is indeed activated under hypoxia upon up-regulation of Jagged2, we examined the Notch reporter activity, protein level of NICD as well as HIF1\\u03b1 expression in MCF7 cells under normoxia and hypoxia. We found that Notch signal is indeed significantly augmented, which was accompanied by up-regulation of HIF1\\u03b1 level ( Fig. 3B ). We also examined the expression of NICD in the MDA-MB231 cells by immunocytochemistry and found that NICD was strongly expressed in the clustered cells but much less so in a separated single cell under hypoxic condition ( Fig. 3C ). These results suggest that hypoxia-induced NICD activation requires Jagged2-Notch interaction through cell-cell contact rather than stabilization of preexisting Notch signaling. Hypoxia-induced cell survival and EMT require Notch signaling How the up-regulation of Jagged2 by hypoxia and concomitant activation of Notch signaling affect tumor progression is the next important question. Rapidly growing tumor cells at the invasive front must survive under hypoxic condition and need to be invasive; therefore, we speculated that hypoxia-induced Jagged2 plays a role in this process. To test this possibility, Jagged2 expression in four different breast cancer cell lines were examined by RT-PCR and Western blot ( Fig. 4A ), and we found that Jagged2 expression was significantly up-regulated in highly metastatic cell line, MDA231LM, which is derived from MDA231 cells while it is significantly lower in non-tumorigenic MCF10A cells. These results suggest an increasing chance of Notch signaling being activated in MDA231LM cells due to the large amount of Jagged2 expression, and this property of MDA231LM cells may be the result of metastatic phenotype selection driven by hypoxia. Next, we examined the role of Notch signaling in promoting cell survival under hypoxia. MDA231, MDA231LM and MDA231 LM-DNMAML which constitutively expresses the dominant negative Mastermind to suppress Notch signaling, were cultured under hypoxia and cell viability was measured at different time points by MTT assay. We found that the cell viability of MDA231LM was not affected by hypoxia; however, MDA231 and MDA231LM-DNMAML cells showed decreased cell viability after 48hr exposure to hypoxia, indicating that hypoxia induced cell survival requires Notch signaling ( Fig. 4B ). To further investigate the role of Jagged2 in promoting cancer cell survival under hypoxia, we knocked down the Jagged2 expression in MDA231LM and also ectopically expressed Jagged2 in MDA231 by infecting these cells with sh-Jagged2 and pSin-EF2-Jagged2 lenti viruses, respectively. As shown in Fig4 C , the knock-down of Jagged2 significantly attenuated cell survival in MDA231LM and MDA231cells under hypoxia, while the ectopic expression of this gene significantly augmented cell survival. Because the activation of Akt signaling is known to be a key mediator of cell survival under hypoxia, and Akt was reported to be a potential target of Notch signaling in melanoma and lung cancer ( Bedogni et al., 2008 ; Zhao et al., 2010 ; Eliasz et al., 2010 ), we evaluated the effect of Notch signaling on Akt by treating MDA231LM cells with DAPT, a \\u03b3-secretase inhibitor which prevents the release of activated Notch intracellular domain. We found that the amount of phospho-Akt-1 was strongly decreased by the DAPT treatment under hypoxia, while the total amount of Akt was not affected by the suppression of Notch signaling ( Fig. 4D ). These results suggest that Notch signaling promotes cell survival under hypoxia through the activation of Akt pathway. Furthermore, to examine whether Notch signaling is involved in hypoxia-induced invasion and EMT in breast cancer cells, we first measured the invasive ability of MDA231LM cells using Matrigel invasion chamber under hypoxia in the presence or absence of DAPT for 24hrs. We found that the DAPT treatment significantly decreased the number of invading cells, suggesting that inhibition of Notch signaling abrogates the invasive ability of cancer cells under hypoxia ( Fig. 4E ). To further understand the molecular basis of Notch mediated cell invasion, we examined mRNA level of the snail gene, which is known to be involved in EMT by decreasing E-cadherin expression, in MCF7, MDA231 and MDA231LM cells. As shown in Fig 4F , the snail gene was highly expressed in MDA231LM which is the most invasive among the tested cell lines with highest level of expression of Jagged2. Hypoxia also induced the morphological change of MDA231LM cells to become a more mesenchymal shape which was reversed by DAPT treatment ( Fig. 4G ). Next, we examined the level of E-cad mRNA in MDA231LM and MCF7 cells under normoxia and hypoxia in the presence or absence of DAPT. We found that E-cad mRNA was significantly down-regulated in both MCF7 and MDA231LM cells under hypoxia, and this hypoxia effect was significantly abrogated by DAPT ( Fig. 4H ). To further examine the role of Jagged2 in promoting snail expression through activating Notch signaling, we cultured MDA231LM cells under normoxia and hypoxia with or without the infection of sh-Jagged2 lentivirus followed by measuring the mRNA levels of snail as well as Hes1 which is a typical downstream target of Notch signaling. As shown in Fig 2I , we found that the knock-down of Jagged2 significantly decreases the expression of both Hes1 and snail mRNA. Taken together, our results strongly suggest that the activation of Notch signaling mediated by hypoxia-induced Jagged2 expression leads to cell survival via Akt activation and cell invasion via promotion of EMT. Hypoxia up-regulates Jagged2 in bone marrow stromal cells Bone is the major metastatic site of breast cancer. Several lines of evidence indicate that bone marrow is generally hypoxic, suggesting that hypoxic pre-metastatic niche may contribute to the progression of bone metastasis in breast cancer. Therefore, to examine the role of Notch signaling in bone metastasis with hypoxic microenvironment, we evaluated Jagged2 expression in immortalized human bone marrow stromal cells HS5 under hypoxic condition. We found that mRNA and protein levels of Jagged2 were significantly elevated by either 1% O 2 culture or hypoxia-mimetic compound DFO treatment in a time dependent manner ( Fig. 5A ). In addition, we also performed immunocytochemical staining of HS5 and HS27cells for Jagged2 expression under hypoxia. We found that the Jagged2 protein was strongly expressed in both HS5 and HS27 cells after being exposed to hypoxia for 48hrs ( Supplemental Fig. S4 ). Next, we used MDA231BoM cell line, which has a strong tendency to metastasize to bones to examine whether interaction between bone marrow stromal cells and cancer cells indeed activates Notch signaling under hypoxia. A plasmid containing the Notch reporter gene was stably cloned into MDA231BoM cells, and the Notch reporter activity was measured after 48hr of co-culture with HS5 under hypoxia or DFO treatment ( Fig. 5B ). We found that the Notch reporter activity was significantly up-regulated in both hypoxia and DFO treated groups, implying that hypoxic bone microenvironment plays a pivotal role in activating Notch signaling in cancer cells and that such activation is contributed by the increased amount of Notch ligands, particularly Jagged2. Notch signaling promotes cancer stem-like cell self renewal under hypoxia Because Notch signaling is known to be involved in stem cell renewal and hypoxic condition generates a favorable niche to these cells, we sought a possibility that the hypoxia \\u2013 induced Notch ligand expression in bone stromal cell plays a role in self-renewal of cancer stem-like cells. To address this question, we first isolated cancer stem-like cell population (CD24 \\u2212 CD44 + ESA + ) from MDA231BoM cells by MACS, and the tumor-initiating ability of this cell population was examined in nude mice ( Supplemental Fig. S5A ). We then cultured the bone stromal cells (HS5) in monolayer and seeded the stem-like cells of MDA231BoM which was labeled with GFP on top of the HS5 cells, and they were further cultured under hypoxia or normoxia for 48 hrs followed by Immunocytochemical analysis using anti-NICD antibody. As shown in Fig. 6A , Notch signaling was highly activated in the cancer stem-like cells that were co-cultured with HS5 under hypoxia compared to the cells that were co-cultured under normoxia or those that were cultured alone under hypoxia. In addition, we co-cultured the cancer stem-like cells with HS5 under normoxia or hypoxia in the presence or absence of DAPT followed by FACS analysis to measure the percentage of NICD + cells in overall stem cell population ( Fig. 6B ). Our result indicates that Notch signaling is significantly activated in cancer stem-like cells that were co-cultured with the HS5 cells under hypoxia. In a parallel experiment, we measured the number of CD24 \\u2212 CD44 + ESA + cells by FACS analysis and found that the number of cancer stem-like cell population was indeed significantly increased when co-cultured with HS5 under hypoxia and that this increase was blocked by DAPT ( Fig. 6C ). To further verify our results, we co-cultured HS5 and MDA231BoM stem-like cells that were labeled with the Cell Tracker dye under hypoxia for 48 hrs. Cells were then subjected to ALDH activity assay which is a recently developed method for identifying cancer stem cells. As shown in Supplemental Fig S5B\\u2013C , we found that hypoxia or ectopic expression of Jagged2 in HS5 cells significantly increased the number of ALDH-positive cells while DAPT treatment significantly abrogated such effect. We also generated a 231BoM/Tet NICD cell line which inducibly expresses the active form of Notch by tetracycline, and CD24 \\u2212 CD44 + ESA + cell population was isolated from this cell line. The sorted cells were then cultured in mammosphere medium in the presence or absence of tetracycline, and the number of stem-like cells was measured by FACS analysis ( Fig. 6D ). We found that the number of cancer stem-like cells was significantly increased upon tetracycline induction. Taken together, our results indicate that hypoxic condition induces Notch ligand, particularly Jagged2, in bone stromal cells and that the interaction of stromal cells with cancer stem-like cells activates their Notch signaling which leads to self renewal of these cells and promotion of further tumor progression.\",\n",
            "        \"discussion\": \"Discussion Notch signaling is a highly conserved pathway which plays a vital role in normal embryonic development and morphogenesis as well as in stem cell maintenance ( Yu et al., 2008 ). However, aberrant expression of Notch signaling is also frequently observed in many solid tumors and leukemia, suggesting a critical role of this pathway in tumor pathology and cancer progression ( Ellisen et al., 1991 ; Koch & Radtke, 2007 ). The results of our immunohistochemical analysis also indicate that Notch signaling is strongly activated in normal glands of breast tissue and generally down-regulated in DCIS and advanced tumor cells; however, it is significantly up-regulated at the invasive front. These results illustrate an intricate role of Notch signaling in tumor progression. A hallmark of Notch signaling is the requirement of the ligand-receptor interaction through direct cell-cell contact, which may occur between tumor cells or tumor cell-stroma interactions ( Felli et al., 1999 ). Therefore, dysregulation of Notch signaling in cancer cells is likely caused by either alteration of the receptor expression in cancer cell and/or abnormal expression of ligand in tumor-surrounding cells. In mammalian cells, there are four Notch receptors, Notch 1 to 4, and they are cleaved by \\u03b3-secretase upon ligand binding to become the activated form, NICD ( Artavanis-Tsakonas et al., 1999 ). On the other hand, less is known about the expression status of Notch ligand in tumor and tumor-associated stroma. The results of our existing microarray data analysis indicate that Jagged2 is the ligand, and in some cases the only ligand, whose expression is significantly correlated with metastasis-free survival in many patient cohorts. It should be noted that Reedijk et al ( Reedijk et al., 2005 ) and Dickson et al ( Dickson et al., 2007 ) previously reported that mRNA expression of Jagged1 had significant correlation to patient survival. The reason for this apparent contradiction is not clear; however, there are several differences between these analyses and our approach. First, their analysis is based on in situ hybridization, while our analysis is based on existing data set of microarray analysis which used gene chip arrays. Although in situ hybridization is more accurate for localization, it is less quantitative. Secondly, the design and location of probes for Jagged1 gene in these analyses are different. The gene chip generally uses multiple probes. Thirdly, the numbers of patients are different. We analyzed all together more than 600 patients in three different cohorts while they examined about 100 patients in each of their experiments. These differences may have contributed to the different outcome of the results of mRNA expression of Jagged1 in breast cancer patients. We also demonstrated that high expression of Jagged2 and concomitant activation of Notch signaling often coincides with the hypoxic regions of invasive front in breast cancer and that hypoxic condition indeed significantly up-regulates the Jagged2 expression in both tumor and stromal cells in vitro . Although there is a general concern whether the cell lines that have been adapted for years to grow in 21% oxygen reflects in vivo hypoxic condition, which may be indeed a limitation of the in vitro experiment, our results using multiple cell lines clearly indicate that Jagged2 specifically responds to hypoxia which is also consistent with the result of immunohistochemical analysis for clinical sample using hypoxic markers. Hypoxia is a hallmark of tumor, which contributes to tumor cell survival, angiogenesis and chemo-resistance ( Maynard & Ohh, 2007 ). The central area of tumor mass is often hypoxic due to necrotic cell death; however, it is well established that invasive front of tumor is also strongly hypoxic because of the rapid rate of proliferation of tumor cells ( Horree et al., 2007 ). It has been reported that hypoxia increases the transcriptional activity of NICD by inhibiting its degradation in neuronal stem cells and myogenic cells ( Gustafsson et al., 2005 ). Similar results were also observed in NSCL ( Chen et al., 2007 ; Eliasz et al., 2010 ). It was also found that HIF1 alpha can directly bind to NICD and stabilize this protein and activity ( Cejudo-Martin & Johnson, 2005 ). This protein stabilization is certainly considered as one of the mechanisms which contribute to the Notch activation at the invasive front; however, our results indicate that the major factor in the Notch activation is the hypoxia-induced expression of Jagged2 through cell-cell interaction of tumor cells. In fact, we have shown that hypoxia can significantly activate Jagged2 expression as well as the Notch signaling in breast tumor cell lines, but this activation is strongly cell-density dependent and the clustered cells showed much higher sensitivity to Notch activation by hypoxia than a single cell in the culture ( Fig. 3C ). How hypoxia promotes invasiveness of tumor cells through activation of Notch signaling is an intriguing question. Our results indicate that inhibition of hypoxia-induced Notch signaling significantly decreased cell survival and invasiveness by blocking Akt pathway and also by suppressing EMT. It should be noted that Akt was previously found to be activated by Notch signaling in melanoma and lung cancers ( Zhao et al., 2010 ; Bedogni et al., 2008 ). In addition, Notch signaling has been known to be associated with chemo-resistance and cell survival ( Wang et al., 2010 ; Eliasz et al., 2010 ). Therefore, hypoxia-induced Notch activation may render tumor cells to become more resistant to cell death through activation of Akt. Taken together, the aggressive nature of tumor cells at the invasive front appears to be mediated by Akt/EMT induction through up-regulation of Jagged2 followed by the activation of Notch signaling. Bone is one of the most common sites of breast cancer metastasis ( Kominsky & Davidson, 2006 ). It is becoming clear that there are many regions that are hypoxic in the bone, and these areas play an important role in the self-renewal ability of hematopoietic stem cells by providing stem cell niche ( Yin & Li, 2006 ). Because metastatic cells must have stem-like characteristics, it is highly plausible that cancer stem-like cells take advantage of these niches once they reach the bone. In fact, our results indicate that bone marrow stromal cells significantly over-expressed Jagged2 under hypoxic condition and that co-culturing the stromal cells with the cancer stem-like cells significantly promoted proliferation of cancer stem-like cells by activating Notch signaling. Therefore, Jagged2 expression in bone marrow stromal cells at the hypoxic regions in the bone is considered to provide appropriate niche for metastatic cancer stem-like cells and promote their self-renewal. Collectively, our results indicate that the expression of Jagged2 is significantly augmented by hypoxia at the invasive front and the interaction between breast cancer cells and the Jagged2-expressing cells significantly enhanced the invasiveness and survival of cancer cells through the activation of Akt/EMT pathways. Furthermore, hypoxic regions in the bone marrow provide the niche for cancer stem-like cells by expressing the hypoxia-induced Jagged2. Therefore, Jagged2 is considered to be a valuable marker to predict the outcome in patients with metastatic disease, and more importantly, intervention in Jagged2 expression and cell-cell interaction may serve as a novel therapeutic target for metastatic breast cancer.\",\n",
            "        \"conclusion\": null\n",
            "    },\n",
            "    {\n",
            "        \"pmcid\": \"PMC3174812\",\n",
            "        \"abstract\": \"The principal visual cue to self-motion (egomotion) is optic flow, which is specified in terms of local 2D velocities in the retinal image without reference to depth cues. However, in general, points near the center of expansion of natural flow fields are distant, whereas those in the periphery are closer, creating gradients of horizontal binocular disparity. To assess whether the brain combines disparity gradients with optic flow when encoding egomotion, stereoscopic gradients were applied to expanding dot patterns presented to observers during functional MRI scanning. The gradients were radially symmetrical, disparity changing as a function of eccentricity. The depth cues were either consistent with egomotion (peripheral dots perceived as near and central dots perceived as far) or inconsistent (the reverse gradient, central dots near, peripheral dots far). The BOLD activity generated by these stimuli was compared in a range of predefined visual regions in 13 participants with good stereoacuity. Visual area V6, in the parieto-occipital sulcus, showed a unique pattern of results, responding well to all optic flow patterns but much more strongly when they were paired with consistent rather than inconsistent or zero-disparity gradients. Of the other areas examined, a region of the precuneus and parietoinsular vestibular cortex also differentiate between consistent and inconsistent gradients, but with weak or suppressive responses. V3A, V7, MT, and ventral intraparietal area responded more strongly in the presence of a depth gradient but were indifferent to its depth-flow congruence. The results suggest that depth and flow cues are integrated in V6 to improve estimation of egomotion.\",\n",
            "        \"intro\": null,\n",
            "        \"method\": \"MATERIALS AND METHODS Participants. Fifteen healthy human volunteers took part in the study. They were screened for normal stereoacuity before being invited to participate. Two were later excluded because of poor binocular fusion during scanning. Therefore, data from 13 participants (10 female; mean age 27 yr) were included in the analysis. All participants were scanned for the main experiment, 12/13 for retinotopic mapping and egomotion-compatible optic flow areas localizers, and 10/13 for MT/MST localizers (the missing localizer scans were due to unavailability of participants for rescanning). All participants were screened according to standard procedures, and written informed consent was obtained. The project was approved by the Royal Holloway ethics committee. Data acquisition. MRI images were obtained with a 3-Tesla Siemens Magnetom Trio scanner (Siemens, Erlangen, Germany) and a custom-designed eight-channel posterior array (PA) head coil designed for imaging posterior regions of the brain (Stark Contrast, Erlangen, Germany). Scanning was conducted over at least two sessions on different days. In one session, the data for the main experiment were acquired; in the others, functional data for the localizers and a 3D, T1-weighted anatomical scan (modified driven equilibrium Fourier transform, MDEFT; 160 \\u00d7 256 \\u00d7 256 resolution; 1-mm isotropic voxels; Deichmann et al. 2004 ) were acquired. This anatomical image, which alone was acquired with a standard Siemens head array coil, was used as a reference to which all the functional images obtained with the PA coil were coregistered. Functional data acquisition employed a standard gradient echoplanar sequence (main experiment and egomotion-sensitive area localizer: repetition time T R = 2.5 s, 35 contiguous axial slices, interleaved acquisition order, 3-mm isotropic voxels, T E = 31 ms; retinotopic mapping and MT/MST localizer: same parameters, except T R = 2 s, 28 slices). Each scan run lasted 6 min 40 s for the main experiment, 8 min 56 s for retinotopic mapping, 4 min 47 s for egomotion-compatible optic flow areas, and 5 min 5 s for MT/MST localizers. For coregistration purposes (see Data analysis ), at the beginning of each scanning session, we acquired two single-volume echo-planar imaging (EPI) sequences that had the same position parameters as the experimental runs: one (BC-EPI) using the scanner's integral whole body coil (BC) to give uniform contrast, and another (PA-EPI), immediately after, acquired with the PA coil. Within each scanning session, a T1-weighted anatomical scan (MP-RAGE, Siemens; 1 \\u00d7 1 \\u00d7 1 mm; field of view: 256 \\u00d7 256) was also acquired to assist with coregistration of the functional data to the MDEFT scan. Stimuli and procedure. The stimuli for the main experiment were viewed via binocular LCD goggles (NordicNeuroLab, Bergen, Norway) that were adjusted to match each participant's interocular distance. The images subtended 30 \\u00d7 23 degree visual angle and had a resolution of 800 \\u00d7 600 pixels. The stimuli consisted of 600 moving white dots on a dark background. Each dot subtended 16-min arc diameter, and each eye's display was updated at 30 Hz. The dots, which were initially randomly located, formed a circular patch of 23-degree diameter. Dot directions were arranged to produce radial motion away from the center of the patch (expansion, simulating forward motion of the observer), with dot speed increasing as a linear function of eccentricity (min = 1.15 degree/s; max = 9.23 degree/s). A central fixation cross was provided, and participants were instructed to maintain fixation at all times. The dots had a radial gradient of horizontal disparity that was centered at the fixation plane and spanned \\u00b152-min arc disparity. There were three disparity-gradient conditions: 1 ) consistent disparity, 2 ) inconsistent disparity, and 3 ) zero disparity. In the consistent condition, horizontal disparity changed linearly with eccentricity in all directions from the center. Dots near the center of the display had the largest uncrossed disparity and were perceived as distant, whereas dots at the periphery had the largest crossed disparity and were perceived as closer to the observer, consistent with the forward self-motion indicated by the optic flow. The fixation cross appeared at a depth midway between the nearest and furthest dots. In the inconsistent condition, the disparity gradient was reversed, the perceptual result being that dots increased in distance from the observer as they moved away from the center. This is not consistent with natural self-motion. The zero disparity condition had no disparity gradient; all the dots were presented at the fixation plane. Thus the middle of the disparity range was zero for all three stimuli. Figure 1 shows sample stereopairs illustrating the consistent and inconsistent disparity gradients. Movies showing the complete, animated stimuli are provided as online supplemental material, available at the J Neurophysiology website. Fig. 1. A: diagram showing the depth structure of the 3 stimuli used in the main experiment. The arrows indicate local dot motion direction, arrow length indicates the presence of a dot speed gradient, and arrow color indicates the magnitude of horizontal binocular disparity (see key; units are min of arc). B\\u2013D : stereopairs illustrating the stereoscopic gradients that were applied to the stimuli. Each stereopair represents 1 frame of an animation in which dots moved outward from the center, simulating forward motion. When the right image is viewed by the right eye and the left by the left, the stereopair in B creates a zero gradient. That in C creates a consistent radial gradient, and D creates an inconsistent gradient. If free fused by converging the eyes, C and D will reverse. An event-related design was used for the main experiment. Participants took part in six to nine experimental runs, each lasting 6 min 40 s. Within each run, there were 48 trials (16 per condition). During each trial, a stimulus was presented for 3 s, followed by an interstimulus interval (ISI) that varied between 2 s and 8 s in steps of 1 s, approximately following a Poisson probability distribution (median 4 s). During the ISI, the screen was a uniform gray apart from the central fixation cross, which was present continuously. A central fixation task was used to ensure attention to the depth structure of the motion stimuli and to monitor successful binocular fusion and stereopsis throughout the experiment. In this task, the color of the fixation cross was randomly reselected at a rate of 2 Hz from six easily distinguished hues, and participants were instructed to maintain fixation at all times and to monitor the color of the cross. Whenever a specific color (dark blue) appeared, the task was to indicate, by pressing one of four buttons, the nature of the displayed stimulus: \\u201chollow\\u201d (consistent), \\u201cpointy\\u201d (inconsistent), \\u201cflat\\u201d (zero disparity), or \\u201cno stimulus\\u201d (ISI). A blue cross could occur at any time except 500 ms after the beginning and before the end of a stimulus or ISI. Only runs in which participants performed the task with at least 70% accuracy for each condition were included in the analysis. The task was possible only when fixation was close to the fixation cross. Occasional saccades resulting in small fixation errors may have occurred but are unlikely to affect the results because they do not alter the stereoscopic gradients. On completion of the main scan, participants were asked whether they experienced vection (the illusory sensation that they were actually moving). All reported that they did not, at any time. This is expected when using brief presentations because vection takes time to develop. The strategy adopted was to quantify activity occurring in the main experiment within various regions of interest, some of which have previously been associated with optic flow or egomotion. These regions were identified with separate localizer scans that enabled us to demarcate their boundaries and also to avoid \\u201cdouble-dipping\\u201d bias when quantifying activity within them. Stimuli for the localizer scans were back projected onto a screen mounted in the rear of the scanner bore by a computer-controlled LCD projector. During the retinotopic mapping and MT/MST localizer scans, all stimuli were seen binocularly in free view via a mirror above the head coil. For retinotopic mapping, standard procedures were used ( Sereno et al. 1995 ; rotating 24-degree flickering red and green \\u201cwedge\\u201d; 8 cycles; 64 s/cycle). MT and MST were mapped using a standard procedure ( Huk et al. 2002 ) in which moving and static dots are contrasted in each hemifield. Areas V6, VIP, p2v, PIVC, and PcM were localized using egomotion-compatible optic flow as described in Cardin and Smith (2010) . Stimuli for this localizer scan were viewed via a custom monocular optical device that magnified the image, to obtain wide-field visual stimulation (60 degrees). The unstimulated eye was occluded. Mean Talairach coordinates for these regions of interest (ROIs) are reported in Table 1 , along with mean cluster sizes. Table 1. Mean Talairach coordinates of ROIs identified with egomotion-compatible optic flow Left Right x y z x y z Number of voxels V6 \\u221217 \\u221278 26 16 \\u221277 26 59 \\u00b1 8.0 PcM \\u221212 \\u221249 46 11 \\u221243 46 22 \\u00b1 2.5 VIP \\u221227 \\u221253 51 26 \\u221258 49 33 \\u00b1 4.0 2v \\u221230 \\u221246 47 27 \\u221245 48 31 \\u00b1 3.7 CSv \\u221211 \\u221219 40 9 \\u221226 42 28 \\u00b1 3.1 PIVC \\u221240 \\u221233 19 36 \\u221233 18 32 \\u00b1 5.2 The numbers of voxels are mean numbers (\\u00b1 SE) of functional voxels included in each region of interest (ROI). PcM, precuneus motion area; VIP, ventral intraparietal area; CSv, cingulate sulcus visual area; PIVC, parietoinsular vestibular cortex. Data analysis: main experiment. All data were preprocessed and analyzed with BrainVoyager QX (version 2.2; Brain Innovation, Maastricht, The Netherlands). EPIs were corrected for head motion and slice timing and were filtered with a temporal high-pass filter of 0.01 Hz. No smoothing was applied. All functional images were aligned to the PA-EPI acquired at the beginning of the scan session. Because of the steep posterior-to-anterior intensity gradient of the EPIs acquired with the PA head coil, coregistration of these images to the anatomy is poor. Therefore, we coregistered the BC-EPI to the intrasession inhomogeneity-corrected MP-RAGE and assumed no head movements between the acquisition of this BC-EPI and the PA-EPI. All images where then transformed to the space of the reference MDEFT image, using parameters obtained from the coregistration of the two anatomical scans. Coregistration accuracy was checked visually. Analysis was conducted by fitting a general linear model with regressors representing the three stimulus categories and four button presses. For every condition, each event was modeled as a boxcar with 3 s (stimuli) and 0.5 s (button presses) duration, convolved with a canonical hemodynamic response function, and entered into a multiple-regression analysis to generate parameter estimates for each regressor at every voxel. Six movement parameters were derived from the realignment of the images and were also included in the model. The first three volumes of each run were discarded to allow for T1 equilibration effects. Effect sizes (\\u03b2 values) for the three experimental conditions were extracted for each independently defined ROI by averaging across all voxels in the ROI. The results were normalized, to account for overall amplitude differences across participants, by expressing each activation in a given ROI as a proportion of the largest activation. This was done independently for each ROI. Finally, the results were averaged across participants. Data analysis: ROI definitions. Areas V6, VIP, p2v, PIVC, and PcM were defined as all contiguous voxels that were more active with a single pattern of expansion-contraction and rotation (egomotion compatible), than with nine simultaneous smaller patterns (egomotion incompatible) in the parieto-occipital sulcus (V6), the intraparietal sulcus (VIP), the junction of the intraparietal sulcus and the postcentral sulcus (p2v), the region of the precuneus dorsal to the ascending arm of the cingulated sulcus (PcM), and the posterior region of the insula, putatively the PIVC. Area V6 was successfully defined in 22/24 hemispheres, VIP in 14/24, p2v in 16/24, CSv in 20/24, PcM in 17/24, and PIVC in 19/24. The method is described in detail in our earlier work ( Cardin and Smith 2010 ). MST was defined as all contiguous voxels within MT+ that were significantly active during ipsilateral motion stimulation. MT was defined as all contiguous active voxels that were active during contralateral but not ipsilateral stimulation, with the proviso that any MT voxels situated further anterior than the median value of the MST ROI on the horizontal (axial) plane were removed from the MT ROI. MT and MST were successfully defined in all the scanned hemispheres ( n = 20). Retinotopic data were analyzed conventionally. ROIs (visual areas V1-V7) were drawn by eye on the basis of boundaries formed by phase reversals, viewed on a flattened version of each participant's reference anatomy. V3B was defined according to the original criteria of Smith et al. (1998) and corresponds to LO1 of Larsson and Heeger (2006) . Retinotopic areas were successfully defined in all the scanned hemispheres ( n = 24).\",\n",
            "        \"subjects\": null,\n",
            "        \"results\": null,\n",
            "        \"discussion\": null,\n",
            "        \"conclusion\": null\n",
            "    },\n",
            "    {\n",
            "        \"pmcid\": \"PMC3212907\",\n",
            "        \"abstract\": null,\n",
            "        \"intro\": null,\n",
            "        \"method\": \"Materials and methods Experimental Design and Subjects Seventy-nine post-menopausal women (aged 58-78) volunteered and gave written consent for this study which was approved by the Human Studies Committee of Urmia University, IRAN. Subjects had bone density (lumbar spine (L 2 -L 4 ) and femoral neck BMD), inflammatory markers (TNF-\\u03b1, IL-6, PGE 2 ), hormone concentrations (estrogen, osteocalcin, 1, 25 Vit D, CTX, PTH, CT), blood ions (Ca 2+ , phosphorus, and fatty acid composition of neutrophil extracts) measured at baseline, the end of week 12 (after 24 hours of recovery), and the end of week 24 (after 24 hours of recovery). The women were sedentary, in good health, at least 8 years past-menopause, and taking no medications. Preliminary screening included a medical history, physical examination, and a Bruce treadmill test [ 34 ]. Subjects were randomly assigned to one of 4 groups: exercise + supplement (E+S, n = 21), exercise only (E, n = 20), supplement only (S, n = 20), and control (Con, n = 18) groups. Leisure, household, and occupational activity was estimated with the Physical Activity Scale for the Elderly Questionnaire [ 35 ] (Table 1 ). Table 1 Individual characteristics of post-menopausal women Variables Groups E + S (n = 21) E (n = 20) S (n = 20) Con (n = 18) P < Age ( yr ) 59.7 \\u00b1 2.3 61.4 \\u00b1 6.9 63.1 \\u00b1 7.5 58.9 \\u00b1 8.1 0.604 Height ( m ) 1.64 \\u00b1 0.14 1.67 \\u00b1 0.08 1.69 \\u00b1 0.09 1.68 \\u00b1 0.16 .6410 Weight ( kg ) 75.2 \\u00b1 12.2 77.5 \\u00b1 10.4 78.3 \\u00b1 16.1 75.9 \\u00b1 17.2 .8150 BMI ( kg\\u00b7m -2 ) 26.3 \\u00b1 4.8 25.1 \\u00b1 7.1 27.9 \\u00b1 5.4 28.5 \\u00b1 3.7 .7700 Fat (%) 29.8 \\u00b1 5.8 27.9 \\u00b1 7.5 29.1 \\u00b1 7.2 28.1 \\u00b1 6.8 .8050 VO 2max 33.7 \\u00b1 4.9 32.2 \\u00b1 3.1 31.2 \\u00b1 4.5 32.7 \\u00b1 4.4 .0660 PAS 135 \\u00b1 60 134 \\u00b1 51 140 \\u00b1 69 138 \\u00b1 61 .7010 E+S = Exercise + Supplement; E = Exercise; S = Supplement; Con = Control; BMI = Body mass index; VO 2max = Maximal oxygen uptake (ml -1 \\u00b7kg -1 \\u00b7min -1 ); PAS = Physical activity score. Values are mean \\u00b1 SD Oral omega-3 The E+S and S groups were supplemented with N-3 PUFA capsules (Viva Omega-3 fish oil, Manufactured by: Viva Pharmaceutical Inc, Richmond B.C. V6V 1K8, Canada), containing 180 mg EPA and 120 mg Docosahexaenoic acid (DHA), to supply a total of 1000 mg/day [ 12 ] of N-3 PUFAs over 24 weeks. The degree of compliance with N-3 PUFA supplements, as determined by pill counts, was 96 \\u00b1 9%. Also, the incorporation of EPA and DHA into the cell membranes of neutrophils was measured. Information was collected from each individual by trained interviewers in face to face interviews based on a structured and previously validated questionnaire that included the following: socio-demographic data; years since menopause; physical activities, including hours spent sitting, standing, walking, sports, and leisure activities; medications; smoking and drinking alcohol; and other factors that may have possible confounding effects on N-3 PUFA consumption and metabolism of bone and lipid [ 36 ]. All subjects were requested to consume their usual diet throughout the period of study. A detailed diary of all types and household measures of food and drinks consumed, including brand names, was kept for baseline, week 12 and week 24 [ 37 ]. Dietary intakes did not alter more than would be expected over the 24 weeks of the study (Table 2 ). Daily intakes of calcium, vitamin D, carbohydrates, lipids, saturated fatty acid, mono-unsaturated fatty acid, PUFA, proteins and total energy were calculated from the daily record by the dietitian on the basis of the fifth revision of the standard tables of food composition in Japan [ 36 ]. Information on use of medications and drugs was also obtained through standard and self-reported questionnaires in accordance to the researchers' recommendations [ 12 ]. Table 2 Differences in dietary intakes for the women Daily intakes Groups E + S E S Con P < Total energy ( kcal ) Before 2116 \\u00b1 371 2101 \\u00b1 358 2121 \\u00b1 360 2099 \\u00b1 381 0.149 After \\u2020 2207 \\u00b1 354 \\u2020 2211 \\u00b1 344 2119 \\u00b1 369 2109 \\u00b1 364 0.074 Proteins ( g ) Before 81 \\u00b1 14 77 \\u00b1 21 82 \\u00b1 17 80 \\u00b1 20 0.198 After 78 \\u00b1 19 81 \\u00b1 10 79 \\u00b1 18 82 \\u00b1 12 0.357 Carbohydrates ( g ) Before 224 \\u00b1 59 218 \\u00b1 51 230 \\u00b1 58 228 \\u00b1 65 0.091 After 230 \\u00b1 66 225 \\u00b1 55 233 \\u00b1 49 227 \\u00b1 60 0.208 Lipids ( g ) Before 74 \\u00b1 12 69 \\u00b1 24 71 \\u00b1 29 73 \\u00b1 22 0.077 After 72 \\u00b1 22 74 \\u00b1 18 68 \\u00b1 23 71 \\u00b1 12 0.201 SFA ( g ) Before 24 \\u00b1 9 23 \\u00b1 6 22 \\u00b1 5 22 \\u00b1 8 0.087 After 23 \\u00b1 5 23 \\u00b1 8 23 \\u00b1 7 22 \\u00b1 5 0.141 MUFA ( g ) Before 26 \\u00b1 7 28 \\u00b1 5 27 \\u00b1 6 29 \\u00b1 9 0.536 After 27 \\u00b1 5 28 \\u00b1 9 27 \\u00b1 8 28 \\u00b1 6 0.339 PUFA ( g ) Before 12 \\u00b1 4 11 \\u00b1 7 11 \\u00b1 6 11 \\u00b1 9 0.067 After 11 \\u00b1 9 11 \\u00b1 8 11 \\u00b1 8 11 \\u00b1 6 0.094 Total fibers ( g ) Before 18 \\u00b1 9 17 \\u00b1 8 17 \\u00b1 5 18 \\u00b1 6 0.091 After 17 \\u00b1 8 18 \\u00b1 6 17 \\u00b1 9 18 \\u00b1 8 0.082 Calcium ( mg ) Before 690.4 \\u00b1 234.7 692.3 \\u00b1 244.9 694.5 \\u00b1 227.3 687.6 \\u00b1 189.9 0.092 After 694.1 \\u00b1 212.1 692.8 \\u00b1 259.1 693.4 \\u00b1 235.2 690.2 \\u00b1 204.8 0.421 Vitamin D ( \\u03bcg ) Before 9.7 \\u00b1 6.5 9.6 \\u00b1 6.3 9.8 \\u00b1 6.9 9.4 \\u00b1 5.7 0.114 After 9.6 \\u00b1 6.8 9.7 \\u00b1 6.8 9.4 \\u00b1 6.5 9.7 \\u00b1 7.1 0.067 Vitamin K ( \\u03bcg ) Before 435.1 \\u00b1 211.7 431.9 \\u00b1 191.2 436.2 \\u00b1 204.7 434.5 \\u00b1 197.1 0.336 After 433.8 \\u00b1 227.2 435.1 \\u00b1 201.7 433.9 \\u00b1 211.1 436.2 \\u00b1 217.4 0.081 E+S = Exercise + Supplement; E = Exercise; S = Supplement; Con = Control; BMI = Body mass index; SFA = Saturated fatty acid; MUFA = Monounsaturated fatty acid; PUFA = Polyunsaturated fatty acid \\u2020 P < 0.05, significantly different from baseline values (within groups, baseline vs . week 24). Values are mean \\u00b1 SD Exercise program In the first 12 weeks of the study, subjects in E+S and E groups walked or jogged on a treadmill 25-30 min\\u00b7day -1 , 3-4 days\\u00b7week -1 , at 45-55% of their individually determined HR max . As their exercise tolerance improved, the intensity and duration of exercise training in the second 12 weeks of the study were increased to 40-45 min\\u00b7day -1 , 4-6 days\\u00b7week -1 ; at an intensity of 55-65% of HR max . Adherence to the exercise prescription was documented through the use of Polar heart rate monitors, and subjects received feedback if training intensities were either too high or low in comparison with desirable intensities. Attendance was taken at each exercise session to monitor compliance with the program. Subjects were contacted if an exercise session was missed. In those women who completed the interventions, there was > 95% compliance for attendance at the exercise sessions. Nine subjects (E + S group, n = 1; E group, n = 2; S group, n = 2; Con group, n = 4) could not complete the study protocol and were excluded from the study. All exercise sessions began between 0600 and 0800 h. The S and Con group subjects were instructed to maintain their current physical activity levels during the study [ 12 ]. Measurements Blood sampling and assays Following a 12-hour overnight fast, blood samples were taken between 0700-0800 h and collected from an antecubital vein. Samples were taken at baseline, 12 and 24 wks. Standard procedures for blood handling and processing were followed. All samples were taken 24 h after the last exercise bout. Blood samples were analyzed for serum TNF-\\u03b1, IL-6, PGE 2 , estrogen, osteocalcin, 1, 25- Vit D, CTX, PTH, CT, Ca 2+ , phosphorus, and fatty acid composition of neutrophil extracts. Serum IL-6 and TNF-\\u03b1 measurements were assayed using an enzyme-linked immunosorbent assay (ELISA) kit from Biosource (Nivelles, Belgium). PGE 2 was determined using competitive enzyme immunoassay (PGE 2 EIA kit- Monoclonal [solid plate], Ceyman-Germany). Serum estrogen levels were detected through a chemiluminescent method (Roche Diagnostics, Indianapolis, IN, USA) using an automatic immunoanalyzer. Circulating osteocalcin was measured using a previously developed radioimmunoassay (RIA). Serum 1, 25- Vit D was measured by a radioreceptor assay. Serum CTX was assessed using Serum Cross Laps One-Step ELISA (Osteometer BioTech, Herlev, Denmark). Serum PTH levels were measured through an electrochemiluminescent method, using Elecsys Systems/Modular analytics E170 (catalog number, 11972103; Roche Diagnostics). CT was determined by radioimmunoassay (RIA) with a polyclonal antiserum directed against the carboxyterminus. The antibodies were produced in rabbits against human CT, using a calcitonin kit (CIS Bio International ORIS groups, France). Serum calcium and phosphorus levels were measured by standard automated laboratory techniques. Incorporation of EPA and DHA into the cell membranes of neutrophils was analyzed using gas chromatography using known standards [ 12 , 37 ]. Bone measurements BMD measurements were made at the anterior-posterior lumbar spine (L 2 -L 4 ) and the non-dominant proximal femur, including femoral neck, using dual-energy X-ray absorptiometry (DXA; Norland XR36; Norland, Fort Atkinson, WI, USA). The coefficients of variation (CV) for the L 2 -L 4 and femoral neck were 1.21% and 1.54% respectively. Statistical analysis Group differences were determined using a one way analysis of variance ( ANOVA ) for repeated measures, for continuous variables. If the main effects F -ratio was significant, differences among groups were subsequently identified using a Bonferroni post-hoc analysis. Partial correlation and mixed model regression coefficients were used to evaluate the association between the variables studied. The statistical software program SPSS (SPSS Co, Chicago IL, version 17) for windows was used for data analysis. All statistical tests were performed and considered significant at a P \\u2264 0.05.\",\n",
            "        \"subjects\": null,\n",
            "        \"results\": null,\n",
            "        \"discussion\": null,\n",
            "        \"conclusion\": null\n",
            "    },\n",
            "    {\n",
            "        \"pmcid\": \"PMC3534646\",\n",
            "        \"abstract\": null,\n",
            "        \"intro\": null,\n",
            "        \"method\": \"Methods Population and Design The present study is cross-sectional and includes subjective and objective measures of hearing, as well as subjective ratings of emotional exhaustion. The sample was drawn from the Swedish Longitudinal Occupational Survey of Health (SLOSH) [22] , which was initiated by the Stress Research Institute at Stockholm University in 2006. The present cohort was established through the second data collection, which was conducted in April 2008 by Statistics Sweden. The sample of the present study was based on two types of inclusion criteria; 1) degree of EE and 2) living in the greater Stockholm area. The rationale for using the EE scores was previous findings indicating higher prevalence of hearing problems among individuals who are exposed to long-term stress [23] , [24] , [25] . For convenience and feasibility reasons, the participants had to be living in Stockholm County. The selection of participants was then based on scorings on the exhaustion dimension of the Maslach Burnout Inventory \\u2013 General survey (MBI-GS). The reason for this was that it should be possible to study possible differences in hearing outcomes in relation to degrees of stress-related pathology. Consequently, the strategy was to select contrasting groups including the 200 women and men (100 each) with the highest EE scores, 200 with medium EE and 200 with the lowest EE scores. After plotting the EE scores for women and men separately, the cutoff was set around the highest quartile, the median and around the lowest quartile. The selection procedures yielded a sample of 720 individuals. Some of them enrolled in a parallel study were excluded. As a result, the final sample included 687 individuals consisting of 143 women and 127 men with low EE scores, 118 women and 110 men with medium EE scores and 119 women and 103 men with high EE scores. With the help of Statistics Sweden, these individuals were invited to participate in the study. After sending out the invitations, 16 participants were removed due to either not fulfilling the inclusion criterion of living in Stockholm or having a protected or invalid address. The final cohort size was thus 671. After two reminders, 348 (52%) individuals enrolled in the study. Figure 1 illustrates the flow of the participants. 10.1371/journal.pone.0052945.g001 Figure 1 The flow of participants from the SLOSH cohort to the Stress and hearing study. The procedure of the study was described in an invitational letter, and was orally explained to each participant upon arrival. The study was approved by the Central Ethical Review Board in Stockholm (protocol no 2009/493-31/3). All gave their written consent to participate. Data Collection Extensive questionnaires were used to assess the participants\\u2019 demographics and various dimensions of mental and physical health (depressive symptoms, sleep and recovery status and emotional exhaustion) and wellbeing (e.g. life satisfaction) as well as hearing status (e.g., tinnitus, hearing loss and hyperacusis) health behaviors (e.g. physical activity and smoking) and psychosocial work environment. Hearing assessment Hearing status was assessed in the laboratory using a pure tone audiogram (PTA, 500\\u20138,000 Hz by octave steps) following the standard Hughson-Westlake procedure in both ears using circumaural headphones (TDH39). After that, a hearing-in-noise-test (HINT) [26] , [27] was conducted in the left and right ears separately. Uncomfortable loudness levels (ULLs) were determined according to the SAME-method (see below for a further description). Each participant was instructed to let the examiner know when the pure tone became uncomfortably loud, by speaking into a microphone. The testing started at 1 kHz with a signal intensity of 70 dB. If this level was perceived as uncomfortably loud, the intensity was decreased by 10 dB and the starting level on the following frequencies was decreased to 60 dB. The sound intensity was increased in 5 dB steps until the participant gave a response that the sound was uncomfortably loud. The tested frequencies were 0.5, 1, 2 and 4 kHz. The left and right ears were tested separately. In three cases ULLs were erroneously measured only to 100 dB and in three cases only to 105 dB. Hyperacusis was also assessed using the Hyperacusis Questionnaire (HQ) [21] . Good hearing status was defined as having no hearing loss in any PTA frequency (i.e. \\u226420 dB HL PTA). Exposure to noise at work was assessed by asking for the proportion of time study participants were exposed to weary noise (from 0 = never to 6 = all or almost all the time). EE Assessment Emotional exhaustion was assessed with the Maslach Burnout Inventory general survey (MBI-GS) using the emotional exhaustion subscale [28] . The scale consists of five items, derived from the Maslach Burnout Inventory human services survey (MBI-HSS) in unmodified form. Scorings reach from 1 (every day) to 6 (a few times a year or less/never). The items included to assess the construct are: \\u201cI feel emotionally drained by my work\\u201d, I feel completely exhausted when the working day is over\\u201d, \\u201cI feel tired when I get up in the morning to face a new working day\\u201d, \\u201cTo work during a whole day is really stressful for me\\u201d, \\u201cI feel burned out of my work\\u201d. Cronbach\\u2019s alpha and stability for the subscale have been reported to be satisfactory. Strong support for the construct validity of the Swedish translation of the MBI-HSS has been found [29] . The index was calculated on the basis that 4 out of 5 items had to be answered in order be included in the index. Acute stress task In order to maximize the likelihood of eliciting stress, the study participants were exposed to three stress-inducing tasks simultaneously. The first task consisted of an emotional Stroop-test [30] , [31] , [32] , where participants were asked to identify the colors of rapidly alternating words on a computer screen. At the bottom of the screen were boxes with the words: blue, brown, grey, green, yellow, pink, red, black, and white, and the task was to click on the box corresponding to the color of the letters of the word currently displayed. In contrast to the traditional Stroop-test, emotionally charged and neutral words were used to elicit a greater stress response. The charged and neutral words were distributed equally and presented in random order. Examples of charged words were: death, hate, and enemy. Examples of non-charged words were rose, senior, and bread. Different interfering colors were blinking on the background of the screen, and random colors were concurrently presented by a speaker voice via headphones. The pace of the visual and auditory presentation was 30 words per minute, while the background color shifted 80 times per minute. The second task, performed simultaneously, was a cold pressor exposure, i.e. hand in ice water (about 4\\u00b0C), which may be more or less painful for different individuals. The non-dominant hand was inserted wrist-deep into a bowl of water and ice for the entire the Stroop-test, i.e. four minutes. The cold-pressor test has been extensively used in laboratory settings to elicit a stress response [33] , [34] , [35] , [36] . It has been demonstrated that adding a social evaluative task further enhances the stress response [37] . Therefore, a social evaluation element was also included, i.e. video recording of the respondent and being observed by the researcher holding the camera. The video camera was stationed approximately 40 cm behind the computer screen, and participants were told that their facial expressions would be recorded for evaluation by a professional. The participants were instructed that they were allowed to remove their hand from the ice-water if the pain was unbearable. The recording started as soon as the headphones were properly equipped and one hand had been fully inserted the bowl of water. Plasma samples for steroid analysis Venous plasma samples were collected before and 20\\u201330 minutes after the stress test. The samples were centrifuged and stored in a deep freezer in immediate conjunction with blood sampling at temperature \\u221280 centigrades. They were analyzed with regard to steroid content by means of liquid chromatography tandem mass spectrometry [38] . The concentration of cortisol and estradiol was recorded for the purposes of the present study. Plasma cortisol concentration is an established stress (energy mobilization) indicator that is known to react within minutes after the onset of the stress exposure [39] . It is also known that the ability to respond with cortisol elevation may be hampered by conditions related to long lasting stress exposure i e stress lasting for weeks and months such as chronic fatigue [40] . It is also known that strong subjective reactions to a standardized stressor may be reduced by a pronounced cortisol reaction [41] . Estradiol is an anabolic hormone which protects against adverse effects of stress [42] . All blood samples were collected between 7.30 am and 11.30 am, between October and December 2009. Statistical analyses A one-way ANOVA with Bonferroni post-hoc tests was conducted in order to assess possible baseline differences in mean values of ULL. A 2-way ANCOVA was used to assess possible differences in mean ULLs over time (pre to post the acute stress task) between the EE-groups. Since there was a ceiling effect in the ULL assessment a new variable was created. If a person had a \\u201c>\\u201d symbol next to a 110 dB score, the value was set to 115 dB. This is a cautious approach to create at least a minor reduction in the ceiling effect. All the parametric analyses were conducted with this variable. Multivariate analyses, proportional odds model (also called ordered logistic regression), were used to calculate possible odds ratios, including interacting or confounding effects of age, gender, ear wax and hearing loss (only PTA). All logistic regression analyses were adjusted for hearing loss and ear wax. For the logistic regression, ULLs were trichotomized. ULL values \\u226485 dB HL were considered to be a sign of severe hyperacusis; 86\\u201395 dB moderate hyperacusis and \\u226596 normal values. EE was also divided by tertile split so that values between <1.2 was considered to be low levels of EE; 1.21 to 2.99 medium EE and \\u22653 high EE. All p-values were 2-sided and significance was set at 0.05. Data analysis was performed using SPSS Statistics version 20 and SAS.\",\n",
            "        \"subjects\": null,\n",
            "        \"results\": null,\n",
            "        \"discussion\": null,\n",
            "        \"conclusion\": null\n",
            "    },\n",
            "    {\n",
            "        \"pmcid\": \"PMC3909226\",\n",
            "        \"abstract\": null,\n",
            "        \"intro\": null,\n",
            "        \"method\": \"Materials and Methods Ethics Statement This project was approved by the Ethical Research Committee of Fiocruz for animal use (CEUA-L0023/08) according to the International Guiding Principles for Biomedical Research Involving Animals developed by the Council for International Organizations of Medical Sciences. The Ethical Research Committee of Fiocruz (CEPSH-03/2008) and the National Brazilian Ethical Board (784/2008, CONEP-14886) approved the human study. The study objectives were explained to all participants and written consent was obtained in writing before admission to this study. Parents/guardians provided written consent on behalf of child participants. Community Survey We performed a study in Buriti Seco and Morro Grande in Pedra Preta, a schistosomiasis endemic area in the rural region of Montes Claros, southeast Brazil [23] . This population has a low population migration index and has never been treated for schistosomiasis. The number of residents participating in the survey was 201 individuals aged 1 to 88 (93 female and 108 male). Four stool samples per individual were collected on four consecutive days for Kato-Katz analysis [24] . This method was performed using 18 slides, which were prepared as follows for each participant: 12 slides of the first sample and two slides each for the second, third and fourth sample in a total of 750 mg of faeces (18\\u00d742.7 mg). Samples were also analyzed by quantitative TF-Test [25] . Briefly, samples were passed through a nylon mesh and quantified in metal plates (500 mg). Each 500 mg portion was transferred to a tube with preservative solution (10% formalin) (Vetec; Rio de Janeiro, Brazil) and processed using neat ethyl acetate. Samples were centrifuged at 500 g for 2 min. The sediment was resuspended in 0.85% saline solution and analyzed using optical microscopy. TF-Test was repeated three times for each sample, with a total of 1500 mg of stool sample and results were equivalent to 36 Kato-Katz slides. Among the individuals participating in the survey, fifty patients aged 8\\u201388 were selected to provide serum samples (24 female and 26 male). Among these individuals, egg output ranged from 1 to 555 eggs per gram of faeces (EPG). Blood was collected by venous puncture and individual serum samples obtained after centrifugation of blood samples at 3000 g for 5 min. Samples were maintained at \\u201320\\u00b0C. All participants with egg-patent schistosomiasis were treated with single dose of praziquantel; 60 mg/kg for children and 50 mg/kg for adults, as recommended by the Brazilian Ministry of Health. Infections with other helminthes were treated with 400 mg albendazole. Thirty six positive patients were re-examined by Kato-Katz (18 slides) and TF-Test 30 and 90 days post therapy and retreated as needed; serum samples were also obtained for these individuals. A schematic showing the distribution of subjects and detection of infection is shown in S1. Healthy Donors Fifty two schistosomiasis-negative volunteers aged 22 to 65 (34 female and 18 male) were selected to be part of the negative control group. These volunteers were residents in non-endemic areas with no history of previous schistosome infection. Serum samples of these donors were processed as described earlier. Besides the historical criteria used to select healthy donors, we also performed ELISA assays for the detection of IgG antibodies against soluble adult worm antigens (ELISA-SWAP) and soluble egg antigens (ELISA-SEA). Production of Monoclonal Antibody Specific for CCA-protein (mAbCCA) Nine-week-old female BALB/c mice were subcutaneously immunized with 0.1 mg of the CCA \\u201ccrude antigen\\u201d previously obtained [17] using a new vaccine delivery method (US patent n.61/476,431) as adjuvant. Two weeks later, mice were boosted. Serum from mice was tested by ELISA to determine the antibody titer against the antigen. Mice were given an additional boost 15 days after the first boost by intraperitoneal injection of 0.1 mg of CCA \\u201ccrude antigen\\u201d. Three days later, spleen cells were fused with Sp2/O-Ag14 myeloma cells using polyethyleneglycol. The fused cells were cultured at 37\\u00b0C and selected with hypoxanthine-aminopterin-thymidine medium (Sigma-Aldrich; St. Louis, United States of America) (HAT). The initial screen of positive growth wells was made by ELISA. Antigen was diluted 1 \\u00b5g/ml in 0.05 M carbonate-bicarbonate buffer pH 9.6 (coating buffer) and microtiter plates MaxiSorp Surface (NUNC, Thermo Scientific; Roskilde, Denmark) were coated at 4\\u00b0C for 16 h. After blocking, 100 \\u00b5l of culture supernatants of the HAT-selected hybridomas were added and incubated for 1 h at room temperature. The bound antibodies were detected using peroxidase-conjugated anti-mouse IgG (1\\u22365,000) after incubation for 1 h at room temperature (Southern Biotech; New Orleans, United States of America) and 100 \\u00b5l of substrate 3,3\\u2032,5,5-tetramethylbenzidine solution (TMB) (Invitrogen; Grand Island, United States of America). ELISA positive hybridomas were selected. A second ELISA was performed to differentiate the hybridomas producing mAbs against the Lewis x epitope. Plates were coated with Lewis x tetrasacharide (Sigma-Aldrich; Saint Louis, United States of America) and the ELISA performed as described. Hybridomas non-reactive for Lewis x were selected and Ig-subclasses were determined by a kit for monoclonal isotyping (Sigma-Aldrich; Saint Louis, United States of America) ( Table 1 ). 10.1371/journal.pone.0087777.t001 Table 1 Characterization of mAbs produced against antigens of purified CCA glycoprotein. Clones Ig-subclass ELISA CCA specificity Lewis x specificity 1.3C2b IgG1 +++ \\u2013 1.2C6 IgG1 ++ \\u2013 4.4C3 IgG1 +++ \\u2013 5.1B4 IgG1 +++ \\u2013 5.1B1 ND ++ +++ 5.2A3 IgG1 ++ \\u2013 5.1D3 IgG1 ++ \\u2013 5F4.E4 ND ++ ++ 16D7.C10 IgM +++ \\u2013 16D7.C4 ND +++ ++ 16D7.B9 ND +++ ++ 12D3.F2 IgM +++ \\u2013 12D3.G8 ND ++ ++ \\u2212: negative reaction (OD<0,033); ++: positive reaction (OD<0,330); +++: positive reaction (OD<0,660); ND = not determined. The selected clones (16D7.C10 IgM and 5.1B4 IgG1) was grown in hybridoma medium (Invitrogen; Grand Island, United States of America) supplemented with penicillin (100 U/ml) and streptomycin (100 mg/ml) (Invitrogen; Grand Island, United States of America) at 37\\u00b0C. Culture supernatants were harvested in a final volume of 1 liter by inverting flask into collection tube without disturbing the cells and used for ammonium sulfate precipitation [26] . Precipitated proteins were dialyzed against PBS and then mAbs were purified by protein G purification column (Sigma-Aldrich; Saint Louis, United States of America). After measuring the OD at 280 nm of the fractions, the protein-containing fractions were stored at \\u221220\\u00b0C. Aliquots of the mAbs (16D7.C10) were conjugated to peroxidase with Zenon Mouse Labeling Kit (Invitrogen; Grand Island, United States of America), and also to Alexa Fluor 647 with Fluorochrome Protein Labeling Kit (Invitrogen; Grand Island, United States of America). Immunological Assay Evaluation for the Direct Detection of CCA in Individual Serum Immunomagnetic separation technique with mAbCCA-protein (IMS-mAbCCA) IMS was standardized according to the previously described [17] with some modifications. Paramagnetic microspheres (0.4 \\u00b5m; 10 6 microspheres/assay) (Estapor Microspheres, Merck; Lyon, France) were sensitized with 1 \\u00b5g/ml of mAbCCA-protein (5.1B4) in coating buffer for a physical adsorption. The following steps were performed with a rotating suspension system to improve antigen-antibody binding. Microspheres were incubated for 16 h at 4\\u00b0C, and then washed four times with 500 \\u00b5l of 0.15 M phosphate buffer saline with 0.05% tween pH 7.2 (washing buffer) using a 1.5 ml tube magnetic base (Invitrogen; Grand Island, United States of America). During washing steps, microspheres stays in contact with the magnetic base for 10 sec when washing buffer are removed. Nonspecific sites were blocked with 20% skim milk at 4\\u00b0C for 16 h. The microspheres were then maintained at 4\\u00b0C until use. On the day of the analysis the microspheres were washed, and then 200 \\u00b5l of undiluted serum samples were added into duplicate tubes, followed by incubation at 37\\u00b0C for 2 h. Microspheres were washed, then incubated at 37\\u00b0C for an hour with 100 \\u00b5l of peroxidase conjugated mAbCCA-protein (16D7.C10) diluted 1\\u2236400. Each tube was washed again and 100 \\u00b5l of TMB (Invitrogen; Grand Island, United States of America) were added to each well. The reaction was stopped after 10 min incubation by addition of 100 \\u00b5l/tube of 2 N sulfuric acid. A magnetic base was used to separate the beads from the supernatant, which was transferred to a microtiter plate and results were obtained at 450 nm in a microplate reader (Model 3550, Bio-Rad Laboratories; Tokyo, Japan). ELISA with mAbCCA-protein (ELISA-mAbCCA) ELISA was standardized based on the technique previously described [17] with some modifications. Microtiter plates MaxiSorp Surface (NUNC, Thermo Scientific; Roskilde, Denmark) were coated with mAbCCA-protein (5.1B4) 1 \\u00b5g/ml in coating buffer. Plates were washed and blocked by addition of 300 \\u00b5l per well of 2.5% skim milk, incubating at 37\\u00b0C for 1 h. After additional washing, 100 \\u00b5l of individual serum diluted 1\\u2236100 was added to the plate wells in duplicate and incubated for 1 h. Plates were then incubated with peroxidase conjugated mAbCCA-protein diluted 1\\u2236400 followed by addition of TMB (Invitrogen; Grand Island, United States of America). The enzymatic reaction stopped after 10 min of incubation in the dark with 50 \\u00b5l/well of 2 N sulfuric acid and the absorbance measured at 450 nm in microplate reader (Model 3550, Bio-Rad Laboratories; Tokyo, Japan). Fluorescent microscopy analysis of IMS products using mAbCCA-protein (FluoIMS-mAbCCA) The same procedure adopted for IMS - mAbCCA was used here for a double blind analysis. After microspheres were sensitized and blocked, 200 \\u00b5l of serum samples were added and incubated for 1 h at 37\\u00b0C. Microspheres were then washed and 100 \\u00b5l of Alexa Fluor 647 conjugated mAbCCA-protein (16D7.C10) (1\\u2236400) were added followed by an incubation for 1 h at 37\\u00b0C. Qualitative analysis of 5 \\u00b5l of microspheres suspension was performed by examination on a glass slide with a fluorescent microscope (Karl Zeiss; Oberkochen, Germany) to visualize fluorescent microspheres (642 nm, emission filter LP590). Photographic records were taken with a digital camera (Canon EOS). Positive and negative controls were assayed for each technique as control of nonspecific adsorption of conjugate. Statistical Analysis Absorbance value data were analyzed with Minitab Inc. by Kolmogorov-Smirnov normality test. Normal distributed data were analyzed by Student\\u2019s t test and non-normal data by Mann-Whitney test. Comparisons between methods were done by 2 proportions\\u2019 Fisher analysis (p\\u22640.05 as significance level). Sensitivity, specificity and cut-off values were determined with Prism 4.0. Agreement between methods was measured using Cohen coefficient [27] and analyzed by Landis & Koch definition [28] .\",\n",
            "        \"subjects\": null,\n",
            "        \"results\": null,\n",
            "        \"discussion\": null,\n",
            "        \"conclusion\": null\n",
            "    },\n",
            "    {\n",
            "        \"pmcid\": \"PMC4393161\",\n",
            "        \"abstract\": \"Epithelial barrier dysfunction has been implicated as one of the major contributors to the pathogenesis of inflammatory bowel disease. The increase in intestinal permeability allows the translocation of luminal antigens across the intestinal epithelium, leading to the exacerbation of colitis. Thus, therapies targeted at specifically restoring tight junction barrier function are thought to have great potential as an alternative or supplement to immunology-based therapies. In this study, we screened\",\n",
            "        \"intro\": null,\n",
            "        \"method\": \"Materials and Methods Caco-2 cell culture The human colonic adenocarcinoma epithelial cell line Caco-2 was obtained from Riken Cell Bank (Ibaraki, Japan) and kept in a humidified incubator at 37\\u00b0C with 5% CO 2 . Caco-2 cells were maintained in Dulbecco's Modified Eagle Medium (DMEM)-High glucose (Wako, Osaka, Japan) supplemented with 10% heat-inactivated (30 min, 56\\u00b0C) fetal bovine serum (Biowest, Nuaill\\u00e9, France), 100 U/mL penicillin, 100 \\u03bc g/mL streptomycin, and 1% nonessential amino acids (NEAA) (Wako). In this study, Caco-2 cells were used between passages 25 and 40. Cells cultivated to 80% confluence were seeded on 12-well Transwell\\u00ae inserts (1.12 cm 2 polycarbonate membrane with 0.4 \\u03bc m pore size; Corning Life Sciences, Corning, NY) at a density of 5 \\u00d7 10 4 cells per well. The culture medium was changed every 2 days until 20 days later when full polarization of the Caco-2 cell monolayer was achieved. The integrity of the Caco-2 cell monolayer was evaluated by measuring the transepithelial electrical resistance (TER) using a milicell-ERS (Millipore, Billerica, MA). The TER values measured in the experiments were summarized as table form below each figure, and all the figures were expressed as the relative TER value compared to the value before experimental treatment. Bacterial isolates and sample preparation Thirty bacterial isolates from human feces were used in this study (Table 1 ). Each bacterial isolate was identified based on a nearly full 16S rRNA sequence, which was deposited in DNA Data Bank of Japan (DDBJ). All bacterial strains were cultured in Difco Lactobacilli MRS (de Man-Rogosa Sharpe) broth (Becton Dickinson, Sparks, MD) at 37\\u00b0C under anaerobic conditions. Heat-killed bacteria were prepared by heating bacteria at 95\\u00b0C for 10 min. Table 1 List of bacterial isolates used in this study Strain Accession number Lactobacillus L . salivarius WU 30 AB932525 L . fermentum WU 30 AB932548 L . fermentum WU 33 AB932537 L . gasseri WU 04 AB932527 L . gasseri WU 06 AB932530 L . rhamnosus WU 08 AB932536 L . gasseri WU 11 AB932519 L . pantheris WU 61 AB932532 L . pantheris WU 21 AB932522 L . rhamnosus WU 07 AB932535 L . rhamnosus WU 09 AB932547 L . rhamnosus WU 12 AB932520 L . rhamnosus WU 14 AB932521 L . salivarius WU 57 AB932528 L . salivarius WU 60 AB932531 L . salivarius WU 63 AB932533 Enterococcus E . cecorum WU 65 AB932534 E . faecium WU 31 AB932549 E . avium WU 58 AB932529 E . avium WU 22 AB932523 E . avium WU 76 AB932546 E . raffinosus WU 27 AB932524 Bifidobacterium B . bifidum WU 11 AB932538 B . bifidum WU 12 AB932539 B . bifidum WU 20 AB932541 B . bifidum WU 57 AB932544 B . longum WU 16 AB932540 B . longum WU 35 AB932543 B . adolescentis WU 22 AB932542 B . pseudocatenulatum WU 06 AB932550 Evaluation of the TJ-barrier-strengthening ability of bacterial isolates To screen the TJ-barrier-strengthening bacteria, polarized Caco-2 cell monolayers were exposed to bacterial isolates. On day 18 of cultivation, both apical and basal compartments of Caco-2 cell monolayers were washed twice with prewarmed PBS and then fresh DMEM without antibiotics added to both compartments of the Transwell\\u00ae cell culture system. On day 20, the TER of each well was measured using a milicell-ERS. Caco-2 cell monolayers with >650 \\u03a9 cm 2 of TER were used for further experiments. After the measurement of TER, the apical compartment of Caco-2 cell monolayers was washed with prewarmed PBS, and then the bacterial suspensions were exposed to the apical side at a multiplicity of infection (MOI, ratio of bacteria number to epithelial cell number) of 1 in a humidified incubator at 37\\u00b0C with 5% CO 2 . Culture medium alone was used as a negative control. After 24 h incubation, the TER value was measured to assess epithelial TJ permeability. All experiments were performed in triplicate. Evaluation of preventive effect of bacterial isolates on TNF-\\u03b1-induced impairment of TJ permeability In the prevention screening, Caco-2 cell monolayers were washed with prewarmed PBS and then bacteria (MOI of 1) added to the apical compartment one hour before TNF- \\u03b1 treatment. Next, the Caco-2 cell monolayers were treated with TNF- \\u03b1 (20 ng/mL) from the basolateral compartment to induce TJ dysfunction and incubated for a further 23 h. The TER value was measured to assess epithelial barrier function after a total of 24 h incubation. The TNF- \\u03b1 treated group without bacteria was used as a negative control. The prevention effect was estimated by the change in TER compared with the value before bacteria was added. All experiments except for B . bifidum WU57 were performed in triplicate. The assay of B . bifidum WU57 was performed in duplicate. Evaluation of restorative effect of bacterial isolates on TNF-\\u03b1-induced impairment of TJ permeability Fully polarized Caco-2 cell monolayers were washed with prewarmed PBS then antibiotic-free DMEM containing TNF- \\u03b1 was added to the basolateral compartment. After 48 h incubation, the Caco-2 cell monolayers were washed on both the apical and basolateral sides with prewarmed PBS. Subsequently, the apical compartment of the Caco-2 cell monolayers was exposed to the bacterial suspensions at a MOI of 1. After 24 h incubation, the TER value was measured to assess the restoration of TJ function. All experiments were performed in triplicate. RNA isolation and Gene expression analysis Total RNA was extracted using the RNeasy mini kit (Qiagen, Tokyo, Japan) according to the manufacturer's instructions, and the RNA concentration determined by absorbance at 260/280 nm using a spectrophotometer (nano-Drop ND-1000; Thermo Scientific, Wilmington, DE). For mRNA expression analysis, cDNA was prepared from 1500 to 2000 ng of total RNA using the High-Capacity cDNA reverse transcription kit (Applied Biosystems, Carlsbad, CA) according to the manufacturer's instructions. The reverse transcription reactions were performed in a thermo cycler (iCycler, Bio-Rad, Hercules, CA) at 25\\u00b0C for 10 min, 37\\u00b0C for 120 min, and 85\\u00b0C for 5 min. Real-Time PCR was performed with a StepOnePlus\\u2122 Real-time PCR System (Applied Biosystems) using the TaqMan\\u00ae Gene Expression Assays No. Hs00268480_m1 (ZO-1), Hs00170162_m1 (occludin), Hs00221623_m1 (claudin-1), Hs00170423_m1 (E-cadherin), and Hs99999905_m1 (GAPDH) with TaqMan gene expression master mix according to the manufacturer's instructions (Applied Biosystems). The time-dependent TJ gene expression analysis during the coculture of B . bifidum WU12 and Caco-2 cells were repeated two independent times, each performed in duplicate or triplicate. The endpoint TJ gene expression analysis of Caco-2 cells treated with B . bifidum WU12 or acetate for 24 h were repeated three independent times, each performed in triplicate. Characterization of metabolites in culture supernatants using 1 H-NMR spectroscopy Supernatants of B . bifidum monocultures and the coculture of TNF- \\u03b1 -treated Caco-2 cells and B . bifidum were collected at 6, 12, and 24 h. DMEM and the culture supernatants of Caco-2 cell monolayers treated with TNF- \\u03b1 for 48 h were also prepared. The metabolic profile of the culture supernatant was analyzed using a NMR spectrometer. The NMR samples were prepared by mixing with 10 mmol/L sodium 2,2-dimethyl-2-silapentane-5-sulfonate (DSS) dissolved in deuterium oxide (D 2 O) and then transferred into 5 mm NMR tubes. The spectra of these supernatant samples were obtained on a Bruker DRU-700 spectrometer equipped with an inverse (proton coils closest to the sample) gradient 5 mm cryogenically cooled 1 H/ 13 C/ 15 N probe (Bruker Biospin, Rheinstetten, Germany), operating at 700.15 MHz for protons. The NMR measurement and data analysis of Carr\\u2013Purcell\\u2013Meiboom\\u2013Gill (CPMG) pulse sequences was performed as per previous studies (Date et al. 2012 ). The spectra acquired at 298 K, and 32,768 data point with a spectral width of 12,500 Hz were collected into 32 transients and 16 dummy scans. The NMR spectra were processed using TopSpin 3.1 software (Bruker Biospin) and assigned using the SpinAssign programs at the PRIMe web site ( http://prime.psc.riken.jp/ ) (Chikayama et al. 2010 ). The CPMG spectra data were reduced by subdividing spectra into sequential 0.04 ppm-designated regions between 1 H chemical shifts of \\u22120.5 to 9.0 ppm. After exclusion of water resonance (4.7\\u20135.0 ppm), each region was integrated and normalized to the total of DSS integral regions. The normalized spectra data were statistically evaluated by Principle Component Analysis (PCA) using the \\u2018R (2.15.2)\\u2019 software. Determination of acetate, formate, and lactate in culture supernatants Supernatant samples were treated with an equal volume of acetonitrile (Wako) at room temperature to precipitate proteins, and then centrifuged for 10 min at 20,000 \\u00d7 g at room temperature. The supernatant was collected by filtration through a 0.2 \\u03bc m Advantec\\u00ae disposable membrane filter unit (Toyo Roshi, Tokyo, Japan) and then stored at \\u221280\\u00b0C until analysis. Samples were diluted 100-fold in ddH 2 O on the eve of analysis. Acetate, formate, and lactate content in culture supernatants was determined by ICS 2100 ion chromatography (Dionex Instruments, Sunnyvale, CA) with a Dionex IonPac AS-19 column and a 1\\u201340 mmol/L KOH gradient mobile phase.\",\n",
            "        \"subjects\": null,\n",
            "        \"results\": \"Results Exploration of microorganisms strengthening tight junction barrier After the polarized Caco-2 cell monolayers were incubated with each bacterial isolate or medium alone for 24 h, the intestinal barrier function was determined by measurement of the TER (Fig. 1 and Table 2 ). All Enterococcus isolates (including E . faecium , E . avium , E . cecorum, and E . raffinosus ) induced an appreciable decrease in TER. Some Lactobacillus species ( L . gasseri , L . rhamnosus , and L . salivarius ) also induced a significant decrease in TER. In contrast, seven Bifidobacterium isolates and six Lactobacillus isolates induced a 5.7\\u201333.9% increase in TER. Therefore, we examined the preventive effects of these screened isolates (7 strains of Bifidobacterium and 6 strains of Lactobacillus ) on TNF- \\u03b1 -induced impairment of TJ permeability. Table 2 The TER value measured at the screening of bacterial isolates for TJ-barrier-strengthening effects in polarized Caco-2 monolayers Genus Species Strain 0 h 24 h coincubation TER (\\u03a9 \\u00d7 cm 2 ) \\u00b1SD TER (\\u03a9 \\u00d7 cm 2 ) \\u00b1SD \\u2013 \\u2013 Blank 1085.1 53.7 1115.3 90.5 Lactobacillus fermentum WU30 1194.5 16.7 1371.8 13.8 WU33 927.9 32.6 1160.9 38.1 gasseri WU04 919.3 37.5 176.4 81.5 WU06 1056.3 24.8 1157.1 38.7 WU11 1074.6 57.8 1229.9 49.8 pantheris WU21 1069.0 23.4 1283.7 35.6 WU61 1098.9 31.4 1059.3 24.9 rhamnosus WU07 1109.4 22.7 948.5 87.2 WU08 1095.9 33.4 1158.6 44.4 WU09 1138.9 31.1 904.4 22.7 WU12 1171.7 18.9 272.3 177.9 WU14 1150.8 24.4 342.5 35.3 salivarius WU30 1160.1 67.2 19.2 2.3 WU57 1081.7 3.4 55.8 94.7 WU60 1089.6 43.5 109.2 89.4 WU63 1159.8 71.9 9.5 15.5 Enterococcus avium WU22 1050.4 51.5 9.9 2.6 WU58 1092.9 30.6 51.0 3.0 WU76 935.8 45.1 0.9 5.6 cecorum WU65 1100.4 31.4 9.9 2.3 faecium WU31 1162.0 27.2 20.7 10.8 raffinosus WU27 1135.5 71.2 6.5 1.7 Bifidobacterium adolescentis WU22 959.7 15.2 1123.9 25.6 bifidum WU11 911.9 60.8 1148.6 54.7 WU12 900.3 74.1 1166.5 82.0 WU20 1085.1 32.1 1284.5 30.4 WU57 873.8 15.5 1170.2 29.8 longum WU16 1029.1 19.5 1160.5 23.6 WU35 1033.9 21.5 1030.2 38.7 pseudocatenulatum WU06 923.8 100.5 997.4 114.5 Figure 1 Screening of bacterial isolates for TJ-barrier-strengthening effects in polarized Caco-2 monolayers at MOI = 1. All the bacteria isolates were incubated with Caco-2 monolayers for 24 h. The average of actual TER values of Caco-2 monolayer used in this experiment was 1057 \\u00b1 98 \\u03a9 cm 2 . Experiments were carried out in triplicate, and data represent the means of relative changes in TER \\u00b1 SD. ** P < 0.01, * P < 0.05 compared with the blank (media alone) control group by t -test. Evaluation of preventive effect of screened bacteria on TNF-\\u03b1-induced impairment of tight junction function TNF- \\u03b1 treatment induced a marked decrease in TER of the Caco-2 cell monolayers. Pretreatment with Lactobacillus strains did not attenuate the TNF- \\u03b1 -induced decrease in TER (Fig. 2 and Table 3 ). In contrast, four of the seven Bifidobacteria strains prevented the TNF- \\u03b1 -induced drop in TER. Interestingly, exposure to three Bifidobacteria strains ( B . bifidum WU12, WU20, and WU57) led to a reproducible and significant 4.2\\u201312.4% increase in TER ( P < 0.01). Additionally, one Bifidobacteria strain ( B . bifidum WU11) and two Lactobacillus strains ( L . gasseri WU11 and L . pantheris WU21) demonstrated partial suppression of the TNF- \\u03b1 -induced TER drop, (\\u22124.2 to \\u22125.8% change). Table 3 The TER value measured at the screening of preventive effect against TNF- \\u03b1 -induced TJ barrier impairment Genus Species Strain 0 h 24 h coincubation TER (\\u03a9 \\u00d7 cm 2 ) \\u00b1SD TER (\\u03a9 \\u00d7 cm 2 ) \\u00b1SD \\u2013 \\u2013 Blank 979.1 79.5 978.3 63.1 \\u2013 \\u2013 TNF- \\u03b1 control 948.5 117.2 774.1 92.9 Lactobacillus fermentum WU30 1262.1 32.4 1129.5 11.4 WU33 996.6 35.1 853.3 27.3 gasseri WU06 1215.8 67.3 1078.4 67.6 WU11 846.2 35.9 810.7 32.6 pantheris WU21 880.5 19.8 837.9 9.8 rhamnosus WU08 1235.2 36.4 1104.1 10.6 Bifidobacterium adolescentis WU22 1031.7 49.1 739.0 34.1 bifidum WU11 858.1 67.3 808.1 51.0 WU12 751.0 34.7 844.3 35.1 WU20 775.2 16.1 834.2 1.3 WU57 794.6 1.6 827.7 18.2 longum WU16 1043.7 47.7 1024.2 28.4 pseudocatenulatum WU06 958.2 21.4 790.5 10.8 Figure 2 The preventive effect against TNF- \\u03b1 -induced TJ barrier impairment. (A) Schedule of Caco-2 monolayer treatment and TER analysis. The bacterial suspensions were added to the apical side of Caco-2 monolayers an hour before TNF- \\u03b1 stimulation. After 1 h, Caco-2 monolayers were stimulated on the basolateral side with TNF- \\u03b1 , and then incubated for 24 h. (B) The preventive effect of bacterial isolates with barrier-strengthening ability at MOI = 1. The average of actual TER values of Caco-2 monolayer used in this experiment was 976 \\u00b1 166 \\u03a9 cm 2 . Experiments were carried out in triplicate, and data represent the means of relative changes of TER \\u00b1 SD. ** P < 0.01, * P < 0.05 compared with the TNF- \\u03b1 control group by t -test. Evaluation of bacteria-induced restoration of epithelial function In this study, the addition of TNF- \\u03b1 (20 ng/mL) to the basolateral compartment produced an approximately 15\\u201320% drop in Caco-2 TER for 48 h. TER of TNF- \\u03b1 -treated Caco-2 cell monolayers did not increase after changing to fresh medium without TNF- \\u03b1 , indicating no spontaneous recovery of the Caco-2 monolayer TJ function. Based on the above results, it was hypothesized that the four Bifidobacteria strains ( B . bifidum strain WU12, WU20, WU57, and B . longum strain WU16) which exhibited protective effects against TNF- \\u03b1 induced damage could also be involved in the restoration of intestinal epithelial TJ function. Thus, we examined whether Bifidobacteria strains can restore the TJ function of Caco-2 cell monolayers damaged by the treatment of TNF- \\u03b1 . As a result, all tested Bifidobacterium (3 strains of B . bifidum , 1 strain of B . longum ) facilitated the restoration of the Caco-2 TJ function (Fig. 3 and Table 4 ). To elucidate the mechanism of Bifidobacteria -induced epithelial restoration, B . bifidum strain WU12 was chosen for further study. Table 4 The TER value measured at the repairing of TNF- \\u03b1 -induced barrier dysfunction by administration of bacterial isolates post TNF- \\u03b1 stimulation Genus Species Strain Before treatment TNF- \\u03b1 treated 24 h coincubation TER (\\u03a9 \\u00d7 cm2) \\u00b1SD TER (\\u03a9 \\u00d7 cm2) \\u00b1SD TER (\\u03a9 \\u00d7 cm2) \\u00b1SD \\u2013 \\u2013 Blank 696.8 14.9 680.8 27.4 697.9 30.2 \\u2013 \\u2013 TNF- \\u03b1 control 1050.6 14.7 860.2 20.7 842.2 12.7 Bifidobacterium longum WU16 1026.9 71.2 800.2 37.8 1077.6 83.0 bifidum WU12 884.6 66.6 712.9 41.8 1058.2 84.1 WU20 903.3 40.0 675.5 32.6 918.6 73.4 WU57 709.1 36.5 613.9 20.2 705.0 44.0 Figure 3 Repair of TNF- \\u03b1 -induced barrier dysfunction by administration of bacterial isolates post TNF- \\u03b1 stimulation. (A) The schedule of TNF- \\u03b1 and bacterial treatment of Caco-2 monolayers. Caco-2 monolayers were stimulated on the basolateral side with TNF- \\u03b1 , and then incubated for 48 h. After 48 h, the bacterial suspension were added into the apical side, and coincubated for 24 h. (B) Repair effect of four Bifidobacterial species on the TJ function of Caco-2 cell monolayer damaged by basolateral TNF- \\u03b1 stimulation. The average of actual TER values of Caco-2 monolayer used in this experiment was 879 \\u00b1 147 \\u03a9 cm 2 . Experiments were carried out in triplicate, and data represent the means of relative changes in TER \\u00b1 SD. ** P < 0.01 compared with the TNF- \\u03b1 treated TER of the same group by t -test. Characterization of the factor responsible for B . bifidum -induced restoration of epithelial function We determined whether the bacterial components or secreted metabolites were responsible for the epithelial TJ restoration induced by B . bifidum strain WU12. Heat-killed B . Bifidum WU12 clearly had a diminished a repair capacity (Fig. 4 A and Table 5 A). Interestingly, only intact cells of B . bifidum strain WU12 resulted in a pronounced increase in TER of the Caco-2 cell monolayers, indicating a high repair capacity on the epithelial TJ. Thus, the interaction between live Bifidobacterium and Caco-2 cells are a prerequisite for Bifidobacteria -induced epithelial restoration. However, high-dose administration of intact bacterial cells at a MOI of 10 resulted in a significant decrease in TER of TNF- \\u03b1 -untreated Caco-2 cell monolayers (Fig. 4 B and Table 5 B). This might be due to the acceleration of medium acidification, as suggested by the medium turning yellow during the coculture of Caco-2 cells and live Bifidobacterium . Table 5 The TER value measured at the (A) The effects of MOI on cocultivation of Caco-2 monolayers with live/heat inactive B . bifidum WU12, (B) characterization of B . bifidum WU12-induced epithelial TJ restoration with live/heat-killed bacteria (A) Sample TNF- \\u03b1 treated Postadministration of bacteria TER (\\u03a9 \\u00d7 cm 2 ) \\u00b1SD TER (\\u03a9 \\u00d7 cm 2 ) \\u00b1SD No bacteria 734.5 13.4 727.1 18.8 Alive bacteria (MOI=1) 652.0 11.7 888.0 12.1 Heat-killed bacteria (MOI=1) 709.1 102.1 739.4 90.0 (B) Sample 0 h 24 h coincubation TER (\\u03a9 \\u00d7 cm 2 ) \\u00b1SD TER (\\u03a9 \\u00d7 cm 2 ) \\u00b1SD No bacteria 867.4 71.2 879.0 78.1 Alive bacteria (MOI=1) 692.7 75.9 791.3 89.5 Alive bacteria (MOI=10) 655.0 42.3 558.7 19.7 Heat-killed bacteria (MOI=1) 739.8 24.2 721.5 17.0 Heat-killed bacteria (MOI=10) 807.7 4.7 824.1 8.5 Figure 4 Characterization of the factor responsible for B . bifidum -induced TJ restoration. (A) Effects of the heat treatment (95\\u00b0C for 10 min) of bacterial cells on B . bifidum WU12-induced epithelial TJ restoration capacity. The average of actual TER values of Caco-2 monolayer before basolateral TNF- \\u03b1 stimulation used in this experiment was 894 \\u00b1 99 \\u03a9 cm 2 . Experiments were carried out in triplicate, and data represent the means of relative changes in TER \\u00b1 SD before and after administration of B . bifidum WU12. Statistical differences between before and after administration of B . bifidum WU12 in each experimental group were calculated by t -test (** P < 0.01). (B) The cytotoxic effect of MOI on cocultivation of Caco-2 monolayers with alive or heat-killed B . bifidum WU12. The average of actual TER values of Caco-2 monolayer used in this experiment was 753 \\u00b1 91 \\u03a9 cm 2 . Experiments were carried out in triplicate, and data represent the means of relative changes of TER \\u00b1 SD. Statistical differences before and after administration of alive or heat-killed B . bifidum WU12 were calculated by t -test (** P < 0.01). Modulation of the mRNA expression of tight junction proteins by B . bifidum The kinetics of TER change during B . bifidum -induced epithelial restoration is shown in Figure 5 A and Table 6 . A rapid increase in TER was observed after 12 h of coincubation which reached 9.3% at 24 h. Quantitative RT-PCR (qRT-PCR) analysis was performed to characterize the changes in gene expression of tight junction proteins during B . bifidum -induced epithelial restoration. Although the expression dynamics of the TJ protein genes during the coincubation period were not correlated with the change in TER in Caco-2 cells incubated with B . bifidum , the administration of B . bifidum significantly enhanced the occludin mRNA expression after 24 h incubation (Fig. 5 B, C, and D). Table 6 The TER value measured during the B . bifidum WU12-induced Caco-2 monolayer restitution in 24 h Sample B . bifidum NO B . bifidum TER (\\u03a9 \\u00d7 cm 2 ) \\u00b1SD TER (\\u03a9 \\u00d7 cm 2 ) \\u00b1SD Na\\u00efve cell 890.0 124.7 762.1 35.8 0 h (TNF- \\u03b1 treated) 682.9 89.1 598.1 20.0 9 h 727.6 71.1 12 h 782.5 111.3 598.5 18.8 14 h 706.3 17.4 22 h 910.9 20.1 24 h 869.9 14.2 596.6 24.8 Figure 5 The kinetics of TER and TJ-related mRNA expression during B . bifidum WU12-induced Caco-2 monolayer restitution. (A) B . bifidum WU12-induced Caco-2 monolayer restitution over time as estimated by TER. Caco-2 monolayers incubated without B . bifidum WU12 were used as the negative control. The average of actual TER values of Caco-2 monolayer before basolateral TNF- \\u03b1 stimulation used in this experiment was 822 \\u00b1 86 \\u03a9 cm 2 . Experiments were carried out in triplicate, and data represent the means of relative changes of TER \\u00b1 SD. (B\\u2013D) Temporal changes of mRNA expression changes in TJ protein genes (Claudin-1, Occludin, ZO-1) during Caco-2 monolayer restitution induced by B . bifidum WU12. The time course of mRNA expression of nontreated group (w/o B . bifidum ) was used as the negative control. Experiments were carried out in triplicate, and data represent the means of relative changes in \\u00b1 SD. Statistical differences were calculated by t -test, comparing conditions with the negative control group at the same time point (** P < 0.01). Time-course characterization of the coculture supernatant Based on the above results, it was hypothesized that metabolites produced by B . bifidum strain WU12 play a crucial role in the process of Bifidobacteria -induced epithelial restoration. To characterize the metabolic dynamics during the epithelial restoration, time-course metabolic profiles of the apical side supernatant were determined using a 1 H-NMR-based metabonomics approach. The principal component analysis (PCA) of the 1 H NMR chemical shift data revealed clear differences in metabolite profiles between the coculture groups, the monoculture of B . bifidum , and the monoculture of Caco-2 cell monolayers (Fig. 6 ). Contributions of the first two components (PC1 and PC2) were 92.5% and 6.97%, respectively (Fig. 6 A). The loading plot analysis showed that the major metabolites contributing to the separation of each component were lactate and glucose in PC1 and acetate and formate in PC2 (Fig. 6 B). In the metabolite profile of B . bifidum alone group, there was only a minute amount of acetate production detected during cultivation. In the Caco-2 monoculture treated with TNF- \\u03b1 for 48 h, glucose was consumed and a large amount of lactate produced. In contrast, the coculture of B . bifidum and Caco-2 notably enhanced the production of acetate after 12 h of the incubation. Furthermore, formate was also produced in the coculture of B . bifidum and Caco-2. Figure 6 Principle component analysis (PCA) score plots and loading plots of the metabolic profile affected by B . bifidum WU12. (A) PCA score plot was computed from blank (square), Caco-2 mono-culture 48 h (with TNF- \\u03b1 ; diamond), time-dependent change of the coculture group (TNF- \\u03b1 treated; circle), time-dependent change in B . bifidum WU 12 mono-culture group (triangle), B . bifidum WU12 coculture control (dash). (B) PCA loading plot derived from the information on the PCA score plots. Experiments were carried out in duplicate. Evaluation of the production of acetate, formate, and lactate in the coculture supernatant of Bifidobacterium isolates and Caco-2 cells We conducted ion chromatography analysis to determine the production of acetate, formate and lactate in the supernatant during coculture of Caco-2 and B . bifidum WU12, with and without TNF- \\u03b1 , and by B . bifidum alone (Fig. 7 ). Interestingly, the production of acetate and formate was amplified by the coculture of B . bifidum WU12 and Caco-2 monolayers. Acetate production in the coculture group increased 1.8 times compared with that of the monoculture of the B . bifidum strain WU12. Furthermore, we determined whether the production of acetate, formate, and lactate in the coculture supernatant differed between the seven Bifidobacterium strains, which fortified the epithelial TJ barrier of naive polarized Caco-2 cells (Fig. 1 ). Analysis of the culture supernatant revealed that the production of lactate was mainly derived from cell metabolism of Caco-2 cells as there were no differences between the monoculture of Caco-2 and the coculture of Caco-2 and Bifidobacterium (Fig. 8 ). In contrast, the production of acetate and formate varied among Bifidobacterium species. Notably, B . bifidum produced higher levels of acetate and formate in comparison with other species ( B . adolescentis, B . longum, and B . pseudocatenulatum ) (Fig. 8 ). Figure 7 Increased production of acetate and formate by the cocultures of Caco-2 monolayer and B . bifidum WU. Acetate, formate, and lactate concentrations in the culture supernatant after 24 h incubation were determined by ion chromatography. Experiments were carried out in triplicate, and data represent the means \\u00b1 SD. Statistical differences were calculated by t -test (** P < 0.01). Figure 8 Production of acetate and formate in the coculture supernatant varied in a strain-dependent manner. (A\\u2013C) Acetate, formate, and lactate concentrations in the supernatant after 24 h cocultures of Caco-2 monolayer and seven different Bifidobacterial strains were determined by ion chromatography. Experiments were carried out in triplicate, and data represent the means \\u00b1 SD. Statistical differences were calculated by t -test (** P < 0.01, * P < 0.05). Effects of acetate and formate on TJ barrier function in Caco-2 monolayers We determined whether acetate and formate might be involved in TJ barrier restoration of Caco-2 monolayers whose TER were decreased approximately 20% by basolateral TNF- \\u03b1 stimulation for 48\\u201372 h. Interestingly, administration of 10 and 20 mmol/L acetate to the apical side to the TNF- \\u03b1 treated Caco-2 monolayers showed a marked restorative effect after 24 h incubation ( P < 0.05) (Fig. 9 A and Table 7 ). On the other hand, the restorative effect of formate was weaker than the effect of acetate. Furthermore, occludin mRNA expression were significantly enhanced by apical stimulation with acetate (20 mmol/L) as the case of administration of B . bifidum WU12 (Fig. 9 B). Occludin mRNA levels by acetate or B . bifidum WU12 were 1.28-fold and 1.49-fold, respectively ( P < 0.05). On the other hand, apical stimulation with acetate had no significant effect on the gene expression of claudin-1 and ZO-1. Table 7 The TER value measured at the experiment of 24 h apical treatment of Caco-2 monolayers with varying concentrations of acetate and formate to induce the TJ barrier restorative effects Sample Before experiment TNF- \\u03b1 treated 24 h treatment TER (\\u03a9 \\u00d7 cm 2 ) \\u00b1SD TER (\\u03a9 \\u00d7 cm 2 ) \\u00b1SD TER (\\u03a9 \\u00d7 cm 2 ) \\u00b1SD Blank 768.1 120.0 787.2 165.6 800.1 162.0 TNF- \\u03b1 875.0 89.5 698.2 91.6 678.8 92.9 Formate 2.5 mmol/L 809.9 95.3 669.3 92.0 669.3 156.1 Formate 5 mmol/L 828.0 89.1 688.4 78.6 743.2 114.9 Acetate 10 mmol/L 822.0 126.3 688.4 109.1 858.2 181.6 Acetate 20 mmol/L 823.8 105.0 691.0 97.7 868.7 144.6 Figure 9 Acetate induced the TJ barrier restorative capacity. (A) Effects of acetate and formate on the TJ barrier restoration. Caco-2 monolayers were stimulated on the basolateral side with TNF- \\u03b1 , and then incubated for 48 h. After 48 h, acetate or formate were added into the apical side, and incubated for 24 h. The average of actual TER values of Caco-2 monolayer used in this experiment was 846 \\u00b1 96 \\u03a9 cm 2 . Data represent the average of two or three independent experiments carried out in triplicate. (B) Effects of acetate or B . bifidum WU12 on mRNA expression of TJ protein genes. TJ protein gene expressions in Caco-2 cells treated with acetate or B . bifidum WU12 for 24 h was were determined by quantitative RT-PCR. Data represent the average of three independent experiments carried out in duplicate or triplicate. Statistical differences were calculated by t -test (* P < 0.05). Dependency of Bifidobacteria-induced TER restoration on the differentiation stage of Caco-2 cells To investigate whether the TER-enhancing effect of B . bifidum depends on polarization of the intestinal epithelial cells, we determined the TER-enhancing activity in untreated Caco-2 cells at three different time points (Fig. 10 A and Table 8 ): day 7, unpolarized; day 14, early-polarization; day 20, stable-polarization. In the early- and stable-polarization stages, the TER values of the Caco-2 cell monolayers significantly increased by approximately 30% in coculture with B . bifidum WU12. In contrast, B . bifidum -induced TER-enhancing effect in unpolarized Caco-2 cells was weak compared with that in polarized Caco-2 cells. Interestingly, the differential stage of Caco-2 cells did not affect the production of acetate and formate in the coculture of B . bifidum WU 12 and Caco-2 monolayers (Fig. 10 B). These results suggest that the acetate-induced TER-enhancing effect depends on the differentiation stage of the intestinal epithelial cells. Table 8 The TER value measured at the characterization of B . bifidum -induced strengthening epithelial barrier function at different timepoints during Caco-2 monolayer polarization Period Sample 0 h 24 h coincubation TER (\\u03a9 \\u00d7 cm 2 ) \\u00b1SD TER (\\u03a9 \\u00d7 cm 2 ) \\u00b1SD Day7 Blank 306.6 86.0 229.0 72.0 B . bifidum WU12 307.5 94.3 224.1 94.8 Day 14 Blank 499.7 7.1 585.4 23.3 B . bifidum WU12 516.9 6.2 794.5 38.8 Day 20 Blank 816.5 50.9 830.0 41.7 B . bifidum WU12 796.1 35.6 1043.5 28.6 Figure 10 Characterization of B . bifidum -induced strengthening epithelial barrier function at different time points during Caco-2 monolayer polarization. (A) B . bifidum -induced TER enhancement: Day7, nonpolarization (actual TER values, 307 \\u00b1 77 \\u03a9 cm 2 ); Day14, early-polarization (actual TER values, 508 \\u00b1 19 \\u03a9 cm 2 ); Day20, stable-polarization (actual TER values, 806 \\u00b1 49 \\u03a9 cm 2 ). TER changes were calculated from the TER value measured before and 24 h after the coincubation with B . bifidum WU12. Data represent the average of three independent experiments carried out in triplicate. (B) Short chain fatty acids production in nonpolarized (Day 7) or stable-polarized Caco-2 cells (Day 20) treated with B . bifidum WU12. Experiments were carried out in triplicate, and data represent the means \\u00b1 SD. Statistical differences were calculated by t -test (** P < 0.01, * P < 0.05).\",\n",
            "        \"discussion\": \"Discussion Dysregulation of intestinal permeability allows paracellular permeation of luminal antigens that initiate or promote intestinal inflammation. Thus, a defective intestinal TJ barrier has been proposed as an etiological factor for various gastrointestinal and systemic diseases with chronic inflammation, including allergy, celiac disease, Crohn's disease, diabetes, multiple sclerosis, and irritable bowel syndrome (Arrieta et al. 2006 ; Groschwitz and Hogan 2009 ). Recent studies demonstrated that various dietary components or probiotics regulate epithelial permeability by modifying intracellular signal transduction involved in the expression and localization of TJ proteins (Ulluwishewa et al. 2011 ). Administration of probiotics capable of colonizing the intestine is expected to induce long-term beneficial effects on intestinal health, whereas orally delivered dietary components have only a transient effect. In this study, we investigated the regulation of epithelial TJ barrier function by human fecal bacterial strains from the three bacterial genera (i.e., Bifidobacterium , Enterococcus, and Lactobacillus ) most commonly used as probiotics for human consumption. This study demonstrated that some Bifidobacterium strains protect the epithelial TJ barrier against TNF- \\u03b1 -induced injury and promote the restoration of TNF- \\u03b1 -induced loss of epithelial barrier integrity. Furthermore, we showed that B . bifidum -induced restoration of epithelial TJ barrier may be attributed to increased production of acetate and formate, as demonstrated in cocultures of B . bifidum and Caco-2 cells. Our present data showed that the permeability of Caco-2 epithelial cell monolayers exposed to live bacterial cells was altered depending on the bacterial strain. Many strains of Bifidobacterium species and some strains of Lactobacillus species significantly increased the TER of the Caco-2 cell monolayers, whereas all strains of Enterococcus species tested in this study dramatically decreased the TER. Therefore, in this study, we did not evaluate whether Enterococcus species have the capacity for the protection or restoration of the epithelial TJ barrier. However, these results do not mean that all of Enterococcus species are incapable of strengthening epithelial TJ function. Compared with probiotic strains belonging to the genera Bifidobacterium and Lactobacillus , Enterococcus species are used as probiotics to a much lesser extent because some strains possess virulence factors (including adhesins, invasins, pili, and hemolysin) and are resistant to various antibiotics (Franz et al. 2011 ). Steck et al. showed that a strain of the gut commensal E . faecalis compromises the epithelial barrier by metalloprotease-triggered degradation of the extracellular domain of E-cadherin (Steck et al. 2011 ). In contrast, some probiotic strains of E . faecium and E . faecalis are produced in the form of pharmaceutical preparations for treatment of diarrhea, immune modulation and lowering of serum cholesterol. Furthermore, Miyauchi et al. showed that pretreatment of heat-killed Enterococcus hirae prevented TNF- \\u03b1 -induced barrier impairment by modulating intracellular signaling pathways, suggesting that its cell wall components lead to the enhancement of the epithelial TJ barrier (Miyauchi et al. 2008 ). However, in this study, the heat-killed B . bifidum WU12 lost the TJ repair capacity completely. This suggests that the main inducing factor of this beneficial effect might be the metabolites of bacteria, not the cell wall components. We found that different species of Bifidobacterium and Lactobacillus differentially attenuated TNF- \\u03b1 -induced reduction in TER of Caco-2 cell monolayers. Similarly, previous studies have shown that pretreatment with some commensal and probiotic bacteria can inhibit the increase in the epithelial TJ permeability caused by infection, proinflammatory cytokines, and stress (Resta-Lenert and Barrett 2003 ; Donato et al. 2010 ; Miyauchi et al. 2012 ). Molecular mechanisms underlying the protective effect are only partially understood. It is well established that treatment of intestinal epithelial cells with TNF- \\u03b1 promotes the redistribution of several TJ proteins (including claudin-1, occludin, and ZO-1) and impairs epithelial barrier function (Groschwitz and Hogan 2009 ). In particular, myosin light chain kinase (MLCK)-mediated phosphorylation of the myosin light chain (MLC), which induces the contraction of perijunctional actin\\u2013myosin filaments and opening of the epithelial TJ structure, appears to be a central molecular mechanism of TNF- \\u03b1 -induced loss of epithelial barrier function (Turner et al. 1997 ; Zolotarevsky et al. 2002 ; Ma et al. 2005 ; Shen et al. 2006 ). Previous studies showed that certain probiotic and commensal bacteria (i.e., Enterococcus hirae and Lactobacillus rhamnosus ) suppressed the TNF- \\u03b1 -induced decrease in TER of Caco-2 cell monolayers by decreasing MLCK expression (Miyauchi et al. 2008 , 2009 ). Furthermore, Ye et al. revealed that the TNF- \\u03b1 -induced increase in MLCK expression was mediated through activation of the nuclear transcription factor NF- \\u03ba B, which is considered as the master regulator of inflammatory responses (Ye et al. 2006 ). Thus, the suppression of TNF- \\u03b1 -induced NF- \\u03ba B activation, rather than modulation of TJ proteins, is thought to be a pivotal process for prevention of TNF- \\u03b1 -induced barrier impairment by commensal and probiotic bacteria. In other words, beneficial bacteria that confer preventive effects against TNF- \\u03b1 -induced TJ disruption could also exert antiinflammatory effects on intestinal epithelial cells by inhibiting NF- \\u03ba B activation. There are only a limited number of studies focusing on the therapeutic effects of probiotics on impaired epithelial barrier function, as compared with multiple reports regarding preventive or protective effects of probiotics on TJ integrity against harmful stimuli (e.g., cytokines, infection, and H 2 O 2 ). In a previous report, the probiotic E . coli Nissle 1917 led to restoration of a disrupted epithelial barrier of T84 cell monolayer caused by pathogenic E . coli infection, which was associated with increased expression and relocalization of zonula occludens-2 (ZO-2) (Zyrek et al. 2007 ). Our study demonstrated that Bifidobacterium species promoted the restoration of the epithelial TJ barrier of Caco-2 cell monolayer, in which TJ barrier integrity was impaired by pretreatment of TNF- \\u03b1 . Notably; B . bifidum WU12 exerted the highest TER-restorative effect on Caco-2 cell monolayers among the tested Bifidobacterium strains. Interestingly, B . bifidum -induced increase in TER was accompanied by an increase in mRNA expression of occludin. Balda et al. showed that the overexpression of occludin was linked to an increase in TER (Balda et al. 1996 ). From these findings, increased occludin gene expression may contribute to the ability of B . bifidum to restore the epithelial TJ barrier. In contrast, B . bifidum did not alter the mRNA expression of other TJ-related proteins (i.e., claudin-1 and ZO-1). In previous studies, the treatment of epithelial monolayers with probiotics (specifically L . rhamnosus GG, L . plantarum, and L . salivarius ) enhanced protein expression of the occludin-associated plaque proteins (ZO-1, ZO-2, and cingulin) and other TJ-related proteins (claudin-1, claudin-3, and JAM-1) by IECs (Anderson et al. 2010 ; Miyauchi et al. 2012 ; Patel et al. 2012 ). Thus, epithelial barrier integrity may be further enhanced by administration of a mixture of multiple probiotic species each capable of modulating specific TJ proteins. Furthermore, the paracellular permeability of IECs are affected by expression levels and localization of TJ proteins in the paracellular space. Further study assaying the expression level and localization of TJ proteins during the B . bifidum -induced TJ restoration will be carried out to elucidate the cellular mechanisms. Live B . bifidum had a restorative effect on the TER in TJ-impaired Caco-2 cell monolayers, whereas this effect was diminished when treated with heat-killed bacteria. It was previously reported that metabolites from B . infantis or B . lactis prevented epithelial barrier dysfunction caused by proinflammatory cytokines or pathogen infection (Ewaschuk et al. 2008 ; Putaala et al. 2008 ). Thus, we speculate that some of B . bifidum -derived metabolites may also regulate the epithelial TJ. In this study, NMR-based metabonomics analysis of the coculture supernatant revealed that SCFAs (acetate and formate) are responsible for the TER-restorative effect induced by live B . bifidum . Interestingly, production of acetate and formate was upregulated in the coculture environment compared with the monoculture of Bifidobacterium strain or Caco-2 monolayers, indicating that cross talk between the host and microbes may alter their metabolic activity. Further studies are needed to clarify which cells enhanced the production of acetate or formate by coculture. Furthermore, we observed that production levels of acetate and formate in the coculture varied depending on the Bifidobacterium strain. There are previous reports showing the beneficial effect of SCFAs on TER and paracellular permeation in Caco-2 cell monolayers (Mariadason et al. 1997 ; Malago et al. 2003 ; Suzuki et al. 2008 ; Elamin et al. 2013 ). Notably, Suzuki et al. presented the finding that the incubation of Caco-2 cell monolayers with acetate (>40 mmol/L) led to a significant increase in the epithelial TJ barrier function. They suggested that acetate-induced TJ enforcement may be mediated by intracellular signal transduction through GPR43, which has been defined as one of the cell surface G-protein coupled receptors for SCFAs (Brown et al. 2003 ; Suzuki et al. 2008 ). However, the molecular mechanism underlying acetate-mediated fortification of intestinal epithelial barrier remains unclear. In contrast, our data suggest that B . bifidum -induced epithelial TJ enforcement was dependent on the differentiation stage of Caco-2 cell monolayers. In future experiments we will elucidate the molecular mechanism involved, including the investigation of the GPR43-acetate signaling pathway. Probiotic microorganisms (e.g., Bifidobacterium and Lactobacillus ) exert their immune-modulatory effect through interaction with Toll-like receptor 2 (TLR2), which recognizes cell wall components such as peptidoglycan, lipoteichoic acid, and lipoprotein (Mohamadzadeh et al. 2005 ; Galdeano and Perdigon 2006 ; Hoarau et al. 2006 ; Zeuthen et al. 2008 ). Furthermore, recent studies have shown that TLR2 stimulation plays a crucial role in maintaining intestinal epithelial barrier integrity. Stimulation with the synthetic TLR2 agonist PCSK efficiently preserves TJ-associated barrier integrity (Cario et al. 2004 , 2007 ). Cell wall components derived from some Bifidobacterium and Enterococcus strains are also reported to markedly enhance epithelial TJ barrier through a TLR2-mediated mechanism (Miyauchi et al. 2008 ; Sultana et al. 2013 ). This study showed that heat-killed B . bifidum slightly increased TER of Caco-2 cell monolayers, whereas lipase and mutanolysin treatment to digest lipid-related components abolished the rise in TER elicited by B . bifidum . These results suggest that cross talk between B . bifidum and TLR2 plays a partial role in the modulation of intestinal epithelial TJ barrier function. In summary, we demonstrated that most of Bifidobacterium strains have the capacity to prevent TNF- \\u03b1 -induced disruption of intestinal epithelial barrier and to promote epithelial TJ integrity. Furthermore, it was found that the upregulation of the production of SCFAs (acetate and formate) leads to the restoration of the epithelial TJ barrier. Our findings show that TJ-strengthening Bifidobacterium strains could make an important contribution to the prevention and treatment of various digestive system disorders associated with intestinal epithelial barrier dysfunction (including diabetes, inflammatory bowel disease, and nonalcoholic fatty liver disease). However, studies using animal models are still required to evaluate in vivo the beneficial effects seen here in vitro.\",\n",
            "        \"conclusion\": null\n",
            "    },\n",
            "    {\n",
            "        \"pmcid\": \"PMC5040013\",\n",
            "        \"abstract\": null,\n",
            "        \"intro\": \"I ntroduction Dysferlinopathy is a group of autosomal recessive muscular dystrophies caused by mutations in dysferlin( DYSF ) gene showing marked clinical heterogeneity.[ 1 2 3 4 5 6 ] The most common phenotypes of dysferlinopathy are proximal limb weakness (limb girdle muscular dystrophy type 2B [LGMD2B])[ 1 ] and distal myopathy (Miyoshi myopathy [MM]).[ 2 ] However, other atypical symptoms such as hyperCKemia,[ 7 ] distal anterior compartment myopathy,[ 8 ] and proximodistal myopathy (PDM)[ 3 ] are not rare and can have a congenital onset.[ 9 ] Clinical variability is also observed within a single family. Western blot analysis and immunohistochemistry are important tools in the initial diagnosis of primary dysferlinopathy due to their low cost and convenience in clinical practice.[ 10 ] However, severe reduction of DYSF can also be observed in other skeletal muscle diseases, such as calpainopathy, caveolinopathy, and anoctaminopathy, which are classified as secondary dysferlinopathies.[ 11 ] In addition, false-negative results can occur in western blot analysis when DYSF has accumulated in the cytoplasm.[ 12 ] Therefore, gene analysis is necessary and still remains the \\u201cgold standard\\u201d for diagnosis.[ 13 ] The DYSF gene is located on chromosome 2p13, which spans a genomic region of more than 230 kbp and comprises 55 exons.[ 1 2 ] It encodes a transmembrane protein DYSF which has been linked to membrane repair,[ 14 ] Ca 2+ signaling,[ 15 ] cell adhesion,[ 16 ] and angiogenesis.[ 17 ] To date, 510 different mutations in this gene have been reported in the Leiden muscular dystrophy database worldwide (Leiden Muscular Dystrophy pages \\u00a9 www.dmd.nl ). Most of these mutations are private and there are no hotspots,[ 18 ] which makes screening of the entire coding sequence of the DYSF gene necessary. In addition, most of the reported mutations are point mutations, small deletion/insertions, and intronic mutations.[ 18 ] Exonic rearrangements had been reported on rare occasions and were identified as the second disease-causing mutation in 5 of 12 patients by multiplex ligation-dependent probe amplification (MLPA).[ 19 ] Given the high frequency of patients with only one pathogenic mutation (the proportion varied from 12.5% to 34.0% in previous studies),[ 4 5 18 ] it is necessary to carry out MLPA testing in these patients as a supplementary tool in the routine screening for DYSF gene mutations. To date, no more than 60 Chinese dysferlinopathy patients with genetic diagnoses had been reported.[ 4 20 21 22 23 ] To better characterize the genetic spectrum of Chinese patients with dysferlinopathy, we described the genetic and clinical findings in the largest cohort of Chinese dysferlinopathy patients. In addition, we performed MLPA assay of DYSF gene in patients with only one pathogenic mutation to confirm the existence of exonic rearrangements in Chinese patients.\",\n",
            "        \"method\": \"M ethods Patient selection criteria and clinical evaluation Eighty-seven patients were included in this study based on the clinical suspicion of primary dysferlinopathy and absent/severely reduced dysferlin expression as evidenced by immunohistochemical analyses of muscle specimen. Written informed consent was obtained from all participants according to the Declaration of Helsinki . All these patients underwent muscle biopsy at Department of Neurology, Peking University First Hospital after providing written informed consent. Immunohistochemical analyses were performed using primary antibodies for DYSF, sarcoglycans, and dystrophin (all from Novocastra Laboratories, Newcastle, UK). Two patients were included with a clinical suspicion of dysferlinopathy without a muscle biopsy. All patients underwent detailed neurological interviews and physical examinations by experienced neurologists (Zhao-Xia Wang, Wei Zhang, or Yun Yuan) at Department of Neurology, Peking University First Hospital. Patients were classified into different phenotypes according to their initial pattern of muscle involvement: hyperCKemia when there are no clinical symptoms, LGMD2B when the proximal leg is first involved, MM when the distal part of leg is first involved, and proximodistal phenotype when there is proximal and distal weakness simultaneously at disease onset. Functional status was evaluated with a modified 0\\u20139 grading system proposed by Gardner-Medwin and Walton (GM-W scale).[ 24 ] Mutation analysis Genomic DNA was extracted from peripheral blood cells or skeletal muscle specimens of the patients. In 41 patients, all 55 exons and the intron/exon boundary of the DYSF gene were amplified by PCR as previously described.[ 25 ] The PCR products were directly sequenced using an ABI 3730XL automatic sequencing machine (Applied Biosystems, Life Technologies, Carlsbad, CA, USA). The sequences were read by Chromas software ( http://technelsium.com.au/wp/chromas ) and compared to the human DYSF sequence (NM_003494.3). In 48 patients, next generation sequencing (NGS) was applied with a neuromuscular disease panel (Agilent, Santa Clara, CA, USA) of 420 genes known to be associated with inherited muscular diseases. The exons and 10 bp of flanking splice sites were captured and subsequently sequenced on an Illumina HiSeq 2500 Sequencer (Illumina, San Diego, CA, USA). The reads were aligned by SOAPaligner for single-nucleotide polymorphism calling and other analyses. The sequencing files were mapped to reference sequences with Burrows-Wheeler Aligner and Picard tools, and then called with control samples with the GATK 3.0 HaplotypeCaller (Broad Institute, USA). Sanger sequencing with specific primers was conducted to confirm the mutations detected by NGS. In patients in whom only one mutation was detected, we further performed MLPA assay using a commercially available MLPA kit (SALSA MLPA probemix P268-A2 DYSF; MRC-Holland BV, Amsterdam, The Netherlands) which covered 40 of the 55 exons. Interpretation of mutations found in this study The mutations found in patients were determined to be disease-causing by the following criteria: (1) mutations reported in literature, in the HGMD database, Leiden Muscular Dystrophy pages database ( www.dmd.nl ), or the UMD-DYSF mutations database ( www.umd.be/DYSF ); (2) novel null mutations, including nonsense mutations, frameshift mutations, canonical \\u00b11 or 2 splice sites, and single exon or multiexon deletions; (3) novel missense mutations predicted to be disease-causing by a combination of four predictive software programs, including UMD-predictor (predicted as pathogenic/probably pathogenic),[ 26 ] Mutation Taster (predicted as disease-causing),[ 27 ] PolyPhen-2 (predicted as probably/possibly damaging),[ 28 ] and SIFT software (J. Craig Venter Institute, USA) (predicted as deleterious); (4) novel intronic mutations predicted as disease-causing by MutationTaster (predicted as disease-causing)[ 27 ] and Human Splicing Finder ( http://www.umd.be/HSF3 ).[ 29 ] Statistical analysis All values were calculated using SPSS version 13.0 (SPSS Inc., Chicago, IL, USA). Values are presented as the mean \\u00b1 standard error (SE) unless otherwise stated. Mann-Whitney U -test was used to test the significance of differences in the GM-W scale between different types of genetic mutations. Student's t -test was used to test the significance of differences in age of onset, disease duration, and serum creatine kinase (CK) level between the different types of genetic mutations. The difference in the clinical phenotypes between the two groups was analyzed by Chi-square test. A value of P \\u2264 0.05 was considered statistically significant (two-tailed).\",\n",
            "        \"subjects\": null,\n",
            "        \"results\": \"R esults Geographic and clinical data The patients in this study came from 27 provinces of mainland China, including 51 men and 38 women. A total of 87 patients were of Han ethnicity, one patient was of Hui ethnicity, and one patient was of Uygur ethnicity. Sixteen patients had a family history of muscle diseases. The mean age of onset was 21.1 \\u00b1 7.3 years (range 10\\u201349 years). The mean disease duration was 7.4 \\u00b1 5.7 years (range 1 month to 25 years). Forty-five patients presented with LGMD2B, 31 with MM, 7 with PDM, and 6 were clinically asymptomatic and diagnosed with hyperCKemia. The median GM-W scale score was 4, ranging from 0 to 9. Eight patients were wheelchair-dependent. Serum CK ratio (defined as CK level/upper limit of normal range) varied widely, ranging from 10 to 187. Myopathic changes were found in 58 of the 63 patients who underwent examination by electromyography (EMG), and neurogenic changes were found in one patient. The remaining four patients showed normal EMG results. The detailed clinical data are provided in Supplementary Table 1 . Supplementary Table 1 Clinical information of patients in this study Patient number Gender/age at diagnosis (years) Age of onset (years) Family history Phenotype GM-W scales at diagnosis CK (\\u00d7 N ) Dysferlin ICH Calf atrophy EMG 1 Female/34 33 MM 2 12 - NA 2 Female/30 26 LGMD2B 3 42 - Normal 3 Male/13 13 HyperCKemia 0 20 - Myopathic 4 Male/16 14 LGMD2B 1 118 - Myopathic 5 Female/28 14 LGMD2B 4 65 - + Myopathic 6 Male/21 15 + MM 3 106 - + NA 7 Female/32 12 + LGMD2B 8 157 - + NA 8 Male/35 20 LGMD2B 9 NA - + Myopathic 9 Male/37 18 MM 5 29 - + Myopathic 10 Female/49 41 LGMD2B 7 15 - + Myopathic 11 Male/24 18 MM 4 52 - + Myopathic 12 Female/28 20 MM 4 27 Reduced + Myopathic 13 Male/18 15 HyperCKemia 1 29 - NA 14 Male/16 14 LGMD2B 4 71 - Myopathic 15 Female/25 24 LGMD2B 3 64 - Myopathic 16 Female/18 13 LGMD2B 4 47 - Myopathic 17 Male/22 16 MM 4 59 - + Myopathic 18 Female/36 24 LGMD2B 7 24 - + Myopathic 19 Male/25 22 LGMD2B 4 40 - Myopathic 20 Male/28 20 LGMD2B 4 43 - + Myopathic 21 Female/36 29 PDM 4 15 - + NA 22 Female/33 30 + LGMD2B 4 32 - NA 23 Female/28 25 LGMD2B 5 23 - + Myopathic 24 Male/26 25 PDM 4 71 - + Myopathic 25 Male/25 16 + MM 4 34 - + Myopathic 26 Male/25 12 LGMD2B 5 NA Reduced + NA 27 Female/19 17 MM 2 80 - + Myopathic 28 Male/15 15 HyperCKemia 1 187 Reduced NA 29 Male/21 14 MM 2 71 - + Myopathic 30 Male/35 31 MM 4 34 - + Myopathic 31 Female/51 49 LGMD2B 3 11 - + Myopathic 32 Female/49 24 LGMD2B 7 10 - + Myopathic 33 Male/22 22 PDM 1 122 - + NA 34 Male/26 23 MM 8 16 - Myopathic 35 Male/45 31 + LGMD2B 5 NA - + NA 36 Male/36 25 LGMD2B 5 38 - + Myopathic 37 Male/23 21 + PDM 2 73 - + Myopathic 38 Male/34 29 MM 8 20 - + Myopathic 39 Female/35 29 LGMD2B 5 33 - Myopathic 40 Female/37 28 LGMD2B 7 17 Reduced + Myopathic 41 Male/22 20 LGMD2B 2 13 - Myopathic 42 Female/25 14 + MM 4 57 - + Myopathic 43 Female/32 26 LGMD2B 2 31 - + Myopathic 44 Female/38 28 MM 5 21 - + Myopathic 45 Male/24 19 MM 4 40 - + Myopathic 46 Female/25 22 MM 1 21 - + Myopathic 47 Male/24 22 LGMD2B 3 30 - Myopathic 48 Male/15 15 LGMD2B 1 31 - NA 49 Male/25 18 MM 3 51 Reduced + Myopathic 50 Male/17 16 + MM 1 116 - NA 51 Male/16 15 PDM 1 120 - + Normal 52 Male/16 10 + LGMD2B 2 118 - + Myopathic 53 Male/36 14 MM 5 29 - + Myopathic 54 Male/42 32 MM NA 24 \\u2013 + Myopathic 55 Male/27 16 MM 5 37 \\u2013 + NA 56 Male/26 22 LGMD2B 2 26 \\u2013 + Myopathic 57 Female/25 23 LGMD2B NA 22 \\u2013 Myopathic 58 Female/43 15 LGMD2B 8 10 \\u2013 + Myopathic 59 Male/27 16 MM 5 76 \\u2013 + Myopathic 60 Male/19 19 HyperCKemia 0 122 \\u2013 Normal 61 Male/33 25 + MM 2 22 \\u2013 + NA 62 Male/25 16 MM 1 38 \\u2013 + Myopathic 63 Female/31 18 LGMD2B 4 NA \\u2013 + NA 64 Female/28 14 + PDM 5 NA \\u2013 + NA 65 Male/41 39 LGMD2B 5 29 \\u2013 + Myopathic 66 Male/24 14 MM 2 36 \\u2013 + Myopathic 67 Female/26 23 LGMD2B 5 20 \\u2013 + Myopathic 68 Male/36 30 + LGMD2B 4 71 \\u2013 + Myopathic 69 Male/26 17 LGMD2B 8 NA NA + NA 70 Male/22 14 LGMD2B 4 55 NA + NA 71 Female/30 28 LGMD2B 5 NA \\u2013 Myopathic 72 Male/29 18 LGMD2B 5 35 \\u2013 + NA 73 Female/23 18 PDM 5 49 Reduced + NA 74 Female/32 29 LGMD2B 4 13 \\u2013 Neurogenic 75 Female/33 26 MM 5 16 \\u2013 + Myopathic 76 Male/32 26 MM 2 43 \\u2013 + Myopathic 77 Female/14 14 LGMD2B 1 93 \\u2013 NA 78 Female/28 15 + MM 4 NA \\u2013 + NA 79 Male/13 13 HyperCKemia 1 47 \\u2013 NA 80 Male/45 42 LGMD2B 2 38 \\u2013 Myopathic 81 Male/30 23 MM 4 79 Reduced + Myopathic 82 Female/30 19 LGMD2B 4 24 \\u2013 Myopathic 83 Female/36 26 LGMD2B 4 17 Reduced + NA 84 Female/18 16 + HyperCKemia 0 48 \\u2013 Normal 85 Female/23 15 + LGMD2B 8 35 \\u2013 + Myopathic 86 Female/37 15 MM 8 18 \\u2013 + Myopathic 87 Female/26 21 MM 4 66 Reduced + Myopathic 88 Male/26 15 + LGMD2B 4 44 \\u2013 + NA 89 Male/31 23 LGMD2B NA 31 \\u2013 + NA NA: Not available; PDM: Proximodistal myopathy; MM: Miyoshi myopathy; LGMD2B: Limb girdle muscular dystrophy type 2B; GM-W: Gardner-Medwin and Walton; CK: Creatine kinase; ICH: Immunohistochemistry; EMG: Electromyography; + (Family history): patients with a family history of skeletal muscle diseases; \\u2013 (Dysferlin ICH): positive staining of dysferlin on muscle biopsy; + (Calf atrophy): patients with calf atrophy. Analysis of mutations identified in this study Among the 89 index patients, 79 were demonstrated to carry two disease-causing (73 cases) or possibly disease-causing mutations (six cases), including 26 patients with homozygous mutations and 53 patients with compound heterozygous mutations. In the remaining 10 patients, only one heterozygous mutation was found. Notably, among 13 patients who were initially found to carry only one pathogenic mutation by Sanger sequencing or NGS, three were further identified to carry exonic deletions with MLPA [ Figure 1 ]. In these patients, we identified 105 different mutations, including 98 disease-causing and seven possibly disease-causing mutations [ Supplementary Table 2 and Figure 2 ]. Fifty-nine novel mutations were found, 52 of which were identified as disease-causing [ Supplementary Table 2 ]. Seven novel missense mutations were determined as possibly disease-causing because of inconsistent results of different predicting softwares. Figure 1 Multiplex ligation-dependent probe amplification results of patients with exon deletions of dysferlin gene. Deletion of exon 48, exon 33, and exon 2 were identified in patient 22, 64, and 72, respectively (arrows). Supplementary Table 2 Dysferlin gene mutations found in this study Patient number Mutation name Protein change Exon/intron Domain State DNA sequencing Mutation type Patients with two disease-causing mutations 1 c.863A>T p.D288V 9 C2B Homozygous NGS Missense 2 c.1375dupA p.M459NfsX15 15 C2C Heterozygous NGS Frameshift c.3036G>C \\u2020 p.W1012C 29 InnerDysF-C Heterozygous Missense 3 c.2643+5G>C \\u2020 Abl.spl 25 InnerDysF-N Heterozygous NGS Splice site c.3827T>C \\u2020 p.L1276P 34 Heterozygous Missense 4 c.3059C>T \\u2020 p.P1020L 28 InnerDysF-C Heterozygous NGS Missense c.3442+1G>A \\u2020 Abl.spl IVS31 Heterozygous Splice site 5 c.5302C>T p.R1768W 47 Homozygous NGS Missense 6 c.1667T>C p.L556P 19 Heterozygous Sanger Missense c.3988C>T p.Q1330X 37 Heterozygous Nonsense 7 c.4894G>T p.E1632X 45 C2F Homozygous NGS Nonsense 8 c.5414dupC \\u2020 p.R1806Tfs*22 48 Homozygous NGS Frameshift 9 c.1523-2A>G \\u2020 Abl.spl IVS17 Homozygous NGS Splice site 10 c.3115C>T \\u2020 p.R1039W 29 InnerDysF-C Heterozygous NGS Missense c.5245C>T \\u2020 p.R1749C 47 Heterozygous Missense c.5525G>A p.G1842D 49 C2G Heterozygous Missense 11 c.1375dupA p.M459Nfs*15 15 C2C Homozygous NGS Frameshift 12 c.895G>C p.G299R 9 C2B Heterozygous Sanger Missense c.4756C>T p.R1586X 43 C2F Heterozygous Nonsense 13 c.1874A>T \\u2020 p.D625V 20 Heterozygous NGS Missense c.2762C>T \\u2020 p.S921L 26 OuterDysF-N Heterozygous Missense c.3785delG \\u2020 p.G1263Afs*82 34 Heterozygous Frameshift 14 c.1535_1553del \\u2020 p.F514Pfs*107 18 Heterozygous Sanger Frameshift c.4167+1G>A Abl.spl IVS38 C2E Heterozygous Splice site 15 c.4988_4989delTC \\u2020 p.V1663Gfs*47 45 C2F Homozygous NGS Frameshift 16 c.1523-2A>G \\u2020 Abl.spl IVS17 Heterozygous NGS Splice site c.2974T>C p.W992R 28 InnerDysF-N Heterozygous Missense 17 c.1180+5G>A Abl.spl IVS12 C2C Heterozygous NGS Splice site c.4194delC p.C1398fs 39 C2E Heterozygous Frameshift 18 c.3601C>T p.Q1200X 33 C2D Heterozygous NGS Nonsense c.4580T>G \\u2020 p.L1527R 42 Heterozygous Missense 19 c.1930+2T>G \\u2020 Abl.spl IVS20 Homozygous NGS Splice site 20 c.5884 C>T p.Q1962X 52 Homozygous NGS Nonsense 21 c.3531_3533delCAT \\u2020 p. 1178Idel 33 C2D Homozygous NGS In frame deletion c.5355G>A*, \\u2020 p.M1785I 48 Homozygous Missense 22 Exon 48 deletion 40 Heterozygous NGS + MLPA Exonic deletion c.4024C>G p.R1342G 38 C2E Heterozygous Missense 23 c.4022T>C p.L1341P 38 C2E Homozygous NGS Missense 24 c.265C>T p.R89X 4 Heterozygous NGS Nonsense c.5509G>A p.D1837N 49 C2G Heterozygous Missense 25 c.1284+2T>C Abl.spl IVS13 C2C Heterozygous NGS Splice site c.4756C>T p.R1586X 43 C2F Heterozygous Nonsense 26 c.144+1G>A \\u2020 Abl.spl IVS2 C2A Heterozygous NGS Splice site c.1393G>C \\u2020 p.D465H 15 C2C Heterozygous Missense 27 c.3258_3259delAG \\u2020 p.D1087Cfs*26 30 OuterDysF-C Homozygous NGS Frameshift 28 c.796_797delCT p.L266FfsX6 8 C2B Heterozygous NGS Frameshift c.1377_1379del p.R460del 15 C2C Heterozygous In frame deletion 29 c.4509+2_c.4509+ 6delTAAGG \\u2020 Abl.spl IVS41 Homozygous Sanger Splice site 30 c.313dupC p.L105PfsX43 4 Heterozygous NGS Frameshift c.5438T>C \\u2020 p.L1813P 49 C2G Heterozygous Missense 31 c.1180+5G>A Abl.spl IVS12 C2C Heterozygous Sanger Splice site c.3601C>T p.Q1201X 33 C2D Heterozygous Nonsense 32 c.863A>T p.D288V 9 C2B Heterozygous NGS Missense c.965T>C p.L322P 11 FerI Heterozygous Missense 33 c.2940delG p.L981FfsX76 28 InnerDysF-N Heterozygous NGS Frameshift c.4200dupC p.I1401HfsX8 39 C2E Heterozygous Frameshift 34 c.252delC \\u2020 p.K85Rfs*66 4 C2A Homozygous Sanger Frameshift 35 c.1992C>A \\u2020 p.Try664X 21 Heterozygous Sanger Nonsense c.2997G>T p.W999C 28 InnerDysF-N Heterozygous Missense 36 c.799_800delTT p.F267LfsX5 8 C2B Homozygous Sanger Frameshift 37 c.937+1G>A Abl.spl IVS10 Heterozygous Sanger Splice site c.3521-1G>T \\u2020 Abl.spl IVS32 C2D Heterozygous Splice site 38 c.2083delG \\u2020 p.A695Pfs*2 22 FerA Homozygous NGS Frameshift 39 c.176delT \\u2020 p.L59Rfs*92 3 C2A Heterozygous NGS Frameshift c.5975delT \\u2020 p.V1992Efs*20 53 Heterozygous Frameshift 40 c.4194delC p.C1398fs 39 C2E Heterozygous NGS Frameshift c.4886+2T>G \\u2020 Abl.spl IVS44 C2F Heterozygous Splice site 41 c.610C>T p.R204X 6 Heterozygous NGS Nonsense c.3516_3517delTT p.S1173X 32 C2D Heterozygous Frameshift 42 c.863A>T p.D288V 9 C2B Heterozygous Sanger Missense c.5077C>T p.R1693W 46 Heterozygous Missense 43 c.1667T>C p.L556P 19 Homozygous NGS Missense 44 c.1180+5G>A Abl.spl IVS12 C2C Homozygous Sanger Splice site 45 c.3516_3517delTT p.S1173X 32 C2D Homozygous Sanger Frameshift 46 c.5444G>T p.C1815F 49 C2G Homozygous NGS Missense 47 c.799_800delTT p.F267LfsX5 8 C2B Heterozygous Sanger Frameshift c.3181C>T p.Q1061RfsX59 30 Heterozygous Nonsense 48 c.1180+5G>A Abl.spl IVS12 C2C Heterozygous Sanger Splice site c.4941_4942del CT \\u2020 p.Y1648* 45 C2F Heterozygous Frameshift 49 c.5302C>T p.R1768W 47 Homozygous Sanger Missense 50 c.799_800delTT p.F267LfsX5 8 C2B Homozygous NGS Frameshift c.863A>T p.D288V 9 C2B Heterozygous Missense 51 c.2997G>A p.W999X 28 InnerDysF-N Heterozygous NGS Nonsense c.4411_4433del p.E1472Qfs*21 41 Heterozygous Frameshift 52 c.863A>T p.D288V 9 C2B Heterozygous Sanger Missense c.1667T>C p.L556P 19 Heterozygous Missense 53 c.1254delC p.F419Lfs*41 13 C2C Heterozygous NGS Frameshift c.1956G>A \\u2020 p.W652X 21 Heterozygous Nonsense 54 c.1375dupA p.M459NfsX15 15 C2C Heterozygous NGS Frameshift c.3137G>A p.R1046H 29 InnerDysF-C Heterozygous Missense 55 c.1375dupA p.M459NfsX15 15 C2C Heterozygous Sanger Frameshift c.1464delT p.G489EfsX4 16 Heterozygous Frameshift 56 c.3137G>A p.R1046H 29 InnerDysF-C Homozygous Sanger Missense 57 c.144+1G>A \\u2020 Abl.spl IVS2 C2A Heterozygous Sanger Splice site c.1375dupA p.M459NfsX15 15 C2C Heterozygous Frameshift 58 c.3988C>T p.Q1330X 37 Heterozygous Sanger Nonsense c.1180+5G>A Abl.spl IVS12 C2C Heterozygous Splice site 59 c.1284+1G>A \\u2020 Abl.spl IVS13 C2C Homozygous Sanger Splice site 60 c.937+1G>A Abl.spl IVS10 Heterozygous Sanger Splice site c.4756C>T p.R1586X 43 C2F Heterozygous Nonsense 61 c.680T>C \\u2020 p.I227T 7 C2B Heterozygous Sanger Missense c.3988C>T p.Q1330X 37 Heterozygous Nonsense 62 c.755C>T p.T252M 7 C2B Heterozygous Sanger Missense c.3789_3796del \\u2020 p.S1264Vfs*9 34 Heterozygous Frameshift 63 c.1708C>T \\u2020 p.Q570X 19 Heterozygous Sanger Nonsense c.4011delG \\u2020 p.A1338Hfs*7 38 C2E Heterozygous Frameshift 64 c.339delA \\u2020 p.A115Pfs*36 4 Heterozygous Sanger + MLPA Frameshift Exon 33 deletion 33 C2D Heterozygous Exonic deletion 65 c.1180+4delC \\u2020 Abl.spl IVS12 C2C Heterozygous Sanger Splice site c.1708C>T \\u2020 p.Q570X 19 Heterozygous Nonsense 66 c.1906C>T \\u2020 p.Q636X 20 Heterozygous NGS Nonsense c.2810+5G>A \\u2020 Abl.spl IVS20 Heterozygous Splice site 67 c.610C>T p.R204X 6 Heterozygous Sanger Nonsense c.1134_1166del p. 359A_368Adel 12 C2C Heterozygous In frame deletion 68 c.4989_4990insCGGT \\u2020 p.V1664Rfs*48 45 Homozygous Sanger Frameshift 69 c.265C>T p.R89X 4 Heterozygous NGS Nonsense c.6080G>A \\u2020 p.W2027X 54 Heterozygous Nonsense 70 c.792+1G>A \\u2020 Abl.spl IVS7 C2B Heterozygous NGS Splice site c 965T>C p.L322P 11 FerI Heterozygous Missense 71 c.5740G>A \\u2020 p.D1914N 51 C2G Heterozygous Sanger + MLPA Missense 72 c.567delA \\u2020 p.P190Lfs*37 6 Heterozygous NGS + MLPA Frameshift Exon 2 deletion 3 C2A Heterozygous Exonic deletion 73 c.1165G>A p.E389K 12 C2C Heterozygous Sanger Missense c.2997G>T p.W999C 28 InnerDysF-N Heterozygous Missense Patients with one disease-causing mutation 74 c.2997G>T p.W999C 28 InnerDysF-N Heterozygous NGS Missense c.5639C>G*, \\u2020 p.A1880G 50 C2G Heterozygous Missense 75 c.3702T>G \\u2020 p.Y1234X 33 C2D Heterozygous NGS Nonsense c.5511C>A*, \\u2020 p.D1837E 49 C2G Heterozygous Missense c.5516A>T*, \\u2020 p.Y1839F 49 C2G Heterozygous Missense 76 c.3032-3C>G \\u2020 Abl.spl IVS28 InnerDysF-N Heterozygous NGS Splice site c.5639C>G \\u2020 p.A1880G 50 C2G Heterozygous Missense 77 c.1375dupA p.M459NfsX15 15 C2C Heterozygous NGS Frameshift c.5197A>G*, \\u2020 p.I1733V 46 Heterozygous Missense 78 c.4497delT p.F1499LfsX4 41 Heterozygous Sanger Frameshift c.5216C>A*, \\u2020 p.P1739Q 41 Heterozygous Missense 79 c.5792G>C*, \\u2020 p.R1931P 52 Heterozygous NGS Missense c.5511C>A*, \\u2020 p.D1837E 49 C2G Heterozygous Missense c.5516A>T*, \\u2020 p.Y1839F 49 C2G Heterozygous Missense 80 c.3112C>T p.R1038X 29 InnerDysF-C Heterozygous NGS + MLPA Nonsense 81 c.4513T>A \\u2020 p.Y1505N 42 Heterozygous Sanger + MLPA Missense 82 c.1464delT p.G489EfsX4 16 Heterozygous Sanger + MLPA Frameshift 83 c.610C>T p.R204X 6 Heterozygous Sanger + MLPA Nonsense 84 c.567delA \\u2020 p.P190Lfs*37 6 Heterozygous Sanger + MLPA Frameshift 85 c.1464delT p.G489EfsX4 16 Heterozygous Sanger + MLPA Frameshift 86 c.3725G>A p.R1241H 34 C2D Heterozygous Sanger + MLPA Missense 87 c.610C>T p.R204X 6 Heterozygous Sanger + MLPA Nonsense 88 c.5803C>A \\u2020 p.P1935T 52 Heterozygous Sanger + MLPA Missense 89 c.4063_4064insT p.P1355SfsX27 38 C2E Heterozygous Sanger + MLPA Frameshift *Mutations with undetermined pathogenicity; \\u2020 Novel mutations. NGS: Next generation sequencing; Sanger: Sanger sequencing; MLPA: Multiplex ligation-dependent probe amplification. Figure 2 Positioning of dysferlin gene mutations identified in this study along the dysferlin protein sequence. Different domains are indicated by rectangles. Vertical lines above the protein indicate homozygous mutations while the vertical lines below the protein denote the heterozygous mutations. The allele frequencies of disease-causing mutations found in this study were as follows [ Table 1 ]: missense mutations (30.06%), nonsense mutations (17.18%), frameshift mutations (30.67%), in-frame deletions (2.45%), intronic mutations (17.79%), and exonic rearrangement (1.84%). These mutations span the whole length of the DYSF gene. However, the C2B and C2C domain demonstrated the highest frequency of mutations in this study [ Table 2 and Figure 2 ]. Table 1 Classification of disease-causing mutations of dysferlin gene in this study Mutation types Mutations, n Allele frequency (%) Point mutations 77 47.24 Missense 49 30.06 Nonsense 28 17.18 Deletion/insertion 54 33.13 Frameshift 50 30.67 In-frame 4 2.45 Intronic mutation 29 17.79 Exonic rearrangement 3 1.84 Total 163 100.00 Table 2 Distribution of dysferlin gene mutations in this study Domains Proportion of mutations in this domain (%) C2 domain A 3.70 C2 domain B 9.88 Ferlin family domain FerI 1.23 C2 Domain C 13.58 Ferlin family domain FerA 1.23 Ferlin family domain FerB 0.00 Outer DysF domain, N.terminal 0.62 Inner DysF domain, N.terminal 4.94 Inner DysF domain, C.terminal 4.32 Outer DysF domain, C.terminal 1.23 C2 domain D 6.17 C2 domain E 5.56 C2 domain F 5.56 C2 domain G 3.70 Transmembrane domain 0.00 Total 61.72 Ten recurrent mutations, which were found in more than three unrelated patients, are listed in Table 3 . Four of these mutations (c.863A>T, c.1375dupA, c.1667T>C, and c.3988C>T) have only been reported in the Chinese population. Table 3 Recurrent mutations of dysferlin gene identified in this study Mutations Protein changes Domains Patient number c.610C>T p.R204X 41, 67, 83, 87 c.799_800delTT p.F267LfsX5 C2B 36, 47, 50 c.863A>T p.D288V C2B 1, 32, 42, 50, 52 c.1180+5G>A Abl.spl C2C 17, 31, 44, 48, 58 c.1375dupA p.M459NfsX15 C2C 2, 11, 54, 55, 57, 77 c.1464delT p.G489EfsX4 55,82,85 c.1667T>C p.L556P 6, 43, 52 c.2997G>T p.W999C InnerDysF-N 35, 73, 74 c.3988C>T p.Q1330X 6, 58, 61 c.4756C>T p.R1586X C2F 12, 25, 60 Genotype-phenotype correlation We divided the patients in this study into two groups: (1) patients with at least one missense mutation or in- frame deletion/insertion and (2) patients with no missense mutation or in- frame deletion/insertion. No statistically significant difference was detected between these two groups regarding age of onset, disease duration, phenotype, GM-W scale, and serum CK level ( P >0.05).\",\n",
            "        \"discussion\": \"D iscussion The patients enrolled in this study came from 27 of the 34 provinces of China, which enabled us to better characterize the genetic spectrum of patients with dysferlinopathy in mainland China. As previously reported, the mutations found in this study span the whole length of the DYSF gene, and no mutational hot spots were identified. However, we found that DYSF mutations in Chinese patients clustered in the N-terminal region of the gene, especially in and around the C2C and C2B domains. In contrast, previously reported DYSF mutations were distributed evenly along the DYSF gene.[ 18 ] N-terminal clustering was only observed in a group of South Korean patients with dysferlinopathy.[ 5 ] Interestingly, four of the ten recurrent mutations found in this study (c.799_800delTT, c.863A>T, c.1180+5G>A, and c.1375dupA) were located in the C2B and C2C domains. The N-terminal clustering of DYSF mutations in Chinese patients was partly attributable to these recurrent mutations. All patients with the c.1375dupA mutation originated from northern China, indicating a potential founder effect of this mutation. The dysferlin protein consists of seven C2 domains (C2A\\u2013C2G) [ Figure 2 ],[ 30 ] which are highly conserved and functions in calcium-dependent phospholipid binding. The affinity of calcium- and phospholipid-binding for each domain varies greatly. For example, the C2B domain was predicted to have no calcium-binding capacity.[ 30 ] In line with this hypothesis, quantitative study revealed that the C2B domain is one of the domains with lowest affinity for calcium-dependent membrane binding.[ 31 ] In addition, by constructing mini-dysferlin molecules, Azakir et al .[ 32 ] found that the deletions of the dysferlin C2B domains have no impact on the sarcolemmal localization of dysferlin and the membrane repair of injured muscle cell. However, in this study, the C2B domain was the second most frequently affected domain, which was partly attributed to the recurrent mutation c.863A>T found in Chinese patients. To date, a very few missense mutations affecting the calcium-binding residues have been identified. c.863A>T results in a substitution of aspartate 288 by valine, which was predicted to be a key calcium-binding residue in this domain.[ 30 ] The recurrence of the missense mutation c.863A>T implied the importance of the C2B domain for the function of dysferlin. The proportion of different types of mutations in Chinese patients with dysferlinopathy varied among previous studies.[ 4 21 23 ] We found that there were fewer missense mutations in the current study than previously reported,[ 18 ] which might be due in part to the exclusion of the possibly disease-causing missense mutations in this study. Interestingly, the missense mutations identified in this study were located mainly in the C2B domain and inner DysF domain. The c.863A>T was found in five of eight patients carrying missense mutations in the C2B domain. Of all nine patients carrying missense mutations in the Inner DysF domain, eight patients were carrying mutations disrupting the arginine/tryptophan (R/W) stacks.[ 33 ] In this study, we first determined the existence of exonic rearrangements in Chinese patients. Only a few reports have described exonic rearrangements in patients with dysferlinopathy.[ 10 19 34 ] Genomic deletions/duplications were found in five of 12 patients with one pathogenic mutation using the MLPA method.[ 10 ] In this study, the frequency of exonic rearrangements was three in 13 patients with one pathogenic mutation. At present, there is no information available on the percentage of defects in the DYSF gene caused by deletions/duplications of complete exons. In our cohort, the allele frequency was estimated to be 3/178. However, we did not perform the MLPA test in patients with compound heterozygous or homozygous mutations. In addition, the MLPA kit we used only covers 40 of the 55 exons in the DYSF gene, so the frequency might be higher than expected. Given the high frequency of exonic rearrangements in patients with one disease-causing mutation, further MLPA analysis in these patients is recommended. In addition, in patient 22, we first identified a single exonic deletion by NGS using a copy number variation (CNV) calling algorithm, which incorporates read-depth statistics, allele zygosity analysis, and breakpoints detection. CNV calls were further confirmed by MLPA assays, providing a conclusive molecular diagnosis that would not be possible by routine Sanger sequencing alone. In this study, we identified 59 novel mutations, 52 of which were determined to be disease-causing. However, confirmation of the seven novel missense mutations was impossible using a bioinformatic approach because of conflicting results among different software programs. The clinical diagnoses of dysferlinopathy in patients (Six patients in total: P21, 74, 75, 77, 78 and 79, Supplementary Table 1 and Supplementary Table 2 ) carrying these mutations were confirmed based on the typical history and pathological study (especially the immunohistochemistry staining of dysferlin). In five of these patients (P21, 74, 75, 77, and 79), mutations of other muscular dystrophies related genes were ruled out by NGS panel based on 420 different genes, and exonic rearrangements were ruled out by the CNV calling algorithm [ Supplementary Table 2 ]. Therefore, the pathogenicity of these mutations could not be ruled out. Patient 78 had a family history of dysferlinopathy, and negative dysferlin expression was confirmed in her and her affected brother. As the mutation c.5216C>A was also identified in the siblings by Sanger sequencing, it is quite possible that c.5216C>A was disease-causing. Further study at the mRNA level is needed to achieve definitive genetic diagnoses in these patients. The novel mutations identified in this study, accounting for about 10% of all mutations reported to date, greatly expand the genetic spectrum of dysferlinopathy. In conclusion, DYSF mutations in Chinese patients clustered in the N-terminal region of the DYSF gene. Exonic rearrangements were found in 23% of patients with only one pathogenic mutation identified by Sanger sequencing or NGS. Novel mutations found in this study greatly expand the mutational spectrum of dysferlinopathy. Supplementary information is linked to the online version of the paper on the Chinese Medical Journal website . Financial support and sponsorship Nil. Conflicts of interest There are no conflicts of interest.\",\n",
            "        \"conclusion\": null\n",
            "    },\n",
            "    {\n",
            "        \"pmcid\": \"PMC5425199\",\n",
            "        \"abstract\": \"Tafazzin (TAZ) is often aberrantly expressed in some cancers, including rectal cancer and thyroid neoplasms. However, the function of TAZ in cervical cancer cells remains unknown. This study aims to explore the expression and function of TAZ in cervical cancer cells. Here, we determined the expression of TAZ protein in normal cervical tissue (NC, n = 27), high-grade squamous intraepithelial lesions (HSIL, n = 26) and squamous cervical carcinoma (SCC, n = 41) by immunohistochemistry, the expression of TAZ protein gradually increased from NC to HSIL to SCC. TAZ was overexpressed or down-regulated in cervical cancer cells by stably transfecting a TAZ-expressing plasmid or a shRNA plasmid targeting TAZ.\",\n",
            "        \"intro\": \"Introduction Cervical cancer is a common cancer that contributes to cancer-related death in females[ 1 ]. Despite advances in detection and prevention, cervical cancer remains the third most frequently diagnosed female cancer worldwide. Approximately 80% of cervical cancer cases occur in developing countries, where approximately 270,000 women die from cervical cancer annually[ 2 , 3 ]. Although infection with high-risk human papillomaviruses (HPV) is intimately related to the development of cervical cancer, HPV is not sufficient for cervical carcinogenesis and tumor progression[ 4 \\u2013 6 ]. Researchers have reported that the aggressive and proliferative nature of human cervical cancer is related to various factors that activate oncogenes and inactivate tumor suppressor genes[ 7 \\u2013 12 ]. The tafazzin (TAZ) gene is located on the distal region of chromosome Xq28 and encodes the tafazzin protein, which has an amino acid sequence homologous to acyltransferases[ 13 ]. TAZ is a mitochondrial protein localized in the mitochondrial membrane and plays a critical role in the remodeling of cardiolipin, a major lipid in the mitochondrial membrane[ 14 ]. Studies have shown that TAZ mutations can cause Barth syndrome, a rare and fatal X-linked genetic disorder[ 15 ]. In recent years, overexpression of TAZ has been observed in several tumors, including colon cancer[ 16 ], rectal cancer[ 17 ] and thyroid neoplasms[ 18 ]. Additionally, abnormal TAZ expression combined with higher IL-6 expression was found to promote inflammatory responses, which are commonly considered a predisposition factor for cancer progression[ 19 ]. However, the function of TAZ in cervical carcinogenesis is still not fully understood. Here, we explored the function and mechanism of TAZ in cervical cancer. In the present study, TAZ protein expression was found to gradually increase in the progression of cervical carcinoma, as detected by IHC and Western blot. Furthermore, TAZ was verified to be able to promote cell growth both in vitro and in vivo and inhibit apoptosis in cervical cancer cells, providing preliminary evidence that TAZ contributes to cervical carcinogenesis.\",\n",
            "        \"method\": \"Materials and methods Human tissue samples and ethics statement A total of 27 normal cervical samples (NC), 26 high-grade squamous intraepithelial lesions (HSIL) and 41 squamous cervical cancer samples (SCC) were obtained from patients at the First Affiliated Hospital of Xi\\u2019an Jiaotong University Medical College from 2008 to 2014. No subject had received chemotherapy, immunotherapy or radiotherapy before specimen collection. Histological classifications and clinical staging were based on the International Federation of Gynecology and Obstetrics classification system. The study was approved by the Ethics Committee of the Medical College of Xi\\u2019an Jiaotong University, and written informed consent was obtained from all subjects before sample collection. Cell lines and cell culture Human cervical cancer cell lines (HeLa, SiHa, C33A, CaSki, HT-3) were purchased from the American Type Culture Collection (ATCC, Rockville, MD, USA) in 2007 and cultured at 37\\u00b0C with 5% CO 2 in our lab. The HeLa, SiHa and C33A cells were cultured in Dulbecco\\u2019s Modified Eagle\\u2019s Medium (DMEM, Sigma- Aldrich, USA. CaSki cells were cultured in RPMI1640 (Sigma-Aldrich, USA). HT-3 cells were cultured in McCoy\\u2019s 5A (Sigma-Aldrich, USA). All media was supplemented with 10% heat-inactivated fetal bovine serum (FBS, Invitrogen, Carlsbad, CA, USA). Immunostaining Using a routine immunohistochemistry protocol, the specimens were fixed in 10% buffered formalin and embedded in paraffin. Then, 4 \\u03bcm sections of the tissue samples were deparaffinized in xylene and rehydrated through descending concentrations of ethanol. Antigen retrieval was performed by heating in 10 mM citrate buffer (pH 6.0) for 2 minutes. The sections were then treated with 3% hydrogen peroxide to block endogenous peroxidases. After washing with phosphate-buffered saline (PBS) at room temperature, the sections were incubated overnight at 4\\u00b0C with a rabbit polyclonal antibody against human TAZ (1:100 dilution; ab93362; Epitomics, USA). The sections were incubated with horseradish peroxidase-conjugated secondary antibody for 30 minutes at room temperature, followed by 3,3\\u2019-diaminobenzidine development. After that, the sections were counterstained with hematoxylin. As a negative control, the primary antibody was replaced with PBS. All slides were examined under an Olympus-CX31 microscope (Olympus, Tokyo, Japan) by two separate researchers. The staining intensity was scored as follows: 0 (negative), 1 (weak), 2 (moderate), 3 (strong). According to the percentage of positively stained cells, the staining extent was scored as 0 (0%), 1 (1%\\u201325%), 2 (26%\\u201350%), 3 (51%\\u201375%) and 4 (76%\\u2013100%). The final immunoreactivity score (IRS) equaled the intensity score multiplied by the quantity score. The staining of TAZ was stratified into two categories according to the IRS: negative (0\\u20134) and positive (5\\u201312). For the immunocytochemistry experiments, cells were seeded on autoclaved cover slips and, after 48 hours, fixed with 4% paraformaldehyde for 30 minutes and then permeabilized with 0.2% Triton X-100 for 20 minutes at room temperature. The staining procedure was similar to the immunohistochemistry process described above. Western blotting Lysates from cells and fresh tissue were prepared on ice using lysis buffer (150 mM NaCl; 50 mM Tris-HCl, pH 7.4; 2 mM EDTA; 1% NP-40; and 0.1% SDS) containing a protease inhibitor cocktail (Complete Mini; Roche Diagnostics, Branchburg, NJ, USA). A BCA Protein Assay Reagent (Pierce, Rockford, IL, USA) was used to determine the protein concentrations. A total of 50 \\u03bcg of protein samples was separated using SDS-PAGE, then transferred to PVDF membranes (Millipore, Billerica, MA, USA). The membranes were then blocked in 5% fat-free milk for one hour. The appropriate primary antibodies were incubated with the membranes at 4\\u00b0C overnight. The primary antibodies included TAZ (1:500 dilution; ab93362; Epitomics, USA), Cleaved Caspase 9 (1:800 dilution; sc-56076, Santa Cruz, USA), Cleaved Caspase 3 (1:1000 dilution; #9661, Cell Signaling Technology, USA) and GAPDH (1:1000 dilution; sc-47724, Santa Cruz, USA). This was followed by secondary incubation using a secondary antibody, either horseradish peroxidase-conjugated anti-rabbit or anti-mouse IgG (Thermo Fisher Scientific, New York, NY, USA). The proteins were visualized on X-ray film with an enhanced chemiluminescence reagent (Millipore, Billerica, MA, USA). The relative densities of the Western blot bands were measured using the Alpha View system (Cell Biosciences, Santa Clara, CA, USA). The Western blot results were normalized to those of GAPDH for quantification purposes. Vector construction and transfection The coding sequence (CDS) of the human TAZ gene was amplified from the cDNA of cervical cancer cell line SiHa by polymerase chain reaction (PCR) using the following primers: F 5\\u2032-GGAAGATCTACCATGCCTCTGCACGTGAAGTGGC-3\\u2032 ; R 5\\u2032-GTTGTCGACCTATCTCCCAGGCTGGAGGTGGTTG-3\\u2019 . The TAZ CDS fragment was subsequently cloned into the pCAG-PIRES2-AcGFP1 expression vector (Clontech, Mountain View, CA, USA) at the sites of SalI and BglII (TaKaRa, Tokyo, Japan). To construct a small interfering RNA expression vector that expressed TAZ-specific short hairpin RNA (shRNA), the following three TAZ sequences were used, which were obtained from GenePharma Co., Ltd. (Shanghai, China): TAZ-Homo-897: 5\\u2032-GCCTGATTGCTGAGTGTCATC-3\\u2032 ; TAZ-Homo-943: 5\\u2032-GCATGTCGGAATGAATGACGT-3\\u2032 ; TAZ-Homo-1105: 5\\u2032-GGACTTCATTCAAGAGGAATT-3\\u2032 . Following the manufacturer\\u2019s instructions, the TAZ overexpression vectors were transfected into SiHa cells and the TAZ shRNA vectors were transfected into SiHa and HeLa cells using the Lipofectamine 2000 reagent (Invitrogen, Carlsbad, CA, USA). The drug-resistant colonies were collected, expanded and identified after the transfected cells were treated for 3 weeks with G418 (Calbiochem, La Jolla, CA, USA). Cell growth and cell viability assays Cervical cancer cells were seeded in 6-well plates at a concentration of 1\\u00d710 4 cells/well and cultured for 7 days. After harvesting on days 1, 3, 5 and 7 days the number of cells was counted using a hemocytometer under light microscopy, and cell growth curves were charted to assess cell growth. To assess cell viability, 3-(4,5-dimethylthiazole-yl)-2,5-diphenyl tetrazolium bromide (MTT, Sigma-Aldrich) was used. Cells were seeded at a density of 1000 cells per well in 96-well plates and cultured for 7 days. Six parallel samples were used for each condition. Assays were performed according to the manufacturer\\u2019s protocol. Flow cytometry analysis FACS Calibur flow cytometry (Becton Dickinson, Franklin Lakes, NJ, USA) was used to analyze the cell cycle. In brief, approximately 1\\u00d710 6 cells were collected and fixed in cool 70% ethanol at 4\\u00b0C overnight. After washing twice with PBS, the cells were incubated in 1 mL of staining solution including 20 mg/mL propidium iodide (Sigma-Aldrich, St. Louis, MO, USA) and 10 U/mL RNase A for 30 minutes at room temperature. Then, cell cycle progression was analyzed by FACS. For apoptosis analysis, a PE Annexin V Apoptosis Detection Kit (559763, BD, Franklin Lakes, NJ, USA) was used for flow cytometry. After being cultured for 48 hours, the cells were harvested and washed with PBS, then incubated with PE Annexin V and 7-AAD for 10 minutes in the dark. Finally, FACS (Becton Dickinson, Franklin Lakes, NJ, USA) was used to detect the stained cells. Tumor xenograft experiment Female BALB/c-nude mice that were 4 to 6 weeks old were obtained from the Model Animal Research Center of Nanjing University (Nanjing, China) and housed in a SFP room. A total of 1\\u00d710 6 tumor cells were injected into the subcutis on the dorsum of each mouse. The tumor sizes were measured every three days using vernier calipers, and tumor volumes were calculated with the standard formula: length\\u00d7width 2 /2. At the end of the experiment, the mice were killed by cervical vertebra dislocation and the tumors were weighed. Then, the tissue from the xenograft tumors were fixed and paraffin-embedded for histological analysis. All animal studies were evaluated and approved by the Animal Care and Use Committee of the Medical School of Xi\\u2019an Jiaotong University. TUNEL assay Apoptosis was detected by using the In Situ Cell Death Detection Kit, POD (Roche Corporation, Switzerland) according to the manufacturer\\u2019s instructions. In brief, the paraffin-embedded tissue was incubated with proteinase K at 37\\u00b0C for 15 minutes after dewaxing and rehydrating and then incubated with the TUNEL reaction mixture (50 \\u03bcl Enzyme Solution mixed with 450 \\u03bcl Label Solution) for 60 minutes at 37\\u00b0C with a humidified atmosphere in the dark. After being washed 3 times with PBS for 5 minutes each, converter-POD was added and the slides were incubated in a humidified chamber for 30 minutes at 37\\u00b0C. Finally, the samples were incubated with DAB substrate for 10 minutes at room temperature. The total number of cells with TUNEL-positive nuclei was counted under a light microscope. Statistical analysis Statistical analysis was performed using SPSS software version 16.0 (SPSS Inc., Chicago, IL, USA). All data were expressed as the mean \\u00b1 standard deviation of the mean (SD). For group comparisons, Student\\u2019s t-test, Mann-Whitney U test, one-way ANOVA and chi-square tests were performed. P <0.05 was defined as statistically significant.\",\n",
            "        \"subjects\": null,\n",
            "        \"results\": \"Results The expression of TAZ in human normal cervix and various cervical lesions To understand the endogenous TAZ expression in cervical carcinogenesis, we detected the expression of TAZ in 27 NC, 26 HSIL and 41 SCC samples by immunohistochemistry. As shown in Fig 1A , TAZ staining was observed in the cytoplasm of positive cells, both in the NC tissue and various cervical lesions. According to the established method of IRS, the rate of positive TAZ expression was 40.74% (11 of 27) in the NC samples and was significantly increased to 73.08% (19 of 26) in HSIL lesions and 80.49% (33 of 41) in SCC lesions ( Table 1 and Fig 1B , P <0.05). Additionally, the IHC score of the TAZ staining gradually increased from 5.67\\u00b13.97 in NC tissue to 8.11\\u00b14.26 in HSIL lesions and 9.30\\u00b13.79 in SCC lesions ( Fig 1C and S1 Table , P <0.05). The expression of TAZ protein in 8 fresh NC samples and 8 fresh SCC lesions from patients undergoing surgery was detected by Western blot ( Fig 1D ), and the level of TAZ expression related to GAPDH was higher in SCC than that in normal cervical tissue ( Fig 1E , P <0.01). All these results indicate that TAZ protein is up-regulated in SCC, suggesting that the TAZ may promote cervical carcinogenesis. 10.1371/journal.pone.0177171.g001 Fig 1 TAZ expression in human normal cervical tissue and in various cervical lesions. (A) Immunohistochemical (IHC) staining showing TAZ expression in normal cervix, high-grade squamous intraepithelial lesion and squamous cervical cancer (magnification, \\u00d7100). (B) TAZ staining is classified into 2 categories (negative and positive). Bar graph showing TAZ-positive percentage and TAZ-negative percentage of 27 normal cervix samples, 26 high-grade squamous intraepithelial lesions and 41 squamous cervical cancer lesions. (C) The IHC score of TAZ staining in normal cervix, high-grade squamous intraepithelial lesions and squamous cervical cancer. (D) Western blot analysis of TAZ expression in 8 fresh normal cervix samples and 8 squamous cervical cancer samples. representative blots are shown. (E) The relative expression of TAZ in normal cervix samples and squamous cervical cancer samples. GAPDH was used as a loading control. NC: normal cervix; HSIL: high-grade squamous intraepithelial lesion; SCC: squamous cervical cancer. Values are shown as the mean\\u00b1SD, * P <0.05 vs . control; ** P <0.01 vs . control. 10.1371/journal.pone.0177171.t001 Table 1 TAZ expression in different cervical tissue specimens. Specimens Total TAZ Staining P- value Negative, No. (%) Strong, No. (%) NC 27 16(59.26) 11(40.74) HSIL 26 7(26.92) 19(73.08) <0.05 a SCC 41 8(19.51) 33(80.49) <0.05 b NC: normal cervix; HSIL: high-grade squamous intraepithelial lesion; SCC: squamous cervical cancer. a: HSIL versus NC; b: SCC versus NC. TAZ promotes the growth of cervical cancer cell in vitro What is the function of TAZ in cervical cancer cells? The expression of TAZ was observed by immunocytochemistry and Western blotting in the cervical cancer cell lines SiHa, HeLa, C33A, CaSki and HT-3 ( Fig 2A and 2B ). TAZ was up-regulated in SiHa cells by stable transfection with a TAZ-expressing plasmid ( Fig 2C ). TAZ was silenced in SiHa and HeLa cells by transfection with an shRNA plasmid targeting TAZ ( Fig 2F and 2I ). To uncover the potential effect of TAZ on the growth and viability of cervical cancer cells, cell growth curves and MTT assays were performed in vitro to evaluate cell growth and viability. The cell growth curve assay and the MTT assay showed that TAZ-overexpressing SiHa (SiHa-TAZ) cells had significantly stronger cell growth and cell viability than the control cells (SiHa-GFP) ( Fig 2D and 2E , P <0.05). However, the TAZ-silenced SiHa (SiHa-shTAZ) and HeLa (HeLa-shTAZ) cells had significantly weaker cell growth and cell viability than the control (SiHa-shControl and HeLa-shControl) cells did ( Fig 2G, 2H, 2J and 2K , P <0.05). The above results demonstrate that TAZ may promote the growth of cervical cancer cells in vitro. 10.1371/journal.pone.0177171.g002 Fig 2 TAZ increases the growth of cervical cancer cells in vitro . (A) Immunocytochemical staining showing TAZ expression in HeLa, SiHa, C33A, Caski and HT3 cells (magnification, \\u00d7100). ( B) Western blot analysis of TAZ expression in HeLa, SiHa, C33A, Caski and HT3 cells. Stably transfected cell lines were identified by western blotting: (C) control (SiHa-GFP) and TAZ-overexpressing SiHa (SiHa-TAZ) cells; (F) control(SiHa-shControl) and TAZ-knockdown SiHa (SiHa-shTAZ) cells; (I) control(HeLa-shControl) and TAZ-knockdown HeLa (HeLa -shTAZ) cells. The growth and viability of cells were detected using growth curves and 3-(4,5-dimethylthiazole-yl)-2,5-diphenyl tetrazolium bromide (MTT) assay in SiHa-GFP and SiHa -TAZ cells (D, E) , SiHa-shControl and SiHa-shTAZ cells (G, H) , and HeLa-shControl and HeLa-shTAZ cells (J, K) . Values are shown as the mean\\u00b1SD obtained from 3 separated experiments, * P <0.05 vs . control; ** P <0.01 vs . control. TAZ enhances the tumor formation of cervical cancer cells in vivo Next, we focused on the effects of TAZ on tumor formation of cervical cancer cells by performing a xenograft experiment in female BALB/c nude mice. A total of 1\\u00d710 6 TAZ-modified HeLa and SiHa cells and their respective control cells were injected into the subcutis on the dorsum of each mouse at the same time. Tumor volume was then continuously measured every three days. The growth of the tumors formed by SiHa-TAZ cells was much faster than those formed by SiHa-GFP cells ( Fig 3B , P <0.05). Furthermore, at the termination of the experiment, the average tumor weight was 1.13\\u00b10.30 g in the SiHa-TAZ cell group, which was much heavier than that in the SiHa-GFP cell group (0.52\\u00b10.22 g) ( Fig 3A , P <0.05). The consistent effect of TAZ on the tumor formation of cervical cancer cells was found in TAZ-silenced SiHa and HeLa cells. The tumors formed by SiHa-shTAZ cells grew more slowly than the SiHa-shControl cells did ( Fig 3D , P <0.05). Additionally, the average weight of tumors formed by SiHa-shTAZ cells (0.33\\u00b10.08 g) was lighter than the average weight of SiHa-shControl tumors (0.58\\u00b10.09 g, Fig 3C , P <0.05). Similarly, the silencing of TAZ in HeLa cells significantly inhibited tumor growth ability in vivo ( Fig 3F , P <0.01); the average tumor weight was 0.36\\u00b10.16 g in the HeLa-shTAZ group and 1.09\\u00b10.35 g in the HeLa-shControl group ( Fig 3E , P <0.01). These results suggest that TAZ may enhance cervical cancer tumor formation in vivo. 10.1371/journal.pone.0177171.g003 Fig 3 TAZ enhances the growth of cervical cancer xenografts in vivo. Tumor formation assays were performed with 6 mice per group. The tumor growth curves were calculated after injection into female nude mice based on monitoring performed every three days:(B) SiHa-GFP and SiHa-TAZ cells; (D) SiHa-shControl and SiHa-shTAZ cells; (F) HeLa-shControl and HeLa-shTAZ cells. The xenograft tumors were dissociated and weighed at the end of experiment: (A) SiHa-GFP and SiHa-TAZ cells; (C) SiHa-shControl and SiHa-shTAZ cells; (E) HeLa-shControl and HeLa-shTAZ cells. Values are shown as the mean\\u00b1SD,* P <0.05 vs . control; ** P <0.01 vs . control. TAZ inhibits cell apoptosis of cervical cancer cells To uncover the function of TAZ in cervical cancer cell growth, cell cycle distribution was monitored by FACS technology. As shown in Fig 4A and 4B , 56.73% of SiHa-TAZ cells were in the G0/G1 phase, 23.07% were in the S phase, and 15.81% were in the G2/M phase, which was not statistically different compared to the SiHa-GFP cells (58.52% in G0/G1, 22.84% in S, 15.54% in G2/M). Consistently, there was no significant difference in the cell cycle stages of the TAZ-silenced SiHa cells (62.36% in G0/G1, 19.62% in S, 12.41% in G2/M) and the SiHa-shControl cells (60.72% in G0/G1, 21.64% in S, 14.58% in G2/M; Fig 4C and 4D ). Similarly, the cell cycle distribution of HeLa-shTAZ cells (56.65% in G0/G1, 20.71% in S, 17.75% in G2/M) was not significantly different from that of the HeLa-shControl cells (55.27% in G0/G1, 22.95% in S, 18.84% in G2/M; Fig 4E and 4F ). These results suggest that TAZ is not involved in the cell cycle distribution of cervical cancer cells. 10.1371/journal.pone.0177171.g004 Fig 4 TAZ does not change the cell cycle distributions in TAZ-mediated cervical cancer cells. The cell cycle in TAZ-mediated cervical cancer cells were monitored with fluorescence-activated cell sorting (FACS) analysis (A, C, E,) . A quantitative analysis of the cell cycle distribution is shown. There was no significant difference in the G0/G1, S or G2/M phases between SiHa-GFP and SiHa -TAZ cells (B) , SiHa-shControl and SiHa-shTAZ cells (D) , and HeLa-shControl and HeLa-shTAZ cells (F) . Values are expressed as the mean\\u00b1SD of three experiments in duplicate. * P <0.05 vs . control; ** P <0.01 vs . control. We then examined apoptosis in cervical cancer cells by FACS using the PE annexin V /7-AAD double staining kit in TAZ-modified cervical cancer cells and the corresponding control cells, respectively. We found that the percentage of SiHa-TAZ cells in apoptosis was 2.16%, which was lower than that of SiHa-GFP cells (7.54%, Fig 5A and 5B , P <0.01). Moreover, the percentage of apoptosis cells in TAZ-silenced SiHa cells was 18.36%, which was higher than that in SiHa-shControl cells(6.15%, Fig 5E and 5F , P <0.01), the percentage of apoptosis cells in TAZ-silenced HeLa cells was 14.42%, which was higher than that in HeLa-shControl cells(5.14%, Fig 5I and 5J , P <0.01). These results indicate that TAZ may inhibit cervical cancer cell apoptosis. 10.1371/journal.pone.0177171.g005 Fig 5 TAZ inhibits apoptosis in cervical cancer cells both in vitro and in vivo. Apoptosis of TAZ-mediated cervical cancer cells was monitored with fluorescence-activated cell sorting (FACS) analysis. The representative cell apoptosis histograms and the percentage of apoptosis cells of TAZ-mediated cervical cancer cells are shown: (A, B) SiHa-GFP and SiHa-TAZ cells; (E, F) SiHa-shControl and SiHa-shTAZ cells; (I, J) HeLa-shControl and HeLa-shTAZ cells. Values are expressed as the mean\\u00b1SD of three experiments in duplicate. Apoptotic cell death in tumor xenografts formed by TAZ-mediated cervical cancer cells was measured by TUNEL assay, and representative micrographs are shown (magnification, \\u00d740):(C) SiHa-GFP and SiHa-TAZ cells; (G) SiHa-shControl and SiHa-shTAZ cells; (K) HeLa-shControl and HeLa-shTAZ cells. Quantitative analysis of apoptosis cells in xenograft samples formed by TAZ-mediated cervical cancer cells: (D) SiHa-GFP and SiHa-TAZ cells; (H) SiHa-shControl and SiHa-shTAZ cells; (L) HeLa-shControl and HeLa-shTAZ cells. Six tumor samples were measured and analysed every group. Values are shown as the mean\\u00b1SD. * P <0.05 vs . control; ** P <0.01 vs . control. To further investigate whether TAZ inhibits cell apoptosis in the tumor formation of cervical cancer cells, a TUNEL assay was performed in xenograft tumor tissues formed by TAZ-mediated SiHa and HeLa cells. The percentage of TUNEL-positive cells in tumor tissues formed by SiHa-TAZ cells(5.55%) was significantly lower than that in tumor tissues formed by SiHa-GFP cells (9.05%, Fig 5C and 5D , P <0.05). The percentage of TUNEL-positive cells in tumor tissues derived from TAZ-silenced SiHa cells was 17.85%, which was higher than that in tumor tissues formed by SiHa-shControl cells (9.68%, Fig 5G and 5H , P <0.01). Similarly, The percentage of TUNEL-positive cells in tumor tissues derived from TAZ-silenced HeLa cells was 19.90%, which was higher than that in tumor tissues formed by HeLa-shControl cells (8.48%, Fig 5K and 5L , P <0.01). These results indicate that TAZ inhibits apoptosis in cervical cancer cells both in vitro and in vivo. TAZ limits the cleavages of Caspase 9 and Caspase 3 in cervical cancer cells Caspase 9 and Caspase 3 have distinct roles in the intrinsic apoptotic pathways. First, Caspase 9 is activated by mitochondria-released cytochrome c, and then, Caspase 3 is activated[ 20 ]. To confirm the decreased induction of apoptosis in TAZ-expressing cervical cancer cells, Western blotting was performed to detect the expression of Cleaved Caspase 9 and Cleaved Caspase 3, which are involved in the mitochondrial apoptosis pathway. The relative expression of Cleaved Caspase 9 and Cleaved Caspase 3 in SiHa-TAZ cells was lower than that in SiHa-GFP cells ( Fig 6A and 6B , P <0.05). In contrast, the expression of Cleaved Caspase 9 and Cleaved Caspase 3 in SiHa-shTAZ cells was higher than in SiHa-shControl cells ( Fig 6C and 6D , P <0.05). Similarly, the expression of Cleaved Caspase 9 and Cleaved Caspase 3 was higher in HeLa-shTAZ cells than that in HeLa-shControl cells ( Fig 6E and 6F , P <0.05). These results suggest that TAZ inhibits apoptosis via limited cleavages of Caspase 9 and Caspase 3 in cervical cancer cells. 10.1371/journal.pone.0177171.g006 Fig 6 TAZ limits the cleavages of Caspase 9 and Caspase 3 in cervical cancer cells. Western blot analysis showing the expression of Cleaved Caspase 9 and Cleaved Caspase 3 in TAZ-mediated cervical cancer cells: (A) SiHa-GFP and SiHa-TAZ cells with the quantitative analysis (B); (C) SiHa- shControl and SiHa-shTAZ cells with the quantitative analysis (D); (E) HeLa-shControl and HeLa-shTAZ cells with the quantitative analysis (F) . Values are shown as the mean\\u00b1SD of three experiments in duplicate. * P <0.05 vs . control; ** P <0.01 vs . control.\",\n",
            "        \"discussion\": null,\n",
            "        \"conclusion\": \"Discussion TAZ, an important cardiolipin-remodeling enzyme, is involved in the maintenance of mitochondrial cardiolipin, which plays an important role in cellular metabolism, as exemplified by its involvement in mitochondrial energy production and apoptosis[ 14 ]. In recent years, TAZ has been observed to be overexpressed in several tumors, including thyroid carcinoma[ 18 ], rectal cancer[ 17 ], and colon cancer[ 16 ], suggesting it may function as a promoter in tumorigenesis. However, to date, the expression and mechanism of the involvement of TAZ in cervical cancer have not been thoroughly investigated. Here, we found that TAZ expression gradually increased from NC (40.74%) to HSIL (73.08%) to SCC (80.49%), suggesting that TAZ may function to promote the development and progression of cervical cancer ( Fig 1 ). The overexpression of TAZ in SiHa cells could enhance cell growth in vitro and in vivo , and reduced expression of TAZ in SiHa and HeLa cells could inhibit cell growth in vitro and in vivo (Figs 2 and 3 ). It has been previously reported that strong expression of the TAZ protein is related to rectal cancer development and RT response[ 16 ]. Our results in vitro and in vivo also support the hypothesis that TAZ is a tumor enhancer in cervical cancer. To uncover how TAZ affects cervical cancer cell growth, cell cycle and apoptosis experiments were performed, and the results show that TAZ may promote cervical cancer growth and inhibit cell apoptosis, but do not affect the cell cycle distribution in cervical cancer cells. In Barth syndrome, cardiolipin remodeling due to TAZ gene mutations can inhibit cell apoptosis[ 15 ], suggesting that abnormal TAZ expression is related to apoptosis. In rectal cancer, strong TAZ expression was positively correlated with expression of Livin [ 17 ], which is an apoptosis inhibitory protein and has been found to be involved in tumorigenesis and metastasis in epithelial tumors[ 21 ]. In cervical cancer, we propose that TAZ accelerates cell growth and inhibits cell apoptosis. In mammalian cells, the induction of apoptosis occurs via two distinct pathways, the extrinsic pathway and the intrinsic pathway. The intrinsic apoptotic cascade involves the release of apoptogenic factors from the mitochondria into the cytosol[ 22 , 23 ]. In the intrinsic pathway, Caspase 9 is first activated by mitochondria-released cytochrome c and then activates Caspase 3. Cardiolipin is required for cytochrome c release from mitochondria[ 23 , 24 ]. Cardiolipin, a phospholipid of the mitochondrial membrane, participates in several mitochondria-dependent apoptotic steps[ 24 ]. The maturation of cardiolipin requires the transacylase TAZ[ 25 ]. To further explore whether TAZ could regulate the expression of mitochondria-derived Caspases, which are important factors in the apoptosis pathway, we detected the levels of Cleaved Caspase 9 and Cleaved Caspase 3 proteins in distinct types of cervical cancer cells via Western blot. The expression of Cleaved Caspase 9 and Cleaved Caspase 3 was down-regulated in TAZ-over-expressing SiHa cells, whereas the levels of Cleaved Caspase 9 and Cleaved Caspase 3 were up-regulated in TAZ-knock-down SiHa and HeLa cells ( Fig 6 ). These results show that TAZ may decrease the cleavages of Caspase 9 and Caspase 3, which are important apoptosis-related factors in cervical cancer cells. In summary, our study demonstrates that TAZ promotes tumorigenesis of cervical cancer cells and inhibits cell apoptosis. Based on our results and previous literature, we propose that TAZ may inhibit the apoptosis of cervical cancer cells through the mitochondrial pathway, as characterized by the decreased cleavages of Caspase9 and Caspase3. However, the precise mechanism of TAZ-limited apoptosis need to be elucidated by further research (i.e. how does TAZ limits the cleavages of Caspase9 and Caspase3?). Our results suggest that TAZ promotes cell growth and tumor formaion in cervical cancer cells and may be a potential therapeutic target in certain types of cervical cancer.\"\n",
            "    },\n",
            "    {\n",
            "        \"pmcid\": \"PMC5441889\",\n",
            "        \"abstract\": \"Artur, S, Bogdan, P, Kawczy\\u0144ski, A, Winiarski, S, Grzegorz, J, and Andrzej, R. Manifestations of proprioception during vertical jumps to specific heights.\",\n",
            "        \"intro\": \"Introduction Jumping is an important ability in many sports. In team sports, the level of jumping ability indirectly determines the team's final success during competition ( 6 , 18 , 33 ). For example, in volleyball, the skill of blocking, in which a player or players jump to a specific or maximal height and extend their upper limbs above and over the net (without touching the net) to block an attack (spike) by the opponent, is crucial to team success ( 9 , 25 ). In basketball, the 2-legged jump shot has become more frequent, accounting for more than 70% of all shots during a game ( 31 ). Therefore, within 20 years the number of jumps during a basketball game increased twofold ( 29 , 36 ). Each sport that involves jumping is characterized by sport-specific jump types. For instance, basketball players and volleyball players perform different types of jumping movements ( 3 ). Jumping abilities are required for both offensive and defensive activities. During attack, basketball players use jumping during jump shots, layups, and dunking. Jumps help to avoid the defensive players and improve scoring chances. Defensive players jump to steal or block the ball, prevent opponents from shooting and passing, and limit the visibility range of offensive players. Similar jumps are observed during handball games. In addition, in basketball, players on both teams fight for the ball in the air near the backboard after a missed shot. In volleyball, jumping is used during blocking, spiking, and serving. Setters also often perform jumps while setting ( 3 , 6 , 18 , 24 ). Other actions that involve jumps but cannot be categorized into the above groups should also be considered. In all of these activities athletes often aim to jump to a specific height rather than to the maximum height, although the target height is often near the maximum. Connected, repeated jumps must also be performed at specific heights (to steal the ball or maintain possession), which often leads to performing a maximum height jump ( 29 ). Thus, the height of a jump depends strongly on the temporary situation on the court. In general, jumping ability is related to the properties of the neuromuscular system, which is responsible for proper muscle coordination during jumping ( 39 ). An external manifestation of the efficiency of the central nervous system is the quality of the movements performed. Proprioception is based on an accurate perception of forces (muscle tension), timing (movement velocity), and space (position of body segments with respect to each other) during the performance of a particular motor activity. Proprioception is essential to properly use the control impulses (based on the incoming information and evaluation and adequate processing of this information) and facilitates the optimal performance of the motor task. Proprioceptors located in the muscles and tendons provide information to the brain about muscle tone. Furthermore, the proprioceptors present in joint capsules are responsible for information about the changes in intraarticular pressure caused by movements. These systems help humans know their current body position (without seeing). Receiving and processing information from proprioceptors are required for proper reflexes, planning and performing movements, regulation of muscular tension cocontraction of antagonistic muscles, and for increased mental activity (especially regarding body position) ( 14 , 17 ). To properly perform a submaximal jump, maximal capabilities must be corrected in terms of movement patterns and muscular tensions. Thus, the proprioceptive system manifests during the performance of jumps to varied heights. Kai et al. ( 19 ) reported a relationship between the vertical jump height and knee joint flexion angle. Furthermore, Vanrenterghem et al. ( 41 ) concluded that movement effectiveness (energy cost reduction) is a driving criterion of the submaximal jump performance technique. Lees et al. ( 22 ) stated that submaximal jumps seem to stress the ankle and knee muscle as adequately as maximal jumps. However, maximal jumps are achieved through greater engagement of the hip extensor muscles ( 22 ). Despite the substantial interest among researchers regarding vertical jumps (because of applications in sports practices), some questions remain unanswered, particularly concerning submaximal jumps ( 23 , 26 , 29 ). A limited number of studies have analyzed jumps to a specific percentage height ( 19 , 40 , 41 ), especially at low heights (25 or 50% of the maximum height). Because of the specific nature of team sports, the proprioceptive system manifests indirectly during the performance of jumps to varied heights. In many team sport games, success requires accurate and well-controlled jumping. Therefore, the efficiency of the proprioceptive system among individuals who play team sports that require jumping to a specific height, such as basketball, volleyball, and handball, is an interesting topic. A limited number of studies have analyzed jumping accuracy ability. If jumping ability is influenced by different training protocols, we assume that athletes from different disciplines and of different ages and sexes will also be characterized by different jumping abilities ( 10 , 27 , 28 , 33 ). Therefore, we evaluated differences in accuracy in the performance of a vertical jump to a specific height among players from different team sports and of different ages and sexes. We also examined whether the level of maximal jumping ability correlated with the accuracy of performing a vertical jump to a specific height. Because jumps are complex movements and are related to the properties of the neuromuscular system, they should include all 3 components of the proprioception system: force, timing, and space. However, to perform a jump to a specific height, adequate force value (muscle tension) must be generated. Therefore, the question is whether the accuracy of a jump to a specific height is related to maximal value of force during the take-off phase. The aim of the present study was to determine the level of ability to control jump height with respect to maximum jumping ability among a group of young athletes in team sport games in which jumping plays an important role.\",\n",
            "        \"method\": \"Materials and Methods Experimental Approach to the Problem Proprioceptive system efficiency was evaluated indirectly in young athletes who played team sports and was estimated based on the ability to control vertical jump height. To properly perform a submaximal jump, maximal capabilities must be corrected in the neuromuscular system. Thus, the proprioceptive system manifests during the performance of jumps to specified heights. The dependent variable was the accuracy of performing a jump to a specific height. The independent variables were age, sex, sport discipline, maximal countermovement jump (CMJ) height, and maximum take-off force. In total, 12 teams were studied at the end of a preparation macrocycle. Subjects The examinations of selected teams members were performed in a group of 305 participants (age range = 12\\u201316) selected among young individuals who trained at team sport games (basketball, volleyball, and handball). The selected members were the most talented players chosen by coaches for regional teams. A detailed characterization of the study groups is presented in Table 1 . Before the tests, each participant was familiarized with the task. Parents were informed about the purpose of the study and provided written permission for the tests. The experiments were performed in the Biomechanical Analysis Laboratory (with PN-EN ISO 9001:2009 certification). The research project was approved by the Senate's Research Bioethics Commission, and the procedures complied with the Declaration of Helsinki regarding human experimentation. Table 1. Characteristics of the research groups (mean \\u00b1 SD ).* Procedures Participants performed a 15-minute warm-up before performing the jumping tasks: 5 minutes of jogging, 5 minutes of dynamic stretching exercises, and 5 minutes of general and test-specific drills, lower leg drills and technique drills, and easy run outs over 30\\u201360 m. Each participant performed 8 CMJs with arm swing. The jumps were repeated 2 times to the following heights: maximum height ( h max ), 25% h max , 50% h max , and 75% h max . The 2 maximal height jumps were performed first. Then, the participants were given the following instructions: \\u201cKnowing the maximum height you are able to jump to, perform a jump at 25%, 50%, and 75% of your maximum abilities.\\u201d During performance of the jump, the subjects were asked not to flex their lower limbs during the flight phase. Whenever a mistake was made during the jump, the jump was repeated. The participant did not receive feedback on the jump heights. A maximum CMJ height (greater value of the 2 attempts) was recorded. The mean accuracy of the 2 attempts for the specific percentage value ( D x for x = 25, 50, or 75) of each jump to a specific height was also calculated. The jump accuracy to a specific height was calculated as a mean of the percentage and the sum of errors methods according to the equations in Struzik et al. ( 37 ). We propose equations 1 and 2 . The jump accuracy was based on the mean value of 2 trials for each percentage value ( D x ), with the error score calculated both as a percentage ( equation 1 ) and as the total error ( equation 2 ): The symbol h x,i denotes successive jumps ( i ) to a particular percentage height x . The best accuracy result when performing a jump to a specific height was 0, and greater values corresponded to lower accuracy levels. The methodology of the sum of errors yields a quantitative absolute value when exceeding the demanded height, whereas the percentage method additionally normalizes this value with respect to the maximum jump height. The ground reaction forces during jumping were recorded with an ACCUPOWER force plate manufactured by AMTI (Watertown, MA, USA) with ACCUPOWER software. This equipment enabled an accurate measurement of the take-off and landing time (using the force plate) and aided the assessment of the duration of the flight phase and, consequently, the jump height. Furthermore, the value of the maximum take-off force ( F max ) during all types of jumps was also recorded. The sampling frequency for the signal from the plate was set at 240 Hz. The jump height ( h ) was calculated using the following formula: where t f was the flight time and g was the acceleration due to gravity ( 5 , 7 , 12 ). In the reliability part, the subjects performed 4 jumps at each percentage of jump (25, 50, 75, and 100%) on 2 consecutive days. During each session, jumps were separated by a 3-minute rest period to avoid fatigue ( 34 ). Statistical Analyses The normality of the distributions of each variable was tested using the Shapiro-Wilk and Lilliefors tests. The data did not follow normal distributions. Thus, Spearman's rank correlation coefficients were used to analyze the relationships between the maximum jump height and accuracy of performing jumps to specific heights. Differences between variables (stratified according to age categories, sex, and sport) were analyzed using a multivariate analysis of variance. Significant differences between the jump accuracies to specific heights were verified with the Wilcoxon signed-rank test for dependent variables. The level of significance was set at \\u03b1 = 0.05 in all tests. The relative and absolute reliability across the reliability session were computed using Intraclass Correlation Coefficient (ICC). The relative reliability was evaluated by calculating a 2-way fixed ICC 2,1 (for absolute agreement). Reliability coefficients (i.e., ICC values) were interpreted according to Landis and Koch ( 21 ): an ICC between 0.00\\u20130.20 was considered poor, 0.21\\u20130.40 fair, 0.41\\u20130.60 moderate, 0.61\\u20130.80 substantial, and 0.81\\u20131.00 almost perfect.\",\n",
            "        \"subjects\": null,\n",
            "        \"results\": \"Results The reliability of the performed measurements for jumping to 100% of the maximum height was almost perfect (ICC 0.929, 95%). The reliability of the performed measurements for jumping to 75% of the maximum height was almost perfect (ICC 0.833, 95%). The reliability of the performed measurements for jumping to 50 and 25% of the maximum height were almost perfect (ICC 0.887, 95%) and substantial (ICC 0.730, 95%), respectively. The statistical power was sufficient to detect the described differences. For significant changes ( p \\u2264 0.05), the partial \\u019e 2 effect size was found between 0.52 and 0.83. Table 2 contains the mean values (\\u00b1 SD ) of the maximum CMJ heights and jump accuracies to specific heights. The jump accuracy to a specific height increased as the percentage value expected to be reached increased (Table 2 ). The exceptions were groups of junior female basketball players, junior male volleyball players, and youth male basketball (BMY) players. In these groups, the difference between D 25 and D 50 (calculated using the sum of errors) was not significant. Jumps to 25% of the maximum height were (significantly) approximately 2 times higher than the target values. Table 2. Mean values (\\u00b1 SD ) of the maximum countermovement jump (CMJ) height ( h max ) and jump accuracy to a specific height ( D x ) for each subgroup.* Table 3 presents the relationships between the maximum CMJ heights and jump accuracy to specific heights. We observed negative relationships between the maximum CMJ height and jump accuracy to specific heights (particularly when using the percentage method). These relationships were significant or insignificant depending on the subgroup (most of the significant relationships were among handball players). However, groups of handball players (handball female youth and handball female junior) demonstrated positive significant relationships between the maximum CMJ height and jump accuracy to a specific height (calculated by means according to the sum of errors method). Table 3. Correlation coefficients between maximum countermovement jump (CMJ) height ( h max ) and jump accuracy to a specific height ( D x ).* No significant differences were observed for maximum CMJ height between groups divided according to age, sex, and sport. Only BMY players and youth male handball (HMY) players demonstrated significant differences in the jump accuracy to a specific height. Basketball players demonstrated greater levels of accuracy for jumps to 25 and 50% of the maximum height (for percentage and sum of errors methods). Table 4 presents the mean values (\\u00b1 SD ) of the maximum take-off force during CMJs. No statistically significant correlations were found between the jump accuracy to specific heights (percentage and total errors methods) and maximum take-off force. Table 4. Mean values (\\u00b1 SD ) of the maximum take-off force ( F max ) during countermovement jumps (CMJs).*\",\n",
            "        \"discussion\": \"Discussion Many motor tasks can be performed at maximal (attempts to achieve the highest possible performance) or submaximal (attempts to achieve a certain level of performance) levels of performance. The central nervous system generates an appropriate set of control signals to all muscles involved during the performance of a motor activity. Performing a motor task at the maximum level can be relatively easy because of the existence of a unique set of control signals yielding maximum performance. These optimal control signals may result from learning a motor task over a longer period of time. Performing a motor task at submaximal capabilities is more difficult because of the presence of many movement patterns (and different sets of control signals) that can be used for these tasks ( 8 , 22 , 40 , 41 ). The neural control of jumping is highly complex, and both feedforward (preprogrammed) and feedback (reflex) mechanisms have to be highly adaptive to ensure balance between achieving maximum performance (power) and the risk of overload injuries ( 39 ). A vertical jump is a complex movement characterized by eccentric-concentric muscle work during the countermovement and take-off phases, with different characteristics and purposes of the 2 phases. The total duration of the countermovement and take-off phases (which contribute to jump effectiveness) is approximately 0.5 seconds ( 38 ). Thus, the movements performed during both phases are regarded as nearly ballistic. Therefore, during the countermovement or take-off phase, movement modifications (corrections) during that respective phase are impossible unless planned before the movement begins. Control signals must (to a large extent) be preprogrammed. Consequently, jump control relies heavily on the storage capacity of the central nervous system ( 17 , 38 , 40 ). A jump performed to a submaximal height has a slightly different movement pattern compared with the maximum jump. A jump to the maximum height is characterized by a greater range of movement in the joints of the lower limbs to achieve greater kinetic energy in the countermovement phase ( 4 , 19 ). Changes are also observed in the proportions of individual muscle and joint contributions ( 8 , 22 , 41 ). Less energy is needed when performing a submaximal jump; thus, to reduce energy waste, countermovement and arm swing are reduced ( 16 , 23 , 41 ). Therefore, during a submaximal performance, active stretch during the stretch-shortening cycle only created minor benefits for the utilization of elastic energy in muscles ( 11 ). Consequently, a decline in the accuracy of performing the movement with a reduction in the jump height seems to be undesirable and may cause fatigue. Young athletes, who jump approximately 2 times higher than necessary during the performance of jumps to 25% of their maximum height, consume (unnecessarily) too much energy ( 42 ). Performing a series of tasks with excessive intensity (excessive load) might lead to the premature depletion of energy sources. This situation might lower the subject's performance during the final phases of the game. It should be noted that during the game, the athlete has to cope with quickly changing situations on the court, time pressure, the defensive actions of opponents, and emotions. These issues might additionally deteriorate sport movement efficiency. To properly perform a task, submaximal capabilities have to be corrected in terms of movement patterns and muscular tensions. Young athletes tended to increase their accuracy when performing a jump to a specific height when the desired percentage value was increased. Thus, young athletes may lack the ability to work within submaximal loads. Young athletes aged 13\\u201315 do not perform a high number of strength exercises during practice. Therefore, a decline in the accuracy of performing a jump to lower target heights may be because of puberty and the changes that occur during physical development ( 6 , 20 ). According to Lloyd et al. ( 26 ), children become more reliant on feedforward mechanisms as they mature. Certain age-related developments in short-latency stretch-reflex responses may influence the neural regulation of the stretch-shortening cycle during submaximal hopping. However, in the study performed by Struzik et al. ( 37 ) in postpubertal basketball players, the accuracy of releasing a particular value of static torque by elbow joint extensors decreases as the target values decrease. This finding might suggest that the efficiency of the proprioceptive system is nonuniform and depends on the type of motor activity. The level of proprioceptive (or kinesthetic) sensitivity is highest in the body parts involved in a given sport and higher in athletes compared with untrained people. For example, table tennis players can more accurately reproduce range of movement and pressure force ( 1 , 2 ). Furthermore, Han et al. ( 15 ) argue that ankle proprioception scores (an active movement extent discrimination test) have a positive relationship with competition level in soccer. Proprioception also has a crucial role in balance control, and ankle proprioception is the most important aspect of balance ( 13 ). Because of the efficiency of the proprioception system, basketball players may organize compensatory behaviors with the joints of their upper limbs, which are used for shooting ( 35 ). Because of the lack of relationships between jump accuracy to specific heights (percentage and total errors methods) and maximum take-off force, this variable cannot be used to evaluate the efficiency of the proprioception system. The lack of relationships may be because of the presence of too many variables related to jump height ( 30 ). In this study, the maximum CMJ height was negatively associated with jump accuracy to specific heights (particularly using the percentage method). Therefore, an increase in maximum jump height is also accompanied by increased jump accuracy to a specific height. This relationship, however, was not observed in all subgroups. Thus, its regular presence cannot be unequivocally demonstrated. The maximum CMJ height and jump accuracy to a specific height did not differ between groups divided according to age, sex, and sport (with one exception). The BMY group demonstrated significantly higher jump accuracies to 25 and 50% of their maximum ability compared with the HMY group. This exception may reflect the greater number of jumps performed in basketball compared with handball games. The level of ability (jumping accuracy) analyzed in this study was relatively constant among the groups divided according to team sport. One limitation of this study may be the different sizes of the groups. In most cases, the groups contained fewer than 30 people. We realize also that the determination of jump height based on the flight phase duration has some limitations and is not a perfect method. However, the measurement error is relatively small compared with other currently used methods.\",\n",
            "        \"conclusion\": \"Conclusions Increasing the target jump height increases the accuracy of reaching the desired height. Jumps to 25% of the maximum height were approximately 2 times higher than the target height, which might reflect a lack of ability to work with submaximal loads. A negative correlation between the maximum CMJ height and accuracy of performing the jump to a specific height was observed (particularly using the percentage method). Individuals who had greater maximum jump heights also exhibited greater accuracy when jumping to a specific height. However, these relationships were not significant in all studied groups (stratified by age, sex, and sport). Significant differences in the jump accuracy to a specific height were found only between groups of youth basketball players and youth handball players. Higher levels of jump accuracy to 25 and 50% of the maximum height were obtained by basketball players. Therefore, there was little difference between the subgroups regarding the jump ability analyzed in this study.\"\n",
            "    },\n",
            "    {\n",
            "        \"pmcid\": \"PMC5601641\",\n",
            "        \"abstract\": \"Vasculogenic mimicry (VM) was an important tumor blood supply to complement the endothelial cell-dependent angiogenesis, while leptin and receptor (ObR) involved in angiogenesis in glioblastoma has been reported on previous study, but the relationship between ObR expression and VM formation in human glioblastoma tissues, as well as their prognostic significance still remains unclear. In our study, we found that VM recognized by CD31-/PAS+ immunohistochemical staining in glioblastoma tissues showed a positive correlation with leptin expression (\",\n",
            "        \"intro\": null,\n",
            "        \"method\": \"MATERIALS AND METHODS Patients and clinical data 82 archived, formalin-fixed, paraffin-embedded glioblastoma tissue specimens were all acquired from Changhai Hospital between January 1996 and December 2006. The glioblastoma tumor grade of patients was diagnosed histologically according to the WHO classification. There were 48 men and 34 women, the age range was 34\\u201377 years and the mean ages were 56 and 62 year respectively. The survival time was measured from the date of surgery to the date of death. Total period of follow up was 8\\u201384 months. The present study was approved by the Hospital Research Ethics Committee and performed in accordance with institutional and state guidelines on the use of human tissue specimens for experimental purposes. Written informed consent was obtained from all patients. Immunohistochemical and periodic acid-schiff (PAS) double-staining Tissue sections (4\\u20135 \\u03bcm) were deparaffinized and dehydrated using a graded series of ethanol solutions and stained with hematoxylin-eosin (H&E) as standard procedures, and the serial sections were stained with double staining of CD31 or ObR and PAS. Endogeneous peroxidase was then inactivated with 3% hydrogen peroxide at room temperature for 20 minutes. Then the slides were soaked in 0.1 mol/L citrate buffer (pH 6.0) and placed in an autoclave at 121\\u00b0C for 2 minutes for antigen retrieval. After washing with PBS (pH 7.4), the sections were blocked with 1% BSA diluted in PBS at 37\\u00b0C for 30 minutes, and then incubated with anti-CD31 protein IgG (1:50, Abcam, Cambridge, UK) or anti-ObR protein IgG (1:50; Dako, Glostrup, Denmark) at 4\\u00b0C overnight, after being rinsed with PBS again, the sections were incubated with HRP-conjugated goat anti-mouse/rabbit antibody and DAB (DAKO, Glostrup, Denmark) washing distilled water, then the section were incubated with 0.5% PAS for 10 min in a dark chamber and washing with distilled water for 3 min. Finally all of these sections were counterstained with hematoxylin. All sections with immunohistochemical staining were observed and the pictures were photographed by an Olympus microscope (IX-70 OLYMPUS, Japan). Four representative fields within each section were randomly chosen and captured under 200X. The integrated optical density (IOD) in each image was measured with the same setting for all the slides, and the density was calculated as IOD/total area of each image. Double-fluorescence immunostaining Double-fluorescence immunostaining of the glioblastoma tumor tissue was performed with a sequential fluorescent method as described [ 33 ]. The primary antibodies against ObR and CD133 were all from Abcam, and Alexa488-conjugated goat anti-mouse IgG and Alexa568-conjugated goat anti-rabbit IgG (all from Invitrogen) were used as secondary antibodies. Immunofluorescence was observed with the Olympus IX-71. Statistical analysis Spearman's rank test was used to analyze the correlation between ObR expression and VM formation, as well as the correlation between ObR expression or VM formation with GMT markers. Survival curves were estimated using Kaplan\\u2013Meier method and compared by log rank test. All statistical analyses were performed using the SPSS software system (version 20.0; SPSS, Chicago, IL, USA). P < 0.05 was considered statistically significant.\",\n",
            "        \"subjects\": null,\n",
            "        \"results\": null,\n",
            "        \"discussion\": null,\n",
            "        \"conclusion\": null\n",
            "    },\n",
            "    {\n",
            "        \"pmcid\": \"PMC5717332\",\n",
            "        \"abstract\": null,\n",
            "        \"intro\": \"INTRODUCTION Giving birth plays an important role in reducing the risk of breast cancer (BC) in women [ 1 - 3 ]. In addition, breastfeeding, which is inseparable from giving birth, is considered to have a risk reduction effect, although controversies exist [ 3 - 8 ]. Pregnancy and lactation prevent BC through the following 2 major mechanisms. The first is promoting the differentiation of mammary cells [ 9 , 10 ]. In parous women, the proportion of differentiated terminal ductal lobular tissues is higher than in nulliparous women, and the risk of malignancy is lower, because undifferentiated breast tissues have a higher risk of malignant transformation than differentiated breast tissues. Breastfeeding triggers a continuation of the cell differentiation process and prolongs the time during which the tissue is in a mature state [ 11 , 12 ]. The second mechanism is the reduction of lifetime estrogen exposure. According to the estrogen hypothesis, higher estrogen exposure increases the risk of BC [ 13 ] in comparison to women in whom ovulatory cycles were attenuated due to pregnancy and lactation [ 14 , 15 ]. Prior meta-analyses have failed to demonstrate a dose-response relationship of breastfeeding on BC risk due to high heterogeneity among studies (>70%), a small number of cohort studies (only 3), and non-significant dose-response relationship in meta-analysis of cohort studies [ 8 , 16 ]. In addition, the combined influence of the number of childbirths and breastfeeding period on the risk of BC has not been clarified in previous epidemiological studies. The reason for this is that women with 4 or more childbirths and longer breastfeeding periods (over 25 months) comprise less than 5 and 10% of the total population, respectively. Furthermore, since the risk of BC by reproductive risk factors depends on menopausal status, the analysis should be stratified by menopausal status. Therefore, larger study populations must be analyzed to identify a dose-response relationship. Furthermore, since the number of childbirths and the breastfeeding period are biologically related, the duration of breastfeeding increases as the number of childbirths increases. In most studies, due to limited sample sizes, it is difficult to distinguish between women who are not breastfed and women who are not breastfed among women who have had one or more childbirths. Because it is known that both factors reduce BC risk, the presence of both a longer breastfeeding duration and more childbirths may exert a combined effect. A previous study was conducted of the interaction of these 2 factors on the risk of ovarian cancer [ 17 ], but their interaction with BC risk has not been confirmed, due to various complex situations causing limitations in statistical power. It has recently been established that BC has different clinical courses depending on the hormone receptor (HR) status, classified in terms of estrogen and progesterone receptors, the HER2-receptor, and/or Ki-67 levels, and the pathological subtype of cancer. We hypothesized that the protective effect of breastfeeding on BC risk would be different depending on the number of childbirths, and therefore we investigated whether there was a synergistic effect between parity and breastfeeding. We designed a population-based case-control study using the Nationwide Breast Cancer Registry and woman participants in large-scale population-based health screening programs. The aims of this case-control study were: 1) to identify the main effects of the number of childbirths and breastfeeding period on BC risk reduction, while sufficiently controlling for confounding factors; 2) to determine whether women with a combined experience of more childbirths and longer breastfeeding duration had a more strongly reduced BC risk than those with exposure to a single protective factor; and 3) to investigate the heterogeneity of the associations of childbirth, breastfeeding, and their combination with BC risk across menopausal status, HR status, and pathological subtypes.\",\n",
            "        \"method\": \"METHODS Study Subjects The Korea Breast Cancer Study has a mixed design of a large-scale population-based case-control study and cohort study, and is composed of man and woman BC patients from throughout the nation and community-based healthy controls from 2004 to 2012. The eligible BC cases were obtained from the database of the Korean National Breast Cancer Registry, maintained by the Korean Breast Cancer Society (KBCS) [ 18 ]. In brief, the KBCS has conducted a nationwide BC registry for man and woman BC patients diagnosed at 102 general hospitals since 1996, and the registration data include more than 95% of new breast cancer patients in Korea. The BC cases included in this study were woman BC patients aged 40-74 who were registered between 2004 and 2012. We excluded individuals with missing data about both the number of childbirths and the breastfeeding period. The eligible controls were obtained from the database of the Health Examinees Study (HEXA) [ 19 ]. In brief, the HEXA is a population-based cohort study of health screening recipients who receive biennial health screenings covered by the National Health Insurance Service, and is one of the cohort studies of the Korean Genome and Epidemiology Study conducted by the Korea Centers for Disease Control and Prevention. The control subjects in this study were defined as healthy women aged between 40 and 74 with no diagnosis of BC who were registered from 2004 to 2012 (n=101 017). We excluded women with missing data about both the number of childbirths and the breastfeeding period. We matched cases and controls 1:1, according to age and enrollment period, which was divided into 3 categories; 2004-2006, 2007-2009, and 2010-2012. Finally, a total of 25 778 case-control sets (12 889 pairs) were selected as study subjects. Data Collection and Management Information about age, family history of BC among first-degree and second-degree relatives, age at menarche, pregnancy history, age at the first full-term pregnancy, parity and the number of childbirths, periods of breastfeeding, oral contraceptive (OC) use, and hormone replacement therapy use was collected by personal interviews. Statistical Analysis Differences between BC cases and controls in the frequencies of general characteristics, such as family history, number of childbirths, breastfeeding experience, OC use, body mass index (BMI), age at menarche, and age at first full-term pregnancy, were tested using the chi-square test. The number of childbirths was classified as 0 (nulliparous women), 1, 2, and 3+. Breastfeeding duration was classified as 0 (never breastfeeding), 1-12, 13-24, and 25+ months. The BC risk associated with parity, number of childbirths, breastfeeding duration, and the combination of the number of childbirths and breastfeeding duration in the entire sample was estimated using conditional logistic regression models with odds ratios (ORs) and 95% confidence intervals (CIs), stratified by age and enrollment year, and additionally adjusted for BC family history, age at menarche, number of childbirths (for breastfeeding and its duration), breastfeeding duration (for parity and number of childbirths), OC use, and BMI. The BC risk associated with these risk factors in premenopausal and postmenopausal women was estimated in unconditional logistic regression models, stratified by age and enrollment year and adjusted for same covariates, because the matching pair was broken. Moreover, we classified the BC patients according to HR status (HR+ or HR\\u2212 BC cases) and pathologic subtype (such as the luminal A, luminal B, HER2-enriched, and triple-negative BC [TNBC] subtypes); and analyzed the BC risk associated with childbirth and breastfeeding duration for each population subset and the total control population in unconditional logistic regression models, accounting for the variability of subgroup-specific OR values across HR and pathological types. To evaluate differences in the association of individual factors (number of childbirths and breastfeeding duration) and their combination with BC risk according to menopausal status, HR status, and pathological subtype, the heterogeneity across menopausal status, HR status, and pathological subtype was tested using the Cochran Q test at a significance level of p <0.05 and quantified by the I 2 statistic, with an I 2 of 50% considered to be evidence of substantial heterogeneity [ 20 ]. Most statistical analyses, except p -heterogeneity analyses, were performed in SAS version 9.4. (SAS Institute Inc., Cary, NC, USA). The 1:1 matching of cases and controls was made using the G-match algorithm [ 21 ] of SAS 9.4. The Cochran Q test was performed in Stata version 14 (StataCorp, College Station, TX, USA). Ethics approval and consent to participate This study was approved by the institutional review board (IRB) of Seoul National University Hospital Biomedical Research Institue (IRB no., 0909-048-295).\",\n",
            "        \"subjects\": null,\n",
            "        \"results\": \"RESULTS The baseline characteristics of the 25 778 women in the study are described in Supplemental Table 1 . Statistically significant differences were found in most of the known risk factors, such as a family history of BC, childbirth, breastfeeding, OC use, high BMI, and late age of the first full-term pregnancy; but the distribution of menarche age was not significantly different between cases and controls. The BC risk by individual factors, such as total childbirths and total breastfeeding duration, premenopausal and postmenopausal status, HR status, and pathological subtype is shown in Tables 1 - 3 and Supplemental Figures 1 - 3 . The BC risk of nulliparous women was higher than that of parous women, especially in postmenopausal women (OR, 1.72; 95% CI, 1.18 to 2.50). This effect was stronger for HR+ BC (OR, 2.06; 95% CI, 1.43 to 3.00) and luminal A BC (OR, 2.15; 95% CI, 1.46 to 3.18) (data not shown). Women with more childbirths had a reduced risk of BC. In premenopausal women, 2+ childbirths were associated with a 20% reduced risk of BC (childbirths: 2 vs. 0, OR, 0.80; 95% CI, 0.65 to 0.97; childbirths: 3+ vs. 1, OR, 0.81; 95% CI, 0.70 to 0.94). Postmenopausal women with 2 childbirths had a 40% reduced risk of BC and those with 3 or more childbirths had a much more strongly reduced risk (childbirths: 3+ vs. 0, OR, 0.40; 95% CI, 0.35 to 0.43; p -heterogeneity between premenopausal and postmenopausal women <0.001). This effect was much stronger for HR+ BC ( p -heterogeneity between HR+ and HR\\u2212 <0.001, 0.009, <0.001 in total, premenopausal, and postmenopausal women, respectively) (childbirths: 3+ vs. 0, OR, 0.55; 95% CI, 0.48 to 0.61; and OR, 0.22; 95% CI, 0.18 to 0.25 in premenopausal and postmenopausal women, respectively) and in luminal A BC ( p -heterogeneity among the 4 BC subtypes in total, premenopausal, and postmenopausal women <0.001) (childbirths: 3+ vs. 0, OR, 0.55; 95% CI, 0.46 to 0.66; and OR, 0.26; 95%CI, 0.20 to 0.33 in premenopausal and postmenopausal women, respectively). In particular, postmenopausal women with 3 or more childbirths showed a 44% reduced risk of TNBC (childbirths: 3+ vs. 0, OR, 0.56; 95% CI, 0.40 to 0.78). Likewise, longer breastfeeding was associated with a reduced risk of BC. Relative to women with no breastfeeding, premenopausal women with a breastfeeding duration of 1-12 months had a reduced risk of BC (OR, 0.67; 95% CI, 0.60 to 0.74), while postmenopausal women with the same breastfeeding duration had no reduced risk for BC ( p -heterogeneity<0.001). Women with a much longer breastfeeding duration (25+ months) had a 56% reduced risk for BC, regardless of menopausal status (OR, 0.44; 95% CI, 0.37 to 0.53; and OR, 0.44; 95% CI, 0.38 to 0.52 in premenopausal and postmenopausal women, respectively). The protective effect of breastfeeding on BC risk was not significantly different according to HR status or pathological subtype. The combined effect of breastfeeding and childbirth is shown in Figure 1 and Table 4 . Since childbirth and breastfeeding were events in premenopausal women, we described to focus on the BC risk in premenopausal women. Compared to BC risk in women with no-childbirth and never-breastfeeding, there were no BC risk reduction in women with 1 childbirth but never-breastfeeding. However, the BC risk in women with 2+ childbirths but never-breastfeeding began to decline and their BC risk was reduced by at least 0.89-fold (3+ childbirths: OR, 0.89 95% CI, 0.86 to 0.94; 2 childbirths: OR, 0.78; 95% CI, 0.69 to 0.92). The BC risk was reduced by at least 0.69-fold for women with 1 childbirth and breastfeeding for \\u22651 month (breastfeeding for 1-12 months: OR, 0.65; 95% CI, 0.58 to 0.74; breastfeeding for \\u226513 months: OR, 0.69; 95% CI, 0.51 to 0.60), at least 0.63-fold for women with 2+ childbirth and breastfeeding for 1-12 months (2 childbirths: OR, 0.52; 95% CI, 0.45 to 0.61; 3+ childbirths: OR, 0.63; 95% CI, 0.59 to 0.68), and at least 0.44-fold for women with 2+ childbirth and breastfeeding for \\u226513 months (2 childbirths and breastfeeding for 13-24 months: OR, 0.44; 2 childbirths and breastfeeding for 13-24 months: OR, 0.36; 3+ childbirths and breastfeeding for 13-24 months: OR, 0.36; 3+ childbirths and breastfeeding for \\u226525 months: OR, 0.34). The combined OR of never breastfeeding and 3+ childbirths was different between premenopausal and postmenopausal women ( p -heterogeneity <0.001). Additionally, the combined OR of a breastfeeding duration of 25+ months and 2+ childbirths was different between premenopausal and postmenopausal women ( p -heterogeneity=0.002, <0.001 for 2 and 3+ childbirths, respectively). The patients analyzed in this study had the following distribution of BC stages: stage 0, 5.3%; stage I, 18.9%; stage II, 17.7%; stage III, 6.5%; and stage IV, 0.7%. The BC risk associated with number of childbirths, breastfeeding, and their combination by stage (0-1, 2, and 3+) was not heterogeneous (data not shown).\",\n",
            "        \"discussion\": \"DISCUSSION In our study, individual risk factors such as childbirth and breastfeeding were associated with a reduced risk for BC. The effect of giving birth on BC risk reduction was heterogeneous between premenopausal and postmenopausal women, and a much stronger association was found in postmenopausal women (childbirths of 3+ vs. 0: 20 and 60% risk reduction in premenopausal and postmenopausal women, respectively). A breastfeeding duration of 1-12 months, relative to never breastfeeding, was associated with a strongly decreased BC risk in premenopausal women, but no such association was found in postmenopausal women. The longer breastfeeding durations of 13-24 and 25+ months were associated with a further decrease in the risk for BC, but the effect was not different by menopausal status. BC risk reduction by 3 or more childbirths was strongly observed in HR+ BC, as well as in the luminal A subtype. By contrast, the reduction of BC risk by a longer breastfeeding period did not vary according to HR or pathological subtype. The combined effect of childbirth and breastfeeding on BC risk reduction showed a multiplicative synergistic effect. The combination of both factors had a heterogeneous association between premenopausal and postmenopausal women who had breastfed for 25+ months or had 3+ childbirths. Previous studies on the individual effect of childbirth on BC risk showed a significantly decreased risk, with a much larger reduction in postmenopausal women [ 1 , 3 , 22 ]. A consistent relationship between breastfeeding and postmenopausal status has not been found [ 3 , 4 , 6 , 8 ] The recent meta-analyses of the relationship between breastfeeding and BC risk confirmed an inverse association with breastfeeding itself and found that a greater reduction of BC risk was associated with longer breastfeeding [ 8 , 16 , 23 ]. Few previous studies have investigated the heterogeneous associations of childbirth and breastfeeding with BC risk, and their results were controversial. Two case-control studies of breastfeeding itself and longer breastfeeding periods showed a reduced BC risk in only premenopausal women [ 5 , 7 ]. Those previous results are similar to our results. However, our results showed some differences according to breastfeeding duration. When the breastfeeding period was 1-12 months, only premenopausal women had a risk reduction in BC, which was a heterogeneous association. A previous cohort study of childbirth showed a significant reduction of BC risk only in postmenopausal women [ 24 ], whereas our findings also showed a heterogeneously lower risk for premenopausal women than postmenopausal women, but a reduction in BC risk associated with 3 or more childbirths was observed in all women, regardless of menopausal status. Our study reported more detailed results than previous studies because the number of subjects in our study is the largest among the studies that have been conducted in this field. Our findings of a heterogeneous association with childhood according to HR status (a stronger association in HR+ BC) and a non-heterogeneous association of breastfeeding according to HR status have consistently been observed in individual studies (for childbirth: [ 25 - 27 ]; for breastfeeding: [ 27 ]) and review papers [ 3 , 28 ]. However, our findings of a heterogeneous association with childbirth according to the pathological subtype and a non-heterogeneous association with breastfeeding do not correspond with those of previous studies. In our study, a greater reduction in BC risk was shown with more childbirths in all 4 subtypes, with a particularly strong trend for the luminal A subtype, whereas previous studies reported inconsistent results, such as a heterogeneous association between the luminal A and B subtypes (inverse association) and the HER2/TNBC subtypes (no significant association) [ 26 , 29 ] or a non-heterogeneous association with all subtypes [ 27 ]. The finding that BC risk was reduced with breastfeeding in all 4 subtypes and non-heterogeneously associated with the 4 subtypes is not concordant with the findings of previous studies [ 26 - 30 ]. Findings of the lowest risk in the luminal A subtype relative to other subtypes agree with our observation of a strongly lower risk in HR+ BC than in HR\\u2212 BC. Previous studies showed various results, such as a significantly reduced risk of luminal A and B BC [ 30 ], a reduced risk of TNBC [ 26 , 28 , 30 ], a non-significant association with HER2-expressing BC [ 26 , 28 ], and a non-significant association with all subtypes [ 27 ]. Although a combined effect of breastfeeding and giving birth on BC risk reduction was not reported in previous studies, a synergistic multiplicative effect was presented in this paper, as the BC risk reduction was higher with an increase in both factors (i.e., more childbirths and a longer breastfeeding duration). It is biologically plausible that the presence of both factors induces a much shorter duration of exposure to hormones, in accordance with the estrogen hypothesis, in addition to the biochemical mechanisms associated with each protective factor [ 14 , 15 ]. In particular, this combined effect was not different across menopausal status, HR, or pathological subtype. This study has several limitations. First, lifestyle factors well known as risk factors for BC (such as physical exercise [ 31 ], smoking and drinking [ 32 , 33 ], and food intake [ 34 - 37 ]) were not included. Second, it can be suspected that there is a recall bias in women\\u2019s response of breastfeeding duration because the response to the total breastfeeding period of women was concentrated in a particular month (such 12-month, 24-month, 36-month by an interval of 12-month). The strength of our research is that it incorporated large-scale data from across the country. In Korea, a total of 115 928 new cases of BC occurred during the 9 years from 2004 to 2012 [ 38 ]. About 11% of these epidemiologic data were used in this study. In addition, participants were matched by age and year of enrollment to improve the comparability by controlling for the age effect and the birth effect. In conclusion, the effect of childbirth on BC risk reduction was heterogeneous according to menopausal status, and the BC risk reduction associated with 3 or more childbirths was much stronger in postmenopausal women than in premenopausal women. By contrast, longer breastfeeding durations of 13+ months were non-heterogeneously associated with menopausal status, but only women with a breastfeeding duration of 1-12 months showed heterogeneity according to menopausal status, with only premenopausal women showing a significant BC risk reduction. The association of breastfeeding and its duration on BC risk was not different according to HR status or pathological subtype. More childbirths and longer breastfeeding exerted a combined effect on BC risk reduction. The heterogeneous association of the combination of breastfeeding (1-12 months) and any childbirths on BC risk by menopausal status was only due to the heterogeneous association of breastfeeding according to menopause, not the combined effect. The combined effects of women\\u2019s childbirth and breastfeeding on BC risk reduction are summarized, focusing on premenopausal women because birth and breastfeeding are important life events of premenopausal women who can only be modifiable in premenopausal women. Compared to BC risk in women with no-childbirth and never-breastfeeding, a woman who have at least one birth can only benefit from BC risk reduction if she experiences breastfeeding, while she does not do breastfeeding she does not have risk reduction effect. The BC risk is decreased by 10-20% for women with 2+ childbirth but never-breastfeeding, about 30% for women with 1 childbirth but breastfeeding for at least 1 months, 40-50% for women with 2+ childbirth but breastfeeding for 1-12 months, and >60% for women with 2+ childbirth but breastfeeding for \\u226513 months. In the view of the population-attributable fraction (PAF), two previous Korean papers suggest that the PAF of breastfeeding and/or pregnancy (including age at first birth) is estimated to be 7-11% for Korean women BC [ 30 , 39 ]. If women with <2 childbirths and <13 months of breastfeeding are considered to be the high-risk group, the PAF for BC prevention through interventions in the high-risk group would be estimated to be about 7.5% according to our data, which is similar to the prior PAF value estimated in prior studies. The PAF (7.5%) associated with the high-risk group defined by the combination of childbirth and breastfeeding period accounts for about one-fourth of the PAF (33%) from all modifiable factors [ 30 ]. Since the fertility rate in Korea has rapidly decreased and still not recovered (6.0 births per fertile woman in 1960; 1.08 in 2005 [the lowest]; 1.17 in 2016; Korean Statistical Information Service [ http://kosis.kr ]), the relative impact of these factors on breast cancer prevention seems to be more important than in the past. Although low childbirth and/or the combination of low childbirth and short breastfeeding can considered to make a partial contribution to BC in Korea, this paper is expected to serve as scientific evidence to support national cancer management policies for BC. Our findings regarding BC risk modification by the combination of childbirth and breastfeeding and the full evaluation of heterogeneous associations by menopausal status, HR, and pathological subtypes are the first such findings worldwide, to the best of our knowledge. This study may provide a scientific basis for national cancer management policies involving the promotion of childbirth and breastfeeding, although childbirth and breastfeeding contribute only partially to the development of BC.\",\n",
            "        \"conclusion\": null\n",
            "    },\n",
            "    {\n",
            "        \"pmcid\": \"PMC5839230\",\n",
            "        \"abstract\": null,\n",
            "        \"intro\": \"Introduction Efforts to evaluate adult attachment have led to the production of a vast number of tools, both narrative and self-report. These instruments examine a diversity of contents (early attachment experience, current representations of attachment, romantic relationships, representations of caregiving) and are used in different fields of research: self-report tools have mainly been developed in social research contexts, and are generally administered to large samples (Crowell et al., 2008 ), while narrative instruments are more commonly adopted in developmental research, which are generally carried out on smaller samples and use meta-analysis to test their validity (van IJzendoorn, 1995 ; Hesse, 2008 ; Bakermans-Kranenburg and van IJzendoorn, 2009 ; Lucassen et al., 2011 ). Finally, different tools evaluate different levels of conscious awareness in relation to attachment representations: typically, narrative instruments such as the Adult Attachment Interview (AAI: George et al., 1984 , 1985 , 1996 ; Main and Goldwin, 1984 ; Main et al., 2002 ) facilitate evaluation of non-conscious aspects of attachment via the external coding of responses, while self-report questionnaires are centered on conscious ideas about attachment and relationships, and are generally not highly correlated with the AAI or other narrative instruments (Crowell et al., 1999 , 2008 ). The AAI is generally viewed as the benchmark tool for evaluations of adult attachment, although it is difficult and time consuming to administer, like all narrative instruments. Self-report tools are easier and more economical to use, but can be over-simplistic and therefore fail to access distinctive features of the different attachment styles, especially in relation to unconscious representations as in the case of dismissing attachment (Maier et al., 2004 ). The literature reflects general agreement on the fact that both types of measure are useful, each serving to evaluate different kinds of construct (Crowell et al., 2008 ): narrative instruments evaluate the \\u201cstate of mind with respect to attachment,\\u201d based on infancy experience, while self-report instruments evaluate conscious representations of attachment in respect to different contents, that is to say, \\u201cattachment style\\u201d emerging from representations of early experience (Bartholomew and Horowitz, 1991 ). Despite their limitations, self-report instruments remain a valuable resource for clinical practice and research, particularly when large samples are involved. The aim of our study is to evaluate the factorial structure of the CA-MIR (CArtes-Mod\\u00e8les Individuels de Relations: Pierrehumbert et al., 1996 ), a self-report instrument designed to tap into the attachment styles of adult, that can be used both as a traditional self-report questionnaire (Likert format), and as a Q-Sort instrument. In both versions, items are written on a card, and respondents are asked to put the cards in five piles, ordered from the less pertinent (\\u201cnot at all true for me\\u201d) to the more pertinent (\\u201cvery true for me\\u201d). In the Q-Sort version items are to be placed according to a forced distribution, resembling a normal distribution, while in the Likert format, the distribution is free. The CA-MIR is a tool that shares aspects of both the above-mention research lines. In fact, one of its strengths lies in the opportunity it gives to be administered in a Q-Sort version that can solve at least some social desirability problems and be more effective in capturing even unaware aspects of individual representations. Nevertheless, in this study, only the Likert format is considered, since responses need to be independent in order to apply factor analysis tools. Informed by attachment theory, the CA-MIR assumes that a self-and-other model drives personal relationships throughout life and adulthood: this model is proposed to have been generated by early experience with caregivers, although it may be modified by subsequent relational experiences. The questionnaire is intended for use with people of different ages (from adolescence to old age), and of both genders, regardless of family background and current family circumstances: parents or currently childless; currently or no longer living with family of origin; single or with a partner; coming from families with two parents, a single parent, or an adoptive family; with or without siblings. The CA-MIR comprises 72 items designed to explore three different areas of representation: the present (questions about respondents' relationships with their current family unit), the past (questions concerning aspects of respondents' past experience with both parents, or with one parent in particular) and their state of mind (questions on respondents' current beliefs about the attachment relationship they had with their parents while growing up, as well as their semantic representations of parenting and the emotional needs of children and adults in general\\u2014the state of mind referred to here is that of which the respondent is consciously aware). Representation of Past experience and State of Mind are two different concepts, because the first concerns the conscious representations of the past experience (namely the memories, the explicit narrative), the second the representations (even unconscious) about attachment, which derive but do not coincide with the past experience, as evidenced by the presence of earned secure adults (Hesse, 2008 ). Present representations, on the contrary, refers to the present experience, and are characterized by the actual experience with significant persons in the life of the subject (partners, children, parents). For a summary outline of the questionnaire's theoretical structure and some example of items, see Table 1 . Table 1 Theoretical structure of the instrument and sample items. Past Present \\u201cState of Mind\\u201d Preoccupied attachment (E) A Parental intrusion B Preoccupation with family C Resentment about childhood experience As a child I was afraid of being abandoned. (39) When I go away from those who are close to me, I feel ill at ease. (56) I feel that I did not have the opportunity to assert myself in the environment I grew up in. (52) Autonomous attachment (F) D Parental support E Family support F Gratitude for support Even if it is not true, I feel that I had the best parents in the world. (53) I trust those who are close to me. (36) I enjoy thinking about my childhood. (25) Dismissing attachment (DS) G Parental unavailability H Family distance I Resentment of childhood rejection When I was a child, my wishes did not count much for the adults around me. (29) I hate feeling dependent on others. (12) Every time I try to think of my parents' good sides, I end up recalling their bad sides. (60) Unresolved attachment (U) J Parent-related trauma K Blocked memories As a child, I was afraid of my parents. (33) I have difficulty accurately recalling childhood events. (51) Structuration L Parental abdication M Regard for authority When I was a child, I would set the grownups against one another to get what I wanted. (44) In a family, it is very important that there be respect for the parents. (8) For each of the three areas of representation, there are items characterizing three different attachment styles, namely autonomous, preoccupied , and dismissing , yielding a total of nine sub-scales. In addition, the groups of items concerning respondents' past experience of family relationships and their current state of mind both include two further sets of questions, related to unresolved attachment and respondents' beliefs about family relationships, respectively. This yields a further four sub-scales: two scales involve the presence of bereavement or traumas unresolved , an additional category that is attributed secondarily to the three main categories, as for the AAI. The two scales on structuration concern the ideas that people have regarding the education of the children and the authority: the educational principles experienced in the past (in the family of origin) and their own beliefs about education and authority. Although these last two scales are not envisaged as directly linked to attachment profiles, they offer useful additional information that can help to build up clinical profiles, e.g., when evaluating parenting skills. In previous studies that adopted the Q-Sort format, the instrument showed good convergent and discriminant validity 1 ; however, the factor structure was only partially reproduced in the Chilean validation study carried out by Garrido et al. ( 2009 ). Garrido and colleagues conducted a principal components analysis, identifying eight components: two of these exactly replicated the original scales labeled Preoccupation with Family (Preoccupied-Present) and Family Support (Autonomous-Present), while three factors partially replicated the original scales named: Parental Abdication (Structuration-Past), Blocked Memories (Unresolved-\\u201cState of mind\\u201d) and Parental Intrusion (Preoccupied -Past). Considering the complexity of the CA-MIR theoretical model and the only partially overlapping results of the analyses just reviewed, additional analysis of the Likert-format items is necessary. The aim of the current study was to contribute to clarifying the factorial structure of the instrument, by performing confirmatory factor analysis (CFA) to directly verify the goodness-of-fit of the original 13-factor structure and assessing the concurrent and discriminant validity of the CA-MIR latent dimensions. According to the attachment literature and the item content, the following expectations were formulated about the latent dimensions' correlation. Due to a certain persistence of relational modalities across the life span, a moderate positive correlation between Past, Present , and State of Mind dimensions of Autonomous, Preoccupied, and Dismissing styles was expected; at the same time, these relations should have been stronger for Past and State of Mind dimensions because in both cases items deal with respondents' childhood experience. Autonomous style dimensions were expected to be negatively correlated with the dimensions of preoccupied and dismissing attachment styles, with weaker correlations when involving Present Autonomous style dimension because of the so-called earned-secure attachment patterns (Hesse, 2008 ). Moreover, Dismissing and Preoccupied dimensions were attended to be unrelated and Blocked Memories scale was expected to correlate with none of the Past dimensions since it deals with the lack in memories. Finally, we expected to find a moderate negative correlation between Autonomous style dimensions and the Past Structuration dimension related to the absence of authoritative parenting when respondent was a child: as highlighted by literature, \\u201csensitive discipline\\u201d (van Zeijl et al., 2006 ) is a key aspect of secure experience in childhood.\",\n",
            "        \"method\": \"Methods Participants We recruited 979 participants during the spring and fall of 2005, using a snowball sampling method. About one third of the sample (30.1%) was made up of people related to each other, such as siblings, partners, parents, and children. To reduce the potentially distorting effect of family ties among participants, we removed one member from each interrelated pair of respondents, two members from each interrelated group of three or four and three members from each interrelated group of five 2 . The percentage of interrelated respondents decreased to 0.9%, while sample size was reduced to 827 individuals (see Tables 2.1 , 2.2 ). Table 2.1 Sample ( n = 827) compared to the Italian Population. Sample percentage Italian popolation percentage Age ISTAT, 2006c 15-19 3.5 5.7 20-29 47.0 13.8 30-39 11.9 18.7 40-49 20.3 17.4 50-59 14.9 15.1 60 and over 2.4 29.3 Gender ISTAT, 2006c Percentage of females 61.2 51.9 Highest level of education completed ISTAT, 2006a Primary and middle school 23.1 64.3 High school 59.6 26.5 University 17.3 9.1 Occupational status ISTAT, 2006a Self-employed 13.2 11.9 In third party employment 45.8 32.7 Housewife 7.1 17.7 Student 29.3 8.9 Retired 3.1 21.9 Other (seeking employment and other categories of unemployed) 1.5 7.0 Marital status ISTAT, 2006b Single 51.0 43.9 Cohabiting 5.6 Married 36.8 52.4 Divorced 5.2 1.2 Widowed 1.5 2.5 Parenthood status (percentage, n 405 selected to compare with Italian population) ISTAT, 2004 Having one or more children: 79.0 74.3 Number of children: 1 43.4 47.8 2 46.3 41.3 3 or more 10.3 10.9 Table 2.2 Other sample characteristics (comparison data for the Italian population not available). Sample percentage Age (min-max, M, SD ) 15-81; 34.2; 13.4 Birthplace Northwestern Italy 80.2 Other 19.8 Parenthood Yes 40.4 No 59.6 Parents alive Mother 84.2 Father 72.1 Issues faced during development Early bereavement 1.0 Parental separation/divorce 1.3 Social disadvantage 0.2 Childhood adoption 0.1 Child of single parent 0.2 Parent-child separation 0.2 Total 3.1 Recent issues Illness or death of a loved one 1.7 Illness 0.1 Trauma 0.4 Separation/divorce 0.1 Psychosocial issues 0.4 Total 2.7 Most of the respondents were from Northwestern Italy (80.2%). Their ages ranged from 15 to 81 years (M = 34 years; SD = 13 years). Table 2.1 reports the sample's demographic characteristics, comparing them to those of the general Italian population: the 20\\u201329 years age group was over-represented in the current sample (47.0%, as opposed to 13.8% of the overall Italian population) while the over 60s segment of the population was under-represented (2.4% in the current sample vs. 29.3% of all Italians; ISTAT, 2006c ). The percentage of women was 61.2% (somewhat higher than in the Italian population: 51.9%). Compared to the general population (ISTAT, 2006a ), the sample was better educated (more subjects with high school diplomas and university degrees), and contained a higher percentage of students and workers in third party employment; vice versa, those who had not completed their secondary education, as well as the categories of housewife, retired worker, job-seeker, and others outside of the active work-force were under-represented. Fifty-one percent of the sample were single (vs. the lower level of 43.9% in the Italian population); 5.6% cohabited with a partner, 36.8% were married (vs. the higher level of 52.4% in the general population); the percentage of divorcees was higher than in the Italian population while the percentage of widowed persons was lower (ISTAT, 2006b ). If we consider the family unit ( n = 405) 3 , levels of parenthood and number of children were comparable to the general population (ISTAT, 2004 ). Table 2.2 reports the percentage of participants whose parents were still alive and the percentages of those reporting various types of recent and childhood challenges: in relation to these aspects, there was no general population data available for comparison. With a view to test and cross-validate the factor structure of CA-MIR, we randomly divided the sample into two sub-samples 4 : Sample 1 ( N = 419) and Sample 2 ( N = 408). Procedure The present study involved human beings in a data collection process aimed at gaining information about attitudes related to close interpersonal relationships and socio-demographical data from people aged 15 years old or more. The research conformed to the Codice Etico AIP (ethical code for research in psychology, Italian Psychological Association) and the provisions of the Italian laws on privacy and data protection (L. 196/2003). The participants volunteered to participate in the research, and signed an informed consent form in which they agreed to anonymously complete a Q-sort questionnaire and allow the researchers to use the data for scientific purposes. In case of minors, also parents' consent was collected. The questionnaires were administered by undergraduate students who had received 20 h of ad hoc training. Each student administered five questionnaires in both Likert and forced Q-Sort formats, after obtaining participants' informed consent. The administration and coding procedure adopted were those recommended by the authors of the original instrument (Pierrehumbert, 2004 ). In the present study, only the Likert format data were analyzed. Instruments CA-MIR questionnaire The first author translated the questionnaire from French to Italian, discussing and revising the translation with the Swiss authors and with a colleague native speaker both in Italian and French, and producing Italian versions of the instrument for both male and female respondents 5 . The 72 items were scored on a 5-point Likert scale, from \\u201cnot at all true for me\\u201d to \\u201cvery true for me.\\u201d The Cronbach's Alpha for each of the 13 theoretical scale scores were the following. Parental intrusion (Preoccupied-Past, 6 items):.59; Preoccupation with family (Preoccupied-Present, 6 items):.70; Resentment about childhood experiences (Preoccupied-State of Mind, 6 items):.78; Parental support (Autonomous-Past, 6 items):.80; Family support (Autonomous-Present, 6 items):.72; Gratitude for support (Autonomous-State of Mind, 6 items):.75; Parental unavailability (Dismissing-Past, 6 items):.79; Family distance (Dismissing -Present, 3 items):.37; Resentment of childhood rejection (Dismissing -State of Mind, 6 items):.78; Parent-related trauma (Unresolved-Past, 6 items):.75; Blocked memories (Unresolved-State of Mind, 3 items):.67; Parental abdication (Structuration-Past, 6 items):.65; Regard for authority (Structuration-State of Mind, 6 items):.59. Socio-demographic information Respondents' socio-demographic data was collected concurrently, and each participant's family ties with other respondents, such as siblings, partners, parents, or children, were recorded. In addition, we collected information regarding non-normative life experiences that could potentially influence participants' attachment representations (e.g., juvenile grief, illness, divorce). Data analysis On the data of Sample 1, the original 13-factor structure of CA-MIR was tested by a confirmatory factor model in which all cross-loadings were constrained to be zero, i.e., by means of an independent clusters model (ICM-CFA, Marsh et al., 2009 ). As reported in literature, ICM-CFA models are very restrictive and can produce biased estimates of factor correlations (Asparouhov and Muth\\u00e9n, 2009 ; Marsh et al., 2009 , 2011 , 2014 ; Morin et al., 2016 ). Non-target loadings motivated by substantive theory or by item formulation are typically present in multidimensional instruments and forcing them to zero can lead to overestimated factor correlations. This is particularly true when instruments assessing conceptually related constructs are involved (Morin et al., 2016 ). As concern the CA-MIR, several scales assess related constructs and we expected that some items exhibit \\u201cconstruct relevant psychometric multidimensionality\\u201d as defined by Morin et al. ( 2016 ). Some items of the Autonomous style scales could be weak reverse indicators of non-autonomous styles and the opposite, or some indicators of \\u201cPast\\u201d scales could be weak indicators of the \\u201cState of mind\\u201d scales and the opposite. Constraining these secondary loadings to be zero artificially increases the correlation between the corresponding factors. According to literature, the ICM-CFA solution need to be compared with the corresponding explorative factor model, following the exploratory structural equation modeling approach (ESEM, Asparouhov and Muth\\u00e9n, 2009 ; Marsh et al., 2014 ) to check for the presence of overestimated factor correlation. In the present work, it was not possible to apply the ESEM approach because the high number of latent factors (13) would have required the estimation of more parameters than the number of participants (930 free parameters); and neither it was feasible to identify the subset of items affected by multidimensionality on theoretical grounds. We thus performed an exploratory factor analysis with target rotation and Minres as a method of estimation, followed by a CFA in which all the EFA secondary loadings >0.30, alongside the target loadings, were estimated (called complex-CFA model in the following) 6 . To identify the latent variables scale, factor variances were set to 1. Given that the data violated the multinormality condition 7 , we used the Maximum Likelihood method (ML) to estimate the parameters, correcting the chi-square and standard errors (Satorra and Bentler, 1994 ). The following cut-off criteria were used to establish the model's goodness of fit: RMSEA < 0.08; CFI > 0.95; SRMR < 0.08 (Browne and Cudeck, 1993 ; Hu and Bentler, 1995 , 1999 ); the Satorra-Bentler scaled difference \\u03c7 2 test (SBDiff, Satorra and Bentler, 2001 ) was used to compare the fit of nested CFA models. In the second sample, we cross-validated the model emerged from the analyses of the data from the first sample. All analyses were conducted using SPSS 24; R 3.4; Prelis and LISREL 8.72 (J\\u00f6reskog and S\\u00f6rbom, 1996a , b ).\",\n",
            "        \"subjects\": null,\n",
            "        \"results\": \"Results Factor analyses on sample 1 The theoretical 13 factor model showed good overall fit to the data as all the fit indices, RMSEA, CFI, and SRMR, fulfiled the cut-off values (Table 3 ). The loadings were all statistically significant ( p < 0.01) and only five of them were below 0.30. Several factors were highly correlated, in particular correlation was > 0.90 (in absolute value) for scales A and C, C and I, D and F, F and I, G and I and 7 correlation values were in the range 0.80\\u20130.90. Table 3 Goodness of fit of the CFA models on Sample 1. Model SB Chi-square DF SBDiff DF p RMSEA (CI) CFI SRMR 13 factor ICM-CFA 4186.8 2406 0.042 (0.040\\u20130.044) 0.95 0.067 13 factor Complex-CFA model 3635.8 2376 389.9 30 <0.0001 0.036 (0.033\\u20130.038) 0.96 0.063 12 factor Complex-CFA model 3694.9 2392 54.5 16 <0.0001 0.036 (0.034\\u20130.038) 0.96 0.064 Based on the results of the explorative target rotated factor analysis reported in Appendix , a second CFA model that included secondary loadings >0.30 (in absolute value) was performed. As depicted in Table 3 , this second model outperformed the previous one, and the Satorra Bentler scaled difference \\u03c7 2 test (SBDiff) was statistically significant. Moreover, the number of factor correlations in absolute value above 0.80 decreased from 12 to 4, with maximum values for scales D and F (0.88), and scales G and I (0.86). In the light of these results, a third model with 12 latent variables in which scales D and F belongs to the same latent factor was estimated, in order to assess whether the two highest correlated latent factors could be collapsed. As shown in Table 3 , the worsening of model fit passing from 13 to 12 latent factors was statistically significant, suggesting retaining the 13 complex-CFA model. Cross-validation on sample 2 In order to gain further understanding of, and further clarify, the factor structure of the CA-MIR, we replicated the complex-CFA model in the second sample. The overall fit statistics were good: SB-Chi-Square (2376) = 4130.6, p < 0.001, RMSEA = 0.043; 90% CI = 0.040\\u20130.045, CFI = 0.96, SRMR = 0.067. As summarized in Table 4 , the large majority of the items loaded on the target factor with values >0.30. Table 4 Complex-CFA model: Standardized parameter estimates (Sample 2, N = 408). Preoccupied Autonomous Dismissing Unresolved Structuration A B C D E F G H I J K L M 7 0.18 - 0.30 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 35 0.28 \\u2013 0.46 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 39 0.15 \\u2013 \\u2013 \\u2013 \\u2013 \\u22120.39 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 48 0.36 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u22120.07 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 54 0.72 \\u2013 \\u2013 0.03 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 62 0.51 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 2 \\u2013 \\u2013 0.68 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 26 \\u2013 \\u2013 0.25 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 0.41 \\u2013 \\u2013 \\u2013 \\u2013 41 \\u2013 \\u2013 0.53 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 52 \\u2013 \\u2013 0.63 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 55 \\u2013 \\u2013 0.71 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 64 0.34 \\u2013 0.46 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 20 \\u2013 0.51 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 22 \\u2013 0.56 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2014 \\u2013 \\u2013 32 \\u2013 0.47 \\u2013 \\u2013 \\u2013 \\u2013 \\u22120.21 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 56 \\u2013 0.56 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 68 \\u2013 0.59 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 72 \\u2013 0.56 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 9 \\u2013 \\u2013 \\u2013 0.64 \\u2013 \\u2013 \\u22120.14 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 21 \\u2013 \\u2013 \\u2013 0.56 \\u2013 0.11 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 40 \\u2013 \\u2013 \\u22120.30 0.37 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 53 \\u2013 \\u2013 \\u2013 0.32 \\u2013 0.25 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 58 \\u2013 \\u2013 \\u2013 0.55 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 66 \\u2013 \\u2013 \\u2013 0.46 \\u2013 \\u2013 \\u22120.30 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 10 \\u2013 \\u2013 \\u2013 0.27 \\u2013 0.14 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 11 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 0.54 \\u2013 \\u2013 \\u2013 \\u22120.37 \\u2013 \\u2013 \\u2013 19 \\u2013 \\u2013 \\u2013 0.38 \\u2013 0.37 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 25 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 0.65 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 28 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 0.51 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 6 \\u2013 \\u2013 \\u2013 \\u2013 0.61 0.12 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 1 \\u2013 \\u2013 \\u2013 \\u2013 0.53 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 4 \\u2013 \\u2013 \\u2013 \\u2013 0.55 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 0.24 \\u2013 18 \\u2013 \\u2013 \\u2013 \\u2013 0.14 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 27 \\u2013 \\u2013 \\u2013 \\u2013 0.63 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 36 \\u2013 \\u2013 \\u2013 \\u2013 0.73 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 69 \\u2013 \\u2013 \\u2013 \\u2013 0.65 \\u2013 \\u2013 \\u2013 0.00 \\u2013 \\u2013 \\u2013 \\u2013 15 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 0.03 \\u2013 \\u2013 0.73 \\u2013 \\u2013 \\u2013 29 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 0.75 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 30 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 0.63 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 31 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 0.27 \\u2013 \\u2013 \\u2013 \\u2013 0.41 \\u2013 38 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 0.52 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 71 \\u2013 \\u2013 \\u2013 \\u22120.42 \\u2013 \\u2013 0.35 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 13 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 0.62 \\u2013 \\u2013 \\u2013 \\u2013 47 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 0.58 \\u2013 \\u2013 \\u2013 \\u2013 50 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 0.42 \\u2013 \\u2013 \\u2013 \\u2013 57 \\u2013 \\u2013 0.18 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 0.53 \\u2013 \\u2013 \\u2013 \\u2013 60 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 0.63 \\u2013 \\u2013 0.18 \\u2013 67 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u22120.01 \\u2013 0.71 \\u2013 \\u2013 \\u2013 \\u2013 12 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 0.79 \\u22120.33 \\u2013 \\u2013 \\u2013 \\u2013 14 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 0.56 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 17 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 0.20 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 3 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 0.48 \\u2013 \\u2013 \\u2013 33 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 0.61 \\u2013 \\u2013 \\u2013 45 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 0.62 \\u2013 \\u2013 \\u2013 59 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 0.69 \\u2013 \\u2013 \\u2013 61 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 0.51 \\u22120.02 \\u2013 0.16 \\u2013 63 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 0.61 \\u2013 \\u2013 \\u2013 37 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 0.66 \\u2013 \\u2013 46 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 0.48 \\u2013 \\u2013 51 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 0.71 \\u2013 \\u2013 5 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 0.57 \\u2013 16 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 0.29 \\u2013 0.53 \\u2013 23 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u22120.05 \\u2013 \\u2013 \\u2013 \\u2013 0.60 \\u2013 42 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 0.38 \\u2013 44 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 0.30 \\u2013 70 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 0.47 \\u2013 8 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 0.67 24 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 0.16 34 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 0.53 43 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 0.46 49 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 0.51 65 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 \\u2013 0.65 A, Parental intrusion; B, Preoccupation with family; C, Resentment about childhood experiences; D, Parental support; E, Family support; F, Gratitude for support; G, Parental unavailability; H, Family distance; I, Resentment of childhood rejection; J, Parent-related trauma K, Blocked memories; L, Parental abdication; M, Regard for authority . Statistically significant (p < 0.05) factor loadings are highlighted in bold . Items 7, 35, 39 resulted to be poor indicators of factor A (Preoccupied-Past); their loadings, albeit statistically significant, were low in value. More specifically, item 7 (\\u201cI' d like my children to be more autonomous than I have been\\u201d) and item 35 (\\u201cMy parents didn 't really realize that a child who is growing up needs to have a life of his own\\u201d) seemed to be indicators of the State of Mind dimension instead of the Past, whereas item 39 (\\u201cAs a child, I was scared of being abandoned\\u201d) loaded heavily (and negatively) on Autonomous-State of Mind factor. Items 26 (\\u201cAs a teenager, no one around me has ever really understood my worries\\u201d) and 64 (\\u201cIn my family, we lived isolated from the rest of the world\\u201d) loaded both on the expected factor (scale C), and on a second factor: factor A (item 26), and factor I (item 64). As regard autonomy factors (D, E, and F), item 40 (\\u201cAs a child, I was encouraged to share my feelings\\u201d) loaded also on scale C, but with negative sign, and item 53 (\\u201cEven if it's not true, I have the feeling l've had the best parents in the world\\u201d) showed a secondary positive loading on factor F. Moreover, item 18 (\\u201cI spend a lot of time talking to the people who are close to me\\u201d) was a poor indicator of factor E (Autonomous-Present), without showing relevant secondary loadings, and item 10 (\\u201cI believe l've known how to give back to my parents the love they've given me\\u201d) was more an indicator of the Past than of the State of Mind dimension. Moving to the Dismissing dimensions (factors G, H, and I), item 15 (\\u201cWhen I was a child, the people close to me were often impatient and irritable\\u201d) resulted to be related to the J factor instead of the G factor and item 31 (\\u201cWhen I was a child, it was difficult for us to make decisions in the family\\u201d) to factor L. As regard item 12 (\\u201cI hate feeling I depend on others\\u201d), an unexpected negative secondary loading on factor I was observed. In the remaining four scales (factors J, K, L, and M), only two items showed poor fit: item 61 (\\u201cI have the feeling I was a rejected child\\u201d), that resulted to be an indicator of factor I instead of factor J, and item 24 (\\u201cAdults have to control their emotions with their children, whether it concerns pleasure, love or anger\\u201d), that emerged as a weak indicator of scale M without showing any secondary loadings. In summary, four items (17, 18, 24, and 44) were weak indicators of the intended factor and did not load on secondary factors. Others four items (10, 15, 61, and 6) did not load on the expected factor but on a plausible related factor; and finally, five items (64, 11, 19, 71, and 12) resulted in a substantial loading (>0.30) on two factors. As reported in Table 5 , six correlations were >0.80 in absolute value, and two of them were >0.90 (the positive one between factors D and F and the negative one involving scale I and D). The signs of the correlations were in line with the theoretical expectation: The Past and State of Mind dimensions of each attachment style were positively associated with one another; Autonomous Attachment was negatively associated with Preoccupied and Dismissing State of mind dimensions, while these last were positively associated with one another. Only for the Autonomous Attachment style there were a strong correlation with the Present dimension (0.69 and 0.70 for Past and State of Mind dimensions, respectively). Table 5 Complex-CFA model: factors correlations (Sample 2, N = 408). Preoccupied Autonomous Dismissing Unresolved Structuration A B C D E F G H I J K L M A 1 B 0.20 1 C 0.68 0.20 1 D \\u22120.36 0.12 \\u22120.65 1 E \\u22120.37 0.12 \\u22120.52 0.69 1 F \\u22120.38 0.09 \\u22120.56 0.90 0.70 1 G 0.46 0.20 0.89 \\u22120.79 \\u22120.51 \\u22120.63 1 H 0.19 \\u22120.01 0.44 \\u22120.41 \\u22120.43 \\u22120.35 0.54 1 I 0.56 0.03 0.78 \\u22120.93 \\u22120.73 \\u22120.86 0.87 0.54 1 J 0.59 0.04 0.66 \\u22120.81 \\u22120.55 \\u22120.62 0.76 0.38 0.85 1 K 0.04 0.13 0.27 \\u22120.14 \\u22120.14 \\u22120.26 0.32 0.10 0.15 \\u22120.01 1 L 0.15 0.14 0.36 \\u22120.42 \\u22120.46 \\u22120.63 0.53 0.27 0.51 0.38 0.23 1 M \\u22120.22 0.25 \\u22120.13 0.29 0.41 0.54 \\u22120.18 0.14 \\u22120.23 \\u22120.23 \\u22120.10 \\u22120.36 1 A, Parental intrusion; B, Preoccupation with family; C, Resentment about childhood experiences; D, Parental support; E, Family support; F, Gratitude for support; G, Parental unavailability; H, Family distance; I, Resentment of childhood rejection; J, Parent\\u2013related trauma K, Blocked memories; L, Parental abdication; M, Regard for authority. Values statistically significant at p < 0.05 are in bold . With regard to Scales J, K, L, and M, the only correlations above 0.30 were that between J and L (0.38), and L and M (\\u22120.36). Factors J and L were also strongly correlated with the attachment style scales (especially the Autonomous and Dismissing scales), whereas scales K and M were those least strongly correlated with all the other scales.\",\n",
            "        \"discussion\": \"Discussion Factorial structure of the CA-MIR In the evaluation of the factorial structure of the CA-MIR, both the ICM-CFA model and a model with the presence of several secondary loadings were estimated. The former was aimed at directly assessing the goodness of fit with the Italian data of the 13-factor theoretical model; the latter, here called complex-CFA model, was considered to eventually take into account the presence of item multidimensionality (Marsh et al., 2014 ; Morin et al., 2016 ). The comparison of the two above-mentioned models in the first sample showed that several secondary loadings were present. The same model, replicated in the second sample, confirmed the presence of some multidimensionality: in a few circumstances, secondary loadings were greater than the loadings in the expected factors and others few items showed not negligible cross-loadings. The content of these unfitting items justifies these findings: in most cases, the items ambiguity was connected to the distinction between Past and State of mind scales of the same attachment style or to a negative relation with a scale of a different attachment style (indicators of Autonomous style with negative loadings on one of the unsecure styles, and vice versa). Four items emerged as poor indicators of the expected latent dimension, without exhibiting secondary loadings. One of these, item 17, belongs to the Family distance scale and its misfit could be responsible for the very low internal consistency of the scale scores. The internal consistency of each of the remaining scales was satisfactory and comparable to that found in the original Swiss validation study. Only Family distance scale displayed an unsatisfactory value (0.37), but this had also been the case for the Chilean sample (0.33, Garrido at al., 2009) and the Swiss sample (0.48, Pierrehumbert et al., 1996 ). The CFA approach allowed us to directly test and confirm the 13-factor model and contributes to advancing our understanding of why exploratory tool like that applied in Garrido et al. ( 2009 ) could tend to portray the CA-MIR as measuring a lower number of dimensions. The extent of the inter-correlations among the 13 scales, the presence of items that load negatively on secondary factors and of some misplaced items could be the reason why in the Chilean study only eight dimensions were obtained. Convergent and discriminant validity of the 13 subscales If we examine the correlations among the scales in light of the theoretical model, we find that the CA-MIR can differentiate between the three main styles of attachment. The signs of the correlation coefficients are theoretically coherent: specifically, the three scales associated with autonomous attachment are negatively correlated with those assessing preoccupied and dismissing attachment, which in turn are positively correlated with one another. The last-mentioned result invites further analysis: indeed, the relationship between the two insecure styles is puzzling and requires cautious interpretation. Within each attachment style, the correlations among individual scales are positive, but vary in strength. Past and State of Mind factors are strongly correlated for each of the three attachment patterns. These robust associations could be due to the formulation of the State of Mind items, which frequently refer to the past. Indeed, as conceived by the authors of the CA-MIR, the State of Mind scales typically investigate respondents' current evaluations of their childhood experience (52: \\u201cI feel that I did not have the opportunity to assert myself in the environment I grew up in\\u201d) (Pierrehumbert et al., 1996 ). The correlations between the dimension of the Present and the other two dimensions ( Past, State of Mind ) differ across the three attachment styles: these correlations are high (>0.60) for autonomous attachment patterns, moderate (around 0.50) for dismissing attachment and low (0.20) for preoccupied attachment. These differences in the correlation of the three dimensions ( Past, Present , and State of Mind ) may be interpreted in light of item semantics. Nevertheless, a theoretical explanation is also plausible: the Past and State of Mind scales concern child experiences, whereas the Present scales regard relationships within the respondent's current family unit, often involving a partner and a different type of relationship (romantic). However, the stronger association among Present, Past , and State of Mind dimensions for the autonomous attachment style could imply that the CA-MIR is not effective in identifying so-called earned-secure attachment patterns. Individuals with earned-secure attachment describe negative childhood experiences, which might be expected to produce some form of insecure attachment, but are nonetheless found to display a secure/autonomous state of mind (Hesse, 2008 ). The last four Scales concern two further constructs. Scales J and K are theoretically related to traumatic experience and are designed to tap into Unresolved or Disorganized attachment. Scales J and K were not correlated with one other: the former concerns extremely negative childhood experiences (the mean score obtained by our sample on this scale was very low), while the latter refers to gaps in memory and consequently does not appear to be linked to any of other scales, particularly those concerning the past. In contrast, Scale J was positively correlated with Scale L ( Parental Abdication ), providing support for the notion that negative experience in childhood may also be related to a lack of sensitive discipline. Although this correlation is interesting, it is difficult to view Scales J and L as two facets of the same unresolved attachment style: they must be interpreted in terms of individual profiles. Scales L and M assess respondents' general views about the role of parents and the importance of authority: theoretically, these scales are not linked to attachment but to \\u201cideological\\u201d/\\u201cabstract\\u201d assumptions about parents and adults. In the current dataset, these scales clearly loaded onto distinct factors, and the pattern of correlations confirms their independent status. Scales L and M were moderately and negatively correlated, as expected: the former concerns the absence of parental discipline in respondents' childhood experience, and the latter, on the contrary, their views regarding the importance of authority. Scales J ( Parent-related trauma ) and L ( Parental abdication ) shared the same correlation profile: they were positively correlated with both types of insecure attachment and negatively correlated with autonomous attachment. This finding suggests that Scale J reflects parental behavior that is highly inadequate rather than just traumatic: according to the attachment literature, markedly negative childhood experience may be related to insecure attachment. The Scale L correlations are lower, but in the same direction: this provides support for the idea that \\u201csensitive discipline\\u201d (Van Zeijl, et al., 2006) is a key aspect of secure experience in childhood. Scale M ( Regard for Authority ) displayed an inverse pattern of associations, but the correlations were markedly weaker. Thus, the CA-MIR appears to be able to distinguish attachment security from ideological beliefs concerning the family.\",\n",
            "        \"conclusion\": \"Conclusion This study is part of a broader research project aimed at completing Italian standardization of the CA-MIR. The objective of the current study was to evaluate the factor structure of the instrument. Indeed, the CA-MIR is an interesting tool precisely because of the richness and complexity of its theoretical model. Nevertheless, this model had never been tested with CFA, whereas previous exploratory analysis had only partially reproduced the original theoretical model (Garrido et al., 2009 ). Our CFA enabled us to confirm the originally proposed structure of the CA-MIR: when used with its Likert response format it adequately differentiates between the three attachment styles, taking into account both past and present experiences of attachment relationships, and providing a complex and rich assessment of multiple features of attachment. From a cross-cultural point of view, our results support the original theoretical model in a novel cultural context, namely Italy. The importance of this generalization is linked to one of the core hypotheses of the attachment theory: the broad cross-cultural stability of attachment representations. Therefore, the 13 scores obtained by administering the CA-MIR may be taken as valid indexes providing key information about subjective experience and conscious representations of attachment. Taken as a whole, the Italian internal consistency values are acceptable and comparable to those obtained in the original Swiss validation study. As mentioned earlier, Scale H displays a number of weakness, including both the low number of items composing the scale (three items vs. the six items forming most of the other scales) and the formulation of Item 17. The last-mentioned item (\\u201c It's better not to lament a bereavement too much so as to get over it faster \\u201d), could be replaced by another question that more unambiguously taps into currently deployed dismissing attachment strategies. The outcomes of the correlation analysis provide some pointers regarding appropriate interpretation of CA-MIR scores. Firstly, we must be cautious in differentiating between respondents' evaluations of past experience and their states of mind about attachment: these two dimensions are strongly correlated for all attachment styles, and individual cases of difference between these scores must be carefully assessed. The dimension of present experience is relatively independent of past experience and state of mind, particularly in relation to dismissing and preoccupied attachment styles, and this is in line with the theoretical expectations. Furthermore, due to the strong correlation between dismissing and preoccupied attachment scales, we must be cautious in viewing them as distinct. Scales J and K are not correlated and must therefore be taken as two separate indicators rather than as forming a single scale of unresolved attachment, but they can nonetheless be of assistance in interpreting a respondent's clinical profile. Although the L and M scales are not envisaged as directly linked to attachment profiles, they offer useful additional information that can help to build up clinical profiles, e.g., when evaluating parenting skills. These characteristics, combined with the advantages of the self-report format, make the CA-MIR a useful and powerful instrument, capable of discriminating between representations of secure and insecure attachment, and of providing information about attachment representations that is of both clinical and research value. The sampling procedure adopted in the current study is one of its limitations. We chose a snowball sampling method in order to recruit a large sample. However, starting out from a northern region and a young adult population, this procedure reduced the representativeness of the sample because of the high proportion of participants from North-western Italy and aged between 20 and 29 years. Nevertheless, we have provided a detailed comparison of the demographic characteristics of our sample with those of the general Italian population, highlighting both differences and similarities, to facilitate contextualization of our findings. Although not completely representative, our sample is comparable to that of the original Swiss-led validation study, and its principal limitation (the over-representation of young adults) may also be viewed as an interesting feature, given that individuals in this age range typically face the key attachment challenges of adult life such as developing romantic relationships, growing more independent of their parents and becoming parents themselves. More in-depth analysis of socio-cultural variables is one of our future research objectives. Moreover, sample size was not big enough to employ the ESEM approach. The comparison between the ICM-CFA and complex-CFA models showed that some \\u201cconstruct relevant item multidimensionality\\u201d (Morin et al., 2016 ) affects the structure of the CA-MIR leading to inflated factors correlations if ignored and it could not be excluded that some item multidimensionality remained unmodeled in the present study. However, results confirm the distinctiveness of the 13 factors, potentially overestimating their correlation and the comparison with the ESEM model is a further research goal to carry out when a bigger sample will be available. In addition, an interesting direction for future research could be to test the concurrent and discriminant validity of the Italian version of the test, by comparing the CA-MIR with other instruments assessing adult attachment. In particular, it would be useful to compare CA-MIR self-report data with outcomes obtained using narrative instruments, such as the AAI. Comparison of AAI categories with CA-MIR scores would be a further test of the CA-MIR's effectiveness in investigating attachment representations, and can be useful to further analyse the behavior of the few unfitting items observed in this study. Finally, the instrument could be administered to clinical groups in order to assess its ability to detect specific aspects of insecure attachment in particular sub-samples, as has already been done in previous studies (Molina et al., 2007 , 2009 ) with a small group of maltreating parents.\"\n",
            "    },\n",
            "    {\n",
            "        \"pmcid\": \"PMC6060212\",\n",
            "        \"abstract\": \"Whether habit stimulation is effective in DOC patient arousal has not been reported. In this paper, we analyzed the responses of DOC patients to habit stimulation. Nineteen DOC patients with alcohol consumption or smoking habits were recruited and 64-channel EEG signals were acquired both at the resting state and at three stimulation states. Wavelet transformation and nonlinear dynamics were used to extract the features of EEG signals and four brain lobes were selected to investigate the degree of EEG response to habit stimulation. Results showed that the highest degree of EEG response was from the call-name stimulation, followed by habit and music stimulations. Significant differences in EEG wavelet energy and response coefficient were found both between habit and music stimulation, and between habit and call-name stimulation. These findings prove that habit stimulation induces relatively more intense EEG responses in DOC patients than music stimulation, suggesting that it may be a relevant additional method for eliciting patient arousal.\",\n",
            "        \"intro\": \"Introduction Exploring an effective method to help the arousal of patients suffering from disorders of consciousness (DOCs) is an ongoing challenge [ 1 , 2 ]. Sensitive stimulation treatments, including call-name and music stimulations, are the most commonly used methods [ 3 , 4 ]. Habits such as alcohol consumption and smoking can arouse intense behavioral responses in normal individuals [ 5 ]. These habits are built on patients\\u2019 specific and unique lifestyle behaviors. The detailed neurobiological mechanisms associated with habits remain unclear. It is still unknown whether habit stimulation can be applied to DOC patients, as few studies have compared patient responses to habit with those to other types of stimulation [ 6 ]. Clinical practice indicates that habit stimulation (e.g. smoking) arouses stronger behavioral responses than non-habit stimulation (e.g. non-smoking) in DOC patients. For example, there was no response when vinegar or sauce was placed on the lips of patients who were alcoholics, but alcohol caused their lips to start moving in a drinking movement. Yet these behavioral responses may be subjective, and a quantitative study is needed to explain this phenomenon. In most cases, qualitative analyses such as neuropsychological scales [e.g. the Coma Recovery Scale \\u2013 Revised (CRS-R) score] are used to evaluate brain functions in the clinical setting. In order to thoroughly investigate these behavioral responses, more quantitative analytical methods are needed to accurately evaluate the brain responses. Diagnostic tools such as functional magnetic resonance imaging [ 7 \\u2013 9 ], positron emission tomography [ 10 , 11 ], and electroencephalogram (EEG) analysis [ 12 \\u2013 17 ] have been used as auxiliary diagnostic methods to evaluate the state of consciousness. EEG has several advantages over the other methods, including low cost, safety, easy access, and convenience for bedside evaluation [ 18 \\u2013 21 ]. Generally, linear and nonlinear dynamic analysis methods are used to extract the features of EEG signals. In linear analysis, one of the common methods is time-frequency analysis, including variable-time-interval Fourier transformation, wavelet transformation, and Wigner\\u2013Ville distribution. Among these, wavelet transformation is a typical time-frequency analysis with multi-resolution and multi-scale characteristics, which enables the conversion of a multi-signal from coarse to fine [ 22 ]. Wavelet transformation is a mature theory which has been widely used in the study of EEG signals [ 23 ]. Besides linear analysis, nonlinear dynamic analysis methods have also been widely used to extract features of EEG signals [ 23 , 24 ]. Studies have shown that the human brain is a complex, multidimensional system, and the use of nonlinear dynamic analysis might accurately reflect brain states [ 25 \\u2013 27 ]. In this study, to investigate the response of DOC patients to habit stimulation, both linear and nonlinear dynamic analysis methods were applied to measure the response intensity of the EEG in DOC patients. In addition, topographic maps of the brain were plotted to evaluate the degree of response to stimulation in different regions.\",\n",
            "        \"method\": \"Materials and Methods Patients A total of 19 patients with alcohol consumption or smoking habits were examined in this study, including 9 MCS (minimally conscious state) and 10 VS (vegetative state) cases assessed by neurologists based on the CRS-R. These patients were recruited from the Rehabilitation Center of Wu Jing Hospital in Zhejiang Province, China. Patients were out of the clinically acute phase, had been in a DOC state for > 1 month, exhibited spontaneous breathing, and had no history of cardiopulmonary resuscitation or neurological disease. The inclusion criteria were as follows: (a) inability to follow commands; (b) inability to clearly express words; (c) inability to open eyes even with stimulation or achieve eye tracking, not due to paralysis; (d) inability to move arms and legs in a non-directed fashion; and (e) having a Glasgow coma scale [ 28 ] score \\u2264 8 points, and with a score for each item < 4-5-6-3-2-3 according to the CRS-R [ 29 ]. If the total score was \\u2265 8, the patient was diagnosed as MCS; if it was < 8, the patient was diagnosed as VS. The exclusion criteria were as follows: (a) drug interventions (which could affect the assessment of brain function) prior to data collection such as nerve-muscle blockers, depressants, or anesthetics; and (b) a coexisting disorder such as metabolic disease, poisoning, or shock that could affect brain activity. The state of consciousness was diagnosed by experienced clinicians from Wu Jing Hospital using the CRS-R scale. The regional Review Board approved the use of human participants in this study. Family members of patients and the attending doctor gave consent for EEG acquisition. Patient information is summarized in Table 1 . Table 1 Patient information. Disease status Number of patients CRS-R score Average age (years) Time after injury (months) Sex Male Female MCS 9 13.7 \\u00b1 2.61 39.3 \\u00b1 11.9 3.10 \\u00b1 1.92 8 1 VS 10 6.2 \\u00b1 1.90 51.1 \\u00b1 10.2 4.05 \\u00b1 1.38 7 3 CRS-R, Coma Recovery Scale \\u2013 Revised; MCS, minimally conscious state; VS, vegetative state. EEG Acquisition EEG signals were recorded in single-electrode channel mode with an Active Two EEG system (BioSemi, Amsterdam, Netherlands). Electrodes were placed over the entire head according to the 10\\u201320 general international standard lead system. Signals were recorded from 64 channels; the left and right earlobes were used as references. The EEG recording was initiated when the signal had been stable for at least 2 min. Signals were digitized at a sampling rate of 256 Hz, a bandwidth range from 0.5 Hz to 70 Hz, and an electrode impedance < 5 K\\u03a9. In the music stimulation, a piece of Chinese classical music \\u201cJasmine\\u201d was truncated into voice fragments and played for 90 s. In the call-name stimulation, the patient\\u2019s name was called by relatives for 90 s. In the habit stimulation, patients were stimulated either by wiping alcohol on the lips for 36 s for alcoholic patients, or by introducing the smell of cigarette smoke for 36 s for smoking patients (Fig. 1 ). The entire process was repeated three times. Fig. 1 Time-course of EEG acquisition. EEG Signal Preprocessing A time-window of 12 s was truncated from the acquired EEG data for analysis. IIR filter, an EEGLAB processing tool (University of California San Diego), was used to remove interference from the 50-Hz power frequency [ 30 ]. A wavelet soft threshold de-noising algorithm was applied to remove noise [ 31 ]. Wavelet De-noising of EEG Studies have shown that the process of wavelet de-noising often uses a given threshold for the de-noising paradigm. The most reliable threshold method is calculated based on trials and errors [ 31 ]. There are several classical threshold methods: (a) The VisuShrink threshold, also known as the general threshold, was the first wavelet threshold de-noising method developed. The probability that the coefficient is greater than the threshold is close to zero, so it is the optimal threshold method based on the minimum maximum estimation. (b) The Sureshrink threshold, also known as the Stein unbiased risk threshold, is close to the ideal threshold. This is an adaptive threshold selection based on Stein\\u2019s unbiased likelihood estimation criterion. (c) In the Heursure threshold, also known as the Heuristic Sure threshold, the threshold is chosen as the optimal predictor threshold, and is a synthesis of the first two thresholds. (d) The Minmax threshold uses a fixed threshold according to the minimum criteria to select the threshold. In order to choose the most suitable threshold estimation method to analyze the data, Gaussian white noise was added to the original signal, and wavelet threshold de-noising and hard threshold de-noising were performed on the mixed signal. The VisuShrink, Sureshrink, Heursure, and Minmax methods were used for threshold estimation. The signal-to-noise ratio and the root-mean-square error were introduced as reference indicators (Table 2 ). Table 2 Comparison of four threshold estimation methods. Threshold estimation method VisuShrink Sureshrink Heursure Minmax SNR 42.541 72.890 72.890 50.997 RSME 1.217 0.037 0.0370 0.460 SNR, signal-to-noise ratio; RSME, root-mean-square error. The signal-to-noise ratio was maximal and the root-mean-square error was minimal in the processing methods of Sureshrink and Heursure threshold estimation. These results indicated that these two methods were equal and superior to the others. Therefore, both methods were used to analyze all our data. Wavelet Energy Extraction The signal was decomposed into eight layers, and a db3 wavelet base was chosen. As the eight-layer decomposition was complex, 0 Hz\\u201332 Hz three-layer decomposition was taken as an example for the following wavelet decomposition corresponding band diagram (Fig. 2 ). Fig. 2 Correspondence of wavelet decomposition band diagram. The energy value of a certain wavelet coefficient can be used to represent the energy value of a signal in a band. Since the wavelet function is an orthogonal basis function with energy conservation characteristics, the sum of the wavelet energy values of each band can represent the energy value of the signal, that is, the wavelet energy value of the signal: \\\\documentclass[12pt]{minimal} \\\\usepackage{amsmath} \\\\usepackage{wasysym} \\\\usepackage{amsfonts} \\\\usepackage{amssymb} \\\\usepackage{amsbsy} \\\\usepackage{mathrsfs} \\\\usepackage{upgreek} \\\\setlength{\\\\oddsidemargin}{-69pt} \\\\begin{document}$$ E_{n} = \\\\sum\\\\limits_{k} {\\\\left| {d_{l}^{j,n} } \\\\right|}^{2} $$\\\\end{document} E n = \\u2211 k d l j , n 2 where both n and k are natural numbers and \\\\documentclass[12pt]{minimal} \\\\usepackage{amsmath} \\\\usepackage{wasysym} \\\\usepackage{amsfonts} \\\\usepackage{amssymb} \\\\usepackage{amsbsy} \\\\usepackage{mathrsfs} \\\\usepackage{upgreek} \\\\setlength{\\\\oddsidemargin}{-69pt} \\\\begin{document}$$ d_{i}^{j,n} $$\\\\end{document} d i j , n are wavelet packet coefficients. Wavelet transformation was used to calculate EEG signal features under resting and stimulation states (music, habit, and call-name). The resting state was considered as the unified normalized standard of response coefficients. Therefore, the ratio between the wavelet transform value of the stimulation state and the wavelet transform value of the resting state was defined as the wavelet energy value. This ratio reflected the change of wavelet energy before and after stimulation. The EEG wavelet energy was calculated as: \\\\documentclass[12pt]{minimal} \\\\usepackage{amsmath} \\\\usepackage{wasysym} \\\\usepackage{amsfonts} \\\\usepackage{amssymb} \\\\usepackage{amsbsy} \\\\usepackage{mathrsfs} \\\\usepackage{upgreek} \\\\setlength{\\\\oddsidemargin}{-69pt} \\\\begin{document}$$ Wavelet \\\\, energy \\\\, value \\\\, \\\\left( {ratio} \\\\right)\\\\; = \\\\;wavelet \\\\, energy \\\\, value \\\\, of \\\\, stimulation \\\\, state{ / }wavelet \\\\, energy \\\\, value \\\\, of \\\\, resting \\\\, state. $$\\\\end{document} W a v e l e t e n e r g y v a l u e ratio = w a v e l e t e n e r g y v a l u e o f s t i m u l a t i o n s t a t e / w a v e l e t e n e r g y v a l u e o f r e s t i n g s t a t e . Nonlinear Dynamic Feature Extraction Correlation dimension, complexity, entropy, and Lyapunov exponents are common nonlinear features in EEG signal analysis. Of these, entropy is the most suitable because of its small dataset demand and high computation speed, whereas correlation dimension and Lyapunov exponent require large datasets and strict dimensions (features that are unsuitable for EEG analysis). Therefore, in this study, approximate entropy (ApEn) [ 32 ], sample entropy (SampEn) [ 33 ], and permutation entropy (PmEn) [ 34 ], were used to determine the patients\\u2019 state of consciousness. A nonlinear dynamic method was used to calculate EEG signal features (ApEn, SampEn, and PmEn) under the resting and stimulation states (music, habit, and call-name) [ 7 ]. During feature extraction, ApEn, SampEn, and PmEn were computed for the EEG data in each time-window (12 s signal truncated from the EEG data). For each nonlinear characteristic, the mean of the values from all time-windows was considered as a feature of the EEG data. The feature value was extracted and calculated from each channel. The means of single features were the average value of all 64 channel features of the EEG data. The resting state was considered as the unified normalized standard of response coefficients (Rc). Therefore, the ratio between the nonlinear dynamic feature of the stimulation state and the nonlinear dynamic feature of the resting state was defined as the Rc. Rc values are the EEG nonlinear dynamic feature response coefficients of the stimulation value, which reflect the changes of a nonlinear dynamic feature before and after stimulation. The Rc values were calculated as: \\\\documentclass[12pt]{minimal} \\\\usepackage{amsmath} \\\\usepackage{wasysym} \\\\usepackage{amsfonts} \\\\usepackage{amssymb} \\\\usepackage{amsbsy} \\\\usepackage{mathrsfs} \\\\usepackage{upgreek} \\\\setlength{\\\\oddsidemargin}{-69pt} \\\\begin{document}$$ {\\\\text{Rc}}\\\\;{\\\\text{value}}\\\\;\\\\left( {\\\\text{ratio}} \\\\right)\\\\; = \\\\;{\\\\text{feature}}\\\\;{\\\\text{of}}\\\\;{\\\\text{stimulation}}\\\\;{\\\\text{state / feature}}\\\\;{\\\\text{of}}\\\\;{\\\\text{resting}}\\\\;{\\\\text{state}}. $$\\\\end{document} Rc value ratio = feature of stimulation state / feature of resting state . RcA, RcS, and RcP refer to the Rc values for the three features ApEn, SampEn, and PmEn, respectively. Statistical Analysis Wavelet energy and Rc values were analyzed using the paired-samples t -test, independent-sample t -test, and one-way ANOVA, using SPSS v.19 software (SPSS Inc., Chicago, IL). P < 0.05 was considered statistically significant.\",\n",
            "        \"subjects\": null,\n",
            "        \"results\": \"Results EEG Wavelet Energy Values for MCS and VS Under Different Stimulations We first investigated the degree of EEG response to various kinds of stimulation by comparing the differences in wavelet energy. The highest wavelet energy of the cases (including MCS and VS) was for call-name stimulation, followed by habit and music stimulation (Table 3 ). Table 3 Comparison of wavelet energy values in different stimulation states. Stimulation Total (MCS + VS) MCS VS Music 1.124 \\u00b1 0.147 1.139 \\u00b1 0.158 1.107 \\u00b1 0.116 Habit 1.346 \\u00b1 0.198 1.391 \\u00b1 0.215 1.296 \\u00b1 0.115 Call-name 1.423 \\u00b1 0.314 1.489 \\u00b1 0.343 1.349 \\u00b1 0.128 To verify the differences among the three stimulations, statistical analysis of the wavelet energy values between the different stimulations were performed. First, differences were analyzed between habit and music stimulation, then between habit and call-name stimulation, using the paired-samples t -test. In all cases, there were significant differences in the wavelet energy both for habit versus music (P = 0.0065) and for habit versus call-name stimulation ( P = 0.0089). In MCS cases, the wavelet energy differed significantly ( P = 0.0074) between habit and music, but not between habit and call-name stimulation. However, the VS cases showed no significant differences in wavelet energy either between habit and music, or between habit and call-name stimulation (Fig. 3 ). Fig. 3 Difference of wavelet energy values in total cases ( n = 19), MCS cases ( n = 9), and VS cases ( n = 10) in the three stimulations (music, habit, and call-name) (error bars, 95% confidence intervals; ** P < 0.01). Further investigations of the differences of wavelet energy values between VS and MCS cases for the three stimulations were performed using the independent-samples t -test. There was a significant difference in these values for habit ( P = 0.023) and call-name stimulation ( P = 0.016) between MCS and VS cases, but not for music stimulation (Fig. 4 ). Fig. 4 Analysis of EEG wavelet energy value differences between VS and MCS cases for the three stimulations (error bars, 95% confidence intervals; * P < 0.05). EEG Rc Values for MCS and VS Under Different Stimulations We then investigated the degree of EEG response to various kinds of stimulation by comparing the nonlinear entropy values in DOC patients. The Rc values (RcA, RcS, and RcP) of the three different stimulations are shown in Table 4 . The results indicated that the highest Rc values among all the cases corresponded to call-name stimulation, followed by habit and music stimulation. Table 4 EEG entropy values in different stimulation states. Sample Rc values RcA RcS RcP Total (MCS+VS) Music 1.044 \\u00b1 0.13 1.071 \\u00b1 0.12 1.004 \\u00b1 0.01 Habit 1.262 \\u00b1 0.05 1.234 \\u00b1 0.09 1.254 \\u00b1 0.31 Call-name 1.446 \\u00b1 0.24 1.254 \\u00b1 0.21 1.394 \\u00b1 0.32 MCS Music 1.025 \\u00b1 0.15 1.067 \\u00b1 0.15 1.004 \\u00b1 0.02 Habit 1.413 \\u00b1 0.07 1.377 \\u00b1 0.13 1.413 \\u00b1 0.23 Call-name 1.668 \\u00b1 0.21 1.644 \\u00b1 0.19 1.602 \\u00b1 0.28 VS Music 1.061 \\u00b1 0.13 1.074 \\u00b1 0.08 1.003 \\u00b1 0.004 Habit 1.127 \\u00b1 0.09 1.105 \\u00b1 0.06 1.111 \\u00b1 0.36 Call-name 1.247 \\u00b1 0.17 1.211 \\u00b1 0.15 1.206 \\u00b1 0.24 In order to verify the statistical differences between the three stimulations, statistical analysis of the Rc values for habit versus music, and for habit versus call-name stimulation was performed using the paired-samples t -test. In all cases, there were significant differences in the three Rc values between habit versus call-name stimulation (RcA, P = 0.029; RcS, P = 0.034; RcP, P = 0.041). Moreover, two Rc values (RcA and RcP) remarkably differed between habit and music stimulations ( P = 0.0082, P = 0.0096) (Fig. 5 A). In MCS cases, there were significant differences in the three Rc values between habit and music stimulations (RcA, P = 0.0097; RcS, P = 0.037; RcP, P = 0.0088); but only RcS differed ( P = 0.038) between habit and call-name stimulations (Fig. 5 B). In VS cases, there were no differences in the three Rc values either between habit and music or between habit and call-name stimulations (Fig. 5 C). Fig. 5 Rc values of A the total patient population ( n = 19), B MCS cases ( n = 9), and C VS cases ( n = 10) in response to the three stimulations (error bars, 95% confidence intervals; * P < 0.05, ** P < 0.01). Further comparison of the Rc values between VS and MCS cases was performed using the independent-samples t -test. RcA and RcS differed significantly (RcA, P = 0.006; RcS, P = 0.004) between MCS and VS cases for habit stimulation; all three differed significantly (RcA, P = 0.007; RcS, P = 0.001; RcP, P = 0.003) between MCS and VS cases for call-name stimulation; and none was significantly different between MCS and VS cases for music stimulation (Table 5 ). Table 5 Analysis of EEG Rc differences ( P values) between VS and MCS for the three stimulations. Feature value Music Habit Call-name RcA 0.177 0.006 0.007 RcS 0.894 0.004 0.001 RcP 0.871 0.032 0.003 EEG Response in Different Brain Lobes Under Different Stimulations Since different stimulations may activate distinct brain areas, we finally investigated the degree of EEG responses in the frontal, temporal, parietal, and occipital lobes at different regions elicited by various types of stimulation. Among the nonlinear EEG features (ApEn, SampEn, and PmEn), PmEn measures the complexity of a signal, with the advantages of being simple, fast, and having a strong anti-noise property. Therefore, PmEn was used to measure the nonlinear dynamic feature responses of the different areas. The differences of entropy values in the different areas during the three stimulations were evaluated using one-way ANOVA. The resulting EEG responses in the different lobes under different stimulations are shown in Table 6 . During habit stimulation, the RcP values in the frontal lobe were significantly higher than those in any other areas in VS cases ( P < 0.05), MCS cases ( P < 0.01), and VS + MCS cases ( P < 0.01). During music stimulation, the RcP values in the temporal lobe were significantly higher ( P < 0.05) than those in any other areas for VS + MCS and VS cases, but not for MCS cases. During call-name stimulation, the RcP values in the temporal lobe were significantly higher than those in any other areas for VS + MCS ( P < 0.05), VS ( P < 0.05), and MCS ( P < 0.01). Table 6 EEG entropy values in different stimulation states. Sample Brain area Frontal lobe Temporal lobe Parietal lobe Occipital lobe Total (MCS+VS) Music 0.986 \\u00b1 0.097 1.048 \\u00b1 0.105* 1.003 \\u00b1 0.079 0.985 \\u00b1 0.103 Habit 1.452 \\u00b1 0.132** 1.116 \\u00b1 0.097 1.179 \\u00b1 0.099 1.294 \\u00b1 0.154 Call-name 1.378 \\u00b1 0.112 1.530 \\u00b1 0.101* 1.315 \\u00b1 0.101 1.393 \\u00b1 0.145 MCS Music 1.014 \\u00b1 0.059 1.024 \\u00b1 0.062 1.005 \\u00b1 0.050 0.972 \\u00b1 0.057 Habit 1.664 \\u00b1 0.163** 1.215 \\u00b1 0.099 1.280 \\u00b1 0.104 1.492 \\u00b1 0.138 Call-name 1.601 \\u00b1 0.068 1.730 \\u00b1 0.110** 1.447 \\u00b1 0.126 1.631 \\u00b1 0.162 VS Music 0.973 \\u00b1 0.122 1.072 \\u00b1 0.135* 0.967 \\u00b1 0.101 0.998 \\u00b1 0.135 Habit 1.240 \\u00b1 0.103* 1.018 \\u00b1 0.100 1.077 \\u00b1 0.099 1.096 \\u00b1 0.165 Call-name 1.155 \\u00b1 0.139 1.331 \\u00b1 0.092* 1.183 \\u00b1 0.064 1.154 \\u00b1 0.099 * P < 0.05, ** P < 0.01. To visualize the results of brain responses under different stimulations, we used the topographic map visualization method. Three VS cases and 4 MCS cases were chosen to plot the maps using the mean PmEn (Fig. 6 ). These patients did not suffer from traumatic brain injury, but from other conditions such as subarachnoid hemorrhage or diffuse axonal injury. Indeed, in these cases, the brain integrity was conserved and the mapping was not influenced by injury of the lobes. Fig. 6 EEG topographic maps of the mean PmEn under the three stimulations. The colors reflect the intensity of the EEG response. MCS and VS results for music stimulation (upper panels), habit stimulation (middle panels), and call-name stimulation (lower panels). With the music stimulation, in both MCS and VS cases, the darkest area was located in the temporal cortex, denoting a significantly more intense EEG response than in other regions. Furthermore, the EEG responses in MCS cases were more intense than those of VS cases. The results of music stimulation showed that, in the selected cases, the temporal region was the most responsive. With the habit stimulation, in both MCS and VS cases, the darkest area was located in the prefrontal region, which had significantly more intense EEG responses than other regions. The EEG responses of the MCS cases were more intense than those of the VS cases. The results of habit stimulation demonstrated that, for the selected cases, the frontal brain region was the most responsive area. With the call-name stimulation, in the MCS cases, the darkest areas were in the left temporal and right occipital areas, which had significantly more intense EEG responses than other regions. In the VS cases with call-name stimulation, the darkest area was in the temporal area, which had significantly more intense EEG responses than other regions. The EEG responses of MCS cases were more intense than those of VS cases.\",\n",
            "        \"discussion\": \"Discussion In this study, using EEG wavelet transformation and nonlinear dynamics analysis, we studied the EEG responses of DOC patients under different stimulations to determine whether habit stimulation is clinically useful. We found that, with the three types of stimulation (music, habit, and call-name), the EEG response under habit stimulation was higher than that under music stimulation, but lower than that under call-name stimulation. In different consciousness states, the MCS response to habit stimulation was more pronounced than that for VS patients. These results provide preliminary evidence for the effectiveness of habit stimulation. Concerning the spatial distribution of EEG responses, the brain maps showed a more intense response in the frontal lobe during habit stimulation and in the temporal lobe during music and call-name stimulation. These results may reflect positive neural-related activity evoked by the stimulations. Results from current studies suggest that habit stimulation can arouse brain responses; however, the detailed neurobiological mechanism underlying habit remains unclear but could be involved in the addiction mechanism. Existing research shows that the mesolimbic dopamine system (MLDS) is the neurobiological basis of the addiction mechanism [ 35 ]. The MLDS is a pathway by which dopamine is carried from one area to another. Dopamine is the major molecule released by the brain\\u2019s reward centers. Dopaminergic neurons are mainly located in the ventral tegmental area, and also have projections into several parts of the brain, including the nucleus accumbens, prefrontal cortex, hippocampi, and amygdala. The MLDS is the common neural pathway of the reward mechanism, and is involved in the physiological process of addiction [ 36 ]. In nicotine habit stimulation, the MLDS is thought to be the key site of its action. However, nicotine also increases the extracellular dopamine concentration by stimulating dopaminergic neurons. In alcohol habit stimulation, it has been reported that alcohol stimulates addiction, triggers dopamine release, enhances brain activity, and stimulates the reward system. Rose [ 37 ] proposed that nicotine and alcohol have positive reinforcement in the MLDS. The specific neural mechanisms involved in habit stimulation need to be further clarified using imaging techniques such as fMRI combined with EEG, as well as neuropsychological scale methods. Future investigations need larger samples.\",\n",
            "        \"conclusion\": null\n",
            "    },\n",
            "    {\n",
            "        \"pmcid\": \"PMC6286024\",\n",
            "        \"abstract\": \"Present study was carried out to find out the association of vitamin D receptor (VDR) gene polymorphism, mRNA gene expression of VDR gene and level of vitamin D with leprosy reactions and leprosy patients. Surprisingly, level of vitamin D in leprosy patients was not found to be associated with the disease and its manifestations. VDR genotypes (Fok1 and Taq1) were found to be associated with leprosy patients. t-F-a haplotype was significantly associated with leprosy patients. Gene expression of vitamin D receptor was lower in leprosy patients in comparison to healthy controls.\",\n",
            "        \"intro\": \"Introduction Leprosy is a chronic, debilitating infectious disease caused by Mycobacterium leprae that involves skin, peripheral nerves and mucous membranes. At one pole is tuberculoid (TT) leprosy, characterized by strong Th1 immunity with few bacilli in skin lesions and at the opposite pole is lepromatous (LL) leprosy, characterized by large number of bacilli scattered throughout the skin lesions expressing high Th2 type of immunity [ 1 ]. In between these two poles the disease is manifested in a spectrum as borderline tuberculoid (BT), borderline (BB) and borderline lepromatous (BL) determined by the immune status of the host. Two types of reactions are commonly observed in leprosy patients. Type 1 or reversal reaction (T1R) mainly occurs after the initiation of treatment in patients at the borderline pole of the leprosy spectrum (BL, BB, and BT) with up-regulation of the cell mediated immunity (CMI) and consequently shifting from a Th2 type to a Th1 type of immune response. Type 2 reaction (T2R) or erythema nodosum leprosum (ENL) occurs in BL/LL group of leprosy patients and is more commonly seen in patients with high bacterial load along with high level of M . leprae antibody resulting in deposition of immune complexes and complement [ 2 ]. Host immune system plays a vital role in monitoring and controlling the spread of infection. Genetic polymorphisms in the immune reactive genes have been involved in making a person more susceptible or resistant to infections. Genetic studies have indicated that association of HLA [ 3 ] and some non-HLA genes are responsible for susceptibility to leprosy phenotypes that occurs upon infection [ 4 , 5 ]. Vitamin D Receptor (VDR) is a member of the nuclear receptor family and it controls transcriptional responses and regulate micro-RNA directed post-transcriptional mechanisms for initiation of an effective immune response. Hence, vitamin D facilitates several immuno-modulatory properties through VDR. A single dose of vitamin D supplementation has been reported to enhance the immunity to mycobacteria [ 6 ]. VDR is present on many immune cells like macrophages, dendritic cells, T and B lymphocytes [ 7 ] and after stimulation it plays a dynamic role in host immune response [ 8 ]. It has been reported earlier that among the allelic genotypes of Taq1 polymorphisms genotypes tt homozygous tend to produce Th1-type responses while TT homozygous produce a Th-2 type response [ 9 ]. Frequency and distribution of VDR gene (Taq1, Fok1 and Apa1) in healthy individuals of Indian population was reported earlier [ 10 ]. Previously in a Bengali population Taq1 polymorphism was found to be associated with leprosy [ 9 ]. It has been shown that The VDR Fok1 ff genotype and Apa1 AA, Aa genotype and haplotype T-f-a, T-F-A were positively associated with leprosy in South-Indian population [ 11 ]. Vitamin D deficiency was found to be linked with increased risk of some infectious and metabolic diseases [ 12 , 13 ]. It has been documented that hypercalcemia suppressed 1, 25-dihyroxyvitamin-D3 (1,25OHD) levels [ 14 ]. It was also reported that serum 1,25OHD and calcium levels remain elevated in patients with leprosy [ 15 ]. However, no differences in serum 1,25OHD levels between leprosy patients and normal individuals have also been reported [ 16 ]. Further, vitamin D in massive doses was used along with sulfones in the treatment tuberculoid leprosy which modified the clinical course of the disease [ 6 ]. Recently, Mandal et al (2015) reported significantly lower levels of vitamin D in leprosy patients compared to their household controls [ 17 ]. In the light of above-mentioned facts, we hypothesize that there might be an association between vitamin D level with the leprosy and that VDR gene polymorphism may show association with leprosy and also VDR mRNA gene expression may be associated with the disease status. Since, osteoporosis is a major problem in lepromatous forms of leprosy along with conflicting reports on vitamin D levels in leprosy, the present study has been undertaken to find out the associations of vitamin D level and mRNA VDR gene expression VDR gene in leprosy patients and in reaction (T1R and T2R) group of leprosy. Further, an attempt has also been made to investigate the association of VDR gene polymorphism, if any, with leprosy patients with and without reactions.\",\n",
            "        \"method\": \"Materials and methods Ethics statement This study was approved by The Leprosy Mission Trust India (TLMTI) human ethics committee following the guidelines of the Indian Council of Medical Research (ICMR) in a meeting held on 29 th August, 2014 with REC No. EC 5/14/a, Protocol No. 1/5/14. All the adult participants (age >18 years) who agreed to participate in the study with written informed consents were enrolled. In the case of minors (age 10\\u201318 years) written informed consent was obtained from their parents/guardians on behalf of all child participants. Participants Leprosy patients were recruited from the outpatient department (OPD) of TLM Community Hospital, Shahdara, Delhi. A total of 305 leprosy patients [NR (218) (TT/BT (n = 128), BL/LL (n = 90)), T1R (n = 74) and T2R (n = 13)] were enrolled in the study. Patients were diagnosed clinically by experienced dermatologists and classified on Ridley-Jopling scale (1966) [ 18 ]. We recruited leprosy patients of TT/BT and BL/LL group without active reaction and no history of leprosy reactions in the past years. T1R group is characterized by development of acute inflammation in skin lesions/nerves/both. T2R is a serious systemic illness with inflamed nodules, papules. Neuritis occurs in both types of leprosy reactions and it is major cause of disability in leprosy patients. Leprosy cases with HIV sero-positive, tuberculosis (TB), diabetes and any other dermatological and immunosuppressive disorders were excluded from the study. Healthy individuals (n = 200) of the same locality with similar social and financial backgrounds were enrolled as healthy controls (HCs). HCs having family history of leprosy or TB and HIV were also excluded from the study. The demographic characteristics of the leprosy patients and healthy controls were tabulated in Table 1 . 10.1371/journal.pntd.0006823.t001 Table 1 Demographic characteristics of leprosy patients and healthy controls. Characteristics Leprosy Patients (n = 305) Healthy Controls (n = 200) P value Age (Mean \\u00b1 SD) 35.01 \\u00b1 13.95 32.69 \\u00b1 10.95 0.096 a Gender (%) Male 194 (63.60) 126 (63) 0.89 b Female 111 (36.39) 74 (37) TT/BT (%) 202 (66.23) - BL/LL (%) 103 (33.78) - T1R (%) 74 (24.26) - T2R (%) 13 (4.26) - a Mean age of leprosy patients and healthy controls was compared by Mann-Whitney U test b Gender of leprosy patients and healthy controls was compared by Mann-Whitney U test Sample collection and analysis Peripheral blood samples were collected from all the participants and sera were separated and stored at -20\\u00b0C until further processing. DNA extraction DNA was extracted from whole blood using DNeasy blood and tissue kit (Qiagen, Hilden, Germany) according to manufacturer\\u2019s instructions. DNA concentration was estimated on a UV Spectrophotometer (Shimadzu Inc. Japan). RNA extraction RNA was extracted from whole blood using RNeasy blood kit (Qiagen, Hilden, Germany) according to the manufacturer\\u2019s instructions. RNA concentration was estimated on a UV Spectrophotometer (Shimadzu Inc. Japan). Genotyping of VDR A total of 505 samples were included for genotyping. Genotyping of VDR Taq1, Fok1 and Apa1 polymorphisms was done by PCR-RFLP method using specific primers [ 19 , 20 , 21 ] and restriction enzymes. Taq1 polymorphism The PCR was done with some minor modifications using published protocol by Bid et al (2005) [ 10 ]. Briefly, PCR cycling conditions were consisted of 35 cycles of 94\\u00b0C for 60s, 63\\u00b0C for 60s and 72\\u00b0C for 2 min each, preceded by a single cycle of 94\\u00b0C for 5 min and followed by a single cycle of 72\\u00b0C for 10 min. The PCR product (740 bp) was digested with Taq1 (NEB) restriction enzyme at 65\\u00b0C for 20 min inactivated at 80\\u00b0C for 20 min. Digested products were either of the homozygous TT (absence of specific restriction site) of 245 bp and 495 bp or homozygous tt yielding 205, 245, 290 bp and heterozygous Tt yielding 205, 245, 290, 495 bp [ 10 ]. Fok1 polymorphism PCR cycling conditions were of 35 cycles at 94\\u00b0C for 30s, 61\\u00b0C for 30s and 72\\u00b0C for 1 min each, preceded by a single cycle of 94\\u00b0C for 5 min and followed by a single cycle of 72\\u00b0C for 10 min. PCR product (265 bp) was digested with Fok1 (NEB) restriction enzyme, reaction was incubated at 37\\u00b0C for 1 hr and inactivated at 65\\u00b0C for 20 min. Digested products were of FF (265 bp), ff (196 and 69 bp) and Ff (265, 196 and 69 bp). Apa1 polymorphism PCR cycle conditions were initiated with denaturation at 94\\u00b0C for 4 min followed by 35 cycles at 94\\u00b0C for 30s, 65\\u00b0C for 30s and 72\\u00b0C for 2 min and with final extension at 72\\u00b0C for 10 min. Amplified product (2000 bp) was digested with restriction enzyme Apa1 (NEB) for 25\\u00b0C for 20 min and was inactivated at 65\\u00b0C for 20 min. Digested product was of AA (2000 bp), aa (1700, 300 bp) and Aa (2000, 1700 and 300 bp). Vitamin D level A total of 119 participants consisting of HC (n = 31), TT/BT (n = 26), BL/LL (n = 21), T1R (n = 28) and T2R (n = 13) were included in the study. Estimation of serum vitamin D levels was outsourced from M/s Thyrocare, India. It has been done by fully automated chemiluminescence immunoassay by using ADVIA Centaur. Gene expression of vitamin D mRNA cDNA was constructed from 1 \\u03bcg of total RNA from each of the sample using cDNA Synthesis Kit (NEB #6300S, New England Biolabs Inc) according to the manufacturer\\u2019s instructions. Briefly, 1\\u03bcg of total RNA was mixed with Random Primer mix and nuclease free water. RNA was then denatured at 70\\u00b0C for 5 min and followed by the addition of 1\\u00d7M-MuLV Reaction Mix containing buffer, Magnesium ions, dNTPs and 1\\u00d7M-MuLV Enzyme Mix containing 0.5 units/\\u03bcl of Reverse Transcriptase and RNase Inhibitor. Temperature cycle conditions were 25\\u00b0C for 5 min, 45\\u00b0C for 1 hr and inactivation of enzyme was carried out at 80\\u00b0C for 5 min. cDNA corresponding to VDR transcript was amplified on Rotor Gene-Q (Qiagen Inc. USA) real-time PCR machine using primers and reaction conditions as published earlier [ 22 ]. Briefly, 10\\u03bcl of Rotor-Gene SYBR Green PCR Master Mix (Qiagen, Cat No. 204074), 200 nM concentration of each of the forward and reverse primers for VDR and 200 ng of cDNA (total of 25 \\u03bcl reaction mix) are amplified in Rotor-Gene Q with cycling conditions as 95\\u00b0C for 10 min (initial denaturation and activation of enzyme) followed by 40 cycles of 95\\u00b0C for 10 s, annealing at 61\\u00b0C for 15s and elongation at 72\\u00b0C for 20 s. The fluorescence was acquired on green channel during the annealing step. A melting curve analysis was performed by heating the amplicons from 65\\u00b0C to 95\\u00b0C with 1\\u00b0C/s raise in temperature at each step. The mRNA expression levels were normalized by using GAPDH (glyceraldehyde 3 phosphate dehydrogenase) mRNA as a house-keeping gene. The threshold fluorescence values were normalized to those of GAPDH values. The mRNA expression levels were calculated after determining the primer efficacy for all the genes using Pfaffl Method [ 23 ] by a standard curve with a 6-fold dilution of cDNA from 1 \\u03bcg/reaction to 10 ng/reaction. Melting curve analysis was performed to determine the integrity of the amplification and to rule out primer\\u2013dimer formation. The percentage efficiency of the primers from the standard graphs were determined to be in the order of 98% for GAPDH, 93% for VDR. The fold difference in expression was calculated based on the below formula: Ratio = ( E target ) \\u0394 C T target ( Control \\u2013 Sample ) / ( E ref ) \\u0394 C T ref ( Control \\u2013 Sample ) Individual expression ratios using the above formula were computed for all the subjects taking the average ratio of HC as the control value for all the cases. Statistical analysis The statistical analysis was performed using Graph-Pad Prism software (Version 6). Chi-square test was done for investigating Hardy-Weinberg equilibrium and for comparing the genotypes and alleles frequencies between the controls and different phenotypes of leprosy. Odds ratio (OR) and 95% confidence interval (CI) were computed for the association between genotype and leprosy. An individual genotype analysis was performed using logistic regression model corrected by sex for the significant detected genotype by SNPstat software. Haplotype analysis and association was carried out using online software SNPstat ( https://www.snpstats.net/snpstats/preproc.php ) [ 24 ]. For genetic association study we adjusted the threshold value (0.02) below which p-value is considered significant. The level of vitamin D and VDR gene expression among all the studied groups were analyzed by One-way ANOVA non-parametric Kruskal-Wallis test. Dunn\\u2019s multiple comparison test was used to find out the difference between healthy controls and groups of leprosy patients. For all analysis p value, less than 0.05 was considered as statistically significant. The real-time data was analyzed on Rotor-Gene Q Series Software (Software Version 2.0.2).\",\n",
            "        \"subjects\": null,\n",
            "        \"results\": \"Results Participants characteristics A total of 305 leprosy patients [NR (218), T1R (n = 74) and T2R (n = 13)] were enrolled in the study. Healthy individuals (n = 200) from the same geographical area and having similar societal and financial background were included as healthy controls (HCs). Genotypic analysis of VDR gene polymorphisms The allele and genotype distribution of each SNP (Taq1, Fok1, Apa1) of VDR gene was in agreement with Hardy-Weinberg equilibrium (p>0.05) in leprosy patients and healthy controls. The frequency distribution of alleles, genotypes and association analysis is described in Table 2 . A significant difference in the distribution of Taq1 position TT, Tt, tt (p = 0.0037) and Fok1 position FF, Ff and ff was observed (p = 0.0048) between leprosy patients and healthy controls. The allelic frequency distribution of T and t allele (p = 0.0037), F and f allele (p = 0.0024) was significantly different between leprosy and healthy controls. However, considering the difference in the frequency of tt genotype in leprosy patients (14.81%) and healthy controls (7.51%), genotypes were analyzed for significant association. The recessive model of Taq1 was found to be associated with leprosy patients [OR (95%CI) 0.47 (0.24\\u20130.89, p = 0.016) ( Table 2 ). ff genotype of Fok1 was found to be associated with leprosy patients [p = 0.0004; OR (95% CI) 3.148 (1.662\\u20135.965)] ( Table 2 ). The recessive model of Fok1 genotype was found to be significantly associated in leprosy patients in comparison to healthy controls [p = 0.00004; OR (95% CI) 2.85 (1.56\\u20135.22)] ( Table 2 ). No significant difference was observed in allele and genotype frequencies for Apa1 polymorphism between leprosy patients and healthy controls (p>0.02) ( Table 2 ). 10.1371/journal.pntd.0006823.t002 Table 2 Genotype and allele frequencies of VDR gene (Taq1, Fok1 and Apa1) polymorphism among leprosy patients and healthy controls. Genotype/allele Leprosy patients (n = 305) Healthy controls (n = 200) OR (95% CI) P value of \\u03c7 2 test Taq1 (rs731236) N = 297 (%) N = 173 (%) HWE (p value) 1.0 0.59 TT 111 (37.37) 85 (49.13) Reference 0.0037 * Tt 142 (47.81) 75 (43.35) 0.6897 (0.4634\\u20131.027) 0.069 tt 44 (14.81) 13 (7.51) 0.3858 (0.1954\\u20130.7618) 0.0053 T allele 364 245 t allele 230 101 0.0037 TT vs Tt+tt 0.6178 (0.4226\\u20130.9032) 0.015 tt vs Tt+TT 0.47 (0.24\\u20130.89) 0.016 Fok1 (rs2228570) (N = 288) (N = 185) HWE (p value) 0.12 0.44 FF 137 (47.57) 71 (38.38) Reference 0.0048* Ff 132 (45.83) 83 (44.86) 1.213 (0.8157\\u20131.805) 0.363 ff 19 (7) 31 (16.76) 3.148 (1.662\\u20135.965) 0.0004 F allele 406 225 f allele 170 145 0.0024 FF vs Ff+ff 1.457 (1.000\\u20132.122) 0.049 ff vs Ff+FF 2.85 (1.56\\u20135.22) 0.00004 Apa1 (rs7975232) (N = 297) (N = 160) HWE (p value) 0.8 0.09 AA 124 (41.75) 56 (35) 0.2716* Aa 138 (46.46) 86 (53.75) aa 35 (11.78) 18 (11.25) A allele 386 198 a allele 208 122 0.3864 AA vs Aa+aa 1.331 (0.8936\\u20131.983) 0.1619 aa vs Aa+AA 1.054 (0.5759\\u20131.928) 1.00 *p value for 3X2 \\u03c7 2 test for comparison among all the genotype frequencies for leprosy patients and healthy control. No significant difference was observed for Taq1, Fok1 and Apa1 genotype in reactional groups of leprosy when compared with non-reactional group of leprosy ( Table 3 ). 10.1371/journal.pntd.0006823.t003 Table 3 Genotype frequencies of VDR gene (Taq1, Fok1 and Apa1) polymorphism among Type 1 reaction (T1R), Type 2 reaction patients (T2R) and Non-reaction (NR) leprosy patients. Genotype/allele Type 1 Reaction (T1R) Type 2 Reaction (T2R) Non-Reaction (NR) OR (95% CI) P value of \\u03c7 2 test Taq1 N = 70 N = 11 N = 216 TT 25 5 81 0.64 a , 0.75 b Tt 39 4 99 0.46 a , 0.73 b tt 6 2 36 0.27 a , 1.00 b TT vs Tt+tt 0.9259 (0.5282\\u20131.623) a , 1.389 (0.4106\\u20134.698) b 0.89 a , 0.75 b tt vs Tt+TT 0.4688 (0.1886\\u20131.165) a , 1.111 (0.2303\\u20135.361) b 0.12 a , 1.00 b Fok1 N = 69 N = 13 N = 206 FF 28 8 101 0.24 a , 0.71 b Ff 36 3 93 0.72 a , 0.22 b ff 5 2 12 0.54 a , 0.32 b FF vs Ff+ff 0.71 (0.4084\\u20131.234) a , 1.663 (0.5264\\u20135.256) b 0.27 a , 0.41 b ff vs Ff+FF 1.263 (0.4284\\u20133.723) a , 2.939 (0.5841\\u201314.79) b 0.77 a , 0.19 b Apa1 N = 72 N = 13 N = 212 AA 28 3 93 0.94 a , 0.33 b Aa 40 9 89 0.20 a , 0.13 b aa 4 1 30 0.23 a , 1.00 b AA vs Aa+aa 0.8143 (0.4716\\u20131.406) a , 0.3839 (0.1027\\u20131.435) b 0.49 a , 0.16 b aa vs Aa+AA 0.3569 (0.1212\\u20131.051) a , 0.1704 (0.02180\\u20131.332) b 0.059 a , 0.068 b P value for 3x2 \\u03c7 2 test of comparison of overall genotype frequencies between a T1R and NR, b T2R and NR group of leprosy Haplotype association between the study groups The frequency of haplotype t-F-a was significantly higher in leprosy patients in comparison to HC (p<0.0001) suggesting a positive association of this haplotype ( Table 4 ) with the disease. No linkage disequilibrium was observed between SNPs tested. 10.1371/journal.pntd.0006823.t004 Table 4 Haplotype frequency distribution among leprosy patients and healthy controls. Haplotype Leprosy Patients Healthy Controls P value T-F-A 0.261 0.238 t-F-A 0.214 0.149 0.41 T-F-a 0.162 0.219 0.18 T-f-a 0.111 0.131 0.21 t-f-A 0.094 0.11 0.46 T-f-A 0.078 0.118 0.088 t-F-a 0.066 0 <0.0001 t-f-a 0.014 0.035 0.27 Level of vitamin D in the participants Average levels of vitamin D among healthy controls and leprosy patients are shown in Table 5 and Fig 1 . It is noted that there was no difference in the values of vitamin D levels amongst any of the groups ( Fig 1 ). We observed average level of vitamin D in healthy controls (mean\\u00b1SD = 35.30\\u00b122.02) and leprosy patients (mean \\u00b1 SD = 26.91\\u00b110.89) is also not significantly different. 10.1371/journal.pntd.0006823.g001 Fig 1 Graph showing levels of vitamin D in healthy controls and leprosy patients. mRNA expression profile of VDR gene in the study subjects. 10.1371/journal.pntd.0006823.t005 Table 5 Average level of vitamin D among healthy controls and leprosy patients. Healthy Controls (n = 31) Leprosy Patients TT/BT (n = 26) BL/LL (n = 21) T1R (n = 28) T2R (n = 13) Mean level of Vitamin D in ng/ml \\u00b1 Standard Deviation p value 35.30\\u00b122.02 28.60\\u00b111.31 p = 0.54 26.37\\u00b14.436 p = 0.28 25.89\\u00b111.78 p = 0.16 26.78\\u00b110.67 p = 0.42 The mean mRNA expression ratios of VDR/GAPDH are identified to be significantly lower in TT/BT and BL/LL group of leprosy in comparison to the HC (TT/BT vs. HC, 0.702 vs. 1.076 and BL/LL vs HC, 0.462 vs 1.076 p < 0.05) ( Fig 2 ). There was no significant difference observed between TT/BT and BL/LL groups of leprosy (TT/BT vs BL/LL, 0.702 vs 0.462 p>0.05). Highest mean mRNA expression ratio of VDR/GAPDH was observed in T2R group of leprosy however it is not significant in comparison to HC (T2R vs HC 5.62 vs 1.076, p>0.05) followed by T1R (1.044) ( Fig 2 ). Further, when we compared mean mRNA expression ratio of VDR/GAPDH between non-reaction (NR) and T1R/T2R groups of leprosy, it was noted that the ratio is not significantly different between the groups (NR vs T1R 0.5894 vs 1.044, p = 0.05; NR vs T2R 0.5894 vs 5.62, p = 0.9529). 10.1371/journal.pntd.0006823.g002 Fig 2 Comparative analysis of mRNA expression ratios of VDR/GAPDH in blood samples among the HC (n = 10), TT/BT (n = 17), BL/LL (n = 15), T1R (n = 17) and T2R (n = 13) using real-time PCR.\",\n",
            "        \"discussion\": null,\n",
            "        \"conclusion\": \"Discussion The association of SNPs of VDR gene (Taq1, Fok1 and Apa1), serum level of vitamin D along with VDR mRNA expression with the susceptibility to different phenotypes of leprosy was assessed in this study. It has been reported earlier that immunomodulatory role of vitamin D occurs through its receptor, VDR and it may help to promote appropriate innate immune responses [ 8 ]. Vitamin D has been shown to promote a T-cell shift from Th1 to Th2 [ 25 ] and transcription of antimicrobial peptides [ 26 ]. The polymorphisms in VDR gene (Taq1, Fok1 and Apa1) may affect mRNA stability which in turn affect protein levels and eventually may alter the balance of the production of the Th1 and Th2 related cytokines [ 25 ] which is a decisive factor for determining the clinical outcome of leprosy. Present study has indicated some interesting findings. Firstly, in our study population mainly from North India Taq1 and Fok1 association with leprosy patients was observed and noted that tt and ff are significantly more prevalent in leprosy patients in comparison to HC. Though power of association being a limitation of the study our observations suggest that ff and tt genotypes might be associated with the susceptibility with leprosy. Similarly, in an earlier study, ff genotype was found to be associated with leprosy in South-Indian population [ 11 ]. In another study, authors observed that Ff genotype frequency was highest in North Indian healthy population [ 10 ]. While, Neela et al (2015) reported that allele A and genotype AA and Aa are associated with leprosy [ 11 ]. We have not found any significant association of Apa1 genotype of VDR gene with leprosy. No association of Apa1 polymorphism has been shown to be associated with leprosy in Nepalese population [ 27 ]. However, allele A of Apa1 was found to be positively associated with tuberculosis in African population [ 28 ]. However, Taq1 polymorphism was not found to be associated with leprosy in south Indian population [ 11 ]. Similarly, Taq1 polymorphism was found to be associated with leprosy in Mexican population and Bengali Indian population [ 29 , 9 ]. It has also been reported that Taq1 polymorphism is associated with tuberculosis in Gambian population [ 30 ]. We also observed significant association of Taq1 polymorphism with leprosy. This difference of the associations might be due to different ethnic groups studied by the researchers. Further, we observed that t-F-a haplotype is associated with leprosy per se. As we know that persons affected with leprosy have bone deformities and manifest osteoporosis hence, we estimated the plasma level of vitamin D in leprosy patients and healthy controls. The present study showed no significant difference in the levels of vitamin D across the spectrum of leprosy including reactional cases in comparison to HC. Similarly, Ribeiro et al (2007) noted same serum level of vitamin D in leprosy patients in comparison to healthy controls [ 16 ]. To the best of our knowledge there is no report on vitamin D level in leprosy patients in India except one reported by Mandal et al (2015) wherein leprosy cases with reactions were shown to have a lower levels of vitamin D compared to HC [ 17 ]. Differences which have been noted in the level of vitamin D in these studies might be due to differences in cultural behaviors, exposure to sunshine and dietary intake of vitamin D in the participants. It has been reported earlier that incubation of human macrophages and monocytic cell lines with vitamin D inhibit intracellular growth of M . tuberculosis [ 31 ]. The mechanism could be that treatment with vitamin D might increase the differentiation and cytotoxicity in macrophages. In the present set up, such in vitro studies are not possible to perform with M . leprae . There are many contentious reports showing association between vitamin D levels and tuberculosis [ 32 , 33 ]. One recent study has reported the association of higher levels of vitamin D with active tuberculosis [ 34 ]. We suggest that sun exposure is the main source of vitamin D and most of the leprosy patients studied here having agricultural background were usually exposed to sun. Further, bone deformities and osteporotic changes in early and advanced stages of leprosy has already provided evidence that the bone damages are mainly due to direct invasion of bone by M . leprae [ 35 , 36 , 37 , 38 ]. We further observed a significantly lower VDR/GAPDH mRNA expression ratio in TT/BT and BL/LL groups of leprosy in comparison to HC group. Highest VDR/GAPDH gene expression ratio was observed in T2R cases in comparison to all the phenotypes of leprosy. In contrast to the present study earlier it has been reported that VDR mRNA expression was significantly lower in T2R group of leprosy patients in comparison to HC group [ 17 ]. It has also been reported that mRNA expression for VDR in skin lesions of lepromatous patients is lower in comparison to that with BT/TT leprosy and in T1R [ 39 ]. We found significantly lower VDR mRNA expression in tuberculoid and lepromatous group of leprosy while we noted no significant difference in the level of vitamin D. Low level of gene expression VDR gene in leprosy patients in comparison to HCs might be associated with the VDR gene polymorphism found in leprosy patients and one study Selvaraj et al (2009) have reported that vitamin D has no effect on the mRNA expression of VDR in tuberculosis patients [ 40 ]. It has been shown earlier that ethnicity influenced the VDR expression and serum level of vitamin D but did not influence VDR expression and its function in South African healthy population [ 41 ]. However, it has been shown by Zella et al (2010) that vitamin D induced the accumulation of VDR in osteosarcoma cells [ 42 ]. All these findings suggest that vitamin D status is dependent on multiple factors and VDR mRNA expression is not only dependent on vitamin D status in an individual but also on expression of multiple factors. This study demonstrated for the first time in north Indian population that Taq1 and Fok1 genotypes are associated with the leprosy and t-F-a haplotype could be useful to detect high risk group for leprosy. Further, low VDR gene expression level in patients might be responsible for impaired immune response in leprosy. Further, genotype-based study with a larger number of samples would be useful in understanding their role in genetic risk factor in leprosy. Present study demonstrated that SNP of VDR gene (Fok1 and Taq1) are associated with the leprosy per se and it might be responsible for lower VDR gene expression in leprosy. No association of Taq1, Fok1 and Apa1 with reactional group of leprosy was noted. t-F-a haplotype is significantly associated with leprosy per se. Haplotype analysis could be useful to screen the higher risk group of HCs and endemic population of leprosy patients. Surprisingly, no significant difference in the levels of vitamin D was noted amongst leprosy patients in comparison to healthy controls. VDR mRNA expression was found to be significantly lower in tuberculoid and lepromatous groups of leprosy. However, VDR mRNA expression was highest in T2R group amongst all the studied groups.\"\n",
            "    },\n",
            "    {\n",
            "        \"pmcid\": \"PMC6730009\",\n",
            "        \"abstract\": \"Bacterial\\u2013host interactions are non-linear and actually threefold, involving significant selection through predatory lytic bacteriophages in the host environment. In studies of human and animal gut microbiome bacteria, it is important to consider phage in all host\\u2013pathogen interactions. We use an important zoonotic pathogen, Shiga toxigenic\",\n",
            "        \"intro\": \"Introduction Bacteria are ubiquitous colonizers of plant and animal hosts. The increasing availability of large bacterial genome datasets is revealing variation in strains and species that reflect genetic isolation in, and adaptation to, the host ( Sheppard et al., 2018 ). It is tempting to consider the host-associated microbiota as a stable community, but this is not the case. Rather, the observed population sampled from a given host is a snapshot of stochastic mechanisms in an ongoing microbial war in which only the fittest survive to pass their genes onto the next generation. The ecological genomics of host\\u2013bacterial interactions is extremely complex with genetic variation in both the host and bacterium contributing to successful colonization and proliferation. Broadly, however, it is useful to consider the extant bacterial population as reflecting the outcome of competition with the host response, with other bacteria and with bacteriophage (viruses that infect bacteria). Shiga-toxigenic Escherichia coli (STEC) serotype O157:H7 is a zoonotic, foodborne pathogen that causes gastrointestinal disease in humans. Despite being relatively rare compared with other bacterial gastrointestinal infections, for example, Campylobacter and Salmonella , STEC O157:H7 infections are of significant public health concern due to potential disease severity. Symptoms range from abdominal cramps and nausea to severe bloody diarrhea and, in 5\\u201314% of cases, infection leads to hemolytic uremic syndrome (HUS), a severe and potentially fatal condition ( Adams et al., 2016 ). Cattle and other ruminants are natural reservoirs for STEC O157:H7, where the bacterium is part of the commensal microbiota, and transmission to humans occurs through direct or indirect animal contact or through consumption of contaminated food or water ( Byrne et al., 2015 ). Around 20% of the STEC O157:H7 genome is made up of prophage ( Hayashi et al., 2001 ) and the key virulence factor, stx , is encoded on lambdoid phages ( Scotland et al., 1985 ). The Shiga toxin ( stx ) can be divided into subtypes stx1a-1d and stx2a-2g ( Scheutz et al., 2012 ). The stx2a subtype is associated with adverse clinical outcomes, including progression to HUS ( Dallman et al., 2015 ). There are three major lineages of STEC O157:H7, lineage I, II, and I/II. The STEC O157:H7 clone is globally disseminated but analysis of the population structure reveals evidence of the expansion of certain lineages and/or sub-lineages in different geographical regions periodically during its evolutionary history ( Dallman et al., 2015 ). When STEC O157:H7 emerged in the UK in the 1980s, the dominant lineage associated with causing gastrointestinal disease in humans was I/II ( Dallman et al., 2015 ; Adams et al., 2016 ). During the 1990s, a rapid decline in the number of human STEC O157:H7 cases caused by strains belonging to lineage I/ll coincided with an equally rapid increase in cases caused by strains belonging to sub-lineage Ic, and a more gradual increase in cases caused by strains belonging to sub-lineage IIc ( Dallman et al., 2015 ; Adams et al., 2016 ). Over the last decade, a gradual decline in human cases caused by STEC O157:H7 sub-lineage Ic has coincided with an increasing trend in the number of cases caused by sub-lineage IIb ( Byrne et al., 2018 ). The emergence of sub-lineages I/II, Ic, and IIb as a cause of human disease coincided with the acquisition of stx2a integrated into the genome on a prophage. Bacteriophage display different host (bacterium) specificity with some phage having broad host ranges and others capable of only infecting a limited strain set. This phenomenon is exploited by phage typing, an in vitro method where the specificity of different bacteriophages is used to differentiate bacterial strains. The STEC O157:H7 phage typing scheme has been used at Public Health England (PHE) for more than 20 years as an epidemiological tool for the rapid detection and subsequent investigation of outbreaks. Following the implementation of whole genome sequencing at PHE, retrospective studies revealed that each bacterial sub-lineage broadly correlates with certain phage types (PT); lineage I/ll comprises PT2, sub-lineage Ic correlates with PT21/28, and the recent emergence of sub-lineage IIb coincided with a change in the dominant PT associated with this sub-lineage from PT4 or PT1 to PT8 ( Dallman et al., 2015 ; Byrne et al., 2018 ). The O157:H7 PT scheme is a phenotypic assay that produces a profile of susceptibility and resistance based on 16 typing phages, 14 T4-like lytic coliphages and 2 T7-like lytic coliphages. The results of phage typing can be condensed into susceptibility (defined as any kind of lysis with those phages) or resistance (defined as no lysis with those phages) for each of the typing phage similarity groups. These groups have been previously defined as T4 Group 1\\u2014TP1, 8, 11, and 15; T4 Group 2\\u2014TP3, 6, 7, and 13; T4 Group 3\\u2014TP4, 5, and 14; and T7\\u2014TP9 and 10 ( Cowley et al., 2015 ). In the same way that pathogenic bacteria and viruses are in a continual struggle to survive the human immune system of their hosts, bacteriophage are in a simultaneous arms race with bacteria to survive and pass on their genetic information through infection of their bacterial hosts. We propose that this dynamic has made a significant contribution to the population structure and therefore the evolutionary history of STEC O157:H7. Phage predation has posed an evolutionary pressure on this pathotype and certain phage resistance profiles have become fixed in the population. STEC O157:H7 is susceptible to lysogenic phage and is known to exclude some prophage through phage\\u2013phage interactions and superinfection immunity ( Labrie et al., 2010 ). Horizontal gene transfer, such as plasmid carriage, and prophage carriage provides resistance mechanisms to lytic phages, thereby conferring a selective advantage to those strains. The interaction of phage and bacterium within the reservoir host (ruminants) is a key factor in shaping the STEC O157:H7 population to which humans are exposed. However, there are multiple phenotypes that affect the population structure, such as super shedding, colonization in the terminal rectum, and host immunity. The population structure observed in human disease is not the natural ecology where selection takes place and different strain diversity is seen in cattle to people. However, we can use measurements of strains causing disease in humans as markers of prevalence in cows. Strains causing disease in humans also have to be successful in their primary ruminant hosts and can provide indications of the selective environment of the primary host. Heterogeneity in phage resistance profiles within the population structure of STEC O157:H7 can potentially be linked to short-term changes in accessory genome but can also become fixed as stable lineage associations. Here, by combining information from phage typing profiles ( Cowley et al., 2015 ) and whole genome sequences, we investigate how the fixation of resistance in lineages relates to phenotypes in the rest of the population structure. Observations of bacteria\\u2013phage interactions ( Cowley et al., 2016 ), in the context of a reconstructed phylogeny, are then used to understand how bacteriophage resistance and susceptibility can influence the evolutionary history of STEC O157.\",\n",
            "        \"method\": \"Materials and Methods Whole Genome Alignment and Recombination Removal Sequencing reads from 580 isolates of STEC O157:H7 from human cases in England and Wales, submitted to the Gastrointestinal Bacterial Reference Unit (GBRU) in 2017, were downloaded from the sequence read archive. The reads were trimmed ( Bolger et al., 2014 ) and mapped using BWA-SW ( Li and Durbin, 2010 ) to the STEC O157:H7reference strain Sakai (GenBank accession BA000007). The alignment map file from BWA was sorted and indexed into a binary alignment map (BAM) with Samtools ( Li et al., 2009 ), and GATK2 ( McKenna et al., 2010 ) was used to produce a Variant Call Format (VCF) file that was parsed to call SNPs of high quality (MQ \\u2265 30, DP \\u2265 10, GQ \\u2265 30, variant ratio \\u2265 0.9). Whole genome alignment files were used with Gubbins ( Croucher et al., 2015 ) to identify regions of high SNP density that were assumed to be areas of recombination. These recombination regions were removed in new alignment files and mutations outside of these regions were used to create a phylogeny that better reflect the clonal frame of the population using RAxML ( Stamatakis, 2014 ). Phage Resistance Phenotyping Public Health England (PHE) have used phage typing as a way of broadly discriminating and clustering strains of STEC O157 for more than 20 years. This involves a panel of 16 typing phages (TP) that the strains are assayed against to produce a resistance profile corresponding to phage type ( Cowley et al., 2015 ). Groupings can be used instead of individual phages as, in most cases, strains have the same reaction to all members of the similarity groupings. In those cases where there is heterogeneity of phenotype within the group, the stronger reaction/clearer phenotype has been chosen as representative. Briefly, STEC O157:H7 strains for typing are flooded on a Difco nutrient agar plate with 1 ml of broth culture. Dried plates are spotted with 10 \\u00b5l of each of the 16 typing phages following a template and incubated overnight. Profiles of lysis for each of the typing phages are then recorded the next day. Stx Prophage Carriage Determination Stx profiling is an important assay in public health surveillance of STEC O157 and is carried out routinely at PHE who provided data for this study. Shiga toxin subtyping is performed through a mapping technique to reference Shiga toxin sequences as previously described ( Ashton et al., 2015 ). Briefly, reads were mapped to stx1a, stx1c, 1d, 2a, 2b, 2c, 2d, 2e, 2f , and 2g sequences (taken from Scheutz et al., 2012 ) using BWA-MEM ( http://bio-bwa.sourceforge.net/ ). Reads that mapped to more than one place in the reference set (i.e., ambiguous reads) were removed from the resultant SAM file using Samtools ( Li et al., 2009 ). If at least 10 reads and 90% of the total reads concordantly mapped to discriminatory positions for a specific subtype, then a positive match was returned for that subtype. OmpC Allele Determination The OmpC coding region was identified in strain 9000 (GenBank accession CP018252.1, protein ID API32766.1, locus tag BFL16_\\u200b16860) ( Shaaban et al., 2016 ). This reference sequence was used to query all the STEC 2017 assemblies using blastn with e-value 0.0000000001. One hundred percent identity matches and coverage were seen for all clade Ic isolates and all other clade isolates had a T at position 921 instead of the C seen in strain 9000.\",\n",
            "        \"subjects\": null,\n",
            "        \"results\": null,\n",
            "        \"discussion\": null,\n",
            "        \"conclusion\": \"Conclusions Chromosomal background determinants of phage susceptibility phenotypes are likely to be established and fixed in specific lineages and sub-lineages at the time of divergence, but transient susceptibility and resistance profiles observed in these lineages may be shaped by the current niches that they occupy. This relationship between clone and phage population in environmental niche is likely to be a significant driver in shaping the population structure of bacteria including STEC O157:H7. Further phage susceptibility testing would expand the work described in this perspective. Additional testing of different lytic phages that infect STEC O157:H7 may provide a better understanding on how phage populations in different environmental niches and animal reservoirs influence the evolution and emergence of the STEC O157:H7 sub-lineages causing the greatest burden of gastrointestinal disease. In conclusion, our results have shown that predatory phage effects in the primary ruminant host affect strains causing disease in the secondary human host and may maintain diversity in strains observed in human disease worldwide.\"\n",
            "    },\n",
            "    {\n",
            "        \"pmcid\": \"PMC6760014\",\n",
            "        \"abstract\": \"This study aimed to explore the psychophysical bases of multisensory surface stickiness perception by investigating how sensitively humans perceive different levels of stickiness intensity conveyed by auditory, tactile, and visual cues. First, we sorted five different sticky stimuli by perceived intensity in ascending order for each modality separately and evaluated the discrimination sensitivities of each participant using a fitted psychometric curve. Results showed that perceptual intensity orders were not identical to physical intensity order and that the sequential order of perceived intensities for different modalities was inconsistent. Moreover, estimated perceptual sensitivities to surface stickiness indicated that auditory cues result in better discrimination sensitivity than tactile and visual cues. Second, we calculated the relative perceptual distances of stickiness intensities using multidimensional scaling. A follow-up statistical test demonstrated that the perceptual mapping of vision and touch are similar but that auditory perception is different. These results suggest that the discriminability of stickiness intensity is best served by auditory cues and that texture information processing in the auditory domain is distinctive from that of other modalities.\",\n",
            "        \"intro\": null,\n",
            "        \"method\": \"Materials and Methods Participants and Ethics Approval Twenty-seven volunteers (10 females, 24.52 \\u00b1 2.79 years old, age range: 20\\u201334 years) participated in this experiment. All participants were all right-handed and had no deficits in auditory, tactile, and visual processing. Experimental procedures were approved by the Ethical Committee of Sungkyunkwan University (IRB# 2018-05-001) and the study was conducted in accordance with the Declaration of Helsinki. All participants were informed about the experimental procedure and gave written informed consent prior to their participation. Stimuli Five different kinds of repositionable tape (9415PC, 665, 9183, 9495, and 9071; 3M Center, St. Paul, MN, United States) were prepared for the experiment. To measure physical stickiness strength of these tapes, we employed a \\u201cprobe tack test\\u201d that measures the peak value of adhesive force indicating the instantaneous adhesion property. Using a probe in the shape of a stainless steel ball (1-inch diameter), we measured the adhesive force that occurs when a probe is peeled off at peeling rate 2 mm/sec. Previous studies have considered this test as a qualitative approach to evaluate tactile sensations of human ( Mith et al., 2008 ; Cakmak et al., 2011 ). The physical stickiness intensities of tape 9415PC, 665, 9183, 9495, and 9071 were estimated as 22.9, 124.5, 330.0, 419.2, and 558.7 gf (gram-force), respectively. According to these values, we labeled five different physical stickiness intensities as level 1 to level 5: 9415PC (level 1), 665 (level 2), 9183 (level 3), 9495 (level 4), and 9071 (level 5). With these tapes, we created auditory, tactile, and visual stimuli. \\u2022 Auditory stimuli: Sound clips of tactile explorations (i.e., a sound when touching and detaching with an index finger) were recorded with a mobile condenser microphone. Each clip lasted 3 s and was sub-divided into a touching and then a detaching period of about 1.5 s each. \\u2022 Tactile stimuli: Tapes were prepared in a size of 5 \\u00d7 1.9 cm and attached to an acrylic plate sized 5 \\u00d7 8 cm. The plastic plate was used to enable the experimenter to present the stimuli easily and without direct contact. All tactile stimuli were used only once, then replaced by new tape. \\u2022 Visual stimuli: Video clips of tactile explorations were recorded with a resolution of 1920 \\u00d7 1080 at 60 frames per second ( Figure 1 ). Each clip displayed a right index finger touching one of the five tapes and lifting off. The recording video camera was positioned at a distance of 20 cm from the stimulus surface and 5 cm above the tabletop. Each video clip lasted 3 s. FIGURE 1 Structure and time course of the experimental design. Participants completed five blocks per modality condition. Following the instruction on the screen, participants perceived stickiness intensity levels through the sensory cues, i.e., auditory, tactile, and visual cues. In each trial, two stimuli were presented one after another for 3 s each and then participants were asked to respond to questions (1) \\u201cDoes the second stimulus feel stickier than the first one?\\u201d and (2) \\u201cHow much stickier was the second stimulus compared to the first one?\\u201d. Note that we did not match the absolute perceptual intensities across sensory modalities. We were aware of that stimulus intensities can be perceived in a different scale depending on the modalities, but the primary consideration was that our stimuli should be expressed its unique characteristics as fully as possible. Experimental Design Twenty-five paired stimuli (counterbalanced pairwise combinations of five distinct intensities, plus five pairs of the same intensity; 5 P 2 + 5 = 25) were used for each modality condition and these pairs were presented once in each block. Participants performed 5 blocks for each modality, thus there were a total of 375 trials (3 modalities \\u00d7 5 blocks \\u00d7 25 pairs). A single trial presented two stimulus pairs one after another for 3 s and there was a pause of 1.5 s between them ( Figure 1 ). Participants were instructed to perceive stickiness intensity from the sensory cues (auditory, tactile, and visual information) and to conduct two different tasks, i.e., 2-Alternative Forced Choice (2-AFC) and magnitude estimation of perceived dissimilarity. The 2-AFC and the magnitude estimation are the common psychophysical methods for measuring the subjective experience ( Lederman and Klatzky, 2009 ). In tactile conditions, participants performed tasks following the instruction \\u201cTouch\\u201d and \\u201cDetach\\u201d displayed on a monitor. To give more detail, a tactile stimulus was given to the participants prior to the stimulation periods. When \\u201cTouch\\u201d was on the monitor, participants put their index finger on the tape and lifted off as soon as they saw \\u201cDetach\\u201d on the monitor. \\u2022 2-AFC: Participants were asked to respond to the question \\u201cDoes the second stimulus feel stickier than the first one?\\u201d If the second stimulus was felt to be stickier, participants pressed a \\u201cYes\\u201d button on the keyboard, otherwise they pressed a \\u201cNo\\u201d button. \\u2022 Magnitude estimation: Participants were asked to respond to the question \\u201cHow much stickier was the second stimulus compared to the first one?\\u201d The response range was from \\u221210 to 10. Participants reported a negative value if the second stimulus was felt to be less sticky and a positive value if the second stimulus was felt to be more sticky. They also could respond 0 if the pair of stimuli were felt to be of the same intensity. The larger the response value, the greater the difference in perceived intensity between the stimulus pair. A short break was provided between the blocks and the entire experiment took approximately 60 min. The presentation order of stimuli as well as modality was randomized to remove ordering effects. Note that we did not notify participants of the number of intensity levels throughout the entire experiment ( Supplementary Material ). Data Analyses 2-Alternative Forced Choice We first computed the sequential order of perceived intensity for the five sticky stimuli to find a relationship between stimulus intensity and sensation. In addition, to fit a psychometric curve to the behavioral responses, we calculated \\u201cYes\\u201d answer rates for each of the stimulus pairs. \\u2022 Ordering perceived intensity: To test whether the order of perceived intensity is consistent with the physical intensity order, we compared the average number of \\u201cYes\\u201d responses for each stimulus pair. Our experiment asked the participants to answer whether the secondly presented stimulus was felt stickier than the first one. To minimize these sequential effects of stimulus presentation, we presented each pair of stimuli in both ascending and descending order, e.g., level 3 - 4 and level 4 - 3. For example, if the average number of \\u201cYes\\u201d across participants was 4.13 for a stimulus pair of levels 3 - 4 and 2.58 for a pair of level 4 - 3, we considered that the perceived intensity of level 4 was greater than the perceived intensity of level 3. Through this process of comparing average number of \\u201cYes\\u201d answer for every stimulus pair, the ascending order of perceived intensity of stimuli was obtained. \\u2022 \\u201cYes\\u201d answer rates: These rates were derived by the ratio of the number of \\u201cYes\\u201d responses to the number of stimulus pairs presented. For example, if the \\u201cYes\\u201d answer rate was 0.3, it meant that the second stimulus was perceived to be stickier three times when a stimulus pair is presented ten times. Psychometric curves were fitted to the \\u201cYes\\u201d answer rates. In particular, to set the values of the x -axis, we assumed that the intensity difference between perceptually adjacent stimuli has a quantity of 1 regardless of the actual physical intensity difference. Hence, the x -axis was set to indicate the perceived intensity difference ranging from \\u22124 to 4, and the y-axis indicated the \\u201cYes\\u201d answer rates. A psychometric function was derived using the Palamedes toolbox for Matlab, which implements a maximum-likelihood method ( Wichmann and Hill, 2001 ; Prins and Kingdom, 2018 ). It showed a cumulative probability that the stickiness intensity of the second stimulus was perceived to be stickier than the first stimulus, as a function of its relative stickiness. Therefore, a steeper slope indicated a better discriminability. Specifically, the discriminability of each participant was estimated as a difference of perceived intensity values between the 25th and 75th percentiles. This difference is known as the just noticeable difference, i.e., JND. To determine the reliability of the JND, we estimated each participant\\u2019s goodness of fit. We fitted a logistic function to the psychometric curve and the individual deviances were evaluated to measure goodness of fit ( Fruend et al., 2011 ). A 95% confidence interval was calculated based on simulations from the bootstrapping procedure ( n = 1000). If the observed deviance was outside the 95% confidence limit, we considered the participant\\u2019s data as an outlier. Magnitude Estimation To compute a dissimilarity matrix for each modality, we normalized participants\\u2019 responses in the magnitude estimation tasks so that the mean value became 0 and the standard deviation became 1. The normalized responses were averaged across participants to obtain a group-level dissimilarity matrix. An entry at the j -th row and the k -th column of the dissimilarity matrix was the average perceived difference in a normalized scale when the j -th intensity was presented followed by the k -th intensity. Then, an MDS was applied to this dissimilarity matrix to yield the spatial organization of perceptual responses for the five different stickiness intensities. To further examine the correspondence of perceived dissimilarities between sensory modalities, we employed the Mantel test to calculate a correlation between the matrices ( Saarimaki et al., 2016 ). The significance was tested from the probability distribution obtained from 10,000 repeated permutations. For each permutation, we permuted the values of the dissimilarity matrix and determined the expected distribution of the statistics under the null hypothesis. The probability of the observed correlation arising by chance was then yielded by observing where the statistic calculated from our data fell in the permuted distribution.\",\n",
            "        \"subjects\": null,\n",
            "        \"results\": null,\n",
            "        \"discussion\": null,\n",
            "        \"conclusion\": null\n",
            "    },\n",
            "    {\n",
            "        \"pmcid\": \"PMC6775309\",\n",
            "        \"abstract\": null,\n",
            "        \"intro\": \"Introduction Women infected with HIV are more likely to have aggressive cervical cancer, and those with HIV infection at the time of consultation are often more serious cases than those without HIV infection 1 - 5 . Compared with that of non-HIV-infected women, the infection rate of HPV in the reproductive tract of HIV-infected women is also significantly higher. This double infection further decreases the cellular and humoral immune functions, resulting in a decrease in HPV clearance ability 6 - 9 . Persistent HPV infection leads to a significant increase in the incidence of cervical epithelial precancerous lesions, which eventually leads to the occurrence and rapid progression of invasive cervical cancer 10 - 12 . It has been found that HPV DNA exists in a free state in benign lesions caused by low-risk HPV, while most HPV DNA is integrated into the host cell DNA in cervical cancer caused by high-risk HPV, which may be one of the reasons for the high incidence of invasive cervical cancer caused by HIV combined with HPV infection 13 - 15 . Intercellular communication is a fundamental process of multicellular organisms to complete their normal physiological functions. Intercellular communication can transmit information through direct cell contact or signal molecules secreted by cells. In the past two decades, a third way of intercellular communication has been proposed that mediates intercellular signal transduction through extracellular vesicles (EVs) 16 - 18 . Although the release of apoptotic bodies during apoptosis has been reported for a long time, the release of vesicles from cell membranes by healthy cells has been recently identified. Exosomes are extracellular vesicles surrounded by a phospholipid bilayer with a diameter of 50-100 nm that can be released or induced by resting cells. The release of exosomes is caused by the formation of a multivesicular body (MVB) by inward budding into the lumen and then by the fusion of the multivesicular body and the cell membrane 19 - 21 . Exosomes not only transports RNA, microRNAs, oncogenic receptors and HIV particles horizontally but also have antigen presenting, immune activation and immunosuppressive activities, expressing D63, CD81, CD9, LAMP1, TSG101 and other markers 22 - 26 . MicroRNAs (microRNAs) are endogenous noncoding RNAs with regulatory functions and are found in eukaryotic organisms 27 , 28 . They are approximately 20-25 nucleotides in size. Recent studies have shown that microRNAs are involved in a variety of regulatory pathways, including development, viral defense, hematopoietic processes, organogenesis, cell proliferation, apoptosis, fat metabolism, and so on 29 - 31 . In this study, we found that exosomes derived from HIV-1-infected cells carrying miR-155-5p could promote the malignant progression of cervical cancer cells and thus constitute pathogenic mechanisms to promote the progression of HIV-1-associated cervical cancer.\",\n",
            "        \"method\": \"Materials and Methods Cell culture and siRNA transfection The J1.1 cell line was obtained from the NIH AIDS Reagent Program. Jurkat cells and human cervical cancer cells CaSki, Siha and HeLa were cultured in high sugar HyClone DMEM. Ten percent fetal bovine serum (Cell-Box) and 1% penicillin-streptomycin were added to the basic DMEM. All cell lines were cultured in a humidified incubator containing 5% CO 2 at 37\\u00b0C. Lipofectamine RNAiMAX was used for siRNA transfection. The sequences of the siRNA used to suppress ARID2 expression were as follows: sense 5'-GGCCGAGGAAUGGGCUCCGTT-3' and antisense 5'-CGGAGCCCAUUCCUCGGCCTT-3'. The scramble control siRNA sequences were as follows: sense 5'-UUCUGGGAACGUGACACGUTT-3' and antisense 5'-ACGUGACACGUUCGGAGAATT-3'. Exosome quantification and identification Exosomes were prepared from cell supernatants by differential ultracentrifugation with filtration steps. Cell culture medium (CM) was a mixture of protease inhibitors with a concentration ratio of 1:1,000, which was derived from cells at 80% confluence with a sterile fusion degree. CM was quickly filtered with a 0.22 \\u00b5m filter (Millipore) to separate intact cells and debris. Ultraspeed centrifugation at 120,000 g and 4 \\u00b0C for 2 hours was performed. The vesicles were washed and resuspended in 1 mL cold PBS and then centrifuged at 120,000 g 4\\u00b0C for 2 hours. Then, the pellet was resuspended in 100 ml cold PBS and transferred to a tube with low adhesion. A drop of the exosome suspension was placed on a copper net, incubated at room temperature for 5 minutes, stained with 2% uranyl acetate for 30 seconds, then dried for 60 minutes, and observed by transmission electron microscopy. Extraction of RNA from exosomes After the exosomes were extracted by ultracentrifugation, 50 \\u00b5l PBS was used to resuspend the exosome pellet. The samples were mixed with 700 \\u00b5l pyrolysis solution, scraped, homogenized, and incubated at room temperature for 5 minutes. After incubation at room temperature for 3 min, the samples were centrifuged at 4\\u00b0C for 15 min at 1.2\\u00d710 4 g. The upper liquid phase was transferred to a new EP tube (avoiding contacting the middle layer), 1.5 times the volume of absolute ethanol was added, and the sample was mixed by pipetting. Then, 700 \\u00b5l of sample was placed in the filter column of a 2 mL collection tube. The cap was tightened and the sample was centrifuged at room temperature (>8 000 g) for 15 seconds to remove the effluent. Then, 700 \\u00b5l of RWT detergent was added to the filter column, the cover was tightened, and the effluent was removed after centrifugation at room temperature (>8 000 g) for 15 seconds. RPE eluent (500 \\u00b5l) was added to the filter column, the cover was tightened, and the effluent was removed after centrifugation at room temperature (>8 000 g) for 15 seconds. Next, 500 \\u00b5l 80% ethanol was added to the filter column, and the cap was tightened. After centrifugation at room temperature (>8 000 g) for 2 min, the effluent and collection tube were removed. Then, the filter column was placed into a new 2 mL collection tube, the cap was opened, and the sample was centrifuged at room temperature for 5 min at 1.6\\u00d710 4 g. The membrane was dried, and the liquid and collection tube were discarded. Then, the filter column was put into a new 1.5 mL collection tube, 14 \\u00b5l of enzymatic water was added to the center of the filter column membrane, the column was lightly covered, the sample was centrifuged at 1.6\\u00d710 4 g for 1 min, and the RNA was eluted. RNA isolation and qRT-PCR Total RNA was isolated using TRIzol reagent (Invitrogen) according to the manufacturer's instructions. Reverse transcription reactions were performed with the PrimeScript RT reagent Kit (TaKaRa). Quantitative PCR was performed with the SYBR Premix ExTaq Kit (TaKaRa) on a CFX96 Real-Time System (Bio-Rad) by following the manufacturer's instructions. The following primer sequences were used to detect each gene: 5'-ACTCACCTCTTCAGAACGAATTG-3' and 5'-CCATCTTTGGAAGGTTCAGGTTG-3' for IL6, 5'-TGGACCCCAAGGAAAACTGG-3' and 5'-ATTTGCTTGAAGTTTCACTGGCA-3' for IL8, 5'-CAGTGTGTCGGATTATCTGCG-3' and 5'-GCATGACGTGCTTGCTTTCATT-3' for ARID2, 5'-ACTGCGGATCTCTGTGTCAT-3' and 5'-AGAGTCCCTGCATCTCAGAGT-3' for TGFB1, 5'-AACGAGACGACGACAGACTTTTTTTTTTTTTTTV-3' for miRNA reverse transcription, universal miRNA qRT-PCR primer 5\\u2032-AACGAGACGACGACAGAC-3\\u2032, 5\\u2032-GCAAATTCGTGAAGCGTTCCATA-3\\u2032 for RNU6, and 5'-CGGGTTAATGCTAATCGTGATAGGGGTT-3' for hsa-miR-6857. PCR was performed under the following conditions: 95 \\u00b0C for 15 s, 40 cycles of 95 \\u00b0C for 5 s and 60 \\u00b0C for 30 s. The relative expression level of each probed mRNA was then normalized to the expression level of the housekeeping gene using the 2 -\\u2206\\u2206CT method. Cell proliferation, migration, and invasion Five thousand cells were added to each well of 96-well plates, and each group was repeated three times. When cells adhered to the wells, the OD value was recorded at 0 h. The OD value at 450 nm was measured for the first time. Then, it was measured five more times every 24 hours. To measure the OD value, the culture medium in the 96-well plate was first removed, 10% CCK8 solution was added to fresh culture medium, and 100 \\u00b5L premixed solution was added to each well to avoid bubbles. The OD 450 nanometer value was measured after standing in an incubator at 37\\u00b0C for 1 hour. Immunoblotting assay Total cellular protein was extracted using RIPA lysis buffer with PMSF and phosphatase inhibitor solution. The concentration of the proteins was determined by a BCA protein quantitative kit. The protein samples (40 \\u03bcg/lane) were separated by SDS-PAGE and transferred onto activated PVDF membranes (Invitrogen, CA). After blocking with 5% skim milk, the membranes were incubated with primary antibodies at 4\\u00b0C overnight. The primary antibodies and dilutions were as follows: anti-ARID2 (1:1000, Cell Signaling Technology, USA), anti-CD63, anti-CD9, anti-p-ERCC5, anti-CD81, anti-p-NF-KB, and anti-NF-KB (1:500, Proteintech, USA). After washing with TBST, the membranes were incubated with the appropriate secondary antibodies. The protein bands were visualized with an ECL kit (Thermo, USA). High-throughput sequencing analysis After running the sequencing program, clean data were obtained by filtering the raw reads. The clean data were aligned to the reference genome and a small RNA database by using the alignment software AASRA (Anchor Alignment-based Small RNA Annotation). AASRA is based on a new alignment algorithm, CG_anchor alignment, which adds a unique anchor sequence to both ends of the sequence and the reference sequence and then aligns them to obtain quantitative results of the expression of unique reads. Small RNAs were identified and classified by comparison with known databases such as microBase, Rfam11.0, UCSC and piRNA Bank. To assign a unique annotation to each small RNA, the small RNAs were annotated in sequence according to the priority order microRNA > piRNA > snoRNA > Rfam > other small RNA. New microRNAs were predicted using the software microDeep2. The TPM algorithm was used to standardize the expression level of small RNAs, and DEGseq software was used to screen the differentially expressed small RNAs. The differentially screened microRNAs were predicted by using miRanda, TargetScan and starBase software. GO significant enrichment analysis and KEGG pathway significant enrichment analysis were carried out with the predicted target genes of the differentially expressed microRNAs to further understand the biological functions of the genes and determine the main biochemical metabolic pathways and signal transduction pathways in which target genes of differentially expressed microRNAs participate. To determine whether there was a significant difference in the expression of microRNAs between two groups of exosome samples, the log2 ratio was used to compare the difference in the expression of microRNAs, and | log 2 (fold change) | (> 1) was significant. Statistical analysis All statistical data were analyzed by SPSS 23 statistical software. All experimental procedures in this study were repeated more than two times. Comparisons between two groups were made by Student's t -test, while comparisons among more than two groups were made by one-way ANOVA. Data were statistically significant when the p value was less than 0.05.\",\n",
            "        \"subjects\": null,\n",
            "        \"results\": \"Results Characterization of exosomes derived from HIV-infected T cells We isolated and purified exosomes from a latently HIV-1-infected human Jurkat T-cell line and the respective non-HIV control Jurkat cells by the standard exosome isolation method of ultracentrifugation at 100,000 g for 16 h at 4\\u00b0C. The structure and size of the particles isolated from cell culture supernatants were determined by electron microscopy and NanoSight particle tracking analysis (Fig. 1 A, B). Interestingly, we found that many more exosomes were secreted from latently HIV-1-infected cells than non-HIV control cells by quantification analysis of exosomes isolated from an equal number of cells. The exosome markers HSP70, CD63, CD9, and CD81 were further detected in exosomes from all four cell lines by immunoblotting (Fig. 1 C). HIV-infected T-cell-derived exosomes promote cervical cancer cell proliferation, migration and invasion To detect the transport of exosomes, we cocultured the exosomes isolated from culture supernatants of HIV-1-infected J1.1 cells or from control Jurkat cells and cervical cancer cells CaSki, SiHa, and HeLa. J1.1 cell-derived exosomes significantly promoted the proliferation and invasion of cervical cancer cells compared with control Jurkat cell-derived exosomes (Fig. 2 A, B). Interestingly, cervical cancer CaSki cells cocultured with exosomes from J1.1 cells expressed higher levels of proinflammatory genes, such as IL-6, IL-8, TGF-\\u03b2, collagen type I (COL1A2), and matrix metallopeptidase 2 (MMP2) (Fig. 2 C), which play an important role in regulating the inflammatory microenvironment and promoting the progression of cervical cancer. In conclusion, these results demonstrate that exosomes from HIV-infected Jurkat cells contribute to the malignant progression of cervical cancer. MiR-155-5p in HIV-infected T-cell exosomes mediates cervical cancer progression It has been reported that microRNAs are abundant in exosomes and play a key role in cell-to-cell communication 32 . Therefore, we hypothesized that microRNAs in HIV-infected T-cell exosomes promote the proliferation, invasion and metastasis of cervical cancer cells. To identify which miRNAs were involved, we conducted an Affymetrix multispecies miRNA-4 array to detect the miRNA expression profile of exosomes derived from J1.1 cells and Jurkat cells. A total of 474 significantly differentially expressed miRNAs were compared and are shown as heatmaps in Fig. 2 A. Twenty-one of the most upregulated miRNAs (fold change > 10) in J1.1 cell-derived exosomes were subjected to validation (Fig. 2 B). Only miR-155-5p mimics promoted inflammatory gene (IL-1\\u03b2, IL-6, and IL-8) expression in CaSki cells (Fig. 2 C). To further investigate the role of miR-155-5p, CaSki cells were stably transfected with miR-155-5p inhibitor (Supplementary Fig. 1 A). As expected, the effect of miR-155-5p on cervical cancer cells was abolished by its specific inhibitor (Fig. 2 D). Collectively, these findings reveal that J1.1-derived exosomal miR-1247-3p promotes the proliferation, invasion and metastasis of cervical cancer cells (Fig. 2 E, F). MiR-155-5p in HIV-infected T-cell exosomes directly targets ARID2 in cervical cancer cells miRDB and microTCDS (bioinformatics tools) were used to predict common target genes of exosomal miR-155-5p. The AT-rich interactive domain (ARID2)-containing family of DNA-binding proteins was verified to be a direct target of miR-155-5p and responsible for the progression of cervical cancer (Fig. 4 A). ARID2 expression could be down-regulated in cervical cancer cells by miR-1247-3p at both the mRNA and protein levels (Fig. 4 B). Potential binding sites of microRNA-155 were found in the ARID2 coding sequence by sequence alignment. Subsequently, we constructed dual luciferase vectors containing wild type and mutant type binding site of microRNA-155 (Fig. 4 C). We found that luciferase activity decreased markedly in CaSki cells cotransfected with the wild-type binding site vector and miR-155-5p mimics. However, the luciferase activity of the mutated binding site vector was not affected by miR-155. The luciferase activity significantly decreased in CaSki cells transfected with the wild-type binding site ARID2 vector after treatment with J1.1-derived exosomes (Fig. 4 D). These results reveal that ARID2 is a direct target of miR-155-5p in cervical cancer. To identify the effect of ARID2 in cervical cancer, we knocked down ARID2 expression in CaSki cells using targeting siRNAs, and the siRNA interference efficiency was determined by qRT-PCR and immunoblotting analyses (Supplementary Fig. 1 B). Migration assays and inflammatory gene expression analyses showed that silencing of ARID2 promoted cell migration and expression of inflammatory genes (Fig. 4 E, F). Moreover, overexpression of ARID2 (Supplementary Fig. 1 C) abolished the effect of miR-155-5p on migration (Fig. 4 G). These data demonstrate that miR-155-5p directly targets ARID2 and mediates cervical cancer invasion. RNA-sequencing analyses revealed that the ARID2-ERCC5-NF-\\u03baB axis promotes the progression of cervical cancer To clarify the downstream molecular mechanism of ARID2 in promoting cervical cancer progression, RNA sequencing was used to compare differential gene expression after ARID silencing. Bioinformatics analysis revealed that there were 100 significant differentially expressed genes after ARID2 silencing, including 205 significantly upregulated and 155 downregulated genes (fold change>4) (Fig. 5 A). GO and pathway enrichment analyses revealed that the differentially expressed genes were mainly involved in NF-kB signaling pathways (Fig. 5 B, C). MiR-155-5p expression or ARID2 suppression also stimulated NF-\\u03baB phosphorylation and I\\u03baB\\u03b1 downregulation (Fig. 5 D). Further analysis showed that ERCC5 was a key downstream target gene for ARID2, and the protein interaction was also verified (Fig. 5 E). Furthermore, we determined that ARID2 and ERCC5 bound to each other in cells by co-IP. (Fig. 5 F). In conclusion, these data demonstrate that exosomal miR-155-5p suppresses ARID2 expression via ERCC5-NF-\\u03baB signaling to promote invasion in cervical cancer.\",\n",
            "        \"discussion\": \"Discussion A high incidence of cancer in HIV-infected people is known as HIV-associated malignancy (HAM). According to the current classification criteria by CDC, HAM can be divided into two categories: AIDS-defining cancers (ADCs) and non-AIDS-defining cancers (NADCs) 33 - 36 . ADC currently includes cervical cancer, non-Hodgkin's lymphoma and Kaposi sarcoma (KS). It was found that these ADCs were co-infected with malignant transformed cells and viruses. Because there are too many variables involved, it is difficult to establish models for relevant research 37 - 39 . Currently, more than 80% of cancers are associated with infection. However, WHO data show that only approximately 23% of cancers are directly related to pathogenic infections. According to this, cancer occurrence must be based, at least in part, on the indirect effect of pathogen infection 40 - 43 . We can reasonably speculate that an indirect effect of infection would include at least one mechanism: the inflammatory response can be mediated by the infected cells, and then inflammatory cytokines will be enriched in the infected areas, ultimately leading to an inflammatory microenvironment conducive to the proliferation, differentiation or dedifferentiation of cancer cells. Here, we found that HIV-1-infected T cell-derived exosomes contained miR-155-5p that promoted the proliferation, migration, and invasion of cervical cancer cells in vitro and in vivo. Most importantly, we determined that miR-155-5p was directly transferred from HIV-1-infected T cells to cervical cancer cells via exosomes and promoted invasion by decreasing the expression of its target gene ARID2 to activate the ERCC5-NF-\\u03baB signaling pathway. Exosomal miR-155-5p derived from HIV-1-infected T cells promotes the malignant progression of cervical cancer by inducing the secretion of IL-1, IL-6 and IL-8. Among these factors, IL-6 has multiple biological effects. IL-6 is an important regulator in different kinds of chronic inflammation, but in addition to activating the immune function of the body to tumors, its main effect is to directly activate STAT3 to promote the proliferation of cancer cells and to inhibit apoptosis to help the survival of cancer cells 44 - 46 . The crosstalk between HIV-1-infected T cells and cervical cancer further elucidates the molecular mechanism of malignant progression. In conclusion, our results indicate that HIV-infected T cells can release exosomes potentially to stimulate cancer cell proliferation and establish the inflammatory tumor microenvironment, thus enhancing the growth and progression of cervical cancer. In addition, our data demonstrate that HIV-infected T cell-derived exosomal miR-155-5p promotes the progression of cervical cancer by the ARID2-ERCC5-NF-\\u03baB signaling pathway. MiR-155-5p induces an increased secretion of IL-6 and IL-8, thereby promoting stemness, EMT, chemoresistance, and tumorigenicity of tumor cells. Our results indicate that HIV-infected patients under ART treatment contain circulating pro-tumor exosomes and that HIV-specific exosome components contribute to the tumor-promoting effect. Our study provides the underlying molecular biological mechanism for the crosstalk between HIV-infected T cells and cervical cancer, which provides a new therapeutic target for the effective prevention and treatment of AIDS-defining cancers.\",\n",
            "        \"conclusion\": null\n",
            "    },\n",
            "    {\n",
            "        \"pmcid\": \"PMC6853912\",\n",
            "        \"abstract\": \"Low vitamin D status is associated with an increased risk of renal cell carcinoma (RCC). This study investigated the association of vitamin D status with serum C-reactive protein (CRP) and adhesion molecules among RCC patients. Fifty newly diagnosed RCC patients and 100 age- and sex-matched controls were recruited. As expected, serum 25(OH)D level was lower in RCC patients than in controls. By contrast, serum levels of CRP, an inflammatory molecule, and ICAM, LAMA4 and EpCAM, three adhesion molecules, were higher in RCC patients than in controls. All RCC patients were divided into two groups: H-VitD (>20\\u2009ng/ml) or L-VitD (<20\\u2009ng/ml). Interestingly, the levels of serum CRP and all adhesion molecules were higher in RCC patients with L-VitD than those with H-VitD. Nuclear vitamin D receptor (VDR) was downregulated and nuclear factor kappa B (NF-\\u03baB) was activated in cancerous tissues. The\",\n",
            "        \"intro\": \"Introduction Renal cell carcinoma (RCC) is common malignant tumor in urinary tract, among which clear cell RCC (ccRCC) is the most common subtype 1 , 2 . Increasing data suggest that hypertension, obesity, long-term sodium intake, high glycemic index diet, and cigarette smoking are the major risk factors for RCC 3 \\u2013 7 . Numerous studies have demonstrated that inflammatory molecules, such as C-reactive protein (CRP), promote RCC progression 8 \\u2013 10 . Other studies have indicated that inflammatory cytokines, such as tumor necrosis factor alpha (TNF-\\u03b1), lead to a poor prognosis 11 . On the other hand, adhesion molecules, such as intercellular adhesion molecule (ICAM)-1, vascular cell adhesion molecule (VCAM)-1, melanoma cell adhesion molecule (MCAM), and epithelial cell adhesion molecule (EpCAM), are involved in invasion and migration of malignant tumors 12 \\u2013 15 . In addition, MCAM and EpCAM are prognostic molecular markers in patients with malignant tumors 16 \\u2013 24 . Nevertheless, which factors up-regulate inflammatory molecules and adhesion molecules in cancer cells have not yet been elucidated. Vitamin D is a secosteroid hormone for calcium and phosphorus metabolism 25 . Several recent studies demonstrate that vitamin D regulates cell proliferation and differentiation, immune response. In addition, vitamin D has anti-cancer, anti-inflammatory and anti-oxidative effects 26 \\u2013 30 . Vitamin D deficiency (VDD) is defined as lower than 20 ng/ml of 25(OH)D 31 , 32 . Numerous epidemiological reports demonstrated that VDD was positively linked with cancer incidence and a poor clinical prognosis 33 \\u2013 36 . Recently, a large epidemiological investigation showed that VDD was associated with an elevated risk of RCC in men and women 37 . Vitamin D3 (VitD3) is metabolized to 25(OH)D3 by cytochrome P450 (CYP)2R1 in the liver and then metabolized into 1,25(OH)2D3 by CYP27B1 in the kidney 38 . The effects of vitamin D3 are mediated by vitamin D receptor (VDR) 39 . Indeed, CYP27B1 and VDR are highly expressed in human kidney 40 . Therefore, it is especially interesting whether there is an association between VDD and increased inflammatory and adhesion molecules among RCC patients. The purpose of this study was to analyze the link among serum 25(OH)D, local inflammatory signaling and increased expression of adhesion molecules among RCC patients. We found that serum 25(OH)D level were lower in RCC patients than in controls. By contrary, the levels of serum CRP and several adhesion molecules were elevated in RCC patients. The present study suggests a link among low vitamin D status, local inflammation and increased expression of adhesion molecules in RCC patients.\",\n",
            "        \"method\": \"Materials and Methods Study participants Total 50 newly diagnosed RCC patients, whose ages ranged from 40 to 75 years old, were recruited as cases. All cases were diagnosed as RCC patients for the first time between January 2013 and December 2014. RCC was confirmed by histopathology. Total 100 control subjects were from physical examination at the Second Affiliated Hospital of the Anhui Medical University. Two control subjects were matched with one RCC case regarding age (within 2 years), sex and season of blood sample. Individuals with a history of cystic nephropathy, tuberous sclerosis, and severe kidney disease were excluded from this study. Patients were staged using the AJCC TNM system 41 . Tumors were graded according to the Fuhrman system 42 . All slides were examined by two pathologists to ensure diagnostic correction. Serum collection and research procedure obtained approval from the ethics committee of Anhui Medical University. Oral and written consents were obtained from all subjects. All participants signed informed consent forms and were told they could quit at any time. All methods, including methods mentioned below, followed the relevant guidelines and regulations. Reagents Antibodies against MCAM, LAMA4, EpCAM and VDR were from Cell Signaling Technology (Beverley, MA). Antibodies against NF-\\u03baB p65, p-I\\u03baB and \\u03b2-actin were from Santa Cruz Biotechnologies (Santa Cruz, CA). Chemiluminescence detection kit was from Pierce Biotechnology (Rockford, IL). All other reagents were purchased from Sigma Chemical Co. (St. Louis, MO) if not otherwise stated. Biochemical measurement Venous blood was drawn in the morning with more than 8 hours fasting. Serum creatinine, uric acid, urea nitrogen, calcium and serum phosphorus were measured with automatic biochemical analyzer (KHBZY-1200). Radioimmunoassay (RIA) Serum 25(OH)D concentration was measured by RIA using a commercial kit (DiaSorin Inc, Stillwater, MN, USA) as our previous study 43 . Serum 25(OH)D concentrations were expressed as ng/ml. Less than 20 ng/mL for 25(OH)D level was defined as VDD. Cell culture and treatments ACHN cells and 786-o cells are two types of renal cell cancer cell lines that are from the cell bank of the Chinese Academy of Sciences (Shanghai).The cells were inoculated in a 10 cm culture dish and cultured in 5% carbon dioxide at 37 \\u00b0C with 100U/ ml penicillin-streptomycin and 10% serum MEM/EBSS (HyClone, for ACHN) or RPMI Medium Modified (HyClone, for 786-O) medium. When the cells grew to 80% full, the mediums were replaced by serum-free mediums. 1,25(OH)D2 was pretreated for 24 hours before LPS was added. Supernatant was collected after 6 hours for ELISA. Cells were collected for Western blot and Co-IP analysis. Co-immunoprecipitation (Co-IP) The collected cells were washed twice with PBS and then lysed for 20 min with 400 \\u03bcl lysis buffer to obtain total protein. The protein was quantified by BCA Protein Quantitation Kit, and 1000 \\u03bcg Protein was diluted to 1.2 ml. About 20 \\u03bcl protein A/G-agarose was added for incubation for 2 h, and supernatant was taken. The supernatant was added with 2 \\u03bcg anti-VDR antibody, and after 3 h incubation, 40 \\u03bcl protein A/G-agarose was added for incubation. After 10 h, agarose was collected, washed, and boiled for 10 min with 60 \\u03bcl loading buffer. Supernatant was collected for immunoblots using VDR and NF-\\u03baB p65 antibodies. Enzyme-linked immunosorbent assay (ELISA) Measurements of CRP and adhesion molecules CRP and adhesion molecules (ICAM-1, VCAM-1, MCAM, LAMA4 and EpCAM) were measured using ELISA kits (R&D Systems, Abingdon, Oxon, UK) according to the manufacturer\\u2019s protocol. Immunohistochemistry (IHC) The methods of IHC referred to our previous study with minor modification 43 . Briefly, human RCC specimens were fixed in 4% formalin and embedded in paraffin according to the standard procedure. For IHC, paraffin-embedded renal sections were deparaffinized and rehydrated in a graded ethanol series. After antigen retrieval and quenching of endogenous peroxidase, sections were incubated with specific antibodies (MCAM and EpCAM) at 4 \\u00b0C overnight. The color reaction was developed with HRP-linked polymer detection system and counterstaining with hematoxylin. Tissue sample collection and research procedure obtained approval from the ethics committee of Anhui Medical University. Western blot Eight specimens were randomly selected from cancer patients, whose mean level of serum 25(OH)D was 20.73 ng/ml. Tissue lysate and protein extraction referred to the previous study 43 . For Western blot, 20 \\u03bcg of protein in loading buffer was subjected to electrophoresis in 12.5% SDS-PAGE for 3 h. The gel was transferred electrophoretically onto a PVDF membrane (Immobilon-P; Millipore powdered milk in Dulbecco\\u2019s PBS). The membranes were blocked by nonfat milk and then incubated for 2 h using following antibodies: either VDR or p-I\\u03baB or p-p65. For total proteins, \\u03b2-actin was used as a loading control. For nuclear protein, lamin A/C was used as a loading control. After washed in Dulbecco\\u2019s PBS-T for four times and PBS once, the membrane was incubated with goat anti-rabbit or horse anti-goat IgG antibody for 2 h. The membrane was then washed 4 times in Dulbecco\\u2019s PBS-T for 10 min and PBS for 10 min, followed by signal development using an enhanced chemiluminescence (ECL) detection kit. Statistical analysis The difference between two independent groups was compared using two student t test or the Mann-Whitney U-test. Comparative analyses of categorical variables were carried out by the chi-square test. The link between 25(OH)D and adhesion molecules was analyzed using linear correlation. A P value of less than 0.05 was considered statistically significant.\",\n",
            "        \"subjects\": null,\n",
            "        \"results\": \"Results Demographic characteristics and grades of renal cell carcinoma The demographic characteristics are shown in Table 1 . As expected, no difference on mean age and sex composition was observed between RCC patients and controls. There was no difference on serum creatinine, uric acid and phosphorus levels between two groups. BMI index and serum urea nitrogen were higher in RCC patients than those of controls (Table 1 ). By contrary, the levels of serum calcium were lower in RCC patients than those of controls (Table 1 ). The number of cases at different stages and grades was then analyzed. As shown in Table 2 , 60% (30/50) of RCC patients belonged to G1/2 and 40% (20/50) belonged to G3/4. Moreover, 68% (34/50) of RCC patients belonged to T1a/b (34/50, 68%). In addition, 18% (9/50) of RCC patients belonged to T2 and only 14% (7/50) belonged to T3/4 (Table 2 ). Table 1 The demographic and serum biochemical parameters between cases and controls Control (n = 100) Case (n = 50) P value Mean age (years), mean (SD) 56.55 (11.57) 56.88 (11.91) 0.871 Sex, male (%) 73 (73) 36 (72) 0.897 BMI (kg/m 2 ) 21.18 (2.57) 24.26 (3.24) <0.001 Serum creatinine (\\u03bcmol/L) 80.09 (18.12) 85.72 (31.89) 0.706 Serum urea nitrogen (mmol/L) 5.03 (1.00) 5.81 (1.53) <0.001 Serum uric acid (mmol/L) 375.35 (96.70) 381.11 (97.16) 0.704 Serum calcium (mmol/L) 2.27 (0.13) 2.21 (0.09) 0.039 Serum phosphorus (mmol/L) 1.15 (0.24) 1.18 (0.26) 0.737 Table 2 Fuhrman Grades and AJCC TNM Stages Case (n = 50) Fuhrman Grades (%) Fuhrman I 2 (4) Fuhrman II 28 (56) Fuhrman III 13 (26) Fuhrman IV 7 (14) AJCC TNM Stages (%) T1a 12 (24) T1b 22 (44) T2 9 (18) T3/4 7 (14) Serum 25(OH)D is reduced in RCC patients Serum 25(OH)D was analyzed among all subjects. As shown in Fig. 1A , 25(OH)D concentration was 20.73 ng/ml in RCC patients, significantly lower than 23.02 ng/ml in controls. VDD is defined by most experts as a 25(OH)D level of less than 20 ng/ml. A 25(OH)D level of greater than 20 ng/ml and less than 30 ng/ml is defined as vitamin D insufficiency (VDI), and 30 ng/ml or greater is defined as vitamin D sufficiency (VDS) 32 , 44 . We compared the vitamin D status between RCC patients and controls. We found that the proportion of subjects with VDD (25(OH)D < 20 ng/ml), VDI (20 \\u2264 25(OH)D < 30 ng/m) and VDS (25(OH)D \\u2265 30 ng/ml) in RCC patients were 44%, 48% and 8%, respectively. The proportion of subjects with VDD, VDI and VDS in controls were 34%, 54% and 12%, respectively. Although there is no significant difference, the proportion of RCC patients with VDD has an increasing trend. Accumulating data have demonstrated that overweight/obesity influences vitamin D status 45 , 46 . The present study found that BMI index of RCC patients was higher than that of controls. Thus, we further analyzed the association between RCC patients and serum 25 (OH) by multivariable logistic regression, adjusting for BMI. The results are presented in Table 3 . As expected, a negative association between serum 25(OH)D and RCC was observed (adjusted OR: 0.946; P < 0.05). Figure 1 Serum 25(OH)D level and VDR distribution among RCC patients. ( A ) Serum 25(OH)D level in RCC patients and controls. Serum 25(OH)D was measured by RIA. All data were expressed as means \\u00b1 S.E.M. **P < 0.01. (N = 50 for RCC patients; N = 100 for control subjects). ( B , C ) Nuclear VDR level was measured using Western blot. ( B ) A representative gel for VDR (upper panel) and Lamin A/C (lower panel) was shown. ( C ) VDR/Lamin A/C. The grouping of blots cropped from different gels of same samples. All data were expressed as means \\u00b1 S.E.M. (N = 8). ** P < 0.01. Table 3 Multivariable logistic regression analysis of correlation between RCC and serum 25(OH)D concentration. Variable \\u03b2 Wald P OR (95%C.I.) Unadjusted 0.053 3.684 0.055 0.948(0.898, 1.001) Adjusted a 0.055 3.905 0.048 0.946(0.896, 0.999) Note: \\u03b2: regression coeffcient; Wald: Wald chi-square value; OR: odds ratio. a Adjusted for BMI and age. Nuclear VDR is reduced in cancerous tissues among RCC patients Nuclear VDR levels were analyzed among RCC patients. As expected, nuclear VDR was reduced in cancerous tissue (Fig. 1B,C ). The levels of serum CRP and adhesion molecules are increased in RCC patients Serum CRP was analyzed in RCC patients and controls. As shown in Fig. 2A , there was a rising trend on serum CRP level in RCC patients. Serum adhesion molecules were then analyzed in RCC patients and controls. Interestingly, serum ICAM was elevated in RCC patients (Fig. 2B ), whereas there was no difference on serum VCAM levels between two groups (Fig. 2C ). As shown in Fig. 2D , there was a rising trend on serum MCAM. Laminin alpha 4 (LAMA4) is extracellular matrix interaction partner of MCAM 22 . As shown in Fig. 2E , the levels of serum LAMH4 were higher in RCC patients than those in controls. In addition, serum EpCAM was higher in RCC patients than that of controls (Fig. 2F ). The distribution of MCAM and EpCAM, two adhesion molecules in cancerous tissues, was then determined among RCC patients. As expected, the expression of MCAM were much higher in cancerous tissue than that of paracancerous tissue (Fig. 3A,C ). In addition, the expression of EpCAM in cancerous tissues was significantly higher than that of paracancerous tissue (Fig. 3B,D ). Figure 2 Serum CRP and adhesion molecule levels in RCC patients and controls. Serum CRP and adhesion molecules were measured using ELISA. Serum CRP, ICAM, VCAM, MCAM, LAMA4 and EpCAM levels were compared between RCC patients and controls. (N = 50 for RCC patients; N = 100 for control subjects). (A) CRP; ( B ) ICAM; ( C ) VCAM; ( D ) MCAM; ( E ) LAMA4; ( F ) EpCAM. All data were expressed as means \\u00b1 S.E.M. * P < 0.05, ** P < 0.01. Figure 3 The distributions of adhesion molecules and activation of NF-\\u03baB in cancerous and paracancerous tissues. ( A \\u2013 D ) The distribution of MCAM and EpCAM in cancerous and paracancerous tissues was determined using IHC. ( A , C ) MCAM; ( B , D ) EpCAM. ( E \\u2013 G ) Phosphorylated p65 and p-I-\\u03baB in cancerous and paracancerous tissues was measured using Western blot. ( E ) A representative gel for p-p65 (upper panel), p-I-\\u03baB (middle panel) and \\u03b2-actin (lower panel) was shown. ( F ) p-I-\\u03baB/\\u03b2-actin. ( G ) p-p65/\\u03b2-actin. The grouping of blots cropped from different gels of same samples. All data were expressed as means \\u00b1 S.E.M. (N = 8). **P < 0.01. NF-\\u03baB is activated in cancerous tissues among RCC patients Phosphorylated I-\\u03baB and NF-\\u03baB p65 were measured in cancerous tissue. As shown in Fig. 3E,F , the levels of p- I\\u03baB were higher in cancerous tissue than in paracancerous tissue. The levels of p-p65 were accordingly higher in cancerous tissue than in paracancerous tissue (Fig. 3E,G ). Vitamin D status is reversely associated with serum CRP and adhesion molecules in RCC patients Limited by the sample size, we defined patients with VDD (serum 25(OH)D < 20 ng/ml) as a low vitamin D status, while patients with 25(OH)D \\u2265 20 ng/ml is defined as a relatively high status. All RCC patients were divided into two groups according to serum 25(OH)D level: H-VitD (\\u226520 ng/ml) or L-VitD (<20 ng/ml). Serum CRP level was compared between two groups. As shown in Fig. 4A , serum CRP level was significantly higher in RCC patients with L-VitD than those with H-VitD. The correlation between serum 25(OH)D and CRP levels was then analyzed among RCC patients and controls. As expected, serum 25(OH)D was not correlated with CRP among controls. Interestingly, serum 25(OH)D was reversely correlated with serum CRP among RCC patients (Table 4 ; r = \\u22120.27, P < 0.05). Serum adhesion molecules were analyzed. As expected, the levels of all detected adhesion molecules were higher in RCC patients with L-VitD than those with H-VitD (Fig. 5B\\u2013F ). The link between serum 25(OH)D and adhesion molecules was then analyzed. As expected, no link between serum 25(OH)D and all adhesion molecules was observed in controls (Table 4 ). Of interest, serum 25(OH)D was reversely correlated with serum ICAM ( r = \\u22120.25, P < 0.05), VCAM ( r = \\u22120.23, P < 0.05), MCAM ( r = \\u22120.27, P < 0.05), LAMA4 ( r = \\u22120.23, P < 0.05) and EpCAM ( r = \\u22120.25, P < 0.05) in RCC patients (Table 4 ). Figure 4 Low vitamin D status is associated with serum CRP and adhesion molecules in RCC patients. RCC patients were divided into two groups according to serum 25(OH)D level: >20 ng/ml or <20 ng/ml. The levels of serum CRP and adhesion molecules were compared between two groups. ( A ) CRP; ( B ) ICAM; ( C ) VCAM; ( D ) MCAM; ( E ) LAMA4; ( F ) EpCAM. All data were expressed as means \\u00b1 S.E.M. *P < 0.05. Table 4 Association between serum 25(OH)D and adhesion molecules. CRP sICAM sVCAM MCAM LAMA4 EpCAM Control 0.20 \\u22120.11 0.05 0.19 0.04 0.02 RCC \\u22120.27 a \\u22120.25 a \\u22120.23 a \\u22120.27 a \\u22120.23 a \\u22120.25 a a P < 0.05. Figure 5 The effects of vitamin D3 on LPS-stimulated NF-\\u03baB and adhesion molecules activation in RCC cells. ( A , B ) ACHN cells were pretreated with 1,25(OH)2D3, the active form of vitamin D3, and then stimulated with LPS for 6 h. Phosphorylated p65 was measured using Western blot. ( A ) A representative gel for p-p65 (upper panel) and \\u03b2-actin (lower panel) was shown. ( B ) p-p65/\\u03b2-actin. All experiments were repeated for six times. The grouping of blots cropped from different gels of same samples. All data were expressed as means \\u00b1 S.E.M. (N = 6). **P < 0.01. ( C \\u2013 E ) ACHN cells were pretreated with 1,25(OH)2D3, the active form of vitamin D3, and then stimulated with LPS for 6 h. The level of MCAM, LAMA4 and EpCAM in medium was measured using ELISA. ( C ) MCAM in medium; ( D ) LAMA4 in medium; ( D ) EpCAM in medium. All data were expressed as means \\u00b1 S.E.M. (N = 8). **P < 0.01. Vitamin D3 inhibits NF-\\u03baB activation and MCAM upregulation in RCC cells To further explore the mechanism through which low vitamin D status upregulates expression of adhesion molecules among RCC patients, ACHN cells were pretreated with 1,25(OH)2D3, the active form of vitamin D3, and then stimulated with LPS to observe whether active vitamin D3 inhibits adhesion molecules in RCC cells. As expected, the levels of MCAM, LAMA4 and EpCAM were elevated in LPS-stimulated ACHN cells (Fig. 5C\\u2013E ). Interestingly, pretreatment with 1,25(OH)2D3 suppressed LPS-evoked elevation of MCAM, LAMA4 and EpCAM in ACHN cells (Fig. 5C\\u2013E ). The effects of pretreatment with 1,25(OH)2D3 on LPS-induced NF-\\u03baB activation were measured. As expected, the level of phosphorylated NF-\\u03baB was significantly elevated in LPS-stimulated ACHN cells (Fig. 5A,B ). Interestingly, pretreatment with 1,25(OH)2D3 inhibited LPS-induced activation of NF-\\u03baB in ACHN cells (Fig. 5A,B ). Vitamin D3 promotess integration between VDR and NF-\\u03baB p65 subunit in RCC cells The integration between VDR and NF-\\u03baB p65 subunit in RCC cells was measured by CoIP. As shown in Fig. 6 , 1,25(OH)2D3 plus LPS elevated level of p65 in immunocomplexes precipitated by anti-VDR antibody. These results suggest an integration between VDR and NF-\\u03baB p65 subunit in RCC cells. Figure 6 Vitamin D3 reinforces interaction between VDR and NF-\\u03baB p65 subunit in RCC cells. ACHN cells were pretreated with 1,25(OH)2D3, the active form of vitamin D3, and then stimulated with LPS for 6 h. The interaction between NF-\\u03baB p65 and VDR was detected using CoIP. All experiments were duplicated for four times. A representative gel for NF-\\u03baB p65 (upper panel) and VDR (lower panel) was shown. The grouping of blots cropped from different gels of same samples. All data were expressed as means \\u00b1 S.E.M. (n = 4). **P < 0.01.\",\n",
            "        \"discussion\": \"Discussion Several epidemiological reports indicated that occupational ultraviolet exposure was negatively correlated with risk of RCC in different population 47 , 48 . A large prospective cohort study demonstrated that plasma 25(OH)D level was reversely associated with the incidence of RCC in men and women 37 . According to a report from the European Prospective Investigation into Cancer and Nutrition, circulating 25(OH)D was negatively associated with Odds Ratio of RCC after adjustment for confounding factors 49 . A recent study demonstrated that circulating 25(OH)D was positively associated with survival of kidney cancer 50 . The present study analyzed the link of low vitamin D status with RCC risk. We found that circulating 25(OH)D in RCC patients was lower than that of controls. Multivariable logistic regression analysis indicated an inverse link between RCC and serum 25(OH)D after adjustment for confounding factors. These results suggest that low vitamin D status is linked with an increased RCC risk. Indeed, all RCC patients recruited in this study are new cases, most of which belong to T1a/b (34/50, 68%) and G1/2 (30/50, 60%). Thus, additional research is necessary to analyze the correlation between serum 25(OH)D level and RCC severity in a large sample of population. The methods for determining serum 25(OH)D levels include HPLC, LC-MS/MS, RIA and ELISA. The LC-MS/MS is the most accurate method that can simultaneously measure serum 25(OH)D2 and 25(OH)D3. LC-MS/MS is usually used a gold standard. However, LC-MS/MS method is time-consuming. In addition, specimen pretreatment for LC-MS/MS is complicated and the results are greatly influenced by human factors. On the other hand, the RIA method is characterized by high sensitivity, specificity and precision, and low requirements for equipment and equipment. In addition, the RIA method has good consistency with LC-MS/MS method and is often used to detect serum 25(OH)D in a large sample of population 51 , 52 . Thus, the RIA method is used to detect serum 25(OH)D in this study. Numerous reports suggest that inflammation plays an important role in RCC metastases and progression 53 , 54 . CRP, a marker of inflammation, is an independent prognostic factor for overall survival of RCC patients 8 , 55 . An early report showed that an increase in preoperative CRP was linked with poor survival in patients with localized RCC 56 . According to a recent study, serum CRP level is a strong independent factor associated with survival of RCC patients treated with sunitinib 57 . This study analyzed the link between 25(OH)D and CRP in RCC patients. We found that serum CRP in RCC patients with L-VitD was higher than those with H-VitD. Further analysis found that serum CRP level was reversely correlated with serum 25(OH)D level among RCC patients. The study indicates that low vitamin D status is linked with increased serum CRP among RCC patients. Accumulating data have demonstrated that adhesion molecules are prognostic molecular markers in RCC patients. Two early studies showed that patients with short survival or advanced RCC had higher levels of serum ICAM-1 than low-grade and/or low-stage RCC patients 58 , 59 . Another report indicated that VCAM-1, commonly overexpressed in RCC, was involved in tumor immune evasion 60 . According to a recent study, MCAM and its extracellular matrix interaction partner LAMA4, highly expressed in locally advanced tumors as well as secondary metastases, were predictive markers for poor RCC prognosis 22 . The present study showed that serum ICAM, LAMA4 and EpCAM were higher in RCC patients than in controls. Importantly, MCAM and EpCAM were highly expressed only in cancerous tissue but not in paracancerous tissue. Further observation found that the levels of all detected adhesion molecules were higher in RCC patients with L-VitD than those with L-VitD. These results suggest that low vitamin D status is associated with increased expression of adhesion molecules among RCC patients. The mechanism through which VDD upregulates expression of adhesion molecules among RCC patients remains obscure. An in vitro study demonstrated that active VitD3 downregulated expression of ICAM-1 and VCAM-1 in endothelial cells through suppressing NF-\\u03baB activation 61 . According to an in vivo report, pretreatment with VitD3 attenuated LPS-induced upregulation of renal ICAM-1 and VCAM-1 by inhibiting activation of NF-\\u03baB in the kidneys 62 . In the present study, we found that nuclear VDR was low in cancerous tissue. By contrary, renal NF-\\u03baB was activated only in cancerous but not in paracancerous tissue. Correspondingly, renal adhesion molecules were highly expressed only in cancerous tissue but not in paracancerous tissue. The in vitro experiments found that VitD3 blocked activation of NF-\\u03baB and suppressed upregulation of adhesion molecules in RCC cells. Moreover, VitD3 suppressed NF-\\u03baB through promoting the integration between VDR and NF-\\u03baB p65 in RCC cells, which provides a mechanistic explanation for linking VDD, low VDR and increased expression of adhesion molecules in RCC patients. The present study placed emphasis on analyzing the association of low vitamin D status with serum CRP and adhesion molecules among RCC patients. We demonstrated that there was a link among low vitamin D status, cancerous NF-\\u03baB activation and increased expression of adhesion molecules in RCC patients. However, the present study is observational. The findings herein described are not concretely able to demonstrate a causal association between both CRP and adhesion molecules and the prognosis of RCC patients with low vitamin D status. Several studies have demonstrated that serum calcium level is a factor of prognosis 62 . Thus, further studies are required to identify a causal association between both CRP and adhesion molecules and the prognosis of RCC patients with low vitamin D status. In addition, an in vivo experiment may be needed to measure the effects of supplementation with VitD3 on serum CRP and adhesion molecules as well as NF-\\u03baB signaling in RCC animal model. In summary, the present study analyzed the association of low vitamin D status with serum CRP and adhesion molecules in RCC patients. We showed that serum 25(OH)D level was reduced among RCC patients. By contrast, the expression of renal adhesion molecules was upregulated among RCC patients. Moreover, vitamin D status was reversely correlated with serum CRP and adhesion molecules among RCC patients. We observed that NF-\\u03baB in cancerous tissue was activated among RCC patients. The in vitro experiments found that active VitD3 suppressed NF-\\u03baB activation and adhesion molecules through promoting integration between VDR and NF-\\u03baB p65 in RCC cells. Our results suggest a link among low vitamin D status, local inflammation and increased expression of adhesion molecules in RCC patients.\",\n",
            "        \"conclusion\": null\n",
            "    },\n",
            "    {\n",
            "        \"pmcid\": \"PMC6955584\",\n",
            "        \"abstract\": null,\n",
            "        \"intro\": null,\n",
            "        \"method\": \"2 MATERIALS AND METHODS 2.1 Study design This prospective randomized, open\\u2010label, exploratory, parallel\\u2010group controlled study was performed in a private infertility clinic (Kinutani Women's Clinic, Hiroshima, Japan) from December 1, 2016, to December 30, 2017. The protocol was approved by the Ethics Committee of the Kinutani Women's Clinic (ethical review number 2016\\u20101110\\u20101), and this study was registered in the UMIN Clinical Trials Registry (registration number UMIN000032997). 2.2 Study population As this study was an exploratory study, no particular sample size was established. We excluded patients who had contraindications that were described on the drug package. This was the only exclusion criterion; we used no other restrictions such as age or cause of infertility. Thus, we requested the participation of all patients who visited our clinic on an outpatient basis, underwent egg retrieval, and had at least one embryo frozen using the vitrification method for cryopreservation. Patients who consented to study participation were enrolled in the study. The patients underwent autologous Frozen\\u2010ET of one cleavage\\u2010stage embryo or one blastocyst. Patients who desired to transfer two embryos or those who desired the use of a different preparation method were not permitted to participate in this study. Patients were permitted to participate two or more times. Even if the patients participated in the study more than once, we allocated them to any of the four drug groups. Therefore, sometimes patients were assigned to the same vaginal drug used previously. 2.3 Randomization and intervention All patients were given a registration number on the basis of the order of their referral. Then, a specific researcher generated a computer\\u2010based random allocation table, and all patients were randomly assigned to one of the four study groups. Those assigned to the first study group received 100 mg of a vaginal P4 tablet three times daily (Lutinus, Ferring Pharmaceuticals); those assigned to the second group received 200 mg of vaginal P4 capsules (Utrogestan, FUJIFILM Pharmaceuticals) three times daily; those assigned to the third group received a 400\\u2010mg vaginal suppository (Luteum, ASKA Pharmaceutical) twice daily; and those in the fourth group received 90 mg of a vaginal gel (Crinone, Merckserono) once daily. The randomization lists were kept on a password\\u2010protected computer. 2.4 Study protocol All patients underwent an IVF cycle with an agonist, antagonist, and mild stimulant protocol. IVF, intracytoplasmic sperm injection (ICSI), or both methods were used in the process. Confirmation of fertilization was tested for at 16\\u201019 hours after IVF or ICSI. Several embryologists affiliated with the clinic performed cryopreservation of early embryos 2\\u20103 days after egg retrieval and cryopreservation of blastocysts 4\\u20107 days after egg retrieval. The vitrification method of cryopreservation was used in all cases. Before freezing, the early embryos were assessed using the Veek method, 24 and the blastocysts were assessed using the Gardner method. 25 In the case of several early embryos and blastocysts, the decision regarding whether to freeze them was left to four physicians affiliated with this clinic. The HRT protocol was conducted using the gonadotropin\\u2010releasing hormone agonist via nasal drops for down\\u2010regulation 1\\u20102 weeks before the start of menstruation. We started with the use of estrogen (E2) tape (Estrana tape, Hisamitsu Pharmaceutical Co., Inc.) or E2 gel (L\\u2019estrogel 0.06%, FUJIFILM Pharmaceuticals), which is a transdermal E2 preparation, from menstrual cycle days 2\\u20105, gradually increased the dosage, checked whether the endometrial thickness was at least 7.0 mm on menstrual cycle days 12\\u201015, and then started P4 vaginal suppositories (day 0), with embryo transfer being performed on days 2 and 3 for a cleavage\\u2010stage embryo and on day 5 for a blastocyst. In some cases, in which the endometrial thickness was <7.0 mm, the period of E2 administration was extended. It has been reported that when the endometrium is <7 mm, the pregnancy rate decreases. 26 However, in some patients, the endometrium does not become thicker even if E2 is increased or prolonged. Thus, embryo transfer was not canceled because of insufficient thickness in any case. Single embryo transfer of a cleavage\\u2010stage embryo or blastocyst was performed. If the patient had a positive beta\\u2010human chorionic gonadotropin blood test result in pregnancy week 4, fetal growth was assessed weekly thereafter by transvaginal ultrasonography. The administration of vaginal suppositories was continued until pregnancy week 10 or pregnancy termination. The primary outcome measure of this study was CPR, and the secondary outcome measures were FHR and MR. 2.5 Adverse events All adverse reactions other than the side effects described on the drug package, such as thrombosis, headache, somnolence, genital bleeding, and diarrhea, were recorded. Patients were able to withdraw from the study if severe adverse events occurred. 2.6 Statistical analysis Clinical pregnancy rate was assessed by checking for a gestational sac via transvaginal ultrasonography during pregnancy week 5. FHR was assessed during pregnancy week 7. MR was also assessed during pregnancy week 8. Pairwise comparisons between the four groups were performed using the Mann\\u2010Whitney or Kruskal\\u2010Wallis test for the continuous variables and the Fisher exact test for categorical variables. Univariable and multivariable logistic regression models were applied to investigate the effect of covariates (age, body mass index [BMI], and number of previous transfers) on CPR, FHR, and MR. All statistical analyses were performed with the statistical software SAS 9.4 (SAS Institute Inc). The level of significance for the univariate and multivariate analyses was set to <5% ( P < .05).\",\n",
            "        \"subjects\": null,\n",
            "        \"results\": \"3 RESULTS We evaluated 259 cycles for eligibility, of which 254 were randomized to four study groups. Twenty\\u2010four cycles could not be followed up or the patients discontinued the study. Reasons for dropping out of the study were as follows: Per patients\\u2019 request, participation in five cycles was declined even after judging that there was no problem in the evaluation of eligibility. Two cycles were withdrawn from the Lutinus group because two patients wanted to transfer two embryos. Three cycles were withdrawn from the Utrogestan group because two patients wanted another drug, and one patient had no implantable embryo. Eight cycles were withdrawn from the Luteum group because one patient wanted another drug, four patients wanted to transfer two embryos, two patients had no implantable embryo, and one patient had severe vaginal bleeding and changed the drug. Six cycles were withdrawn from the Crinone group because three patients wanted to transfer two embryos, two patients had no implantable embryo, and one patient wanted another drug. No serious adverse events occurred in the women involved in this study. As a result, the number of cycles included in the final analysis was 235 (Figure 1 ). The number of patients who participated in this study was 183; thus, the average participation was 1.2 times. The number of cycles assigned to the same vaginal P4 as the previously assigned cycle was only 10. Figure 1 CONSORT flow diagram of the study. The diagram shows participant flow through the study, including patient eligibility, reasons for exclusion, treatment group allocation, loss to follow\\u2010up, and number of cycles included in the final analysis The patient demographics are summarized in Table 1 . Blastocysts that were assessed as 3BB or higher using the Gardner classification were identified as high\\u2010quality embryos. The results showed no significant differences between the study groups regarding the baseline characteristics: age ( P = .169), BMI ( P = .384), pregnancy history ( P = .732), previous transfers ( P = .679), endometrial thickness ( P = .132), transferred embryo ( P = .909), and quality of the blastocyst ( P = .384). Table 1 Patient demographics for frozen embryo transfer cycles using four vaginal progesterones for luteal phase support Group Lutinus (n = 63) Utrogestan (n = 60) Luteum (n = 56) Crinone (n = 56) P \\u2010value Age (y) 37.0 (33.0\\u201040.0) 35.0 (32.0\\u201039.0) 37.5 (33.0\\u201041.0) 36.0 (33.0\\u201038.0) .169 BMI (kg/m 2 ) 20.8 (18.9\\u201022.2) 20.2 (18.7\\u201021.7) 20.2 (18.8\\u201021.8) 19.9 (18.5\\u201021.0) .384 Pregnancy history Primary 30 (47.6) 31 (51.6) 26 (46.4) 23 (41.0) .732 Secondary 33 (52.3) 29 (48.3) 30 (53.5) 33 (58.9) Previous transfers (times) 0 21 (33.3) 24 (40.0) 15 (26.7) 15 (26.7) .679 1 16 (25.4) 17 (28.3) 16 (28.5) 16 (28.5) \\u22652 26 (41.2) 19 (31.6) 25 (44.6) 25 (44.6) Endometrial thickness (mm) 11.2 (10.2\\u201012.8) 10.8 (9.9\\u201012.1) 10.5 (9.9\\u201011.7) 11.0 (9.75\\u201012.0) .132 Transferred embryo Cleavage\\u2010stage 31 (49.2) 30 (50.0) 25 (44.6) 25 (44.6) .909 Blastocyst 32 (50.7) 30 (50.0) 31 (55.3) 31 (55.3) Quality of blastocyst High 23 (71.8) 25 (83.3) 21 (67.7) 20 (64.5) .384 Poor 9 (28.1) 5 (16.6) 10 (32.2) 11 (35.4) Note Data are presented as a median (IQR) or count (%). The IQR represents the 25th and 75th percentiles. The P \\u2010value is >.05 for all variables when the four groups are compared. IQR, interquartile range. John Wiley & Sons, Ltd Although we consider the median age of the patients (35\\u201037 years) to be higher than that of those in other countries, this age group is commonly dealt with in fertility clinics located in Japan. Pregnancy outcome data are shown in Table 2 . There were no statistically significant differences between the four groups in the rates of CPR (Lutinus, Utrogestan, Luteum, and Crinone: 34.9%, 33.3%, 37.5%, and 35.7%, respectively; P = .976), FHR (26.9%, 31.6%, 30.3%, and 25.0%, respectively; P = .858), and MR (31.8%, 10.0%, 19.0%, and 30.0%, respectively; P = .306). The crude odds ratios (ORs) for these comparisons are shown in Table 3 . We did not find any significant difference between the four study groups regarding CPR, FHR, and MR in univariate OR. We performed a multivariate logistic regression analysis to eliminate the confounding effects of age, BMI, and the number of previous transfers (Table 3 ). We also found that after controlling for confounders, there was no significant difference between the study groups. Table 2 Comparison of pregnancy outcomes in 235 cycles of frozen\\u2010thawed embryo transfer between four vaginal progesterones Lutinus (n = 63) Utrogestan (n = 60) Luteum (n = 56) Crinone (n = 56) P \\u2010value Clinical pregnancy rate 22 (34.9) 20 (33.3) 21 (37.5) 20 (35.7) .976 Fetal heart beat 17 (26.9) 19 (31.6) 17 (30.3) 14 (25.0) .858 Miscarriage rate 7 (31.8) 2 (10.0) 4 (19.0) 6 (30.0) .306 Note Data are presented as a count (%). John Wiley & Sons, Ltd Table 3 Univariate and multivariate analyses of the clinical outcomes between the four vaginal progesterones Outcome Group Univariate analysis Multivariate analysis OR (95% CI) P \\u2010value OR (95% CI) P \\u2010value Clinical pregnancy rate Lutinus a 1 1 Utrogestan 0.93 (0.44\\u20101.97) .853 0.42 (0.13\\u20101.30) .131 Luteum 1.12 (0.53\\u20102.37) .770 0.60 (0.21\\u20101.75) .351 Crinone 1.04 (0.49\\u20102.20) .927 0.76 (0.27\\u20102.18) .611 Fetal heart beat Lutinus a 1 1 Utrogestan 1.25 (0.58\\u20102.73) .569 0.61 (0.19\\u20101.92) .394 Luteum 1.18 (0.53\\u20102.62) .685 0.53 (0.17\\u20101.67) .279 Crinone 0.90 (0.40\\u20102.05) .806 0.61 (0.20\\u20101.88) .390 Miscarriage Lutinus a 1 1 Utrogestan 0.24 (0.04\\u20101.32) .101 0.17 (0.02\\u20101.83) .146 Luteum 0.50 (0.12\\u20102.07) .342 0.55 (0.10\\u20103.14) .503 Crinone 0.92 (0.25\\u20103.41) .899 0.82 (0.17\\u20103.92) .807 Note Data are presented as an OR with 95% CI. CI, confidence interval; OR, odds ratio. a The Lutinus group was considered as the reference group for comparison. Adjusted variables: age, body mass index, and number of previous transfers. John Wiley & Sons, Ltd\",\n",
            "        \"discussion\": \"4 DISCUSSION This study's results indicated that no significant differences were found between the four preparations in terms of CPR, FHR, and MR. Currently, vaginal suppositories are commonly used throughout the world for LPS in ART. 15 A systematic review conducted in 2018 concluded that there was no difference between four vaginal preparations when used in Fresh\\u2010ET in terms of safety or effectiveness. 22 However, an insufficient number of studies have compared the effectiveness of four preparations in HRT\\u2010FET. Lan et al indicated that there was no difference between the Crinone group (90 mg/d) and Utrogestan group (200 mg three times per day) in terms of CPR in HRT\\u2010FET. 27 This result is consistent with our study's finding that no significant differences were found between the four vaginal formulations in terms of CPR. Other studies have shown the effectiveness of vaginal preparations in HRT\\u2010FET 28 , 29 , 30 , 31 , 32 ; however, they compared the effectiveness between intramuscular injection and vaginal preparations, not between several vaginal preparations. All four of the preparations are the same natural P4 agents, but their daily doses range from 90 mg to 300 mg, 600 mg, and 800 mg. Regardless of this, the fact that all four have the same pregnancy rates and MRs is extremely interesting. Over the history of vaginal preparations, the form of the preparations has undergone development from vaginal suppositories to gelatin capsules, bioadhesive gel, and most recently, foam agents. 33 , 34 , 35 , 36 , 37 , 38 , 39 , 40 , 41 Crinone and Lutinus, which are newer preparations, contain smaller amounts of P4, presumably because they have superior solubility and absorption rates. In addition, Crinone and Lutinus are applied with the use of an applicator. It has been reported that the use of an applicator to place the preparation deeper into the vagina allows it to be efficiently transferred to the endometrium. 42 This is the likely reason why these two preparations require smaller amounts of P4. The methods of application and doses currently recommended by the manufacturers of all four preparations are indicated for use in Fresh\\u2010ET procedures. 43 , 44 , 45 , 46 As a result, it is unknown whether the same doses are effective when the preparations are used for HRT\\u2010FET. It has been reported that higher doses are better when used in HRT\\u2010FET. One such study reported that the pregnancy rate was higher when the dose of Crinone is double the recommended dose of 90 mg/d (ie, 180 mg/d). 47 Since this was a retrospective study, bias may have been involved. A second such study reported that the pregnancy rate was higher when Utrogestan was used at a dose of 1200 mg/d rather than 900 mg/d. 48 Because additional doses are required when the blood P4 level is <9 ng/mL on day 5 of the luteal phase, this study was not properly conducted. The third such study investigated three groups: 50 mg daily intramuscular injection of P4, Lutinus (200 mg twice daily), and Lutinus (200 mg twice daily) with a 50\\u2010mg intramuscular injection of P4 every third day. They found that the vaginal preparation alone, that is, Lutinus (200 mg twice daily) had a worse outcome than the other two, and therefore, it should be avoided. 49 However, the dose of Lutinus that was used differed from that recommended by the manufacturer. The present study's results of LPS for HRT\\u2010FET indicated that there were no significant differences between the four vaginal P4 suppositories in terms of CPR, FHR, or MR. Nevertheless, this study has some limitations. First, as it was a single\\u2010center study, there may have been a patient bias, and as all patients were Japanese individuals, their BMI may have been lower than in other races; thus, they may have required smaller doses than women from other countries. Second, the drug amount presented by each pharmaceutical company is the recommended amount based on Fresh\\u2010ET; hence, it is unknown whether the amount is enough in HRT\\u2010FET. Assisted reproductive treatments performed as LPS requires an extremely long time, which places a high degree of stress on the patient. Currently, the use of Frozen\\u2010ET is increasing worldwide, so establishment of effective LPS for HRT\\u2010FET is urgently needed. Future large\\u2010scale multi\\u2010center cohort studies on this issue are required in order to obtain improved results. In conclusion, there was no clinically significant difference in pregnancy outcomes between Lutinus, Utrogestan, Luteum, and Crinone when used in HRT\\u2010FET for LPS. The four vaginal preparations differ in the number of administrations and use of an applicator. The period of corpus luteum supplementation in ART is long and stressful for women. If there is no difference in the pregnancy rate between the four drugs, then women can choose their preferred vaginal P4.\",\n",
            "        \"conclusion\": null\n",
            "    },\n",
            "    {\n",
            "        \"pmcid\": \"PMC7007877\",\n",
            "        \"abstract\": null,\n",
            "        \"intro\": null,\n",
            "        \"method\": \"Material and Methods Subjects In total, serum samples from 1838 healthy subjects (772 boys and men/1066 girls and women) selected from 5 different cohorts were analyzed for estrogen concentrations: cohort 1, a subset from the Copenhagen Mother\\u2013Child Cohort: 50 boys and 50 girls aged 3.0 to 5.3 months, 17 boys and 15 girls aged 10 to 14 months, and 49 boys and 48 girls aged 4.2 to 6.0 years ( http://www.edmarc.net/mother-child-cohort.html , ( 16 , 17 ); cohort 2, a subset of 48 girls aged 7.9 months to 6.0 years from the study of the follicle-stimulating hormone (FSH) and luteinizing hormone (LH) response to a gonadotropin-releasing hormone test in this age group ( 18 ); cohort 3, 499 boys and 604 girls aged 5.9 to 23.4 years and additionally 62 adolescent girls using oral contraceptives aged 14.8 to 19.4 years from the Copenhagen Puberty Study ( http://www.edmarc.net/puberty-cohort.html , ( 19 , 20 ); cohort 4, 157 men aged 31.4 to 60.9 years and 93 postmenopausal women aged 54.6 to 60.7 years from Health2008, a population-based cross-sectional study conducted at the Research Centre for Prevention and Health, Glostrup University Hospital, Denmark ( 21 , 22 ); and cohort 5, 194 women aged 24.7 to 43.9 years with regular menstrual cycles. The women were seen in the Fertility Assessment and Counselling Clinic (FAC Clinic), Copenhagen University Hospital between 2012 and 2016 ( 23 ). Clinical examinations In all cohorts, participants were examined for their general health status according to the respective study protocols. In cohort 3, pubertal stage was assessed by clinical examination, breast stages (B1\\u2013B5) by palpation, and genital stage (G1\\u2013G6) by inspection according to the classification by Marshall and Tanner ( 19 , 20 , 24 , 25 ), and information on age at menarche and on use of oral contraceptives were given as self-reported data in a questionnaire ( 19 ). In cohort 5, information on the given menstrual cycle day (the day the blood samples were withdrawn) was obtained by the examining physician ( 23 ). Hormone analysis All blood samples were drawn from an antecubital vein, clotted and centrifugated, and serum was stored at \\u201320\\u00b0C until analysis. All hormone analyses were performed at Department of Growth and Reproduction, Rigshospitalet, Copenhagen, Denmark. For analysis of E1 and E2, a new method for simultaneous quantitative determination of estrogens, also including estriol (E3) in human serum by isotope dilution online TurboFlow-LC-MS/MS was developed and validated (all supplementary material and figures are located in a digital research materials repository ( 26) ). In short, after the thawing of 200 \\u00b5L of each serum sample, isotope-labelled internal standards were added to serum calibration and control samples, and the estrogens were purified from the serum by liquid\\u2013liquid extraction using heptane/ethyl acetate. The analysis was performed on a Dionex UltiMate 3000 UHPLC system with integrated Transcend TLX TurboFlow sample preparation system coupled with triple quadrupole mass spectrometer (TSQ Quantiva) from Thermo Scientific controlled by Aria MX 2.2 and Xcalibur 4.0 software. For further sample extraction and chromatographic separation of the estrogens, the TurboFlow-LC system was equipped with a loading Cyclone-P TurboFlow column followed by an analytical Kinetex\\u00ae Phenyl-Hexyl column. The MS/MS system was equipped with a heated electrospray ionization source (HESI) running in negative mode. The total duration time was 5.50 minutes. Injection volume, flow rate, solvents and solvent programming, optimized MS/MS interphase settings and MS transitions, retention times, collision energies, and S-lens settings optimized for each single analyte are shown in the supplementary material see ( 26 ). LC-MS/MS-extracted ion chromatograms of E1, E2, and E3 including internal standards, qualifier, and quantifier ions in the lowest calibration standard are shown in ( 26 ). By linear regression based on area ratios (sample area/internal standard area) the concentrations of unknown samples and control material were determined. For method validation and all other analyses two calibration curves in Milli-Q water were included at the beginning and the end of all sample batches. Method validation was based on repeated calibration curves made in a serum pool and Milli-Q water. Matrix effect, ion suppression, linearity, limit of detection (LOD), and quantification (LOQ) were calculated, and the intraday variability including accuracy (% recovery) and precision (relative standard deviation (RSD)) for a low, medium, and high concentration levels (Q low, Q middle, and Q high) were estimated using the regression function in Analysis Toolpak for Microsoft Excel 2007 according to international guidelines ( 27 ). The interday variation (precision) was estimated from the analysis of control material analyzed in duplicates in 20 different batches during a period of 2 months. Validation of the new estrogen LC-MS/MS method for clinical use included testing of stability in full blood and serum, storage at different temperatures over time, repeated freezing/thawing cycles, and interference by hemolysis. Our lab participates in the external quality assessment program Steroid Hormones from the United Kingdom National External Quality Assessment Service ( 28 ). The analysis is accredited by the Danish Accreditation Fund ( 29 ) according to the DS/EN 15189 standard for medical laboratories. All serum samples used in the present study for establishment of sex- and age-specific reference ranges were analyzed in 48 batches over a period of 8 months. Each batch included standards for calibration curves, approximately 40 unknown samples, 2 blanks, and a total of 8 control samples, including 2 serum pool controls, and 2 sets of serum pool controls spiked with native estrogen standards at low, medium, and high levels, and an unspiked serum pool. Progesterone, 17-hydroxyprogesterone (17-OHP), androstenedione, testosterone (T), dehydroepiandrosterone sulfate (DHEAS), and E1S were analyzed in cohort 4 samples by LC-MS/MS as previously described ( 18 ). Quantification limits were 0.036 nmol/L (progesterone), 0.1 nmol/L (17-OHP), 0.042 nmol/L (androstenedione), 0.012 nmol/L (T), 19 nmol/L (DHEAS), and 0.026 nmol/L (E1-S). Samples were analyzed in 4 batches. The interday variation expressed as the relative standard deviation (RSD) were <6% for these 6 steroids. Until September 2014, serum concentrations of sex hormone-binding globulin (SHBG) were determined by a time-resolved fluoroimmunoassay (AutoDELFIA; PerkinElmer, Turku, Finland) with a LOD of 0.23 nmol/L and interassay coefficients of variation (CVs) of the SHBG assay of <7%. The assay has previously been validated in our laboratory ( 30 ). From September 2014 onward, SHBG was determined by a chemiluminescence immunoassay (Access2, Beckman Coulter, Brea, CA, USA) with a LOD of 0.33 nmol/L and interassay CV of the SHBG assay of <9%. Calculations and statistics Free estradiol (free E2) and free estrone (free E1) were calculated based on Mazer ( 31 ), who determined new constants, mean K SHBG_x -values, and mean of K Alb_x -values based on SHBG and albumin data from 6 different studies and integrated them into the original equation by Vermeulen et al. ( 32 ): X = X t o t a l ( 1 + K S H B G X \\u2217 [ S H B G ] M W S H B G + K A l b X \\u2217 [ A l b ] M W A l b + K C B G X \\u2217 [ C B G ] M W C B G ) where X = E1 or E2 In our calculations, binding to cortisol binding globulin (CBG) was not included due to missing cortisol concentrations. Thus, the equation for free E2 (or E1) is: F r e e E 2 = E 2 m e a s u r e d p m o l / L 1 + ( K S H B G E 2 L m o l \\u2217 S H B G m e a s u r e d g L M W S H B G g m o l ) + ( K A L B E 2 L m o l \\u2217 43 g L M W A L B g m o l ) = X X p m o l / L where the binding constants used were 5 \\u00d7 10 8 L/mol (K SHBG E2 ), 4.55 \\u00d7 10 4 L/mol (K ALB E2 ), 9.4 \\u00d7 10 7 L/mol (K SHBG E1 ), and 3.6 \\u00d7 10 4 L/mol (K SHBG E1 ) according to Mazer ( 31 ). Reference charts were developed using the Generalized Additive Models for Location, Scale and Shape (GAMLSS). In the analysis, the data were summarized in 3 smoothed age-dependent curves: L, M, and S, of which the L-curve adjusts age-dependent skewness, the M curve corresponds to the age-dependent median, and the S-curve is approximately the age-dependent coefficient of variation. The applied method was based on the Box\\u2013Cox power transformation, which transformed the data to follow a Gaussian distribution for each age. Calculation of the standards deviation (SD) score was based on the following equation: S D S c o r e = ( X M ) L \\u2212 1 L \\u00d7 S . where X was the hormone concentration and L \\u2260 0 ( 33 ). The GAMLSS method allows back-calculation to obtain selected percentiles in the original distribution. In the reference charts, we show the 2.5th, 16th, 50th, 84th, and 97.5th percentiles corresponding to mean \\u2013 2 \\u00d7 SD, mean \\u2013 1 \\u00d7 SD, mean, mean + 1 \\u00d7 *SD, and mean + 2 \\u00d7 SD, respectively. GAMLSS analysis was performed using R (version 3.5.2; R Core Team, R Foundation for Statistical Computing, Vienna, Austria ( http://www.R-project.org/ )) and the GAMLSS package. For graphic illustration of estrogen reference areas, the concentration values of E1 and E2 below LODs, that is, 2.93 pmol/L and 4.04 pmol/L, respectively, were replaced with values of 1 pmol/L. The corresponding calculated values of free E1 and free E2, where E1 and E2 concentrations were below LODs, were replaced with the value 0.01 pmol/L. To compare medians across menstrual cycle groups (categorized as day 1\\u20137, 8\\u201314, and 15+), the 2-tailed Mann\\u2013Whitney U-test was used. P < .05 was considered statistically significant. Calculations were performed using IBM SPSS Statistics, version 22 (IBM Corporation, Armonk, NY, USA). Graphical presentations were performed in Excel and GraphPad Prism 7. Ethical considerations For all participants, a written, informed consent form was signed before study participation, either by the participant or by their parents, in the case of children below 18 years. The cohort studies were approved by the ethics committees of the Capital Region of Denmark (RegionH): cohort 1, no. [KF] 01-030/97; cohort 2, no. 1-10-72-631-12, the study is registered in ClinicalTrials.gov (identifier NCT01944488); cohort 3, no. KF 01 282214 and V200.1996/90, the study is registered in ClinicalTrials.gov (identifier NCT01411527); cohort 4, no. H-KA20060011; and cohort 5, no. H-I-2011\\u2013081. Furthermore, the studies were approved by the Danish Data Protection Agency: cohort 1, no. 2003-41-2996; cohort 2, no. 2007-58-0010; cohort 3, no. 2010-41-5042; cohort 4, RH-2015\\u2013299; and cohort 5, no. 2012-58-0015. The present study is in accordance with the Helsinki II declaration.\",\n",
            "        \"subjects\": null,\n",
            "        \"results\": null,\n",
            "        \"discussion\": null,\n",
            "        \"conclusion\": null\n",
            "    },\n",
            "    {\n",
            "        \"pmcid\": \"PMC7449478\",\n",
            "        \"abstract\": null,\n",
            "        \"intro\": \"Introduction Life expectancy is increasing worldwide [ 1 ], contributing towards an increasing demand on health and social care resources [ 2 ], because human function deteriorates with age [ 3 , 4 ]. There is an identified need for research on maintaining wellbeing of older people [ 5 ], to assist declining numbers of professional care workers [ 1 ]. Improving wellbeing is essential for those in long term nursing facilities, who are vulnerable to feelings of isolation and loneliness [ 6 ], and those with dementia, a condition associated with changes referred to as behavioural and psychological symptoms of dementia (BPSD), and includes agitation, anxiety, depression, delusions and hallucinations [ 7 ]. BPSD can reduce wellbeing, but also increase care provider burden and distress [ 7 , 8 ], hospitalisation and healthcare costs [ 7 ] and is associated with institutionalisation and medication use, including antipsychotics, which have serious side effects [ 8 ], including cardiovascular issues [ 9 ], and mortality [ 10 ]. Companion robots may provide a non-pharmacological psychosocial intervention to assist with these healthcare challenges. A systematic review showed that there was a wealth of research available on the use of social robots, or companion robots in care and long term nursing homes [ 11 ], with various robots and interactive toys available [ 12 , 13 ]. Much of the previous research focused on Paro the robot seal [ 14 ]. The benefits of interaction with Paro for older adults, including those with dementia, are reduced depression and agitation [ 15 ], more adaptive stress response [ 16 ], reduced loneliness [ 17 ], and reduced nursing staff stress [ 16 , 18 ]. Paro may also reduce use of psychoactive and analgesic medications [ 19 ], and even lower blood pressure [ 20 ]. Nursing staff previously discussed perceptions of Paro, noting the usefulness for older people and potential social benefits, with the device aiding interpersonal relationships [ 21 ]. It should be noted, the aim of companion robots is to augment human care, rather than replace. Similar is true of robots used in other care contexts (for example children with autism) [ 22 , 23 ], and support has been reported for the social mediation effect of such devices [ 17 , 24 ]. However, little has been published on practical maintenance considerations of companion robot use. A review of benefits of and barriers to Paro implementation in care settings noted infection concerns as a key barrier [ 25 ]. The Health Protection Agency [ 26 ] provides guidance for community infection control nurses, health protection nurses, and care home staff including the decontamination of equipment, but little is known about how to do this for new technologies such as companion robots. We demonstrated Paro and other robot animals and toys to hundreds of people as part of the eHealth Productivity and Innovation in Cornwall and the Isles of Scilly (EPIC) project [ 27 ] in 2017\\u201318 in Cornwall, including many nurses and care home staff, who frequently raised concerns of hygiene and infection control. We also found in other work [ 28 ] that relevant stakeholders expressed concerns regarding cleaning. The Department of Health and Social Care [ 29 ] suggests good infection control is imperative to ensure service users receive safe care. A previous large-scale randomised controlled trial of Paro in long-term care facilities described the employed hygiene protocol [ 30 ], including cleaning Paro after each use with disinfectant spray and wipes, and cleaning the storage box weekly. This reflects the cleaning procedure suggested by the Paro website [ 31 ]. However, research was lacking on the efficacy of such procedures, or any potential risk that companion robots pose for care home residents in terms of microbial transmission. Background We are aware of only two studies on infection control and Paro [ 32 ], only one of which reported a cleaning procedure based on use of the robot on a UK National Health Service (NHS) dementia ward for 9 months [ 33 ]. Dodds et al. included a broad cleaning protocol discussing risk reduction measures and processes before, during and after use of Paro. Results suggested cleaning was successful based on Adenosine Triphosphate (ATP) luminometer readings of below 50 relative light units (RLU). The authors, however, acknowledged the limitations of the assessment method [ 34 ], as although it provided an estimation of surface cleanliness it is impossible to convert luminometric results to number of microorganisms [ 35 ]. Sygula-Cholewinska et al. [ 35 ] suggested many studies indicated that intracellular ATP levels vary so much between microbial taxa that tests of ATP should not be viewed as indicative of the presence of microbial pathogens. They suggested the method should not be commonly applied due to limitations such as low sensitivity of commercial luminometers for microbe detection, poor result reproducibility, and environmental factors influencing measurement outcomes [ 35 ]. A literature review by Health Protection Scotland [ 36 ] found most studies showed no correlation between ATP and microbial contamination. They concluded there was insufficient evidence to support using ATP as a marker of microbiological cleanliness. The protocol described by Dodds et al. [ 33 ], therefore, has limited quantitative microbiological support, as noted by Rowson and colleagues [ 34 ]. Furthermore, the research was limited only to Paro, that is reported to have anti-bacterial fur [ 37 ], thus restricting generalisability of results to a wider selection of companion robots that do not generally have anti-microbial coverings. There was also no identification of microbes conducted, and samples were taken periodically over 9 months, rather than before and after cleaning [ 33 ]. Thus, no comparison was provided to demonstrate the impact of the cleaning on either microbial load or removal of specific microbes. There was, therefore, still a strong requirement for research using more valid and standardised methods [ 34 ], as well as a range of companion robot alternatives that do not have the anti-bacterial properties of Paro, to begin establishing a tested cleaning procedure for companion robots used by older adults. Previous research investigating general cleaning efficacy includes work by Santos-Junior et al. [ 38 ], who sampled high-touch surfaces in a nursing ward before and after cleaning. They used ATP bioluminescence assay, aerobic colony counts (ACC), staphylococcus aureus colony count, and resistance to methicillin [ 38 ]. They collected 80 samples over four weeks, 40 before cleaning and 40 samples 10 minutes after cleaning to allow disinfectants to dry. The disinfectant used was NIPPO-BAC PLUS. They collected samples with contact plates containing tryptone soya agar with neutralizers. Results were analysed following incubation, and suggested only two of the five sites tested demonstrated significant decrease in RLU. ACC results showed that on two sites, microbial load was higher after cleaning and disinfection. They concluded the cleaning and disinfection process showed little effectiveness. Kenters et al. [ 39 ] also tested cleaning efficacy, exploring effectiveness of various disinfectants, using a known positive method of contaminating tiles with a test solution of clostridium difficile strains. The authors compared wipes and sprays of various ingredients using colony count and ATP. Their results suggested that wipes performed better than sprays with the same active ingredient. Wipes including hydrogen peroxide (1.5%) demonstrated the highest bactericidal activity. Woodland, Whitham, O\\u2019Neil & Otter [ 40 ], assessed colony counts on healthcare cubical curtains before and after cleaning. They used swabs to sample from high-touch areas of 20 curtains. Samples were incubated then colony-counts were conducted and micro-organisms were identified using gram stain and colony morphology. Colony counts increased slightly immediately after laundering before declining by 56% after one week, and the two most frequently present microorganisms were coagulase negative staphylococcus and micrococcus species . They suggested current laundry procedures may not be completely effective. A limitation, however, of this study was reliance on swabbing, which can create greater variation in sampling than more standardised methods such as contact plates [ 41 ]. Similar research on infection control for companion robots appears lacking, other than that of Dodds et al. [ 33 ]. Indeed, a literature review of hygiene for robotic animals in hospitals identified that related research focused only on children\\u2019s toys and dolls [ 42 ]. The authors concluded little is known about the hygienic application of robotic animals in the clinical setting [ 42 ]. Previous research investigating microbiological hazards on children\\u2019s toys and play equipment included Mart\\u00ednez-Bastidas et al. [ 43 ], who found interaction with play-park equipment influenced microbial presence on both children\\u2019s hands and toys. E . coli was predominant, but staphylococcus aureus , klebsiella pneumonia , serratia , giardia lamblia and hepatitis A were also found. The importance of these results is emphasised by other studies that suggested a chain of transmission of infection not only from person to person, but from fomites (objects) to people [ 44 , 45 ]. Randle and Fleming [ 46 ] supported this concern, finding toys specifically can spread infection between children in healthcare settings. Rowson and colleagues [ 34 ] discussed infection control concerns with Paro noting that soft-toy type shells are notoriously difficult to decontaminate, with no clear guidelines present on best practice. They also acknowledged the need for quantitative microbiological evidence on adequacy of any decontamination procedures, particularly when considering robot use with vulnerable older adults and those with dementia [ 34 ]. Older adults may be particularly vulnerable to health consequences when exposed to pathogens due to a decline in immune function with ageing [ 47 ]. Older people also have reduced levels of gastric acid, and consequently experience increased risk of developing infectious gastroenteritis [ 48 ]. Furthermore, older adults residing in care homes are at particular risk, due to concentration of high-risk individuals in the environment, and the susceptibility of this environment to spreading pathogens [ 49 ]. Infections in nursing home samples are associated with higher rates of morbidity and mortality, hospitalisation, and healthcare expense [ 48 ]. It is therefore important to establish if companion robots can transmit potentially harmful microorganisms between users and to assess efficacy of cleaning methods to allow safe use of companion robots in such settings. This paper therefore begins to contribute to the necessity noted by Scholten et al. [ 42 ], for research furthering our knowledge on robot animals and infection control. Although Paro appears to be the most well researched companion animal robot [ 14 ], other interactive toys and robots are commercially available, such as the dinosaur Pleo, Miro, or the Joy for All cat and dog. Some of these cheaper devices have been used in previous research with older adults [ 12 , 13 ]. We therefore included a range of commercially available toys and robots with potential for use with older adults. As Paro has been designed with anti-bacterial fur that can be washed with anti-bacterial products [ 37 ], our study provides a comparison with the surfaces of possible alternative robots. Our study thus has implications for: (i) the use of current companion robots in health and social care settings, (ii) the materials to be used in future robot design, (iii) cleaning procedures for robots and toys in care homes and similar contexts, either for real-world or research purposes.\",\n",
            "        \"method\": \"Method Setting This investigation formed part of a collaborative action research project exploring use of companion robots and alternatives in care homes for older adults and people with dementia. Non-probabilistic convenience sampling was used to select two care homes as research sites. Both homes provide residential care for individuals with and without dementia. Four residents in each home volunteered to take part. In the first home, four females participated with a mean age of 86 (SD 14.84). In the second home, three females and one male participated, with a mean age of 90.75 (SD 4.09). The study also involved collaborating with a microbiology laboratory, which follows UKNEQAS [ 50 ] and LABQUALTY [ 51 ] for external quality assurance of bacterial identification, and is also UKAS accredited [ 52 ]. Ethical approval for this study was discussed and waived by the Faculty of Science and Engineering committee at the University of Plymouth, as data collection involved no human participants, older adults volunteered to assist in handling companion robots, as they are familiar with them for non-research purposes. A highly ethical approach was taken, with written consent gained from collaborators who were fully informed on research aims and potential implications. The Microbiology Investigation Criteria for Reporting Objectively (MICRO) checklist was used to guide the writing of this manuscript ( S1 File ), although not all points were deemed relevant to this study design [ 53 ]. Design Our study had two parts: In stage one we investigated the microbial load on eight devices ( Fig 1 ) following use, to establish contamination and infection risk. Tests were repeated after cleaning by the researcher, to assess efficacy of the procedure. 10.1371/journal.pone.0237069.g001 Fig 1 Eight robot and toy animals used in stage one. From left: Paro, Miro, Pleo rb, Joy for All dog, Joy for All cat, Furby Connect, Perfect Petzzz dog, Handmade Hedgehog. In stage two, we repeated this using only two animals (Joy for All dog and cat) with care staff themselves conducting the cleaning. The cat and dog had been present in the home for eight weeks, undergoing cleaning after each use by the care staff. Our procedure and materials were otherwise identical to stage one. Both stages involved collection of environmental specimens during December 2018, in Cornwall, UK. Materials Robots A range of robots and alternatives were used ( Fig 1 ). Selection was based on current involvement in the larger project, and through providing a range of shell types and materials currently used on socially assistive robots ( Table 1 ). 10.1371/journal.pone.0237069.t001 Table 1 Shell types of the robot animals and alternatives. Animal Shell Type Fur Length (approx.) Paro [ 54 ] Anti-bacterial, anti-static soft fur (exact composition protected under intellectual property, but includes silver particles for anti-bacterial properties) 1cm Miro [ 55 ] Hard, smooth plastic N/A Pleo rb Dinosaur [ 56 ] Soft textured plastic (SEBS thermoplastic elastomer) N/A Joy Dog [ 57 ] Soft-toy fur (polyester, acrylic mix) 1cm Joy Cat [ 58 ] Soft-toy fur (polyester, acrylic mix) 2.5cm Furby [ 59 ] Soft-toy fur (polyester and acrylic mix) and hard smooth plastic 0.8cm Perfect Petzzz Breathing Dog [ 60 ] Soft-toy fur (100% polyester) 0.6cm Knitted Hedgehog [ 61 ] Soft toy fur (polyester and lurex mix) 2cm Cleaning products We used the following cleaning products for disinfection of the devices: (i) Sirafan Speed Disinfection Spray for Surfaces by Ecolab [ 62 ], and (ii) Super-Sani Germicidal Wipes by PDI [ 63 ]. Both companies currently supply disinfectants to health care providers. The use of both a spray and wipes was suggested by Moyle et al. [ 30 ] and the Paro user manual [ 31 ]. The PDI Super Sani-Cloths were selected as they are recommended for use in health care and medical settings to control cross contamination hazard, and also in the Paro cleaning instructions [ 31 ]. The wipes also allow for wiping of hard surfaces on devices, such as noses or eyes, and to allow the anti-bacterial product to be worked thoroughly into fur-type shells. Furthermore, research suggesting superiority of wipes over sprays despite similar composition [ 39 ]. The PDI company suggests these wipes are bactericidal, tuberculocidal and virudicidal, with broad coverage of microorganisms, including multi-drug resistant organisms [ 63 ]. The active ingredients include Isopropyl Alcohol, n-Alkyl dimethyl ethylbenzyl ammonium and chlorides. Although we have not tested for viruses here, this product also appears on the USA Environment Protection Agency List N of disinfectants meeting criterion for use against SARS-CoV-2 [ 64 ]. The Sirafan Speed Spray was suggested for trialling by contacts at Ecolab, due to the speed of disinfection and lack of rinse necessity, as rinsing is unfeasible for devices without removable skins. The disinfectant is suggested to be effective against bacterial, viral and fungal infections [ 62 ]. The active ingredients include Isopropyl Alcohol and 1-Propanol. Products were selected for being more powerful than everyday disinfectants, due to the importance of intensifying disinfection on high-touch surfaces that could allow transmission of pathogens to service users [ 38 ]. Although both products are designed for hard surfaces, there is a lack of disinfectant products available specific to soft surfaces, and therefore currently available products may provide adequate substitutes. PDI and Ecolab currently supply to health and social care facilities, so the chosen products are easily accessible. Agar plates We used agar filled contact plates, supplied by Cherwell Laboratories. Irradiated tryptone soya agar was used, with four neutralisers to inactivate residual disinfectants. Plates were triple vent contact plates with a surface area of 25cm 2 . This type of agar is a general purpose nutrient agar currently used in environmental sampling, and is recommended for recovering a variety of microorganisms. Tryptone soya agar was used in previous research [ 38 ]. Procedure The research was conducted in two care homes, reflecting the intended \\u2018real-world\\u2019 use of companion robots [ 11 ]. Devices were taken to two care homes providing residential care for older adults with and without dementia. Devices were cleaned using the described procedure ( Fig 2 ) on site to minimise any influence of microbes collected during transportation. 10.1371/journal.pone.0237069.g002 Fig 2 Cleaning procedure for use with socially assistive robots in care homes or other health and social care contexts. The process of cleaning each robot or toy. The procedure took approximately six minutes, with additional drying time. This was applied to both soft-furry and hard-plastic shell types. Storage boxes and associated components such as chargers were also cleaned weekly using the same method. Once cleaned, four care home residents interacted with the robots, in a group session reflective of real-world use and research practice [ 11 , 14 ]. The four participants were invited to interact with each robot for five minutes with each robot receiving 20 minutes of interaction. The researcher then sampled from the robots using contact agar plates to gain a measure of microbial load after use. Contact plates were applied to the sections of the robots most commonly touched based on review analysis of previous video recordings of 45 different care home residents interacting with each of the eight animals. This sampling of high-touch areas reflects previous methodology [ 38 , 40 ]. The plate was in contact with the robot for 10 seconds, as in previous research [ 41 ]. The robots were cleaned again using the suggested hygiene procedure ( Fig 2 ), then sampling was repeated to examine the efficacy of the cleaning method. This before and after cleaning sampling is suggested to be an established method of evaluating cleaning and disinfection practices [ 34 , 65 ]. All sampling from the robots was conducted by the same researcher to standardise sample collection. Sixteen samples were collected in stage one, with each of the eight animals being sampled from once before cleaning and once after. Four samples were collected in stage two, with two animals being sampled before and after cleaning. Previous research by Woodland et al. [ 40 ], used swabs for testing microbial contamination of cubicle curtains in a health care setting, however the contact plate method allowed greater standardisation, and was used in previous research [ 38 ]. Sampling via swabbing requires two processes; sampling from the object itself and inoculation of the plate, while the contact plate method allows for inoculating of any bacteria directly from the object to the agar [ 41 ]. Analysis Samples were transported straight to the collaborating microbiology laboratory and incubated at 30\\u201335\\u00b0C for 5 days to grow any environmental organisms or enteric/pathogenic bacteria sampled from the animals. Colony counts were conducted at 48 and 120 hours, and CFU/cm 2 calculated, providing an indicator of how \\u2018unclean\\u2019 robots become during standard care home use, and to assess the efficacy of the cleaning procedure, and initial comparisons of shell type. A threshold of \\u22642.5 CFU/cm 2 was considered acceptable, based on previous research [ 36 , 38 , 65 ]. In stage one, identification was conducted on colonies remaining after cleaning using gram stain, colony morphology and coagulase agglutination as in previous research [ 40 ]. This was to ascertain what microbes had remained following cleaning. In stage two, identification was conducted on micro-organisms present before cleaning, using the same methods. This allowed insight into microbes potentially transmitted on companion animals, and analysis of what microbes were removed during cleaning. The datasets generated and analysed during the current study are available at the Open Science Framework using the following link: https://osf.io/4qud9/?view_only=183ae25f030a4e0b905a50286f99ca8c\",\n",
            "        \"subjects\": null,\n",
            "        \"results\": \"Results Stage one Most of the devices gathered enough microbes during 20 minutes of standard use to have a microbial load above the acceptable threshold of 2.5 CFU/cm 2 ( Table 2 ). 10.1371/journal.pone.0237069.t002 Table 2 CFU/cm 2 on each robot before cleaning and after cleaning at 48 and 120 hours incubated. Animal Before Cleaning After Cleaning 48 120 48 120 Paro 3.20 3.20 0 0 Miro 0.04 1.08 0 0.64 Pleo 3.84 4.48 0.04 0.04 Joy for all Dog 8.96 9.60 0 0 Joy for all Cat 1.28 1.92 0 0 Furby 10.88 10.88 0.04 0.04 Perfect Petzzz Dog 17.28 19.20 0 0 Hedgehog 2.56 3.20 0.08 0.08 The Perfect Petzzz dog demonstrated particularly unacceptable levels, followed by the Furby and Joy for All dog. Only two of the animals remained within acceptable levels following use, the Joy for All cat and Miro. The post-cleaning CFU/cm 2 , however, demonstrates that regardless of material type, or previous microbial load, the described cleaning procedure effectively brought the CFU/cm 2 on each animal down to well below acceptable levels, thus strongly supporting cleaning efficacy for bacterial contamination. Further to post-cleaning results being well within recommended limits, the remaining colonies following cleaning were identified as aerobic spore-bearers which are ubiquitous in the environment and pose relatively little risk. Stage two The cleaning procedure was effective when carried out by care home staff (rather than the researcher). Using the benchmark of \\u22642.5 CFU/cm 2 , it is clear microbial load on the animals was high following a group session, but that cleaning by a care staff member, following the standard procedure ( Fig 2 ) removed microbes ( Table 3 ). 10.1371/journal.pone.0237069.t003 Table 3 CFU/cm 2 before cleaning, and after cleaning by a care staff member, at 48 hours and 120 hours incubation. Animal Before Cleaning After Cleaning 48hr 120hr 48hr 120hr Joy for All cat 24.32 29.44 0 0 Joy for All dog 5.76 10.24 0 0 Identification conducted on the samples taken before cleaning suggested the presence of diptheriods , ASB , micrococcus species , coagulase negative staphylococcus and staphylococcus aureus . Some of these bacteria can present a risk to human health [ 66 , 67 ]. No gram-negative bacteria were present suggesting faecal contamination at time of sampling was unlikely. No colonies were present following cleaning.\",\n",
            "        \"discussion\": null,\n",
            "        \"conclusion\": \"Conclusion Companion robots hold significant potential for improving aspects of health and wellbeing for older adults. Numerous benefits have been reported, however research has been lacking on the important factor of infection control. We have demonstrated through colony counts and microbe identification that robots and toys can pose a bacterial infection control risk in health and social care contexts such as care homes. Our simple cleaning procedure has efficacy and gives some confidence that companion devices with a range of soft and hard shell types can be used relatively safely and that cheaper devices are no more risky than Paro. However, further research is needed both addressing viral infections and the effectiveness of our procedures in situ in the longer term.\"\n",
            "    },\n",
            "    {\n",
            "        \"pmcid\": \"PMC7657407\",\n",
            "        \"abstract\": null,\n",
            "        \"intro\": \"Introduction Infertility is defined as the failure to achieve a pregnancy after 1 year of regular unprotected intercourse.[ 1 ] An estimated 9%\\u201332% of couples around the world suffer from infertility.[ 2 ] According to a review study, the rate of infertility in Iran was estimated at 34.1%.[ 3 ] The prevalence of primary infertility has been reported by 20.2% in Iran.[ 4 ] Infertility by creating a disturbing and debilitating condition is an important individual problem.[ 5 ] Both infertility and its treatment techniques are associated with serious mental issues.[ 6 7 ] Infertility stress and anxiety can cause marital maladjustment, sexual dysfunction, reduced self-confidence, and reduced sexual relationship,[ 8 ] and in many infertile couples, these problems may prevent the continuation and completion of the treatment.[ 9 ] A study has reported that the divorce rate is 3.7 times higher in infertile women.[ 10 ] In addition, the prevalence of depression in this group equals to cancer- or heart disease-induced depression.[ 11 ] In recent years, the mental health of infertile couples has been given a serious attention, and infertility has been an undeniable severe experience of helplessness for many couples.[ 12 ] According to the World Health Organization (WHO), mental health is defined as \\u201csubjective well-being, perceived self-efficacy, autonomy, competence, intergenerational dependence, and self-actualization of one's intellectual and emotional potential among others.\\u201d[ 13 ] In a study, the presence of mental disorders was reported in 40.2% of infertile women, with a prevalence of 23.3% for general anxiety disorder, 17% for major depression, and 9.8% for chronic depression.[ 14 ] Peterson and Eifert (2011) observed severe depressive symptoms in 11.6% of infertile women and 4.3% of infertile men.[ 15 ] Many studies have shown that infertility has a negative effect on the quality of life.[ 16 ] The WHO defines the quality of life as \\u201can individual's perception of his/her position in life in the context of the culture and value systems in which he/she lives and in relation to his/her goals, expectations, standards and priorities.\\u201d[ 7 ] A systematic review study showed that infertility affects couples in the following four aspects of their life including mental health, marital relationships, sexual relationships, and quality of life.[ 17 ] Irrational beliefs about the parenthood and economic burden of infertility treatments have an adverse impact on mental health of this group of patients, necessitating psychological and counseling supports.[ 16 ] One of the most important approaches in this regard is counseling.[ 18 ] Acceptance and commitment therapy (ACT) is a novel technique for behavioral therapy that employs mindfulness, acceptance, and cognitive diffusion skills to enhance psychological flexibility and prevent behavioral changes in favor of chosen values.[ 19 ] Given the high prevalence of infertility and its adverse individual and social effects, the importance of mental health in this group, lack of using ACT in Iranian infertile couples, and previous recommendation of a meta-analysis study for further study on the effect of ACT on quality of life,[ 20 ] this study aimed to determine the impact of counseling based on ACT on mental health and quality of life in infertile couples. Holding counseling classes in centers can be a successful step in improving the quality of provided services and achieving healthier communities and greater prosperity.\",\n",
            "        \"method\": \"Methods Study design and participants This randomized controlled trial was carried out on 54 infertile couples (both spouses), referring to the Besat Infertility Clinic in Sanandaj, Iran, between July 23, 2017, and February 20, 2018. Inclusion criteria included couples with initial infertility of 1\\u201310 years, couples 20\\u201345 years old, having at least reading and writing literacy, and at least once failed treatment of infertility experience. Exclusion criteria included earning a score of 41 from the Mental Health Questionnaire, occurrence of an unpleasant event during the 6 months before the study (death or acute illness of relatives), not having any known mental or physical illness such as diabetes, depression (according to the participant's statement), participation in another intervention study, and positive pregnancy test during the sampling and intervention. Sample size The estimated sample size was 10 based on Narimani et al .'s study[ 21 ] into the quality of infertile women's life (Narimani et al ., 2014), considering m 1 = 11.05, m 2 = 13.44, SD 1 = 1.32, SD 2 = 1.76, test power of 80%, and one-sided \\u03b1 = 0.05. In addition, the estimated sample size was 24 based on Hashemi et al .'s study[ 22 ] into infertile women's mental health (Hashemi et al ., 2014), considering m 1 = 41.83, m 2 = 33.47, SD 1 = 11.46, SD 2 = 1, test power of 80%, and one-sided \\u03b1 = 0.05. As a result, since the sample size was greater based on the mental health, the ultimate sample size estimation was 27 couples, considering a possible sample loss of 10%. Sampling For sampling, the researcher attended at the infertility clinic to obtain the phone numbers of the infertile couples. Then, through phone calls, the researcher was introduced and they were informed about the general objective of the study and provided with a brief description of the research. In this stage, the initial enrollment of the consenting infertile couples was performed, and they were scheduled to attend the clinic. The eligibility of the couples in terms of the inclusion and exclusion criteria was ensured at the clinic. Then, the eligible and consenting couples were asked to sign the consent form. After obtaining their informed written consent, the pretest questionnaires, including demographic information, General Health Questionnaire-28 (GHQ-28), and Fertility Quality of Life (FertiQoL) questionnaire, were completed through self-reporting. Among 116 couples visiting the Besat Infertility Clinic, 24 couple were excluded for unwillingness to participate, 26 couple for ineligibility (7 couples aged over 45 years, 13 couples with secondary infertility, and 6 couples for illiteracy), 8 couples with scores of higher than 41 in the GHQ-28, and 3 couples for a death in the family within 6 months before the study; in addition, one couple was excluded as the female spouse developed diabetes. The remaining 54 couples were randomized into two equal-sized groups, namely intervention and control. There was no sample loss, and all participants were analyzed [ Figure 1 ]. The researchers paid the cost of transportation of participants to the clinic. They appreciated the participants for their time by providing them with the results. Figure 1 Flow chart of the study Randomization and intervention The participants were equally allocated into the counseling and control groups (1:1), using the randomized block design with blocks of size 4 and 6. The allocation process was done by someone not involved in data sampling and analysis. To conceal the allocation, the type of intervention was written on a paper and sealed in an opaque and sequentially numbered envelope according to random allocation sequence. The first author (MH) provided counseling under supervision of a psychologist (KS) in groups of 8\\u201312 based on the ACT approach. The counseling sessions were held in a room of infertility clinics at a friendly atmosphere that reinforced self-confidence of participants and facilitated their participation in group discussion. The control group did not receive any intervention. To avoid the contamination, participants receiving the intervention were requested not to share information with other participants. At the end of the study, the contents of the counseling sessions were prepared in the form of a booklet and given to the control group. A summary of the counseling sessions is as follows. At the end of each session, the couples were asked to practice the lessons covered during the week and to attend the next session well-prepared. During the intervention period, no couple was absent. During the sessions, exercises, questioning, and feedback were reviewed too. A summary of the counseling sessions was as follows: First session: Introducing the members, describing the rules and responsibilities of group members, presenting the objectives, and explaining the ACT-based counseling. General assessment: Determining the main complaints of the participants, determining the experimental avoidance of the participants, specifying the thoughts and their underlying feelings, and identifying the strengths of the participants Second session: Collecting the previous session's assignments; providing a brief description of the reproductive system physiology and anatomy, defining infertility, presenting infertility statistics and prevalence, and describing the effects of infertility on different life aspects. Informing about the methods of pharmacotherapy and surgical treatments for infertility and their success rates Third session: Collecting the previous session's assignments; delivering mindfulness technique education. Teaching mindfulness techniques to deal with empirical avoidance was reinforced by practice and repetition. Hence, the participants did mindfulness exercises several times a day by learning how to realize their feelings, emotions, and thoughts and observed everyday tasks such as conscious breathing and eating with full awareness. These exercises were performed in groups and in practice in the classroom, and the participants were asked to repeat them regularly at home Fourth session: The previous session assignments were retrieved from the couples at the beginning of the session, and the following subjects were discussed. Teaching hopelessness technique using hole and shovel metaphor; exercise: Thought suppression to show uncontrollability of mind; homework: Self-monitoring Fifth session: Collecting the previous session's assignments; introducing the acceptance concept: Using the guest metaphor and introducing concept of self-monitoring: Thought suppression techniques and relevant exercises: Watching the thoughts, taking the role of television reporter, or singing the thoughts Sixth session: Collecting the previous session's assignments; introducing the concept of values and commitment actions, difference between value and objective, and the list of value domains, and using birthday ceremony metaphor; homework: Completing the worksheet of objectives, values, effective measures, and barriers Seventh session: Collecting the previous session's assignments; starting the session with exercise of contact with the present moment, controlling and assessing worksheets, and introducing and teaching cognitive dissonance using the bus metaphor Eighth session: Collecting the previous session's assignments; reviewing mindfulness techniques, and reviewing exercises. Data collection tool The demographic inventory, GHQ-28 (General Health Questionnaire-28), and FertiQoL questionnaire were used for data collection. The demographic inventory was administered before the intervention. The GHQ-28 and FertiQoL were completed by the participants before and 1 month after the intervention in the form of self-report. The participants referred for posttest session 1 month after the intervention as individually (the woman and her spouse together). The demographic inventory included items on age, weight, length of marriage, duration of infertility, job, education, income, source of infertility knowledge, and cause of infertility. The GHQ-28 was developed in 1970 by Hiller and Goldberg to measure four areas of physical function, anxiety symptoms and sleep disturbance, social function, and depression symptoms. Each area was comprised of 7 items. Each response was scored on a four-point scale (not at all, not more than normal, more than normal, and much more than normal), with lower scores indicating the normal mental health and higher scores reflecting a mental health problem. Each item was weighted on a four-point Likert scale scored from 0 to 3, resulting in an overall score between 0 and 84 and a cutoff score of 23. Participants with scores higher than 23 were considered susceptible to mental health problem, and those with scores higher than 41 were considered to have an acute condition. This questionnaire was validated by Norbala et al . in Tehran in 2001. They selected 879 men and women using random cluster sampling in a two-stage assessment study and reported, respectively, the overall classification error rate, specificity, and sensitivity of 12.3, 92.3, and 70.5, with a cutoff score of 23.[ 23 ] The FertiQoL questionnaire consists of 36 items with 24 items assessing the fertility-specific quality of life, 10 items assessing the fertility treatment quality of life, and 2 items assessing the overall life and medical health.[ 7 ] This instrument is scored on a Likert scale scored from 0 to 4 with a greater score reflecting the higher quality of life. This is a valid and reliable instrument to measure the effects of fertility problems and their treatment methods on quality of life in Iran and throughout the world. It has been widely used by different studies in Iran.[ 7 23 24 ] The face and content validity of this questionnaire was proved by some specialists in Iran. The Cronbach's alpha of the whole scale was 0.90 and acceptable for the subscales of fertility-specific quality of life and fertility treatment quality of life within the range of 0.72 and 0.92.[ 24 ] Data analysis After data collection, the obtained data were analyzed with SPSS21. The normality of quantitative data was evaluated using the K-S test. The intergroup comparison of demographic information was done using the independent t -test, Chi-square test, Chi-square test for trend, and Fisher's exact test. The independent t -test (if data were normally distributed) and the Mann\\u2013Whitney U-test (if data were not normally distributed) were employed to compare the mean overall score of mental health and its subdomains, as well as the mean score of quality of life and its subdomains before the intervention. With respect to the postintervention, ANCOVA with controlled baseline values (if data were normally distributed) and Mann\\u2013Whitney U-test (if data were not normally distributed) were utilized. All analyses were carried out based on the intention-to-treat analysis. In this study, P < 0.05 was set statistically significant. Ethical consideration All procedures performed in studies involving human participants were in accordance with the ethical standards of the institutional and/or national research committee and with the 1964 Helsinki declaration and its later amendments or comparable ethical standards. In this manuscript after obtaining the approval of the Ethics Committee of Tabriz University of Medical Sciences (Ethics committee 83 reference number: IR.TBZMED.REC.1395.1179) and registering the study in the Iranian Registry Clinical Trial (IRCT20111122008170N13), the written informed consent was obtained from the participants.\",\n",
            "        \"subjects\": null,\n",
            "        \"results\": \"Results There was no significant intergroup difference in demographic information ( P > 0.05). The mean (standard deviation [SD]) age of the females in the intervention and control groups was 31.5 (5.3) and 30.3 (4.6) years, respectively. The mean (SD) age of the males in the intervention and control groups was 33.3 (5.5) and 31.9 (4.8), respectively. The mean (SD) of the duration of infertility in the intervention and control groups was 3.0 (2.1) and 2.4 (1.2) years, respectively. The other sociodemographic characteristics of participants in the study groups are reported in Table 1 . Table 1 Comparison of sociodemographic characteristics of the participants in two groups of counseling and control Variable Counseling ( n =54), n (%) Control ( n =54), n (%) P Woman\\u2019s age ) year) 31.5 (5.3) \\u2020 30.3 (4.6) \\u2020 0.390* Man\\u2019s age 33.3 (5.5) \\u2020 31.9 (4.8) \\u2020 0.296* Female weight (kg) 70.0 (12.5) \\u2020 73.0 (12.5) \\u2020 0.359* Man weight (kg) 84.9 (13.0) \\u2020 82.9 (11.8) \\u2020 0.557* Duration of marriage (years) 5.0 (2.2) \\u2020 5.0 (1.3) \\u2020 1* Duration of infertility (years) 3.0 (2.1) \\u2020 2.4 (1.2) \\u2020 0.227* Woman job Homemaker 19.0 (70.4) 22.0 (81.5) 0.396 \\u01c2 Employed 8.0 (29.6) 5.0 (18.5) Man job Unemployed 2.0 (7.4) 0 0.745 \\u01c2 worker 3.0 (11.1) 3.0 (11.1) Employee 5.0 (18.5) 9.0 (33.3) Self-employment 17.0 (63.0) 15.0 (55.6) Income level At a sufficient level 16.0 (29.6) 16.0 (29.6) 0.914 \\u01c2 Too much 9.0 (16.7) 8.0 (14.8) Less than full 29.0 (53.7) 30.0 (55.6) Female education Elementary 12.0 (44.4) 6.0 (22.2) 0.128 \\u01c2 Secondary 6.0 (22.2) 4.0 (14.8) Diploma 4.0 (14.8) 11.0 (40.7) Bachelor 3.0 (11.1) 5.0 (18.5) Master and higher 2.0 (7.4) 1.0 (3.7) Male education Elementary 7.0 (25.9) 8.0 (29.6) 0.458 \\u01c2 Secondary 7.0 (25.9) 5.0 (18.5) Diploma 8.0 (29.6) 3.0 (11.1) Bachelor 4.0 (14.8) 9.0 (33.3) Master and higher 1.0 (3.7) 2.0 (7.4) Female infertility education Yes 20.0 (76.9) 17.0 (63.0) 0.268 \\u00a5 No 6.0 (37.0) 10.0 (23.1) Male infertility education Yes 22.0 (84.6) 17.0 (63.0) 0.458 \\u00a5 No 4.0 (15.4) 10.0 (37.0) The source of female information Book 4.0 (20.0) 4.0 (23.5) 0.833 \\u00a7 Family physician 10.0 (50.0) 7.0 (41.2) Radio and TV 5.0 (25.0) 6.0 (35.3) Advice class 1.0 (5.0) 0 The source of female information Book 1.0 (4.5) 6.0 (35.3) 0.051 \\u00a7 Family physician 12.0 (54.5) 6.0 (35.3) Radio and TV 8.0 (36.4) 5.0 (29.4) Advice class 1.0 (4.5) 0 Cause of infertility Female 24.0 (50.0) 27.0 (44.4) 0.616 \\u00a7 Male 10.0 (24.0) 13.0 (18.5) Female and male 16.0 (18.5) 10.0 (29.6) Unknown 4.0 (7.4) 4.0 (7.4) All numbers are reported in terms of numbers (percentages) except those given with \\u2020 which represents the mean (standard deviation). *Independent t -test, \\u01c2 linear-by-linear association, \\u00a5 Pearson \\u03c7 2 Fisher\\u2019s exact test Based on ANCOVA adjusted for baseline values, the mental health total score was significantly lower in the intervention women than the control women 1 month after the intervention (adjusted mean difference [aMD]: \\u22129.1; 95% confidence interval [CI]: \\u221211.9\\u2013\\u22126.3; P < 0.001). Furthermore, the total score of mental health was significantly lower in the intervention men than the control men 1 month after the intervention (aMD: \\u22127.2; 95% CI: \\u221210.1\\u2013\\u22124.2; P < 0.001) [ Table 2 ]. Table 2 Comparison of total mean score of mental health and its subscales in infertile couples Variable Mean (SD) Adjusted mean difference (95% CI) P Counseling ( n =27) Control ( n =27) Overall mental healtd score Wife Before the counseling 29.4 (8.2) 32.2 (4.7) \\u22122.8 (\\u22126.5-0.8) 0.128 \\u2020 A month after the end of the counseling 21.9 (8.5) 33.5 (7.0) \\u22129.1 (\\u221211.9-\\u22126.3) <0.001 \\u00a5 Husband Before the counseling 27.8 (8.2) 27.1 (5.4) 0.6 (\\u22123.1-4.4) 0.743 \\u2020 A month after the end of the counseling 22.0 (6.3) 28.9 (7.2) \\u22127.2 (\\u221210.1-\\u22124.2) <0.001 \\u00a5 Physical scale Wife Before the counseling 9.0 (3.8) 11.5 (2.9) \\u22122.5 (\\u22124.4-\\u22120.7) 0.007 \\u2020 A month after the end of the counseling 7.3 (3.7) 12.0 (3.3) \\u22122.4 (\\u22123.7-\\u22121.3) <0.001 \\u00a5 Husband Before the counseling 8.5 (3.2) 8.4 (2.2) 0.07 (\\u22121.5-1.6) 0.923 \\u2020 A month after the end of the counseling 7.8 (2.7) 7.9 (2.5) \\u22122 (\\u22123.3-\\u22120.8) 0.003 \\u00a5 Anxiety and sleep disorders Wife Before the counseling 7.9 (4.0) 7.7 (2.1) 0.1 (\\u22121.6-1.9) 0.866 \\u2020 A month after the end of the counseling* 7.0 (3-8)* 9.0 (7-10)* - 0.013 \\u01c2 Husband Before the counseling 7.0 (4-9)* 7.0 (6-7)* - 0.803 \\u01c2 A month after the end of the counseling 7.0 (3-8)* 7.0 (7-9)* - 0.013 \\u01c2 Social function Wife Before the counseling 11.0 (2.9) 11.5 (2.2) \\u22120.6 (\\u22122.0-0.8) 0.397 \\u2020 A month after the end of the counseling 7.8 (2.8) 10.1 (2.4) \\u22122.3 (\\u22123.8-\\u22120.9) <0.001 \\u01c2 Husband Before the counseling 12.0 (9-13)* 11.0 (9-13)* - 0.570 \\u01c2 A month after the end of the counseling 7.0 (7-10)* 10.0 (7-13)* - <0.001 \\u01c2 Depression Wife Before the counseling 1 (0-3)* 1 (0-2)* - 0.812 \\u01c2 A month after the end of the counseling 0 (0-1)* 3 (0-4)* - 0.007 \\u01c2 Husband Before the counseling 0 (0-3)* 0 (0-2)* - 0.871 \\u01c2 A month after the end of the counseling 0 (0-0)* 0 (0-2)* - 0.054 \\u01c2 *Median (percentile 25-75), \\u00a5 ANCOVA; \\u00c7 Mann-Whitney U-Test; \\u01c2 Independent t -test. SD=Standard deviation, CI=Confidence interval Based on ANCOVA adjusted for baseline values, the FertiQoL total score was significantly higher in the intervention women than the control women 1 month after the intervention (aMD: 15.4; 95% CI: 10.2\\u201320.5; P < 0.001). Furthermore, this score was significantly higher in the intervention men than the control men 1 month after the intervention (aMD: 14.0; 95% CI: 10.2\\u201317.7; P < 0.001) [ Table 3 ]. The results of mental health and quality of life subdomains are presented in Tables 2 and 3 , respectively. Table 3 Comparison of the mean total fertility quality of life score and its subscales in infertile couples Variable Mean (SD) Mean difference (95% CI) P Counseling ( n =27) Control ( n =27) Overall quality of life of fertility Wife Before the counseling 75.3 (18.9) 61.4 (11.4) 14 (5.4-22.4) 0.002 \\u00a5 A month after the end of the counseling 85.1 (15.2) 58.1 (15.8) 15.4 (10.2-20.5) >0.001 \\u2020 Husband Before the counseling 79.7 (17.0) 74.4 (15.5) 5.2 (-3.7-14.1) 0.243 \\u00a5 A month after the end of the counseling 86.8 (12.4) 68.5 (16.5) 14.0 (10.2-17.7) >0.001 \\u2020 Emotional part Wife Before the counseling 12.9 (4.0) 10.4 (2.9) 0.9 (0.5-4.3) 0.014 \\u00a5 A month after the end of the counseling 14.9 (3.0) 9.1 (3.3) 4.0 (2.9-5.1) >0.001 \\u2020 Husband Before the counseling 14.0 (3.5) 14.0 (3.0) 0.03 (-1.8-1.8) 0.967 \\u00a5 A month after the end of the counseling 15.8 (2.4) 12.2 (3.9) 3.4 (2.3-4.5) >0.001 \\u2020 Mental/physical part Wife Before the counseling 12.4 (3.6) 9.8 (2.8) 2.6 (0.9-4.4) 0.004 \\u00a5 A month after the end of the counseling 14.5 (3.1) 9.5 (3.2) 2.9 (1.8-4.0) >0.001 \\u2020 Husband Before the counseling 14.4 (3.1) 13.4 (3.1) 1 (-0.7-2.7) 0.230 \\u00a5 A month after the end of the counseling 15.5 (2.3) 12.7 (3.1) 2.0 (1.2-2.9) >0.001 \\u2020 Interpersonal relations Wife Before the counseling 11 (8-15)* 8 (7-11)* - 0.007 \\u01c2 A month after the end of the counseling 14.5 (3.1) 9.5 (3.8) 3 (1.7-4.3) >0.001 \\u2020 Husband Before the counseling 13 (12-18)* 12 (11-16)* - 0.374 \\u01c2 A month after the end of the counseling 15.4 (2.8) 12.9 (3.9) 2 (1.1-3) >0.001 \\u2020 Social relations Wife Before the counseling 14 (3.8) 11.3 (3.2) 2.7 (0.8-4.6) 0.007 \\u00a5 A month after the end of the counseling 15.5 (3.3) 10.2 (3.8) 3.3 (1.9-4.8) >0.001 \\u2020 Husband Before the counseling 14.3 (3.6) 13.6 (3.5) 0.7 (-1.2-2.6) 0.469 \\u00a5 A month after the end of the counseling 15.5 (2.8) 12.8 (3.9) 2.1 (1.2-3.0) >0.001 \\u2020 Section for the treatment of infertility Wife - Before the counseling 23 (20-30 )* 20 (18-22)* 0.023 \\u01c2 A month after the end of the counseling 25.8 (4.7) 19.7 (4.9) 3.4 (1.8-5.5) >0.001 \\u2020 Husband Before the counseling 22 (20-30)* 20 (20-23)* - 0.025 \\u01c2 A month after the end of the counseling 24 (20-30)* 20 (16-20)* - >0.001 \\u01c2 *Median (percentile 25-75), \\u00a5 Independent t -test, \\u00c7 Mann-Whitney U-Test, \\u2020 ANCOVA. SD=Standard deviation, CI=Confidence interval\",\n",
            "        \"discussion\": \"Discussion The results of this study indicate that counseling has been helpful on improving mental health and FertiQoL of infertile couples. These findings are consistent with other studies. For example, a pilot study (2011) in California on a couple with 5 years of infertility and failed in vitro fertilization (IVF) used ACT to treat infertility stress. They received a 1-year follow-up during two failed IVFs and for 1 year after it. The reported anxiety and depression were higher in the female spouse than her husband before the treatment; however, a significant reduction was observed in the general infertility stress, social infertility stress, sexual infertility stress, psychological distress, and depression. In addition, the level of stress after the second failed IVF was lower. The male participant reported a significant reduction in sexual infertility anxiety. This study showed the effectiveness of ACT in reducing the stress in infertile couples.[ 15 ] Consistent with the current study, the above study showed the effectiveness of the ACT approach in infertile couple. The only differences between these studies were that the sample size included only one couple, and the lengths of intervention and follow-up were longer. A clinical study in Tabriz, on 92 infertile women between 2012 and 2013, the intervention group received 2-h weekly sessions of infertility therapy based on the guidelines for counseling in infertility for 4 weeks, and the control group received the routine clinical care. Results showed an improvement in the adjustment of the counseling group. Researchers concluded the effectiveness of counseling as a complementary therapy in enhancing the adjustment of infertile women. It was recommended to investigate both spouses to achieve more complete results.[ 25 ] There are differences between the current study and the above study in the counseling content and participants. This study also confirmed the positive effect of group counseling on the condition of infertile women. In addition, a clinical trial was conducted in Iran on 30 infertile women to investigate the effect of ACT counseling on infertility stress and adoption. These participants were divided into two groups of intervention and control groups. The first group received eight ACT-based sessions, and the second group did not receive any intervention. The results showed that the ACT-based counseling reduced infertility stress and improved attitude toward adoption.[ 26 ] This study similar to the current study used the same intervention approach and both showed significant positive results. The only difference was that the former study did not include the spouses of the participants and infertile men. Sadri et al . investigated the effect of affective couple therapy on the quality of relationship, marital conflict, and simplicity of infertile couples in the two groups, each containing 18 participants. In their study, the intervention group received 10 counseling sessions, whereas the control group did not receive any intervention. Data analysis indicated the improvement of couple relationship, reduction of marital conflict, and enhancement of simplicity among infertile couples.[ 27 ] Similar to the current study, they applied group counseling to infertile couples and showed a significant improvement. The only difference was the employed approach. However, this counseling was focused on infertile women and not much on infertile men or infertile couples. Reports indicated positive effects of psychological intervention on the rate of fertility and psychological consequences of infertility, such as anxiety, depression, stress, and marital relationship.[ 28 ] In a meta-analytical study (2014), 1821 patients with mental disorders and physical problems have been examined. Results showed that ACT outperformed the placebo and standard treatments in controlling the posttreatment conditions and follow-up evaluation of primary objectives; in addition, better results were obtained with respect to the secondary objectives, i.e., satisfaction or quality of life. ACT can be used as an effective psychological intervention to treat anxiety, depression, addiction, and physical problems. However, better identification of ACT requires more concentrated studies into its effect on the quality of life.[ 20 ] Limitations and strengths Among the limitations of the study was the lack of long-run follow-up to investigate the effect of counseling on the fertility success rate. Another limitation is a small sample size of this study. Among the strengths of the study are the participation of men and adherence to all clinical trial principles, including the random and blinded allocation process. Suggestions and findings implication Although more relevant studies are required, it is recommended to hold individual counseling sessions in future studies. Furthermore, conducting of studies with large sample size is suggested. Regarding the positive effect of the ACT intervention on the mental health and infertile couple's quality of life, it can be used in the infertility clinics. It is also recommended that the therapist takes the ACT courses.\",\n",
            "        \"conclusion\": \"Conclusion Results of the current study suggested that the ACT counseling can improve the mental health and quality of life\\u2019 of infertile couples. As a result, in addition to routine care, the infertile couples can be provided with suitable educational and counseling packages. Financial support and sponsorship This study was funded by the Tabriz University of Medical Sciences. Conflicts of interest There are no conflicts of interest.\"\n",
            "    },\n",
            "    {\n",
            "        \"pmcid\": \"PMC7661891\",\n",
            "        \"abstract\": null,\n",
            "        \"intro\": \"Introduction Endothelial progenitor cells (EPCs), also called angioblasts, originate from the blood island of the mesoderm yolk sac and are the precursors of endothelial cells ( 1 ). EPCs not only take part in the embryonic stage of angiogenesis but are also involved in angiogenesis after birth. When the body suffers from ischemia, trauma, and stress reactions, or EPCs are induced by cytokines or drugs, EPCs in the bone marrow can be stimulated and mobilized into the systemic circulation. These EPCs gradually migrate to the site of ischemia or injury to differentiate into mature endothelial cells and participate in vasculogenesis and angiogenesis in the damaged region ( 2 ). Using EPCs as seed cells have broad clinical applications in ischemic diseases, including wound healing, ischemic cerebrovascular disease, and other fields. The notch signaling pathway plays an important role not only in vascular development but also in the pathological angiogenesis of tumors and the repair processes of tissue injury ( 3 , 4 ). There are Notch signaling pathways and molecular structures on the surface of EPCs ( 5 ). The biological behavior of EPCs, including proliferation and migration, is the foundation of the promotion of neovascularization. However, the current mechanism of regulating EPC biological behavior, especially in the notch-signaling pathway, is still not understood. This study set out to over-express hNotch1.ICN in the EPCs, directly activate the notch-signaling pathways and examine the notch pathway\\u2019s influence on the EPCs biological behavior. We present the following article in accordance with the MDAR reporting checklist (available at http://dx.doi.org/10.21037/atm-20-6362 ).\",\n",
            "        \"method\": \"Methods The ethics committee approved the study of the Affiliated Tumor Hospital of Guangxi Medical University, Institute of Cancer Prevention and Treatment of Guangxi Zhuang Autonomous Region. Materials and reagents PSB40 (virus control, TU =7.14\\u00d710 8 ), PSB1661 (virus control, TU =1.02\\u00d710 8 ), primer, and Polybrene were obtained from Sunbio (Shanghai, China). Fetal calf serum and Dulbecco\\u2019s modified Eagle\\u2019s medium-high glucose (DMEM-HG) were acquired from Gibco (Grand Island, New York, USA). Trypsin and Trizol were obtained from Invitrogen (Carlsbad, California, USA), while the apoptosis assay kits (FITC/PI) were bought from the Bi Yun Tian Company (Shanghai, China). Antibodies directed against Hes 1 and Hey 1 were acquired from Abcam (Cambridge, MA, USA). Prestained Protein Ladder was obtained from Thermo (Waltham, MA, USA), and TAKARA sold the SYBR Master Mixture (Japan). Transwell was from Millipore (MA, USA). Cell Counting Kit-8 was obtained from the BOSTER Company (Wuhan, China). Cell culture and transfection Canine EPCs were acquired from Shanghai Jing Kang bioengineering company. A human embryonic kidney cell line (293T cells) was acquired from the Shanghai Cell Bank of the Chinese Academy of Sciences. First, the 293T cells were cultured in the DMEM-HG complete medium with 10% FBS, 100 U/mL penicillin, and 100 \\u00b5g/mL streptomycin in a humidified atmosphere holding 5% CO 2 at 37 \\u00b0C. The cells were digested and passaged by 1:3 when they reached >90% confluence. The 293T cells were then transfected using a standard lentivirus method (Invitrogen). hNotch1.ICN gene overexpressed with lentivirus The hNotch1.ICN gene was overexpressed with lentivirus. Gene overexpression technology was used to understand the effects of hNotch1.ICN on the biological behavior of transfected EPCs in vitro. Lentivirus packaging of plasmid pCMV-dR8.9 and pCMV-VSV-G were acquired from Addgene. EPCs were transfected with lentivirus according to the manufacturer\\u2019s protocol (Sunbio Medical Biotechnology Co., Ltd., Shanghai, China). Cell counting kit-8 assay EPCs in good condition were respectively digested, counted, and placed into 96 cell cultures, with 2,000 cells per hole. After the cells had adhered to the culture dish (overnight, about 16 hours), according to the MOI =10, the purpose and control viruses were respectively added. Simultaneously, 8 \\u00b5g/mL Polybrene was added to cause an enhanced infection. Then, 10 \\u00b5L CCK-8 were added to each well 0 hours after infection. Then, the cells were placed in a 37 \\u00b0C incubator for 3 hours; then, absorbances at 450 and 620 nm were measured. The OD values at 450 nm were measured values, while the OD values at 620 nm were the values of blanks. Therefore, the final value = measured value-blank value. At specific time intervals following virus infection (24, 48, and 72 hours), the cell supernatant was replaced. Simultaneously, 10 \\u00b5L CCK-8 were added into each well. The cells were put in a 37 \\u00b0C incubator for 3 hours, and then their absorbances at 450 and 620 nm were measured. The final values of the empty cells and the two virus-infected cells were analyzed at four points in time. Cell cycle analysis apoptosis assay Logarithmic phase EPCs were digested, counted, and placed in three 6-cm dishes, with 6.5\\u00d710 5 cells per dish. After being adherent cells (overnight, about 16 hours), according to the MOI =10, the purpose and control viruses were respectively added. Simultaneously, 8 \\u00b5g/mL Polybrene was added to cause an enhanced infection. Twenty-four hours after the virus infection, the cell supernatant was replaced. The culture supernatant was collected 48 hours following virus infection, and the cells were respectively digested with pancreatic enzyme and then collected. The cells were centrifuged for 5 minutes at 1,000 rpm to discard the supernatant. Then, the cells were rinsed with PBS and centrifuged for another 5 minutes at 1,000 rpm. The cells were fixed under 4 \\u00b0C for 1.5 hours, with 70% precooling ethanol. The fixed sample was taken out and centrifuged for 5 minutes at 1,000 rpm to abandon any remaining alcohol. The cells were washed with PBS and then resuspended in 400 \\u00b5L precooling RNase PI solution. Afterward, the cell suspension was incubated for 15 minutes at room temperature. The cells were resuspended in 300 \\u00b5L PBS, and the cell cycle was at once analyzed using a flow cytometer. After that, the cells were resuspended in 195 \\u00b5L AnnexinV-FITC and incubated with 10 \\u00b5L PI for 10 minutes at room temperature. Then, the cells were centrifuged to abandon the supernatant and were resuspended in 190 \\u00b5L AnnexinV-FITC and then incubated with 10 \\u00b5L PI for 10 minutes at room temperature. Finally, the cells were respectively resuspended in 100 \\u00b5L PBS and at once analyzed for cell apoptosis using a flow cytometer. In vitro trans-well migration assay In vitro migration of EPCs was performed using trans-wells. Monolayer ECs were firstly loaded onto trans-well inserts (5\\u00d710 5 cells/well). The ECs were cultured and identified in our prior study. When the ECs had been wholly fused, they were then washed three times with PBS. The next step involved the addition of 10 ng/mL TNF-\\u03b1and 10 ng/mL IL-to activates the monolayer ECs. Three independent experiments were performed in duplicate. Blank, PSB40, and PSB166 cells were respectively resuspended and loaded onto trans-well inserts (2\\u00d710 5 /mL in serum-free medium). The lower chambers had a 20% volume fraction serum culture medium. The cells were incubated for 12 hours at 37 \\u00b0C and 5%. Then, any cells that migrated to the lower chambers were collected. Five random vision cells with green fluorescence in each trans-well were counted. Adhesion of an activating endothelial cells assay ECs were put into 24 cell cultures (1\\u00d710 6 per well). When the ECs had fused entirely, they were then washed three times with PBS. Ten ng/mL TNF-\\u03b1and 10 ng/mL IL-1\\u03b2 were used for activating the monolayer ECs. Cell holes were respectively set for Group A, Group B, and Group C. Each group was evaluated in triplicate. Blank, PSB40, and PSB166 cells were uniformly covered onto a monolayer of ECs and incubated for 20 minutes at 37 \\u00b0C and 5%. The suspension cells were then washed using PBS, and the numbers of cells were counted in five random fields under green fluorescence. Angiogenesis A tube formation assay was performed to investigate the effect of the hNotch1.ICN gene on the angiogenic activity of EPCs in vitro. Forty-eight-well culture dishes were coated with 150 \\u00b5L of matrigel matrix and incubated for 25 minutes at 37 \\u00b0C. These cells were divided into blank, PSB40, and PSB166 groups. The cells were starved for 10 hours and were then seeded onto the solidified gels at a density of 10 6 cells/well in 50 \\u00b5L of culture medium. After incubation for 15 hours, the total tube-like structures were photographed using phase-contrast microscopy (100\\u00d7), and five randomly selected microscopic fields per photograph were quantified using Image J software. RNA\\u2019s extraction and quantitative RT-PCR The total RNA was extracted from the three groups of cells using Trizol Reagent. First-strand cDNA was synthesized with the PrimeScript Reverse Transcriptase Reagent Kit (Takara Biotechnology, Japan) with oligo (dT) and random primer. The gene expression of the EPCs was quantified using the KAPA Kit (Kapa Biosystems, USA), and 50 ng of cDNA was amplified in a 10 \\u00b5L reaction using the Applied Biosystems 7500 Real-Time PCR (RT-PCR) system with SYBR Green dye (Applied Biosystems). The primers were acquired from Nanjing ChemGen Biotech (Nanjing, China). The amplification primers used were forward primer AGCACAGAAAGTCATCAA and reverse primer TTCTTCAGAGCATCCAAA for the Hes 1 gene. For the Hey 1 gene, the amplification primers were forward primer GGATGAGAATGGAAACTTGA and reverse primer ATGATGCCTCTTCGTCTT. QRT-PCR was performed with the following cycling conditions: 95 \\u00b0C for 15 seconds, followed by 45 cycles of 95 \\u00b0C for 0.05 seconds and 60 \\u00b0C for 30 seconds. Quadruplicate cycle threshold (CT) values were analyzed with SDS software (Applied Biosystems, USA) using the comparative CT method. \\u03b2-actin was used as an internal control to confirm the success of the reaction. The procedure was repeated three times, and each measurement was set for three repeats. Western blot analysis Total protein lysates from the blank, PSB40, and PSB166 cells were subjected to standard Western blot analysis. The total protein was loaded onto a 4\\u201316% gradient SDS-PAGE gel and transferred onto a PVDF membrane. Anti-Hey 1 polyclonal antibody (1:500 dilution, Novus) and Anti-Hes1 antibody (1:1,000 dilution, Abcam) were used to detect Hey 1 and Hes 1 protein, respectively. \\u03b2-Actin (Santa Cruz) was used as an internal control. Statistical analysis The experimental data were obtained from experiments replicated three times. The Statistical Package for the Social Sciences (SPSS), version 17.0, the software was used for all statistical analyses. Experimental data are shown as mean \\u00b1 standard deviations (SD) and have been compared with student\\u2019s t -test or one-way analysis of variance (ANOVA). A P value <0.05 was determined to show statistically significant differences.\",\n",
            "        \"subjects\": null,\n",
            "        \"results\": \"Results No significant differences in growth activity among the three groups The survival vitality of EPCs in the three groups had no noticeable difference (P>0.05), which showed that an overexpressed hNotch1.ICN gene had no significant effect on the EPCs growth ( Figure 1 ). Figure 1 There are no apparent differences in the survival vitality of EPCs in the three groups (P>0.05). EPC, endothelial progenitor cell. Overexpressed hNotch1 ICN gene strengthened the EPCs antiapoptotic ability Cell apoptosis was detected in infected EPCs after 48 hours using a flow cytometer. The early apoptosis rate of the blank cells was 10.65%\\u00b10.12%, while the late apoptosis rate was 17.57%\\u00b10.09%. The early apoptosis rate of PSB40 cells was 8.29%\\u00b10.21%, while the late apoptosis rate was 13.85%\\u00b10.08%. Finally, for PSB1661 cells, the early apoptosis rate was 3.84%\\u00b10.11%, and the late apoptosis rate was 9.65%\\u00b10.10%. Both the early and the late apoptosis rates of PSB1661 were lower than the other two groups (P<0.05, Figure 2 ). Figure 2 Overexpression of hNotch1.ICN gene can enhance the anti-apoptotic ability of EPCs. (A) Normal EPCs; (B) EPCs transfected with an empty vector; (C) genetically modified EPCs; (D) early and late apoptosis rates were lower for PSB1661 than in the other two groups (*, P<0.05). EPC, endothelial progenitor cell. An overexpressed hNotch1.ICN gene increased numbers of EPCs in the G2 phase The cell cycle was detected in infected EPCs after 48 hours using flow cytometry. The rates of G1, G2, and S phase in the blank group were 57.63%\\u00b13.25%, 8.60%\\u00b12.12%, and 33.77%\\u00b10.12%, respectively. In the PSB40 group, the rates of G1, G2, and S phase cells were 65.62%\\u00b14.11%, 7.47%\\u00b10.19%, and 27.00%\\u00b12.58%, respectively. Finally, the G1, G2, and S phase rates in the PSB1661 group were 54.49%\\u00b13.33%, 23.66%\\u00b12.24%, and 31.86%\\u00b11.36%, respectively. The numbers of cells in the G2 phase of the PSB1661 group was significantly higher than the other two groups (P<0.05, Figure 3 ). Figure 3 Overexpression of hNotch1.ICN gene increased the number of EPCs in G2 phase. (A) EPCs; (B) EPCs infected with a control virus; (C) infected EPCs with a target virus; (D) numbers of cells in the G2 phase of the PSB1661 group was significantly higher than the other two groups (*, P<0.05). EPC, endothelial progenitor cell. An overexpressed hNotch1 ICN gene promoted EPCs to migrate across the ECs A migrating cross-vascular endothelial ability reflects the ability of EPCs to form blood vessels. Our experimental results showed that overexpressed hNotch1.ICN gene EPCs of PSB1661 were significantly higher than in the PSB40 and blank groups (88.1\\u00b111.03 vs. 31.0\\u00b16.32 and 35.2\\u00b13.28, respectively *, P<0.05, Figure 4A ). Figure 4 Overexpression of hNotch1.ICN gene can promote EPCs to pass through vascular endothelium, to adhere to activated endothelial cells, and to form lumenoid structure. (A) A comparison of migration across the endothelial cell among three groups ( x \\u00af \\u00b1 s , n=3); PSB1661 was significantly higher than the PSB40 or the blank group (*, P<0.05). (B) Comparison of adhesion with activation of endothelial cells among the three groups ( x \\u00af \\u00b1 s , n=3), the capacity of PSB1661 cells to adhere to activating endothelial cells was significantly stronger than PSB40, blank cells (*, P<0.05). (C) A comparison of the forming lumen structure among the three groups ( x \\u00af \\u00b1 s , n=3). The percentage of PSB1661 cells on matrix-forming blood vessels was significantly higher than the other two groups (*, P<0.05). EPC, endothelial progenitor cell. An overexpressed hNotch1.ICN gene promoted the EPCs to adhere to activating endothelial cells The adherence of EPCs to activating endothelial cells is the crucial step in angiogenesis; it directly affects the time duration of EPCs to stay in the injured or ischemic area and its ability to migrate endothelially. Promoting the EPCs to adhere to activating endothelial cells can significantly enhance the ability of EPCs to form new blood vessels. These experimental results showed that the capacity of PSB1661 cells to adhere to activating endothelial cells was significantly stronger than PSB40 and blank cells (361\\u00b132.9 vs. 51.1\\u00b117.6 and 57.6\\u00b121.5, respectively *, P<0.05, Figure 4B ). Overexpressed hNotch1.ICN gene in EPCs induces angiogenesis in vitro An in vitro tube cavity structure forming ability is an essential index of the EPCs that participate in angiogenesis. The results showed that overexpression of hNotch1.ICN EPCs onto matrix-forming blood vessels were significantly higher than the other two groups and could effectively improve the ability of EPCs to form a sample tube structure (2.68\\u00b10.24 vs. 1\\u00b10.21 and 1.3\\u00b10.19, respectively *, P<0.05, Figure 4C ). An overexpressed hNotch1.ICN gene in EPCs enhances the Hes 1 and Hey 1 expression of Notch downstream signaling molecules Overexpression of the hNotch1.ICN gene in PSB1661 cells, as well as Hes 1 and Hey 1 gene expression of the Notch signaling pathways\\u2019 downstream signaling molecules, were higher than those values for either the PSB40 group or the blank group (*, P<0.05). The difference was statistically significant ( Figure 5 ). Protein expression was consistent with the results of the expression of genes ( Figures 6,7 ). Figure 5 Notch downstream signaling molecules Hes 1 and Hey 1 gene expression. Hes 1 and Hey 1 gene expression rates in the PSB1661 group were higher than those found in the PSB40 group and the blank group (*, P<0.05). Figure 6 Notch downstream signaling introduces molecule Hes 1 protein expression. The relative gray value of Hes 1 in the PSB1661 group was higher than the other two groups (*, P<0.05). Figure 7 Notch downstream signaling introduces molecule Hey 1 protein expression. The relative gray value of Hey 1 in the PSB1661 group was higher than the other two groups (*, P<0.05).\",\n",
            "        \"discussion\": \"Discussion EPCs are precursor cells of endothelial cells that are mainly derived from bone marrow ( 6 ), but a few may also be found in umbilical cord blood, peripheral blood, and myocardial tissue ( 7 - 9 ). EPCs play an essential role in angiogenesis and endothelial repair ( 10 ). In many ischemic animal research models, EPCs from umbilical cord blood, peripheral blood, or bone marrow can cure or even alleviate ischemia ( 11 , 12 ). The angiogenesis process includes several steps that involve EPCs. Silent bone marrow EPCs are activated, mobilized into the peripheral blood circulation, and finally remain in the area of tissue ischemia to differentiate into endothelial cells in situ to form new blood vessels ( 13 ). Because they are directly involved in angiogenesis, EPCs can also have a chemotaxis effect on vascular defect points and promote the formation of new blood vessels. Many studies have found that EPCs can promote angiogenesis. For example, in tissue engineering, EPCs are used as vascularized seeds to promote bone vascularization and improve osteogenesis ( 5 ). For repairing bone defects, EPCs combined with the tissue-engineered bone are better than simplex tissue-engineered bone ( 14 ). The functions of EPCs are closely related to their biological characteristics. In general, EPCs are resting and can also support an immature state. However, when a stimulus induces EPCs or some signal molecule that can adjust for angiogenesis, EPCs will migrate, proliferate, and differentiate into mature endothelial cells. These mature endothelial cells promote new blood vessel formation to repair the damaged vasculature ( 15 ). Many signaling pathways regulate EPCs that take part in angiogenesis. Many studies have shown that notch signaling plays a crucial role in regulating cell differentiation and angiogenesis. Recent studies have shown the characteristics of vascular endothelial cell differentiation towards arteriovenous are mainly regulated by notch signaling molecules ( 16 ), and the notch signaling pathway has a close relationship with smooth muscle cell differentiation and tumor angiogenesis. The notch signaling pathway is made up of notch ligands, receptors, and effector molecules into cells ( 17 ). When notch receptors combine with ligands, the notch signaling pathway is activated, the Notch intracellular domain (Notch.ICN) is released, and then Notch.ICN transfers into the cell nucleus and activates the downstream effector molecules, which inhibit the differentiation and transcription of related genes. Notch.ICN codes Notch l proteins across the membrane area and in front of the PEST structure domain. The fragments of ICN have a complete ARM and ANK structure domains. These domains activated Notch 1 directly through the notch signaling pathway in the absence of ligands. The notch signaling pathway plays an important signal transduction role in the interaction between cells and stem cell self-renewal, proliferation, adhesion, apoptosis, and migration ( 18 , 19 ). To study how Notch signaling molecules influence the biological characteristics of EPCs, we adopted a gene transfection method. Genetic modification enhances the function of EPCs. EPCs can be used as target cells of gene therapy, and they also produce therapeutic proteins at the site of injury. This result showed that the overexpression of the hNotch1.ICN gene had no significant impact on the activity or proliferation ability of EPCs. There was no apparent difference among the blank group, the PSB40 group, and PSB1661 cells. The notch signaling pathway may be crucial in supporting the undifferentiated state of progenitor cells. Duncan observed that in the bone marrow and peripheral blood, notch activity was down-regulated when hematopoietic progenitor cells differentiated ( 20 ). To verify this conclusion, the researchers used interleukin 3 and steel factor cultured KLS GFP + cells. Three days later, KLS GFP + cells were differentiated into Lin+ cells. The result of 13% Lin+ cells that expressed GFP, which they addressed the finding of 12% obtained before Lin\\u2019s - cell differentiation, and the GFP fluorescence density in Lin\\u2013 cells was lower. These results showed that an activated notch signal decreased the differentiation, and the fluorescent signal level also decreased. YU also found the Hes 1 gene, the primary target molecule of Notch signaling pathways downstream, was highly expressed in the CD34+, CD38-, and Lin- cells but was lowly expressed in more mature CD34+ and CD38+ cells. In this study, overexpression of the hNotch1.ICN gene caused the Notch signaling pathway to be continuously activated ( 21 ). We found Hes 1 and Hey 1, which are Notch signaling downstream effector molecules, increased their expression. Hes 1 and Hey 1 inhibited specific target gene transcription. Additionally, overexpression of the hNotch1.ICN gene increased numbers of EPCs that underwent cell cycle arrest in the G2 phase, delayed the cell cycle\\u2019s progress, reduced the differentiation velocity of EPCs, and inhibited the differentiation of stem cells. In this study, we used the Annexin V/PI method to detect a blank group, PSB40 group, and PSB1661 group cells. We found the early and the late apoptosis rates of PSB1661 were 3.84%\\u00b10.11% and 9.65%\\u00b10.10%, respectively, which was lower than the other two groups. The results showed that overexpression of the hNotch1.ICN gene enhanced bone marrow EPCs\\u2019 ability to resist apoptosis, inhibited hematopoietic progenitor cell differentiation, and contributed a long-term recovery effect. EPCs can gather, adhere, stay in the vascular injury area, proliferate, differentiate, and secrete cytokines, and finally repair damaged endothelial cells or blood vessels. Adhesion and migration are essential links in EPC homing. In this study, by simulation of the existence, movement, and the interaction of cells with the endothelial microenvironment, we described EPCs involvement in essential steps of angiogenesis. The experimental results showed that overexpression of the hNotch1.ICN gene can significantly enhance the ability of EPCs to adhere to activated endothelium, migrate across the endothelium, and improve EPC tube structure formation in vitro. This work indicated that overexpression of the hNotch1.ICN gene can significantly enhance the ability of canine bone marrow EPCs to resist apoptosis, inhibit hematopoietic progenitor cell differentiation, and help EPCs to exert a long-term recovery effect. Overexpression of the hNotch1.ICN gene can enhance the angiogenesis ability of canine EPCs and supply a new strategy for EPCs as seed cells to treat many types of ischemic diseases.\",\n",
            "        \"conclusion\": null\n",
            "    },\n",
            "    {\n",
            "        \"pmcid\": \"PMC8288503\",\n",
            "        \"abstract\": \"This work shows that Engrailed 1 (EN1) is a key mediator of TGF\\u03b2-induced myofibroblast differentiation and fibrotic tissue remodeling. Mechanistically, EN1 induces a profibrotic gene expression profile by modulating the activity of SP transcription factors to coordinate the microtubule\\u2013stress fiber cytoskeletal rearrangements required for fibroblast activation in a ROCK-dependent manner. Transforming growth factor-\\u03b2 (TGF\\u03b2) is a key mediator of fibroblast activation in fibrotic diseases, including systemic sclerosis. Here we show that Engrailed 1 (EN1) is reexpressed in multiple fibroblast subpopulations in the skin of SSc patients. We characterize EN1 as a molecular amplifier of TGF\\u03b2 signaling in myofibroblast differentiation: TGF\\u03b2 induces EN1 expression in a SMAD3-dependent manner, and in turn, EN1 mediates the profibrotic effects of TGF\\u03b2. RNA sequencing demonstrates that EN1 induces a profibrotic gene expression profile functionally related to cytoskeleton organization and ROCK activation. EN1 regulates gene expression by modulating the activity of SP1 and other SP transcription factors, as confirmed by ChIP-seq experiments for EN1 and SP1. Functional experiments confirm the coordinating role of EN1 on ROCK activity and the reorganization of cytoskeleton during myofibroblast differentiation, in both standard fibroblast culture systems and in vitro skin models. Consistently, mice with fibroblast-specific knockout of\",\n",
            "        \"intro\": \"Introduction Fibrotic diseases are a major cause of morbidity and mortality, accounting for \\u226440% of the deaths in Western societies ( Wynn, 2008 ). Systemic sclerosis (SSc) is the prototypical systemic fibrotic disease, characterized by excessive deposition of collagen in skin and multiple other organs, leading to their dysfunction ( Gabrielli et al., 2009 ). The major effector cells in fibrosis are activated fibroblasts, so-called myofibroblasts, which express contractile proteins such as \\u03b1-smooth muscle actin (\\u03b1SMA) and produce excess collagen ( Hinz et al., 2012 ; Wynn and Ramalingam, 2012 ). Resting fibroblasts are initially activated by cytokines released from infiltrating leukocytes, both in physiological responses such as wound healing and in fibrosis ( Distler et al., 2019 ). Of these cytokines, TGF\\u03b2 plays a key role, driving persistent fibroblast activation not only during disease initiation, but also at later stages of fibrosis, when the inflammation has subsided and fibroblasts remain endogenously activated ( Distler et al., 2019 ). TGF\\u03b2 activates fibroblasts via a complex network of multiple intracellular cascades ( Lafyatis, 2014 ). These TGF\\u03b2-regulated intracellular pathways are main targets of emerging antifibrotic therapies ( Gy\\u00f6rfi et al., 2018 ). Thus, discovery and characterization of novel mediators of the profibrotic effects of TGF\\u03b2 can provide new therapeutic opportunities in SSc ( Distler et al., 2017 ). Engrailed 1 (EN1) is a homeodomain-containing transcription factor with essential, widespread roles in embryonic development of different tissues including cerebellum, midbrain, skeleton, and limbs ( Loomis et al., 1996 ; Wurst et al., 1994 ). While EN1 expression persists in some cells such as mesencephalic dopaminergic neurons during adulthood, most other cell types silence the expression of EN1 after lineage commitment and do not express EN1 under homeostatic conditions in adulthood ( Rekaik et al., 2015 ). However, a pathological environment can induce the expression of EN1 in these cells to promote phenotypical adaptation. In tissues from patients with triple-negative breast cancer, for example, EN1 expression is up-regulated and correlates with poor patient survival; moreover, EN1 expression induces proliferation and migration of quintuple-negative breast cancer cells ( Kim et al., 2018 ). Depending on the context, EN1 can regulate gene transcription either directly by binding to conserved EN1-binding sites in the promoters of target genes or indirectly by trans-activation or trans-repression by interacting with other transcription factors, with subsequent modulation of their transcriptional outcome ( Alexandre and Vincent, 2003 ). Of particular interest, En1 is transiently expressed during dermal development in murine embryos in a distinct fibroblast lineage ( Rinkevich et al., 2015 ). In mice, En1-positive fibroblasts gradually replace En1-negative fibroblasts in the developing dermis ( Jiang et al., 2018 ). The expression of En1, however, decreases rapidly before birth. Former EN1-positive cells give rise to a subpopulation of fibroblasts that has a high capacity for extracellular matrix (ECM) production and important roles in wound healing in adult murine skin ( Jiang et al., 2018 ; Rinkevich et al., 2015 ). However, the regulation of EN1 in adult skin and its role in the pathological activation of fibroblasts in tissue fibrosis have not been studied thus far. In this study, we show that TGF\\u03b2 reactivates the expression of EN1 in a SMAD-dependent manner with increased numbers of EN1-positive fibroblasts in SSc skin. Imaging mass cytometry (IMC) of paired skin biopsies from SSc patients (from ventral and dorsal skin) show that EN1 is expressed by multiple fibroblast subpopulations in SSc. EN1 is a critical mediator of the profibrotic effects of TGF\\u03b2 in adult fibroblasts: EN1 overexpression in adult human dermal fibroblasts promotes fibroblast-to-myofibroblast transition and collagen release, whereas knockdown of EN1 prevents TGF\\u03b2-induced fibroblast-to-myofibroblast transition. EN1 knockdown induces global transcriptomic changes in TGF\\u03b2-stimulated fibroblasts, preventing induction of a profibrotic gene expression profile. EN1 trans-regulates gene expression by modulating the activity of transcription factors from the C2H2 zinc finger family, in particular those from the specificity protein (SP) subfamily. Chromatin immunoprecipitation followed by sequencing (ChIP-seq) experiments for both EN1 and SP1 demonstrate no direct binding of EN1 to the promoters of EN1-DEGs, but EN1-driven modulation of SP1 binding to the promoter of fibrosis-relevant genes. Mechanistically, changes of gene expression induced indirectly by EN1 knockdown, in particular genes with predicted regulation by SP transcription factors and especially with SP1 binding, lead to stabilization of microtubules and reduced Rho-associated protein kinase (ROCK) activity, with subsequent prevention of TGF\\u03b2-induced stress fiber formation, myofibroblast transition, and fibrotic transformation of three-dimensional (3D) full-thickness skin equivalents. Consistently, fibroblast-specific En1 knockout ameliorates dermal fibrosis in three complementary inflammation-driven and inflammation-independent mouse models of SSc. We thus characterize EN1 as a mediator of TGF\\u03b2-induced fibroblast activation and fibrosis.\",\n",
            "        \"method\": \"Materials and methods Patient samples Skin biopsies from 10 SSc patients and 10 age- and sex-matched healthy volunteers were used for staining or fibroblast isolation. Skin biopsies from SSc patients were taken from the affected skin of the forearm. All patients fulfilled the American College of Rheumatology/European League Against Rheumatism 2013 criteria ( van den Hoogen et al., 2013 ). Seven patients were female, and three were male. The median age of SSc patients was 44 yr (range 19\\u201365), and median disease duration was 5 yr (range 1\\u201312). Further clinical data are provided in Table S2 . The human studies were approved by the ethics committee of the Medical Faculty of the University of Erlangen-Nuremberg. All patients and controls signed a consent form approved by the local institutional review board. Animal studies Mice carrying two conditional alleles of En1 ( En1 fl/fl ), En1 tm8.1Alj /J (stock no. #007918, The Jackson Laboratory; Sgaier et al., 2007 ), were crossbred with Col6Cre mice, B6.Cg-Tg(Col6a1-cre)1Gkl/Flmg (obtained from G. Kollias, Institute of Immunology, Biomedical Sciences Research Center \\u201cAlexander Fleming,\\u201d Vari, Greece; Armaka et al., 2008 ). For all mouse experiments, for En1 fl/fl \\u00d7 Col6Cre mice with fibroblast-specific knockout of En1 , En1 fl/fl \\u00d7 wt/wt littermates without Cre-mediated recombination were used as controls. The role of EN1 signaling in fibrosis was investigated in three different mouse models: (1) In the model of bleomycin-induced dermal fibrosis, fibrosis was induced by subcutaneous injections of bleomycin (50 \\u00b5g) every other day for 4 wk starting at age 6 wk ( Dees et al., 2015 ; Soare et al., 2016 ). Littermates injected with 0.9% NaCl served as controls. (2) In the TBRI CA skin fibrosis model, fibrosis was induced by subcutaneous injections of replication-deficient type 5 adenoviruses (6.67 \\u00d7 10 7 viral particles/administration/mouse) encoding for a constitutively active TBRI construct every second week ( Beyer et al., 2010 ). Fibrosis was evaluated after 8 wk. Mice injected with type 5 adenoviruses encoding for LacZ served as controls. (3) Tsk-1 mice are a genetic model of skin fibrosis with progressive accumulation of ECM in the hypodermal layer of the skin ( Beyer et al., 2010 ). Tsk-1 mice, B6.Cg- Fbn1 Tsk /J (stock no. #014632, The Jackson Laboratory; Siracusa et al., 1996 ) were analyzed at an age of 10 wk ( Beyer et al., 2012a ). En1 fl/fl \\u00d7 Col6Cre \\u00d7 wt/wt littermates were used as healthy controls. All animal experiments were approved by the government of Unterfranken in W\\u00fcrzburg, Germany. Histological analyses Skin sections were stained with hematoxylin and eosin or trichrome. Dermal or hypodermal thickness was analyzed in three consecutive skin sections from each mouse at four different sites/section in a blinded manner, as described ( Akhmetshina et al., 2012 ). Pictures were taken at 40- or 100-fold magnification with a Nikon Eclipse 80i microscope. Immunohistochemistry staining using \\u03b1SMA antibody (1:1,000, #A5228; Sigma-Aldrich) and goat anti-mouse HRP-conjugated secondary antibody (1:5,000, #P0447; Dako) was performed for the visualization and quantification of myofibroblast numbers in murine skin. Cell culture Human and murine fibroblasts were isolated and cultured as described previously ( Chakraborty et al., 2017 ; Palumbo-Zerr et al., 2015 ). In healthy human dermal fibroblasts, EN1 overexpression was induced by transfection of 0.5 \\u00b5g of plasmid encoding the human EN1 (Origene). Knockdown of EN1 or of SMAD3 was induced by transfection of 3 \\u00b5g of EN1 -targeting or SMAD3 -targeting siRNA using an Amaxa 4D-Nucleofector. Healthy dermal fibroblasts transfected with an equal amount of empty vector or nontargeting siRNA served as controls. In murine fibroblasts isolated from En1 fl/fl mice, Cre-mediated recombination was induced by infection with type 5 adenoviral vectors encoding for Cre recombinase at a multiplicity of infection of 80 ( Palumbo-Zerr et al., 2017 ). Type 5 adenoviral vectors encoding for LacZ served as controls. In selective experiments, cells were stimulated with recombinant TGF\\u03b2 (10 ng/ml; PeproTech) for 24 h, unless stated otherwise. In certain experiments, microtubules were stabilized by addition of paclitaxel (1 \\u00b5M); depolymerized by addition of vinblastine (1 \\u00b5M); or activity of Rho kinase 1 and 2 was inhibited by Y27632 (1 \\u00b5M). 3D full-thickness skin equivalents 3D full-thickness skin equivalents were generated with human fibroblasts transfected with 0.5 \\u00b5g of plasmid encoding for EN1 or a CMV control vector. After transfection, fibroblasts (4 \\u00d7 10 5 cells for each replicate) were suspended in neutralization solution (232.5 ml DMEM/F12, 7.5 ml FBS, 7.5 ml 3 M HEPES, and 2.5 ml chondroitin sulfate) and mixed with rat tail collagen type 1 (10 mg/ml). 500 \\u00b5l of neutralized fibroblast\\u2013containing collagen (2 parts of collagen to 1 part neutralization solution containing fibroblasts) was added in each cell-culture insert with porous membranes (8 \\u00b5m; Greiner Bio One), to create the dermal-like part. The dermal components were cultured for 1 d in DMEM supplemented with 10% FBS, 1% penicillin/streptomycin at 37\\u00b0C, 5% CO 2 , and atmospheric O 2 . The epidermal component was generated by seeding 5 \\u00d7 10 5 normal human epidermal keratinocytes for each replicate resuspended in Epilife medium with 1% human keratinocyte growth supplement (Gibco BRL) and with extra 1.44 mM CaCl 2 on the apical surface of the dermal components on the following day. 1 d later, the full-thickness skin models were cultured in Epilife medium supplemented with 1% human keratinocyte growth supplement (Gibco BRL), 1.44 mM CaCl 2 , 0.125 mM L-ascorbic acid 2-phosphate, and 10 ng/ml keratinocyte growth factor (Sigma-Aldrich) at an air-liquid interface and treated with paclitaxel (1 \\u00b5M) or Y27632 (1 \\u00b5M). RNA-seq Total RNA from EN1 -knockdown and control human fibroblasts stimulated with TGF\\u03b2 (with three biological replicates per condition) was extracted as described ( Wohlfahrt et al., 2019 ). RNA-seq was performed by Novogene (Cambridge, UK) on an Illumina NovaSeq platform using a paired-end 150-bp sequencing strategy. Read counts were normalized by trimmed mean of M values method for analysis with edgeR and limma-voom. For analysis with DESeq2, the internal normalization method listed on the package was used ( Fuchs et al., 2020 ). Differential gene expression analysis was performed using edgeR, limma-voom, and DESeq2 R packages, with the following thresholds for differential expression: Benjamini\\u2013Hochberg adjusted P value/false discovery rate <0.25 and fold-change >1.5. Genes that were identified as differentially expressed by at least two of the three methods were defined as high-confidence DEGs and used for downstream analyses. Heatmaps were generated using pheatmap package. Dot plots were generated with ggplot2 R package. Functional enrichment analysis was performed using the g:Profiler web tool with a threshold of 0.05 for adjusted P values using the GO, Reactome, KEGG, and Wiki Pathways databases ( Raudvere et al., 2019 ). Gene ratios of selected GO or Reactome terms were plotted as dot plots. GSEA was performed using GSEA software v3.0 (Broad Institute) with previously curated gene sets from the Molecular Signatures Database and a threshold of 0.25 for false discovery rate\\u2013corrected q value ( Subramanian et al., 2005 ). The RNA-seq datasets were deposited in the GEO database (accession no. GSE174598 ). In silico analysis of the promoters of DEGs Flanking regions (\\u2212800) of up- and down-regulated DEGs were retrieved from Ensembl database (GRCh38.p13). Negative control sequences of the same length were generated with random ACTG dummy sequences using Perl. Transcription factor profiling analysis against all human matrices extracted from JASPAR2018 database was performed with sequence scanning from TFBStools package v1.26.0 ( Tan and Lenhard, 2016 ) using R version 4.0.0. The top 1% of results sorted by calculated binding scores was used for further analysis. Transcription factor enrichment was calculated by the ratio of genes predicted for one transcription factor relative to the total number of up- and down-regulated genes used in the analysis. Transcription factors with >20% promotor binding were selected as significant. Functional analysis was performed using g:Profiler web tool ( Raudvere et al., 2019 ). Results of the transcription factor analysis are presented in circular packing plots generated with ggraph package v2.0.3 ( https://cran.rstudio.com/web/packages/ggraph/ ). Quantitative real-time PCR Total RNA was isolated with the NucleoSpin RNA II extraction system (Macherey-Nagel) and reverse transcribed into cDNA. Gene expression was quantified by real-time PCR using the Mx3005P Sequence Detection System (Agilent Technologies). Primers were designed with Primer3 software and are listed in Table S1 . Samples without enzyme in the reverse transcription reaction were used as negative controls. Unspecific signals caused by primer dimers were excluded by nontemplate controls and by dissociation curve analysis. \\u03b2-Actin was used to normalize for the amounts of cDNA within each sample. Differences were calculated with the threshold cycle (Ct) and the comparative Ct method for relative quantification ( Dees et al., 2020 ). Western blot analysis The soluble and insoluble fractions of \\u03b1-Tubulin were extracted as described ( Sandbo et al., 2013 ). The total protein concentration of cell lysates was determined by Bradford assay (#5000001; Bio-Rad). Proteins were separated by SDS-PAGE and transferred to a polyvinylidene difluoride membrane. The membrane was incubated with primary antibodies against EN1 (1:200; Santa Cruz Biotechnology), \\u03b2-ACTIN (1:10,000, #A5441; Sigma-Aldrich), p-SMAD3 (1:1,000, #9520; Cell Signaling), \\u03b1SMA (1:1,000, #A5228; Sigma-Aldrich), \\u03b1-Tubulin (1:1,000, #14-4502-82; Thermo Fisher Scientific), SP1 (#9389S; Cell Signaling), and HRP-conjugated secondary antibodies (1:5,000; Dako). Blots were visualized by ECL. \\u03b2-Actin was used as loading control. Western blots were quantified using ImageJ software. Coimmunoprecipitation Fibroblasts were collected in lysis buffer, as for Western blot. 10% of the amount of protein used for immunoprecipitation was used as input. Cell extracts were incubated with 30 \\u00b5l protein A/G Sepharose and 5 \\u00b5g of either SP1 or rabbit IgG antibody (#9389S; Cell Signaling; and #NI01; Millipore). Unbound proteins were removed by washing with lysis buffer. Sepharose-bound protein complexes were separated via SDS-PAGE followed by Western blotting on a polyvinylidene difluoride membrane (see Western blot analysis). ChIP PCR and sequencing ChIP assays, including purification of the enriched DNA, were performed using the ChIP-IT High Sensitivity Kit (Active Motif). In brief, 20\\u201330 \\u00b5g of sonicated chromatin extract was incubated with 1 \\u00b5g of antibodies against SMAD3, EN1, or SP1 (SMAD3, #9523S; SP1, #9389S; both Cell Signaling; EN1, NBP2-57757; Novus Biologicals) or rabbit IgG antibody (#NI01; Millipore). SMAD3 binding in EN1 promoter was determined by PCR of the enriched DNA (after immunoprecipitation with an antibody against SMAD3) using the following primer pairs: 5\\u2032-TTC\\u200bTAT\\u200bGGC\\u200bCTC\\u200bTGG\\u200bACG\\u200bTG-3\\u2032 (forward) and 5\\u2032-ACT\\u200bGTC\\u200bGCT\\u200bGAG\\u200bAGA\\u200bTTC\\u200bGT-3\\u2032 (reverse) and 5\\u2032-CTG\\u200bAGT\\u200bGTC\\u200bAGC\\u200bGCG\\u200bAGT\\u200bT-3\\u2032 (forward) and 5\\u2032-TTT\\u200bTGC\\u200bCTT\\u200bCAA\\u200bACC\\u200bGGA\\u200bAGC-3\\u2032 (reverse). Enriched DNA after immunoprecipitation with antibodies against EN1 or SP1 from EN1 knockdown and control human fibroblasts stimulated with TGF\\u03b2 (with three biological replicates per condition) was sequenced by Novogene on an Illumina NovaSeq 6000 platform. Alignment to the GRCh37 reference genome was performed using Bowtie2 software ( Langmead and Salzberg, 2012 ), and the generated result bam files were sorted with samtools ( Li, 2011 ) and parsed by MACS v2.2.7.1 ( Feng et al., 2012 ) for peak calling. Consequently, the results were assessed by ChIPQC library ( Carroll et al., 2014 ), and peak annotation was performed using ChIPseeker ( Yu et al., 2015 ). The promoter regions were then carefully extracted for pairwise comparison with the differential gene expression results, down- and up-regulated, respectively. Functional enrichment analysis was completed using g:Profiler, and figures were plotted using ggplot2 library. The ChIP-seq datasets were deposited in the GEO database (accession no. GSE174580 ). Quantification of collagen protein The amount of soluble collagen in cell culture supernatants was quantified using the SirCol collagen assay (Biocolor). The total collagen content of skin tissue samples was determined by hydroxyproline assay as described previously ( Zehender et al., 2018 ; Zhang et al., 2017 ). ECM staining ECM staining was performed as described ( Chakraborty et al., 2020 ). An equal number of fibroblasts (3 \\u00d7 10 3 /well for 384-well plates) was seeded in each well of dark-walled imaging plates (BD Biosciences). After 5 d of incubation with or without stimulation with recombinant TGF\\u03b2 (10 ng/ml), IL-11 (10 ng/ml), IL-13 (100 ng/ml), or CTGF (50 ng/ml) at 37\\u00b0C and 5% CO 2 , cells were washed in PBS and lysed with 0.25 M ammonium hydroxide in 25 mM Tris for 3 min at 37\\u00b0C. Before cell lysis, Preston Blue staining was performed to account for differences in cell viability between conditions. The matrix was washed three times in PBS, fixed using 100% methanol for 30 min at \\u221220\\u00b0C, and then stained with anti-Fibronectin Alexa Fluor 488 (1:200; eBioscience), anti-collagen type I antibody (1:200; Sigma-Aldrich), and anti-collagen type III (1:200, Millipore). The ECM stainings were imaged using the CellInsight CX5 High Content Screening Platform (Thermo Fisher Scientific), and total intensity per well was automatically quantified using a custom pipeline in HCS Studio Cell Analysis software, normalized to cell viability, and represented as a violin plot for each condition using ggplot2. Immunohistochemistry and immunofluorescence staining Formalin-fixed, paraffin-embedded skin sections or fibroblasts fixed in 4% paraformaldehyde and permeabilized by 0.25% Triton X-100 were stained as described ( Matei et al., 2019 ). Primary antibodies used were against EN1 (1:50, #83693; Abcam), P4H\\u03b2 (1:50, #MA3-019; Thermo Fisher Scientific), vimentin (1:200, #20346; Abcam), CD45 (1:100, #sc-59070; Santa Cruz Biotechnology), CD31 (1:100, #AF3628; R&D Systems), \\u03b1SMA (1:1,000, #A5228; Sigma-Aldrich), and \\u03b1-Tubulin (1:500, #T9026; Sigma-Aldrich). Alexa Fluor\\u2013conjugated secondary antibodies (Thermo Fisher Scientific) were used (1:200). Isotype-matched antibodies were used as negative controls. Stress fibers were visualized with rhodamine-conjugated phalloidin (1:40, #R415; Sigma-Aldrich). Nuclei were counterstained using DAPI (1:800, #sc-3598; Santa, Cruz Biotechnology). The stains of human or murine skin tissues were analyzed using an Eclipse 80i microscope (Nikon), and quantification of percentages of EN1-positive fibroblasts was performed using a custom CellProfiler 3.0 pipeline ( McQuin et al., 2018 ). Voronoi tessellation of immunofluorescence stainings was performed using Fiji software as described ( Matei et al., 2018 ; Schindelin et al., 2012 ). The stains of cultured cells were analyzed using the CellInsight CX5 High Content Screening Platform (Thermo Fisher Scientific), and average or total intensity per cell was automatically quantified using a custom pipeline in HCS Studio Cell Analysis software and represented as a violin plot for each condition using ggplot2. IMC: Staining and data analysis IMC was performed as described ( Giesen et al., 2014 ). The antibodies were acquired preconjugated (Fluidigm) or in purified preparations. All the purified antibodies were first validated by standard immunofluorescence staining. Purified antibodies were consequently conjugated to lanthanide metals using the Maxpar X8 antibody labeling kit (Fluidigm) following the manufacturer\\u2019s instructions. The complete panel was once more validated in IMC, and all the antibodies were titrated in IMC. Paraffin-embedded skin sections (5 \\u00b5m) were deparaffinized with xylene for 30 min and rehydrated in a graded series of alcohol (ethanol:deionized water 100:0, 100:0, 90:10, and 80:20 for 5 min each). For epitope retrieval, the slides were incubated for 30 min in preheated Tris-EDTA buffer (10 mM Tris base, 1 mM EDTA, and 0.05% Tween 20, pH 9). After slides were cooled, they were blocked with 2% BSA in PBS for 1 h at room temperature. Samples were incubated overnight at 4\\u00b0C with the metal-labeled antibody mix (in 0.5% BSA). The antibodies are listed in Table S3 . Tissue samples were washed once in PBS-T (PBS and 0.2% Tween 20) and twice in PBS for 5 min each. DNA staining was performed with Iridium-Intercalator (125 \\u00b5M) 1/400 for 5 min at room temperature. Afterward, the samples were washed three times in PBS and once in deionized water for 5 min each. Finally, the tissue sections were dried, stored, and shipped at room temperature. Imaging was performed using a Hyperion Imaging System (Fluidigm) by the group of Prof. Dr. Bertram Bengsch (Department of Medicine II: Gastroenterology, Hepatology, Endocrinology, and Infectious Disease, University Medical Center Freiburg). The MCD files were converted to TIFF format, segmented into single cells using a publicly available analysis pipeline ( https://zenodo.org/record/3841961 ), and analyzed with histoCAT with logarithmic data normalization ( Schapiro et al., 2017 ) and FlowJo software (BD Life Sciences). Before applying the Phenograph clustering ( Levine et al., 2015 ), all epithelial cells (E-cadherin + ) and immune cells (CD45 + ) were excluded. Finally, t-distributed stochastic neighbor embedding (tSNE) plots were generated using a custom Python script. ROCK activity assay The activity of ROCK in lysates of human fibroblasts was measured by ROCK activity assay according to the manufacturer\\u2019s instructions (#CSA001; Millipore). Statistics All data are presented as median \\u00b1 interquartile range. For a two-group comparison, Mann\\u2013Whitney U test for nonparametric data was used. When more than two groups of samples were compared, one-way ANOVA with Tukey\\u2019s range test as post hoc analysis was used. P values <0.05 were considered statistically significant. P values are expressed as follows: *, 0.05 > P > 0.01; **, 0.01 > P > 0.001; and ***, P < 0.001. GraphPad Prism software v8.0 was used for statistical analysis. The sample size was estimated based on previous experiments. No statistical method was used to predetermine sample size. Online supplemental material Fig. S1 shows EN1 expression in healthy and SSc skin in endothelial cells, leukocytes and keratinocytes, as well as compositions of IMC clusters and EN1 expression across clusters in dorsal and ventral skin. Fig. S2 shows that knockout of En1 in murine fibroblasts prevents TGF\\u03b2-induced fibroblast activation. Fig. S3 represents the effects of EN1 overexpression on IL-11\\u2013, IL-13\\u2013, and CTGF-induced fibroblast activation. Fig. S4 lists the transcription factors with enriched binding motifs in the promoters of EN1-DEGs and the transcription factors with predicted cobinding with EN1-SP-DEGs with enriched motifs only in up- or down-regulated DEGs. Fig. S5 shows that knockdown of EN1 prevents myofibroblast differentiation by inducing microtubule stabilization. Table S1 lists the primers used for quantitative real-time PCR. Table S2 lists the clinical data of the SSc patients. Table S3 lists the antibodies used in the IMC panel. Table S4 and Table S5 show transcription factor binding site enrichments in the promoters of EN1-DEGs and EN1-SP-DEGs, respectively. Table S6 shows functional enrichment analysis of the EN1-SP-DEGs.\",\n",
            "        \"subjects\": null,\n",
            "        \"results\": \"Results EN1 expression is up-regulated in fibrotic skin We first analyzed the expression of different members of the EN family of homeodomain-containing transcription factors in the skin of patients with SSc. Immunofluorescence staining demonstrated increased expression of EN1 and prominent staining, with nucleocytoplasmic pattern, in dermal fibroblasts (identified as prolyl-4-hydroxylase-\\u03b2 [P4H\\u03b2]\\u2013positive, CD45-negative cells) in the skin of SSc patients ( Fig. 1 A ). The percentage of EN1-positive fibroblasts was increased in SSc patients as compared with healthy individuals ( Fig. 1 A ). Other cell populations such as endothelial cells, leukocytes, and keratinocytes were stained with lower intensity for EN1 than fibroblasts, and EN1 expression was not changed in SSc skin in these cells ( Fig. S1 ). Higher expression levels of EN1 were maintained in SSc fibroblasts for several passages in vitro. Cultured fibroblasts from SSc patients expressed higher levels of EN1 mRNA and EN1 intracellular protein and secreted higher amounts of EN1 as compared with healthy dermal fibroblasts ( Fig. 1, B and C ). The expression of En1 was also up-regulated in murine models of skin fibrosis, with higher En1 protein levels in fibrotic mice as compared with nonfibrotic controls ( Fig. 1, D and E ). Moreover, the percentage of En1-positive fibroblasts (defined as En1 + , Vimentin + , CD45 \\u2212 cells) was highly increased in the dermis of bleomycin-challenged mice and in the hypodermis of Tsk-1 mice in comparison with their respective controls ( Fig. 1, F and G ). In contrast to EN1, the other member of the EN family, EN2, was expressed at very low levels and did not differ between fibrotic and healthy human skin. Figure 1. EN1 expression is up-regulated in fibrotic skin. (A) Representative immunofluorescence stainings for EN1 (green) and costainings with P4H (red) and CD45 (magenta) in the dermis of healthy donors and SSc patients at 1,000-fold magnification (scale bars = 20 \\u00b5m). Hematoxylin and eosin stainings (200-fold magnification, scale bars = 100 \\u00b5m), Voronoi diagrams, and percentages of EN1-positive fibroblasts from all fibroblasts (defined as P4H + CD45 \\u2212 cells) are included. Data are shown from one experiment ( n \\u2265 4). (B and C) EN1 mRNA (B) and EN1 intracellular or secreted protein levels (C) in cultured fibroblasts or their supernatants from healthy donors and SSc patients. Representative Western blot images and quantifications are included (C). Data are shown from one experiment ( n = 3). (D and E) En1 protein levels in the skin of mice challenged with bleomycin (D) or Tsk-1 mice (E) and their respective controls. Representative Western blot images and quantifications are included. Data are compiled from two independent experiments ( n \\u2265 4). (F and G) Representative immunofluorescence stainings for EN1 (green) and costainings with Vimentin (red) and CD45 (magenta) in the dermis of mice challenged with bleomycin and their controls at 1,000-fold magnification (F; scale bars = 20 \\u00b5m) and in the hypodermis of Tsk-1 mice and their controls, respectively, at 1,000-fold magnification (G; scale bars = 20 \\u00b5m). Hematoxylin and eosin stainings (100-fold magnification, scale bars = 200 \\u00b5m [F] and 40-fold magnification, scale bars = 500 \\u00b5m [G]), Voronoi diagrams, and percentages of EN1-positive fibroblasts from all fibroblasts (defined as Vimentin + CD45 \\u2212 cells) are included. Data are shown from one experiment each ( n \\u2265 3). All data are represented as median \\u00b1 interquartile range. P values are expressed as follows: *, 0.05 > P > 0.01 (Mann-Whitney U test). Figure S1. EN1 expression in healthy and SSc skin. (A\\u2013C) Representative immunofluorescence stainings for EN1 (green) and costainings with CD31 (A) and CD45 (B; both magenta) in the dermis of healthy donors and SSc patients at 1,000-fold magnification (scale bars = 20 \\u00b5m). Hematoxylin and eosin stainings (200-fold magnification, scale bars = 100 \\u00b5m) are included. Data for A\\u2013C were obtained from n \\u2265 4 biological samples from two independent experiments. (D) Composition of clusters from the segmented IMC images in ventral and dorsal skin biopsies from SSc patients. (E) tSNE plot showing distribution of cells from ventral and dorsal SSc skin across clusters. (F) tSNE plot showing expression of EN1 across the ventral and dorsal SSc clusters. Data for D\\u2013F were obtained from n = 8 biological samples from four SSc patients (paired ventral and dorsal biopsies). EN1 is expressed by multiple fibroblast subpopulations in both ventral and dorsal SSc skin To evaluate whether EN1 expression is a common characteristic of all fibroblasts or whether it is rather restricted to a fibroblast subpopulation in SSc, we performed multiplexed IMC from paired biopsies from ventral and dorsal skin of SSc patients ( Fig. 2 A ). We designed and validated a 22-plex panel for IMC that comprises well-characterized fibroblast markers such as P4H\\u03b2, platelet-derived growth factor receptor \\u03b1 (PDGFR\\u03b1), \\u03b1SMA, fibroblast activation protein (FAP), FSP-1, Thy1, DPP4, and Cdh-11, allowing simultaneous identification of multiple fibroblast subpopulations. We used image segmentation algorithms to generate cell masks ( Fig. 2 A ), which allowed extraction of high-dimensional single-cell data of marker expression. Subsequent clustering using the Phenograph algorithm identified 11 cellular clusters ( Fig. 2 B ). EN1 was expressed at various levels in all clusters, with the highest expression (>1 SD above the mean) in clusters 6 and 7 and the lowest expression in clusters 1 and 4 ( Fig. 2, C and D ). Clusters 6 and 7 also showed particularly high expression of \\u03b1SMA as the typical myofibroblast marker ( Fig. 2, D\\u2013H ). Cluster 6 identifies a subpopulation with high expression of most fibroblast markers, but also of CD31/von Willebrand factor and with a particular perivascular localization, suggestive of cells that may have undergone endothelial-to-mesenchymal transition ( Fig. 2, D\\u2013H ). Other clusters that express EN1 at intermediate levels are characterized by marker expression of different fibroblast subpopulations: cluster 8 predominantly expresses Cdh-11; clusters 9 and 10 highly express FAP; and cluster 5 expresses FSP-1 ( Fig. 2 D ). EN1 expression was similar in dorsal and ventral skin, but clusters 6 and 7 were underrepresented and cluster 9 was nearly absent in ventral skin ( Fig. S1, D\\u2013F ). In contrast, cluster 8, expressing EN1 at intermediate levels, was overrepresented in ventral skin ( Fig. S1, D\\u2013F ). Figure 2. EN1 is expressed in multiple fibroblast subpopulations in the skin of SSc patients. (A) Schematic representation of the experimental and analytical workflow. IMC was performed on paired skin biopsies from ventral and dorsal skin from four SSc patients stained with a panel consisting of 22 metal-labeled antibodies. Segmentation of the multiplexed images comprising the signals from all markers into masks representing single cells was performed. (B) Phenograph clustering and visualization on a tSNE map of CD45 \\u2212 E-Cadherin \\u2212 cells from all patients, regardless of biopsy location. (C) Expression of EN1 (as log[normalized intensity]) across the clusters. (D) Heatmap of marker expression across the clusters (as Z-score). (E and F) Representative hematoxylin and eosin staining (E) and schematic representation of clusters 6 and 7 (as the clusters with the highest expression of EN1, >1 SD above the mean; F), illustrating the spatial relationships of cells belonging to these clusters (scale bars = 200 \\u00b5m). (G and H) Visualization of the area of interest marked in F, with EN1 signals (green) and \\u03b1SMA (G) or CD31 and von Willebrand factor (H; red) as the markers with the highest expression from the clusters 6 and 7 (scale bars = 50 \\u00b5m). Data were obtained from n = 8 biological samples from 4 SSc patients (paired ventral and dorsal biopsies) from one experiment. TGF\\u03b2 signaling induces EN1 expression in a SMAD3-dependent manner SSc fibroblasts are characterized by persistent up-regulation of TGF\\u03b2 signaling under culture conditions. Considering the up-regulation of EN1 in cultured SSc fibroblasts, as well as in SSc patients and different mouse models of skin fibrosis, we hypothesized that TGF\\u03b2 may reactivate the expression of EN1 in SSc skin. Indeed, stimulation of cultured fibroblasts with recombinant TGF\\u03b2 induced a persistent up-regulation of EN1 protein levels ( Fig. 3 A ). In murine skin, activation of TGF\\u03b2 signaling by overexpression of a constitutively active TGF\\u03b2 receptor type I (TBRI CA ) increased the protein levels of En1 and induced the expression of En1 in dermal fibroblasts (Vimentin + , CD45 \\u2212 cells; Fig. 3, B and C ). Inhibition of TGF\\u03b2 signaling by treatment with the selective TBRI inhibitor SD-208 prevented the up-regulation of En1 in bleomycin-induced skin fibrosis and in fibrotic skin of Tsk-1 mice, further highlighting that TGF\\u03b2 is required for the up-regulation of En1 in fibrotic conditions ( Fig. 3, D and E ). The stimulatory effect of TGF\\u03b2 on EN1 expression is SMAD3-dependent, as siRNA-mediated knockdown of SMAD3 abrogated the induction of EN1 by TGF\\u03b2 in vitro ( Fig. 3 F ). In vivo, inhibition of canonical TGF\\u03b2 signaling by Smad3 siRNA prevented En1 up-regulation in the skin of TBRI CA - and bleomycin-challenged mice, confirming the Smad3 dependence of the TGF\\u03b2-induced En1 up-regulation ( Fig. 3, G and H ). In silico analysis of the EN1 promoter revealed two SMAD3 binding sites. ChIP PCR confirmed enhanced binding of SMAD3 to both predicted SMAD3-binding sites in the EN1 promoter upon TGF\\u03b2 stimulation ( Fig. 3 I ). Figure 3. TGF\\u03b2 up-regulates EN1 in a SMAD3-dependent manner. (A) EN1 protein levels in cultured human fibroblasts at 1, 3, 6, 24, and 48 h after TGF\\u03b2 stimulation. Representative Western blot images and quantification are included. Data are compiled from two experiments ( n \\u2265 4). (B) En1 protein levels in the skin of mice expressing TBRI CA and their controls. Representative Western blot images and quantifications are included. Data are compiled from two experiments ( n \\u2265 4). (C) Representative immunofluorescence stainings for En1 (green) and costainings with Vimentin (red) and CD45 (magenta) in the dermis of mice expressing TBRI CA and their controls at 1,000-fold magnification (scale bars = 20 \\u00b5m). Data are shown from one experiment ( n \\u2265 4). Hematoxylin and eosin stainings (100-fold magnification, scale bars = 200 \\u00b5m), Voronoi diagrams, and percentages of EN1-positive fibroblasts from all fibroblasts (defined as Vimentin + CD45 \\u2212 cells) are included. (D and E) En1 protein levels in the skin of mice challenged with bleomycin (D) or Tsk-1 mice (E) and/or treated with the TBRI inhibitor SD-208 and their respective controls. Representative Western blot images and quantifications are included. Data are compiled from two experiments ( n \\u2265 4). (F) EN1 protein levels in cultured human fibroblasts with siRNA-mediated SMAD3 knockdown with or without stimulation with TGF\\u03b2 (24 h) and their controls (fibroblasts transfected with nontargeting siRNA). Representative Western blot images and quantification are included. Data are compiled from two experiments ( n = 4). (G and H) En1 protein levels in the skin of TBRI CA mice (or, as controls, mice injected with an adenovirus expressing LacZ; G) or of mice challenged with bleomycin (or, as controls, mice injected with NaCl; H) and/or treated with SMAD3 siRNA and their controls (mice injected with nontargeting siRNA). Representative Western blot images and quantifications are included. Data are compiled from two experiments ( n = 5). (I) ChIP PCR for SMAD3 binding at predicted SMAD3 motifs in the promoter of EN1. Data are compiled from two experiments ( n = 4). All data are represented as median \\u00b1 interquartile range. P values are expressed as follows: *, 0.05 > P > 0.01; **, 0.01 > P > 0.001; ***, P < 0.001 (Mann\\u2013Whitney U test for B and C; ANOVA with Tukey\\u2019s post hoc test for A and D\\u2013I). EN1 promotes fibroblast activation and collagen production We next analyzed whether EN1 can regulate fibroblast activation. siRNA-mediated knockdown of EN1 in adult human dermal fibroblasts rendered them less sensitive to the profibrotic effects of TGF\\u03b2, with lower levels of COL1A1 and COL1A2 mRNA, and reduced release of collagen protein (compared with nontargeting siRNA\\u2013treated fibroblasts; Fig. 4, A and B ). Moreover, EN1 knockdown prevented TGF\\u03b2-induced fibroblast-to-myofibroblast transition, with reduced levels of ACTA2 mRNA and \\u03b1SMA protein and impaired stress fiber formation ( Fig. 4, C\\u2013E ). En1 fl/fl murine fibroblasts transduced with AdCre ( En1 -knockout fibroblasts) released less collagen and expressed lower levels of myofibroblast markers in response to TGF\\u03b2 than AdLacZ-transduced controls (with unaltered expression of En1; Fig. S2 ). Overexpression of EN1 in human fibroblasts fostered TGF\\u03b2-induced fibroblast activation with higher levels of COL1A1 and COL1A2 mRNA and collagen protein and increased levels of ACTA2 mRNA and \\u03b1SMA protein and enhanced formation of stress fibers ( Fig. 4, F\\u2013J ). EN1 overexpression also enhanced the profibrotic effects of IL-11, IL-13, and connective tissue growth factor (CTGF) on deposition of fibronectin and collagen type I, expression of \\u03b1SMA, and formation of stress fibers, but to a much lesser extent than the effects of TGF\\u03b2 ( Fig. S3 ). Figure 4. EN1 promotes fibroblast activation and collagen production. (A\\u2013E) siRNA-mediated knockdown of EN1 prevents TGF\\u03b2-induced fibroblast activation. Relative mRNA levels of COL1A1 and COL1A2 (A). Relative protein levels of secreted collagen type I (B). Relative mRNA levels of ACTA2 (C). Representative immunofluorescence stainings for \\u03b1SMA (D) and stress fibers (E) at 400-fold magnification (scale bars = 50 \\u00b5m). Quantification of average signal intensity for each cell (relative to control) is included. Data for A\\u2013E were obtained from n \\u2265 4 independent biological samples per group from two independent experiments. (F\\u2013J) EN1 overexpression promotes fibroblast activation. Relative mRNA levels of COL1A1 and COL1A2 (F). Relative protein levels of secreted collagen type I (G). Relative mRNA levels of ACTA2 (H). Data for F\\u2013J were obtained from n \\u2265 4 independent biological samples per group from two independent experiments. Representative immunofluorescence stainings for \\u03b1SMA (I) and stress fibers (J) at 400-fold magnification (scale bars = 50 \\u00b5m). Quantification of average signal intensity for each cell (relative to control) is included. All data are represented as median \\u00b1 interquartile range. P values are expressed as follows: *, 0.05 > P > 0.01; **, 0.01 > P > 0.001; ***, P < 0.001 (ANOVA with Tukey\\u2019s post hoc test). Figure S2. Knockout of En1 ameliorates TGF\\u03b2-induced fibroblast activation. (A) Representative immunofluorescence stainings for En1 (green) and costaining with Vimentin (red) in the dermis of mice with fibroblast-specific knockout of En1 and controls, both groups challenged with bleomycin, at 1,000-fold magnification (scale bars = 20 \\u00b5m). Data were obtained from one experiment ( n = 4). (B) En1 protein levels (representative Western blot images). Data were obtained from one experiment ( n = 3). (C) Relative mRNA levels of Col1a1 and Col1a2 . (D) Secreted collagen 1 protein levels. (E) Relative mRNA levels of Acta2 . Data for C\\u2013E were compiled from two independent experiments ( n \\u2265 4). (F) Representative immunofluorescence stainings for \\u03b1SMA (green) and stress fibers (red) at 400-fold magnification (scale bars = 50 \\u00b5m). Quantification of signal intensity relative to control is included. Data for F were obtained from two independent experiments ( n \\u2265 4). All data are represented as median \\u00b1 interquartile range. P values are expressed as follows: **, 0.01 > P > 0.001; ***, P < 0.001 (ANOVA with Tukey\\u2019s post hoc test). Figure S3. EN1 overexpression promotes IL-11\\u2013, IL-13\\u2013, and CTGF-induced fibroblast activation and ECM deposition to a lesser extent than TGF\\u03b2-induced fibroblast activation and ECM deposition. (A and B) Representative immunofluorescence stainings for fibronectin (A) and collagen type I (B) at 100-fold magnification (scale bars = 200 \\u00b5m). Quantification of average signal intensity (relative to control) is included. (C and D) Representative immunofluorescence stainings for \\u03b1SMA (C) and stress fibers (D) at 200-fold magnification (scale bars = 100 \\u00b5m). Quantification of average signal intensity for each cell line (relative to control) is included. All data are represented as median \\u00b1 interquartile range of n \\u2265 3 independent biological samples per group from at least two independent experiments. P values are expressed as follows: *, 0.05 > P > 0.01; **, 0.01 > P > 0.001; ***, P < 0.001 (ANOVA with Tukey\\u2019s post hoc test). Data for A\\u2013D were obtained from two independent experiments ( n = 4). EN1 induces a profibrotic profile of gene expression by modulation of SP binding to the promoters of profibrotic genes To evaluate whether EN1 induces transcriptomic changes to promote fibroblast activation, we performed RNA sequencing (RNA-seq) from adult human dermal fibroblasts with knockdown of EN1 . We identified 632 high-confidence differentially expressed genes (EN1-DEGs; 239 down-regulated and 315 up-regulated) between TGF\\u03b2-stimulated fibroblasts with siRNA-induced EN1 knockdown and TGF\\u03b2-stimulated fibroblasts transfected with nontargeting siRNA ( Fig. 5 A ). gProfiler enrichment analysis of the EN1-DEGs revealed enrichment of multiple gene ontology (GO) and Reactome terms related to ECM production and organization (such as collagen formation or ECM organization), cytoskeleton organization (such as microtubule cytoskeleton organization or cytoskeleton), and Rho GTPases activity (such as signaling by Rho GTPases), confirming the functional relevance of EN1-regulated gene expression for TGF\\u03b2-induced fibroblast activation ( Fig. 5 B ). Gene set enrichment analysis (GSEA) revealed that EN1 expression levels did not influence apoptosis-related gene sets ( Fig. 5 C ). Figure 5. EN1 induces a profibrotic gene expression profile. (A\\u2013C) RNA-seq results showing that knockdown of EN1 partially reverses the profibrotic gene expression profile induced by TGF\\u03b2, without affecting apoptosis. Heatmap of expression levels of the 632 EN1-DEGs across the three replicates for each condition. Hierarchical clustering of the replicates and of the EN1-DEGs\\u2019 expression are included (A). Dot plot showing gene ratio (on the x axis), gene count (as dot size), and adjusted P value (as color gradient) of selected enriched GO and Reactome terms (after g:Profiler enrichment analysis) related to fibrosis (B). GSEA of the apoptosis Reactome term (C). Data for A\\u2013C were obtained from n = 3 independent biological samples per group. (D) Circular packing plots illustrating enrichment of fibrosis-related GO and Reactome terms (after g:Profiler enrichment analysis) of the EN1-DEGs with predicted binding motifs for SP1/2/3/8 (SP) and cobinding with other members of the C2H2 or E2F families, or with NFYB (expressed as percentages from the total fibrosis-related processes enriched in EN1-DEGs). The threshold for DEGs was false discovery rate < 0.25 and fold-change >1.5, and the conditions were as follows: nontargeting siRNA treated human fibroblasts stimulated with TGF\\u03b2 as reference group and EN1 siRNA treated human fibroblasts stimulated with TGF\\u03b2 as treatment group. (E) Coimmunoprecipitation of EN1 with SP1 in human fibroblasts \\u00b1 TGF\\u03b2 (24 h). Representative Western blot images are included. Data were obtained from two independent experiments ( n = 3). We next performed in silico transcription factor profiling of the promoters of EN1-DEGs. We screened the promoter regions of all EN1-DEGs for potential binding sites against all available human transcription factor matrices ( Table S4 ). Of note, we did not identify statistically enriched EN1-binding sites in EN1-DEGs ( Table S4 ), indicating that EN1 may rather indirectly regulate gene expression in TGF\\u03b2-stimulated fibroblasts via modulation of the transcriptional activity of other transcription factors. We found, in particular, members of the SP subfamily of C2H2 zinc finger transcription factors to be enriched in EN1-DEGs, with more than twofold increases of binding motifs for SP1, SP2, SP3, and SP8 as compared with the mean of two randomly generated gene sets as negative controls ( Fig. S4 A ). Coimmunoprecipitation experiments demonstrated that TGF\\u03b2 promotes interaction of EN1 with SP1, providing experimental validation for the in silico model of EN1-mediated transcriptional regulation ( Fig. 5 E ). Although more common in up-regulated EN1-DEGs, SP1-, SP2-, SP3-, and SP8-binding motifs were present in the promoters of up- and down-regulated EN1-DEGs, suggesting that recruitment of additional transcriptional regulators may determine the transcriptional outcome of EN1-SP1/2/3/8 binding ( Fig. S4 A ). Figure S4. EN1 promotes formation of SP1/2/3/8-containing regulatory modules with specialized profibrotic function. (A) Transcription factors with enriched binding motifs in the promoters of up- and down-regulated EN1-DEGs, with gene ratio in EN1-DEGs (ratio of genes that contain motifs for the respective transcription factor) more than twofold higher than the gene ratio in negative controls, colored by families. (B) Circular packing plots illustrating predicted SP1/2/3/8 (SP) binding in the promoters of up- and down-regulated EN1-DEGs and cobinding with other members of the C2H2 or E2F families and with NFYB or TFDP1, expressed as percentages. Further transcription factor coregulation analysis of the promoters of these EN1-DEGs with SP1-, SP2-, SP3-, and SP8- binding motifs (EN1-SP-DEGs) revealed a consistent pattern for all four SP members, with major enrichment of binding sites for other members of the C2H2 family (in particular, KLF5, 16, and 9 and ZNF263 and 384), as well as for members of the EF family (E2F4 and E2F6; Table S5 ). Binding sites for these transcription factors were found in similar frequencies in up- and down-regulated EN1-SP-DEGs ( Fig. S4 B ). Interestingly, we also identified binding motifs for other transcription factors that were exclusively present in up- or in down-regulated genes: binding motifs for transcription factor AP-2\\u03b1 (TFAP2A) were enriched in up-regulated EN1-SP-DEGs, indicating that in conditions of increased expression of EN1 recruitment of TFAP2A may promote transcriptional repression; in contrast, binding motifs for nuclear transcription factor Y \\u03b2 (NFYB) and for transcription factor Dp-1 (TFDP1) were exclusively found in down-regulated EN1-SP-DEGs ( Fig. S4 B and Table S5 ), suggesting that their recruitment may promote transcriptional activation. Functional analysis of the EN1-SP-DEGs and EN1-SP-DEGs with binding motifs for other C2H2 transcription factors revealed major enrichment of processes related to fibroblast activation, in particular of those pertaining to cytoskeleton organization and Rho GTPases activity ( Fig. 5 D and Table S6 ). EN1 ChIP-seq showed binding of EN1 predominantly to intergenic and intronic regions, while SP1 ChIP-seq revealed binding of SP1 predominantly to promoter regions in all three biological replicates ( Fig. 6 A ). SP1 ChIP-seq demonstrated that knockdown of EN1 significantly decreases the genomic binding of SP1, with reduced peak counts and reduced activity of SP1 in peaks ( Fig. 6, B and C ). From the few genes with EN1 binding in promoter regions, only the following genes were shared among all three biological replicates: ROCK1P1 , DUX4 , and PACSIN2 . Of note, all of these genes also showed binding of SP1 in their promoter region. We found no direct binding of EN1 to the promoters of EN1-DEGs; instead, we found binding of SP1 to the promoter of multiple EN1-DEGs, especially to the down-regulated EN1-DEGs. gProfiler enrichment analysis of the EN1-DEGs with direct binding of SP1 revealed enrichment of multiple GO and Reactome terms related to fibrosis such as cytoskeleton, microtubule, and signaling by Rho GTPases ( Fig. 6 D ). Figure 6. Direct genomic targets of EN1 and SP1. (A) Genomic localization of EN1 and SP1 peaks showing enrichment of EN1 peaks in intergenic and intronic regions, while SP1 peaks show great enrichment in promoter regions defined as 300\\u20133,000 bp upstream from transcription start site. (B and C) Knockdown of EN1 reduces the number and activity of SP1 peaks across all three biological replicates. Number of SP1 peaks (B). Read counts in SP1 peaks (C). (D) Dot plot showing gene ratio (on the x axis), gene count (as dot size), and adjusted P value (as color gradient) of selected enriched GO and Reactome terms after g:Profiler enrichment analysis of the EN1-DEGs with direct SP1 binding related to fibrosis. Data were obtained from n = 3 independent biological samples per group. EN1 promotes rearrangements of the microtubule-actin cytoskeleton in a ROCK-dependent manner to induce fibroblast activation GSEA showed negative enrichment scores (\\u201cde-enrichment\\u201d) for the GO gene sets regulation of actin nucleation and microtubule depolymerization in EN1 -knockdown fibroblasts stimulated with TGF\\u03b2, suggesting that EN1 might induce myofibroblast differentiation by promoting microtubule-dependent cytoskeletal rearrangements ( Fig. 7 A ). Figure 7. EN1 overexpression promotes microtubule depolymerization to induce myofibroblast differentiation. (A) GSEA of the indicated GO terms, showing that EN1 knockdown de-enriches genes related to microtubule depolymerization and actin nucleation. (B and C) EN1 overexpression induces microtubule depolymerization. Representative immunofluorescence stainings for \\u03b1-Tubulin at 400-fold magnification (scale bars = 50 \\u00b5m). Quantification of total signal intensity for each cell (relative to control) is included (B). Protein levels of soluble and insoluble \\u03b1-Tubulin. Representative Western blot images and quantification of insoluble/soluble \\u03b1-Tubulin ratio are included (C). Data for B and C were obtained from two independent experiments ( n \\u2265 4). (D\\u2013H) Pharmacological stabilization of microtubules prevents EN1-induced myofibroblast differentiation. Relative mRNA levels of COL1A1 (D). Relative protein levels of secreted collagen type I (E). Relative mRNA levels of ACTA2 (F). Representative immunofluorescence stainings for \\u03b1SMA (G) and stress fibers (H) at 400-fold magnification (scale bars = 50 \\u00b5m). Quantification of average signal intensity for each cell (relative to control) is included. Data for D\\u2013H were obtained from two independent experiments ( n \\u2265 4). All data are represented as median \\u00b1 interquartile range. P values are expressed as follows: *, 0.05 > P > 0.01; **, 0.01 > P > 0.001; ***, P < 0.001 (ANOVA with Tukey\\u2019s post hoc test). Microtubule depolymerization induces stress fiber assembly at focal adhesions and promotes cell contraction ( Liu et al., 1998 ; Ng et al., 2014 ). To experimentally validate an EN1-dependent regulation of microtubules in fibroblasts, we analyzed the microtubule polymerization state in human dermal fibroblasts with overexpression or knockdown of EN1 by two methods: Western blot for insoluble and soluble tubulin and a previously described tubulin polymerization high-content imaging assay ( Sum et al., 2014 ). Overexpression of EN1 promotes depolymerization of the microtubular network, with reduced tubulin intensity on immunofluorescence stainings and decreased insoluble/soluble tubulin ratio on Western blot ( Fig. 7, B and C ). In contrast, knockdown of EN1 enhanced the formation of microtubular networks, with increased tubulin intensity and higher insoluble/soluble tubulin ratio ( Fig. S5, A and B ). Incubation with paclitaxel, a stabilizer of microtubules, phenocopied the stimulatory effects of EN1 knockdown and prevented the inhibitory effects of EN1 overexpression on microtubule polymerization ( Fig. S5, A and B ; and Fig. 7, B and C ). In contrast, vinblastine, an inhibitor of microtubule polymerization, abrogated the stimulatory effects of EN1 knockdown on microtubule elongation ( Fig. S5, A and B ). Figure S5. Knockdown of EN1 stabilizes the microtubular network to inhibit stress fiber formation and myofibroblast differentiation. (A and B) Knockdown of EN1 stabilizes microtubules. Representative immunofluorescence stainings for \\u03b1-Tubulin at 400-fold magnification (scale bars = 50 \\u00b5m). Quantification of total signal intensity for each cell (relative to control) is included (A). Protein levels of soluble and insoluble \\u03b1-Tubulin. Representative Western blot images and quantification of insoluble/soluble \\u03b1-Tubulin ratio are included (B). Data for A and B were obtained from two independent experiments ( n \\u2265 4). (C\\u2013G) Pharmacological depolymerization of microtubules abrogates the inhibitory effects of EN1 knockdown on TGF\\u03b2-induced myofibroblast differentiation. Relative mRNA levels of COL1A1 (C). Relative protein levels of secreted collagen type I (D). Relative mRNA levels of ACTA2 (E). Representative immunofluorescence stainings for \\u03b1SMA (F) and stress fibers (G) at 400-fold magnification (scale bars = 50 \\u00b5m). Data for C\\u2013G were obtained from two independent experiments ( n \\u2265 4). Quantification of average signal intensity for each cell (relative to control) is included. All data are represented as median \\u00b1 interquartile range. P values are expressed as follows: *, 0.05 > P > 0.01; **, 0.01 > P > 0.001; ***, P < 0.001 (ANOVA with Tukey\\u2019s post hoc test). We next tested whether EN1 might regulate stress fiber formation and myofibroblast differentiation by modulation of the microtubular network. Indeed, microtubule stabilization by paclitaxel abrogated myofibroblast differentiation induced by EN1 overexpression, with reduced collagen production, \\u03b1SMA expression, and stress fiber formation in fibroblasts from standard 2D culture ( Fig. 7, D\\u2013H ). Moreover, microtubule stabilization with paclitaxel prevented the profibrotic effects of EN1 on 3D full-thickness skin equivalents, with lower levels of COL1A1 and ACTA2 mRNA, decreased myofibroblast counts, and reduced thickening of the dermal portion of the skin equivalents compared with vehicle-treated controls ( Fig. 9 ). In contrast, microtubule depolymerization by vinblastine abrogated the inhibitory effects of EN1 knockdown on TGF\\u03b2-induced fibroblast-to-myofibroblast transition ( Fig. S5, C\\u2013G ). De-enriched GO and Reactome terms also included signaling by Rho GTPases and Rho GTPase effectors ( Fig. 8 A ). The activity of Rho GTPases and of one of their downstream effectors, ROCK, plays a critical role in regulation of the cytoskeleton ( Chang et al., 2008 ; Krendel et al., 2002 ). We thus hypothesized that EN1 may induce ROCK activity by depolymerizing microtubules to promote stress fiber assembly and fibroblast activation. Indeed, knockdown of EN1 prevented the stimulatory effects of TGF\\u03b2 on ROCK activity, whereas overexpression of EN1 increased ROCK activity at baseline and in response to TGF\\u03b2 ( Fig. 8, B and C ). Stabilization of microtubules with paclitaxel prevented the induction of ROCK activity by EN1, highlighting that microtubule depolymerization is required for EN1 to induce ROCK activity ( Fig. 8 C ). Moreover, the selective ROCK inhibitor Y27632 prevented EN1-induced myofibroblast differentiation, stress fiber formation, and fibrotic transformation of 3D full-thickness skin equivalents ( Fig. 8, D\\u2013H ; and Fig. 9 ). Thus, EN1 promotes depolymerization of microtubules to induce ROCK activity, which facilitates TGF\\u03b2-induced stress fiber formation, myofibroblast differentiation, and collagen deposition. Figure 8. EN1 induces ROCK activity to coordinate microtubule-actin cytoskeletal rearrangements and myofibroblast differentiation. (A) GSEA of the indicated Reactome terms, showing that EN1 knockdown de-enriches genes related to signaling by Rho GTPases and Rho GTPases effectors. (B and C) EN1 induces ROCK activity by depolymerizing microtubules. ROCK activity after EN1 knockdown (B). ROCK activity after EN1 overexpression with/without microtubule stabilization (C). Data for B and C were obtained from two independent experiments ( n \\u2265 4). (D\\u2013H) Pharmacological inhibition of ROCK activity prevents EN1-induced myofibroblast differentiation. Relative mRNA levels of COL1A1 (D). Relative protein levels of secreted collagen type I (E). Relative mRNA levels of ACTA2 (F). Representative immunofluorescence stainings for \\u03b1SMA (G) and stress fibers (H) at 400-fold magnification (scale bars = 50 \\u00b5m). Quantification of average signal intensity for each cell (relative to control) is included. Data for D\\u2013H were obtained from two independent experiments ( n \\u2265 4). All data are represented as median \\u00b1 interquartile range. P values are expressed as follows: *, 0.05 > P > 0.01; **, 0.01 > P > 0.001; ***, P < 0.001 (ANOVA with Tukey\\u2019s post hoc test). Figure 9. EN1 promotes TGF\\u03b2-induced fibrotic transformation of 3D skin equivalents in a microtubule- and ROCK-dependent manner. (A\\u2013E) Pharmacological stabilization of microtubules or inhibition of ROCK activity prevents EN1-mediated fibrotic transformation of 3D skin equivalents. Relative mRNA levels of COL1A1 (A) and ACTA2 (B). Representative trichrome stainings of skin models, acquired with a slide scanner at 40\\u00d7 magnification; adjacent pictures of the same slide were stitched to generate overview pictures; white bars indicate dermal thickness (scale bars = 200 \\u00b5m; C). Quantification of dermal thickness (D) and of myofibroblast counts (E). All data are represented as median \\u00b1 interquartile range of n = 6 independent biological samples per group from two independent experiments. P values are expressed as follows: ***, P < 0.001 (ANOVA with Tukey\\u2019s post hoc test). Knockout of En1 ameliorates experimental skin fibrosis To analyze whether knockout of En1 reduces fibroblast activation in vivo and protects from experimental fibrosis, we generated En1 fl/fl \\u00d7 Col6Cre mice, with fibroblast-specific constitutive En1 knockout. Mice with selective depletion of En1 in fibroblasts did not show a basic phenotype in the absence of profibrotic stimuli as compared with controls, and their dermal structure was preserved ( Fig. 10 ). We employed three complementary models of skin fibrosis: (1) the TBRI CA model; (2) the bleomycin-induced model; and (3) the Tsk-1 model. Skin fibrosis was strongly ameliorated in mice with fibroblast-specific knockout of En1 in comparison with control mice in all three models, with reduced dermal or hypodermal thickness, lower myofibroblast counts, and reduced hydroxyproline content ( Fig. 10, A\\u2013L ). Figure 10. Fibroblast-specific knockout of En1 ameliorates experimental skin fibrosis. (A\\u2013D) TBRI CA mice with fibroblast-specific knockout of En1 are protected from skin fibrosis. Representative trichrome stainings of the skin shown at 100\\u00d7 magnification (scale bars = 200 \\u00b5m; A). Quantification of dermal thickness (B), myofibroblast counts (C), and hydroxyproline content (D), with figure legend under the plots. (E\\u2013H) Bleomycin-challenged mice with fibroblast-specific knockout of En1 are protected from skin fibrosis. Representative trichrome stainings of the skin shown at 100\\u00d7 magnification (scale bars = 200 \\u00b5m; E). Quantification of dermal thickness (F), myofibroblast counts (G), and hydroxyproline content (H), with figure legend under the plots. (I\\u2013L) Tsk-1 mice with fibroblast-specific knockout of En1 are protected from skin fibrosis. Representative trichrome stainings of the skin shown at 40\\u00d7 magnification (I). Quantification of dermal thickness (J), myofibroblast counts (K), and hydroxyproline content (L), with figure legend under the plots (scale bars = 500 \\u00b5m). All data for A\\u2013L are represented as median \\u00b1 interquartile range of n \\u2265 5 independent biological samples per group from at least two independent experiments. P values are expressed as follows: *, 0.05 > P > 0.01; ***, P < 0.001 (ANOVA with Tukey\\u2019s post hoc test).\",\n",
            "        \"discussion\": \"Discussion A landmark study by Rinkevich et al. (2015) and a follow up study by Jiang et al. (2018) showed that a subpopulation of fibroblasts transiently express En1 during skin development in murine embryos. The number of En1-positive fibroblasts increases during embryonic development, but these cells silence the expression of En1 before birth under homeostatic conditions ( Jiang et al., 2018 ). However, the regulation of En1 expression was not investigated in these studies. Here, we demonstrate that EN1 expression is up-regulated in fibrotic skin of SSc patients compared with skin from healthy donors, as well as in three different murine models of skin fibrosis. Multiple fibroblast subpopulations express EN1 in adult skin of SSc patients, in particular fibroblast subpopulations with high expression of \\u03b1SMA. A previous study evaluated the expression of EN1 in adult human skin by immunohistochemistry ( Miura et al., 2018 ). Although not specifically followed up by the authors, EN1-positive single cells can be observed in the dermis, suggesting that, in line with our results, adult fibroblasts in human skin can express EN1. In contrast to the redundant expression of EN1 and EN2 in dopaminergic neurons ( Hanks et al., 1995 ), the low expression of EN2 in adult skin and the lack of induction in fibrotic skin excluded a major role of EN2 in skin fibrosis. We demonstrate on multiple experimental levels that canonical TGF\\u03b2 signaling reactivates the expression of EN1 in cultured fibroblasts and fibrotic skin. (1) Stimulation of cultured dermal fibroblasts with recombinant TGF\\u03b2 up-regulates the intracellular levels of EN1. (2) EN1 levels are increased in SSc fibroblasts as compared with fibroblasts isolated from healthy individuals even after several passages in culture. As persistent activation of TGF\\u03b2 signaling is a major characteristic of SSc fibroblasts, this finding provides indirect evidence for a TGF\\u03b2-dependent regulation of EN1 in SSc. (3) Overexpression of TBRI CA increases En1 protein levels in murine skin, thus confirming that TGF\\u03b2 is sufficient to increase the expression levels of En1 in fibroblasts in vivo. (4) The induction of EN1 by TGF\\u03b2 is mediated by canonical TGF\\u03b2/SMAD3 signaling, as knockdown of SMAD3 abrogates the stimulatory effects of TGF\\u03b2 on EN1 expression in cultured fibroblasts and in murine skin. (5) Selective inhibition of TGF\\u03b2 signaling prevented the up-regulation of En1 in bleomycin-induced and Tsk-1 models of experimental skin fibrosis, highlighting that TGF\\u03b2 signaling is essentially required for the up-regulation of En1 in fibrotic skin. However, additional factors other than TGF\\u03b2 may also promote the expression of EN1. EN1 has been shown to be a target of WNT1 in vertebrate midbrain development, and WNT1 is also up-regulated in multiple fibrotic disorders including SSc ( Akhmetshina et al., 2012 ; Bergmann and Distler, 2016 ; Beyer et al., 2012b ; Danielian and McMahon, 1996 ; K\\u00f6nigshoff et al., 2009 ). Notch signaling, another stem cell pathway that is hyperactive in SSc, is required to maintain the expression of engrailed during the development of the ocellar complex in Drosophila melanogaster ( Aguilar-Hidalgo et al., 2013 ; Beyer and Distler, 2013 ; Dees et al., 2011a ; Dees et al., 2011b ). Finally, ischemia-reperfusion injury, which may occur as a consequence of microvascular endothelial dysfunction in SSc, can also induce EN1 expression and may contribute to its up-regulation in SSc ( Beyer et al., 2009 ; Villanueva et al., 2006 ). In extension of the findings by Rinkevich et al. (2015) , we demonstrate that EN1 not only serves as an embryonic marker for a defined population of fibroblasts with a high capacity to produce ECM proteins, but EN1 also regulates the activation status of adult human dermal fibroblasts. EN1-deficient fibroblasts are less sensitive to TGF\\u03b2-induced activation, whereas forced overexpression of EN1 renders fibroblasts more susceptible to the profibrotic effects of TGF\\u03b2 and, to a lesser extent, to those of other profibrotic molecules such as IL-11, IL-13, and CTGF ( Ng et al., 2019 ; Schafer et al., 2017 ), with increased myofibroblast differentiation and enhanced collagen release. Moreover, EN1 overexpression induces myofibroblast differentiation in the absence of external profibrotic stimuli. EN1 is thus both necessary to fully mount the fibroblast response to profibrotic stimuli (in particular to TGF\\u03b2) and also sufficient to induce fibroblast activation in the absence of external stimuli. Selective depletion of En1 in fibroblasts also reduced the profibrotic effects of TGF\\u03b2 signaling in vivo. Mice with fibroblast-specific knockout of En1 were protected from experimental fibrosis induced by overexpression of a constitutively active TGF\\u03b2 receptor type I. Fibroblast-specific depletion of En1 also ameliorated skin fibrosis in Tsk-1 mice and mice challenged with bleomycin. Lineage commitment of En1-expressing fibroblasts seemed to be unaffected in mice with fibroblast-specific En1 knockout, since the structure and collagen composition of their dermis were comparable to those of littermates with unaltered En1 expression. Although the frequency of recombination in Col6Cre transgenic mice is almost absent in nonmesenchymal cells ( Armaka et al., 2008 ), Col6Cre -induced recombination is less specific to fibroblasts than in, for example, PDGFR\\u03b1-Cre ( Horikawa et al., 2015 ). Despite this limitation of our study, the strong in vitro effects on fibroblast activation suggest that EN1 up-regulation and aberrant signaling in fibroblasts play a pivotal role in skin fibrosis. RNA-seq of adult human dermal fibroblasts with siRNA-mediated knockdown of EN1 combined with functional experiments demonstrated that EN1 induces depolymerization of microtubules with a decreased ratio of insoluble to soluble tubulin. Disruption of the microtubule network promotes formation and maturation of focal adhesions in a ROCK-dependent manner, stimulating the assembly of actin stress fibers ( Bershadsky et al., 1996 ; Liu et al., 1998 ; Ng et al., 2014 ), which is required to promote fibroblast-to-myofibroblast differentiation ( Dugina et al., 2001 ; Hinz et al., 2003 ). Pharmacologic stabilization of microtubules or inhibition of ROCK prevents EN1-induced fibroblast-to-myofibroblast transition and collagen release in both standard 2D fibroblast cultures and 3D full-thickness skin equivalents. In contrast, pharmacologically induced microtubule depolymerization abrogates the antifibrotic effects of EN1 knockdown. These data unravel a novel mechanism of regulation of cytoskeletal organization during fibroblast-to-myofibroblast differentiation in fibrotic diseases. Mechanistically, the effects of EN1 on cytoskeleton and ROCK activity are not mediated by direct binding to EN1 motifs in EN1-DEGs, but by trans-regulation, i.e., by modulating the activity/binding of SP family members of transcription factors. ChIP-seq experiments for EN1 and SP1, the SP member with the most predicted binding motifs in the EN1-DEGs, showed that EN1 predominantly binds to intergenic and intronic regions, while SP1 predominantly binds to promoter regions. Our results are in line with previous EN1 ChIP-seq data from triple-negative breast cancers, which also demonstrate predominant binding of EN1 to noncoding regions ( Peluffo et al., 2019 ). ChIP-seq for EN1 supports the conclusion that EN1 regulates the binding of SP1 to the promoter of profibrotic genes related to the microtubule cytoskeleton and Rho GTPases. In silico transcription factor profiling suggests that the profibrotic roles of EN1 are likely mediated by interference with a network of profibrotic transcription factors including SP1 (validated by SP1 ChIP-seq), SP2, SP3, SP8, and other members of the C2H2 family ( C\\u00f3rdova et al., 2015 ; Jiang et al., 2013 ; Manabe et al., 2002 ; Sysa et al., 2009 ; Verrecchia et al., 2001 ). The potent antifibrotic effects of EN1 knockdown might thus be attributed to the cumulative changes in target specificity of multiple transcription factors. In silico transcription factor profiling and functional enrichment analysis support an EN1-transcriptional regulation of gene expression with specific functional modularity: EN1-DEGs with predicted transcriptional regulation by SP family members (and coregulation by other members of the C2H2 family) are functionally related to fibroblast activation (in particular to cytoskeleton organization and Rho GTPases activity). These data implicate that EN1 can direct the specificity of SP-containing regulatory modules toward genes with functional relevance for fibrosis. Reducing EN1 levels or inhibiting the interaction of EN1 with SP family members may thus interfere with this pathological regulation of gene expression and reestablish the physiological target specificity of SP transcription factors. Targeted inactivation of EN1, for instance with recently described synthetically designed EN1 interference peptides (EN1-iPep; Beltran et al., 2014 ; Gandhi et al., 2018 ; Sorolla et al., 2019 ) might thus offer therapeutic potential for the treatment of fibrotic diseases such as SSc. Considering the antiapoptotic roles of EN1 in dopaminergic neurons ( Rekaik et al., 2015 ), a systemic therapy that targets EN1 may have adverse effects on the central nervous system. However, in the case of EN1-iPep, occurrence of major nervous system toxicity should be hindered by several factors: (1) the low diffusibility of EN1-iPep through the blood\\u2013brain barrier; (2) the C-terminal RGD domain of EN1-iPep ( Sorolla et al., 2019 ) that increases the specificity for cells with high expression of integrin \\u03b1v, such as fibroblasts ( Margadant and Sonnenberg, 2010 ); and (3) the selective targeting of EN1, since EN2 can functionally substitute EN1 in the dopaminergic neurons. However, further studies would be required to test the efficacy and safety of targeting EN1 in fibrotic diseases. In summary, we demonstrate that EN1, beyond its function as an embryonic marker for a population of metabolically active fibroblasts during embryonic development, is also expressed by multiple fibroblast subpopulations in the skin of SSc patients and functionally regulates fibroblast-to-myofibroblast transition in adult skin. EN1 interacts with SP1 (and presumably other members of the SP/C2H2 family of transcription factors) upon stimulation with TGF\\u03b2 and enhances its binding to the promoter of profibrotic genes related to the microtubule cytoskeleton and Rho GTPases to promote depolymerization of microtubules. This induces stress fiber formation, myofibroblast differentiation, and collagen release in a ROCK-dependent manner. In contrast, EN1-deficient fibroblasts are less sensitive to the profibrotic effects of TGF\\u03b2. Moreover, fibroblast-specific knockout of En1 ameliorates experimental skin fibrosis in several complementary models. As EN1 is up-regulated in fibrotic diseases in a TGF\\u03b2-dependent manner, it functions as a molecular amplifier of TGF\\u03b2 signaling and TGF\\u03b2-induced fibroblast differentiation. Targeted inactivation of EN1 may thus offer potential to inhibit TGF\\u03b2 signaling and fibroblast activation in fibrotic diseases.\",\n",
            "        \"conclusion\": null\n",
            "    },\n",
            "    {\n",
            "        \"pmcid\": \"PMC8382172\",\n",
            "        \"abstract\": \"Children with severe acute malnutrition (SAM) remain vulnerable after treatment at nutritional rehabilitation units (NRUs). The objective was to assess the concurrent pathways in a hypothesized model between caregiver body mass index (BMI), the home environment, and child nutritional status, and development (gross motor, fine motor, language, and social domains) in children with SAM following discharge from inpatient treatment. Structural equation modelling (SEM) was performed with data from a cluster-randomized controlled trial at the Moyo Nutritional Rehabilitation and Research Unit in Blantyre, Malawi. This approach was undertaken to explore simultaneous relationships between caregiver BMI, the home environment (Home Observation for Measurement of the Environment Inventory scores), child nutritional status (anthropometric indicators including weight-for-age z-scores [WAZ]), and child development (Malawi Developmental Assessment Tool (MDAT) z-scores as a latent variable) in children with SAM. These data were collected at participants\\u2019 homes six months after discharge from NRU treatment. This analysis included 85 children aged 6\\u201359 months with SAM and their caregivers recruited to the trial at the NRU and followed up successfully six months after discharge. The model with WAZ as the nutritional indicator fit the data according to model fit indices (\\u03c7\",\n",
            "        \"intro\": \"Introduction In low- and middle-income countries, an estimated 250 million children are unlikely to achieve optimal cognitive, motor, and socioemotional development [ 1 ]. Children are at high risk of compromised development if they live in poverty, are exposed to an unstimulating environment, and experience acute illness or malnutrition [ 2 \\u2013 4 ]. Children with complicated severe acute malnutrition (SAM) are often subjected to many of these risk factors. When children have SAM in combination with an acute illness, loss of appetite, or severe oedema, they require inpatient treatment at nutritional rehabilitation units (NRUs) according to the Community Based Management of Acute Malnutrition guidelines [ 5 ]. Aside from an increased risk of mortality within two years after discharge from inpatient treatment, these children have poor nutritional outcomes including low height-for-age z-score (HAZ), representative of stunting, and inhibited gross motor, fine motor, language, and social development [ 6 \\u2013 9 ]. There is strong evidence supporting the association between HAZ and child development [ 1 , 2 , 10 \\u2013 12 ]. A systematic review examining the effects of nutritional supplementation and caregiving interventions have shown that improvements in HAZ are associated with better developmental outcomes, while caregiving interventions lead to better developmental outcomes but not higher HAZ [ 13 ]. The pathways between nutritional status and developmental outcomes require further exploration particularly in children with SAM. There is a particular need to understand how different anthropometric indicators like WHZ and weight-for-age z-scores (WAZ) in children with SAM predict child development. For children to reach their developmental potential, care practices that promote health, nutrition, and early learning of children are essential. Therefore, caregivers are central in this narrative [ 14 \\u2013 17 ]. Based on qualitative research conducted in Malawi, Gladstone et al (2018) identified that caregiver well-being and care practices are important to achieve better child health and development [ 18 ]. Another qualitative study done in Malawi examined perceptions and experiences of caregivers of children admitted for inpatient treatment of SAM [ 19 ]. This study revealed a number of themes reported by caregivers that relate to difficulties in caring for their children at home, including caregiver well-being [ 19 ]. Yet specific caregiver factors such as nutritional status, which could relate to well-being, has not been a strong focus of research in caregivers of children with SAM. We hypothesized that maternal nutritional status according to body mass index (BMI) could be associated with factors such as income and the number of children in the household which could influence care practices. Because there are many social and nutritional factors that influence developmental outcomes in children with SAM, analytic techniques that consider simultaneous pathways are advantageous. Structural equation modelling (SEM) is a type of analysis that confirms and measures structural pathways between measured and latent variables. The objective of this analysis was to evaluate concurrent pathways between caregiver body mass index (BMI), the home environment, child nutritional status according to multiple different anthropometric indicators, and development in children with SAM in the home setting following discharge from inpatient treatment. An additional objective was to understand which anthropometric indicator is most predictive of child development in terms of gross motor, fine motor, language, and social domains within this SEM. This will allow for a deeper understanding of mechanistic pathways and potential areas to intervene with the goal of having the most influence on developmental outcomes in children with SAM.\",\n",
            "        \"method\": \"Materials and methods This analysis was conducted with data from a cluster-randomized controlled trial evaluating the effectiveness of a hospital-based interactive counselling intervention, known as the Kusamala Program, to improve gross motor, fine motor, language, and social development in children admitted for inpatient treatment of SAM [ 20 , 21 ]. The trial was conducted at the Moyo NRU at Queen Elizabeth Central Hospital in Blantyre, Malawi. Ethical approval for this study was obtained from the University of Malawi College of Medicine Research Ethics Committee (P.04/16/1930) and the Hospital for Sick Children Research Ethics Board (1000053578). Written or verbal informed consent was obtained from all subjects/patients. Verbal consent was witnessed and formally recorded. The protocol for this trial was published and can be accessed for more details [ 20 ]. Trial inclusion criteria Child 6\\u201359 months of age with SAM, identified by bilateral pitting oedema, WHZ below -3 SD, and/or MUAC below 115 mm; Child admitted to hospital because of SAM with medical complications or appetite-loss per World Health Organization (WHO) guidelines [ 22 ]; Primary caregiver (self-identified) present at hospital. Trial exclusion criteria Primary caregiver declined to give informed consent; Child with a known terminal illness; Child requiring a surgical procedure. An additional exclusion criterion was applied in that children with identified neurodisability were omitted from the analysis per the original trial protocol because of the influence on developmental scores [ 20 ]. Furthermore, caregivers and children who were lost to follow-up were not included in the main analysis, but their baseline characteristics were examined in comparison to caregivers and children with follow-up data. Screening and informed consent procedures, including written or verbal consent in the presence of a witness in the case that caregivers were unable to read or write, were completed by trained research staff members. Children and their caregivers were recruited and enrolled to the trial within three days of admission. Groups of one to six children and their caregivers were randomized each week to an intervention arm in which caregivers participated in the Kusamala Program or to a comparison arm with no intervention beyond the current practices in the NRU including basic nutrition and WASH counselling at the time of discharge [ 20 ]. Data collection and variables for hypothesized pathway model Data for the cluster-randomized controlled trial were collected at the time of enrollment, discharge from hospital, and at follow-up six months after discharge at the homes of caregivers and children. Baseline data included household and caregiver characteristics as well as child characteristics and anthropometric indices calculated per the 2006 WHO Child Growth Standards [ 23 ]. Caregiver and child weight and height were measured in duplicate, and if there were discrepancies in these two measurements then a third measurement was taken. An a priori hypothesized pathway model was developed with specific pathways leading to child development with all data in the model from follow-up at six months after discharge ( Fig 1 ). Specific variables in the model were pre-selected as they were hypothesized to relate to developmental outcomes directly or indirectly in children with SAM. Caregiver BMI is the exogenous variable in the model leading to the home environment and to child nutritional status. This analysis evaluated four different anthropometric indicators of nutritional status in different versions of the SEM. WHZ and MUAC are both indicators of wasting [ 23 ]. WAZ, an indicator of underweight status, was also examined [ 23 ]. HAZ was computed to indicate linear growth relative to the child\\u2019s age [ 23 ]. 10.1371/journal.pone.0255967.g001 Fig 1 Hypothesized pathways between caregiver BMI, the home environment, child nutritional status and development in children with SAM. Caregiver body mass index (bmi); home environment according to the Home Observation for Measurement of the Environment Inventory; child nutritional status according to weight-for-height z-scores (whz), weight-for-age z-scores (waz), height-for-age z-scores (haz), or mid-upper arm circumference (muac); and development according to MDAT (Malawi Developmental Assessment Tool) z-scores. The Home Observation for Measurement of the Environment (HOME) Inventory, which has been widely used in low- and middle-income settings, was adapted for relevance to this particular study population and included 18 observations of care practices and seven interview questions answered by caregivers [ 24 , 25 ]. The HOME Inventory used included 25 items evaluating responsivity (11 items), language stimulation (2 items), acceptance (6 items), learning materials (1 item), involvement (3 items), and variety (2 items) [ 24 , 25 ]. The home environment and child nutritional status are both hypothesized to be directly linked to child development as the main outcome. Child development is represented by a combination of developmental domains as a latent variable including gross motor, fine motor, language, and social development z-scores according to the Malawi Developmental Assessment Tool (MDAT) [ 26 ]. The MDAT is a culturally validated measure of child development specifically for use in low-resource African settings and has been used in several recent studies [ 26 \\u2013 29 ]. It included 36 items in each of four domains including gross motor, fine motor, language, and social development. The MDAT has good reliability for all domains and is sensitive to nutritional status as indicated by 28% of malnourished children passing the test using a pass/fail technique compared to 94% of non-malnourished passing in a previous validation study [ 26 ]. MDAT z-scores for each respective domain were calculated by comparing scores to a reference population in Malawi [ 26 ]. Statistical analysis All data were entered into a Research Electronic Data Capture database and subsequently analyzed in Stata 16 (Statacorp LP, College Station, Texas, USA) [ 30 , 31 ]. Descriptive statistics were computed including means (SD) or median (interquartile range) for continuous variables as appropriate, and proportions and percentages for categorical variables. The zscore06 macro was used in Stata 16 to calculate anthropometric z-scores per the 2006 WHO Child Growth Standards [ 23 , 32 ]. MDAT z-scores were computed based on a reference population of Malawian children [ 26 , 33 ]. HAZ and MDAT z-scores were considered valid if they were within -6 and +6 SD, though for WHZ and WAZ they were considered between -6 SD and +5 SD [ 23 ]. Characteristics of children who died or were lost to follow-up were compared to surviving children who were successfully followed up using one-sample t-tests for continuous variables and Pearson\\u2019s \\u03c7 2 test for categorical variables. P-values below 0.05 were considered statistically significant for all analyses. Children who died during inpatient treatment or within six months after discharge were excluded from the main analysis for this paper which examined follow-up data only. Multiple linear regression was done to characterize and understand factors related to caregiver BMI as the exogenous variable in the hypothesized pathway model. Predictor variables included monthly household income, caregiver age, caregiver height, and number of children in the household. Standardized beta-coefficients were examined for this analysis. This analysis included all participants followed up in the first three years of the cluster-randomized controlled trial, between November 2016 and November 2019, to reach a target sample size exceeding 70 children and caregivers with follow-up data for ten observations per variable in the SEM [ 34 \\u2013 36 ]. Q\\u2013Q plots were examined for each variable in the SEM to check that the residuals were normally distributed ( S1 Fig ). SEM was then done to evaluate the hypothesized pathway model, which first involved confirmatory factor analysis (CFA) of MDAT as a latent variable. The full information maximum likelihood method was used for the SEM to account for missing data. Covariates in the model included child human immunodeficiency virus (HIV) status, sex, and age. Standardized beta-coefficients were computed for all pathways and their direct and indirect effects were calculated. Furthermore, simple linear regression was done to assess the relationships between individual variables in the SEM. Model fit for the SEM was examined using the \\u03c7 2 test statistic (\\u03c7 2 p-value >0.05 acceptable, indicating no statistically significant difference between the hypothesized and fitted model), comparative fit index (CFI >0.90 represents good fit), Tucker-Lewis index (TLI >0.90 represents good fit), and root mean square error for approximation (RMSEA <0.05 represents good fit, <0.08 acceptable fit) [ 37 \\u2013 39 ]. Bootstrap estimates and confidence intervals were also evaluated by completing SEM on the estimation sample to understand the stability of the model and to further verify that the sample size was adequate. Reporting guidelines The Strengthening the Reporting of Observational Studies in Epidemiology statement was followed in the reporting of this manuscript [ 40 ].\",\n",
            "        \"subjects\": null,\n",
            "        \"results\": \"Results After screening for eligibility, 170 children and their caregivers were enrolled in the first three years of the cluster-randomized controlled trial ( S2 Fig ). Eight participants were excluded because they were deemed ineligible for the trial after enrollment (3/170, 1.8%) or withdrew from the study (5/170, 3.0%). Twenty-seven children (27/162, 16.7%) were excluded from the analysis due to pre-existing neurodisability. Of the remaining children, there were 18 inpatient deaths (18/135, 13.3%). In children that survived inpatient treatment, there were 17 losses to follow-up (17/117, 14.5%) in addition to 15 outpatient deaths (15/100, 15.0%). This left 85 children and their caregivers available for analysis. The mean follow-up time was 6.4 months after discharge. A small proportion of data were missing for certain variables ( S1 Table ). Baseline characteristics Child, caregiver, and household characteristics at baseline are summarized in Table 1 . Children who died had lower WHZ compared to those who survived and were successfully followed up (-4.0 \\u00b1 0.8 versus -3.4 \\u00b1 1.2, p = 0.046). The proportion of children who died that had HIV tended to be higher compared to surviving children with follow-up data (38.7% versus 26.5%, p = 0.2). WAZ also appeared lower in children who died compared to those that survived and were followed up (-4.6 \\u00b1 0.8 versus -3.9 \\u00b1 1.4, p = 0.10). Baseline characteristics between participants successfully followed up and those lost to follow-up were similar. However, caregivers that were lost to follow-up had a lower average number of children in the household compared to those that were followed up (2.2 \\u00b1 1.3 versus 3.0 \\u00b1 1.4, p = 0.03). 10.1371/journal.pone.0255967.t001 Table 1 Baseline characteristics of children admitted for inpatient treatment of SAM and their caregivers and households. Children with follow-up data (n = 85) Children who died (n = 33) Children lost to follow-up (n = 17) Age (months), mean (SD) 19.5 (9.8) 19.8 (12.6) 22.8 (12.7) Sex female (%) 41/85 (48.2) 17/33 (51.5) 8/17 (47.1) HIV positive (%) 22/83 (26.5) 12/31 (38.7) 6/16 (37.5) Previous inpatient admission (%) 12/84 (14.3) 5/30 (16.7) 2/16 (12.5) Oedema (%) 28/85 (32.9) 10/32 (31.3) 9/16 (56.3) MUAC (cm) a , mean (SD) 10.9 (1.1) 10.5 (1.0) 10.5 (0.9) WHZ a , mean (SD) -3.4 (1.2) -4.0 (0.8) -3.5 (1.3) HAZ, mean (SD) -3.1 (1.3) -3.1 (1.8) -2.6 (2.2) WAZ a , mean (SD) -3.9 (1.4) -4.6 (0.8) -4.3 (0.8) Caregivers with follow-up data (n = 85) Caregivers of children who died (n = 33) Caregivers lost to follow-up (n = 17) Relationship to child (%) Mother 80/85 (94.1) 31/32 (96.9) 16/17 (94.1) Other 5/85 (5.9) 1/32 (3.1) 1/17 (5.9) Age (years), mean (SD) 28.0 (7.8) 25.8 (6.7) 24.7 (6.3) MUAC (cm), mean (SD) 26.1 (3.5) 26.1 (2.6) 25.0 (2.3) Body mass index (kg/m 2 ), mean (SD) 22.3 (4.2) 22.3 (2.7) 21.5 (2.6) Caregiver education (%) Preschool 18/85 (21.2) 2/32 (6.3) 1/17 (5.9) Primary 41/85 (48.2) 20/32 (62.5) 10/17 (58.8) Secondary 26/85 (30.6) 9/32 (28.1) 6/17 (35.3) Higher 0/85 (0) 1/32 (3.1) 0 (0) Caregiver depressive symptoms (Self-Reporting Questionnaire 20), median (IQR) 3.0 (7.0) 3.5 (7.5) 3.0 (8.5) Monthly household income (Malawi kwacha), mean (SD) 20451 (24661) 16015 (11812) 24812 (35501) Number of children in the household, mean (SD) 3.0 (1.4) 2.9 (1.5) 2.2 (1.3) Household area (%) Urban 50/85 (58.8) 22/32 (68.8) 14/17 (82.4) Rural 35/85 (41.2) 10/32 (31.3) 3/17 (17.7) HAZ, height-for-age z-score. HIV, human immunodeficiency virus. MUAC, mid-upper arm circumference. SAM, severe acute malnutrition. SD, standard deviation. WAZ, weight-for-age z-score. WHZ, weight-for-height z-score. a Excluding children with oedema. Caregiver body mass index Multiple linear regression results showed no significant relationships for monthly household income (\\u03b2 = 0.22, p = 0.064), caregiver age (\\u03b2 = -0.026, p = 0.83), caregiver height (\\u03b2 = -0.14, p = 0.23), or the number of children in the household (\\u03b2 = 0.17, p = 0.16) as predictors of caregiver BMI. Latent variable of child development Child development represented as a latent variable of MDAT z-scores showed acceptable model fit based on the CFA (\\u03c7 2 = 3.21, p = 0.20, CFI = 0.994, TLI = 0.983, RMSEA = 0.085). All four MDAT domains had significant factor loadings in the measurement model (\\u03b2 = 0.80, p<0.001 for gross motor, \\u03b2 = 0.87, p<0.001 for fine motor, \\u03b2 = 0.84, p<0.001 for language, and \\u03b2 = 0.92, p<0.001 for social). Structural equation modelling results The model with WHZ as the nutritional indicator fit the data based on the various fit indices (\\u03c7 2 = 33.99, p = 0.20, CFI = 0.976, TLI = 0.962, RMSEA = 0.050) ( Fig 2 and Table 2 ). All pathways in the model were significant except from caregiver BMI to child WHZ (\\u03b2 = 0.15, p = 0.21). There was also an indirect pathway from caregiver BMI to MDAT z-scores (\\u03b2 = 0.12, p = 0.046), mediated by the child WHZ and home environment variables. The results of the SEM remained the same with bootstrapping, demonstrating the stability of the model. 10.1371/journal.pone.0255967.g002 Fig 2 Pathways between caregiver BMI, the home environment, child nutritional status according to WHZ and development in children with SAM. Caregiver body mass index (bmi); home environment according to the Home Observation for Measurement of the Environment Inventory; child nutritional status according to weight-for-height z-scores (whz); and development according to MDAT (Malawi Developmental Assessment Tool) z-scores. Estimates represent standardized beta-coefficients. Analysis adjusted for child HIV status, sex, and age. *p<0.05, **p<0.01, ***p<0.001. 10.1371/journal.pone.0255967.t002 Table 2 Standardized beta-coefficients and bootstrap results for direct pathways between caregiver BMI, the home environment, child nutritional status according to WHZ, and development in children with SAM. Structural equation modelling results Bootstrap results Pathways \\u03b2 P-values 95% confidence interval \\u03b2 P-values 95% confidence interval home environment caregiver bmi 0.23 0.03 0.027, 0.44 0.23 0.03 0.018, 0.46 child whz caregiver bmi 0.15 0.21 -0.086, 0.38 0.15 0.20 -0.078, 0.37 MDAT home environment 0.33 0.001 0.14, 0.52 0.33 0.001 0.14, 0.51 child whz 0.32 0.001 0.13, 0.51 0.32 0.003 0.11, 0.53 \\u03b2, beta-coefficient (standardized). bmi, body mass index. MDAT, Malawi Developmental Assessment Tool. whz, weight-for-height z-score. Analysis adjusted for child HIV status, sex, and age. SEM was then done to evaluate the hypothesized pathway model with WAZ as an indicator of nutritional status in place of WHZ ( Fig 3 and Table 3 ). This model also fit the data well (\\u03c7 2 = 31.89, p = 0.28, CFI = 0.985, TLI = 0.975, RMSEA = 0.041). In this SEM, all pathways were significant, including an indirect pathway between caregiver BMI and the MDAT latent variable (\\u03b2 = 0.19, p = 0.008). Results were similar upon completing bootstrapping indicating model stability. 10.1371/journal.pone.0255967.g003 Fig 3 Pathways between caregiver BMI, the home environment, child nutritional status according to WAZ, and development in children with SAM. Caregiver body mass index (bmi); home environment according to the Home Observation for Measurement of the Environment Inventory; child nutritional status according to weight-for-age z-scores (waz); and development according to MDAT (Malawi Developmental Assessment Tool) z-scores. Estimates represent standardized beta-coefficients. Analysis adjusted for child HIV status, sex, and age. *p<0.05, **p<0.01, ***p<0.001. 10.1371/journal.pone.0255967.t003 Table 3 Standardized beta-coefficients and bootstrap results for direct pathways between caregiver BMI, the home environment, child nutritional status according to WAZ, and development in children with SAM. Structural equation modelling results Bootstrap results Pathways \\u03b2 P-values 95% confidence interval \\u03b2 P-values 95% confidence interval home environment caregiver bmi 0.23 0.03 0.029, 0.44 0.23 0.008 0.061, 0.46 child waz caregiver bmi 0.30 0.005 0.089, 0.52 0.30 0.005 0.094, 0.51 MDAT home environment 0.32 0.001 0.13, 0.51 0.32 0.002 0.12, 0.52 child waz 0.37 <0.001 0.18, 0.55 0.37 0.001 0.16, 0.58 \\u03b2, beta-coefficient (standardized). bmi, body mass index. MDAT, Malawi Developmental Assessment Tool. waz, weight-for-age z-score. Analysis adjusted for child HIV status, sex, and age. The third version of SEM was completed using HAZ as an indicator of nutritional status ( Fig 4 and Table 4 ). This version of the model also fit the data based on the model fit indices (\\u03c7 2 = 28.92, p = 0.42, CFI = 0.996, TLI = 0.994, RMSEA = 0.020). All pathways were significant, including the indirect pathway from caregiver BMI to the MDAT latent variable (\\u03b2 = 0.15, p = 0.014), and bootstrapping indicated that the model was stable. 10.1371/journal.pone.0255967.g004 Fig 4 Pathways between caregiver BMI, the home environment, child nutritional status according to HAZ, and development in children with SAM. Caregiver body mass index (bmi); home environment according to the Home Observation for Measurement of the Environment Inventory; child nutritional status according to height-for-age z-scores (haz); and development according to MDAT (Malawi Developmental Assessment Tool) z-scores. Estimates represent standardized beta-coefficients. Analysis adjusted for child HIV status, sex, and age. *p<0.05, **p<0.01, ***p<0.001. 10.1371/journal.pone.0255967.t004 Table 4 Standardized beta-coefficients and bootstrap results for direct pathways between caregiver BMI, the home environment, child nutritional status according to HAZ, and development in children with SAM. Structural equation modelling results Bootstrap results Pathways \\u03b2 P-values 95% confidence interval \\u03b2 P-values 95% confidence interval home environment 0.23 0.03 0.029, 0.44 0.23 0.01 0.050, 0.41 caregiver bmi child haz 0.23 0.03 0.024, 0.44 0.23 0.008 0.060, 0.41 caregiver bmi MDAT home environment child haz 0.32 0.001 0.14, 0.51 0.32 0.002 0.12, 0.53 0.34 <0.001 0.16, 0.53 0.34 0.02 0.047, 0.64 \\u03b2, beta-coefficient (standardized). bmi, body mass index. haz, height-for-age z-score. MDAT, Malawi Developmental Assessment Tool. Analysis adjusted for child HIV status, sex, and age. Lastly, MUAC was evaluated as an indicator of nutritional status, yet this SEM differed from the other three versions in that there was no direct relationship between child MUAC and the MDAT latent variable (\\u03b2 = 0.14, p = 0.19) ( S3 Fig and S2 Table ). There was a small indirect relationship between caregiver BMI and the MDAT latent variable (\\u03b2 = 0.10, p = 0.043). Summary statistics of the variables in the SEM are included in S3 Table and simple linear regression with individual pathways are shown in S4 Table .\",\n",
            "        \"discussion\": null,\n",
            "        \"conclusion\": \"Conclusions This analysis showed the pathways and their relative importance between caregiver BMI, the home environment, and nutritional status in predicting MDAT z-scores of gross motor, fine motor, language, and social domains in children with SAM. Child WAZ was the strongest of the anthropometric indices in predicting MDAT z-scores. Interventions aimed at improving developmental trajectories of these children following discharge from inpatient treatment at NRUs should also consider addressing caregiver nutritional status. Future research should be done to evaluate the effects of social and nutritional interventions on these pathways leading to development in children with SAM.\"\n",
            "    },\n",
            "    {\n",
            "        \"pmcid\": \"PMC8976245\",\n",
            "        \"abstract\": null,\n",
            "        \"intro\": \"Introduction Background The COVID-19 pandemic has accelerated widespread adoption of digital health interventions internationally. This rapid shift to digital delivery has laid bare the impact of pre-existing and emerging systemic health inequalities on communities most in need of accessible health care, specifically people from ethnic minority backgrounds, people from lower socioeconomic backgrounds, older people (aged \\u226565), and people living with disabilities [ 1 , 2 ]. Belonging to one or more of these groups (intersectionality) is a risk factor for experiencing more severe illness and mortality [ 3 ]. These same groups are likely to make up a large proportion of people vulnerable to health inequalities regardless of initial illness severity [ 4 ]. Health inequalities have been defined as \\u201cthe systematic, avoidable and unfair differences in health outcomes that can be observed between populations, between social groups within the same population or as a gradient across a population ranked by social position\\u201d [ 5 ], but the term is also used for differences in access to health care, quality of care received, wider determinants of health such as housing and education, and opportunities to lead healthy lives, including differences in risky behaviors such as smoking [ 6 ]. The related term \\u201chealth inequity\\u201d implies a normative judgement about the fairness or otherwise of these differences, \\u201cexpressing a moral commitment to social justice,\\u201d as described by Kawachi et al [ 7 ]. For this review, we use the term health inequalities. The digital divide\\u2014the gap between populations able to benefit from access to and use of health information and services online and populations unable to take up such opportunities\\u2014is a clear example of health inequality and exacerbates the inverse care law [ 8 ] such that digitally delivered health care runs the risk of excluding the people who could most benefit. This review seeks to address issues of social justice in the digital delivery of health care by providing a comprehensive overview of the literature on strategies to reduce the digital divide through the design and deployment aspects of digital health interventions. The Digital Divide and Digital Health Literacy Health literacy and digital literacy are both key determinants of health [ 9 ]. The term digital health literacy (also referred to as eHealth literacy) brings both literacies together to describe the degree to which individuals can access, understand, and apply digitally delivered health information and services to make informed decisions about their health. Importantly, digital health literacy extends beyond personal responsibility to encompass the responsibility of digital health systems and services to support and dynamically respond to the digital health literacy skills, and the confidence and motivation to develop such skills, in the populations they serve [ 10 ]. Digital health literacy competence fluctuates depending on context, but a clear link has been demonstrated between low digital health literacy and poor health outcomes [ 11 ]. The accelerated shift to digital health care to comply with social distancing measures arising from COVID-19 heightens the risk of poor health outcomes in digitally excluded groups and further complicates an already complex interplay between intersectionality and digital health literacy as a distributed, or outsourced , concept. For example, many older people may have the financial capital to afford digitally enabled devices and data plans but not the skills, motivation, or confidence to engage directly with such technology; instead, they may be reliant on their networks\\u2014family members, friends\\u2014to navigate digital health care on their behalf [ 12 , 13 ]. Digital health intervention design and deployment therefore need to be proportionate and prioritize those groups most affected by low digital health literacy on both individual and systemic levels via a two-pronged approach: first, to develop patients\\u2019 skills in accessing, understanding, and using digital health information and services, and second, to develop the digital health literacy responsiveness of the systems and health care professionals (HCPs) supporting digital health intervention deployment in practice. This review will have wider implications for research into digital health interventions and will address: The digital divide: How should digital health interventions be designed and deployed to mitigate the digital divide (eg, patient access to WiFi-enabled digital devices or subsidized data plans, the inclusiveness of recruitment methods for studies evaluating digital health interventions, strategies to increase uptake and use of a digital health intervention app by people from demographic groups likely to be affected by the digital divide, or the content of onboarding scripts used by HCPs when introducing patients to a digitally delivered health service)? Digital health literacy: How can app design and digital skills training be optimized to reduce digital health inequalities arising from low digital health literacy (eg, integration of user testing into the design process, readability of app content, gamification/social features to engage users, or support for HCPs to train as \\u201cdigital health champions\\u201d as part of improving their own skills)? Rationale for Conducting a Systematic Scoping Review The decision to conduct a systematic scoping review rather than a systematic review is informed by the purpose of scoping reviews, which is to provide a comprehensive picture of the available evidence to guide further research. This purpose is of relevance to the study of digital health interventions because digital health intervention design and deployment is frequently captured in gray literature (eg, charity reports). This review will be conducted using methods outlined by the Joanna Briggs Institute (JBI) Reviewers\\u2019 Manual [ 14 ] and will conform to the PRISMA-ScR (Preferred Reporting Items for Systematic Reviews and Meta-Analyses Extension for Scoping Reviews) [ 15 ]. The protocol for this review is based on the JBI framework by Peters et al [ 16 ] with further process updates from Levac et al [ 17 ] and is structured as follows: identification of the review aims and search questions, selection of evidence sources, charting of data extracted from the evidence sources, reporting of results, and consultation with stakeholders (including HCPs, digital health experts, and digital health intervention users). Results from this review will support decision-making when designing and deploying digital health interventions. Review Question The purpose of this systematic scoping review is to identify research that reports on the design and deployment of digital health interventions to reduce the digital divide and increase digital health literacy at macro- (national policy), meso- (national program), and micro- (localized or individual) levels [ 18 ]. It aims to do this by discovering peer-reviewed qualitative, quantitative, and mixed methods research, and gray literature relevant to mapping the contextual and process factors that enable or hinder engagement with digital health services by people vulnerable to the digital divide and individual or systemic low digital health literacy. This review will seek to answer the review question, \\u201cWhat features of digital health intervention design and deployment enable or hinder engagement with digital health interventions by people from demographic groups likely to be affected by the digital divide?\\u201d Health interventions are increasingly delivered through digital platforms, and it is important that they do not exacerbate or create health inequalities. Our hypothesis is that existing knowledge in the literature can inform our objective to find ways to bridge the digital divide and improve digital health literacy in underserved groups, for example, through digital health intervention design, development, and deployment. Figure 1 outlines how this review will be conducted. Figure 1 Systematic scoping review flow diagram. DHI: digital health intervention.\",\n",
            "        \"method\": \"Methods We will follow the methods recommended by the PRISMA-ScR framework [ 15 ]. Protocol Registration At the time of writing, scoping reviews are ineligible for registration in PROSPERO. For transparency and to enable the systematic scoping review method outlined here to be repurposed for onward research, a summary of the protocol and any supplementary material will be registered with the Open Science Framework [ 19 ] and assigned a Digital Object Identifier (DOI) for long-term retrieval. Eligibility Criteria We will include primary research studies reporting on the design or deployment of digital health interventions that meet the following inclusion criteria. Population Adults from different demographic groups (defined based on gender, age, ethnicity, income, and education) in high-income and low-income countries. Intervention Digital health interventions are defined as any service intended to improve physical or mental health, or to promote health improvement through, for example, lifestyle change delivered digitally (formally or informally), such as via smartphone apps, social media, email, SMS text message, using wearable technologies, video games (eg, for motor or cognitive training), websites, or telehealth (eg, remote consultations) but excluding telemedicine if this consists solely of remote monitoring without any input from the patient [ 15 ]. Comparator There may or may not be a comparator, depending on the study design. Outcome The primary outcome of interest is the capacity to mitigate the digital divide and increase digital health literacy either through the digital health intervention itself or its deployment. Health impact is not considered an outcome of interest for this review. \\u201cMitigate the digital divide\\u201d refers to removing barriers to digital inclusion, for example, an intervention that provides free smartphones or tablets, or low-cost cellular data or WiFi for people with low incomes [ 10 ]. \\u201cIncrease digital health literacy\\u201d refers to digital skills development [ 9 ], either through the design of the digital health intervention or through programs to deploy the digital health intervention, resulting in higher levels or use or confidence in using the digital health interventions, measured either by interview or self-report (eg, the eHealth Literacy Scale questionnaire [ 20 ] or use data). Changes in these outcome measures suggest that both users and the health professionals administering the digital health intervention are confident in the use of digital resources for safe and sustainable self-management and to promote safe and sustainable self-management. For the purposes of this review, gray literature covers conference abstracts and proceedings, white papers, and stakeholder reports (eg, by charities with an interest in digital literacy or who promote the health and well-being of people from different demographic groups) that enhance the contextual understanding of the field. There will be no limitation by geography because lessons from low-income countries could inform strategies to reduce digital health inequities in high-income countries. We will report and interpret data within the context of the country where the study was based. Limitations will be placed on user population and on study date. The user population will prioritize the marginalized demographics previously outlined; the study date will be limited to post-2011 (for preprint or publication) in recognition of the rapid change of pace in digital health intervention development [ 21 ]. No limitations will be placed on the study design or publication status. However, we will limit included studies to those published in English. Decisions on whether data should be included or excluded from further analysis will be guided by criteria that, in line with a systematic application of scoping review method [ 16 ], will be iteratively refined based on increasing familiarity with the literature under review. The use of the term digital health intervention is a possible challenge as it covers a range of digital health technologies [ 22 ] and may contribute to an imbalance in this review that favors comprehensiveness over precision. This will be addressed by working closely with information specialists to peer review the preliminary search strategy in line with the PRESS (Peer Review of Electronic Search Strategies) procedure [ 23 ], ensuring a feasible and focused approach that is also flexible enough to be iteratively reworked in response to the results retrieved. Evidence Sources Reviews A preliminary search for registered, preprint, or published systematic, scoping, and other review types will be conducted to pilot the search strategy, covering the following databases (via the Ovid interface, where applicable): Cochrane Library Epistemonikos NICE Evidence PROSPERO PubMed (with MEDLINE and Europe PMC) Trip The preliminary search will provide keywords that can be incorporated into searches across evidence tiers below that of the gold standard represented by reviews. It will also serve to identify Medical Subject Headings (MeSH) and PubMed IDs, which can be inserted into the Yale MeSH Analyzer [ 24 ] to extract further MeSH and to identify keywords to capture preprints not yet indexed with MeSH. A cap of 100 review articles will be applied in cases where unfeasible amounts of hits are retrieved. Model papers for citation \\u201c pearl-growing \\u201d\\u2014papers already identified as highly relevant to this review\\u2014will be mined for their keywords and applied as test cases for honing the search strategy. Vernacular search terms will be identified from hand searching, including reference list scanning, forward/backward citation snowballing, and table of contents scanning of relevant journals online, such as the Journal of Medical Internet Research and its sister journals. Further sources for other tiers of evidence include: Databases of peer-reviewed primary research (qualitative, quantitative, mixed methods studies): CINAHL EMBASE PsycINFO OTseeker PubMed Gray literature Conference Proceedings Citation Index Health Management Information Consortium International HTA Database OpenGrey GreyNet The Grey Literature Report The Grey Literature Report [ 25 ], available up to 2016, is a key source of gray literature that, unusually, is indexed using MeSH. It may be helpful in providing alternative MeSH to feed back into the search strand for peer-reviewed literature. Alongside database searching, web searching will be conducted systematically, to the extent that this is feasible given that web searching is vulnerable to changing web content and algorithms [ 26 ]. Search terms will be used consistently between database and web searching and screen captures of content, and where available, DOIs will be saved for transparency. Web searching will also support the discovery of human-computer interaction (HCI) literature, which is not reliably indexed in MEDLINE. Searching the HCI literature will be helpful for the provision of further perspectives on design challenges (eg, designing for people with low vision or motor impairments). Web Searching The following databases will be used for the web searching: Google Scholar Basic Search UK, with use of the \\u201cCited by\\u201d function (see Figure 2 ), for the first 100 results retrieved; MedNar Deep Web Search Engine, for the first 100 results retrieved; and Carrot2. Targeted consultation with experts in digital health intervention design and deployment and digital health intervention users will also be conducted in parallel. This is part of the added value of scoping reviews. Ethics approval is not required for such consultation. Figure 2 \\\"Cited by\\\" feature for forward citation snowballing in Google Scholar. Search Strategy The following search strategy was used for PubMed (November 2021): ((\\u201cdigital health\\u201d OR \\u201cdigital health intervention*\\u201d OR \\u201cDHI\\u201d OR \\u201cdigital rehabilitation\\u201d) OR \\u201ceHealth\\u201d OR \\u201cmHealth\\u201d AND (app OR intervention OR technology) AND (barrier* OR design OR disparit* OR divi* OR engage* OR exclu* OR inequ*)) AND literacy. This search strategy will be modified by incorporating MeSH and keywords used to index relevant papers and will inform the search strings for parallel searching in Google Scholar Advanced Search UK and MedNar. Screening of identified literature will follow the recommendations of the JBI [ 16 ]. Of the studies and gray literature initially identified from the searches, 25 publications will be selected at random by dividing the total by four (the number of screeners), then selecting every nth paper (where n=the total divided by four) to assign for double-screening of titles, abstracts, or full-text by four reviewers; if there is greater than 75% agreement for included/excluded publications, then the team will continue to screen all of the identified publications. If agreement is 75% or less, we will hold a further training session to resolve differences in interpretation of the criteria and, if necessary, tighten the wording. Further rounds of double-screening and calculation of interrater agreement will continue until we meet concordance. For included publications, study characteristics will be extracted by one author and checked for agreement by a second author, and any uncertainty over data retrieved will be discussed and resolved via consensus with all contributing authors. Retrieval of a model paper [ 27 ] served as the test case for the search strategy. PRISMA-ScR The PRISMA-ScR framework will be used to transparently record the selection, deduplication, and screening decision process, supplemented by updated recommendations from Tricco et al [ 28 ]. Gray literature may not be searchable by abstract (due to a lack of abstract fields in the gray literature discovery interfaces), so the PRISMA-ScR flow diagram will be adapted to accommodate this. Critical appraisal, for example, of the underlying evidence informing the studies retrieved, is not generally operationalized in scoping reviews [ 16 ]. Instead, the focus of this review type is on providing as complete a picture as possible of what is currently known about the topic of interest. Data Extraction and Charting Data will be extracted from full-text sources relevant to the review question using the JBI data extraction guidance for scoping reviews [ 14 ]. If required, missing data will be requested from authors. Google Sheets [ 29 ] will be used to record the initial data extraction of all included sources. In accordance with the iterative approach taken by scoping reviews, the following headers will be trialed and refined as needed [ 30 ]: Reference Region and setting of digital health intervention design/deployment Study type (qualitative, quantitative, mixed methods, review) Funding source Stakeholder involvement Digital health intervention type Digital health intervention purpose User population Summary of results relevant to this review question Collating, Summarizing, and Reporting Results Unlike a systematic review\\u2014where evidence sources that do not meet the quality criteria applied are excluded\\u2014scoping reviews seek to present an overview of all material reviewed (including gray literature) [ 24 ]. We will summarize the results visually [ 30 ] to assist validation of the results by stakeholders (described in the following section) and later dissemination. Consultation Exercise In parallel with the aforementioned stages, consultation with stakeholders (eg, representatives from charities, experts in researching digital health interventions, and patients or members of the public) will be conducted to validate emerging results. Stakeholders will be identified both a priori (known to the authors) and iteratively as this review progresses (through their representation in relevant sources).\",\n",
            "        \"subjects\": null,\n",
            "        \"results\": \"Results This systematic scoping review is in progress. The final draft of this review will be submitted by September 2022. We expect to report a narrative summary of the findings of included peer-reviewed primary studies and gray literature that describe differences in the ability of people from different demographic groups or people with lower health literacy or digital health literacy to access and use digital health interventions, and any change in this following specific measures taken by the researchers to make digital health interventions more accessible and usable by people from these groups; and the results of qualitative studies that discuss barriers and facilitators to access and use digital health interventions by people from different demographic groups.\",\n",
            "        \"discussion\": \"Discussion This review will present a summary of evidence regarding strategies for optimizing the design and deployment of digital health interventions to mitigate the effects of the digital divide and low digital health literacy on populations disproportionately affected by health inequalities and the digital divide. Its results will inform the design and deployment of digital health interventions at a time when they are more used than ever before and will have wider implications for researchers and policy makers using digital health interventions for health improvement. For example, HCI strategies for developing content for websites and apps, such as use of personas with a range of demographic characteristics (including those of digitally excluded populations) [ 31 , 32 ] and use of video and audio to help people of low health literacy use digital health interventions [ 33 ]. In conclusion, this review is among the first to examine the design and deployment of digital health interventions specifically in the context of the challenges presented by the digital divide and low digital health literacy at individual and systemic levels. The results of this review will support researchers, digital health intervention developers, and HCPs in identifying what works to optimize digital health intervention design and deployment, with the aim of promoting social justice.\",\n",
            "        \"conclusion\": null\n",
            "    },\n",
            "    {\n",
            "        \"pmcid\": \"PMC9333080\",\n",
            "        \"abstract\": \"This qualitative case study explored the impact of lockdown and social distancing measures used during COVID-19 on the lives of individuals with low vision. A purposive sample of 6 participants with low vision was recruited. Data were collected using in-depth semistructured interviews consisting of 10 open-ended questions. A thematic analysis was conducted to understand the unified subjective experiences of the participants. The analysis revealed 4 themes: (1) Changes, a lot of them; (2) Emotional roller coaster; (3) Life goes on; and (4) Technology as the catalyst. Participants demonstrated ongoing adaptation to meet the changing demands imposed by the pandemic.\",\n",
            "        \"intro\": null,\n",
            "        \"method\": \"METHODS A qualitative case study method was used to understand the impact of social distancing and lockdown measures during the COVID-19 pandemic on individuals with low vision. 30 Social distancing and lockdown in response to COVID-19 were an unforeseen and inexperienced event of this century. With the minimal transformation of the data, this method provided the foundation for gaining greater insight into the lives of individuals with low vision during a pandemic and answering the questions that are relevant to practitioners and policy makers. 31 , 32 The question \\u201cHow did lockdown and social distancing measures used during COVID-19 pandemic impact the lives of individuals with low vision?\\u201d guided this study. Subsidiary questions included the following: \\u201cHow were the challenges you faced during the pandemic?\\u201d \\u201cWhat emotional experiences did you face during this pandemic?\\u201d and \\u201cHow did your social participation and relationships have changed due to COVID-19?\\u201d Recruitment procedures and participant contexts A purposive sample of 6 participants with low vision was recruited from a private nonprofit agency located in New York in accordance with the Hofstra Institutional Review Board (IRB) (HUIRB#20190305-OCC-HPHS-HSU-2). The background of participants and their context is explained in the following text based on the information collected during the interview. Pseudo names were assigned to protect the participants' identities. Participant 1: Bryce Bryce is a 55-year-old man with the diagnosis of retinopathy prematurity. Prior to the pandemic, he lived with his wife and daughter in Florida, where he assisted the elderly in a community center as a part of a job training program. However, he was let go during the pandemic. As a result, he moved to New York State. He uses his smartphone with various apps to communicate with his family and friends. He has been using braille since childhood. Participant 2: Claire Claire is a 56-year-old woman with the diagnosis of rubella retinopathy. She had visual acuity of 20/200 in her youth and as the condition progressed, her vision gradually declined to 20/400. She developed glaucoma 8 years ago, leading to further decline of vision. Currently, she only has light perception. She lives with her daughter and 3 grandchildren. She uses a guide dog and a van provided by the Blind Association. Participant 3: Daisy Daisy is a 56-year-old woman with the diagnosis of congenital rubella syndrome. She was born without one eye and a torn retina in the other. Her visual acuity is 20/400 since birth. She also has decreased depth perception. She lives with her husband and her adolescent daughter. She uses a cane. She has a master's degree as a speech pathologist. Participant 4: Layla Layla is a 55-year-old woman with the diagnosis of an underdeveloped retina. She has a low vision, which further declined when she was 30 years old. At this time, she can only see hand movements. She lives alone (separated from her husband), and her adult children and sister would help her to go places. Prior to COVID-19, she was independent in taking public transportation such as subway, train, and Access-A-Ride or train. She uses a cane and a sight dog. Layla worked as a special education teacher for many years and decided to retire early due to her vision condition, and she has been very active in her athletic activities. Participant 5: Lauren Lauren is a 55-year-old woman who was diagnosed with retinopathy of prematurity at birth. Prior to the pandemic, she lived with her husband. However, she separated from her husband during the pandemic. She uses a dog or a cane and walks most places. Occasionally, she uses shared rides for transportation. She uses Braille and sighted assistance over the phone or the Internet. She has 2 master's degrees and works as a vision rehabilitation teacher. Since the beginning of the pandemic, she has been working from home 3 days a week. Participant 6: Yuna Yuna is a 53-year-old woman with the diagnosis of Leber's congenital amaurosis. She is color-blind and has a functional vision. She is a single parent of a 17-year-old son who also has Leber's congenital amaurosis. She said she is very comfortable using technology with accessibility features. She uses a cane and is independent in her mobility. She works as a special education early childhood teacher. Data collection and analysis Data were collected using an in-depth semistructured interview consisting of 10 open-ended questions. Because of pandemic restrictions, interviews were conducted virtually, either over the phone or encrypted service at the discretion of the subject. At least 2 research team members conducted each interview. Interview recordings were transcribed for thematic analysis. The thematic analysis was conducted to understand the common themes that represented the unified subjective experiences of all participants. Two researchers conducted thematic analysis 33 independent of each other. NVivo (version 12 Pro) was used for the analysis. Themes were developed in 3 stages: initialization, where each researcher read transcribed interviews and units of meaning and took notes independent of each other. Then during the construction phase, themes were classified, labeled, and translated. At this stage, data and themes generated by both researchers were compared and, if needed, combined. Finally, in the rectification phase, themes were appraised against the data, checked, and confirmed. 33\",\n",
            "        \"subjects\": null,\n",
            "        \"results\": \"RESULTS The thematic analysis produced 4 themes. These themes included the following: (1) Changes, a lot of them; (2) Emotional roller coaster; (3) Life goes on; and (4) Technology as the catalyst. Changes, a lot of them The lockdown and social distancing measures used to curb the pandemic led to disruption in all areas of participants' life. Some of the main areas disrupted by the pandemic included work, social participation, community participation, leisure activities, and grocery shopping. For example, Bryce had to move from Florida to New York after he lost his job. These measures also impacted participants' relationships and their interaction with family and friends. Lauren and her husband separated, and Denis could not see her friends or go out as reflected in her statement, \\u201cDuring the height of the pandemic, basically, I was holed up at my house with three other beings. Nobody left anywhere.\\u201d On the other hand, the pandemic also led to positive changes for some participants. Yuna had \\u201cmore work now,\\u201d and Claire got to be around her daughter and grandchildren \\u201ca lot.\\u201d Availability of family and social support, and sharing of resources, influenced their ability to cope up with the changes. While sharing space put a strain on the relationship for Lauren, having family members in the house helped Claire manage changes. The pandemic influenced participants' patterns of daily activities as reflected in the following statements: Well, I, you know, I would go into stores more independently before the pandemic, and I'm not really doing that because I, I don't feel that comfortable going by myself because of the whole social distancing aspect of it. I don't know, so I depend on going with somebody now where I would do that by myself before. (Layla) I did a lot of physical activity like running with Achilles and in tandem bike riding things like that which I haven't been doing. Seeing friends\\u2014it's it's hard to get together with people but I don't think that's different than anybody else pretty much. (Layla) It became very difficult to work (with everyone home) in the very small space with just no break for the tension, so that part I don't miss it all. As COVID-19 is concerned, it has pretty much trashed my marriage, so he (husband) was gone. (Lauren) Emotional roller coaster Participants experienced a range of emotions from anxiety to frustration to sadness in response to the sudden and sustained effect of the pandemic. The fear of catching COVID-19, the uncertainty of the course of the pandemic, changing guidelines, and inconsistencies in policies contributed to the theme of the emotional roller coaster , which is reflected in the following statements: Definitely nervousness of traveling and just being around people because about time people approach you, you know, very nicely you know to see if you need assistance but like you don't know if they're wearing masks or if they're not wearing a mask. (Layla) Changes in life situations and lack of opportunities for social interaction further affected their emotional well-being as reflected in Layla and Lauren's statements: Also just spending a lot more time alone has been kind of depressing and you know when people come over it just puts you in an awkward position like, like an electrician came over a couple of weeks ago and he didn't sound like yeah mask on and I asked him and he didn't have a mask you know so it's like kind of awkward to always have to ask are you wearing a mask, aren't you wearing a mask. (Layla) You know people that are close to me. I don't, even if I see my sister who lives close by we don't give each other hugs we give each other fist bumps or elbow bumps and it's really not the same thing, so I do feel a lot more cut off that way. (Lauren) Life goes on Although lockdown measures and social distancing posed challenges across many aspects of life, participants adapted to changes and uncertainty posed by the pandemic. Work and weekly schedules changed in response to the lockdown measures. Participants also worked remotely and engaged in social interactions using technology due to the lockdown measures. Bryce moved across the states to adapt to lost jobs, while Yuna had to take up more work and at times turn down some of the work opportunities. Lauren worked only 2 times a week. Participants changed or adapted leisure activities and social interactions as summarized in the following statements: I'll find something to do and I do I just find stuff to do I listen to my book so I'll be on my phone checking emails and before you know it the day is gone I have had tons and tons of like the Alexa devices so I play games on that and I don't play music and before you know it the day is done. (Claire) When they first shut down we would do it Zoom meetings and now ... what they're doing since this second shutdown is they're doing teleconference calls \\u2018cause like say Monday activity is exercising life skills so we're not doing the teleconference for exercise or doing one with the dietitian that talks about life skills and then the first.... Thursday of the month we have our book club so we've done ... that via you know Zoom. (Claire) We meet virtually for bands and instead of recording tracks live in you know, in the room together as a band, we have to record our instruments separately and send it to our you know, bandleader so we can mix the track, so our relationships have become like in a way we kind of depend on each other, even more now because in a room when you're recording a song. (Yuna) Technology as a catalyst Technology played an important role during the pandemic. Technology was used to accommodate changes in work, leisure, and social participation. Phone apps, Zoom, and other technology were used to continue with life. Layla used a screen recorder on her computer in order to grocery shop online and visited the Metropolitan Museum of Art over Zoom, and Daisy used WhatsApp or Voxer to communicate with her friends. Layla continued her guitar lessons on Zoom. Participants also had technology fatigue as reflected in the following statement: When the pandemic first started it was exciting learning all these new things and trying all these new like WhatsApp and you know doing all this stuff and now people like tired of doing Zoom. (Daisy)\",\n",
            "        \"discussion\": \"DISCUSSION This study was conducted to understand how social distancing and lockdown measures used during the pandemic impacted individuals with low vision. The results of the thematic analysis showed 4 themes: Changes, a lot of them; Emotional roller coaster; Life goes on; and Technology as the catalyst. The pandemic, social distancing, and lockdown measures led to changes in participants' social participation, community participation, work, and leisure activities. Similar results were reported in the general population by researchers. 8 , 9 In addition to demands posed by lockdown and social distancing measures, uncertainty related to the pandemic had a negative influence on the participants' emotional well-being. Similarly, as reported by researchers, 5 , 25 individuals who demonstrated negative emotions, and feelings of loneliness, were reported by participants in this study due to decreased social interaction and participation in meaningful activities. However, participants carried on with their lives; they found new meaningful activities or adapted the activities and continued meaningful relationships with the help of technology. Inner drive to engage in meaningful activities and social interactions facilitated the process of adaptation to meet the demands posed by the pandemic. The study shows that the pandemic affected the lives of individuals with low vision, and they needed to be prepared for the changing life situation. Although participants were able to manage challenges, it was not without exerting a toll on their emotional well-being. Availability of resources and social support positively influenced their ability to cope with the changes. The findings of this study can be used to prepare for future life-changing events. Limitations This study is a qualitative case study, and the findings represent the perspectives of the participants within their personal and environmental contexts; therefore, the results cannot be generalized. Most of the participants received higher education and either had worked prior to the pandemic or continued to work. All were very active within the community and engaged in many social activities prior to the pandemic. They also utilized social support and were accustomed to the technology. In addition, this group sample had adapted their gradual vision loss throughout their life with the technology. Therefore, they may not represent the population with low vision who do not have access to technology or other support. Future research A further study with a larger sample with individuals with low vision would help gain a deeper understanding of how the pandemic affected individuals with low vision.\",\n",
            "        \"conclusion\": null\n",
            "    },\n",
            "    {\n",
            "        \"pmcid\": \"PMC9381901\",\n",
            "        \"abstract\": \"Vitamin D deficiency has been epidemiologically linked to Alzheimer's disease (AD) and other dementias, but no interventional studies have proved causality. Our previous work revealed that the genomic vitamin D receptor (VDR) is already converted into a non\\u2010genomic signaling pathway by forming a complex with p53 in the AD brain. Here, we extend our previous work to assess whether it is beneficial to supplement AD mice and humans with vitamin D. Intriguingly, we first observed that APP/PS1 mice fed a vitamin D\\u2010sufficient diet showed significantly lower levels of serum vitamin D, suggesting its deficiency may be a consequence not a cause of AD. Moreover, supplementation of vitamin D led to increased A\\u03b2 deposition and exacerbated AD. Mechanistically, vitamin D supplementation did not rescue the genomic VDR/RXR complex but instead enhanced the non\\u2010genomic VDR/p53 complex in AD brains. Consistently, our population\\u2010based longitudinal study also showed that dementia\\u2010free older adults ( We studied whether it is beneficial to supplement vitamin D to APP/PS1 mice and humans. The animal studies show vitamin D deficiency is an early disease outcome and supplementation of vitamin D worsens AD brain pathology via enhancing VDR/p53 complex to impair autophagosome. Nationwide longitudinal cohort studies also suggest long\\u2010term vitamin D supplementation is associated with higher risk of dementia in dementia\\u2010free older adults and higher mortality in dementia individuals. Our results caution against prolonged use of vitamin D by AD patients and older adults.\",\n",
            "        \"intro\": null,\n",
            "        \"method\": \"2 METHODS 2.1 Mice Double transgenic APP/PS1 mice (Cat# 037565\\u2010JAX, RRID: MMRRC_037565\\u2010JAX) were purchased from Jackson Laboratory (Bar Harbor, ME, USA) and bred with wild\\u2010type B6C3F1/Bltw (C57BL/6 N background) mice. The sample size of mouse studies was chosen following the guidelines of \\u201cGuidelines for care and uses of mammals in neuroscience and behavioral research\\u201d (Van Sluyters & Obernier, 2004 ). Mice were allocated randomly in all experiments. Mice of 4.5 months of age were fed on a normal diet (Altromin; Cat# Altromin 1320) containing sufficient vitamin D 3 (600 IU/Kg of cholecalciferol) or D 3 \\u2010supplemented diet (Match to Altromin 1320 with 8044 IU/Kg of cholecalciferol; Research Diets, Inc; Cat# D13031002) with or without intraperitoneally injected weekly with 3 mg/kg of p53 inhibitor Pifithrin\\u2010\\u03b1 (PFT\\u03b1, Sigma\\u2010Aldrich, Cat# P4359) for 3 or 7.5 months before subsequent assays were conducted. Submandibular blood collection and cerebrospinal fluid (CSF) collection from anesthetized mice as described (Lim et al., 2018 ) were performed on APP/PS1 and wild\\u2010type mice at the indicated time points for quantification of serum 25(OH) D 3 levels by using Vitamin D 3 EIA Kit (Cayman Chemical, Cat# 501050). The Morris water maze test were performed in NHRI Animal Behavior Core Facility following the procedures as described previously (Lai et al., 2021 ). Animal procedures and protocols were approved by the Institutional Animal Care and Use Committee at NHRI (approved protocol no. NHRI\\u2010IACUC\\u2010101057\\u2010A and NHRI\\u2010IACUC\\u2010103136\\u2010A). 2.2 Cell culture, cell viability, TUNEL staining and mammalian two\\u2010hybrid assay SH\\u2010SY5Y (human neuroblastoma, ATCC CRL\\u20102266) cells were cultured as described (Lai et al., 2021 ). The cell viability was treatment with or without calcitriol (Sigma\\u2010Aldrich, Cat# D1530, 1,25(OH) 2 D 3 ) or calcidiol (Sigma\\u2010Aldrich, Cat# 17938, 25(OH)D 3 ) and measured by colorimetric WST\\u20101 assay. The TUNEL assay was performed by using ApoAlert\\u2122 DNA Fragmentation Assay Kit (Clontech, Cat# 630107). Mammalian two\\u2010hybrid assay was used to assess the interaction between VDR and RXR by using Dual\\u2010Luciferase Reporter Assay (Promega, Cat# E1910) for SH\\u2010SY5Y cells with pretreatment with A\\u03b242 (1\\u201310 \\u03bcM) followed by treatment with or without calcitriol for 6 h before the assay. 2.3 Antibody The antibodies used in this study are listed as follows: VDR (C20), Santa Cruz Biotechnology, Cat# sc\\u20101008; VDR(D6), Santa Cruz, Biotechnology Cat# sc\\u20101313; GAPDH, GeneTex, Cat# GTX10011; PARP\\u20101/2 (H\\u2010250), Santa Cruz Biotechnology, Cat# sc\\u2010715; LC3B, Cell Signaling Technology, Cat# 4108; Beta\\u2010Amyloid\\u20101\\u201016 antibody, BioLegend, Cat# 803014; \\u03b2\\u2010Amyloid Antibody, Cell Signaling Technology, Cat# 245; BACE (M\\u201083), Santa Cruz Biotechnology, Cat# sc\\u201010748; GFAP (Clone SP78), MybioSourse, Cat# MBS302899; GFAP (GA5), Cell Signaling Technology, Cat# 3670; p53 (DO\\u20101) Santa Cruz Biotechnology Cat# sc\\u201012; Cat# sc\\u201013985; Phospho\\u2010SQSTM1/p62 (Ser349), Cell Signaling Technology, Cat# 95697; SQATM1/p62 (GT1478), Thermo Fisher Scientific, Cat# MA5\\u201327800; TNF\\u2010\\u03b1 (D2D4) XP\\u00ae Rabbit mAb, Cell Signaling Technology, Cat# 1194; Alexa 488 chicken anti\\u2010rabbit IgG (H + L), Thermo Fisher Scientific, Cat# A\\u201021441; Alexa 594 chicken anti\\u2010goat IgG (H + L), Thermo Fisher Scientific Cat# A\\u20102146; Peroxidase\\u2010AffiniPure Goat Anti\\u2010Rabbit IgG (H + L), Jackson ImmunoResearch Labs, Cat# 111\\u2010035\\u2010144; Peroxidase\\u2010AffiniPure Goat Anti\\u2010Mouse IgG (H + L), Jackson ImmunoResearch Labs, Cat# 115\\u2010035\\u201014; Peroxidase\\u2010AffiniPure Rabbit Anti\\u2010Goat IgG (H + L), Jackson ImmunoResearch Labs, Cat# 305\\u2010035\\u2010003; Mouse anti\\u2010Rabbit light chain; HRP conjugate, Millipore, at# MAB201P; HRP\\u2010conjugated AffiniPure Mouse Anti\\u2010Rabbit IgG Light Chain, Bclonal Cat# AS061. 2.4 Population\\u2010based study 2.4.1 Study design and data sources We conducted a retrospective population\\u2010based cohort study aiming at delineating the effects of calcitriol use on dementia development in the dementia\\u2010free older adults and the mortality impact on subjects with dementia. There was about 99% of Taiwan's population of 23 million enrolled in the National Health Insurance (NHI) program since 1995. The Taiwan NHI Research Database (NHIRD), derived from the reimbursement claims within the NHI program, provides detailed medical utilization information of the NHI beneficiaries. The NHIRD was the data source we used to select calcitriol users for this study. The Longitudinal Health Insurance Database for the year 2000 (LHID2000), one of the data components in NHIRD, consists of claims information of one million subjects who were randomly selected from the 2000 Registry of Beneficiaries of the NHI program. The LHID2000 served as the sampling pool from which we selected the calcitriol nonusers for the first analysis. The Registry of Catastrophic Illness Patients (RCIP) is another dataset of the NHIRD, containing information of NHI utilization of all patients with catastrophic illnesses defined by the Taiwan's government. A rigorous clinical review and evaluation would precede entitlement for the RCIP to assure valid diagnoses. The catastrophic injuries/illnesses in Taiwan included 31 categories of major illnesses (e.g., cancer, dialysis, hemophilia, etc.), with which patients are exempt from co\\u2010payment and may thus avoid financial hardship. Dementia (ICD9 290 or 294) is one of the listed catastrophic illnesses. The RCIP is another dataset we used to conduct the second analysis. 2.4.2 Analysis of the risk of incident dementia From the data source of NHIRD, we identified those who were 65 years or older, free of dementia diagnosis before 2000, and had received at least one calcitriol prescription in 2000\\u20132009 ( n = 20,108) as the calcitriol users (Figure 3a ). To exclude subjects less adherent to the study medication, we excluded the users with very short\\u2010term calcitriol use (less than ten 2.5 mcg capsules per year) and the users who were diagnosed with dementia within 6 months after the first prescription of calcitriol had been made. From the LHID2000, we considered subjects as the calcitriol nonusers if they were 65 years of age or older, free of dementia diagnosis before 2000, and never had calcitriol prescription in 2000\\u20132010 ( n = 89766). To increase comparability, the calcitriol users were propensity score\\u2010matched to their nonuser counterparts on a 1:1 ratio based on age (birthday year), gender, calcium prescription (yes/no), and comorbidities (yes/no) including chronic kidney disease (ICD9: 582, 585, 586, 583.0, 583.1, 583.2, 583.3, 583.4, 583.5, 583.6, 583.7, A\\u2010code: A350), osteoporosis (ICD9: 733.0 V17.81 V82.81, A\\u2010code: A439 AV09 AV05), acquired hypothyroidism (ICD9:240\\u2013246, A\\u2010code: A180), Diabetes (ICD9: 250, A\\u2010code: A181), hyperlipidemia (ICD9: 272, A\\u2010code: A182), hypertension (ICD9: 401 402 403 404 405, A\\u2010code: A269 A260), and the Charlson's index score. After matching procedures, we got equal number of calcitriol users and nonusers ( n = 7324) for further analysis. The subgroup analysis was conducted with the gender of male ( n = 5362) and female ( n = 9286), or the age of 65\\u201375 years old ( n = 5196) and older than 75 years old ( n = 9452). 2.4.3 Analysis of dementia mortality From the RCIP in 2000\\u20132009, we identified 51,606 individuals who were 60 years or older and had a diagnosis of dementia (ICD9 290 or 294), of whom 636 subjects had their first calcitriol prescription after the diagnosis and the remaining 50,970 subjects never on calcitriol were considered as the calcitriol nonusers (Figure 4a ). To exclude subjects less adherent to the study medication, we excluded the users with very short\\u2010term calcitriol use (less than ten 2.5 mcg capsules per year) and the users who died within 3 months after the first calcitriol prescription. To increase comparability, we also matched the calcitriol users to the nonusers on a 1:4 ratio based on the same covariates as described in the analysis of risk of incident dementia. Frequency matching was performed to ensure a similar distribution of the variables in each group. Baseline characteristics showed no difference between the two groups. After matching procedures, we kept 196 dementia patients who were also the calcitriol users and 784 dementia patients without taking calcitriol (the nonusers) for further analysis. The subgroup analysis was conducted with the gender of male ( n = 315) and female ( n = 665), or the age of 60\\u201375 years old ( n = 239) and older than 75 years old ( n = 741). 2.4.4 Study outcome The study outcome in the analysis for the risk of incident dementia was designated to be dementia development, which was defined as the first diagnosis of dementia (ICD9: 290.0, 290.1, 290.2, 290.3, 290.4, 294.1, 331.0, 331.1, 331.2, A\\u2010code: A210, A213, A22). For the analysis of the risk of dementia mortality, the study outcome was all\\u2010cause mortality. 2.4.5 Statistical analysis We used chi\\u2010square test and t \\u2010test to detect the differences for categorical and continuous variables, respectively. Associations between calcitriol use and study outcomes were analyzed using Kaplan\\u2013Meier survival curves and log\\u2010rank tests. We used Cox proportional hazard models to assess adjusted effects of calcitriol use on the study outcomes. The proportional hazards assumption was ascertained by comparing log\\u2013log survival curves for all time\\u2010independent covariates. All assessed log\\u2013log survival plots indicated no violation of the assumption. Hazard ratios (HR) with 95% confidence interval (CI) were used to measure the risk of dementia or mortality. For calcitriol users, study entry was defined as the date of first calcitriol use; while for the nonusers, study entry was assigned as the same date as their matched user counterparts. In analysis 1, the outcome was dementia development and observations were censored on last clinical visits by 31 December 2010. In analysis 2, the outcome was death defined as withdrawal from the national health insurance registry and sustained lack of healthcare utilization thereafter for at least 1 year. The observation was censored on the date of the last clinic visit by 31 December 2010. A two\\u2010tailed p value less than 0.005 was considered significant. We conducted analyses using SAS version 9.4.\",\n",
            "        \"subjects\": null,\n",
            "        \"results\": \"3 RESULTS 3.1 APP / PS1 AD mice given vitamin D\\u2010sufficient diet exhibit decreased serum vitamin D levels Because vitamin D deficiency is linked to AD, it was imperative to clarify whether vitamin D deficiency is a risk factor or an outcome of AD. To test this hypothesis, we fed both APP/PS1 and wild\\u2010type (WT) mice a vitamin D 3 \\u2010sufficient diet (600 IU/Kg of cholecalciferol) and measured their serum vitamin D levels during the early stages of life. Intriguingly, beginning as early as 4 months, the AD mice started exhibiting significantly lower serum 25(OH)D 3 levels compared with those 2 months old. There was no difference in serum vitamin D levels in WT over the study period (Figure 1a ). We then tested whether the 25(OH)D 3 levels were also decreased in the CSF of AD mice as seen in the serum. Indeed, the EIA assays showed that the 25(OH)D 3 levels in CSF were also markedly decreased in AD mice compared with WT controls (Figure S1 ). These results may suggest that vitamin D deficiency may be caused by AD rather than caused by a lack of dietary vitamin D. FIGURE 1 Dietary supplementation of vitamin D 3 aggravates AD pathology in APP/PS1 AD mice. (a) Serum 25(OH)D 3 levels in APP/PS1 (TG) and wild\\u2010type (WT) mice. Mice were weaned at 4\\u2010weeks of age (\\u00b13 days) and maintained on a vitamin D 3 \\u2010sufficient diet (600 IU/Kg of cholecalciferol). Serum vitamin D 3 levels were determined by 25(OH)D 3 enzyme\\u2010linked immunosorbent assay (EMSA) at the indicated time points ( n = 5). Results are shown as mean \\u00b1 SD. * p < 0.05; ** p < 0.01; *** p < 0.001 by unpaired t \\u2010test. (b) WST\\u20101 cell viability assay. SH\\u2010SY5Y cells were exposed to vitamin D 3 alone or \\u0391\\u03b2 (4 \\u03bcM) plus vitamin D 3 (calcitriol or calcidiol) for 6 h prior to assays. Results are shown as mean \\u00b1 SD. * p < 0.05 by One\\u2010way ANOVA. (c) Western blot analysis of VDR, apoptotic and autophagic marker proteins in SH\\u2010SY5Y cells exposed to A\\u03b242 or plus without or without Vitamin D 3 . SH\\u2010SY5Y cells were treated with 4 \\u03bcM A\\u03b242 alone or A\\u03b242 plus 10, 30, or 100 nM calcitriol for 6 h before harvesting cell lysates for analysis. (d) Representative immunofluorescent micrographs of gliosis (anti\\u2010GFAP, GA5) and amyloid aggregates (anti\\u2010\\u0391\\u03b2, D54D2) in hippocampal tissues of APP/PS1 mice. 4.5\\u2010month\\u2010old APP/PS1 mice were fed with vitamin D 3 \\u2010supplemented (8044 IU/Kg cholecalciferol/day, Vit. D) or vitamin D 3 \\u2010sufficient diets (600 IU/Kg cholecalciferol/day, Ctrl) for 3 months before harvesting brain tissues for analysis. Sections of cortex or hippocampus were stained with the indicated antibodies. The average percentage of surface area with \\u0391\\u03b2 plaques in five consecutive sections per animal ( n = 4\\u20137) was quantified by ImageJ in right panel. (e) Western blot analysis of \\u0391\\u03b2 production and \\u03b2\\u2010secretase 1 (BACE1) levels in hippocampal lysates of APP/PS1 mice supplemented with or without vitamin D 3. Densitometrical quantification of A\\u03b2 and BACE bands were normalized to GAPDH (right panel). (f) Cognitive performance for AD mice supplemented with vitamin D 3 . 4.5\\u2010month\\u2010old APP/PS1 mice were fed with vitamin D 3 \\u2013fortified (Vit. D) or vitamin D 3 \\u2010sufficient diets (Ctrl) for 7.5 months before Morris Water Maze. * p < 0.05 by One\\u2010way ANOVA 3.2 Faster disease progression after vitamin D supplementation in AD mice Our previous finding that the VDR\\u2010RXR heterodimer for transducing the genomic vitamin D signal was impaired in AD led us to question the seemingly common assumption that vitamin D supplementation may exert a protective effect on AD (Groves et al., 2014 ). We first assessed the potential impact of vitamin D supplementation on an A\\u03b242\\u2010treated neuronal cell line. The results showed that the incubation of vitamin D (calcitriol or calcidiol) with SH\\u2010SY5Y cells exposed to A\\u03b242 resulted in a significant dose\\u2010dependent increase in apoptosis and autophagy (Figure 1b,c ), suggesting that vitamin D might have a potentially damaging effect on neuronal cells exposed to A\\u03b242. Encouraged by this result, we proceeded to explore whether vitamin D supplementation could exert a similar detrimental effect on the progression of AD in mice. Adding 8044 IU/kg of cholecalciferol (vitamin D 3 ) to the diets of APP/PS1 AD mice 4.5 months old for 3 months resulted in more severe A\\u03b2 plaque deposits and reactive gliosis in the hippocampus compared to controls (Figures 1d and S2 ). Western blot analyses also revealed increased levels of pro\\u2010degenerative factors, including A\\u03b2, \\u03b2\\u2010secretase 1 (BACE1), Nicastrin (a subunit of the \\u03b3\\u2010secretase complex), TNF\\u2010\\u03b1, and autophagy in the hippocampal lysates of the experimental mice (Figures 1e and S2 ). Moreover, to exclude the possibility of a toxic dose effect in the supplementation of vitamin D, we examined the serum 25(OH)D 3 levels and found that both the WT and AD mice maintained a physiological level of 25(OH)D 3 after supplementation of 8,044 IU/kg calcidiol (Figure S2 a). Mice were then also subjected to the Morris water maze test. Mice administered vitamin D displayed worse cognitive functioning and performance behavior than the controls (Figure 1f ). The results of these experiments suggest that over\\u2010supplementation of vitamin D can exacerbate AD neurodegeneration. 3.3 Vitamin D supplementation enhances non\\u2010genomic VDR /p53 signaling in worsening brain pathology in AD mice It is known that vitamin D normally induces VDR and RXR to combine, allowing the transduction of the genomic vitamin D signal (Yasmin et al., 2005 ). However, it is not known whether this occurs in the context of AD. To determine whether vitamin D would enhance the VDR/RXR complex formation in cells being exposed to A\\u03b242, we performed a biochemical study in which we added vitamin D 3 to the SH\\u2010SY5Y cells exposed to A\\u03b242. Surprisingly, vitamin D 3 treatment did not enhance the VDR/RXR interaction in cell exposed to A\\u03b242 compared with cells exposed to vitamin D 3 only (Figure 2a ). This result suggested that the A\\u03b242 impairment of the VDR/RXR pathway was not rescued by vitamin D. Because our previous work had demonstrated that A\\u03b242 switched VDR binding partner from RXR to p53 to transduce the non\\u2010genomic vitamin D signal in AD brain (Lai et al., 2021 ), we wanted to investigate whether the VDR/p53 complex formation was enhanced by vitamin D. We performed co\\u2010immunoprecipitation assays and found that the A\\u03b2\\u2010triggered VDR/p53 complex was indeed further enhanced by the treatment of vitamin D 3 (Figure 2b ). Therefore, we continued to investigate whether vitamin D supplementation might also lead to a similar consequence enhancing VDR/p53 complex in AD mice. Indeed, after dietary supplementation of cholecalciferol for 3 months, APP/PS1 mice were found to have enhanced VDR/p53 interaction in the hippocampal tissues (Figure 2c ). These results suggest vitamin D 3 could aggravate A\\u03b242\\u2010trigged VDR/p53 signaling promoting neuropathogenesis. FIGURE 2 Vitamin D supplementation enhances VDR/p53 but not VDR/RXR complex in worsening brain pathology in APP/PS1 AD mice. (a) Mammalian two\\u2010hybrid assays for studies of the interaction of VDR with RXR in neuronal cells exposed to \\u0391\\u03b2 plus with vitamin D 3 . SH\\u2010SY5Y cells were treated A\\u03b242 for 6 h and then co\\u2010treated with 10 nM calcitriol for additional 6 h prior to harvesting for mammalian two\\u2010hybrid luciferase assays. (b,c) Western blot analysis of co\\u2010immunoprecipitation of VDR/p53 complex in SH\\u2010SY5Y cells and hippocampal tissues of APP/PS1 mice. (d) Western blot analysis of VDR, p53, and MDM2 in the hippocampal lysates of APP/PS1 mice treated with or without p53 inhibitor. 4.5\\u2010month\\u2010old APP/PS1 mice raised on vitamin D 3 \\u2010sufficient diets were intraperitoneally injected weekly with 3 mg/kg of p53 inhibitor pifithrin\\u2010\\u03b1 (PFT\\u03b1) for 7.5 months before harvesting hippocampal tissues for analysis. Densitometrical quantification of VDR, p53, and MDM2 bands were normalized to GAPDH (lower panel). * p < 0.05; ** p < 0.01; *** p < 0.001 by unpaired t \\u2010test. (e) Western blot analysis of autophagic markers LC3, p62, and ser349 phosphorylated p62 (p62\\u2010S349) in the hippocampal lysates of APP/PS1 mice injected with or without PFT\\u03b1. Densitometrical quantification of LC3, p62\\u2010S349, and p62 bands were normalized to GAPDH (right panel). (f) Western blot analysis of \\u0391\\u03b2 and BACE levels in the hippocampal lysates of APP/PS1 mice injected with or without PFT\\u03b1. Densitometrical quantification of A\\u03b2 and BACE bands were normalized to GAPDH (right panel). (g,h) p53 inhibitor amelioration of vitamin D 3 \\u2010aggravated A\\u03b2 aggregation and apoptosis. The A\\u03b2, GFAP, and TUNEL\\u2010positive signals in five consecutive sections per animal ( n = 5) was quantified by ImageJ and presented as the mean \\u00b1 SD. Scale bars, 50 \\u03bcm. (i) Cognitive performance assays for the AD mice treated with p53 inhibitor. APP/PS1 mice were given with or without weekly injections of PTF\\u03b1 ( n = 6 mice) starting at the age of 4.5\\u2010month. APP/PS1 mice at 12\\u2010month of age were used for the Morris Water Maze test To further investigate whether the aggravation of VDR/p53 pathway by vitamin D would contribute to brain pathology in AD mice, we investigated whether we could reverse the adverse effects of vitamin D on AD by administering p53 inhibitor PFT\\u03b1. Indeed, the p53 inhibitor PFT\\u03b1 not only markedly decreased protein levels of p53 and VDR but also increased the levels of MDM2, a protein that interacts with p53 promoting its degradation (Figure 2d ). We also found that PFT\\u03b1 decreased their autophagic protein LC3II levels (Figure 2e ). Because site\\u2010specific phosphorylation of p62 has been implicated in the disruption of autophagy\\u2010mediated protein degradation in AD brains, we also measured S349\\u2010phosphorylated p62 (P\\u2010S349) levels. We found p53 inhibitors had decreased P\\u2010S349 levels, which are normally increased in the AD brain (Figure 2e , second panel). Based on these findings, we believed we would also be able to observe improvement in brain lesions. Indeed, Western blot and immunohistochemistry studies both revealed significant attenuation in A\\u03b2 deposits, BACE activity, reactive gliosis, and neuronal apoptosis in the AD brains with vitamin D 3 supplementation (Figure 2f\\u2013h ). Finally, we wanted to know whether cognitive functioning and performance behavior would also be improved by treatment with a p53 inhibitor. Mice that were given the p53 inhibitor showed significant improvement in the Morris Water Maze test (Figure 2i ). Taken together, these results suggest that Vitamin D 3 supplementation aggravates VDR/p53 pathway in promoting brain pathology in APP/PS1 mice. 3.4 Vitamin D supplementation is associated with risk of dementia Thus, we began to question whether vitamin D supplementation would, as has been suggested, be able to decrease the risk of AD. To find out, we performed a retrospective population\\u2010based study to assess the effects of continuous use of an optimal dosage of calcitriol on the risk of dementia in older people (Figure 3a ). We identified and enrolled 14,648 dementia\\u2010free people aged over 65 years who had or had not been prescribed calcitriol from Taiwan's National Health Insurance Research Database (NHIRD) and then followed them for 11 years (2000\\u20132010) to find out if they had received a diagnosis of incident dementia. Most claimants prescribed the vitamin took one tablet of calcitriol (0.25 mcg or 0.5 mcg) daily, the suggested optimal daily intake for this vitamin. Frequency matching was performed for age, gender, and eight comorbidities (prescriptions for calcium, chronic renal disease, osteoporosis, thyroid, diabetes, hyperlipidemia, hypertension, and Charlson score) and Cox proportional hazard model was used to assess the effects of calcitriol on incident dementia in three different dosage groups (Table 1 ). As can be seen in Figure 3b and Table S1 , a low cumulative dose of calcitriol (<10.95 mcg/year or <43.8 capsules of 0.25 mcg per year) did not appear to affect the incident dementia. However, those taking a high (average >36.5 mcg/year or >146 capsules of 0.25 mcg per year) and medium (average 10.95\\u201336.6 mcg/year or 43.8\\u2013146 capsules of 0.25 mcg per year) cumulative dose had a 1.80\\u2010 and 1.27\\u2010fold increase in risk of incident dementia, respectively, compared with nonusers, suggesting a potential link between prolonged use of calcitriol and increased risk of dementia in older people. Similar effects were also observed in the subgroup analysis of sex (Tables S2 and S3 ). When incidence of dementia was compared between sex, it was interesting to note that dementia\\u2010free females appeared to be more sensitive than dementia\\u2010free males to calcitriol supplementation (comparing Table S2 with Table S3 ). Similar effects were also observed in the subgroup analysis of age (Tables S4 and S5 ). When incidence of dementia was compared between two age groups, dementia\\u2010free adults aged 65\\u201375 showed more sensitivity than adults aged 75 or older to vitamin D supplementation (when comparing Table S4 with Table S5 ). FIGURE 3 Population\\u2010based cohort study of the associations between the incidence of dementia and calcitriol supplementation. (a) Flow chart of cohort formation for epidemiological study of association between incident dementia and calcitriol supplementation. NHIRD, National Health Insurance Research Database; LHID2000, Longitudinal Health Insurance Database 2000; Comorbidities including calcium prescription, chronic renal disease, osteoporosis, thyroid, diabetes, hyperlipidemia, hypertension, and Charlson score. (b) The adjusted curves of dementia development in study subjects aged over 65 years with different average cumulative dosages of calcitriol with a follow\\u2010up of up to 10 years ( n = 14,648). The definition of the \\u2018dosage/year\\u2019 is the assumed average maintenance dose (mcg) per year for calcitriol used in the whole follow\\u2010up. *** p < 0.001 by Maximum Likelihood\\u2010ratio test TABLE 1 Baseline characteristics of the matched calcitriol users and nonusers Before propensity score match After propensity score match Users Nonusers p Value a Users Nonusers p Value a N = 20108 N = 89766 N = 7324 N = 7324 Age 74.5 (6.5) 73.9 (6.8) <.0001 77.0 (5.6) 77.0 (5.6) 1.000 Sex/gender Female 13,236 (65.82) 41295 (46.00) <.0001 4643 (63.39) 4643 (63.39) 1.000 Male 6872 (34,018) 48471 (54.00) 2681 (36.61) 2681 (36.61) Osteoporosis No 16,733 (83.22) 81035 (90.27) <.0001 6023 (82.24) 6023 (82.24) 1.000 Yes 3375 (16.78) 8731 (9.73) 1301 (17.76) 1301 (17.76) Hyperlipidemia No 16,955 (84.32) 83,550 (93.08) <.0001 6421 (87.67) 6421 (87.67) 1.000 Yes 3153 (15.68) 6216 (6.92) 903 (12.33) 903 (12.33) Hypertension No 8064 (40.1) 59,677 (66.48) <.0001 2981 (40.7) 2981 (40.7) 1.000 Yes 12,044 (59.9) 30,089 (33.52) 4343 (59.3) 4343 (59.3) Thyroid disorders No 19,348 (96.22) 88,916 (99.05) <.0001 7214 (98.5) 7214 (98.5) 1.000 Yes 760 (3.78) 850 (0.95) 110 (1.5) 110 (1.5) Diabetes No 14,188 (70.56) 78,234 (87.15) <.0001 5570 (76.05) 5570 (76.05) 1.000 Yes 5920 (29.44) 11,532 (12.85) 1754 (23.95) 1754 (23.95) Renal disease No 17,964 (89.34) 88,523 (98.62) <.0001 7016 (95.79) 7016 (95.79) 1.000 Yes 2144 (10.66) 1243 (1.38) 308 (4.21) 308 (4.21) CCI score Score 0 9016 (44.84) 47,421 (52.83) <.0001 3141 (42.89) 3141 (42.89) 1.000 Score 1 5700 (28.35) 17,704 (19.72) 2061 (28.14) 2061 (28.14) Score \\u2265 2 5392 (26.82) 24,641 (27.45) 2122 (28.97) 2122 (28.97) Calcium Prescription No 2988 60,062 (66.91) <.0001 1053 (14.38) 1053 (14.38) 1.000 Yes 17,120 29,704 (33.09) 6271 (85.62) 6271 (85.62) Note : Age was expressed as mean (SD) and others data were expressed as n (%). a Continuous variables were analyzed using Kruskal\\u2013Wallis test, whereas categorical variables (proportions) were analyzed using the chi\\u2010square test. 3.5 Vitamin D supplementation is associated with mortality of dementia Since vitamin D supplementation led to increased A\\u03b2 depositions and exacerbated AD led, we wanted to know whether vitamin D supplementation would also increase the risk of mortality in people with pre\\u2010existing dementia. We identified 980 patients diagnosed with dementia who were and were not prescribed calcitriol in the NHIRD and followed them over a 11\\u2010year period (2000\\u20132010) (Figure 4a ). They were frequency matched for similar distributions in age, gender, and eight comorbidities (Table 2 ). The dementia patients prescribed a high cumulative dose of calcitriol (>146 capsules of 0.25 mcg per year) were found to have a 2.17\\u2010fold increase in risk of death, compared with those not prescribed the drug (Figure 4b and Table S6 ). We found no significant difference in mortality among those taking medium doses (43.8\\u2013146 capsules/year) or low cumulative doses (<43.8 capsules/year), compared with nonusers (Figure 4b and Table S6 ). Similar effects were also observed in the subgroup analysis of sex (Tables S7 and S8 ). When survival was compared between sex, females with pre\\u2010existing dementia showed worse survival than males with pre\\u2010existing dementia in response to calcitriol supplementation (comparing Table S7 with Table S8 ). When survival was compared between two age groups, the pre\\u2010existing dementia adults aged 75 and older showed worse survival than the aged 65\\u201375 adults with pre\\u2010existing dementia to calcitriol supplementation (when comparing Table S9 with Table S10 ). Taken together, these animal model and population\\u2010based results support the conclusion that vitamin D supplementation aggravates the progression of AD. FIGURE 4 Population\\u2010based cohort study of the associations between risk of mortality and calcitriol use in dementia. (a) Flow chart of cohort formation for epidemiological study of association between survival and calcitriol supplementation in patients with pre\\u2010existing dementia. HV, Registry for catastrophic illness patients; ICD9, International Statistical Classification of Diseases and Related Health Problems (ICD) 9; NHIRD, National Health Insurance Research Database; ID, Registry for beneficiaries; RCIP, Registry of Catastrophic Illness Patients; Comorbidities including calcium prescription, chronic renal disease, osteoporosis, thyroid, parathyroid disorders, hyperlipidemia, hypertension, and Charlson score. (b) The adjusted survival curves among dementia patients with different average dosages of calcitriol. The relationship between mortality and calcitriol use was determined by using the Kaplan\\u2013Meier survival curves and log\\u2010rank tests with a follow\\u2010up of up to 10 years ( n = 980). The model was adjusted for age, sex/gender, calcium prescription, chronic renal disease, osteoporosis, thyroid, parathyroid disorders, hyperlipidemia, and hypertension. *** p < 0.001 by Maximum Likelihood\\u2010ratio test (Ctrl vs. high cumulative doses with >36.5 mcg/year) TABLE 2 Baseline characteristics of the matched calcitriol users and nonusers in the survival of dementia cohort Users (dosage (mcg/year)) Nonusers p Value a >36.5 ( n = 63) 10.95\\u201336.5 ( n = 64) 2.5\\u201310.95 ( n = 69) ( n = 784) Sex/gender Female 42 (66.67) 44 (68.75) 47 (68.12) 532 (67.86) 0.9955 Male 21 (33.33) 20 (31.25) 22 (31.88) 252 (32.14) Age 80.0 \\u00b1 7.0 79.4 \\u00b1 6.1 78.9 \\u00b1 6.4 79.3 \\u00b1 6.3 0.6801 Calcium prescription No 18 (28.57) 15 (23.44) 18 (26.09) 204 (26.02) 0.9329 Yes 45 (71.43) 49 (76.56) 51 (73.91) 580 (73.98) Renal disease No 34 (53.97) 37 (57.81) 47 (68.12) 472 (60.20) 0.395 Yes 29 (46.03) 27 (42.19) 22 (31.88) 312 (39.80) Thyroid disorder No 61 (96.83) 64 (100.00) 69 (100.00) 776 (98.98) 0.2342 Yes 2 (3.17) 0 (0.00) 0 (0.00) 8 (1.02) Osteoporosis/osteopenia No 32 (50.79) 30 (46.88) 28 (40.58) 360 (45.92) 0.7012 Yes 31 (49.21) 34 (53.13) 41 (59.42) 424 (54.08) Charlson comorbidity score Score 0\\u20132 19 (30.16) 16 (25.00) 27 (39.13) 248 (31.63) 0.7173 Score 3\\u20134 21 (33.33) 26 (40.63) 23 (33.33) 280 (35.71) Score \\u2265 5 23 (36.51) 22 (34.38) 19 (27.54) 256 (32.65) Hyperlipidemia No 47 (74.60) 43 (67.18) 45 (65.22) 540 (68.89) 0.6869 Yes 16 (25.40) 21 (32.81) 24 (34.78) 244 (31.12) Hypertension No 13 (20.63) 13 (20.31) 15 (21.74) 164 (20.92) 0.9975 Yes 50 (79.37) 51 (79.69) 54 (78.26) 620 (79.08) Note : Data were expressed as n (%). a Continuous variables were analyzed using Kruskal\\u2013Wallis test, whereas categorical variables (proportions) were analyzed using the chi\\u2010square test.\",\n",
            "        \"discussion\": \"4 DISCUSSES The level of VDR is usually positively associated with the serum level of 25(OH)D 3 in healthy adults (Medeiros et al., 2020 ). Normally, when VDR is inactivated, the serum 1,25(OH) 2 D 3 levels will be increased to maintain homeostasis. However, a converse relationship between vitamin D concentration and VDR levels can be seen when the 1,25(OH)D 3 \\u2010VDR signaling pathway is impaired during the pathogenesis of chronic diseases. In line with this notion, the findings of this study using an AD mouse model suggest that vitamin D deficiency may be actually more of an early feature or an outcome of AD than a cause of the disease. Other studies report findings that may also support this notion. Older African\\u2010Americans are two to three times more likely to develop AD than elderly whites (Alzheimer's's, 2014 ; Amadori et al., 2017 ), while the African\\u2010Americans have higher mean VDR levels (Amadori et al., 2017 ; Neill et al., 2013 ; Richards et al., 2017 ) but much lower serum levels of vitamin D (Dawson\\u2010Hughes, 2004 ). Another example of the converse relationship between vitamin D concentration and VDR levels has been reported in patients with insulin resistance and obesity, who have been found to have deficient levels of vitamin D on the one hand but increased levels of VDR in adipose tissue on the other (Kang et al., 2015 ). Future studies may want to explore whether the decrease of vitamin D is actually a common pathological response that occurs in many aging\\u2010associated diseases, because, in addition to AD, patients with vascular disease, thyroid disorders, and osteoporosis are likely to have decreased levels of serum vitamin D and, of course, be at higher risk for dementia (Autier et al., 2014 ; Duthie et al., 2011 ). The findings of our animal experiments also suggest that the prolonged vitamin D supplementation might actually exacerbate AD. To try to understand how vitamin D might have an adverse effect on development and progress of AD, we explored how the VDR pathway might somehow be involved. In our exploration, we found the vitamin D did not rescue the canonical VDR\\u2010RXR pathway but instead further exacerbated the non\\u2010genomic VDR/p53 complex in causing damage to AD brains. The results of our mechanistic analysis are important as we try to clarify why supplementation with vitamin D may not be the best way to address vitamin D deficiency with AD and may not protect older people from dementia. Importantly, the epidemiological studies that we performed using two nationwide longitudinal cohorts also supported our finding that prolonged supplementation of vitamin D 3 had adverse effect in AD. We found that long\\u2010term supplementation of vitamin D did not have any benefit on dementia and, it is likely that it increased the risk of dementia in older people and increased mortality in people with dementia. This large\\u2010scale cohort finding is important in that it helps bring into question the wisdom behind the assumption that AD\\u2010associated vitamin D deficiency necessitates need for vitamin D supplementation among older people seeking protection from dementia. These findings, however, do not preclude the potential clinical benefit of vitamin D supplementation on lower AD risk in younger or middle\\u2010aged people before the disease has taken its toll and before AD damage has become irreversible. Supporting this notion, several animal studies have reported that starting vitamin D supplementation at very early stage of disease in AD mice, when amyloid plaques are far from being developed, may benefit AD (Landel, Millet, et al., 2016b ; Morello et al., 2018 ; Wong et al., 2021 ; Yu et al., 2011 ). Therefore, timing of supplementation is perhaps an important factor to consider. Accordingly, it may be more prudent to discourage older adults or individuals with dementia from long\\u2010term vitamin D supplementation until large careful clinical trials are performed to prove otherwise. This cohort study has some limitations. One limitation is that it is a cross\\u2010sectional study and so a causal relationship between long\\u2010term vitamin D supplementation and increased risk of dementia cannot be firmly established. Another limitation of the present study is the lack of generalizability of our findings to other racial and ethnic groups. Finally, the implications of vitamin D deficiency for many other chronic diseases also need to be further investigated by exploring the mechanistic link of initiation of vitamin D deficiency and its causality in those diseases. Vitamin D is also known the sunshine vitamin because it is synthesized in our skin during exposure to sunlight. This seemingly super nutrient was initially recognized as a key regulator of calcium homeostasis in maintaining bone integrity. Its use has been expanded to cover a wide range of body functions from early life to old age, because a large number of observational studies have demonstrated that a deficiency in vitamin D has been associated with many disorders throughout the body, including not only osteoporosis and fractures but also heart disease, high blood pressure, COVID\\u201019 infection, immune system disorders, cancer, stroke, and several metabolic disorders (Holick & Chen, 2008 ; Pereira\\u2010Santos et al., 2015 ). The hormonal\\u2010like vitamin D exerts a neuroprotective role through genomic action (Brewer et al., 2001 ). Specifically, vitamin D metabolite 1,25(OH) 2 D 3 binds to VDR to confer transcriptional activity in the nucleus. However, the results of the current study suggest supplementation of vitamin D may actually worsen brain health in older adults whose blood levels of vitamin D are low. While this finding in surprising, a more interesting research topic would be the mechanism through which vitamin D supplementation worsens the effect of vitamin deficiency. It is possible that the vitamin D\\u2010dependent genomic signaling is impaired in patients with dementia. VDR is also known to act as an important regulator of xenobiotic metabolism in a vitamin D\\u2010independent manner (Krasowski et al., 2011 ; Reschly & Krasowski, 2006 ). Thus, it could be that dementia\\u2010related toxic amyloid formation elicits xenobiotic responses through non\\u2010genomic VDR signaling. Supporting this hypothesis, our study using AD mouse model suggests that the non\\u2010genomic VDR/p53 signaling is further activated in AD brains and contributes to neuronal apoptosis. We, therefore, believe the findings of this longitudinal cross\\u2010sectional study, which is supplementation vitamin D exacerbates dementia, may better align with our hypothesis that non\\u2010genomic VDR signaling might contribute to the promotion of dementia. In conclusion, the results of the current study suggest supplementation of vitamin D may increase the risk of dementia in older adults and increase the risk of mortality in older adults with pre\\u2010existing AD whose blood levels of vitamin D are low. Until our findings are proved otherwise, the use of vitamin D supplementation to prevent dementia should be reconsidered.\",\n",
            "        \"conclusion\": null\n",
            "    },\n",
            "    {\n",
            "        \"pmcid\": \"PMC9529122\",\n",
            "        \"abstract\": \"Epithelial ovarian carcinoma (EOC) is the most lethal gynecological tumor, that almost inevitably relapses and develops chemo-resistance. A better understanding of molecular events underlying the biological behavior of this tumor, as well as identification of new biomarkers and therapeutic targets are the prerequisite to improve its clinical management. ZNF521 gene amplifications are present in >6% of OCs and its overexpression is associated with poor prognosis, suggesting that it may play an important role in OC. Increased ZNF521 expression resulted in an enhancement of OC HeyA8 and ES-2 cell growth and motility. Analysis of RNA isolated from transduced cells by RNA-Seq and qRT-PCR revealed that several genes involved in growth, proliferation, migration and tumor invasiveness are differentially expressed following increased ZNF521 expression. The data illustrate a novel biological role of ZNF521 in OC that, thanks to the early and easy detection by RNA-Seq, can be used as biomarker for identification and treatment of OC patients.\",\n",
            "        \"intro\": \"Introduction Epithelial ovarian carcinoma is the leading cause of gynecological cancer-related mortality, with 295,000 new cases and 184,000 deaths worldwide in 2020 [ 1 ]. Due to the lack of effective screening programs two-thirds of patients with OC are diagnosed with advanced disease. Despite a generally good response to surgery and first-line chemotherapy, OC almost inevitably relapses in different peritoneal sites and develops chemo-resistance. Only 45% of OC-affected women are likely to survive for five years, compared to 89% of breast cancer patients [ 2 , 3 ]. Despite increased radicality of debulking surgery and modifications of conventional chemotherapies there have been only modest gains for overall survival (OS) in OC in the last decades [ 4 ]. Based on their histological and morphological classification OC can be divided in serous, mucinous, endometrial, clear cells and transitional cells (depending on the tissue of origin involved). Moreover, OCs can be divided in type I (low-grade tumor with BRAF and KRAS mutations) or type II (high-grade tumor with a variety of mutations including HOX, PTEN, KRAS, AKT1, BRCA1/2 genes) [ 5 \\u2013 10 ] and various dysregulated mechanisms underlying the pathology [ 11 \\u2013 13 ]. An important therapeutic target in OC may be represented by a subpopulation of cells defined as ovarian cancer stem cells (OCSCs) [ 14 ] and considered a key factor in cancer initiation, relapse and resistance to therapy [ 15 , 16 ]. The stem cell-associated transcription co-factor, zinc finger protein 521 (ZNF521) is a transcription factor abundant in OC, where a high expression is associated with poor prognosis [ 17 , 18 ]. ZNF521 has been recently included in a list of top 15 genes associated to poor survival in serous cystadenocarcinomas [ 19 ]. A significant number of gene amplifications for ZNF521 are detected in ovarian cancers (~6%) [ 20 ]. This association with oncogenesis with a role in control of HSCs (Hematopoietic Stem Cells) [ 21 ], LSCs (Leukemic Stem Cells) [ 22 \\u2013 25 ] and in iNSCs (immortalized Neural Stem Cells) [ 26 \\u2013 28 ] as well as in mesenchymal differentiation [ 29 \\u2013 34 ], involves ZNF521 in the control of the Stem Cells (SCs) compartment and in Cancer Stem Cells (CSCs). ZNF521 has also been implicated in miRNA regulation [ 35 ] where miR-9, via targeting Zfp521 (murine ortholog to human ZNF521), could promote the neural differentiation of mouse bone marrow mesenchymal stem cells (MSCs) [ 36 ]. It is notable that breast and ovarian cancers with BRCA1 mutations are inhibited by miR-9 [ 37 ]. Regulation of ZNF521 was also found in gastric cancer cells by miRNA-204-5p [ 38 ] and in hepatocellular carcinoma by miR-802 [ 39 ]. Additionally, in ovarian cancer an integration analysis of microRNA and mRNA gave a positive association of ZNF521 with the miR34-family and miR133b [ 18 ]. As the mortality rate of this tumor has improved only marginally over the past decades, a better understanding of the molecular events that drive its development, maintenance and progression, and the identification of new biomarkers and therapeutic targets, are paramount to improve OC treatment and hence its prognosis. In this study the role of ZNF521 is investigated in the regulation of human ovarian carcinoma cells.\",\n",
            "        \"method\": \"Materials and methods Cell lines and culture conditions The differentiated papillary human ovary cystoadenocarcinoma cell line HeyA8, the human desmoplastic cerebellar medulloblastoma DAOY cells and the human embryonic kidney HEK293T cells were cultured in DMEM. The human ovarian adenocarcinoma cell line ES-2, the acute monocytic leukemia THP1 cells and the lymphoblastoid B (multiple myeloma) IM9 cells were cultured in RPMI. Cell culture medium are supplemented with 10% fetal bovine serum, 50U of penicillin and 50\\u03bcg of streptomycin/ml (Thermo Fisher Scientific, Milan, Italy) and cell lines were maintained at 37\\u00b0C in 5% CO 2 . Transfection and transduction of cell lines Lentiviral production was carried out in 100 mm tissue culture plates, where the cell line HEK293T were transfected with 10\\u03bcg of plasmid for FUIGW (control vector) or FUIGW-ZNF521 [ 40 ] plus lentiviral packaging plasmids (2\\u03bcg of pCMV-VSVG and 10\\u03bcg of pCMV-deltaR8-91), using the calcium phosphate method. Lentiviral media were collected at 24, 48 and 72 hours after transfection and supplemented with 6\\u03bcg/ml of polybrene [ 41 ] and ovarian carcinoma cell lines HeyA8 and ES-2 were stably transduced with the control vector FUIGW or FUIGW-ZNF521. Three rounds of transduction were performed by centrifuging the cells with lentivirus at 3200 rpm at 32\\u00b0C for 50min. Cells were 70\\u201380% positive for the transgene EGFP by FACS analysis and stably expressed the ZNF521 protein. The transduction was performed in three independent experiments and cells were further sorted for EGFP giving a homogeneous population over 90% positive. Proteins extracts and Western blotting HeyA8 and ES-2 cells were processed for nuclear and cytoplasmic extracts. Cells were scraped, resuspended in hypotonic lysis buffer 10mM HEPES pH7.9, 10mM KCl, 0.1mM EDTA, protease inhibitors (P8849, Sigma-Aldrich) and phosphatase inhibitor cocktails 2 and 3 (P0044, P5726, Sigma-Aldrich) and incubated on ice for 20 min. After the addition of 0.25% Igepal-630 (NP40) (Sigma-Aldrich), samples were centrifuged at 3000 rpm for 5 min and supernatants containing the cytoplasmic extracts were recovered. Nuclear pellets were resuspended in 20mM HEPES pH7.9, 0.4M NaCl, 1mM EDTA with protease and phosphatase inhibitors. After three cycles of vortex and ice, samples were centrifuged at 12000 rpm for 20 min and the supernatants containing the nuclear extracts were collected. Proteins were separated on 4\\u201312% NuPAGE Novex Bis-Tris Protein gradient polyacrylamide gels (Thermo Fisher Scientific) and blotted onto nitrocellulose. Membranes were quenched with 5% blotto (BioRad) [ 42 ] and the proteins detected with: rabbit anti-ZNF521 (EHZF S15 sc-84808, Santa Cruz Biotechnology, DBA, Milan, Italy) at 1:1000, rabbit anti- HDAC1 (H3284, Sigma) 1:10000. Secondary rabbit HRP antibody at 1:2000 (65\\u20136120 Thermo Fisher Scientific) was detected by the ImmunoCruz Western blotting luminal reagent (sc-2004, Santa Cruz, Biotechnology) and exposure to autoradiographic film (GE Healthcare, Milan, Italy). Cell proliferation assay Transduced cells (HeyA8 and ES-2) were seeded into 6 well plates at 2.0x10 5 cells/well in complete DMEM or RPMI respectively. Every 48 hours cells were trypsinized, counted and replated at 2.0x10 5 /well. The average of three independent cell counts, normalized to account for the dilutions performed in the individual cultures at each replating, was plotted against time to obtain a cumulative growth curve for each cell population. The growth of HeyA8 was also evaluated using the Real-Time Glo MT Cell Viability Assay (Promega, Milan, Italy) kit, which allows the evaluation of cell proliferation based on their ATP consumption. The cells were plated in 96 wells and then the substrate and the enzyme are added in a 1:1 ratio. The plate was then incubated at 37\\u00b0C and the readings (GloMax Explorer, Promega) taken at 0, 24 and 48 hours of growth. ES-2 proliferation was also measured by MTS [3-(4,5-dimethylthiazol-2-yl)-5-(3-carboxymethoxyphenyl)-2-(4-sulfophenyl)-2H-tetrazolium, inner salt; MTS(a)] colorimetric assay. Cells were seeded at 5.0x10 2 per well in 96-well plates in DMEM 10% FBS and assayed after 24, 48 and 72 hours of incubation. At each time point, 20\\u03bcl of MTS solution were added to the wells. After 4 hours of incubation at 37\\u00b0C the absorbance, which is proportional to the number of viable cells, was measured at \\u03bb = 490 nm (GloMax Explorer, Promega). All the experiments were performed in triplicate. Clonogenicity assay The transduced cells are trypsinized, counted and resuspended in a medium containing DMEM F12 (GIBCO, Milan, Italy), L-glutamine (Thermo Fisher Scientific) 1%, Pen/Strep (Thermo Fisher Scientific) 1%, B27 (GIBCO) 50x, 20ng/ml hEGF (PeproTech, DBA, Milan, Italy), 2 ng/ml hFGFb (PeproTech). For ES-2 cells 10\\u03bcg/ml of insulin (Sigma-Aldrich, Milan, Italy) and 4 \\u03bcg/ml of heparin (Sigma-Aldrich) are added and plated in ultra-low attachment 6 wells (Corning Inc., Milan, Italy). HeyA8 were plated at a concentration of 3.0x10 4 cells/well and ES-2 at concentration of 4.5x10 4 cells/well. After 7 days spheres formation was observed, the cell number for every culture was calculated and the size of spheres were estimated from acquired images (at 10x magnification) by ImageJ 1.51j8. All the experiments were performed in triplicate. Wound healing assay\\u2013migration assay Transduced cells were counted, seeded at the same density and cultured until >90% confluence in 6 well plates in appropriate medium with 10% FBS and then in DMEM or RPMI with reduced serum 0.5% FBS to block their proliferation. After 24 hours the monolayer of cells was scratched with a sterile 200\\u03bcl pipet tip, washed with PBS 1X and incubated at 37\\u00b0C in DMEM 10% FBS [ 43 ] Images were captured by phase contrast microscopy at 10x magnification, at the beginning and at regular intervals during cell migration. The migration of the cells was analysed and quantified by ImageJ 1.51j8. Experiments were performed in triplicate. Next generation sequencing\\u2013total RNA-Seq RNA concentration was determined by using Quant-IT RNA Assay Kit-High Sensitivity and a Qubit Fluorometer (Life Technologies) and its quality and integrity assessed with the Agilent 4200 Tapestation System (Agilent Technologies). Total RNA\\u2013seq procedures were performed as described previously [ 44 ]. Indexed libraries were prepared using 1 \\u03bcg of total RNA in technical and biological triplicate for each sample as starting material, with a TruSeq Stranded Total RNA Sample Prep Kit (Illumina Inc.). Libraries were sequenced (paired-end, 2x75 cycles) at a concentration of 8 pM/lane on NextSeq 500 platform (Illumina Inc.) [ 45 ]. Aliquots of the amplified RNA samples used for RNA-Seq remained available to be used to confirm by qRT-PCR the differential expression of selected genes. Fastq files were aligned to the hg38 genome assembly using STAR [ 46 ]. STAR gene counts were normalized applying the median of ratios method implemented in DESeq2 R package [ 47 ]. Briefly, the normalization process implies different steps: i) for each gene, a pseudo-reference sample is created and is equal to the geometric mean across all samples; ii) for every gene in a sample and for each sample, the ratios sample/ref are calculated; iii) the median value of all ratios for a given sample is taken as the normalization factor (size factor) for that sample; iv) for each gene in each sample the normalized count values is calculated dividing each raw count value by the sample\\u2019s normalization factor. For differential gene expression (DEG) analysis, we applied DESeq2 R package to normalized counts. Heatmaps were generated using Cluster 3.0 for Mac OS X (C Clustering Library 1.56, Tokyo, Japan) with uncentered correlation and centroid linkage, and Java TreeView software environment (version 1.1.6r4; http://jtreeview.sourceforge.net ). Differentially expressed genes were used as input for the Molecular Signature Database Investigate gene sets analysis [ 48 ]. Expression analysis by qRT- PCR cDNA was synthesized using SuperScript III reverse transcriptase at 42\\u00b0C and 2.5\\u03bcM random hexamers (Thermo Fisher Scientific) from 1\\u03bcg RNA previously prepared with Tri Reagent (Sigma-Aldrich) and verified by the NanoDrop 2000/2000c Spectrophotometer (Thermo Fisher Scientific) [ 49 ]. One cycle of 3 min at 95\\u00b0C was followed by 45 cycles of 10 sec at 95\\u00b0C, 10 sec at 60\\u00b0C and 10 sec at 72\\u00b0C, finishing with a melting curve, amplified with the iQ\\u2122 SYBR\\u00ae green super mix (BioRad, Milan, Italy) using the qRT-PCR amplifier QuantStudio3 (Applied Biosystems, Milan, Italy). Analysis of gene expression was calculated as 2 \\u2212ddCt and normalized for the housekeeping gene ( GAPDH ). Primers used in this study were as follows (5\\u2019-3\\u2019): h-ZNF521 were previously described [ 50 ]; h-CTSK-fwd CAGGGTCAGTGTGGTTCCTG , h-CTSK-rev CCCCGGTTCTTCTGCACATA ; h-RB1CC1 fwd TTCCACTGTTGGAGTGCCTAA , h-RB1CC1 rev CATCTGCGGTATCTGGGGAC ; h-AIMP1-fwd ACAGCAGTAACAACCGTATCTTCTGG , h-AIMP1-rev CTATTGGCTTAGAGTCGGCACTTCC ; h-PIBF1-fwd CTTACAAAGATTGAAGAATTGGAGG , h-PIBF1-rev AATTCTTGATATTTGCTGGCATCT ; h-FXYD5-fwd TCCCACTGATGACACCACGA , h-FXYD5 rev AAAACCAGATGGCTTGAGGGT ; h-HMMR-fwd ACCAACTCAAGCAACAGGAGGA , h-HMMR-rev CCTGAGCTGCACCATGTTCATT ; h-SAMD9-fwd GGGAACTACCTTGGCTATGCAC , h-SAMD9-rev CGTATTCCTGACGGTTCATTGCC ; h-GAPDH-fwd CACCATCTTCCAGGAGCGAG , h-GAPDH-rev TCACGCCACAGTTTCCCGGA . Experiments were performed in triplicate. Statistical analysis The student\\u2019s t-test assuming unequal variances between two samples was used to determine the significant differences (p <0,05 *, p <0,01 **, p <0,001 ***, p <0,0001 ****).\",\n",
            "        \"subjects\": null,\n",
            "        \"results\": \"Results Enforced expression of ZNF521 enhances growth of ovarian carcinoma cell lines ZNF521 is expressed in ovary and ovarian cancers [ 18 ] and mutations of its gene (almost invariably amplifications) are present in >6% of OCs ( S1A Fig ) [ 20 ]. Increased expression of ZNF521 was found associated with a poor prognosis in OC patients [ 17 \\u2013 19 ]. Analysis of a public genomic database [ 51 ] comprising 655 OC patients confirmed an adverse prognosis for patients with ZNF521-high OCs ( S1B Fig ) [ 52 ]. To assess whether the modulation of ZNF521 expression could affect some biological characteristics of OC cells the mRNA levels of ZNF521 for HeyA8 and ES-2 OC cells were quantified by qRT-PCR and compared to that of THP1 cells, DAOY cells and of IM9 that, respectively, express very high, intermediate and very low levels of ZNF521 ( Fig 1A ). The HeyA8 and ES-2 cell lines, were stably transduced with two different lentiviral vectors: FUIGW (carrying only the EGFP) or FUIGW-ZNF521 (expressing the ZNF521 cDNA fused to 3xFLAG tag and EGFP) and immunoblot analysis as well as qRT-PCR confirmed a significant increase of ZNF521 expression ( Fig 1B and 1C ). 10.1371/journal.pone.0274785.g001 Fig 1 Expression of ZNF521 in HeyA8 and ES-2 human ovarian carcinoma cell lines. Endogenous ZNF521 mRNA (A, top panel: qRT-PCR) and protein (A, bottom panel: Nuclear protein detected by Western Blotting) expression in human ovarian carcinoma cell lines (HeyA8 and ES-2) compared to THP1 cells, that express high levels of ZNF521, DAOY cells, that express intermediate levels and IM9 cells that express low levels of ZNF521. Modulation of ZNF521 by lentiviral transduction mRNA (B and C, top panel: qRT-PCR) and protein expression (B and C, bottom panel: Nuclear protein detected by Western Blotting) in HeyA8 (B) and ES-2 (C) cells transduced with lentiviral vectors containing the cDNAs for ZNF521 compared to the control (CTL). \\u201cT\\u201d in 1T, 2T or 3T indicate each \\u201cTransduction\\u201d. The two bands for ZNF521 correspond to the monomer and dimers at 150kDa and 300kDa identified by the S15 anti-ZNF521 antibody. The nuclear extracts are normalized for the HDAC1 protein. Asterisks indicate p <0,05 *, p <0,01 **, p <0,001 ***, p <0,0001 ****. Cell growth was investigated in adherent conditions and was measured by cumulative cell numbers ( Fig 2A and 2B ), MT ( Fig 2C ) and MTS assays ( Fig 2D ). In HeyA8 ( Fig 2A and 2C ), overexpression (OE) of ZNF521 induces a significant (p<0.01) increase in cell growth both at 48h (~1.5-fold increase; 1.06x10 5 ZNF521 vs 7.30x10 4 CTL cells) and after 8 days (~2-fold; 6.96x10 7 RLU in ZNF521 vs 3.4x10 7 RLU in CTL cells; MT assay) ( Fig 2B and 2D ). Similarly, exogenous ZNF521 expression in ES-2 increased cellular growth at 48h and up to 72 hours (~1.2-fold; 0.905 OD ZNF521 vs 0.745 OD CTL; MTS assay), and after 8 days (2.4-fold; 3.12x10 7 ZNF521 vs 1.3x10 7 CTL). 10.1371/journal.pone.0274785.g002 Fig 2 Forced ZNF521 expression confers an advantage in proliferation of OC cells. The growth of HeyA8 (A and C) and ES-2 (B and D) transduced cells was assessed by cell counts (A and B) or by MT (C) and MTS assays (D). Data shown the mean of 3 sets, each conducted in triplicate. Asterisks indicate p <0,05 *, p <0,01 **, p <0,001 ***. ZNF521 improves the migration ability of human ovarian cell lines The cell migration and invasion of ZNF521-overexpressing and CTL cells was investigated by wound-healing assay. Cells were plated and grown to confluence, in 6 well plates, and then starved for 24 hours. When cell confluence approximately reached 90%, a longitudinal wound was done using a sterile tip. Cells cultures were observed at regular time intervals, monitoring the extent of the healing of the wound. Both ZNF521-overexpressing cell lines displayed a ~2-fold increase in wound healing capacity ( Fig 3 ) compared to CTL cells. 10.1371/journal.pone.0274785.g003 Fig 3 Effect of enhanced ZNF521 expression on migration ability of HeyA8 and ES-2 cell lines. ZNF521-transduced HeyA8 (A and B) and ES-2 (C and D) displayed an increased wound healing ability compared to control cells. Scale bars correspond to 250\\u03bcm. As percentage of wound closure the mean of 3 sets, each conducted in triplicate, is illustrated (B and D). Asterisks indicate p <0,05 *, p <0,01 **, p <0,001 ***. ZNF521 enhances clonogenicity of HeyA8 and ES-2 cells The proliferation of HeyA8 and ES-2, ZNF521-overexpressing or CTL cells was assessed in anchorage-independent conditions to test their ability to form spheroids as readout for enrichment in CSC subpopulation in OC cell lines. Overall, ZNF521-overexpressing cells showed a greater number of spheroids and of cells forming spheres ( Fig 4A, 4B, 4D and 4E ) compared to the controls which confirmed an enrichment in CSCs. The average areas of spheres, from images of cultures acquired by phase contrast microscopy at 10x, were calculated for 50 spheres in each culture by ImageJ 1.51j8. 10.1371/journal.pone.0274785.g004 Fig 4 ZNF521 enforced expression enhances spheres-forming ability in HeyA8 and ES-2 cell lines. Representative images (10x) are shown for HeyA8 (A) and in ES-2 (D). Scale bars correspond to 250\\u03bcm. Lentiviral-mediated enforced expression of ZNF521 in ovarian cell lines increases spheres number and size in HeyA8 (B, C) and ES-2 (E, F). Average areas of the 50 spheres are shown. The data shown here illustrate the mean of 3 sets, each conducted in triplicate. Asterisks indicate p <0,05 *, p <0,01 **, p <0,001 ***. Notably, ZNF521 transduced cell lines formed larger spheres than the controls ( Fig 4C and 4F ). ZNF521 over-expression reprograms the transcriptome of HeyA8 and ES-2 cells Since ZNF521 is a transcription co-factor we then sought at the transcriptome of OC cell lines upon ZNF521 expression modulation. RNA-Seq analysis of both ES-2 and HeyA8 cell lines, ZNF521-OE or CTL, revealed that ZNF521 overexpression induced a large transcriptional reprogramming involving a total of 1375 and 742 differentially expressed genes (p<0.05), respectively ( Fig 5A ). Furthermore, Molecular Signature Database (MSigDB) focused on hallmark gene sets (see Methods ) scored the enrichment of 4 hallmarks related to cancer proliferation (i.e., E2F targets, EMT, G2M checkpoint and mitotic spindle) both in HeyA8 and in ES-2 cell lines ( Fig 5B and 5C ), which is in line with previously described enhanced proliferation and migration of HeyA8 and ES-2 cells upon ZNF521 overexpression (Figs 2 and 3 ). Yet, a total of 106 genes were found overlapping between the 1375 and 742 differentially expressed genes in ZNF521-OE cells (p<0.0001, Fisher\\u2019s exact test; Fig 5D ). Of these, 10 genes previously described to be associated to tumour growth, metastasis, cell migration and proliferation, were found consistently down- or up-modulated in ZNF521-OE cells vs CTLs ( Fig 5E ). Lastly, qRT-PCR analysis of 7 out of 10 genes confirmed their trend of regulation as shown in Fig 6 (A: HeyA8, B: ES-2) in both OC cell lines. 10.1371/journal.pone.0274785.g005 Fig 5 Differentially expressed genes (DEGs) by ZNF521 in hOC cell lines. (A) Hierarchical clustering of the 1375 and 742 differentially expressed genes (p<0.05) in HeyA8 and ES-2 cells, respectively. Molecular Signatures Database (MSigDB) analysis of HeyA8 (B) and ES-2 (C). (D) Size of dots represents the inverse logarithm (base 10) of statistical significance (FDR q-value) as per the legend. X-axes, fraction (k/K) of overlapping genes (k) in each Gene Set with the total number of genes represented in the specific hallmark (K). In bold, commonly enriched hallmark gene sets. Venn diagram analysis of the 1375 genes differentially expressed in HeyA8 (blu circle) and 742 genes were differentially expressed in ES-2 (red circle); 106 genes were differentially expressed in common in both cell lines. (E) Heatmap of DEGs specifically involved in growth, metastasis, migration and proliferation commonly regulated in both OC cell lines. FC, fold change (ZNF521 vs control). 10.1371/journal.pone.0274785.g006 Fig 6 Validation of RNA-Seq data by qRT-PCR: HeyA8 (A), ES-2 (B). The data shown here illustrate the mean of three experiments each conducted in triplicate. Asterisks indicate p <0,05 *, p <0,01 **, p <0,001 ***.\",\n",
            "        \"discussion\": null,\n",
            "        \"conclusion\": \"Discussion Ovarian carcinoma represents an important cancer-related cause of death in women, even considering that the overall survival has only modest advantage from radical debulking surgery and conventional chemotherapy [ 4 ]. The major unresolved clinical problems include malignant progression and rapid emergence of drug resistance against conventional chemotherapy. In this regard, novel strategies are warranted based on molecularly targeted therapies directed against components of specific signaling pathways required for tumour development and progression [ 53 ]. Additionally, identification of biomarkers with functional and prognostic significance, may greatly aid early diagnosis and allow timely implementation of therapeutic strategies. Profiling of 157 advanced stage serous ovarian cancers, ZNF521 (known regulator of the homeostasis in stem cells compartment and in cancer [ 18 , 22 \\u2013 28 ]) results in a set of 86-genes with significant poor overall survival profiles [ 17 ]. In exosomes derived from Bone Marrow Stromal Cells (BMSCs), ZNF521 was identified as hub protein and associated with poor outcome in gastric and ovarian cancers [ 18 ] where OCSC are shown to be implicated in the progression of ovarian carcinoma and resistance to therapy [ 14 \\u2013 16 ]. Moreover, recently, ZNF521 was identified as one of the top 15 genes significantly correlated with the overall survival in 1692 serous ovarian cystadenocarcinomas [ 19 ]. Taken together the analyses of different public genomic databases indicate that ZNF521 is: i) amplified in 6% of ovarian carcinoma cases and ii) associated to poor survival in ovarian cancer patients [ 18 \\u2013 20 ] ( S1 Fig ). Here, we highlight a new role of the transcriptional co-factor ZNF521 in OC. Overall, we found that higher levels of ZNF521 in HeyA8 and ES-2 human ovarian cancer cell lines enhances their growth ( Fig 2 ), ability to migrate ( Fig 3 ) and to form OC spheroids ( Fig 4 ). Such a role of ZNF521 in hOC may be exerted via the modulated expression of several regulatory genes which are relevant in cancer [ 54 , 55 ] ( Fig 5 and S1 \\u2013 S3 Tables). qRT-PCR analysis confirmed the regulation by ZNF521 of a subset of these genes by ZNF521 (N = 7; Fig 6 ) that, once dysregulated, induce cellular transformation and tumorigenesis. More in depth, the data presented so far demonstrate that ZNF521 is able to regulate the expression of HMMR ( Hyaluronan-mediated motility receptor ) [ 56 \\u2013 60 ], CTSK ( Cathepsin K ) [ 61 \\u2013 65 ], SAMD9 ( Sterile Alpha Motif Domain-Containing Protein 9 ) [ 66 , 67 ], AIMP1 ( Aminoacyl-tRNA synthases interacting multi-functional protein 1 ) [ 68 \\u2013 71 ], PIBF1 ( Progesterone-Induced Blocking Factor 1 ) [ 72 \\u2013 75 ], RB1CC1 ( RB1-inducible coiled-coil 1 ) [ 76 \\u2013 78 ] and FXYD5 [ 79 \\u2013 81 ]. All these genes are well known to regulate tumour progression (proliferation, migration and metastasis) by regulating EMT pathway in cancers and are also linked to poor overall survival in different types of diseases. Human OC HeyA8 and ES-2 cell lines overexpressing ZNF521 also increase their ability to form spheroids (enrichment of CSCs). CSCs have the ability to self-renew, differentiate and possess tumorigenic properties. CSCs are also involved in drug resistance, and it has been shown that some populations of CSCs share EMT-like cellular characteristics [ 82 ]. Indeed, whole-transcriptome profiling of OC cell lines confirmed the profound transcriptional impact of ZNF521 expression modulation on a set of genes involved in tumour growth, proliferation, migration and metastasis. Forced ZNF521 expression altered the transcriptional profile of a total of 1375 and 742 genes in HeyA8 and in ES-2 human OC cell lines, respectively, as well as the enrichment of several cancer-related hallmarks including EMT regulatory genes. Further in-depth analysis is necessary to better understand the molecular mechanism that underlines these activities of ZNF521 in human OC. Taken together, these data are the first evidence of new role of the stem cell transcription co-factor ZNF521 (so far known for its involvement in the stem pathway) in the regulation of human epithelial ovarian carcinoma (hEOC) cells, suggesting that this could be due to the modulation exerted by ZNF521 on regulatory genes of the relevant molecular mechanisms of EMT in human ovarian cancer.\"\n",
            "    },\n",
            "    {\n",
            "        \"pmcid\": \"PMC9616492\",\n",
            "        \"abstract\": \"Quiescent leukemic cells survive chemotherapy, with translation changes. Our data reveal that FXR1, a protein amplified in several aggressive cancers, is elevated in quiescent and chemo-treated leukemic cells and promotes chemosurvival. This suggests undiscovered roles for this RNA- and ribosome-associated protein in chemosurvival. We find that FXR1 depletion reduces translation, with altered rRNAs, snoRNAs, and ribosomal proteins (RPs). FXR1 regulates factors that promote transcription and processing of ribosomal genes and snoRNAs. Ribosome changes in FXR1-overexpressing cells, including RPLP0/uL10 levels, activate eIF2\\u03b1 kinases. Accordingly, phospho-eIF2\\u03b1 increases, enabling selective translation of survival and immune regulators in FXR1-overexpressing cells. Overriding these genes or phospho-eIF2\\u03b1 with inhibitors reduces chemosurvival. Thus, elevated FXR1 in quiescent or chemo-treated leukemic cells alters ribosomes that trigger stress signals to redirect translation for chemosurvival. Modification of ribosomal properties in therapy-treated AML drives production of survival proteins and thereby AML survival.\",\n",
            "        \"intro\": \"INTRODUCTION Cancer cells can enter a reversible arrest phase called quiescence or G0 that is resistant to harsh conditions including chemotherapy ( 1 , 2 ). We previously found that leukemic cells induced to G0 by growth factor deprivation not only are chemoresistant but also exhibit similar posttranscriptional changes, as that of leukemic cells surviving chemotherapy ( 1 ). This indicated translation of specific genes in these chemoresistant cells, which are needed for their chemosurvival. Understanding the altered translation program and how chemosurviving G0 cells translate specific genes can reveal undiscovered insights on chemoresistance and new strategies to reduce chemosurvival. Canonical translation initiation is mediated by two rate-limiting steps: cap-dependent recruitment of mRNAs and recruitment of the initiator transfer RNA (tRNA)/ternary complex for translation initiation ( 3 , 4 ). This conventional translation promotes proliferation-associated genes ( 5 ). One of the two rate-limiting steps of canonical translation initiation is inhibited by dephosphorylation of the canonical cap complex inhibitor, EIF4EBP (4EBP), due to low mammalian Target of Rapamycin (mTOR) activity ( 3 , 5 ). The second rate-limiting step of canonical translation initiation is inhibited by phosphorylation of the tRNA recruitment complex factor, eIF2\\u03b1. This is brought about by four eIF2\\u03b1 kinases (eIF2aks) that are triggered by various stress responses and cause the integrated stress response (ISR) ( 4 , 6 ). We found that eIF2\\u03b1 phosphorylation is enhanced in G0 and chemotherapy-treated cells ( 1 ), which can inhibit canonical translation at the tRNA recruitment step and enable specific genes to get translated. How G0 and chemotherapy-treated cells induce eIF2\\u03b1 phosphorylation, to suppress canonical translation and express specific genes that lead to chemosurvival and AML persistence, remains to be uncovered. Our previous data revealed that the RNA binding protein, fragile X mental retardation\\u2013related protein 1 (FXR1) ( 7 , 8 ), is elevated in serum-starved G0 acute monocytic leukemic (AML) THP1 cells. FXR1 has been shown to be important for tumor progression as it is amplified in several aggressive cancers, where posttranscriptional expression of specific mRNAs is altered ( 9 ). FXR1 is associated with ribosomes, translation, mRNA stability, and export and localizes in the nucleus, cytoplasm, and stress granules ( 7 , 8 ). In serum-starved G0 cells, we found that FXR1a splice isoform is enhanced and can promote specific mRNA translation ( 10 , 11 ). Given that FXR1 increases and promotes specific mRNA translation in G0 AML cells ( 10 , 11 ) that are chemoresistant ( 1 ), the role of FXR1 on chemosurvival via translation mechanisms needs to be uncovered to understand the impact of translation regulation in refractory cancers. In this study, we investigated the changes in translation, and the role of FXR1, in G0 and chemosurviving AML cells. Our findings demonstrate that as in serum-starved G0 cells that are chemoresistant, FXR1 increases in therapy-surviving AML cells. Consistently, we find that FXR1 depletion reduces chemosurvival, while FXR1 overexpression promotes chemosurvival. Our data reveal that FXR1 associates with ribosome regulators and alters ribosome components. These ribosomal changes trigger stress signaling via eIF2\\u03b1 kinase activation that causes eIF2\\u03b1 phosphorylation. This reduces canonical translation and permits translation of specific prosurvival and immune genes, leading to chemotherapy and immune survival. Pharmacological inhibition of this specific translation, or of translated prosurvival genes, suppresses chemosurvival, indicating new avenues to therapeutically target refractory AML.\",\n",
            "        \"method\": \"MATERIALS AND METHODS Cell culture THP1 cells were cultured in RPMI 1640 medium supplemented with 10% fetal bovine serum (FBS), 2 mM l -glutamine, streptomycin (100 \\u03bcg/ml), and penicillin (100 U/ml) at 37\\u00b0C in 5% CO 2 . SS or G0 THP1 cells were prepared by washing with phosphate-buffered saline (PBS) followed by serum starvation at a density of 2 \\u00d7 10 5 cells/ml. AraC-treated cells were prepared by treatment with indicated concentrations of AraC for indicated periods of time. THP1 (TIB-202) and monocytes (CRL9855) were obtained from the American Type Culture Collection (ATCC). NOMO1 and MOLM13 were obtained from ATCC and from the Scadden group ( 1 ). Cell lines were tested for mycoplasma (Promega) and authenticated by the ATCC Cell Authentication Testing Service ( 1 ). Plasmids TRIPZ and GIPZ plasmids expressing shRNAs against human FXR1, RPLP0, and control vector expressing miR30a primiR sequences (RHS4750) were obtained from Open Biosystems\\u2013Dharmacon (shRNA target sequences are in table S5). Stable cell lines were constructed as described previously ( 10 , 11 ). The stable cells expressing shRNA against FXR1 were induced with doxycycline (1 \\u03bcg/ml) for 3 days (once each day) to knock down FXR1. Control cells were treated similarly. THP1 FXR1 OE cell lines were created by transducing cells with pHAGE retroviral vector containing FXR1a for constitutive overexpression of FXR1a ( 10 , 11 ). Renilla was obtained and used as done previously ( 10 ). The 5\\u2032UTR of BCL6 mRNA was amplified (with primers flanked by Nhe I restriction sites). CX plasmid ( 10 ) and BCL6 5\\u2032UTR-containing amplicon were digested with Nhe I [New England Biolabs (NEB)] followed by ligation with T4 DNA ligase (NEB). Ligation mixes were transformed in Escherichia coli cells. Plasmids were purified from E. coli cells after transformation. Positive clones were confirmed by sequencing. DDX21 plasmid ( 71 ) and GUG and AUG luciferase reporters ( 72 ) were obtained from Addgene (p23-DDX21 WT was a gift from L.-L. Chen, Addgene plasmid #128803; pGL4.13/AUG-FFLuc-3XFLAG and pGL4.13/GUG-FFLuc-3XFLAG were gifts from J. Wilusz, Addgene plasmids #127333 and #127334). Polysome profiling and microarray Cells (30 \\u00d7 10 6 ) were grown for each sample and harvested on ice. In case of the FXR1 KD and its respective control, and FXR1 OE and its respective control, treated with ISRIB or dimethyl sulfoxide (DMSO), cycloheximide was not used. Sucrose was dissolved in polysome buffer containing 100 mM KCl, 5 mM MgCl 2 , 2 mM dithiothreitol (DTT), and 10 mM tris-HCl (pH 7.4). Sucrose gradients from 10 to 55% were prepared in ultracentrifuge tubes (Beckman) as previously described ( 1 ). Harvested cells were rinsed with ice-cold PBS and resuspended in polysome buffer with 1% Triton X-100 and murine ribonuclease (RNase) inhibitor (40 U/ml; NEB) for 20 min with intermittent tapping on ice for lysis. After centrifugation of cell lysates at 10,000 rpm for 10 min, supernatants were loaded onto sucrose gradients followed by ultracentrifugation (Beckman Coulter Optima L90) at 32,500 rpm at 4\\u00b0C for 80 min in the SW40 rotor. Samples were separated by density gradient fractionation system (Biocomp Piston Gradient Fractionation). RNAs were purified from heavy polysome fractions and whole-cell lysates. The synthesized complementary DNA (cDNA) probes from the WT Expression Kit (Ambion) were hybridized to Gene Chip Human Transcriptome Array 2.0 (Affymetrix) and analyzed by the Partners Healthcare Center for Personalized Genetic Medicine Microarray and BUMC facilities. Gene ontology (GO) analysis for differentially expressed translatome or proteome was conducted by DAVID 6.7 tools ( 73 ). Molecular signatures enriched in FXR1 OE, KD, and control cells were identified by gene set enrichment analysis (GSEA) ( 74 ). Western blot analysis Cells were collected and resuspended in lysis buffer containing 40 mM tris-HCl (pH 7.4), 6 mM MgCl 2 , 150 mM NaCl, 0.1% NP-40, 1 mM DTT, 17.5 mM \\u00df-glycerophosphate, 5 mM NaF, and protease inhibitors. Cell lysates were heated at 95\\u00b0C with 200 mM DTT and 1\\u00d7 SDS loading dye for 10 min. Samples were loaded onto 4 to 20% gradient SDS-PAGE (Bio-Rad) or 16% SDS-PAGE (Invitrogen), transferred to nitrocellulose membranes, and processed for immunoblotting. Antibodies against FXR1 (05-1529) (used for Western), actin (MAB1501), and tubulin (05-829) were from Millipore; NOLC1 (11815-1-AP), DDX21 (10528-1-AP), PolR1D (12254-1-AP), RPLP0 (11290-2-AP), RPLP2 (16805-1AP), RPL29 (15799-1-AP), RPL19 (14701-1-AP), vinculin (26520-1-AP), ATF4 (10835-1-AP), GADD34 (10449-1-AP), and FXR1 (13194-1-AP) (used for Immunoprecipitation) were from ProteinTech; PKR (3072S), P-PKR (2611S), EIF2\\u03b1 (9722S), P-EIF2\\u03b1 (3597S), and GCN2 (3302S) (used for immunoprecipitation) were from Cell Signaling Technology; c-Myc (ab32072), P-GCN2 (ab75836), GCN2 (ab134053), and ATF4 (ab23760) (used for Western blot) were from Abcam; and anti-puromycin [3RH11] (EQ0001) was from Kerafast. Mass spectrometry Multiplex quantitative proteomics analysis (TMT spectrometry) was conducted, as done previously ( 10 ), from THP1 leukemic cells and cell lines created that were treated as described. Quantitative RT-PCR Total RNA was extracted using proteinase K buffer and TRIzol (Invitrogen) as performed previously ( 10 ). The cDNA was synthesized from 1 \\u03bcg of RNA using M-MuLV Reverse Transcriptase (NEB) and random hexamer primer (Promega). qPCRs were run on LightCycler 480 Instrument II (Roche) using 2\\u00d7 SYBR green mix (Bio-Rad). All primers used are listed in table S5. ddPCR assay ddPCR was performed using purified cDNA obtained from reverse transcription of RNA isolated from cell lines. The assay development for ddPCR validation of target genes of interest was performed in compliance with the updated 2020 Minimum Information for Publication of Digital PCR Experiments (dMIQE) Guidelines. The ddPCR was performed in the Applied Biosystems 96-well Thermal Cycler (Thermo Fisher Scientific) in a final volume of 20 \\u03bcl. The optimized reaction mixture was prepared using 10 \\u03bcl of ddPCR EvaGreen Supermix (Bio-Rad), 1000 nM forward and reverse primers, and 1 \\u03bcl of diluted cDNA. Dilution of cDNA for different assays was performed as follows: FXR1 (4 ng/\\u03bcl), U3 (0.2 ng/\\u03bcl), and 28 S (0.04 ng/\\u03bcl). The prepared reaction mixture was transferred to the wells of DG8 cartridge. To generate the droplets, 70 \\u03bcl of droplet generation oil for EvaGreen (Bio-Rad) was added and the plate was loaded into the QX200 droplet generator (Bio-Rad). The droplet emulsions were then carefully transferred to a semi-skirted, PCR-clean 96 well plate (Eppendorf) using a multichannel pipette. Last, the plate was sealed using PX1 PCR plate sealer (Bio-Rad). Annealing temperature for each assay was determined using a temperature gradient as follows: FXR1 (57\\u00b0C), U3 (55\\u00b0C), and 28S (55\\u00b0C). The final thermal cycling conditions were as follows: initial enzyme activation at 95\\u00b0C (51% ramp) for 5 min, then 40 cycles of denaturation at 94\\u00b0C (51% ramp) for 30 s, and annealing/extension at optimized temperature for 1 min, followed by enzyme inactivation at 98\\u00b0C for 10 min and final hold at 4\\u00b0C until analysis. Droplets were analyzed using a QX 200 droplet reader (Bio-Rad), and data were acquired and analyzed with QuantaSoft analysis software (Bio-Rad). The number of target gene copies per 20 \\u03bcl of reaction was calculated from QuantaSoft data. Only samples with >10,000 droplets per well were included in the analysis. Three wells are analyzed in parallel for each sample. Mass spectrometry for RNA modification analysis Total RNA was isolated using TRIzol as described previously ( 1 , 10 ). Isolated RNAs were cleaned using an RNeasy kit (Qiagen). Poly(A)-containing RNAs were separated from the RNA pool using Poly(A) mRNA Isolation System IV (PolyATtract, Promega). The remaining nonpolyadenylated RNA was sent for nucleoside digestion and LC-MS/MS analysis to Arraystar Inc. ( www.arraystar.com/lc-ms-based-rna-modification-analysis-service-selected/isolated-rna/ ). Nascent translation level analysis Global translation was measured by metabolic labeling for a short period followed by PAGE and scintillation analysis. THP1 stable cell lines were grown in normal RPMI medium to prevent additional cellular stress from methionine-free medium. Nascent translation was analyzed using HPG, an amino acid analog of methionine containing an alkyne moiety that can be biotinylated by Click-iT chemistry (Thermo Fisher Scientific). Nascent protein translation labeling with HPG was followed by SDS-PAGE and Western analysis of labeled nascent proteins with HRP-streptavidin antibody. HPG was added as described in the manufacturer\\u2019s protocol, or alternatively, 100 \\u03bcCi of 35 S-methionine was added to 10 ml of cells. After incubation at 37\\u00b0C for 45 min, cells were washed once with PBS and lysed in buffer [40 mM tris-HCl (pH 7.4), 6 mM MgCl 2 , 150 mM NaCl, 0.1% NP-40, 1 mM DTT, and protease inhibitors]. The lysate was first separated by electrophoresis on an SDS-PAGE gel and then transferred to a nitrocellulose membrane by Western blotting with anti-biotin antibody, or the radiolabeled blot was exposed to a phosphoimager (GE Healthcare). These were quantified by ImageJ. Radiolabeled lysates were also measured by scintillation counter. Low dNTP RT-qPCR assay For analysis of 2\\u2032- O -methylation, RNA was prepared from indicated cells. Reverse transcription (as described above in qPCR) was performed with primers in the reverse orientation from the modification site on rRNA. Two different dNTP concentrations were used for each primer, low (0.025 mM) and high (2.5 mM). qPCR was done with the resulting cDNA using primers at forward and reverse orientation to the modified sites. Fold change in modification was calculated on the basis of difference of C t values in the low and high dNTP conditions ( 24 ). For analyzing pseudouridylation, first, the RNA was treated with CMC metho- p -toluene sulfonate [1-cyclohexyl-(2-morpholinoethyl) carbodiimide metho- p -toluene sulfonate] under alkaline conditions ( 21 , 23 , 24 ). This was followed by the process for low dNTP RT-qPCR, as described above for 2\\u2032- O - methylation. Primers are listed in table S5. In vivo crosslinking and immunoprecipitation In vivo crosslinking using 0.3% formaldehyde and nuclear-cytoplasmic separation were done as described earlier ( 1 , 10 ). Briefly, around 10 \\u00d7 10 6 to 15 \\u00d7 10 6 cells were harvested on ice followed by washing with cold PBS. Cell pellets were resuspended in hypotonic buffer [10 mM tris (pH 8), 1.5 mM MgCl 2 , and 10 mM KCl]. A syringe with 25-gauge \\u00d7 5 / 8 precision glide needle was used to lyse the cells by repetitive passes through the needle (up to 5 to 10 times to ensure lysis) or, alternatively, a douncer was used to appropriately lyse the cells. The decanted supernatant (cytoplasmic fraction) was centrifuged at 2000 rpm for 10 min at 4\\u00b0C. The supernatant was again decanted at 2000 rpm for 10 min, producing the final cytoplasmic fraction. The remaining pellet was washed with 10 times the pellet volume with buffer B [20 mM tris-HCl (pH 8.0), 25% glycerol, 1.5 mM MgCl2, 0.2 mM EDTA (pH 8.0), and 20 mM KCl]. After removal of the wash buffer, the pellet was again suspended in a mix of 10 times the pellet volume of buffer B, and five times the pellet volume of buffer C [20 mM tris-HCl (pH 8.0), 25% glycerol, 1.5 mM MgCl2, 0.2 mM EDTA (pH 8.0), and 1.2 M KCl], and the solution was mixed by vortex before a 45-min incubation with nutation at 4\\u00b0C. If the cells had been crosslinked, this fraction was sonicated for 5 s and then 30-s cooling on ice (six times, 90% duty cycle, output control at 2) and deoxyribonuclease I (Dnase I)\\u2013treated before being centrifuged at 2000 rpm for 10 min at 4\\u00b0C. The decanted supernatant was collected as the nuclear fraction. The supernatant was mixed with the cytoplasmic fraction. Where mentioned, only cytoplasmic fraction was used. The lysates were precleared of nonspecific binders by incubating with protein G (Santa Cruz Biotechnology) beads and IgG. The precleared lysates were incubated with antibodies and IgG (control) overnight in buffer (40 mM Hepes, 100 mM NaCl, 6 mM MgCl 2 , 0.025% NP-40, 1 mM DTT, 10% glycerol, 1 mM phenylmethylsulfonyl fluoride). Lysates were then incubated with equilibrated and blocked protein G beads for 2 hours. Beads were then pelleted and washed four times with radioimmunoprecipitation assay buffer. Beads were then used to analyze proteins or RNA after using heat to break Schiff\\u2019s linkages from formaldehyde, followed by proteinase K digestion buffer treatment for the fractions for RNA analysis, and RNase and Micrococcal Nuclease (to remove RNA and DNA) treatment for the fraction for protein analysis as described earlier ( 1 , 10 ). DEAE fractionation assay DEAE fractionation was performed with in vivo formaldehyde crosslinked extracts or Y10B antibody immunoprecipitates as done previously ( 10 ). Cell lysates or fractions after Y10B immunoprecipitation were incubated with equilibrated DEAE beads for 2 hours in buffer (40 mM Hepes, 6 mM MgCl 2 , 2 mM DTT, 10% glycerol, and 150 mM NaCl). After collection of the flow-through, beads were incubated with wash buffer of increasing salt concentrations (40 mM Hepes, 6 mM MgCl 2 , 2 mM DTT, and 10% glycerol, with NaCl ranging from 250 to 500 to 750 to 1000 to 2000 mM). RNA was isolated from each salt fraction. Amount of rRNA in each fraction was analyzed by qRT-PCR, normalized to input levels. Luciferase assay Plasmids containing firefly luciferase reporters downstream of AUG or GUG start sites ( 72 ) or the BCL6 5\\u2032UTR were co-nucleofected using Lonza Nucleofector ( 10 ) with Renilla luciferase in FXR1 OE, control vector, and THP1 cells. Nucleofected cells were then grown under conditions of S+, G0, and 5 \\u03bcM AraC. Cells were harvested, washed, and lysed in 1\\u00d7 passive lysis buffer (Promega). Luciferase activity in the lysates was analyzed using the Luciferase Assay System (Promega) as per the manufacturer\\u2019s instructions and as conducted previously ( 10 ). Harringtonine initiation block assay We used harringtonine to block initiation followed by a short time course to observe translated products from preexisting elongating ribosomes, with puromycin termination and labeling [SunRiSE method ( 61 )]. Cells were divided into six-well plates and treated with harringtonine (2 \\u03bcg/ml; Cayman Chemicals, catalog no. 26833-85-2). After indicated time of harringtonine treatment, cells were treated with puromycin (10 \\u03bcg/ml; Santa Cruz Biotechnology, catalog no. CAS-58-58-2) for 8 min. Cells were pelleted, washed with cold PBS, and lysed in lysis buffer. Lysates were fractionated on a 4 to 20% gradient SDS-PAGE followed by immunoblotting. Anti-puromycin antibody (Kerafast) at a concentration of 1:1000 was used for immunoblotting. ImageJ was used to quantify the whole lane as well as the puromycin labeling of smaller proteins that report differences first ( 61 ). Coculture of leukemic cells with immune cells followed by flow cytometry CRL9855 monocytes were tested as monocytes or differentiated to M0 macrophages with PMA (phorbol 12-myristate 13-acetate; 185 ng/ml) for 24 hours. M0 macrophages were then treated with interleukin-4 (20 ng/ml) and interleukin-13 (20 ng/ml) for 48 hours for polarization to M2, and with interferon-\\u03b3 (20 ng/ml) and tumor necrosis factor \\u03b1 (TNF\\u03b1) (20 ng/ml) for 48 hours for polarization to M1 ( 75 ). Control and FXR1 OE cells were stained with Far-Red CellTrace dye (Thermo Fisher Scientific) as per the manufacturer\\u2019s instructions. Macrophages were cocultured with stained control and FXR1 OE cells for 12 hours at a ratio of 1:2. Coculture cells were collected after trypsinization and washed with PBS. Non-cocultured control and FXR1 OE cells, stained with Far Red CellTrace, were used as controls. Harvested cells were thoroughly washed and stained with Hoechst 33342 (Thermo Fisher Scientific) as per the manufacturer\\u2019s instructions. Samples were filtered through a nylon mesh filter and analyzed for the population stained with both dyes by flow cytometry ( 76 ). Control cells without the Far Red CellTrace but stained with Hoechst were used as baseline, and the data were analyzed using FlowJo software. Cell migration assay Cell migration assay was performed as previously described ( 11 ). Transwell chambers (8 \\u03bcm pore, Corning) were preequilibrated with serum-free medium. CRL9855 monocytes (2 \\u00d7 10 4 per chamber) that were prestained for 1 hour per the manufacturer\\u2019s instructions with Far Red CellTrace dye (Thermo Fisher Scientific) were placed in the top chamber, and 700 \\u03bcl of control or FXR1 overexpression cells was placed in the bottom chamber. The chambers were incubated at 37\\u00b0C for 18 hours in 5% CO 2 . Cells on the upper surface of the filter were removed with a cotton swab. Migrated stained monocyte cells were observed in the bottom chamber and visualized using a microscope. Microscope images were taken, and the numbers of migrated cells were determined. Inhibitors Cytarabine (AraC) ( 77 ), trans-ISRIB ( 4 , 63 ), FX-1 ( 49 ), and toyocamycin ( 64 , 65 ) were obtained from Cayman Chemicals. FXR1 OE and control cells were treated with 1 \\u03bcM trans-ISRIB for 24 hours. For cell viability that was measured by trypan blue staining and cell counts ( 1 ), FXR1 OE and control cells were treated individually or with a combination of 5 \\u03bcM AraC and 1 \\u03bcM trans-ISRIB for 24 hours, 100 nM toyocamycin and 1 \\u03bcM AraC, and 10 \\u03bcM FX-1 and 5 \\u03bcM AraC. Motif, GO, GSEA, and RNAFOLD analysis Multiple Em for Motif Elicitation (MEME) software was used to search for 5\\u2032UTR elements as described earlier ( 1 , 78 ), using 5\\u2032 UTR sequences from GenBank. GO analysis for differentially expressed translatome or proteome was conducted by DAVID 6.7 tools ( 73 ), as described earlier ( 1 ) with our datasets. GSEA ( 74 ) was performed as described earlier ( 1 ) with our datasets. 5\\u2032UTR folding as shown in fig. S4 (A and B) was performed using RNAFold web server from the Vienna RNA package (rna.tbi.univie.ac.at/cgi-bin/RNAWebSuite/RNAfold.cgi) ( 79 ). Statistical analyses Each experiment was repeated at least three times. Sample sizes were estimated on the basis of availability and previous experiments ( 10 ). No samples were excluded from analyses. Statistical tests were conducted for each figure. SEM values are shown as error bars in all figures. P values less than 0.05 were indicated with an asterisk. E values were used for the statistical significance in the motif analysis. Statistical analyses for the datasets were performed as described. For the microarray data, signed linear fold changes were used (e.g., +2 = twofold higher in FXR1 KD or OE than in control; \\u22122 = twofold lower in FXR1 KD or OE than in control). Paired t tests (including replicate as a covariate) were performed for each gene between experimental groups to obtain a t statistic and P value for each gene. t statistics and nominal P values for each effect in the two-factor linear model, and t statistics and nominal P values for each pairwise comparison were generated. Benjamini-Hochberg false discovery rate (FDR) correction was applied across all genes to obtain FDR-corrected P values (FDR q values), which represent the probability that a given result is a false positive based on the distribution of all P values on the array. FDR q (filtered) values were recomputed separately for the two-factor model or each pairwise comparison, across only the genes that were expressed above the median value of at least as many samples as in the smallest experimental group involved in the model/comparison. NES is normalized enrichment score for GSEA. For mass spectrometry data in other tables, SD, SEM, confidence interval (CI), P value derived by Student\\u2019s t test, P value derived by t test with unequal variance, and P value derived by Welch\\u2019s test were derived using Excel.\",\n",
            "        \"subjects\": null,\n",
            "        \"results\": \"RESULTS FXR1 is elevated in cytarabine-treated, chemosurviving AML cells We previously found that both serum-starved G0 THP1 AML cells where FXR1 increases ( 10 ) and chemosurviving AML cells show similar translation factor changes compared to untreated, proliferating AML [ Fig. 1, A and B , mass spectrometry data ( 1 )]\\u2014which may underlie their common ability to survive chemotherapy ( 1 ). Therefore, we asked the question whether FXR1, an RNA and translation regulator, could be amplified in chemotherapy-treated cells. We tested FXR1 levels by Western blot analysis in cytarabine (AraC) chemotherapy-treated cells ( 1 ) at clinically relevant concentrations ( 12 , 13 ). We find that FXR1 is transiently elevated in AraC-treated THP1 cells (AraC; Fig. 1C and fig. S1A), as in serum-starved G0 cells ( 10 ) that are chemoresistant ( 1 ). This was also observed upon AraC treatment of another AML cell line, NOMO1 (fig. S1B), indicating that FXR1 elevation upon AraC treatment was not unique to THP1 cells. These data reveal transient elevation of FXR1 levels upon AraC therapy treatment in AML. Fig. 1. FXR1 is required for translation. ( A and B ) Graphs showing proteomic levels [TMT spectrometry from ( 1 )] of commonly up-regulated RPs and regulators including the RNA- and ribosome-associated regulator, FXR1. (A) THP1 cells grown in serum-starved medium (G0) for 4 days. (B) THP1 cells treated with cytarabine (AraC) at 5 \\u03bcM [within clinical range ( 12 , 13 )] for 3 days, as conducted previously ( 1 ). ( C ) FXR1 levels over time of AraC (1 \\u03bcM) treatment. FXR1 runs as multiple bands that include posttranslational modifications and splice isoforms ( 7 , 8 ). ( D and E ) AraC survival after 24-hour treatment, observed by trypan blue stain exclusion cell counts, (D) in FXR1 overexpression (FXR1 OE) compared to control vector, and (E) in FXR1 knockdown (FXR1 KD) compared to control shRNA cells; Western blot below of FXR1, with tubulin and actin as loading controls. ( F ) Comparison of global translation by labeling nascently translated proteins in FXR1 KD compared to shRNA control cells, and in FXR1 OE compared to control vector cells, with l -homopropargylglycine (HPG), an amino acid analog of methionine containing an alkyne moiety that is biotinylated by Click-iT chemistry (Thermo Fisher Scientific), followed by SDS-PAGE and HRP-streptavidin Western analysis; Western analysis of FXR1 levels below and quantification of labeled protein levels on the right. Data are average of three replicates \\u00b1 SEM. See also fig. S1. THP1 cell survival, upon treatment with AraC chemotherapy, is promoted by FXR1 overexpression and reduced by FXR1 depletion To investigate whether FXR1 plays a role in chemosurvival of G0 and AraC-treated cells where it is elevated, we used a previously constructed THP1 cell line for FXR1 depletion ( 10 , 11 ). This is stably transduced with a doxycycline-inducible short hairpin RNA (shRNA) lentiviral vector (FXR1 KD), compared to a parallel control shRNA-expressing cell line. These cells are induced with doxycycline to express the shRNA for 3 days to effectively deplete all FXR1 isoforms ( 10 , 11 ). We previously observed that FXR1a isoform increases in G0 THP1 cells ( 10 ). Therefore, we also constructed a THP1 cell line that constitutively overexpresses FXR1a (FXR1 OE) compared to a control vector cell line. Western blot analysis shows that FXR1 was effectively depleted or overexpressed ( Fig. 1, D and E , Western blots). These were treated with AraC for 24 hours to test the impact of FXR1 levels on chemosurvival. Consistent with its increase in AraC-surviving cells, we find that FXR1 overexpression promotes chemosurvival by 1.9-fold, while FXR1 depletion reduces chemosurvival to less than 50% ( Fig. 1, D and E , graphs). These data suggest that FXR1 promotes chemosurvival of THP1 cells. FXR1 depletion decreases, while overexpression promotes, translation We next explored the mechanism of how FXR1 may promote chemosurvival. To investigate the effect of FXR1 on translation, we performed nascent translation labeling analysis in FXR1 KD compared to control shRNA cells and in FXR1 OE compared to vector control cells. We used l -homopropargylglycine (HPG), an amino acid analog of methionine containing an alkyne moiety that can be biotinylated by Click-iT chemistry (Thermo Fisher Scientific), for labeling nascently translated proteins, followed by SDS\\u2013polyacrylamide gel electrophoresis (SDS-PAGE) and horseradish peroxidase (HRP)\\u2013streptavidin Western analysis and quantification. We verified this by 35 S-Met incorporation ( 14 ) to label nascently translated proteins in FXR1 KD cells followed by SDS-PAGE. We find that FXR1 depletion decreases protein synthesis by almost half, compared to control shRNA cells ( Fig. 1F and fig. S1C). Conversely, FXR1a overexpression increases nascent translation by 1.7-fold compared to control vector cells ( Fig. 1F ). These data suggest that FXR1 is needed for translation. FXR1 depletion decreases, while overexpression increases, ribosomal RNAs Given the decrease in global translation, we investigated whether FXR1 levels alter ribosome biogenesis. Ribosome biogenesis includes ribosomal RNA (rRNA) transcription and processing, rRNA modification, and ribosomal proteins (RPs) and their assembly, mediated by several regulators ( 15 \\u2013 18 ). Analysis of rRNAs by quantitative polymerase chain reaction (qPCR) and droplet digital PCR (ddPCR) revealed a significant decrease in 28 S , 18 S , and 5.8 S rRNAs upon FXR1 depletion, compared to control shRNA cells ( Fig. 2A and fig. S2A). Conversely, 28 S , 18 S , and 5.8 S rRNAs increase upon FXR1 overexpression compared to control vector cells. FXR1 depletion also reduces, while overexpression increased, the RNA Polymerase I (Pol I) rRNA precursor 45 S rRNA ( Fig. 2A ), indicating that ribosome gene transcription and processing are affected by FXR1. Apart from Pol I\\u2013derived 28 S , 18 S , and 5.8 S rRNAs, the fourth Pol III\\u2013transcribed 5 S rRNA is also decreased and enhanced with FXR1 ( Fig. 2A ), suggesting that FXR1 may be altering a common Pol I and Pol III regulator and/or multiple ribosome biogenesis regulators. These data are consistent with decreased translation in FXR1 KD cells and enhanced translation in FXR1 OE cells ( Fig. 1F ) and suggest that FXR1 levels affect ribosome biogenesis. Fig. 2. FXR1 regulates rRNAs and snoRNAs and associates with ribosome and snoRNA regulators. qPCR of rRNAs and snoRNAs normalized to actin mRNA: ( A ) 45 S , 18 S , 28 S , 5.8 S , and 5 S ; ( B ) snoRNAs U3 and U8 in FXR1 KD compared to control cells and in FXR1-OE compared to control cells. ( C ) SnoRNAs from normalized microarray profiles in FXR1 KD G0 cells compared to control cells (red) and G0 cells compared to proliferating cells (blue) (table S1, A to C). ( D ) 2\\u2032- O -methylation assay by low dNTP RT-qPCR, in FXR1-OE cells compared to control cells, of 28 S rRNA site (G4198m) modified by snoRD58A (table S1, A to C). ( E ) Pseudouridylation of 28 S rRNA (U4966 and U4975) by snoRA22 (table S1, A to C), in AraC-treated, G0, and S+ cells, measured by CMC treatment followed by low dNTP RT-qPCR. ( F ) Mass spectrometry of pseudouridine in rRNA from G0 FXR1 KD compared to control cells (table S1D). In vivo formaldehyde crosslinking-coupled FXR1 immunoprecipitation followed by Western blots of FXR1, ( G ) NOLC1, ( H ) DDX21, and ( I ) qPCR of partial rescue of 45 S rRNA by DDX21 overexpression in FXR1 KD cells. ( J ) Western blots of c-MYC in FXR1-OE versus control cells. Western blots of POLR1D in ( K ) FXR1 KD G0 and S+ cells and ( L ) FXR1-OE versus control cells. ( M ) qPCR for PolR1D mRNA in FXR1 immunoprecipitates compared to IgG from in vivo\\u2013crosslinked G0 cells. Data are average of three replicates \\u00b1 SEM. See also fig. S2 and tables S1 and S2. Specific snoRNAs, involved in rRNA processing and modification, are elevated in G0 or FXR1-overexpressing cells and decrease upon FXR1 depletion Ribosome biogenesis involves RNA regulators called small nucleolar RNAs (snoRNAs) that process or modify rRNAs. SnoRNAs involved in rRNA processing, such as U3 (snoRD3) and U8 (snoRD118), bind RNA-protein complexes (RNPs) that are required to cleave and process the 47 S rRNA precursor into mature 18 S , 28 S , and 5.8 S rRNAs. Other snoRNAs bind modification enzymes and RNA binding proteins to form snoRNPs that are known to base pair and modify rRNAs at specific sites. Box C/D snoRNAs (snoRD) recruit fibrillarin enzyme to cause 2\\u2032- O -methylation, while Box H/ACA snoRNAs (snoRA) recruit Dyskerin enzyme to pseudo-uridylate rRNA sites. These modifications affect ribosome interactions and are important for translation ( 16 , 19 \\u2013 22 ). We first examined levels of snoRNAs involved in rRNA cleavage, U3 and U8. RNA profiling in G0 and proliferating cells revealed that U3 (snoRD3) increases in G0 cells (table S1, A to C) ( 1 , 10 , 11 ). Consistently, qPCR analysis reveals that U3 and U8 decrease upon FXR1 depletion and are enhanced upon FXR1 overexpression ( Fig. 2B and fig. S2A), which can affect 45 S , 28 S , 18 S , and 5.8 S rRNA levels in Fig. 2A . Profiling also revealed that several snoRNAs that modify distinct rRNA sites are elevated in G0 compared to proliferating cells (table S1, A to C) and decrease in RNA profiles of FXR1 KD cells compared to control shRNA cells ( Fig. 2C and table S1, A to C). These data are consistent with the impact of FXR1 observed on translation and rRNA levels ( Figs. 1F and 2A ). SnoRNAs regulated by FXR1 alter rRNA modification To test the outcome of FXR1 regulation of snoRNA levels, we analyzed FXR1 KD or OE cells for 2\\u2032- O -methylation [by low deoxynucleotide triphosphate (dNTP) reverse transcription qPCR (RT-qPCR)] and pseudouridylation [after N -cyclohexyl- N \\u2032-(2-morpholinoethyl) carbodiimide metho- p -toluenesulfonate (CMC metho- p -toluene sulfonate) treatment followed by low dNTP RT-qPCR ( 21 , 23 , 24 )] at the rRNA sites of specific snoRNAs that increase in G0 cells but decrease in G0 FXR1 KD cells (table S1, A to C). To globally examine rRNA modification changes, with or without FXR1, we enriched rRNA [devoid of poly(A) RNA and RNAs fractionated, to remove tRNAs and other RNAs less than 200 nucleotides (nt)] from FXR1 KD G0 and control shRNA G0 cells and subjected the nucleosides to liquid chromatography\\u2013mass spectrometry (LC-MS) analysis (fig. S2B and table S1D). Our data show changes in modification at rRNA sites targeted by the snoRNAs that are increased in G0 and regulated by FXR1. As shown in Fig. 2D , the target modification site on 28 S rRNA (G4198m) of the box C/D (2\\u2032- O -methylation eliciting, fibrillarin-bound) snoRNA, snoRD58A, shows enhanced 2\\u2032- O -methylation in FXR1 OE cells, consistent with snoRD58A elevation in G0 cells where FXR1 increases, and its decrease in FXR1 KD cells ( Fig. 2C ). snoRA22 is a H/ACA Dyskerin-bound snoRNA that causes pseudouridylation, which is elevated in G0 (table S1, A to C); in accord, rRNA pseudouridylation at snoRA22 target sites on 28 S rRNA (U4966, U4975) is enhanced in AraC-treated cells ( Fig. 2E ). Consistent with the decrease of snoRA22 and other such pseudouridylation-causing, Dyskerin-associated, H/ACA snoRNAs, upon FXR1 depletion (table S1, A to C), we find that FXR1 knockdown reduces rRNA pseudouridylation by mass spectrometry, along with alterations in other known rRNA modifications ( Fig. 2F , fig. S2B, and table S1D). The rRNA sites targeted by these FXR1-regulated snoRNAs can affect tRNA interaction and translation ( 25 \\u2013 29 ). These data suggest that FXR1 regulates levels of specific snoRNAs in G0, which can affect rRNA levels and modifications, to alter translation in these chemosurviving cells. FXR1 interacts with rRNA and snoRNA regulators and modulates the levels of ribosome gene transcription factors To identify how FXR1 affects snoRNAs and rRNAs, we examined FXR1-interacting regulators by in vivo crosslinking G0 cells followed by FXR1 immunoprecipitation and tandem mass tag (TMT) spectrometric analysis ( 10 ) of coimmunoprecipitates (table S2). We find that FXR1 interacts with multiple ribosome- and translation-related proteins in G0 (fig. S2C). In vivo crosslinking-coupled FXR1 immunoprecipitation in control or FXR1 OE cells reveals FXR1 association with rRNAs (fig. S2D). These data suggest that FXR1 may interact with ribosome regulators. We find that snoRNA and ribosome regulators, NOLC1 and DDX21, coimmunoprecipitated with FXR1 ( Fig. 2, G and H ; fig. S2C; and table S2), indicating that FXR1 may affect ribosome levels through interactions with these ribosome regulators. NOLC1 is a snoRNA and ribosome regulator that is involved in their assembly and biogenesis ( 30 , 31 ). DDX21 ( 32 , 33 ) binds and affects the roles of snoRNAs that control rRNA modification and processing. DDX21 also promotes ribosome transcription via Pol I for rRNAs and Pol II for snoRNA and RP expression ( 34 ). Consistently, DDX21 overexpression in FXR1 KD cells, where rRNA levels decrease ( Fig. 2A ), partially rescued the rRNA defect by increasing 45 S rRNA levels ( Fig. 2I ). In addition, FXR1 has been recently shown to bind and regulate the 3\\u2032 untranslated region (UTR) of c-MYC mRNA ( 35 ), a major ribosome gene transcription regulator that affects all three polymerases and regulates ribosome biogenesis ( 36 ). Consistently, we find that c-MYC increases upon FXR1 overexpression ( Fig. 2J ), which could lead to enhanced ribosome biogenesis. Since FXR1 affects both Pol I transcript 45 S rRNA and Pol III transcript 5 S rRNA ( Fig. 2A ), we hypothesized that a common component of both Pol I and Pol III complexes may be affected by FXR1 and G0 chemoresistant cell conditions, leading to this coordinated impact on both Pol I\\u2013 and Pol and Pol III components are increased in G0 and are affected in FXR1-depleted or FXR1-overexpressed cells. POLR1D is an essential component of both Pol I and Pol III complexes and is needed for DDX21 to locate to the nucleolus ( 32 , 34 ); thus, POLR1D can affect 45 S and 5 S rRNAs, and DDX21 functions on snoRNAs, Pol I, and Pol II. Mass spectrometry dataset from G0 and AraC-treated cells reveals that POLR1D increases in G0 and AraC-treated cells (table S3A) but is reduced upon FXR1 depletion (fig. S2E and table S3, A to D, sheet d). Consistently, we find that POLR1D is reduced and overexpressed with FXR1 knockdown or overexpression, respectively ( Fig. 2, K and L ). Accordingly, we find that POLR1D mRNA binds FXR1 ( Fig. 2M ), indicating that FXR1 associates with POLR1D mRNA to modulate levels of this rRNA transcription regulator. These data suggest that FXR1 interacts with or regulates levels of multiple ribosome biogenesis factors to alter ribosomes in G0 and AraC-treated cells where FXR1 is elevated. RP levels are altered in G0 cells, AraC-treated cells, and FXR1-overexpressing cells SnoRNAs, rRNAs, and their regulators are modulated by FXR1 that increases in G0 and AraC-treated cells, where RPs and regulators are also altered ( Figs. 1, A and B , and 2 and table S3, A to H). Many RPs modulate rRNA processing ( 16 \\u2013 18 ), and RP genes are regulated transcriptionally by DDX21 and c-MYC that are modulated by FXR1 ( Fig. 2, H to J , and tables S1 and S3, A to G). In addition, we find that some RPs interact with FXR1 in G0 (fig. S2C and table S2), consistent with previous studies that showed that RPs interact with FXR1 ( 7 ). FXR1 amplification in G0 and AraC-treated cells could alter RPs to elicit changes in ribosome complexes ( 37 ) and translation output; therefore, we examined RPs upon FXR1 depletion and overexpression. We find that a specific set of RPs are commonly increased or decreased, in both serum-starved G0 and AraC-treated cells in proteomic analyses [ Fig. 1, A and B , and table S3B; TMT mass spectrometry G0 and AraC data from table S3A ( 1 )], indicating that common changes in RP composition occur in these chemoresistant cells. Some RPs that increase in G0 and AraC (table S3B; TMT mass spectrometry G0 and AraC) decrease upon FXR1 depletion (FXR1 KD cells compared to control shRNA cells) or are enhanced upon FXR1 overexpression (FXR1 OE cells compared to control vector cells), as seen by qPCR ( Fig. 3A , RPL8 and RPS14), Western blot ( Fig. 3, B to D , RPLP0, P2, L19, and L29), or profiling analyses [fig. S3A and table S3, C, E, and G: RNA profiling data in FXR1 KD compared to control shRNA cells in table S3C from table S1A, proteomic data of FXR1 KD compared to control shRNA cells in table S3D, and translatome data from FXR1 KD compared to control shRNA cells in table S3G from table S3F ( 10 , 11 )]. These data suggest ribosome changes in cells with elevated FXR1 levels, as in G0 and AraC-treated cells, which can affect ribosome complexes and translation. Fig. 3. eIF2\\u03b1 phosphorylation and noncanonical start site translation increase with altered ribosomal components upon FXR1 overexpression. ( A ) RP levels by qPCR normalized to actin mRNA in FXR1 KD and in FXR1-OE compared to control cells (table S3C). Western blots of ( B ) RPLP0, ( C ) RPLP2, and ( D ) RPL19 and RPL29 in FXR1-OE and control cells. ( E ) Y10B immunoprecipitation from in vivo\\u2013crosslinked cytoplasmic extracts, followed by DEAE fractionation to analyze ribosome complex migration in salt fractions in G0 and AraC-treated cells compared to S+ cells by qPCR for 18 S and 28 S rRNAs (representing 80 S ribosomes) comigration. ( F ) DEAE fractionation and qPCR of rRNAs of Y10B-immunopurified ribosome complexes (18 S , 28 S rRNAs) in the 1 M fraction normalized to 500 mM fraction, from G0, and AraC-treated cells, versus S+ cells; FXR1-OE cells versus control cells. ( G ) Western blots of phospho-PKR in FXR1-OE, AraC-treated versus control cells. ( H ) snoRA2A and snoRD46 overexpression, followed by Western blot for phospho-eIF2\\u03b1. Western blot of ( I ) phospho-GCN2 and ( J ) phospho-eIF2\\u03b1 in FXR1-OE versus control cells. ( K ) GCN2 immunoprecipitation, followed by rRNA qPCR; Western blot of RPLP0 and GCN2 (In = 10% input), in FXR1-OE versus control cells. ( L ) Western blot of phospho-eIF2\\u03b1 in FXR1 KD versus control cells. ( M ) Western blot of phospho-GCN2 and phospho-eIF2\\u03b1 in FXR1-OE cells transfected with control or RPLP0 shRNAs. ( N ) Translation ratios of GUG start site luciferase over AUG reporter normalized to cotransfection Renilla. G0, AraC-treated compared to S+ cells. ( O ) FXR1-OE compared to control cells. Data are average of three replicates \\u00b1 SEM. See also fig. S3 and table S3. Distinct ribosome complexes are observed in G0 or AraC-treated cells, compared to proliferating, untreated cells, but are similar to ribosome complexes in FXR1-overexpressing cells As rRNA processing, RP level, and modification changes are induced in G0 by FXR1 overexpression ( Figs. 2 and 3, A to D ), we analyzed ribosome complexes formed in AraC-treated and serum-starved G0 cells to test whether they are distinct compared to proliferating cells. We also compared these with ribosome complexes in FXR1-overexpressing cells compared to control vector cells, as they show similar chemosurvival and ribosome changes to G0 and AraC-treated cells. These cells were first subject to formaldehyde crosslinking to freeze in vivo complexes. Cytoplasmic extracts were prepared to avoid nuclear, preribosome complexes that are not assembled. To identify changes in ribosome complexes, the extracts were bound on an anionic diethylaminoethanol (DEAE) column and eluted by increasing salt concentrations to separate complexes ( 10 ). Fractions of the in vivo crosslinked complexes that contained both 18 S and 28 S rRNAs, depicting small and large subunits, were examined as composite 80 S ribosome complexes. We observed one peak at 500 mM salt in untreated, serum-grown cells; in contrast, we find a second distinct peak (1 M salt) of ribosome complexes in G0 and AraC-treated cells (fig. S3B). Strikingly, we observed similar two peaks, eluting in the same salt fractions (500 mM, 1 M), in FXR1-overexpressing cells compared to one (500 mM) in vector control cells (fig. S3B). These data suggest changes in ribosome complexes that could affect translation and chemosurvival that is commonly observed in G0, AraC-treated, and FXR1-overexpressing cells, but not in untreated, control vector cells ( Figs. 1 and 2 ). We wanted to ensure that these complexes separated on DEAE are enriched for ribosomes. Therefore, we immunopurified ribosomes from G0 and AraC-treated cells compared to untreated, serum-grown cells, as well as FXR1-overexpressing cells compared to control vector cells, and then examined them by DEAE fractionation. These cells were first subject to formaldehyde crosslinking to freeze in vivo ribosome complexes. Ribosome complexes were purified from cytoplasmic extracts (to avoid nuclear, preribosome complexes) by Y10B, an antibody that recognizes 5.8 S complexes in assembled 80 S complexes (fig. S3C) ( 38 ). Y10B immunopurification of assembled 80 S ribosomes was verified by qPCR analysis of enrichment of 5.8 S rRNA and Western analysis of ribosomal protein RPLP0 in Y10B immunoprecipitates compared to immunoglobulin G (IgG) control (fig. S3, D and E). Lack of unprocessed 45 S rRNA in Y10B immunoprecipitates verified the absence of partially processed, nuclear preribosomes in Y10B immunoprecipitates of 80 S cytoplasmic ribosome complexes (fig. S3D). To test whether Y10B-purified ribosome complexes migrate distinctly in G0, AraC-treated, and FXR1-overexpressing cells, compared to control vector or untreated, serum-grown cells, the Y10B immunoprecipitates were fractionated over DEAE and eluted with increasing salt. These were examined for fractions that showed both 18 S and 28 S rRNAs depicting small and large subunit rRNAs, as composite 80 S ribosome complexes, by qPCR analysis. While untreated, serum-grown cells showed a single peak at 500 mM salt, a second complex is observed at 1 M salt in G0 and AraC-treated cells ( Fig. 3, E and F ), consistent with the ribosome component changes observed in these conditions in Figs. 2 and 3 (A to D) and in fig. S3B. We observe a similar pattern of two distinct peaks of ribosome complexes seen in G0 and AraC-treated cells, eluting in the same salt fractions (500 mM, 1 M) in FXR1-overexpressing cells compared to the single lower salt peak in control vector cells ( Fig. 3F ). These data suggest that in FXR1 overexpression conditions that include exogenously overexpressed, G0, and AraC-treated cells, ribosomal complex changes are observed. While distinct RNPs associated with the ribosome, or partially formed 80S could cause altered complex migration, these data suggest that ribosomes are differentially bound or comprised in FXR1-overexpressing cells and migrate in fractions that are similarly observed in G0 and AraC-treated cells, but not in untreated, proliferating cells. Altered P stalk proteins and snoRNAs in G0 cells, AraC-treated cells, and FXR1-overexpressing cells enhance eIF2\\u03b1 phosphorylation These multiple changes on the ribosome can alter many downstream mechanisms to affect translation; one way is by activating stress signaling pathways ( 39 \\u2013 42 ). G0 and AraC-surviving cells where FXR1 increases show inhibition of canonical translation with increased phosphorylation of eIF2\\u03b1 ( 1 ). This is brought about by eIF2\\u03b1 kinases that respond to multiple stress signals and can be induced by ribosomal component changes elicited by increased FXR1: via enhanced snoRNAs ( 43 , 44 ) and via ribosome changes in the P stalk RPs ( 39 , 40 ). FXR1 enhances levels of several snoRNAs (table S1, A to C) that have been shown to bind RNA-activated protein kinase (PKR) eIF2\\u03b1 kinase (eIF2ak2), a double-stranded RNA (dsRNA) binding protein ( 43 , 44 ) that causes eIF2\\u03b1 phosphorylation. We hypothesized that increased snoRNAs by FXR1 amplification, as in G0 and AraC-treated cells, could, in part, lead to eIF2\\u03b1 phosphorylation, as snoRNAs have been shown to activate eIF2ak2 or PKR ( 43 , 44 ). Consistently, we find that FXR1-overexpressing cells and AraC-treated cells show increased phosphorylation and thus activation of PKR ( Fig. 3G ). Therefore, we tested the impact on eIF2\\u03b1 phosphorylation upon overexpression of two snoRNAs that are enhanced in G0 cells, snoRD46 and snoRA2A. We verified overexpression of these transfected snoRNAs by qPCR compared to a control vector (fig. S3F). We find that overexpression of these snoRNAs leads to elevated eIF2\\u03b1 phosphorylation, compared to expression of a control vector ( Fig. 3H ). This suggests that overexpression of FXR1 alters snoRNAs, which can activate PKR ( 43 , 44 ) and phosphorylate eIF2\\u03b1. Recent studies also show that P stalk proteins, RPLP2, RPLP1, and RPLP0 or uL10, promote phosphorylation of eIF2\\u03b1 by activation of GCN2 eIF2\\u03b1 kinase (eIF2ak4), via RPLP0 interaction ( 39 , 40 ). Significantly, we find that RPLP0 increases in FXR1-overexpressing cells ( Fig. 3B ) and decreases upon FXR1 knockdown (fig. S3A and table S3C). Similarly, we find RPLP2 elevated in FXR1-overexpressing cells ( Fig. 3C ). Consistent with studies that demonstrated that P stalk proteins can interact with and activate GCN2 and cause eIF2\\u03b1 phosphorylation ( 39 , 40 ), we tested FXR1-overexpressing cells for GCN2 phosphorylation that marks its activation and for downstream eIF2\\u03b1 phosphorylation. We find that FXR1-overexpressing cells\\u2014where RPLP0 is enhanced, as in AraC-treated cells ( Fig. 3B )\\u2014show increased GCN2 phosphorylation ( Fig. 3I ) and concordantly enhanced eIF2\\u03b1 phosphorylation ( Fig. 3J ). Concurrently, we find that GCN2 associates more significantly with ribosomes and RPLP0 in FXR1-overexpressing cells ( Fig. 3K ), which may enable GCN2 activation and eIF2\\u03b1 phosphorylation ( 39 , 40 ). Conversely, eIF2\\u03b1 phosphorylation decreases upon FXR1 depletion in THP1 and other cell types ( Fig. 3L and fig. S3G). This increase in FXR1 and eIF2\\u03b1 phosphorylation is regulated over time of AraC treatment but is variable at higher concentrations of AraC (fig. S3, H and I), likely due to multiple effects including cell death and feedback regulation of phosphatases and mTOR that affect eIF2\\u03b1 phosphorylation and FXR1 levels ( 1 , 10 , 45 ). GCN2 phosphorylation and activation, and eIF2\\u03b1 phosphorylation in FXR1-overexpressing cells, is reduced upon RPLP0 knockdown ( Fig. 3M ), consistent with the need for RPLP0 in FXR1 overexpression conditions for GCN2 activation and eIF2\\u03b1 phosphorylation. These data suggest that ribosome changes in FXR1-overexpressing cells, as in G0 and AraC-treated cells, elicit eIF2\\u03b1 phosphorylation. FXR1-overexpressing cells, similar to G0 and AraC-treated cells, promote translation of noncanonical start sites eIF2\\u03b1 phosphorylation inhibits canonical translation and reduces the stringency of selecting canonical AUG start sites that are present within a strong Kozak consensus sequence. This permits unconventional translation, including use of mRNAs with structured 5\\u2032UTRs, start sites in poor Kozak consensus, or non-AUG start sites ( 3 , 46 \\u2013 48 ). FXR1 amplification in G0, AraC-treated, and FXR1 OE cells leads to multiple ribosome alterations ( Figs. 2 and 3, A to F ). Increased snoRNA modification can promote noncanonical translation ( 24 ), while altered ribosomes can promote specific mRNA 5\\u2032UTR translation ( 37 ), and eIF2\\u03b1 phosphorylation can inhibit canonical translation to permit translation of specific 5\\u2032UTRs and noncanonical start sites ( 47 ). As these changes, upon FXR1 elevation in G0 and AraC-treated cells, can redirect translation, we tested whether luciferase reporters with a noncanonical GUG start site would be translationally up-regulated, over a canonical AUG start site reporter. We normalized both these reporters to a Renilla reporter as cotransfection control and for RNA levels. In G0 and AraC-surviving cells, we find that the translation of GUG reporter over that of the AUG reporter is enhanced, compared to the translation ratio in untreated, serum-grown proliferating cells ( Fig. 3N ), indicating that translation of unconventional start sites is enabled in G0 and AraC-treated conditions. We tested whether FXR1 overexpression affects translation of noncanonical start sites, as this translation could be due to FXR1 amplification, or other factors in G0 and AraC-treated cells. We find that, as in G0 cells, the translation of GUG reporter over that of the AUG reporter is enhanced in FXR1-overexpressing cells, compared to control vector cells, even without induction of G0 or AraC treatment ( Fig. 3O ). These changes in luciferase expression are not seen at the RNA level (fig. S3J). These data suggest that translation of atypical start sites is enabled in G0 cells, AraC-treated cells, and FXR1 OE cells. FXR1 overexpression promotes translation of survival genes The above data suggested that FXR1 overexpression in G0 and AraC-treated cells may promote translation of specific mRNAs. To identify which mRNAs are translated because of FXR1 amplification seen in G0 and AraC-surviving cells but without G0 or AraC conditions that could induce other effectors, we conducted polysome profiling of control vector and FXR1 OE cells without G0 or AraC treatment ( Fig. 4A ). The heavy polysomes (>2) were pooled and profiled by microarray compared to input samples to identify mRNAs that are promoted or repressed on polysomes by FXR1 overexpression (table S4, A and B). We find that ~10% [312 (AraC-treated proteome; table S3A) to 407 (G0 proteome; table S3A) genes of 3575 genes above 1.3-fold in table S4A shown in table S4C] are genes that are commonly up-regulated in FXR1 overexpression translatome (table S4, A to C) and in the proteome of AraC-treated and G0 cells (table S3A). Many of these proteins are also known to be increased in G0 but decreased upon FXR1 depletion from our datasets (table S3) ( 1 , 10 ). Apart from prosurvival oncoproteins like BCL6 ( 49 ) and antitumor immune regulators such as CD47 ( 50 ) that we previously noted to be up-regulated in G0 and AraC-treated cells ( 1 ) ( Fig. 4B and tables S3A and S4, A to D), mRNAs translated upon FXR1 overexpression also include stress response genes like XBP1 ( 51 ), cell adhesion genes like PECAM1 ( 52 ), signaling regulators that include phosphatidylinositol 3-kinase (PI3K)\\u2013AKT\\u2013mTOR pathway ( 45 ), and metabolic enzymes like PDK3 ( 53 ) that are known to promote tumor survival ( Fig. 4B and table S4, A to D). Fig. 4. FXR1 overexpression promotes translation of prosurvival genes with distinct 5\\u2032UTRs. ( A ) Polysome analysis in FXR1 OE cells compared to control cells. Polysome/monosome (P/M) ratio graphed from three replicates. Global analysis of translatome in FXR1 OE compared to control vector cells. ( B ) Graph showing gene ontology (GO) terms with highest and lowest normalized enrichment score (NES), based on GSEA of FXR1 OE translatome compared to control vector translatome (table S4, A to E). ( C ) qPCR analysis of polysome enrichment of selected mRNAs from FXR1 OE translatome compared to control vector translatome in table S4 (A to C), normalized to inputs. ( D ) Translation efficiency of luciferase reporter bearing BCL6 GC-rich 5\\u2032UTR normalized to cotransfection Renilla control reporter and for reporter RNA levels over control 5\\u2032UTR CX reporter ( 10 ) normalized to cotransfection Renilla control reporter and for reporter RNA levels, in G0 compared to serum-grown S+ cells. ( E ) Polysome analysis of FXR1 KD compared to control shRNA cells. P/M ratio graphed from three replicates. ( F ) Harringtonine initiation inhibition assay to assess translation ( 61 ) by existing elongating ribosomes over time by puromycin labeling, followed by Western analysis (below) for puromycin-labeled proteins and loading control vinculin. Top, graphs of 37- to 10-kDa proteins (orange line) over time in FXR1 KD versus control shRNA cells, of final time points in FXR1 KD versus control shRNA cells, and in FXR1 OE compared to control vector cells. Data are average of three replicates \\u00b1 SEM. See also fig. S4 and table S4. In addition, several classes of critical genes such as immune cell receptors that would be recognized by immune cells, and immune cell\\u2013attracting chemokines, are decreased in the translatome of FXR1-overexpressing cells compared to control vector cells ( Fig. 4B , down in translatome genes, and table S4E). The suppression of immune recognition receptors and chemokines, while inducing immune evasion receptors like CD47, suggests specific mRNA translation to survive antitumor immunity. These data suggest that specific gene categories are modulated by FXR1 increase to enable AML survival. Specific mRNA translation in G0 cells with distinct 5\\u2032UTRs To verify the translationally up-regulated mRNAs in FXR1 OE cells in table S4 (A to D), we performed qPCR analysis of polysome fractions that are normalized for input levels, which reveal that these mRNAs are enriched on polysomes of FXR1 OE cells compared to control vector cells ( Fig. 4C ), indicating their enhanced translation when FXR1 is amplified. mRNAs that are up-regulated in FXR1 overexpression translatome that overlap with increase in AraC-treated proteome are enriched for GC-rich 5\\u2032UTRs (fig. S4, A and B) ( 1 , 54 , 55 ). Several genes known to be translated by non-AUG start sites ( 48 ) are included among the up-regulated translatome in FXR1 overexpression cells and have GC-rich 5\\u2032UTRs (fig. S4, C and D). These GC-rich 5\\u2032UTRs and noncanonical start sites are not preferred for conventional tRNA recruitment by eIF2 in the canonical translation mechanism. Therefore, translation of these mRNAs could be enabled by unconventional translation conditions in G0, AraC-treated, and FXR1-overexpressing cells where eIF2\\u03b1 is phosphorylated to reduce its activity. In accord, we find that these unconventional target genes, including those regulated by upstream open reading frames [uORFs; activating transcription factor 4 (ATF4)] ( 56 ), and noncanonical start sites (CITED1) ( 48 ) are increased in FXR1 OE but decreased in FXR1 KD cells (fig. S4, E and F), consistent with their need for eIF2\\u03b1 phosphorylation to reduce conventional translation. To test whether these 5\\u2032UTRs contribute to specific translation in G0 cells, we constructed a reporter with the 5\\u2032UTR of BCL6, one of the translated genes in FXR1 overexpression cells (table S4A) that is also enhanced in G0 and AraC-treated cells (table S3A). We find that the translation efficiency of BCL6 5\\u2032UTR luciferase over a control 5\\u2032UTR [CX ( 10 )] reporter (normalized for Renilla co-transfection control and RNA levels) is elevated in G0 compared to proliferating cells and in FXR1 OE compared to control vector cells ( Fig. 4D and fig. S4G). These data suggest that specific mRNA translation involve, in part, distinct 5\\u2032UTRs that are enabled upon eIF2\\u03b1 phosphorylation, elicited by FXR1-mediated ribosome changes. Impact on translation mechanism To understand the mechanism of translation, we analyzed polysomes in FXR1-depleted cells compared to control shRNA cells. As in the case of FXR1 OE cells compared to control vector cells ( Fig. 4A ), while specific genes are altered on polysomes upon FXR1 depletion (table S3, F to H), no significant difference was observed in polysome profiles and in the polysome to monosome (P/M) ratio in profiles conducted in FXR1-depleted cells compared to control shRNA cells ( Fig. 4E ). The P stalk proteins are at the guanosine triphosphatase (GTPase) activation center (GAC) near the A site, where eEF1A mediates translation elongation; increased binding of GCN2 in this region via RPLP0 enables GCN2 activation, which is prevented if eEF1A binds GCN2. Thus, GCN2 activation may suggest reduced eEF1A activity and elongation as seen in starvation stress conditions ( 39 , 40 , 57 \\u2013 59 ). Altered elongation has been observed with the FXR1 paralogue, FMRP ( 60 ). Therefore, as our data showed that GCN2 associates with ribosomes in FXR1 OE cells, but not in control vector cells ( Fig. 3K ), and is activated because of RPLP0 that increases in FXR1 OE cells ( Fig. 3, B and M ), we tested whether elongation may be altered in FXR1 OE and KD cells, compared to control cells. We treated cells with harringtonine to block initiation followed by a time course to observe translated products from preexisting elongating ribosomes, with puromycin termination and labeling (SunRiSE method) ( 61 ). We observed a difference in labeling at the earliest time points (fig. S4H), similar to the nascent global translation in Fig. 1F , suggesting increased translation with FXR1 OE compared to control vector cells that is then blocked by harringtonine. We examined labeled products that were smaller in size, as these would reveal differences in elongation rates first ( 61 ) (also all proteins). The harringtonine time course shows moderately enhanced labeling with FXR1 KD compared to control shRNA cells ( Fig. 4F ). FXR1 OE showed a slight, but not significant, trend of decreased labeling, compared to control vector cells. While additional studies are needed, these data hint that FXR1 depletion may moderately increase translation elongation. This is in addition to FXR1\\u2019s role in increasing translation initiation of specific 5\\u2032UTR-bearing mRNAs ( Fig. 3, N and O , and fig. S3J, GUG reporter; Fig. 4D and fig. S4G, BCL6 5\\u2032UTR reporter), rewiring translation for surviving these conditions. FXR1-overexpressing cells show decreased monocyte migration and enhanced survival with macrophages Our data show altered translation of immune regulators in FXR1-overexpressing cells. On the basis of our data, these cells may subvert immune cells by decreasing translation of immune susceptibility genes (receptors, immune cell\\u2013attracting chemokines) while also promoting translation of immune evasion genes (CD47) that inhibit antitumor macrophages ( 50 ) ( Fig. 4B , up and down in translatome genes, and table S4, D and E). Since FXR1-overexpressing cells show decreased translation of immune-attracting chemokines ( Fig. 4B , down in translatome genes, and table S4E), we first tested whether FXR1 overexpression alters migration of monocytes. We find that cocultured monocytes, in a transwell assay, show 50% reduction in monocyte migration toward FXR1 OE cells compared to control vector cells ( Fig. 5A and fig. S5A). This is consistent with our data that show decreased translation of immune-attracting chemokines in FXR1-overexpressing cells ( Fig. 4B , down in translatome genes, and table S4E) and our data that showed altered immune regulators in G0 and AraC-treated cells ( 1 ). These data suggest that monocytes are precluded from the environment around FXR1-overexpressing cells through decreased translation of immune-attracting chemokines, which could allow FXR1-overexpressing cells to evade associated antitumor immune activity. Fig. 5. Chemosurvival in G0 and AraC-treated cells, and upon FXR1 overexpression, can be reduced by overriding eIF2\\u03b1 phosphorylation, and thereby decreasing translation of survival genes, or by targeting the translated survival genes. ( A ) Graph of fold change in monocyte migration (image in fig. S5A, stained with Far Red CellTrace dye) in a transwell assay, with FXR1 OE cells or control vector cells in the bottom chamber. ( B ) Flow cytometry data showing survival of control vector cells compared to FXR1 OE cells (stained with Far Red CellTrace dye and Hoechst 33342 to detect live cells in quadrant 2 marked by an arrow) after coculturing with macrophages. ( C ) Polysome analysis of FXR1 OE cells, with or without ISRIB treatment for 24 hours, ( D ) P/M ratio, and ( E ) qPCR analysis for enrichment of FXR1 OE target and control mRNAs (from table S4) on polysomes of ISRIB-treated or untreated FXR1 OE cells, normalized to Input. ( F ) Chemosurvival of FXR1 OE compared to control vector cells, treated with AraC and ISRIB (cotreatment, restores canonical translation, suppressing specific mRNA translation), ( G ) with toyocamycin, XBP1 inhibitor (cotreatment with AraC and post-AraC treatment), and ( H ) with FX-1, BCL6 inhibitor (post-AraC treatment) in FXR1 OE compared to control vector cells, and to FXR1 OE and control vector cells treated with AraC and buffer. ( I ) Model for FXR1-mediated ribosome alterations that induce translation changes to promote AML survival. Data are average of three replicates \\u00b1 SEM. See also fig. S5 and table S4. Second, we find that immune evasion genes, elevated in the FXR1 overexpression translatome (table S4, A to D), are also increased in G0 and AraC-treated cells (table S3A) ( 1 ) and include SLAMF6 ( 62 ) and CD47 ( 50 ) that block antitumor immune activity ( 50 ). Therefore, we tested whether FXR1 OE cells would show increased survival when cocultured with macrophages that can be evaded by CD47, compared to control vector cells. Flow cytometry analysis revealed greater survival of FXR1 OE cells (by 22%) compared to control vector cells, after coculturing with macrophages ( Fig. 5B and fig. S5B). These data suggest that FXR1-overexpressing cells translate immune evasion regulators to survive antitumor immunity. Chemosurvival of FXR1-overexpressing cells is reduced by drugs that override eIF2\\u03b1 phosphorylation or block translated survival genes, BCL6 and XBP1 The enhanced eIF2\\u03b1 phosphorylation in G0, AraC-treated, and FXR1 OE cells ( Fig. 3J and fig. S3H) is a potential vulnerability as these cells use reduced canonical translation to permit specific, prosurvival mRNA translation. To test whether the elevated eIF2\\u03b1 phosphorylation and specific mRNA translation is needed for chemosurvival, we used the small molecule, ISRIB. ISRIB overrides eIF2\\u03b1 phosphorylation and restores canonical translation, which would suppress unconventional, specific mRNA translation ( 4 , 63 ). We first tested whether the translation observed with FXR1 overexpression can be reversed by ISRIB. Western blot analysis revealed decrease of known noncanonical targets such as ATF4 that are not translated in the presence of ISRIB (fig. S5C). As shown in Fig. 5 (C to E) , we performed polysome analysis in FXR1 overexpression cells with or without ISRIB treatment followed by qPCR of mRNAs that are translationally increased in FXR1-overexpressing cells (from table S4A). We find that ISRIB reduced polysome association of several mRNAs that are up-regulated in FXR1-overexpressing cells, without affecting other genes like vinculin ( Fig. 5E ). These data suggest that FXR1 overexpression promotes specific, prosurvival mRNA translation under eIF2\\u03b1 phosphorylation conditions. If so, then overriding this unconventional translation should reduce chemosurvival of FXR1-overexpressing cells. Consistently, we find that FXR1 OE cells that are treated with ISRIB and AraC have reduced chemosurvival compared to FXR1 OE cells that are treated with buffer and AraC ( Fig. 5F ). To confirm that the translated genes are important for survival in FXR1-overexpressing cells, chemosurvival was tested after inhibiting two known survival genes (XBP1 and BCL6) that are up-regulated in FXR1-overexpressing cells, compared to untreated, control vector cells (table S4, A to D). We find that treatment of FXR1-overexpressing cells with AraC and toyocamycin, an inhibitor of XBP1 prosurvival gene ( 64 , 65 ) ( Fig. 5G , together, or adding toyocamycin post-AraC treatment), or AraC and FX-1, an inhibitor of BCL6 prosurvival oncoprotein ( 49 ) ( Fig. 5H , adding FX-1 post-AraC treatment), reduced chemosurvival compared to treatment with buffer and AraC and to control vector cells. These data suggest that specific mRNA translation mediated by FXR1 contributes to chemosurvival ( Fig. 5I ) and is a vulnerability that can be targeted in FXR1 elevated conditions.\",\n",
            "        \"discussion\": \"DISCUSSION Cancer cells can enter a reversible arrest phase called quiescence or G0 that is resistant to harsh conditions including chemotherapy ( 1 , 2 ). G0 cells induced by serum starvation are chemoresistant-like chemotherapy-surviving leukemic cells and exhibit similar gene expression at the posttranscriptional level ( 1 ). These data suggest that there is translation of a specific set of survival genes, along with suppression of genes that are detrimental upon therapy treatment. How cells translate specific genes in these clinically relevant conditions is not known and could involve translation mechanism changes. We find that ribosome-associated regulators are similarly altered in both G0 and chemosurviving cells ( 1 ) ( Fig. 1, A and B ), indicating that they may mediate this gene expression via mechanistic changes. FXR1 is an RNA-binding protein that was found associated with ribosomes ( 7 ) and is a posttranscriptional regulator ( 10 ) that is amplified at its chromosome locus in several aggressive cancers, where it induces dysregulation of critical gene expression ( 9 ). Our data revealed that FXR1a isoform is elevated in G0 AML cells ( 10 ). Since G0 cells are chemoresistant ( 1 ), this suggests that FXR1 that is increased in G0 could play a role in chemosurvival. In accord, we find that similar to G0 THP1 cells, AraC chemotherapy-treated THP1 and other AML cells (NOMO1) transiently up-regulate FXR1 ( Fig. 1C and fig. S1, A and B). These data suggest that FXR1 could be increased in some cancers treated with therapies like AraC to promote chemosurvival. Consistently, we find that overexpression of FXR1a that is elevated in G0 ( 10 ) promotes AraC survival by almost twofold, while knockdown of FXR1 reduces chemosurvival to less than 50% ( Fig. 1, D and E ). These data suggest that FXR1 mediates chemosurvival in G0 and chemo-treated AML, where it is elevated. FXR1 was identified associated with the ribosome ( 7 ), and we previously found that FXR1 associates with and promotes translation of specific mRNAs in G0 ( 10 ). Therefore, the role of FXR1 in chemosurvival could involve translation regulation. To test this, we analyzed translation in cells with or without FXR1 depletion by nascent translation labeling. We find that in FXR1-depleted cells, translation is decreased by almost half, compared to control shRNA cells ( Fig. 1F and fig. S1C), while increased by 1.7-fold in FXR1-overexpressing cells. These data suggest a role for FXR1 in G0 and chemo-treated cells via translation regulation that may promote chemosurvival observed here. FXR1 is known to be nucleolar, nuclear, and cytoplasmic ( 8 ) and can participate in ribosome biogenesis at multiple levels: through known associations with ribosomes and RPs ( 7 ), with RNA regulators, and by posttranscriptional control over ribosome regulators as detected previously ( 10 , 35 ). We find that FXR1 depletion reduces all rRNAs and decreases several snoRNAs and RPs that increase in G0; conversely, FXR1a overexpression leads to their increase ( Fig. 2, A to C ; fig. S2A; and table S1). FXR1 regulates levels of snoRNAs (table S1, A to C) that modify rRNA sites that affect translation ( 25 \\u2013 29 ). In accord with the increase of snoRD58A and snoRA22 upon FXR1 elevation in AraC-treated cells and FXR1-overexpressing cells, we find increased modification at their rRNA target sites on 28 S rRNA ( Fig. 2, D and E ). In addition, with multiple snoRNA changes, alterations in other rRNA modifications are observed by mass spectrometry upon FXR1 depletion ( Fig. 2F , fig. S2B, and table S1D). FXR1 depletion and overexpression also alters U3 and U8 snoRNAs ( Fig. 2B and fig. S2A) that cleave and process 47 S Pol I precursor rRNA ( 16 , 19 \\u2013 22 ), which would lead to the observed alteration in mature 28 S , 18 S , and 5.8 S rRNA levels ( Fig. 2A and fig. S2A). These data suggest that FXR1 affects multiple ribosomal components (table S1, Fig. 2 , and fig. S2), which could cause the translation change observed in AraC-treated and G0 cells ( Fig. 1F and fig. S1C) where FXR1 is amplified ( Fig. 1, A to C , and fig. S1, A and B). To understand how FXR1 enables these ribosome changes, we analyzed the FXR1 protein interactome (table S2). We find that FXR1 associates with snoRNA and ribosome biogenesis regulators, such as NOLC1 ( 30 , 31 ) ( Fig. 2G ), other ribosome-associated factors (table S2 and fig. S2C), and rRNAs (fig. S2D). Consistent with the fact that both Pol I\\u2013transcribed rRNAs and Pol II\\u2013transcribed snoRNAs and RPs are affected by FXR1, we find that FXR1 interacts with DDX21 ( Fig. 2H ), a regulator that promotes Pol I and Pol II ribosome gene transcription ( 32 , 33 ). In accord, we find that DDX21 overexpression, but not of a control vector, partially rescues the reduced 45 S rRNA precursor levels in FXR1 knockdown cells ( Fig. 2I ). This is consistent with regulation of ribosomes, as DDX21 also binds snoRNAs including U3 to promote their levels and functions and enhances Pol I\\u2013 and Pol II\\u2013derived ribosome gene transcription ( 32 \\u2013 34 ). Given that all four rRNAs are affected by FXR1, including 18 S , 28 S , and 5.8 S rRNAs from Pol I transcription, and Pol III transcript, 5 S rRNA, our data suggested that FXR1 may influence a common rRNA transcription factor for Pol I and Pol III, and/or for all three RNA polymerases, as FXR1 also affects Pol II\\u2013derived RPs and snoRNAs. Consistent with recent data showing FXR1 regulation of c-MYC ( 35 ) that controls ribosome gene transcription from all three polymerases ( 36 ), we find that c-MYC increases with FXR1 overexpression ( Fig. 2J ). In addition, we find that FXR1 associates with the mRNA of POLR1D, a Pol I and Pol III transcription factor; consistently, FXR1 depletion decreases, while overexpression increases, POLR1D ( Fig. 2, K and M , and fig. S2E). As POLR1D is needed for rRNA transcription from Pol I and Pol III, and for DDX21 localization and thus DDX21-controlled snoRNAs and Pol I and Pol II ribosome biogenesis ( 32 , 34 ), FXR1 regulation of POLR1D would affect all four rRNAs ( Fig. 2A ). The levels of mature rRNAs and the precursor 45 S are differentially affected. This may be due to additional effects, apart from transcription and processing regulation by MYC, DDX21, and PolR1D, where rRNA levels and folding, and thereby stability, may be affected by altered modifications by snoRNAs and RPs, whose levels are regulated by FXR1. Thus, our data reveal that FXR1 modulates multiple ribosome regulators, which can lead to ribosome changes upon FXR1 elevation in G0 chemoresistant cells. These data suggest multiple changes in the ribosome, upon FXR1 overexpression that is seen in G0 and AraC-treated cells ( Figs. 2 and 3, A to D ; fig. S3A; and tables S1 and S3). Consistently, we find that ribosomes migrate differently in two distinct complexes in G0 and AraC-chemoresistant cells compared to one in proliferating cells; this is also observed with FXR1 overexpression compared to control vector ( Fig. 3, E and F , and fig. S3, B to E), indicating that FXR1 induces similar differences in ribosome complexes. As these complexes in cytoplasmic extracts are associated with Y10B that binds 5.8 S in 80 S ( 38 ), they may include partially processed 80 S complexes in the cytoplasm where 18 S is still being processed, causing differences in 18 S and 28 S rRNA levels observed. These ribosome changes can alter translation, which could enable the cells to adapt to chemotherapy and survive. One way that ribosome changes could alter translation would be by activating stress signaling. Ribosomes not only function as translation machineries but also can directly activate stress signaling pathways ( 39 \\u2013 42 ). This includes GCN2 that is activated by stalled ribosomes ( 57 ) or by P stalk RPs ( 39 , 40 ), and PKR that can be activated by snoRNAs ( 43 , 44 ). These kinases disable canonical translation via eIF2\\u03b1 phosphorylation, which permits translation on specific mRNAs such as those with noncanonical start sites or complex 5\\u2032UTRs ( 3 , 46 \\u2013 48 ). In accord, we find that FXR1 overexpression leads to activation of PKR and GCN2 ( Fig. 3, G and I ). Overexpression of FXR1 can increase specific snoRNAs that can bind and activate PKR, a dsRNA binding protein ( 43 , 44 ), which can cause eIF2\\u03b1 phosphorylation. Consistently, we find that overexpression of G0 up-regulated snoRNAs, snoRD46, or snoRA2A leads to increased eIF2\\u03b1 phosphorylation ( Fig. 3H and fig. S3F). In accord, previous data had shown that overexpression of fibrillarin that leads to altered snoRNA-mediated rRNA modification induces noncanonical translation ( 24 ). We find that RPs including P stalk RPs are altered in FXR1 OE and KD cells and in G0 and AraC-treated cells ( Fig. 3, B and C ; fig. S3A; and table S3, B, C, E, and G). Changes in the P stalk RPs can lead to GCN2 activation and eIF2\\u03b1 phosphorylation via GCN2 interaction with RPLP0 ( 39 , 40 ). Consistently, we find that FXR1 overexpression enhances RPLP0 that is also seen in AraC-treated cells, and leads to GCN2 phosphorylation and eIF2\\u03b1 phosphorylation ( Fig. 3, I and J ). RPLP0 associates with GCN2; consistently, in FXR1 overexpression cells where RPLP0 is elevated along with eIF2\\u03b1 phosphorylation, we find increased association of GCN2 with ribosomes and RPLP0 ( Fig. 3K ). In accord, in FXR1-depleted cells, we find that eIF2\\u03b1 phosphorylation is decreased ( Fig. 3L and fig. S3G). Accordingly, RPLP0 depletion in FXR1-overexpressing cells attenuated phosphorylation of GCN2 and eIF2\\u03b1 ( Fig. 3M ), indicating that RPLP0 is needed for eIF2\\u03b1 phosphorylation, elicited in FXR1 overexpression conditions. This increase in FXR1 and eIF2\\u03b1 phosphorylation is regulated over time of AraC treatment and is variable at high AraC concentrations (fig. S3, H and I), likely due to multiple effects including cell death and feedback regulation of phosphatases and mTOR that are shown to affect eIF2\\u03b1 phosphorylation and FXR1 levels ( 1 , 10 , 45 ). Together, these data suggest that increased FXR1 in G0 chemosurviving cells alters ribosomes to induce stress signals that inhibit canonical translation and permit specific mRNA translation. Such stress signaling by the ribosome has been observed with colliding ribosomes that trigger GCN2-mediated eIF2\\u03b1 phosphorylation that leads to survival or, at increased levels, induce stress c-Jun N-terminal kinase (JNK)/p38 mitogen-activated protein kinase (MAPK) signaling pathway to trigger apoptosis. This has also been observed with c-GAS signaling in other conditions, as well as with ribotoxic agents, leading to gene expression changes ( 41 , 42 , 66 , 67 ). With transient, acute ISR, there is global decrease in translation followed by restoration, but with chronic, moderate ISR, such as seen in some stress conditions, there is distinct gene translation ( 68 , 69 ). This can result in reprogramming of translation without drastic reduction in overall translation as seen in FXR1 OE cells where translation increases ( Fig. 1F ), despite eIF2\\u03b1 phosphorylation ( Fig. 3, I to M , and fig. S3, H and I). How this is elicited, whether this is due to MYC deregulation ( 35 ) and of other ribosome regulators ( Fig. 2, G to M ) and stress mechanisms by FXR1 in these conditions, needs to be further investigated. With eIF2\\u03b1 phosphorylation, stringent use of conventional Kozak start sites for initiator tRNA recruitment ( 3 ) is reduced\\u2014and can permit translation of specific mRNAs such as those with complex 5\\u2032UTRs, unconventional non-AUG start sites, or AUGs in poor Kozak start sites that would normally be poorly translated ( 47 ). In accord, we find that translation of a reporter with a noncanonical GUG start site over that of a reporter with a conventional AUG start site is promoted in FXR1-overexpressing cells and in G0 and chemo-treated cells, compared to untreated serum-grown proliferating control vector cells ( Fig. 3, N and O , and fig. S3J). Consistently, global profiling revealed that many of the genes up-regulated in G0, AraC-treated, and FXR1-overexpressing cells ( Fig. 4, A to C , and table S4) have GC-rich 5\\u2032UTRs (fig. S4, A to D), and non-AUG start sites ( 48 ), and include noncanonical translation targets (table S4 and fig. S4, C to F); these genes may need eIF2\\u03b1 phosphorylation to be translationally up-regulated. In accord, we find that a reporter bearing the GC-rich 5\\u2032UTR of one of these up-regulated genes, BCL6, shows enhanced translation over a control 5\\u2032UTR reporter, in G0 and FXR1 overexpression cells, compared to serum-grown, control vector cells ( Fig. 4D and fig. S4G), independent of RNA levels. Other features that are unique to these transcripts such as previously identified 3\\u2032UTR elements ( 1 , 10 ), and other ribosomal and translational changes, may also contribute to the specific translation observed. Our findings of multiple effects of FXR1 on the ribosome ( Figs. 1 and 3 ) reveal the complexity of translation changes in these chemoresistant AML cells where FXR1 is amplified. Analysis of P/M ratios did not reveal differences between FXR1 OE and FXR1 KD compared to control cells ( Fig. 4, A and E ). The P stalk RPLP0/uL10 is at the GAC near the A site, where eEF1A mediates translation elongation; increased binding of GCN2 in this region via RPLP0 enables GCN2 activation, which is prevented if eEF1A binds GCN2. Thus, GCN2 activation may suggest reduced eEF1A activity and elongation as seen in starvation stress conditions ( 39 , 40 , 57 \\u2013 59 ). To test this, we did a harringtonine initiation block experiment to block initiation and see preinitiated, elongating ribosome runoff with puromycin labeling ( 61 ). We observe more labeling over time in FXR1 KD compared to control cells ( Fig. 4F and fig. S4H). These data hint that runoff production from elongating ribosomes after initiation shutdown with harringtonine may be moderately enhanced in FXR1 KD compared to control cells. This was first observed with the paralogue of FXR1, FMRP, that not only suppresses translation elongation of specific targets but also promotes initiation of other distinct mRNAs ( 60 , 70 ). While additional studies are needed, these data suggest that FXR1 rewires translation\\u2014by increasing initiation of specific 5\\u2032UTR-bearing or noncanonical start site\\u2013bearing mRNAs ( Fig. 3, N and O , and fig. S3J, GUG reporter; fig. S4G and Fig. 4D , BCL6 5\\u2032UTR reporter) that are likely needed for survival\\u2014and may also possibly affect elongation slightly ( Fig. 4F ), potentially of mRNAs that were being translated in untreated cells and need not be translated upon therapy or G0 conditions. Our data reveal that prosurvival genes\\u2014stress response genes like XBP1 ( 64 , 65 ), cell adhesion genes like PECAM1 ( 52 ), immune genes like CD47 ( 50 ), and oncoproteins like BCL6 ( 49 ) (table S4, A to D, and Fig. 4B )\\u2014are translated via these ribosome changes that are triggered by FXR1 in G0 chemosurviving cells. Genes that are down-regulated in the FXR1 overexpression translatome compared to control vector cells include immune susceptibility genes (receptors, immune-attracting chemokines, Fig. 4B , down in translatome genes, table S4E). Consistently, we find that monocyte migration is reduced by 50% in transwell assays with FXR1-overexpressing cells compared to control vector cells ( Fig. 5A and fig. S5A), which could allow FXR1-overexpressing cells to evade antitumor immune activity. Concurrently, the increased translatome in FXR1-overexpressing cells include antitumor immune evasion genes such as CD47 ( Fig. 4, B and C , and table S4, A to D) that inactivate antitumor macrophages ( 50 ). In accord, we find that FXR1-overexpressing cells show increased survival compared to control vector cells when cocultured with macrophages ( Fig. 5B and fig. S5B). Together, these data suggest that the specific translatome elicited in FXR1-amplified conditions, as in G0 and AraC-treated cells, subverts antitumor immunity by decreasing immune susceptibility genes and by translating antitumor, immune evasion genes to survive antitumor immune activity. Inhibition of canonical translation by eIF2\\u03b1 phosphorylation can lead to expression of specific survival mRNAs that are poorly translated by the canonical mechanism ( 1 , 47 , 55 ) due to unconventional start sites or GC-rich 5\\u2032UTRs (fig. S4, A to D). Consistently, treatment with ISRIB that overrides eIF2\\u03b1 phosphorylation to restore canonical translation ( 4 , 63 ) prevents this specific mRNA translation ( Fig. 5, C to E , and fig. S5C). Thus, these data suggest that FXR1 increase in G0 and AraC-treated cells causes ribosomal changes that can reduce canonical translation and enable translation of specific genes by triggering eIF2\\u03b1 phosphorylation. Together, our data suggest that the enhanced chemosurvival in FXR1-overexpressing cells\\u2014as in G0 and AraC-treated cells, where FXR1 is elevated and causes ribosome changes and eIF2\\u03b1 phosphorylation\\u2014could be reversed by targeting the eIF2\\u03b1 phosphorylation\\u2013induced, specialized translation mechanism, or such translated genes. Consistently, we find that inhibition of this specific mRNA translation with ISRIB ( Fig. 5F ), or inhibition of the downstream target genes such as that of XBP1 ( 64 , 65 ) or of BCL6 ( 49 ) with pharmacological inhibitors ( Fig. 5, G and H ), reduces the chemosurvival of FXR1 overexpression cells, compared to control vector cells or to FXR1 overexpression cells treated with chemotherapy alone. Together, our data reveal that FXR1 is a critical translation regulator that is amplified in G0 and chemosurviving AML, which induces ribosome changes that trigger stress signals to enable specialized translation of survival genes to promote tumor persistence ( Fig. 5I ). Limitations While we verified our findings in two different AML backgrounds, further studies in vivo are needed. Given the complexity of ribosomal changes observed ( Figs. 1 to 4 and figs. S1 to S4), apart from the eIF2\\u03b1 phosphorylation mechanism, the impact on G0 and chemosurviving AML will include effects of other ribosome changes mediated by FXR1, other gene expression regulation by FXR1, and other dysregulated RNA mechanisms in G0 cells ( 1 ) to elicit survival of G0 and chemoresistant AML.\",\n",
            "        \"conclusion\": null\n",
            "    },\n",
            "    {\n",
            "        \"pmcid\": \"PMC9697589\",\n",
            "        \"abstract\": null,\n",
            "        \"intro\": null,\n",
            "        \"method\": \"METHODS Experimental Design Between 8 June and 8 July 2021, we placed aerosol capturing equipment on a cart ( Figure 1 ). The cart contained a viable virus aerosol sampler (VIVAS) (Aerosol Dynamics, Berkeley, California) and 2 SKC BioSamplers (SKC Inc, Eighty Four, Pennsylvania). VIVAS collects airborne particle by using water vapor condensation method, and previous studies have shown highly efficient (74 \\u00b1 12%) collection of particles in the size range from 10 nm to 10 \\u00b5m [ 13 ]. The SKC BioSampler efficiently traps airborne particles into swirling liquid for subsequent analysis and is considered the industry-standard aerosol capture device [ 14 ]. One SKC BioSampler was placed 5.5 feet above the ground, and the second was placed approximately 3 feet from the ground. This was to determine if there was a qualitative and quantitative difference in bacterial specimen collection between low and high heights. The VIVAS and SKC BioSampler aerosol suction rate was set at 8 L per minute to approximate human respiration. The cart also included a GRIMM aerosol spectrometer (Model 11D, GRIMM Aerosol, Germany) to quantify the aerosol concentration and particle size during the procedure, similar to our previous studies [ 13 ]. Figure 1. Aerosol sampling cart setup. Pumps were set at 8 L/minute. Abbreviation: VIVAS, viable virus aerosol sampler. Operatory Layout and Equipment Placement We placed the equipment cart in a variety of operatory layouts (eg, single dental chair with door closed, semi-open bay, and large multioperator space). Each layout is depicted in Figure 2 A \\u2013 D . We placed the equipment cart as close to the patient without disrupting clinical care in order to capture aerosols within close proximity to the DHCP. Specifically, we connected sterile tubing to the VIVAS aerosol intake port; the other side of the tubing was then taped to the dental overhead lamp to approximate the distance between the DHCP and the aerosol emission source. Figure 2. Experimental layouts. A , Operatory. B , Semi-open bay type 1. C , Large multioperator space. D , Semi-open bay type 2. Abbreviation: HEPA, high-efficiency particular air filter. Equipment Preparation and Specimen Collection Sterile SKC BioSampler impingers were loaded with 20 mL of sterile phosphate-buffered saline (PBS). For the VIVAS, 2 mL of sterile PBS was placed in the sterile collection plate. As the SKC BioSamplers capture aerosols via an air vortex, these were turned on only during aerosol-generating portions of the dental procedure (eg, high-speed drill or ultrasonic scaler use). In settings where the PBS was at risk of evaporating (\\u223c30 minutes), old impingers were exchanged for a new impinger full of PBS, and fluid from the 2 impingers were consolidated into 1 specimen for analysis. The VIVAS, which uses a condensation approach to capture aerosols, continuously collected specimens from beginning to end of dental procedures. This allowed for a more accurate assessment of individual procedure\\u2013level risk for DHCPs. The samples were then transported in sealed and labeled containers within 4 hours for microbiological analysis. The detailed protocol is available in the Supplementary File. This process was repeated for each dental procedure. One baseline sample was collected from SKC BioSamplers and the VIVAS in each dental operatory layout while no procedure was being performed for comparison to procedure samples. Total sampling collection time was measured for both SKC BioSamplers and VIVAS. Aerosol mitigation interventions employed by dentists were documented for each procedure. Microbiological Analysis We plated each specimen onto (1) tryptic soy agar with 5% sheep's blood (BAP), (2) chocolate agar (CHOC), and (3) brucella blood agar (BBA). BAP and CHOC agars were incubated for up to 48 hours in a 35\\u00b0C carbon dioxide incubator. BBA was incubated in anaerobic conditions for 5\\u20137 days. After 24 hours of incubation for the BAB and CHOC agars and 48 hours for the BBA, the plates were examined for growth and the number of bacterial colonies was counted. Subculturing was performed if >1 bacterial colony type was observed. Colonies were identified at the genus and species level using matrix-assisted laser desorption/ionization time-of-flight mass spectrometry (MALDI-TOF) with the VITEK MS MALDI-TOF MS (bioM\\u00e9rieux). We performed viral analyses by sampling 300 \\u00b5L each of the SKC BioSampler and VIVAS specimens, and testing with the BioFire Respiratory Pathogen Panel 2.1 (RP2.1; bioM\\u00e9rieux, Salt Lake City, Utah). The RP2.1 can detect up to 22 respiratory pathogens. Chemical Analyses During amalgam removal procedures, we collected aerosol particles on a stub with carbon tape by holding it 15 cm away from the patient's mouth. The weight of the stub was noted before and after collection. Scanning electron microscopy was performed on the sample with voltage 20 kV and working distance of 9.9 mm followed by energy dispersive X-ray (EDX) analysis to find the elemental composition of the area in focus. This study was reviewed and approved by the Washington University Human Rights Protection Office.\",\n",
            "        \"subjects\": null,\n",
            "        \"results\": \"RESULTS We evaluated the bacterial and viral composition of dental aerosols for 12 procedures in 4 different dental office layouts. A description of each procedure and the microbiological results are presented in Table 1 . Of note, individual dental procedures ran from 30 to 74 minutes and involved both anterior and posterior teeth. DHCPs utilized between 1 and 3 aerosol mitigation strategies, with high-volume evacuation the most common (large-bore dental suction devices), followed by saliva ejectors (standard-bore dental suction devices), high-efficiency particulate air (HEPA) filters, and rubber dams (rubber material that covers part of the mouth). Table 1. Bacterial and Viral Sampling Results for Common Dental Procedures in Real-World Settings Procedure Type Sampling Time, min Aerosol Mitigation Approach Dental Space Bacterial Culture Respiratory Virus VIVAS SKC HVE Saliva Ejector HEPA Filter Rubber Dam VIVAS SKC VIVAS and SKC Implant 45 17 1 1 0 0 Operating room NG NG \\u2026 Ultrasonic cleaning 30 25 1 1 1 0 Semi-open bay type 1 NG NG \\u2026 Gingival flap with cavitron 74 30 1 1 1 0 Semi-open bay type 1 NG a \\u2026 Root canal amalgam removal 43 30 1 0 0 1 Semi-open bay type 1 NG NG \\u2026 Root canal drilling through composite 40 30 1 0 1 1 Semi-open bay type 1 NG NG \\u2026 Root canal amalgam removed; temporary crown placement \\u2026 22 1 0 0 1 Operating room NG b \\u2026 Braces debonding 31 30 0 1 0 0 Large multioperator space c NG \\u2026 Braces debonding 59 30 1 1 0 0 Large multioperator space NG NG \\u2026 Braces debonding 30 21 0 1 0 0 Large multioperator space d NG \\u2026 Amalgam removal and replacement 30 14 1 0 0 0 Semi-open bay type 2 e NG \\u2026 Post and core CEREC crown 30 26 1 0 1 0 Semi-open bay type 2 NG f \\u2026 Amalgam removal and composite filling 30 16 1 0 1 0 Semi-open bay type 2 g h \\u2026 SKC BioSampler run times were the same for high and low settings. Abbreviations: CEREC, chairside economical restoration of esthetic ceramics; HEPA, high-efficiency particular air; HVE, high-volume evacuator; NG, no growth; VIVAS, viable virus aerosol sampler. a One Micrococcus luteus , 2 Staphylococcus hominis. b One Cutibacterium acnes. c Seventeen Cutibacterium acnes. d One Streptococcus gordonii , 1 Actinomyces oris/viscosus. e One Prevotella oris. f One Stenotrophomonas maltophilia. g One Sphingomonas parapaucimobilis. h One mold. Microbiological Analyses Bacterial analysis showed that 5 of 13 procedures had no growth ( Table 1 ). No differences were observed in specimen yield between high- and low-placed SKC BioSamplers ( Supplementary Table 1 ). These results are supportive that we captured aerosols, as a lower placed device should more readily capture large droplets, which settle toward the floor quickly. No substantial differences in microbiological yield were observed between VIVAS and SKC BioSamplers. We identified gram-positive and gram-negative bacteria, and an environmental mold. Organisms were normal human oral flora (eg, Actinomyces oris , Prevotella oris , Streptococcus gordonii ), common environmental flora (eg, Micrococcus luteus , Stenotrophomonas maltophilia , and Sphingomonas parapaucimobilis ), or human skin flora (eg, Cutibacterium acnes and Staphylococcus hominis ). None of the respiratory viral panel tests (from the VIVAS and both SKC BioSamplers) detected any pathogens. All baseline testing (negative controls) was negative for bacteria and viruses. Aerosol concentration and distribution was similar to our previous findings [ 15 ]. Chemical Analyses SEM analysis of a sample collected on a stub placed in close proximity to the dentist during amalgam removal is depicted in Figure 3 . EDX analyses demonstrated that the sample primarily comprised of amalgam alloy metals ( Figure 4 ). Additional SEM images from the stub demonstrate that the amalgam particles are very fine. However, these fine particles largely accumulated in oval- and circular-shaped droplets mixed with water ( Figure 5 ). Figure 3. Scanning electron microscopy image of the emissions during amalgam removal. The black and white photo represents all elements. Figure 4. Energy dispersive X-ray elemental analysis of the aerosols emitted during amalgam removal. Elements are color coded and labeled by periodic table name. Abbreviations: Ag, silver; Ca, calcium; Cl, chlorine; Cu, copper; Hg, mercury; Sn, tin. Figure 5. Scanning electron microscopy image showing amalgam particles concentrating within dried water droplets during routine amalgam removal. Water droplets can be identified by accumulations of fine aerosol materials in a circular or ovoid pattern.\",\n",
            "        \"discussion\": \"DISCUSSION Our study identified relatively few viable bacteria and no viruses in dental aerosols when DHCPs were applying common aerosol mitigation techniques. In comparison to other studies, our comparatively low yield of bacteria and viruses is notable. Indeed, the number of bacteria we captured based on colony counts is far lower than what one would expect from standard saliva. We suspect that this is because the majority of dental aerosols captured were comprised of small amounts of saliva diluted in large volumes of water from water cooled instruments, such as high-speed drills. Previous work using 16S RNA gene sequencing confirms these findings with saliva contributing to a median of 0% of the total aerosol microbiota [ 11 ]. Our findings demonstrate that a substantial proportion of captured bacteria are from a nonoral source, such as the environmental water and sloughed human skin cells. The particularly low overall yield of our findings suggest that dental aerosols do not pose a substantial microbiological health hazard to DHCPs. These results should be reassuring to DHCPs and patients. Indeed, with the colony counts our team observed, we believe that when standard aerosol mitigation strategies are employed, standard surgical masks should provide adequate protection against bioaerosols for DHCPs during aerosol-generating procedures. To our knowledge, this is the first study to measure the inorganic composition of dental aerosols. Although we used indirect approaches to measure the composition of dental droplets and aerosols for our inorganic analyses, SEM images reveal that metal particles may be aerosolized during high-speed drilling. This is concerning because standard surgical masks may not fully capture finer amalgam particles. However, the patterns of the particles suggest that these pieces of amalgam may collect in larger droplets, which would be more likely to be filtered out with standard personal protective equipment. Furthermore, these results suggest that \\u201cwet\\u201d cutting (using high-speed drills with water cooling) would increase the size and number of droplets and aerosols would thus have a protective effect. Studies evaluating urinary mercury levels among dentists are variable. Recent studies suggest that dentists have similar mercury levels to nondentists [ 16 , 17 ] and may be decreasing over time. In the future, real-time monitoring of amalgam particles could provide detailed information about particles suspended in the air for periods longer than the duration of the dental procedure. We believe that our analyses provides a more realistic assessment for DHCPs than prior studies. To our knowledge, this is the second study to directly evaluate the bacterial composition of dental aerosols [ 18 ] with the use of aerosol mitigation strategies. This is likely to provide a more accurate evaluation of pathogen inhalational risk compared to prior analyses. This is because larger particles, such as splatter and droplets, are likely to fall to the floor or get caught by a face shield or a mask [ 18 ]. Our team's experimental design included live patients, placed aerosol collection devices in close proximity to DHCPs, and used suction rates that approximate human respiration to provide the most accurate assessment of occupational health risks. Indeed, the use of specimen collection in real-world clinical settings for the duration of aerosol-generating procedures allowed us to determine the quantity of bacteria an unmasked DHCP would inhale during a dental procedure. Furthermore, we used multiple aerosol collection methods to confirm our findings. Although our results are similar to those of Tan et al, who found that root canal treatments and dental scaling procedures were associated with an increase in oral bacteria in the air, we observed very few total bacteria compared to their study. This discrepancy is likely because Tan et al used supraphysiologic suction rates to capture dental aerosols in their experimental design [ 18 ]. There are several limitations to our study. First, due to the real-world nature of these data, we were unable to assess the impact of aerosol mitigation strategies on the bacterial, viral, and inorganic chemical composition of dental aerosols. However, our results reflect the true risks for dentists who overwhelmingly use aerosol mitigation approaches during routine care. Second, due to the small sample size and low bacterial/viral yield of our study, we were unable to properly assess if certain dental clinic layouts or aerosol mitigation strategies have a marginally more protective effect than others. However, the overall results of our study suggest that the marginal benefits would likely be low. Third, our bacterial analyses relied on standard culture approaches. This may underestimate the measurement of bacterial materials in the aerosols when compared to 16S ribosomal RNA gene sequencing. However, we believe that this approach provides a more realistic evaluation of viable microorganisms, which better reflects occupational health hazards for DHCPs. Fourth, our relatively small sample size limited our ability to draw firm conclusions on the viral occupational health risks for dentists. It is quite possible that none of our patients had a viral infection during our sampling time. Furthermore, even if one of the patients did, the performance characteristics for the respiratory panel we used are based on nasopharyngeal swabs rather than aerosols. We anticipate that this would reduce the sensitivity of our viral analyses. However, we believe that our bacterial assessments can provide useful surrogate data, as every participant would have viable bacteria in their saliva. The low bioburden of bacteria in the aerosols compared to saliva demonstrates that substantial dilution of saliva is occurring. This indirectly suggests that the same would be the case for viral pathogens present in the saliva. Fifth, our studies were limited to capturing aerosols during individual aerosol-generating procedures. DHCPs will also be exposed to pathogens from unmasked patients who may be coughing or gagging during non-aerosol-generating procedures, which we did not measure during this study. Similarly, these data are evaluating dental aerosols from an oral source. For example, patients with a lower respiratory tract infection, such as tuberculosis, may still be able to transmit an infection via standard respiration. However, we would assume that patients with common acute lower respiratory tract infections, such as pneumonia would defer dental care while acutely ill. Sixth, this study was performed when the amount of circulating severe acute respiratory syndrome coronavirus 2 infections were lower in the community. This may have reduced the likelihood of capturing an asymptomatic patient with coronavirus disease 2019. Finally, the bacterial estimates presented here are likely a substantial overestimate of the procedure specific bacterial and viral exposures DHCPs would receive, as our experimental design used an unfiltered tube to mimic normal human respiration, when DHCPs would be wearing a mask and/or face shield during aerosol-generating procedures.\",\n",
            "        \"conclusion\": \"CONCLUSIONS To our knowledge, we have provided the most detailed characterization and accurate picture of the true risks of aerosols among dental healthcare providers to date. The bacterial composition of dental aerosols appears to be extremely low during routine dental procedures. Our findings suggest that commonly employed aerosol mitigation interventions and standard personal protective equipment likely allows for the safe practice of dentistry, regardless of operatory layout or procedure type. These findings suggest that community leaders in infectious diseases and infection control should not advocate for substantial changes in community dental clinic design or ventilation systems. However, due to the small sample size of our study, additional studies confirming these findings for patients with known viral infections would be beneficial.\"\n",
            "    }\n",
            "]\n"
          ]
        }
      ]
    },
    {
      "cell_type": "markdown",
      "source": [
        "## Setting up LLM and Prompting"
      ],
      "metadata": {
        "id": "AMAMtEI0M8oM"
      }
    },
    {
      "cell_type": "markdown",
      "source": [
        "I decided to use an Instruct model to do the data extraction pipeline. However, a NER pipline could also be used to complete this task"
      ],
      "metadata": {
        "id": "dLRc-UmNEfmV"
      }
    },
    {
      "cell_type": "code",
      "source": [
        "import torch\n",
        "from transformers import pipeline\n",
        "\n",
        "# Set Device to CPU if GPU is out of RAM\n",
        "\n",
        "# model_id = \"meta-llama/Meta-Llama-3-8B-Instruct\"   # Uses more GPU RAM\n",
        "model_id = \"meta-llama/Llama-3.2-3B-Instruct\"        # Uses less GPU RAM\n",
        "# model_id = \"meta-llama/Llama-3.2-1B-Instruct\"        # Uses the least GPU RAM\n",
        "pipe = pipeline(\n",
        "    \"text-generation\",\n",
        "    model=model_id,\n",
        "    torch_dtype=torch.bfloat16,\n",
        "    device_map='auto',\n",
        ")"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 67,
          "referenced_widgets": [
            "cdae6812242242ddb245ddf6f12e9a8a",
            "5a9781b987fe4b47b6f3587e75446e24",
            "f315e6ffd6f7479f9b413414b3aa66a6",
            "b0bc553bdbf44bed8ee7a4b487451b4b",
            "56fdedbfd10a4b59a095308407366643",
            "554a89277d3043d68d9b06fa38245c24",
            "35a600674e0f446eb9b353c2c6481581",
            "e8080aeb4ec9462e94dc9b7035a9bf04",
            "0165c1007b224108ad38f6d7408a4036",
            "0bab08a1f34a477a9022f4da601bd177",
            "a2952d2dc2c34d3a8810053c96466b1f"
          ]
        },
        "id": "Zg2r5DBJYHp9",
        "outputId": "5c76bfb8-8c76-4ae4-e362-e78284db6ddd"
      },
      "execution_count": 25,
      "outputs": [
        {
          "output_type": "display_data",
          "data": {
            "text/plain": [
              "Loading checkpoint shards:   0%|          | 0/2 [00:00<?, ?it/s]"
            ],
            "application/vnd.jupyter.widget-view+json": {
              "version_major": 2,
              "version_minor": 0,
              "model_id": "cdae6812242242ddb245ddf6f12e9a8a"
            }
          },
          "metadata": {}
        },
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "Device set to use cuda:0\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "# Function to process a single article and extract relevant information\n",
        "def extract_information_from_article(article):\n",
        "    # Combine relevant fields into a single text input\n",
        "    article_content = (\n",
        "        f\"Abstract: {article.get('abstract', 'Not available')}\\n\"\n",
        "        f\"Introduction: {article.get('intro', 'Not available')}\\n\"\n",
        "        f\"Methods: {article.get('method', 'Not available')}\\n\"\n",
        "        f\"Subjects: {article.get('subjects', 'Not available')}\\n\"\n",
        "        f\"Results: {article.get('results', 'Not available')}\\n\"\n",
        "        f\"Discussion: {article.get('discussion', 'Not available')}\\n\"\n",
        "        f\"Conclusion: {article.get('conclusion', 'Not available')}\\n\"\n",
        "    )\n",
        "\n",
        "    # Define the prompt for extracting information\n",
        "    # Prompt Engineering Reasoning:\n",
        "    # After experimenting with multiple prompting approaches, this approach consistently produced the best results.\n",
        "    #\n",
        "    # Key Design Choices:\n",
        "    # 1. Role Specification: Clearly defining the LLM's role as an \"assistant trained to extract structured data\"\n",
        "    #    improves accuracy and consistency in extraction.\n",
        "    # 2. JSON Format Enforcement: The output is explicitly structured in JSON, ensuring machine-readable and\n",
        "    #    structured responses. If any information is missing, it is assigned a `null` value to maintain consistency.\n",
        "    # 3. Field Definition: Each field is clearly specified, detailing the type of information expected.\n",
        "    # 4. Flexibility in Study Design: Initially, a stricter JSON structure was enforced, but due to variability in\n",
        "    #    research methodologies (e.g., some studies not having an experimental-control group setup), the prompt was\n",
        "    #    adjusted to allow for different study designs. This ensures broader applicability across various scientific articles.\n",
        "    # 5. Changes to patient origin: Seperated categories into study site and patient demographics to provide more accurate information.\n",
        "\n",
        "    system_prompt = f\"\"\"\n",
        "You are an assistant trained to extract structured data from the scientific article provided.\n",
        "Extract the following information fields and return the output in JSON. If some information is missing, you respond with null.\n",
        "1. disease_condition: Studied disease condition, health condition, disorder, syndrome, etc.\n",
        "2. total_patients: Total number of patients\n",
        "3. case_group_patients: Number of patients in experimental groups\n",
        "4. control_group_patients: Number of patients in control groups\n",
        "5. patient_sex: Sex of the patients\n",
        "7. patient_demographics: demographics of the patients\n",
        "8. study_site: location or place or hospital of study conducted\n",
        "\"\"\"\n",
        "\n",
        "    user_prompt = f\"\"\" Scientific article:\n",
        "{article_content}\n",
        "\"\"\"\n",
        "    # Configure the messages for the LLM\n",
        "    messages = [\n",
        "        {\"role\": \"system\", \"content\": system_prompt},\n",
        "        {\"role\": \"user\", \"content\": user_prompt},\n",
        "    ]\n",
        "\n",
        "    outputs = pipe(\n",
        "    messages,\n",
        "    max_new_tokens=500,\n",
        ")\n",
        "    return outputs[0][\"generated_text\"][-1]['content']"
      ],
      "metadata": {
        "id": "7E0JhYpxXdsN"
      },
      "execution_count": 26,
      "outputs": []
    },
    {
      "cell_type": "markdown",
      "source": [
        "## Processes the articles and saves extracted information"
      ],
      "metadata": {
        "id": "E6EhRT7eOAPE"
      }
    },
    {
      "cell_type": "markdown",
      "source": [
        "This is to provide the output in a similar JSON format as the original data provided."
      ],
      "metadata": {
        "id": "RlV2IfV4Df0U"
      }
    },
    {
      "cell_type": "code",
      "source": [
        "# Formats extracted information into the correct format and return it in json format\n",
        "def format_json(pmcid, content):\n",
        "\n",
        "  extracted_info = {}\n",
        "  assistant_content = content\n",
        "\n",
        "  # Look for the part of the string between the triple backticks\n",
        "  if \"```\" in assistant_content:\n",
        "      # Split based on the '```' markers\n",
        "      parts = assistant_content.split('```')\n",
        "      if len(parts) > 1:\n",
        "          # The second part should be the JSON content between the backticks\n",
        "          json_content = parts[1].strip()  # Strip out unnecessary spaces/newlines\n",
        "          try:\n",
        "              # Parse the JSON content\n",
        "              extracted_json = json.loads(json_content)\n",
        "              extracted_info = extracted_json  # Store the extracted content\n",
        "          except json.JSONDecodeError:\n",
        "              # print(f\"Error decoding JSON for pmcid: {pmcid}\")\n",
        "              return 0, None\n",
        "\n",
        "  return 1, {'pmcid':pmcid, 'extracted_info':extracted_info}"
      ],
      "metadata": {
        "id": "XaXXKZZoCogA"
      },
      "execution_count": 27,
      "outputs": []
    },
    {
      "cell_type": "markdown",
      "source": [
        "This code processes a dataset of articles by extracting relevant information from each one, validating the extracted data, and retrying up to five times if errors are found. Successfully processed articles are stored in a results list, while errors are logged. Finally, the extracted information is saved as a JSON file at a specified location."
      ],
      "metadata": {
        "id": "tN8z_1uLO-r6"
      }
    },
    {
      "cell_type": "code",
      "source": [
        "import json\n",
        "from pathlib import Path\n",
        "\n",
        "# Process all articles in the dataset\n",
        "results = []\n",
        "\n",
        "for article in filtered_articles:\n",
        "    attempts = 0\n",
        "    content = extract_information_from_article(article)\n",
        "\n",
        "    # Checks for errors and sends back to LLM to reprocess if error is found\n",
        "    while attempts < 5:\n",
        "        check, formatted_extracted_info = format_json(article['pmcid'], content)\n",
        "        if check == 1:\n",
        "            results.append(formatted_extracted_info)\n",
        "            print(f\"PROCESSED article with PMCID: {article['pmcid']}\")\n",
        "            break\n",
        "        # Re-extract content only if necessary\n",
        "        content = extract_information_from_article(article)\n",
        "        attempts += 1\n",
        "    else:\n",
        "        print(f\"ERROR processing article with PMCID: {article['pmcid']}\\nCONTENT:{content}\\n\")\n",
        "\n",
        "# Save the results to a JSON file\n",
        "output_path = Path(\"/content/recruitment_test/data/extracted_info.json\")\n",
        "output_path.write_text(json.dumps(results, indent=4))\n",
        "\n",
        "print(f\"Extraction complete. Results saved to {output_path}.\")\n"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "collapsed": true,
        "id": "Hpomkm8vmKKQ",
        "outputId": "0257d5d7-85ab-47a1-fa04-e89f708c81d7"
      },
      "execution_count": 28,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "Setting `pad_token_id` to `eos_token_id`:128001 for open-end generation.\n",
            "Setting `pad_token_id` to `eos_token_id`:128001 for open-end generation.\n",
            "Setting `pad_token_id` to `eos_token_id`:128001 for open-end generation.\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "PROCESSED article with PMCID: PMC10167034\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "Setting `pad_token_id` to `eos_token_id`:128001 for open-end generation.\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "PROCESSED article with PMCID: PMC10191296\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "Setting `pad_token_id` to `eos_token_id`:128001 for open-end generation.\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "PROCESSED article with PMCID: PMC10390885\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "Setting `pad_token_id` to `eos_token_id`:128001 for open-end generation.\n",
            "Setting `pad_token_id` to `eos_token_id`:128001 for open-end generation.\n",
            "Setting `pad_token_id` to `eos_token_id`:128001 for open-end generation.\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "PROCESSED article with PMCID: PMC2383879\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "Setting `pad_token_id` to `eos_token_id`:128001 for open-end generation.\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "PROCESSED article with PMCID: PMC2693442\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "Setting `pad_token_id` to `eos_token_id`:128001 for open-end generation.\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "PROCESSED article with PMCID: PMC3145824\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "Setting `pad_token_id` to `eos_token_id`:128001 for open-end generation.\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "PROCESSED article with PMCID: PMC3174812\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "Setting `pad_token_id` to `eos_token_id`:128001 for open-end generation.\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "PROCESSED article with PMCID: PMC3212907\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "Setting `pad_token_id` to `eos_token_id`:128001 for open-end generation.\n",
            "Setting `pad_token_id` to `eos_token_id`:128001 for open-end generation.\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "PROCESSED article with PMCID: PMC3534646\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "Setting `pad_token_id` to `eos_token_id`:128001 for open-end generation.\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "PROCESSED article with PMCID: PMC3909226\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "Setting `pad_token_id` to `eos_token_id`:128001 for open-end generation.\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "PROCESSED article with PMCID: PMC4393161\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "Setting `pad_token_id` to `eos_token_id`:128001 for open-end generation.\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "PROCESSED article with PMCID: PMC5040013\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "Setting `pad_token_id` to `eos_token_id`:128001 for open-end generation.\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "PROCESSED article with PMCID: PMC5425199\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "Setting `pad_token_id` to `eos_token_id`:128001 for open-end generation.\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "PROCESSED article with PMCID: PMC5441889\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "Setting `pad_token_id` to `eos_token_id`:128001 for open-end generation.\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "PROCESSED article with PMCID: PMC5601641\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "Setting `pad_token_id` to `eos_token_id`:128001 for open-end generation.\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "PROCESSED article with PMCID: PMC5717332\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "Setting `pad_token_id` to `eos_token_id`:128001 for open-end generation.\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "PROCESSED article with PMCID: PMC5839230\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "Setting `pad_token_id` to `eos_token_id`:128001 for open-end generation.\n",
            "Setting `pad_token_id` to `eos_token_id`:128001 for open-end generation.\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "PROCESSED article with PMCID: PMC6060212\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "Setting `pad_token_id` to `eos_token_id`:128001 for open-end generation.\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "PROCESSED article with PMCID: PMC6286024\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "Setting `pad_token_id` to `eos_token_id`:128001 for open-end generation.\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "PROCESSED article with PMCID: PMC6730009\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "Setting `pad_token_id` to `eos_token_id`:128001 for open-end generation.\n",
            "Setting `pad_token_id` to `eos_token_id`:128001 for open-end generation.\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "PROCESSED article with PMCID: PMC6760014\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "Setting `pad_token_id` to `eos_token_id`:128001 for open-end generation.\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "PROCESSED article with PMCID: PMC6775309\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "Setting `pad_token_id` to `eos_token_id`:128001 for open-end generation.\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "PROCESSED article with PMCID: PMC6853912\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "Setting `pad_token_id` to `eos_token_id`:128001 for open-end generation.\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "PROCESSED article with PMCID: PMC6955584\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "Setting `pad_token_id` to `eos_token_id`:128001 for open-end generation.\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "PROCESSED article with PMCID: PMC7007877\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "Setting `pad_token_id` to `eos_token_id`:128001 for open-end generation.\n",
            "Setting `pad_token_id` to `eos_token_id`:128001 for open-end generation.\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "PROCESSED article with PMCID: PMC7449478\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "Setting `pad_token_id` to `eos_token_id`:128001 for open-end generation.\n",
            "Setting `pad_token_id` to `eos_token_id`:128001 for open-end generation.\n",
            "Setting `pad_token_id` to `eos_token_id`:128001 for open-end generation.\n",
            "Setting `pad_token_id` to `eos_token_id`:128001 for open-end generation.\n",
            "Setting `pad_token_id` to `eos_token_id`:128001 for open-end generation.\n",
            "Setting `pad_token_id` to `eos_token_id`:128001 for open-end generation.\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "ERROR processing article with PMCID: PMC7657407\n",
            "CONTENT:Here is the extracted information in JSON format:\n",
            "\n",
            "```\n",
            "{\n",
            "  \"disease_condition\": \"Infertility\",\n",
            "  \"total_patients\": 108,\n",
            "  \"case_group_patients\": 54,\n",
            "  \"control_group_patients\": 54,\n",
            "  \"patient_sex\": {\n",
            "    \"male\": 54,\n",
            "    \"female\": 54\n",
            "  },\n",
            "  \"patient_demographics\": {\n",
            "    \"age\": {\n",
            "      \"male\": 33.3 (5.5),\n",
            "      \"female\": 31.5 (5.3)\n",
            "    },\n",
            "    \"weight\": {\n",
            "      \"male\": 84.9 (13.0),\n",
            "      \"female\": 70.0 (12.5)\n",
            "    },\n",
            "    \"duration_of_marriage\": {\n",
            "      \"male\": 5.0 (1.3),\n",
            "      \"female\": 5.0 (2.2)\n",
            "    },\n",
            "    \"job\": {\n",
            "      \"male\": {\n",
            "        \"unemployed\": 2.0 (7.4),\n",
            "        \"worker\": 3.0 (11.1),\n",
            "        \"employee\": 5.0 (18.5),\n",
            "        \"self-employment\": 17.0 (63.0)\n",
            "      },\n",
            "      \"female\": {\n",
            "        \"homemaker\": 19.0 (70.4),\n",
            "        \"employed\": 8.0 (29.6),\n",
            "        \"unemployed\": 0,\n",
            "        \"worker\": 3.0 (11.1),\n",
            "        \"self-employment\": 15.0 (55.6)\n",
            "      }\n",
            "    },\n",
            "    \"income_level\": {\n",
            "      \"male\": {\n",
            "        \"at a sufficient level\": 16.0 (29.6),\n",
            "        \"too much\": 9.0 (16.7),\n",
            "        \"less than full\": 29.0 (53.7)\n",
            "      },\n",
            "      \"female\": {\n",
            "        \"at a sufficient level\": 16.0 (29.6),\n",
            "        \"too much\": 9.0 (16.7),\n",
            "        \"less than full\": 30.0 (55.6)\n",
            "      }\n",
            "    },\n",
            "    \"education\": {\n",
            "      \"male\": {\n",
            "        \"elementary\": 7.0 (25.9),\n",
            "        \"secondary\": 7.0 (25.9),\n",
            "        \"diploma\": 8.0 (29.6),\n",
            "        \"bachelor\": 4.0 (14.8),\n",
            "       \n",
            "\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "Setting `pad_token_id` to `eos_token_id`:128001 for open-end generation.\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "PROCESSED article with PMCID: PMC7661891\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "Setting `pad_token_id` to `eos_token_id`:128001 for open-end generation.\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "PROCESSED article with PMCID: PMC8288503\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "Setting `pad_token_id` to `eos_token_id`:128001 for open-end generation.\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "PROCESSED article with PMCID: PMC8382172\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "Setting `pad_token_id` to `eos_token_id`:128001 for open-end generation.\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "PROCESSED article with PMCID: PMC8976245\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "Setting `pad_token_id` to `eos_token_id`:128001 for open-end generation.\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "PROCESSED article with PMCID: PMC9333080\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "Setting `pad_token_id` to `eos_token_id`:128001 for open-end generation.\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "PROCESSED article with PMCID: PMC9381901\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "Setting `pad_token_id` to `eos_token_id`:128001 for open-end generation.\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "PROCESSED article with PMCID: PMC9529122\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "Setting `pad_token_id` to `eos_token_id`:128001 for open-end generation.\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "PROCESSED article with PMCID: PMC9616492\n",
            "PROCESSED article with PMCID: PMC9697589\n",
            "Extraction complete. Results saved to /content/recruitment_test/data/extracted_info.json.\n"
          ]
        }
      ]
    }
  ]
}